{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "ef39665b",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "418b2277",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"pubmed-crisprcasn-set.txt\", errors=\"ignore\") as l: \n",
    "    my_text = l.read()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "91150688",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "898883"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(my_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "5fc00898",
   "metadata": {},
   "outputs": [],
   "source": [
    "my_text = re.sub(r'\\n\\s{6}', ' ', my_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "0e9e2012",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PMID- 33571114\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "LR  - 20210214\n",
      "IS  - 2375-2548 (Electronic)\n",
      "IS  - 2375-2548 (Linking)\n",
      "VI  - 7\n",
      "IP  - 5\n",
      "DP  - 2021 Jan\n",
      "TI  - A CRISPR-Cas autocatalysis-driven feedback amplification network for supersensitive  DNA diagnostics.\n",
      "LID - 10.1126/sciadv.abc7802 [doi]\n",
      "LID - eabc7802\n",
      "AB  - Artificial nucleic acid circuits with precisely controllable dynamic and function  have shown great promise in biosensing, but their utility in molecular diagnostics  is still restrained by the inability to process genomic DNA directly and moderate  sensitivity. To address this limitation, we present a CRISPR-Cas-powered catalytic  nucleic acid circuit, namely, CRISPR-Cas-only amplification network (CONAN), for  isothermally amplified detection of genomic DNA. By integrating the stringent target  recognition, helicase activity, and trans-cleavage activity of Cas12a, a Cas12a  autocatalysis-driven artificial reaction network is programmed to construct a  positive feedback circuit with exponential dynamic in CONAN. Consequently, CONAN  achieves one-enzyme, one-step, real-time detection of genomic DNA with attomolar  sensitivity. Moreover, CONAN increases the intrinsic single-base specificity of  Cas12a, and enables the effective detection of hepatitis B virus infection and human  bladder cancer-associated single-nucleotide mutation in clinical samples,  highlighting its potential as a powerful tool for disease diagnostics.\n",
      "CI  - Copyright Â© 2021 The Authors, some rights reserved; exclusive licensee American  Association for the Advancement of Science. No claim to original U.S. Government  Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0  (CC BY-NC).\n",
      "FAU - Shi, Kai\n",
      "AU  - Shi K\n",
      "AUID- ORCID: 0000-0002-9181-5664\n",
      "AD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R\n"
     ]
    }
   ],
   "source": [
    "print(my_text[:2000])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "6508d639",
   "metadata": {},
   "outputs": [],
   "source": [
    " x = re.split('^AD+[\\D]+[.=$]', my_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "f8b04b6d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['PMID- 33571114\\nOWN - NLM\\nSTAT- In-Data-Review\\nLR  - 20210214\\nIS  - 2375-2548 (Electronic)\\nIS  - 2375-2548 (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2021 Jan\\nTI  - A CRISPR-Cas autocatalysis-driven feedback amplification network for supersensitive  DNA diagnostics.\\nLID - 10.1126/sciadv.abc7802 [doi]\\nLID - eabc7802\\nAB  - Artificial nucleic acid circuits with precisely controllable dynamic and function  have shown great promise in biosensing, but their utility in molecular diagnostics  is still restrained by the inability to process genomic DNA directly and moderate  sensitivity. To address this limitation, we present a CRISPR-Cas-powered catalytic  nucleic acid circuit, namely, CRISPR-Cas-only amplification network (CONAN), for  isothermally amplified detection of genomic DNA. By integrating the stringent target  recognition, helicase activity, and trans-cleavage activity of Cas12a, a Cas12a  autocatalysis-driven artificial reaction network is programmed to construct a  positive feedback circuit with exponential dynamic in CONAN. Consequently, CONAN  achieves one-enzyme, one-step, real-time detection of genomic DNA with attomolar  sensitivity. Moreover, CONAN increases the intrinsic single-base specificity of  Cas12a, and enables the effective detection of hepatitis B virus infection and human  bladder cancer-associated single-nucleotide mutation in clinical samples,  highlighting its potential as a powerful tool for disease diagnostics.\\nCI  - Copyright Â© 2021 The Authors, some rights reserved; exclusive licensee American  Association for the Advancement of Science. No claim to original U.S. Government  Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0  (CC BY-NC).\\nFAU - Shi, Kai\\nAU  - Shi K\\nAUID- ORCID: 0000-0002-9181-5664\\nAD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R. China.\\nFAU - Xie, Shiyi\\nAU  - Xie S\\nAD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R. China.\\nFAU - Tian, Renyun\\nAU  - Tian R\\nAUID- ORCID: 0000-0003-1566-1269\\nAD  - Institute of Pathogen Biology and Immunology of College of Biology, State Key  Laboratory of Chemo/Biosensing and Chemometrics, Hunan University, Changsha 410082,  China.\\nFAU - Wang, Shuo\\nAU  - Wang S\\nAD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R. China.\\nFAU - Lu, Qin\\nAU  - Lu Q\\nAD  - Hunan Research Center for Big Data Application in Genomics, Genetalks Inc., Changsha  410152, China.\\nFAU - Gao, Denghui\\nAU  - Gao D\\nAD  - Third Hospital of Hebei Medical University, Shijiazhuang 050051, China.\\nFAU - Lei, Chunyang\\nAU  - Lei C\\nAUID- ORCID: 0000-0002-3078-5553\\nAD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R. China.\\nFAU - Zhu, Haizhen\\nAU  - Zhu H\\nAD  - Institute of Pathogen Biology and Immunology of College of Biology, State Key  Laboratory of Chemo/Biosensing and Chemometrics, Hunan University, Changsha 410082,  China.\\nFAU - Nie, Zhou\\nAU  - Nie Z\\nAUID- ORCID: 0000-0001-9864-2965\\nAD  - State Key Laboratory of Chemo/Bio-Sensing and Chemometrics, College of Chemistry and  Chemical Engineering, Hunan Provincial Key Laboratory of Biomacromolecular Chemical  Biology, Hunan University, Changsha 410082, P. R. China. niezhou.hnu@gmail.com.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210127\\nTA  - Sci Adv\\nJT  - Science advances\\nJID - 101653440\\nSB  - IM\\nPMC - PMC7840123\\nEDAT- 2021/02/12 06:00\\nMHDA- 2021/02/12 06:00\\nCRDT- 2021/02/11 17:13\\nPHST- 2020/05/16 00:00 [received]\\nPHST- 2020/12/10 00:00 [accepted]\\nPHST- 2021/02/11 17:13 [entrez]\\nPHST- 2021/02/12 06:00 [pubmed]\\nPHST- 2021/02/12 06:00 [medline]\\nAID - 7/5/eabc7802 [pii]\\nAID - abc7802 [pii]\\nAID - 10.1126/sciadv.abc7802 [doi]\\nPST - epublish\\nSO  - Sci Adv. 2021 Jan 27;7(5):eabc7802. doi: 10.1126/sciadv.abc7802. Print 2021 Jan.\\n\\nPMID- 31759123\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210930\\nLR  - 20210930\\nIS  - 1872-8294 (Electronic)\\nIS  - 0169-409X (Linking)\\nVI  - 168\\nDP  - 2021 Jan\\nTI  - Rational designs of in vivo CRISPR-Cas delivery systems.\\nPG  - 3-29\\nLID - S0169-409X(19)30208-X [pii]\\nLID - 10.1016/j.addr.2019.11.005 [doi]\\nAB  - The CRISPR-Cas system initiated a revolution in genome editing when it was, for the  first time, demonstrated success in the mammalian cells. Today, scientists are able  to readily edit genomes, regulate gene transcription, engineer posttranscriptional  events, and image nucleic acids using CRISPR-Cas-based tools. However, to  efficiently transport CRISPR-Cas into target tissues/cells remains challenging due  to many extra- and intra-cellular barriers, therefore largely limiting the  applications of CRISPR-based therapeutics in vivo. In this review, we summarize the  features of plasmid-, RNA- and ribonucleoprotein (RNP)-based CRISPR-Cas  therapeutics. Then, we survey the current in vivo delivery systems. We specify the  requirements for efficient in vivo delivery in clinical settings, and highlight both  efficiency and safety for different CRISPR-Cas tools.\\nCI  - Copyright Â© 2019. Published by Elsevier B.V.\\nFAU - Xu, Cong-Fei\\nAU  - Xu CF\\nAD  - Guangzhou First People\\'s Hospital, School of Biomedical Sciences and Engineering,  Guangzhou International Campus, South China University of Technology, Guangzhou  510006, PR China.\\nFAU - Chen, Guo-Jun\\nAU  - Chen GJ\\nAD  - Department of Bioengineering, California Nanosystems Institute, University of  California, Los Angeles 90095, USA; Jonsson Comprehensive Cancer Center, California  NanoSystems Institute, Center for Minimally Invasive Therapeutics, University of  California, Los Angeles 90095, USA.\\nFAU - Luo, Ying-Li\\nAU  - Luo YL\\nAD  - Institutes for Life Sciences, School of Medicine, South China University of  Technology, Guangzhou 510006, PR China.\\nFAU - Zhang, Yue\\nAU  - Zhang Y\\nAD  - Guangzhou First People\\'s Hospital, School of Biomedical Sciences and Engineering,  Guangzhou International Campus, South China University of Technology, Guangzhou  510006, PR China; Key Laboratory of Biomedical Engineering of Guangdong Province,  and Innovation Center for Tissue Restoration and Reconstruction, South China  University of Technology, Guangzhou 510006, PR China.\\nFAU - Zhao, Gui\\nAU  - Zhao G\\nAD  - Guangzhou First People\\'s Hospital, School of Biomedical Sciences and Engineering,  Guangzhou International Campus, South China University of Technology, Guangzhou  510006, PR China; Key Laboratory of Biomedical Materials and Engineering of the  Ministry of Education, South China University of Technology, Guangzhou 510006, PR  China.\\nFAU - Lu, Zi-Dong\\nAU  - Lu ZD\\nAD  - Institutes for Life Sciences, School of Medicine, South China University of  Technology, Guangzhou 510006, PR China.\\nFAU - Czarna, Anna\\nAU  - Czarna A\\nAD  - Institutes for Life Sciences, School of Medicine, South China University of  Technology, Guangzhou 510006, PR China.\\nFAU - Gu, Zhen\\nAU  - Gu Z\\nAD  - Department of Bioengineering, California Nanosystems Institute, University of  California, Los Angeles 90095, USA; Jonsson Comprehensive Cancer Center, California  NanoSystems Institute, Center for Minimally Invasive Therapeutics, University of  California, Los Angeles 90095, USA. Electronic address: guzhen@ucla.edu.\\nFAU - Wang, Jun\\nAU  - Wang J\\nAD  - Guangzhou First People\\'s Hospital, School of Biomedical Sciences and Engineering,  Guangzhou International Campus, South China University of Technology, Guangzhou  510006, PR China; National Engineering Research Center for Tissue Restoration and  Reconstruction, South China University of Technology, Guangzhou 510006, PR China;  Key Laboratory of Biomedical Engineering of Guangdong Province, and Innovation  Center for Tissue Restoration and Reconstruction, South China University of  Technology, Guangzhou 510006, PR China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, Guangzhou 510005, PR China. Electronic address:  mcjwang@scut.edu.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20191121\\nPL  - Netherlands\\nTA  - Adv Drug Deliv Rev\\nJT  - Advanced drug delivery reviews\\nJID - 8710523\\nRN  - 0 (Lipids)\\nRN  - 63231-63-0 (RNA)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Drug Delivery Systems\\nMH  - Epigenome/genetics\\nMH  - Exosomes/metabolism\\nMH  - Gene Editing/*methods\\nMH  - Gene Regulatory Networks/physiology\\nMH  - Genetic Vectors/metabolism\\nMH  - Lipids/chemistry\\nMH  - Nanoparticles/chemistry\\nMH  - RNA/metabolism\\nMH  - Transcription, Genetic/physiology\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *Drug delivery\\nOT  - *Genome editing\\nOT  - *Nanomedicine\\nEDAT- 2019/11/24 06:00\\nMHDA- 2021/10/01 06:00\\nCRDT- 2019/11/24 06:00\\nPHST- 2019/10/14 00:00 [received]\\nPHST- 2019/11/09 00:00 [revised]\\nPHST- 2019/11/19 00:00 [accepted]\\nPHST- 2019/11/24 06:00 [pubmed]\\nPHST- 2021/10/01 06:00 [medline]\\nPHST- 2019/11/24 06:00 [entrez]\\nAID - S0169-409X(19)30208-X [pii]\\nAID - 10.1016/j.addr.2019.11.005 [doi]\\nPST - ppublish\\nSO  - Adv Drug Deliv Rev. 2021 Jan;168:3-29. doi: 10.1016/j.addr.2019.11.005. Epub 2019  Nov 21.\\n\\nPMID- 31519332\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210607\\nLR  - 20210607\\nIS  - 1878-4380 (Electronic)\\nIS  - 0966-842X (Linking)\\nVI  - 28\\nIP  - 1\\nDP  - 2020 Jan\\nTI  - CRISPR-Cas, DNA Supercoiling, and Nucleoid-Associated Proteins.\\nPG  - 19-27\\nLID - S0966-842X(19)30217-3 [pii]\\nLID - 10.1016/j.tim.2019.08.004 [doi]\\nAB  - In this opinion article we highlight links between the H-NS nucleoid-associated  protein, variable DNA topology, the regulation of CRISPR-cas locus expression,  CRISPR-Cas activity, and the recruitment of novel genetic information by the CRISPR  array. We propose that the requirement that the invading mobile genetic element be  negatively supercoiled limits effective CRISPR action to a window in the bacterial  growth cycle when DNA topology is optimal, and that this same window is used for the  efficient integration of new spacer sequences at the CRISPR array. H-NS silences  CRISPR promoters, and we propose that antagonists of H-NS, such as the LeuO  transcription factor, provide a basis for a stochastic genetic switch that acts at  random in each cell in the bacterial population. In addition, we wish to propose a  mechanism by which mobile genetic elements can suppress CRISPR-cas transcription  using H-NS homologues. Although the individual components of this network are known,  we propose a new model in which they are integrated and linked to the physiological  state of the bacterium. The model provides a basis for cell-to-cell variation in the  expression and performance of CRISPR systems in bacterial populations.\\nCI  - Copyright Â© 2019. Published by Elsevier Ltd.\\nFAU - Dorman, Charles J\\nAU  - Dorman CJ\\nAD  - Department of Microbiology, Trinity College Dublin, Dublin 2, Ireland. Electronic  address: cjdorman@tcd.ie.\\nFAU - NÃ\\xad Bhriain, Niamh\\nAU  - NÃ\\xad Bhriain N\\nAD  - Department of Microbiology, Trinity College Dublin, Dublin 2, Ireland.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20190910\\nPL  - England\\nTA  - Trends Microbiol\\nJT  - Trends in microbiology\\nJID - 9310916\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (Integration Host Factors)\\nRN  - 0 (Transcription Factors)\\nRN  - 9007-49-2 (DNA)\\nSB  - IM\\nMH  - Bacteria/*genetics\\nMH  - Bacterial Physiological Phenomena\\nMH  - Bacterial Proteins/*genetics/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - *Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - *DNA\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - Gene Transfer, Horizontal\\nMH  - Immunity\\nMH  - Integration Host Factors\\nMH  - Interspersed Repetitive Sequences\\nMH  - Promoter Regions, Genetic\\nMH  - Transcription Factors\\nMH  - Transcriptome\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *DNA supercoiling\\nOT  - *H-NS\\nOT  - *LeuO\\nOT  - *horizontal gene transfer\\nOT  - *integration host factor\\nEDAT- 2019/09/15 06:00\\nMHDA- 2021/06/08 06:00\\nCRDT- 2019/09/15 06:00\\nPHST- 2019/06/05 00:00 [received]\\nPHST- 2019/08/02 00:00 [revised]\\nPHST- 2019/08/08 00:00 [accepted]\\nPHST- 2019/09/15 06:00 [pubmed]\\nPHST- 2021/06/08 06:00 [medline]\\nPHST- 2019/09/15 06:00 [entrez]\\nAID - S0966-842X(19)30217-3 [pii]\\nAID - 10.1016/j.tim.2019.08.004 [doi]\\nPST - ppublish\\nSO  - Trends Microbiol. 2020 Jan;28(1):19-27. doi: 10.1016/j.tim.2019.08.004. Epub 2019  Sep 10.\\n\\nPMID- 31136835\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200402\\nLR  - 20200402\\nIS  - 1879-0402 (Electronic)\\nIS  - 1367-5931 (Linking)\\nVI  - 52\\nDP  - 2019 Oct\\nTI  - CRISPR/Cas-based devices for mammalian synthetic biology.\\nPG  - 23-30\\nLID - S1367-5931(19)30024-9 [pii]\\nLID - 10.1016/j.cbpa.2019.04.015 [doi]\\nAB  - Since its first demonstration for mammalian gene editing, CRISPR/Cas technology has  been widely adopted in research, industry, and medicine. Beyond indel mutations  induced by Cas9 activity, recent advances in CRISPR/Cas have enabled DNA or RNA base  editing. In addition, multiple orthogonal methods for the spatiotemporal regulation  of CRISPR/Cas activity and repurposed Cas proteins for the visualization and  relocation of specific genomic loci in living cells have been described. By  harnessing the versatility of CRISPR/Cas-based devices and gene circuits, synthetic  biologists are developing memory devices for lineage tracing and technologies for  unbiased, high-throughput interrogation of combinatorial gene perturbations. We  envision that such approaches will enable researchers to gain deeper insights into  the translation of genotypes to phenotypes in healthy and diseased states.\\nCI  - Copyright Â© 2019. Published by Elsevier Ltd.\\nFAU - Kim, Tackhoon\\nAU  - Kim T\\nAD  - Research Laboratory of Electronics, Massachusetts Institute of Technology, 77  Massachusetts Avenue, Cambridge, MA 02139, USA.\\nFAU - Lu, Timothy K\\nAU  - Lu TK\\nAD  - Research Laboratory of Electronics, Massachusetts Institute of Technology, 77  Massachusetts Avenue, Cambridge, MA 02139, USA. Electronic address: timlu@mit.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20190525\\nPL  - England\\nTA  - Curr Opin Chem Biol\\nJT  - Current opinion in chemical biology\\nJID - 9811312\\nRN  - 9007-49-2 (DNA)\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems\\nMH  - DNA/genetics\\nMH  - Gene Regulatory Networks\\nMH  - *Mammals\\nMH  - RNA Editing\\nMH  - *Synthetic Biology\\nEDAT- 2019/05/29 06:00\\nMHDA- 2020/04/03 06:00\\nCRDT- 2019/05/29 06:00\\nPHST- 2019/02/04 00:00 [received]\\nPHST- 2019/04/13 00:00 [revised]\\nPHST- 2019/04/16 00:00 [accepted]\\nPHST- 2019/05/29 06:00 [pubmed]\\nPHST- 2020/04/03 06:00 [medline]\\nPHST- 2019/05/29 06:00 [entrez]\\nAID - S1367-5931(19)30024-9 [pii]\\nAID - 10.1016/j.cbpa.2019.04.015 [doi]\\nPST - ppublish\\nSO  - Curr Opin Chem Biol. 2019 Oct;52:23-30. doi: 10.1016/j.cbpa.2019.04.015. Epub 2019  May 25.\\n\\nPMID- 32664329\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210412\\nLR  - 20210412\\nIS  - 2073-4409 (Electronic)\\nIS  - 2073-4409 (Linking)\\nVI  - 9\\nIP  - 7\\nDP  - 2020 Jul 10\\nTI  - DNA Damage: From Threat to Treatment.\\nLID - 10.3390/cells9071665 [doi]\\nLID - 1665\\nAB  - DNA is the source of genetic information, and preserving its integrity is essential  in order to sustain life. The genome is continuously threatened by different types  of DNA lesions, such as abasic sites, mismatches, interstrand crosslinks, or  single-stranded and double-stranded breaks. As a consequence, cells have evolved  specialized DNA damage response (DDR) mechanisms to sustain genome integrity. By  orchestrating multilayer signaling cascades specific for the type of lesion that  occurred, the DDR ensures that genetic information is preserved overtime. In the  last decades, DNA repair mechanisms have been thoroughly investigated to untangle  these complex networks of pathways and processes. As a result, key factors have been  identified that control and coordinate DDR circuits in time and space. In the first  part of this review, we describe the critical processes encompassing DNA damage  sensing and resolution. In the second part, we illustrate the consequences of  partial or complete failure of the DNA repair machinery. Lastly, we will report  examples in which this knowledge has been instrumental to develop novel therapies  based on genome editing technologies, such as CRISPR-Cas.\\nFAU - Carusillo, Antonio\\nAU  - Carusillo A\\nAD  - Institute for Transfusion Medicine and Gene Therapy, Medical Center-University of  Freiburg, 79106 Freiburg, Germany.\\nAD  - Center for Chronic Immunodeficiency, Faculty of Medicine, University of Freiburg,  79106 Freiburg, Germany.\\nFAU - Mussolino, Claudio\\nAU  - Mussolino C\\nAD  - Institute for Transfusion Medicine and Gene Therapy, Medical Center-University of  Freiburg, 79106 Freiburg, Germany.\\nAD  - Center for Chronic Immunodeficiency, Faculty of Medicine, University of Freiburg,  79106 Freiburg, Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20200710\\nTA  - Cells\\nJT  - Cells\\nJID - 101600052\\nSB  - IM\\nMH  - Animals\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - DNA Damage/genetics/*physiology\\nMH  - DNA Repair/genetics/physiology\\nMH  - Gene Editing\\nMH  - Humans\\nPMC - PMC7408370\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *DNA damage\\nOT  - *DNA damage response (DDR)\\nOT  - *HDR\\nOT  - *NHEJ\\nOT  - *cancer\\nOT  - *cell-cycle\\nOT  - *gene editing\\nOT  - *genome integrity\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2020/07/16 06:00\\nMHDA- 2021/04/13 06:00\\nCRDT- 2020/07/16 06:00\\nPHST- 2020/06/04 00:00 [received]\\nPHST- 2020/07/07 00:00 [revised]\\nPHST- 2020/07/07 00:00 [accepted]\\nPHST- 2020/07/16 06:00 [entrez]\\nPHST- 2020/07/16 06:00 [pubmed]\\nPHST- 2021/04/13 06:00 [medline]\\nAID - cells9071665 [pii]\\nAID - cells-09-01665 [pii]\\nAID - 10.3390/cells9071665 [doi]\\nPST - epublish\\nSO  - Cells. 2020 Jul 10;9(7):1665. doi: 10.3390/cells9071665.\\n\\nPMID- 30193985\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200304\\nLR  - 20200304\\nIS  - 1089-8638 (Electronic)\\nIS  - 0022-2836 (Linking)\\nVI  - 431\\nIP  - 1\\nDP  - 2019 Jan 4\\nTI  - CRISPR-Cas: Complex Functional Networks and Multiple Roles beyond Adaptive Immunity.\\nPG  - 3-20\\nLID - S0022-2836(18)31013-1 [pii]\\nLID - 10.1016/j.jmb.2018.08.030 [doi]\\nAB  - CRISPR-Cas is a prokaryotic adaptive immune system that functions by incorporating  fragments of foreign DNA into CRISPR arrays. The arrays containing spacers derived  from foreign DNA are transcribed, and the transcripts are processed to generate  spacer-containing mature CRISPR-RNAs that are employed as guides to specifically  recognize and cleave the DNA or RNA of the cognate parasitic genetic elements. The  CRISPR-Cas systems show remarkable complexity and diversity of molecular  organization and appear to be involved in various cellular functions that are  distinct from, even if connected to, adaptive immunity. In this review, we discuss  some of such functional links of CRISPR-Cas systems including their effect on  horizontal gene transfer that can be either inhibitory or stimulatory, connections  between CRISPR-Cas and DNA repair systems as well as programmed cell death and  signal transduction mechanisms, and potential role of CRISPR-Cas in transposon  integration and plasmid maintenance. The interplay between the primary function of  CRISPR-Cas as an adaptive immunity mechanism and these other roles defines the  richness of the biological effects of these systems and affects their spread among  bacteria and archaea.\\nCI  - Copyright Â© 2018 The Authors. Published by Elsevier Ltd.. All rights reserved.\\nFAU - Faure, Guilhem\\nAU  - Faure G\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA.\\nFAU - Makarova, Kira S\\nAU  - Makarova KS\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA.\\nFAU - Koonin, Eugene V\\nAU  - Koonin EV\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA. Electronic address:  koonin@ncbi.nlm.nih.gov.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Review\\nDEP - 20180905\\nPL  - England\\nTA  - J Mol Biol\\nJT  - Journal of molecular biology\\nJID - 2985088R\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Bacterial)\\nSB  - IM\\nMH  - Adaptive Immunity/genetics\\nMH  - Archaea/*genetics\\nMH  - Bacteria/*genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - DNA Repair/genetics\\nMH  - DNA Transposable Elements/*genetics\\nMH  - DNA, Bacterial/genetics\\nMH  - Gene Transfer, Horizontal/genetics\\nMH  - Plasmids/genetics\\nMH  - Signal Transduction/genetics\\nOTO - NOTNLM\\nOT  - *CRISPRâ€“Cas\\nOT  - *DNA repair\\nOT  - *programmed cell death\\nOT  - *signal transduction\\nOT  - *transposable elements\\nEDAT- 2018/09/09 06:00\\nMHDA- 2020/03/05 06:00\\nCRDT- 2018/09/09 06:00\\nPHST- 2018/08/02 00:00 [received]\\nPHST- 2018/08/28 00:00 [revised]\\nPHST- 2018/08/29 00:00 [accepted]\\nPHST- 2018/09/09 06:00 [pubmed]\\nPHST- 2020/03/05 06:00 [medline]\\nPHST- 2018/09/09 06:00 [entrez]\\nAID - S0022-2836(18)31013-1 [pii]\\nAID - 10.1016/j.jmb.2018.08.030 [doi]\\nPST - ppublish\\nSO  - J Mol Biol. 2019 Jan 4;431(1):3-20. doi: 10.1016/j.jmb.2018.08.030. Epub 2018 Sep 5.\\n\\nPMID- 33685599\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211026\\nLR  - 20211026\\nIS  - 1878-0814 (Electronic)\\nIS  - 1877-1173 (Linking)\\nVI  - 178\\nDP  - 2021\\nTI  - Application of CRISPR-Cas systems in neuroscience.\\nPG  - 231-264\\nLID - S1877-1173(20)30180-0 [pii]\\nLID - 10.1016/bs.pmbts.2020.12.010 [doi]\\nAB  - CRISPR-Cas systems have, over the years, emerged as indispensable tools for Genetic  interrogation in contexts of clinical interventions, elucidation of genetic pathways  and metabolic engineering and have pervaded almost every aspect of modern biology.  Within this repertoire, the nervous system comes with its own set of perplexities  and mysteries. Scientists have, over the years, tried to draw up a clearer genetic  picture of the neuron and how it functions in a network, mainly in an endeavor to  mitigate diseases of the human nervous system like Alzheimer\\'s, Parkinson\\'s,  Huntington\\'s, Autism Spectrum Disorder (ASD), etc. With most being progressive in  nature, these diseases have plagued mankind for centuries. In spite of our immense  progress in modern biology, we are yet to get a grasp over these diseases and  unraveling their mechanisms is of utmost importance. Before CRISPR-Cas systems came  along, the elucidation of the complex interactome of the mammalian nervous system  was attempted with erstwhile existing electrophysiological, histological and  pharmacological techniques coupled with Next Generation Sequencing and cell-specific  targeting technologies. Zinc finger nucleases (ZFNs) and transcription  activator-like effector nucleases (TALENs), imparted excellent sequence specific DNA  targeting capabilities but came with their huge baggage of extensive protein  engineering requirements, which practically rendered them unsuitable for high  throughput exercises. With the discovery of Clustered Regularly Interspaced  Palindromic Repeats (CRISPR) and CRISPR Associated proteins(CAS) systems by Ishino  (1987)(1), the era of extensive custom made endonuclease targeting was ushered in.  For the first time in 2012, Jinek et al. (2012)(2) repurposed the CRISPR-Cas  mediated bacterial immune system for customizable mammalian gene editing. The  CRISPR-Cas technology made it possible to easily customize Cas9 endonucleases to  cleave near specifically targeted sequences, thereby facilitating knock-ins or  knock-outs, silencing or activating or editing any gene, at any locus of the genome,  both at the base-pair level or at the epigenetic level. With this enhanced degree of  freedom, decrypting the nervous system and therapeutic interventions for  neuropathies became significantly less cumbersome an exercise. Here we take a brisk  walk through the several endeavors of research that show how the humble bacteria\\'s  CRISPR-Cas system gave us the \"nerves\" to \"talk\" to our nerves with ease.\\nCI  - Â© 2021 Elsevier Inc. All rights reserved.\\nFAU - Bonnerjee, Deepro\\nAU  - Bonnerjee D\\nAD  - Biophysics and Structural Genomics Division, Saha Institute of Nuclear Physics, Homi  Bhabha National Institute (HBNI), Kolkata, India. Electronic address:  deepro.bonnerjee@saha.ac.in.\\nFAU - Bagh, Sangram\\nAU  - Bagh S\\nAD  - Biophysics and Structural Genomics Division, Saha Institute of Nuclear Physics, Homi  Bhabha National Institute (HBNI), Kolkata, India.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210224\\nPL  - Netherlands\\nTA  - Prog Mol Biol Transl Sci\\nJT  - Progress in molecular biology and translational science\\nJID - 101498165\\nRN  - EC 3.1.- (Transcription Activator-Like Effector Nucleases)\\nRN  - EC 3.1.- (Zinc Finger Nucleases)\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems/genetics\\nMH  - *Gene Editing\\nMH  - Humans\\nMH  - *Nervous System Diseases/genetics/therapy\\nMH  - Transcription Activator-Like Effector Nucleases\\nMH  - Zinc Finger Nucleases\\nOTO - NOTNLM\\nOT  - *Alzheimer\\'s\\nOT  - *CRISPR-Cas9\\nOT  - *CRISPRa\\nOT  - *CRISPRi\\nOT  - *Fragile X\\nOT  - *Gene editing\\nOT  - *Neuroscience\\nOT  - *Parkinson\\'s\\nOT  - *dcas9\\nEDAT- 2021/03/10 06:00\\nMHDA- 2021/10/27 06:00\\nCRDT- 2021/03/09 05:42\\nPHST- 2021/03/09 05:42 [entrez]\\nPHST- 2021/03/10 06:00 [pubmed]\\nPHST- 2021/10/27 06:00 [medline]\\nAID - S1877-1173(20)30180-0 [pii]\\nAID - 10.1016/bs.pmbts.2020.12.010 [doi]\\nPST - ppublish\\nSO  - Prog Mol Biol Transl Sci. 2021;178:231-264. doi: 10.1016/bs.pmbts.2020.12.010. Epub  2021 Feb 24.\\n\\nPMID- 23761437\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20131028\\nLR  - 20211021\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 41\\nIP  - 15\\nDP  - 2013 Aug\\nTI  - Programmable repression and activation of bacterial gene expression using an  engineered CRISPR-Cas system.\\nPG  - 7429-37\\nLID - 10.1093/nar/gkt520 [doi]\\nAB  - The ability to artificially control transcription is essential both to the study of  gene function and to the construction of synthetic gene networks with desired  properties. Cas9 is an RNA-guided double-stranded DNA nuclease that participates in  the CRISPR-Cas immune defense against prokaryotic viruses. We describe the use of a  Cas9 nuclease mutant that retains DNA-binding activity and can be engineered as a  programmable transcription repressor by preventing the binding of the RNA polymerase  (RNAP) to promoter sequences or as a transcription terminator by blocking the  running RNAP. In addition, a fusion between the omega subunit of the RNAP and a Cas9  nuclease mutant directed to bind upstream promoter regions can achieve programmable  transcription activation. The simple and efficient modulation of gene expression  achieved by this technology is a useful asset for the study of gene networks and for  the development of synthetic biology and biotechnological applications.\\nFAU - Bikard, David\\nAU  - Bikard D\\nAD  - Laboratory of Bacteriology, The Rockefeller University, 1230 York Avenue, New York,  NY 10065, USA, Department of Microbiology and Immunobiology, Harvard Medical School,  4 Blackfan Circle, Boston, MA 02115, USA, Broad Institute of MIT and Harvard, 7  Cambridge Center, Cambridge, MA 02142, USA, McGovern Institute for Brain Research,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA, Department of Brain  and Cognitive Sciences, Massachusetts Institute of Technology, Cambridge, MA 02139,  USA and Department of Biological Engineering, Massachusetts Institute of Technology,  Cambridge, MA 02139, USA.\\nFAU - Jiang, Wenyan\\nAU  - Jiang W\\nFAU - Samai, Poulami\\nAU  - Samai P\\nFAU - Hochschild, Ann\\nAU  - Hochschild A\\nFAU - Zhang, Feng\\nAU  - Zhang F\\nFAU - Marraffini, Luciano A\\nAU  - Marraffini LA\\nLA  - eng\\nGR  - DP2 AI104556/AI/NIAID NIH HHS/United States\\nGR  - R01 GM044025/GM/NIGMS NIH HHS/United States\\nGR  - DP1MH100706/DP/NCCDPHP CDC HHS/United States\\nGR  - 1DP2AI104556-01/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20130612\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (RNA, Bacterial)\\nRN  - 0 (Repressor Proteins)\\nRN  - EC 3.1.- (Endoribonucleases)\\nSB  - IM\\nMH  - Bacterial Proteins/genetics/metabolism\\nMH  - Endoribonucleases/genetics/metabolism\\nMH  - Escherichia coli/genetics/metabolism\\nMH  - *Gene Expression Regulation, Bacterial\\nMH  - Gene Regulatory Networks\\nMH  - *Genes, Bacterial\\nMH  - Genes, Synthetic\\nMH  - Genetic Loci\\nMH  - Inverted Repeat Sequences\\nMH  - Promoter Regions, Genetic\\nMH  - Protein Binding\\nMH  - RNA, Bacterial/genetics/*metabolism\\nMH  - Repressor Proteins/genetics/metabolism\\nMH  - Streptococcus pneumoniae/genetics/metabolism\\nMH  - Streptococcus pyogenes/enzymology/genetics\\nMH  - *Transcriptional Activation\\nPMC - PMC3753641\\nEDAT- 2013/06/14 06:00\\nMHDA- 2013/10/29 06:00\\nCRDT- 2013/06/14 06:00\\nPHST- 2013/06/14 06:00 [entrez]\\nPHST- 2013/06/14 06:00 [pubmed]\\nPHST- 2013/10/29 06:00 [medline]\\nAID - gkt520 [pii]\\nAID - 10.1093/nar/gkt520 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2013 Aug;41(15):7429-37. doi: 10.1093/nar/gkt520. Epub 2013 Jun  12.\\n\\nPMID- 28778606\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20181126\\nLR  - 20181202\\nIS  - 1879-3096 (Electronic)\\nIS  - 0167-7799 (Linking)\\nVI  - 36\\nIP  - 2\\nDP  - 2018 Feb\\nTI  - Advances in Industrial Biotechnology Using CRISPR-Cas Systems.\\nPG  - 134-146\\nLID - S0167-7799(17)30187-7 [pii]\\nLID - 10.1016/j.tibtech.2017.07.007 [doi]\\nAB  - The term \\'clustered regularly interspaced short palindromic repeats\\' (CRISPR) has  recently become synonymous with the genome-editing revolution. The RNA-guided  endonuclease CRISPR-associated protein 9 (Cas9), in particular, has attracted  attention for its promise in basic research and gene editing-based therapeutics.  CRISPR-Cas systems are efficient and easily programmable nucleic acid-targeting  tools, with uses reaching beyond research and therapeutic development into the  precision breeding of plants and animals and the engineering of industrial microbes.  CRISPR-Cas systems have potential for many microbial engineering applications,  including bacterial strain typing, immunization of cultures, autoimmunity or  self-targeted cell killing, and the engineering or control of metabolic pathways for  improved biochemical synthesis. In this review, we explore the fundamental  characteristics of CRISPR-Cas systems and highlight how these features can be used  in industrial settings.\\nCI  - Copyright Â© 2017 Elsevier Ltd. All rights reserved.\\nFAU - Donohoue, Paul D\\nAU  - Donohoue PD\\nAD  - Caribou Biosciences, Inc., 2929 7th St., Suite 105, Berkeley, CA 94710, USA.\\nFAU - Barrangou, Rodolphe\\nAU  - Barrangou R\\nAD  - Department of Food, Bioprocessing, and Nutrition Sciences, North Carolina State  University, Raleigh, NC 27695, USA.\\nFAU - May, Andrew P\\nAU  - May AP\\nAD  - Caribou Biosciences, Inc., 2929 7th St., Suite 105, Berkeley, CA 94710, USA; Current  address: Chan Zuckerberg Biohub, 499 Illinois St, San Francisco, CA 94158, USA.  Electronic address: andy.may@czbiohub.org.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20170801\\nPL  - England\\nTA  - Trends Biotechnol\\nJT  - Trends in biotechnology\\nJID - 8310903\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (RNA, Guide)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nRN  - EC 3.1.- (Cas9 endonuclease Streptococcus pyogenes)\\nRN  - EC 3.1.- (Endonucleases)\\nSB  - IM\\nMH  - Animals\\nMH  - Bacterial Proteins/genetics/metabolism\\nMH  - Biotechnology/*methods\\nMH  - CRISPR-Associated Protein 9\\nMH  - *CRISPR-Cas Systems\\nMH  - Endonucleases/genetics/metabolism\\nMH  - Gene Editing/*methods\\nMH  - Genetic Engineering/methods\\nMH  - Industrial Microbiology/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Plants/genetics\\nMH  - RNA, Guide/genetics/metabolism\\nEDAT- 2017/08/06 06:00\\nMHDA- 2018/11/27 06:00\\nCRDT- 2017/08/06 06:00\\nPHST- 2017/04/18 00:00 [received]\\nPHST- 2017/07/11 00:00 [revised]\\nPHST- 2017/07/12 00:00 [accepted]\\nPHST- 2017/08/06 06:00 [pubmed]\\nPHST- 2018/11/27 06:00 [medline]\\nPHST- 2017/08/06 06:00 [entrez]\\nAID - S0167-7799(17)30187-7 [pii]\\nAID - 10.1016/j.tibtech.2017.07.007 [doi]\\nPST - ppublish\\nSO  - Trends Biotechnol. 2018 Feb;36(2):134-146. doi: 10.1016/j.tibtech.2017.07.007. Epub  2017 Aug 1.\\n\\nPMID- 24837679\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20140708\\nLR  - 20211021\\nIS  - 1097-4164 (Electronic)\\nIS  - 1097-2765 (Print)\\nIS  - 1097-2765 (Linking)\\nVI  - 54\\nIP  - 4\\nDP  - 2014 May 22\\nTI  - Multiplexed and programmable regulation of gene networks with an integrated RNA and  CRISPR/Cas toolkit in human cells.\\nPG  - 698-710\\nLID - S1097-2765(14)00355-4 [pii]\\nLID - 10.1016/j.molcel.2014.04.022 [doi]\\nAB  - RNA-based regulation and CRISPR/Cas transcription factors (CRISPR-TFs) have the  potential to be integrated for the tunable modulation of gene networks. A major  limitation of this methodology is that guide RNAs (gRNAs) for CRISPR-TFs can only be  expressed from RNA polymerase III promoters in human cells, limiting their use for  conditional gene regulation. We present new strategies that enable expression of  functional gRNAs from RNA polymerase II promoters and multiplexed production of  proteins and gRNAs from a single transcript in human cells. We use multiple RNA  regulatory strategies, including RNA-triple-helix structures, introns, microRNAs,  and ribozymes, with Cas9-based CRISPR-TFs and Cas6/Csy4-based RNA processing. Using  these tools, we efficiently modulate endogenous promoters and implement tunable  synthetic circuits, including multistage cascades and RNA-dependent networks that  can be rewired with Csy4 to achieve complex behaviors. This toolkit can be used for  programming scalable gene circuits and perturbing endogenous networks for biology,  therapeutic, and synthetic biology applications.\\nCI  - Copyright Â© 2014 Elsevier Inc. All rights reserved.\\nFAU - Nissim, Lior\\nAU  - Nissim L\\nAD  - Synthetic Biology Group, Research Laboratory of Electronics, Department of  Biological Engineering and Electrical Engineering & Computer Science, Massachusetts  Institute of Technology, Cambridge, MA 02142, USA.\\nFAU - Perli, Samuel D\\nAU  - Perli SD\\nAD  - Synthetic Biology Group, Research Laboratory of Electronics, Department of  Biological Engineering and Electrical Engineering & Computer Science, Massachusetts  Institute of Technology, Cambridge, MA 02142, USA.\\nFAU - Fridkin, Alexandra\\nAU  - Fridkin A\\nAD  - Synthetic Biology Group, Research Laboratory of Electronics, Department of  Biological Engineering and Electrical Engineering & Computer Science, Massachusetts  Institute of Technology, Cambridge, MA 02142, USA.\\nFAU - Perez-Pinera, Pablo\\nAU  - Perez-Pinera P\\nAD  - Synthetic Biology Group, Research Laboratory of Electronics, Department of  Biological Engineering and Electrical Engineering & Computer Science, Massachusetts  Institute of Technology, Cambridge, MA 02142, USA.\\nFAU - Lu, Timothy K\\nAU  - Lu TK\\nAD  - Synthetic Biology Group, Research Laboratory of Electronics, Department of  Biological Engineering and Electrical Engineering & Computer Science, Massachusetts  Institute of Technology, Cambridge, MA 02142, USA. Electronic address:  timlu@mit.edu.\\nLA  - eng\\nGR  - DP2 OD008435/OD/NIH HHS/United States\\nGR  - P50 GM098792/GM/NIGMS NIH HHS/United States\\nGR  - DP2OD008435/OD/NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20140515\\nTA  - Mol Cell\\nJT  - Molecular cell\\nJID - 9802571\\nRN  - 0 (MicroRNAs)\\nRN  - 0 (RNA, Catalytic)\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Transcription Factors)\\nRN  - EC 2.7.7.- (RNA Polymerase II)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - *Gene Expression Regulation/physiology\\nMH  - *Gene Regulatory Networks\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Introns/genetics/physiology\\nMH  - MicroRNAs/genetics/metabolism\\nMH  - Promoter Regions, Genetic\\nMH  - RNA Polymerase II/genetics/*metabolism\\nMH  - RNA, Catalytic/metabolism\\nMH  - RNA, Guide/genetics\\nMH  - Synthetic Biology\\nMH  - Transcription Factors/genetics/*metabolism\\nPMC - PMC4077618\\nMID - NIHMS590695\\nEDAT- 2014/05/20 06:00\\nMHDA- 2014/07/09 06:00\\nCRDT- 2014/05/20 06:00\\nPHST- 2014/03/22 00:00 [received]\\nPHST- 2014/04/15 00:00 [revised]\\nPHST- 2014/04/18 00:00 [accepted]\\nPHST- 2014/05/20 06:00 [entrez]\\nPHST- 2014/05/20 06:00 [pubmed]\\nPHST- 2014/07/09 06:00 [medline]\\nAID - S1097-2765(14)00355-4 [pii]\\nAID - 10.1016/j.molcel.2014.04.022 [doi]\\nPST - ppublish\\nSO  - Mol Cell. 2014 May 22;54(4):698-710. doi: 10.1016/j.molcel.2014.04.022. Epub 2014  May 15.\\n\\nPMID- 28136159\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180727\\nLR  - 20181113\\nIS  - 1555-8584 (Electronic)\\nIS  - 1547-6286 (Print)\\nIS  - 1547-6286 (Linking)\\nVI  - 14\\nIP  - 10\\nDP  - 2017 Oct 3\\nTI  - CRISPR-Cas type II-based Synthetic Biology applications in eukaryotic cells.\\nPG  - 1286-1293\\nLID - 10.1080/15476286.2017.1282024 [doi]\\nAB  - The CRISPR-Cas system has rapidly reached a huge popularity as a new, powerful  method for precise DNA editing and genome reengineering. In Synthetic Biology, the  CRISPR-Cas type II system has inspired the construction of a novel class of  RNA-based transcription factors. In their simplest form, they are made of a CRISPR  RNA molecule, which targets a promoter sequence, and a deficient Cas9 (i.e. deprived  of any nuclease activity) that has been fused to an activation or a repression  domain. Up- and downregulation of single genes in mammalian and yeast cells have  been achieved with satisfactory results. Moreover, the construction of CRISPR-based  transcription factors is much simpler than the assembly of synthetic proteins such  as the Transcription Activator-Like effectors. However, the feasibility of complex  synthetic networks fully based on the CRISPR-dCas9 technology has still to be proved  and new designs, which take into account different CRISPR types, shall be  investigated.\\nFAU - Marchisio, Mario Andrea\\nAU  - Marchisio MA\\nAUID- ORCID: 0000-0002-5102-1069\\nAD  - a School of Life Science and Technology, Harbin Institute of Technology , Harbin ,  P.R. China.\\nFAU - Huang, Zhiwei\\nAU  - Huang Z\\nAD  - a School of Life Science and Technology, Harbin Institute of Technology , Harbin ,  P.R. China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20170131\\nTA  - RNA Biol\\nJT  - RNA biology\\nJID - 101235328\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Transcription Factors)\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems\\nMH  - Eukaryotic Cells\\nMH  - Gene Expression Regulation\\nMH  - Humans\\nMH  - RNA, Guide/metabolism\\nMH  - Synthetic Biology/*methods\\nMH  - Transcription Factors/*genetics\\nPMC - PMC5711462\\nOTO - NOTNLM\\nOT  - *CRISPR\\nOT  - *Cas9\\nOT  - *Synthetic Biology\\nOT  - *gene circuits\\nOT  - *guide RNA\\nEDAT- 2017/02/01 06:00\\nMHDA- 2018/07/28 06:00\\nCRDT- 2017/02/01 06:00\\nPHST- 2017/02/01 06:00 [pubmed]\\nPHST- 2018/07/28 06:00 [medline]\\nPHST- 2017/02/01 06:00 [entrez]\\nAID - 1282024 [pii]\\nAID - 10.1080/15476286.2017.1282024 [doi]\\nPST - ppublish\\nSO  - RNA Biol. 2017 Oct 3;14(10):1286-1293. doi: 10.1080/15476286.2017.1282024. Epub 2017  Jan 31.\\n\\nPMID- 32843437\\nOWN - NLM\\nSTAT- Publisher\\nLR  - 20200826\\nIS  - 1532-298X (Electronic)\\nIS  - 1040-4651 (Linking)\\nDP  - 2020 Aug 25\\nTI  - CRISPR Screens in Plants: Approaches, Guidelines, and Future Prospects.\\nLID - tpc.00463.2020 [pii]\\nLID - 10.1105/tpc.20.00463 [doi]\\nAB  - CRISPR-Cas systems have revolutionized genome engineering by facilitating a wide  range of targeted DNA perturbations. These systems have resulted in new powerful  screens to test gene functions at the genomic scale. While there is tremendous  potential for CRISPR screens to map and interrogate gene regulatory networks at  unprecedented speed and scale, their implementation in plants remains in its  infancy. Here we discuss the general concepts, tools and workflows for establishing  CRISPR screens in plants and analyze the handful of recent reports using this  strategy to generate mutant knockout collections or diversify DNA sequences. In  addition, we provide insight on how to design CRISPR knockout screens in plants  given the current challenges and limitations and examine multiple design options.  Finally, we discuss the unique multiplexing capabilities of CRISPR screens to  investigate redundant gene function in highly duplicated plant genomes.  Combinatorial mutant screens have the potential to routinely generate higher-order  mutant collections and facilitate the characterization of gene networks. By  integrating this approach with the large resource of genomic profiles that were  generated in the last two decades, the implementation of CRISPR screens offers new  opportunities to analyze plant genomes at deeper resolution and will greatly advance  plant functional and synthetic biology.\\nCI  - Â© 2020 American Society of Plant Biologists. All rights reserved.\\nFAU - Gaillochet, Christophe\\nAU  - Gaillochet C\\nAD  - VIB CITY: Ghent POSTAL_CODE: 9052 Belgium [BE].\\nFAU - Develtere, Ward\\nAU  - Develtere W\\nAD  - VIB-UGent Center for Plant Systems Biology CITY: Gent Belgium [BE].\\nFAU - Jacobs, Thomas B\\nAU  - Jacobs TB\\nAD  - VIB-UGent Center for Plant Systems Biology CITY: Gent POSTAL_CODE: 9052 Belgium [BE]  thomas.jacobs@ugent.vib.be.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200825\\nPL  - England\\nTA  - Plant Cell\\nJT  - The Plant cell\\nJID - 9208688\\nSB  - IM\\nEDAT- 2020/08/28 06:00\\nMHDA- 2020/08/28 06:00\\nCRDT- 2020/08/27 06:00\\nPHST- 2020/08/24 00:00 [accepted]\\nPHST- 2020/06/15 00:00 [received]\\nPHST- 2020/08/27 06:00 [entrez]\\nPHST- 2020/08/28 06:00 [pubmed]\\nPHST- 2020/08/28 06:00 [medline]\\nAID - tpc.20.00463 [pii]\\nAID - 10.1105/tpc.20.00463 [doi]\\nPST - aheadofprint\\nSO  - Plant Cell. 2020 Aug 25:tpc.00463.2020. doi: 10.1105/tpc.20.00463.\\n\\nPMID- 28782570\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190724\\nLR  - 20191210\\nIS  - 1872-8294 (Electronic)\\nIS  - 0169-409X (Print)\\nIS  - 0169-409X (Linking)\\nVI  - 123\\nDP  - 2018 Jan 1\\nTI  - Nanomedicine and advanced technologies for burns: Preventing infection and  facilitating wound healing.\\nPG  - 33-64\\nLID - S0169-409X(17)30129-1 [pii]\\nLID - 10.1016/j.addr.2017.08.001 [doi]\\nAB  - According to the latest report from the World Health Organization, an estimated  265,000 deaths still occur every year as a direct result of burn injuries. A  widespread range of these deaths induced by burn wound happens in low- and  middle-income countries, where survivors face a lifetime of morbidity. Most of the  deaths occur due to infections when a high percentage of the external regions of the  body area is affected. Microbial nutrient availability, skin barrier disruption, and  vascular supply destruction in burn injuries as well as systemic immunosuppression  are important parameters that cause burns to be susceptible to infections. Topical  antimicrobials and dressings are generally employed to inhibit burn infections  followed by a burn wound therapy, because systemic antibiotics have problems in  reaching the infected site, coupled with increasing microbial drug resistance.  Nanotechnology has provided a range of molecular designed nanostructures (NS) that  can be used in both therapeutic and diagnostic applications in burns. These NSs can  be divided into organic and non-organic (such as polymeric nanoparticles (NPs) and  silver NPs, respectively), and many have been designed to display multifunctional  activity. The present review covers the physiology of skin, burn classification,  burn wound pathogenesis, animal models of burn wound infection, and various topical  therapeutic approaches designed to combat infection and stimulate healing. These  include biological based approaches (e.g. immune-based antimicrobial molecules,  therapeutic microorganisms, antimicrobial agents, etc.), antimicrobial photo- and  ultrasound-therapy, as well as nanotechnology-based wound healing approaches as a  revolutionizing area. Thus, we focus on organic and non-organic NSs designed to  deliver growth factors to burned skin, and scaffolds, dressings, etc. for exogenous  stem cells to aid skin regeneration. Eventually, recent breakthroughs and  technologies with substantial potentials in tissue regeneration and skin wound  therapy (that are as the basis of burn wound therapies) are briefly taken into  consideration including 3D-printing, cell-imprinted substrates, nano-architectured  surfaces, and novel gene-editing tools such as CRISPR-Cas.\\nCI  - Copyright Â© 2017 Elsevier B.V. All rights reserved.\\nFAU - Mofazzal Jahromi, Mirza Ali\\nAU  - Mofazzal Jahromi MA\\nAD  - Department of Advanced Medical Sciences & Technologies, School of Medicine, Jahrom  University of Medical Sciences (JUMS), Jahrom, Iran; Research Center for  Noncommunicable Diseases, School of Medicine, Jahrom University of Medical Sciences  (JUMS), Jahrom, Iran.\\nFAU - Sahandi Zangabad, Parham\\nAU  - Sahandi Zangabad P\\nAD  - Research Center for Pharmaceutical Nanotechnology (RCPN), Tabriz University of  Medical Science (TUOMS), Tabriz, Iran; Cellular and Molecular Research Center, Iran  University of Medical Sciences, Tehran, Iran; Bio-Nano-Interfaces: Convergence of  Sciences (BNICS), Universal Scientific Education and Research Network (USERN),  Tehran, Iran; Department of Medical Nanotechnology, Faculty of Advanced Technologies  in Medicine, Iran University of Medical Sciences, Tehran, Iran.\\nFAU - Moosavi Basri, Seyed Masoud\\nAU  - Moosavi Basri SM\\nAD  - Bio-Nano-Interfaces: Convergence of Sciences (BNICS), Universal Scientific Education  and Research Network (USERN), Tehran, Iran; Bioenvironmental Research Center, Sharif  University of Technology, Tehran, Iran; Civil & Environmental Engineering  Department, Shahid Beheshti University, Tehran, Iran.\\nFAU - Sahandi Zangabad, Keyvan\\nAU  - Sahandi Zangabad K\\nAD  - Bio-Nano-Interfaces: Convergence of Sciences (BNICS), Universal Scientific Education  and Research Network (USERN), Tehran, Iran; Department of Polymer Engineering,  Sahand University of Technology, PO Box 51335-1996, Tabriz, Iran; Nanomedicine  Research Association (NRA), Universal Scientific Education and Research Network  (USERN), Tehran, Iran.\\nFAU - Ghamarypour, Ameneh\\nAU  - Ghamarypour A\\nAD  - Bio-Nano-Interfaces: Convergence of Sciences (BNICS), Universal Scientific Education  and Research Network (USERN), Tehran, Iran; Department of Biology, Science and  Research Branch, Islamic Azad university, Tehran, Iran.\\nFAU - Aref, Amir R\\nAU  - Aref AR\\nAD  - Department of Medical Oncology, Belfer Center for Applied Cancer Science,  Dana-Farber Cancer Institute, Harvard Medical School, Boston, MA 02215, USA.\\nFAU - Karimi, Mahdi\\nAU  - Karimi M\\nAD  - Cellular and Molecular Research Center, Iran University of Medical Sciences, Tehran,  Iran; Department of Medical Nanotechnology, Faculty of Advanced Technologies in  Medicine, Iran University of Medical Sciences, Tehran, Iran; Research Center for  Science and Technology in Medicine, Tehran University of Medical Sciences, Tehran,  Iran; Wellman Center for Photomedicine, Massachusetts General Hospital, Harvard  Medical School, Boston, USA. Electronic address: m_karimy2006@yahoo.com.\\nFAU - Hamblin, Michael R\\nAU  - Hamblin MR\\nAD  - Wellman Center for Photomedicine, Massachusetts General Hospital, Harvard Medical  School, Boston, USA; Department of Dermatology, Harvard Medical School, Boston, USA;  Harvard-MIT Division of Health Sciences and Technology, Cambridge, USA. Electronic  address: hamblin@helix.mgh.harvard.edu.\\nLA  - eng\\nGR  - R01 AI050875/AI/NIAID NIH HHS/United States\\nGR  - R21 AI121700/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Review\\nDEP - 20170804\\nTA  - Adv Drug Deliv Rev\\nJT  - Advanced drug delivery reviews\\nJID - 8710523\\nRN  - 0 (Anti-Bacterial Agents)\\nRN  - 0 (Antimicrobial Cationic Peptides)\\nMH  - Animals\\nMH  - Anti-Bacterial Agents/*therapeutic use\\nMH  - Antimicrobial Cationic Peptides/*therapeutic use\\nMH  - Burns/*therapy\\nMH  - Humans\\nMH  - *Immunotherapy\\nMH  - Infections/*drug therapy\\nMH  - *Nanomedicine\\nMH  - Nanostructures/chemistry\\nMH  - Wound Healing/*drug effects\\nPMC - PMC5742034\\nMID - NIHMS898183\\nOTO - NOTNLM\\nOT  - *3D printing\\nOT  - *Burn wound infection\\nOT  - *CRISPR\\nOT  - *Cell-Imprinting\\nOT  - *Gene therapy\\nOT  - *Growth factors\\nOT  - *Nanomedicine\\nOT  - *Nanoparticles\\nOT  - *Stem cells\\nOT  - *Stimulus-responsive drug delivery\\nOT  - *Topical treatment\\nOT  - *Wound healing\\nEDAT- 2017/08/08 06:00\\nMHDA- 2019/07/25 06:00\\nCRDT- 2017/08/08 06:00\\nPHST- 2017/03/24 00:00 [received]\\nPHST- 2017/07/20 00:00 [revised]\\nPHST- 2017/08/01 00:00 [accepted]\\nPHST- 2017/08/08 06:00 [pubmed]\\nPHST- 2019/07/25 06:00 [medline]\\nPHST- 2017/08/08 06:00 [entrez]\\nAID - S0169-409X(17)30129-1 [pii]\\nAID - 10.1016/j.addr.2017.08.001 [doi]\\nPST - ppublish\\nSO  - Adv Drug Deliv Rev. 2018 Jan 1;123:33-64. doi: 10.1016/j.addr.2017.08.001. Epub 2017  Aug 4.\\n\\nPMID- 32051592\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200529\\nLR  - 20210121\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Print)\\nIS  - 0028-0836 (Linking)\\nVI  - 578\\nIP  - 7795\\nDP  - 2020 Feb\\nTI  - Clades of huge phages from across Earth\\'s ecosystems.\\nPG  - 425-431\\nLID - 10.1038/s41586-020-2007-4 [doi]\\nAB  - Bacteriophages typically have small genomes(1) and depend on their bacterial hosts  for replication(2). Here we sequenced DNA from diverse ecosystems and found hundreds  of phage genomes with lengths of more than 200Â\\xa0kilobases (kb), including a genome of  735Â\\xa0kb, which is-to our knowledge-the largest phage genome to be described to date.  Thirty-five genomes were manually curated to completion (circular and no gaps).  Expanded genetic repertoires include diverse and previously undescribed CRISPR-Cas  systems, transfer RNAs (tRNAs), tRNA synthetases, tRNA-modification enzymes,  translation-initiation and elongation factors, and ribosomal proteins. The  CRISPR-Cas systems of phages have the capacity to silence host transcription factors  and translational genes, potentially as part of a larger interaction network that  intercepts translation to redirect biosynthesis to phage-encoded functions. In  addition, some phages may repurpose bacterial CRISPR-Cas systems to eliminate  competing phages. We phylogenetically define the major clades of huge phages from  human and other animal microbiomes, as well as from oceans, lakes, sediments, soils  and the built environment. We conclude that the large gene inventories of huge  phages reflect a conserved biological strategy, and that the phages are distributed  across a broad bacterial host range and across Earth\\'s ecosystems.\\nFAU - Al-Shayeb, Basem\\nAU  - Al-Shayeb B\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Sachdeva, Rohan\\nAU  - Sachdeva R\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Chen, Lin-Xing\\nAU  - Chen LX\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Ward, Fred\\nAU  - Ward F\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Munk, Patrick\\nAU  - Munk P\\nAD  - National Food Institute, Technical University of Denmark, Kongens Lyngby, Denmark.\\nFAU - Devoto, Audra\\nAU  - Devoto A\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Castelle, Cindy J\\nAU  - Castelle CJ\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Olm, Matthew R\\nAU  - Olm MR\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Bouma-Gregson, Keith\\nAU  - Bouma-Gregson K\\nAD  - Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Amano, Yuki\\nAU  - Amano Y\\nAD  - Nuclear Fuel Cycle Engineering Laboratories, Japan Atomic Energy Agency, Tokai-mura,  Japan.\\nFAU - He, Christine\\nAU  - He C\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - MÃ©heust, RaphaÃ«l\\nAU  - MÃ©heust R\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Brooks, Brandon\\nAU  - Brooks B\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Thomas, Alex\\nAU  - Thomas A\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Lavy, Adi\\nAU  - Lavy A\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Matheus-Carnevali, Paula\\nAU  - Matheus-Carnevali P\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Sun, Christine\\nAU  - Sun C\\nAD  - Department of Microbiology & Immunology, Stanford University, Stanford, CA, USA.\\nFAU - Goltsman, Daniela S A\\nAU  - Goltsman DSA\\nAD  - Department of Microbiology & Immunology, Stanford University, Stanford, CA, USA.\\nFAU - Borton, Mikayla A\\nAU  - Borton MA\\nAD  - Department of Soil and Crop Sciences, Colorado State University, Fort Collins, CO,  USA.\\nFAU - Sharrar, Allison\\nAU  - Sharrar A\\nAD  - Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Jaffe, Alexander L\\nAU  - Jaffe AL\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Nelson, Tara C\\nAU  - Nelson TC\\nAD  - Department of Civil and Mineral Engineering, University of Toronto, Toronto,  Ontario, Canada.\\nFAU - Kantor, Rose\\nAU  - Kantor R\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Keren, Ray\\nAU  - Keren R\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Lane, Katherine R\\nAU  - Lane KR\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Farag, Ibrahim F\\nAU  - Farag IF\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Lei, Shufei\\nAU  - Lei S\\nAD  - Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Finstad, Kari\\nAU  - Finstad K\\nAD  - Environmental Science, Policy and Management, University of California Berkeley,  Berkeley, CA, USA.\\nFAU - Amundson, Ronald\\nAU  - Amundson R\\nAD  - Environmental Science, Policy and Management, University of California Berkeley,  Berkeley, CA, USA.\\nFAU - Anantharaman, Karthik\\nAU  - Anantharaman K\\nAD  - Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Zhou, Jinglie\\nAU  - Zhou J\\nAD  - DOE Joint Genome Institute, Berkeley, CA, USA.\\nFAU - Probst, Alexander J\\nAU  - Probst AJ\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Power, Mary E\\nAU  - Power ME\\nAD  - Integrative Biology, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Tringe, Susannah G\\nAU  - Tringe SG\\nAD  - DOE Joint Genome Institute, Berkeley, CA, USA.\\nFAU - Li, Wen-Jun\\nAU  - Li WJ\\nAD  - School of Life Sciences, Sun Yat-Sen University, Guangzhou, China.\\nFAU - Wrighton, Kelly\\nAU  - Wrighton K\\nAD  - Department of Soil and Crop Sciences, Colorado State University, Fort Collins, CO,  USA.\\nFAU - Harrison, Sue\\nAU  - Harrison S\\nAD  - Centre for Bioprocess Engineering Research, University of Cape Town, Cape Town,  South Africa.\\nFAU - Morowitz, Michael\\nAU  - Morowitz M\\nAD  - Department of Surgery, University of Pittsburgh School of Medicine, Pittsburgh, PA,  USA.\\nFAU - Relman, David A\\nAU  - Relman DA\\nAD  - Department of Microbiology & Immunology, Stanford University, Stanford, CA, USA.\\nFAU - Doudna, Jennifer A\\nAU  - Doudna JA\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Lehours, Anne-Catherine\\nAU  - Lehours AC\\nAD  - Laboratoire Microorganismes: GÃ©nome et Environnement, UniversitÃ© Clermont Auvergne,  CNRS, Clermont-Ferrand, France.\\nFAU - Warren, Lesley\\nAU  - Warren L\\nAD  - Department of Civil and Mineral Engineering, University of Toronto, Toronto,  Ontario, Canada.\\nFAU - Cate, Jamie H D\\nAU  - Cate JHD\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.\\nFAU - Santini, Joanne M\\nAU  - Santini JM\\nAD  - Institute of Structural and Molecular Biology, University College London, London,  UK.\\nFAU - Banfield, Jillian F\\nAU  - Banfield JF\\nAD  - Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA.  jbanfield@berkeley.edu.\\nAD  - Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA.  jbanfield@berkeley.edu.\\nAD  - Environmental Science, Policy and Management, University of California Berkeley,  Berkeley, CA, USA. jbanfield@berkeley.edu.\\nAD  - School of Earth Sciences, University of Melbourne, Melbourne, Victoria, Australia.  jbanfield@berkeley.edu.\\nLA  - eng\\nGR  - R01 AI092531/AI/NIAID NIH HHS/United States\\nGR  - U01 AI142817/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20200212\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nRN  - 0 (Ribosomal Proteins)\\nRN  - 9014-25-9 (RNA, Transfer)\\nRN  - EC 6.1.1.- (Amino Acyl-tRNA Synthetases)\\nSB  - IM\\nMH  - Amino Acyl-tRNA Synthetases/genetics\\nMH  - Animals\\nMH  - Bacteria/genetics/*virology\\nMH  - Bacteriophages/*classification/*genetics/isolation & purification/metabolism\\nMH  - Biodiversity\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - *Earth, Planet\\nMH  - *Ecosystem\\nMH  - Evolution, Molecular\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - Gene Expression Regulation, Viral\\nMH  - Genome, Viral/*genetics\\nMH  - Host Specificity\\nMH  - Humans\\nMH  - Lakes/virology\\nMH  - Molecular Sequence Annotation\\nMH  - Oceans and Seas\\nMH  - *Phylogeny\\nMH  - Prophages/genetics\\nMH  - Protein Biosynthesis\\nMH  - RNA, Transfer/genetics\\nMH  - Ribosomal Proteins/genetics\\nMH  - Seawater/virology\\nMH  - Soil Microbiology\\nMH  - Transcription, Genetic\\nPMC - PMC7162821\\nCOIS- The Regents of the University of California have patents pending for CRISPR  technologies on which the authors are inventors. J.A.D. is a co-founder of Caribou  Biosciences, Editas Medicine, Intellia Therapeutics, Scribe Therapeutics and Mammoth  Biosciences, a scientific advisory board member of Caribous Biosciences, Intellia  Therapeutics, eFFECTOR Therapeutics, Scribe Therapeutics, Synthego, Mammoth  Biosciences and Inari, and is a Director at Johnson & Johnson and has sponsored  research projects by Biogen and Pfizer. J.F.B. is a founder of Metagenomi.\\nEDAT- 2020/02/14 06:00\\nMHDA- 2020/05/30 06:00\\nCRDT- 2020/02/14 06:00\\nPHST- 2019/03/22 00:00 [received]\\nPHST- 2020/01/02 00:00 [accepted]\\nPHST- 2020/02/14 06:00 [pubmed]\\nPHST- 2020/05/30 06:00 [medline]\\nPHST- 2020/02/14 06:00 [entrez]\\nAID - 10.1038/s41586-020-2007-4 [pii]\\nAID - 2007 [pii]\\nAID - 10.1038/s41586-020-2007-4 [doi]\\nPST - ppublish\\nSO  - Nature. 2020 Feb;578(7795):425-431. doi: 10.1038/s41586-020-2007-4. Epub 2020 Feb  12.\\n\\nPMID- 32348779\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200918\\nLR  - 20210605\\nIS  - 1097-4164 (Electronic)\\nIS  - 1097-2765 (Print)\\nIS  - 1097-2765 (Linking)\\nVI  - 78\\nIP  - 5\\nDP  - 2020 Jun 4\\nTI  - CRISPR-Cas13 Inhibitors Block RNA Editing in Bacteria and Mammalian Cells.\\nPG  - 850-861.e5\\nLID - S1097-2765(20)30225-2 [pii]\\nLID - 10.1016/j.molcel.2020.03.033 [doi]\\nAB  - Cas13 has demonstrated unique and broad utility in RNA editing, nucleic acid  detection, and disease diagnosis; however, a constantly active Cas enzyme may induce  unwanted effects. Bacteriophage- or prophage-region-encoded anti-CRISPR (acr) gene  molecules provide the potential to control targeting specificity and potency to  allow for optimal RNA editing and nucleic acid detection by spatiotemporally  modulating endonuclease activities. Using integrated approaches to screen acrVI  candidates and evaluate their effects on Cas13 function, we discovered a series of  acrVIA1-7 genes that block the activities of Cas13a. These VI-A CRISPR inhibitors  substantially attenuate RNA targeting and editing by Cas13a in human cells.  Strikingly, type VI-A anti-CRISPRs (AcrVIAs) also significantly muffle the  single-nucleic-acid editing ability of the dCas13a RNA-editing system.  Mechanistically, AcrVIA1, -4, -5, and -6 bind LwaCas13a, while AcrVIA2 and -3 can  only bind the LwaCas13-crRNA (CRISPR RNA) complex. These identified acr molecules  may enable precise RNA editing in Cas13-based application and study of  phage-bacterium interaction.\\nCI  - Copyright Â© 2020 Elsevier Inc. All rights reserved.\\nFAU - Lin, Ping\\nAU  - Lin P\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; Wound Trauma Medical Center,  State Key Laboratory of Trauma, Burns and Combined Injury, Daping Hospital, Army  Medical University, Chongqing 400042, China.\\nFAU - Qin, Shugang\\nAU  - Qin S\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; State Key Laboratory of  Biotherapy, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, China.\\nFAU - Pu, Qinqin\\nAU  - Pu Q\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; State Key Laboratory of  Biotherapy, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, China.\\nFAU - Wang, Zhihan\\nAU  - Wang Z\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; West China School of Basic  Medical Sciences and Forensic Medicine, Sichuan University, Chengdu, Sichuan 610041,  China.\\nFAU - Wu, Qun\\nAU  - Wu Q\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; Department of Pediatrics,  Ruijin Hospital affiliated to Shanghai Jiao Tong University of Medicine, Shanghai  200025, China.\\nFAU - Gao, Pan\\nAU  - Gao P\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA; State Key Laboratory of  Biotherapy, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, China.\\nFAU - Schettler, Jacob\\nAU  - Schettler J\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA.\\nFAU - Guo, Kai\\nAU  - Guo K\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA.\\nFAU - Li, Rongpeng\\nAU  - Li R\\nAD  - Key Laboratory of Biotechnology for Medicinal Plants of Jiangsu Province, Jiangsu  Normal University, Xuzhou, Jiangsu 221116, China.\\nFAU - Li, Guoping\\nAU  - Li G\\nAD  - Inflammations & Allergic Diseases Research Laboratory, Affiliated Hospital of  Southwest Medical University, Luzhou, Sichuan 646004, China.\\nFAU - Huang, Canhua\\nAU  - Huang C\\nAD  - West China School of Basic Medical Sciences and Forensic Medicine, Sichuan  University, Chengdu, Sichuan 610041, China.\\nFAU - Wei, Yuquan\\nAU  - Wei Y\\nAD  - State Key Laboratory of Biotherapy, West China Hospital, Sichuan University,  Chengdu, Sichuan 610041, China.\\nFAU - Gao, George Fu\\nAU  - Gao GF\\nAD  - Chinese Academy of Sciences (CAS) Key Laboratory of Pathogenic Microbiology and  Immunology, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101,  China; Savaid Medical School, University of Chinese Academy of Sciences, Beijing  101408, China; National Institute for Viral Disease Control and Prevention, Chinese  Center for Disease Control and Prevention, Beijing 102206, China; Research Network  of Immunity and Health, Beijing Institutes of Life Science, Chinese Academy of  Sciences, Beijing 100101, China.\\nFAU - Jiang, Jianxin\\nAU  - Jiang J\\nAD  - Wound Trauma Medical Center, State Key Laboratory of Trauma, Burns and Combined  Injury, Daping Hospital, Army Medical University, Chongqing 400042, China.  Electronic address: hellojjx@126.com.\\nFAU - Wu, Min\\nAU  - Wu M\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA. Electronic address:  min.wu@und.edu.\\nLA  - eng\\nGR  - R03 AI097532/AI/NIAID NIH HHS/United States\\nGR  - R01 AI138203/AI/NIAID NIH HHS/United States\\nGR  - P20 GM113123/GM/NIGMS NIH HHS/United States\\nGR  - R01 AI109317/AI/NIAID NIH HHS/United States\\nGR  - P20 GM103442/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200428\\nTA  - Mol Cell\\nJT  - Molecular cell\\nJID - 9802571\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - 63231-63-0 (RNA)\\nRN  - Leptotrichia buccalis\\nSB  - IM\\nMH  - Animals\\nMH  - Bacteria/genetics\\nMH  - Bacteriophages/genetics\\nMH  - CRISPR-Associated Proteins/*antagonists & inhibitors/genetics\\nMH  - CRISPR-Cas Systems/genetics/*physiology\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - Escherichia coli/genetics/metabolism\\nMH  - Gene Editing\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Leptotrichia/genetics/metabolism\\nMH  - RNA/genetics\\nMH  - RNA Editing/genetics/*physiology\\nPMC - PMC7299153\\nMID - NIHMS1587825\\nOTO - NOTNLM\\nOT  - *AcrVIA\\nOT  - *Cas13a\\nOT  - *RNA editing\\nOT  - *RNA targeting\\nOT  - *anti-CRISPR\\nCOIS- Declaration of Interests A provisional patent application pertaining to AcrVIA genes  for CRISPR-related technologies and their applications has been filed by the  University of North Dakota; Daping Hospital; and Army Medical University, Chongqing,  China.\\nEDAT- 2020/04/30 06:00\\nMHDA- 2020/09/20 06:00\\nCRDT- 2020/04/30 06:00\\nPHST- 2020/02/27 00:00 [received]\\nPHST- 2020/03/10 00:00 [revised]\\nPHST- 2020/03/25 00:00 [accepted]\\nPHST- 2020/04/30 06:00 [pubmed]\\nPHST- 2020/09/20 06:00 [medline]\\nPHST- 2020/04/30 06:00 [entrez]\\nAID - S1097-2765(20)30225-2 [pii]\\nAID - 10.1016/j.molcel.2020.03.033 [doi]\\nPST - ppublish\\nSO  - Mol Cell. 2020 Jun 4;78(5):850-861.e5. doi: 10.1016/j.molcel.2020.03.033. Epub 2020  Apr 28.\\n\\nPMID- 31937679\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200930\\nIS  - 2379-5077 (Print)\\nIS  - 2379-5077 (Electronic)\\nIS  - 2379-5077 (Linking)\\nVI  - 5\\nIP  - 1\\nDP  - 2020 Jan 14\\nTI  - Network-Based Prediction of Novel CRISPR-Associated Genes in Metagenomes.\\nLID - 10.1128/mSystems.00752-19 [doi]\\nLID - e00752-19\\nAB  - A diversity of clustered regularly interspaced short palindromic repeat (CRISPR)-Cas  systems provide adaptive immunity to bacteria and archaea through recording  \"memories\" of past viral infections. Recently, many novel CRISPR-associated proteins  have been discovered via computational studies, but those studies relied on biased  and incomplete databases of assembled genomes. We avoided these biases and applied a  network theory approach to search for novel CRISPR-associated genes by leveraging  subtle ecological cooccurrence patterns identified from environmental metagenomes.  We validated our method using existing annotations and discovered 32 novel  CRISPR-associated gene families. These genes span a range of putative functions,  with many potentially regulating the response to infection.IMPORTANCE Every branch  on the tree of life, including microbial life, faces the threat of viral pathogens.  Over the course of billions of years of coevolution, prokaryotes have evolved a  great diversity of strategies to defend against viral infections. One of these is  the CRISPR adaptive immune system, which allows microbes to \"remember\" past  infections in order to better fight them in the future. There has been much interest  among molecular biologists in CRISPR immunity because this system can be repurposed  as a tool for precise genome editing. Recently, a number of comparative genomics  approaches have been used to detect novel CRISPR-associated genes in databases of  genomes with great success, potentially leading to the development of new  genome-editing tools. Here, we developed novel methods to search for these distinct  classes of genes directly in environmental samples (\"metagenomes\"), thus capturing a  more complete picture of the natural diversity of CRISPR-associated genes.\\nCI  - Copyright Â© 2020 Weissman and Johnson.\\nFAU - Weissman, Jake L\\nAU  - Weissman JL\\nAUID- ORCID: 0000-0002-4237-4807\\nAD  - Department of Biology, University of Maryland, College Park, Maryland, USA.\\nFAU - Johnson, Philip L F\\nAU  - Johnson PLF\\nAUID- ORCID: 0000-0001-6087-7064\\nAD  - Department of Biology, University of Maryland, College Park, Maryland, USA  plfj@umd.edu.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200114\\nTA  - mSystems\\nJT  - mSystems\\nJID - 101680636\\nPMC - PMC6967390\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - metagenomics\\nOT  - microbial ecology\\nOT  - network\\nEDAT- 2020/01/16 06:00\\nMHDA- 2020/01/16 06:01\\nCRDT- 2020/01/16 06:00\\nPHST- 2020/01/16 06:00 [entrez]\\nPHST- 2020/01/16 06:00 [pubmed]\\nPHST- 2020/01/16 06:01 [medline]\\nAID - 5/1/e00752-19 [pii]\\nAID - mSystems00752-19 [pii]\\nAID - 10.1128/mSystems.00752-19 [doi]\\nPST - epublish\\nSO  - mSystems. 2020 Jan 14;5(1):e00752-19. doi: 10.1128/mSystems.00752-19.\\n\\nPMID- 31658261\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200203\\nLR  - 20200223\\nIS  - 1553-7358 (Electronic)\\nIS  - 1553-734X (Print)\\nIS  - 1553-734X (Linking)\\nVI  - 15\\nIP  - 10\\nDP  - 2019 Oct\\nTI  - Prediction of off-target specificity and cell-specific fitness of CRISPR-Cas System  using attention boosted deep learning and network-based gene feature.\\nPG  - e1007480\\nLID - 10.1371/journal.pcbi.1007480 [doi]\\nLID - e1007480\\nAB  - CRISPR-Cas is a powerful genome editing technology and has a great potential for in  vivo gene therapy. Successful translational application of CRISPR-Cas to biomedicine  still faces many safety concerns, including off-target side effect, cell fitness  problem after CRISPR-Cas treatment, and on-target genome editing side effect in  undesired tissues. To solve these issues, it is needed to design sgRNA with high  cell-specific efficacy and specificity. Existing single-guide RNA (sgRNA) design  tools mainly depend on a sgRNA sequence and the local information of the targeted  genome, thus are not sufficient to account for the difference in the cellular  response of the same gene in different cell types. To incorporate cell-specific  information into the sgRNA design, we develop novel interpretable machine learning  models, which integrate features learned from advanced transformer-based deep neural  network with cell-specific gene property derived from biological network and gene  expression profile, for the prediction of CRISPR-Cas9 and CRISPR-Cas12a efficacy and  specificity. In benchmark studies, our models significantly outperform  state-of-the-art algorithms. Furthermore, we find that the network-based gene  property is critical for the prediction of cell-specific post-treatment cellular  response. Our results suggest that the design of efficient and safe CRISPR-Cas needs  to consider cell-specific information of genes. Our findings may bolster developing  more accurate predictive models of CRISPR-Cas across a broad spectrum of biological  conditions as well as provide new insight into developing efficient and safe  CRISPR-based gene therapy.\\nFAU - Liu, Qiao\\nAU  - Liu Q\\nAD  - Department of Computer Science, Hunter College, The City University of New York, New  York City, NY, United States of America.\\nFAU - He, Di\\nAU  - He D\\nAUID- ORCID: 0000-0002-0908-9123\\nAD  - Ph.D. Program in Computer Science, The Graduate Center, The City University of New  York, New York City, NY, United States of America.\\nFAU - Xie, Lei\\nAU  - Xie L\\nAD  - Department of Computer Science, Hunter College, The City University of New York, New  York City, NY, United States of America.\\nAD  - Ph.D. Program in Computer Science, The Graduate Center, The City University of New  York, New York City, NY, United States of America.\\nAD  - Ph.D. Program in Biochemistry and Biology, The Graduate Center, The City University  of New York, New York City, NY, United States of America.\\nAD  - Helen and Robert Appel Alzheimer\\'s Disease Research Institute, Feil Family Brain &  Mind Research Institute, Weill Cornell Medicine, Cornell University, New York City,  NY, United States of America.\\nLA  - eng\\nGR  - R01 AG057555/AG/NIA NIH HHS/United States\\nGR  - R01 GM122845/GM/NIGMS NIH HHS/United States\\nGR  - R01 LM011986/LM/NLM NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20191028\\nTA  - PLoS Comput Biol\\nJT  - PLoS computational biology\\nJID - 101238922\\nRN  - 0 (RNA, Guide)\\nSB  - IM\\nMH  - Algorithms\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Cell Line\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Databases, Genetic\\nMH  - Deep Learning\\nMH  - Forecasting/*methods\\nMH  - Gene Editing/methods\\nMH  - Gene Regulatory Networks/genetics\\nMH  - Genome/genetics\\nMH  - Humans\\nMH  - RNA Editing/genetics\\nMH  - RNA, Guide/*genetics\\nMH  - Substrate Specificity/*genetics\\nPMC - PMC6837542\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2019/10/29 06:00\\nMHDA- 2020/02/06 06:00\\nCRDT- 2019/10/29 06:00\\nPHST- 2019/05/22 00:00 [received]\\nPHST- 2019/10/08 00:00 [accepted]\\nPHST- 2019/11/07 00:00 [revised]\\nPHST- 2019/10/29 06:00 [pubmed]\\nPHST- 2020/02/06 06:00 [medline]\\nPHST- 2019/10/29 06:00 [entrez]\\nAID - PCOMPBIOL-D-19-00817 [pii]\\nAID - 10.1371/journal.pcbi.1007480 [doi]\\nPST - epublish\\nSO  - PLoS Comput Biol. 2019 Oct 28;15(10):e1007480. doi: 10.1371/journal.pcbi.1007480.  eCollection 2019 Oct.\\n\\nPMID- 31744829\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210129\\nLR  - 20210808\\nIS  - 2159-8290 (Electronic)\\nIS  - 2159-8274 (Print)\\nIS  - 2159-8274 (Linking)\\nVI  - 10\\nIP  - 2\\nDP  - 2020 Feb\\nTI  - In Vivo Epigenetic CRISPR Screen Identifies Asf1a as an Immunotherapeutic Target in  Kras-Mutant Lung Adenocarcinoma.\\nPG  - 270-287\\nLID - 10.1158/2159-8290.CD-19-0780 [doi]\\nAB  - Despite substantial progress in lung cancer immunotherapy, the overall response rate  in patients with KRAS-mutant lung adenocarcinoma (LUAD) remains low. Combining  standard immunotherapy with adjuvant approaches that enhance adaptive immune  responses-such as epigenetic modulation of antitumor immunity-is therefore an  attractive strategy. To identify epigenetic regulators of tumor immunity, we  constructed an epigenetic-focused single guide RNA library and performed an in vivo  CRISPR screen in a Kras (G12D)/Trp53 (-/-) LUAD model. Our data showed that loss of  the histone chaperone Asf1a in tumor cells sensitizes tumors to anti-PD-1 treatment.  Mechanistic studies revealed that tumor cell-intrinsic Asf1a deficiency induced  immunogenic macrophage differentiation in the tumor microenvironment by upregulating  GM-CSF expression and potentiated T-cell activation in combination with anti-PD-1.  Our results provide a rationale for a novel combination therapy consisting of ASF1A  inhibition and anti-PD-1 immunotherapy. SIGNIFICANCE: Using an in vivo epigenetic  CRISPR screen, we identified Asf1a as a critical regulator of LUAD sensitivity to  anti-PD-1 therapy. Asf1a deficiency synergized with anti-PD-1 immunotherapy by  promoting M1-like macrophage polarization and T-cell activation. Thus, we provide a  new immunotherapeutic strategy for this subtype of patients with LUAD.See related  commentary by Menzel and Black, p. 179.This article is highlighted in the In This  Issue feature, p. 161.\\nCI  - Â©2019 American Association for Cancer Research.\\nFAU - Li, Fei\\nAU  - Li F\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Huang, Qingyuan\\nAU  - Huang Q\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Luster, Troy A\\nAU  - Luster TA\\nAD  - Belfer Center for Applied Cancer Science, Dana-Farber Cancer Institute, Boston,  Massachusetts.\\nFAU - Hu, Hai\\nAU  - Hu H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Zhang, Hua\\nAU  - Zhang H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Ng, Wai-Lung\\nAU  - Ng WL\\nAUID- ORCID: 0000-0003-2892-6318\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nAD  - Department of Medicine, Harvard Medical School, Boston, Massachusetts.\\nAD  - School of Pharmacy, Faculty of Medicine, The Chinese University of Hong Kong, Sha  Tin, Hong Kong SAR.\\nFAU - Khodadadi-Jamayran, Alireza\\nAU  - Khodadadi-Jamayran A\\nAD  - Applied Bioinformatics Laboratories and Genome Technology Center, Division of  Advanced Research Technologies, New York University Grossman School of Medicine, NYU  Langone Health, New York, New York.\\nFAU - Wang, Wei\\nAU  - Wang W\\nAD  - S. Arthur Localio Laboratory, Department of Surgery, New York University Grossman  School of Medicine, NYU Langone Health, New York, New York.\\nFAU - Chen, Ting\\nAU  - Chen T\\nAUID- ORCID: 0000-0003-1725-7956\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Deng, Jiehui\\nAU  - Deng J\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Ranieri, Michela\\nAU  - Ranieri M\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Fang, Zhaoyuan\\nAU  - Fang Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Pyon, Val\\nAU  - Pyon V\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Dowling, CatrÃ\\xadona M\\nAU  - Dowling CM\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Bagdatlioglu, Ece\\nAU  - Bagdatlioglu E\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Almonte, Christina\\nAU  - Almonte C\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Labbe, Kristen\\nAU  - Labbe K\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Silver, Heather\\nAU  - Silver H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Rabin, Alexandra R\\nAU  - Rabin AR\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Jani, Kandarp\\nAU  - Jani K\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Tsirigos, Aristotelis\\nAU  - Tsirigos A\\nAUID- ORCID: 0000-0002-7512-8477\\nAD  - Applied Bioinformatics Laboratories and Genome Technology Center, Division of  Advanced Research Technologies, New York University Grossman School of Medicine, NYU  Langone Health, New York, New York.\\nAD  - Department of Pathology, New York University Grossman School of Medicine, NYU  Langone Health, New York, New York.\\nFAU - Papagiannakopoulos, Thales\\nAU  - Papagiannakopoulos T\\nAD  - Department of Pathology, New York University Grossman School of Medicine, NYU  Langone Health, New York, New York.\\nFAU - Hammerman, Peter S\\nAU  - Hammerman PS\\nAD  - Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nFAU - Velcheti, Vamsidhar\\nAU  - Velcheti V\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Freeman, Gordon J\\nAU  - Freeman GJ\\nAD  - Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nAD  - Department of Medicine, Brigham and Women\\'s Hospital, Harvard Medical School,  Boston, Massachusetts.\\nFAU - Qi, Jun\\nAU  - Qi J\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nAD  - Department of Medicine, Harvard Medical School, Boston, Massachusetts.\\nFAU - Miller, George\\nAU  - Miller G\\nAD  - S. Arthur Localio Laboratory, Department of Surgery, New York University Grossman  School of Medicine, NYU Langone Health, New York, New York.\\nFAU - Wong, Kwok-Kin\\nAU  - Wong KK\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York. kwok-kin.wong@nyumc.org.\\nLA  - eng\\nGR  - R01 CA216188/CA/NCI NIH HHS/United States\\nGR  - P50 CA101942/CA/NCI NIH HHS/United States\\nGR  - R01 CA219670/CA/NCI NIH HHS/United States\\nGR  - U01 CA233084/CA/NCI NIH HHS/United States\\nGR  - R01 CA166480/CA/NCI NIH HHS/United States\\nGR  - R01 CA222218/CA/NCI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20191119\\nTA  - Cancer Discov\\nJT  - Cancer discovery\\nJID - 101561693\\nRN  - 0 (ASF1A protein, human)\\nRN  - 0 (Asf1a protein, mouse)\\nRN  - 0 (Cell Cycle Proteins)\\nRN  - 0 (Immune Checkpoint Inhibitors)\\nRN  - 0 (KRAS protein, human)\\nRN  - 0 (Molecular Chaperones)\\nRN  - 0 (Pdcd1 protein, mouse)\\nRN  - 0 (Programmed Cell Death 1 Receptor)\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (RNA, Small Interfering)\\nRN  - 0 (Trp53 protein, mouse)\\nRN  - 0 (Tumor Suppressor Protein p53)\\nRN  - EC 3.6.5.2 (Hras protein, mouse)\\nRN  - EC 3.6.5.2 (Proto-Oncogene Proteins p21(ras))\\nSB  - IM\\nCIN - Cancer Discov. 2020 Feb;10(2):179-181. PMID: 32024681\\nMH  - Adenocarcinoma of Lung/*drug therapy/genetics/immunology/pathology\\nMH  - Animals\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Cell Cycle Proteins/genetics/*metabolism\\nMH  - Cell Differentiation/drug effects/genetics\\nMH  - Cell Line, Tumor\\nMH  - Disease Models, Animal\\nMH  - Drug Resistance, Neoplasm/*genetics\\nMH  - Epigenesis, Genetic/immunology\\nMH  - Gene Expression Regulation, Neoplastic/immunology\\nMH  - Gene Knockout Techniques\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Immune Checkpoint Inhibitors/*pharmacology/therapeutic use\\nMH  - Lung/pathology\\nMH  - Lung Neoplasms/*drug therapy/genetics/immunology/pathology\\nMH  - Macrophages/drug effects/immunology/metabolism\\nMH  - Male\\nMH  - Mice\\nMH  - Molecular Chaperones/genetics/*metabolism\\nMH  - Programmed Cell Death 1 Receptor/antagonists & inhibitors/immunology\\nMH  - Proto-Oncogene Proteins p21(ras)/genetics\\nMH  - RNA, Guide/genetics\\nMH  - RNA, Small Interfering/metabolism\\nMH  - RNA-Seq\\nMH  - Tumor Microenvironment/drug effects/immunology\\nMH  - Tumor Suppressor Protein p53/genetics\\nPMC - PMC7007372\\nMID - NIHMS1544097\\nCOIS- No potential conflicts of interest were disclosed by the other authors.\\nEDAT- 2019/11/21 06:00\\nMHDA- 2021/01/30 06:00\\nCRDT- 2019/11/21 06:00\\nPHST- 2019/07/03 00:00 [received]\\nPHST- 2019/10/11 00:00 [revised]\\nPHST- 2019/11/13 00:00 [accepted]\\nPHST- 2019/11/21 06:00 [pubmed]\\nPHST- 2021/01/30 06:00 [medline]\\nPHST- 2019/11/21 06:00 [entrez]\\nAID - 2159-8290.CD-19-0780 [pii]\\nAID - 10.1158/2159-8290.CD-19-0780 [doi]\\nPST - ppublish\\nSO  - Cancer Discov. 2020 Feb;10(2):270-287. doi: 10.1158/2159-8290.CD-19-0780. Epub 2019  Nov 19.\\n\\nPMID- 34206474\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210725\\nIS  - 2079-6382 (Print)\\nIS  - 2079-6382 (Electronic)\\nIS  - 2079-6382 (Linking)\\nVI  - 10\\nIP  - 7\\nDP  - 2021 Jun 22\\nTI  - CRISPR-Cas, a Revolution in the Treatment and Study of ESKAPE Infections:  Pre-Clinical Studies.\\nLID - 10.3390/antibiotics10070756 [doi]\\nLID - 756\\nAB  - One of the biggest threats we face globally is the emergence of  antimicrobial-resistant (AMR) bacteria, which runs in parallel with the lack in the  development of new antimicrobials. Among these AMR bacteria pathogens belonging to  the ESKAPE group can be highlighted (Enterococcus spp., Staphylococcus aureus,  Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and  Enterobacter spp.) due to their profile of drug resistance and virulence. Therefore,  innovative lines of treatment must be developed for these bacteria. In this review,  we summarize the different strategies for the treatment and study of molecular  mechanisms of AMR in the ESKAPE pathogens based on the clustered regularly  interspaced short palindromic repeats (CRISPR) and CRISPR-associated (Cas) proteins\\'  technologies: loss of plasmid or cellular viability, random mutation or gene  deletion as well directed mutations that lead to a gene\\'s loss of function.\\nFAU - GonzÃ¡lez de Aledo, Manuel\\nAU  - GonzÃ¡lez de Aledo M\\nAUID- ORCID: 0000-0001-7222-4941\\nAD  - Microbiology Department-A CoruÃ±a University Hospital (CHUAC), 15006 A CoruÃ±a, Spain.\\nFAU - GonzÃ¡lez-Bardanca, MÃ³nica\\nAU  - GonzÃ¡lez-Bardanca M\\nAD  - Microbiology Department-A CoruÃ±a University Hospital (CHUAC), 15006 A CoruÃ±a, Spain.\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - Blasco, LucÃ\\xada\\nAU  - Blasco L\\nAUID- ORCID: 0000-0002-4039-4142\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - Pacios, Olga\\nAU  - Pacios O\\nAUID- ORCID: 0000-0002-4476-856X\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - Bleriot, InÃ©s\\nAU  - Bleriot I\\nAUID- ORCID: 0000-0002-1846-4693\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - FernÃ¡ndez-GarcÃ\\xada, Laura\\nAU  - FernÃ¡ndez-GarcÃ\\xada L\\nAUID- ORCID: 0000-0002-8531-6105\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - FernÃ¡ndez-Quejo, Melisa\\nAU  - FernÃ¡ndez-Quejo M\\nAD  - Microbiology Department-A CoruÃ±a University Hospital (CHUAC), 15006 A CoruÃ±a, Spain.\\nFAU - LÃ³pez, MarÃ\\xada\\nAU  - LÃ³pez M\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nFAU - Bou, GermÃ¡n\\nAU  - Bou G\\nAD  - Microbiology Department-A CoruÃ±a University Hospital (CHUAC), 15006 A CoruÃ±a, Spain.\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nAD  - Spanish Network for Research in Infectious Diseases (REIPI), 41071 Seville, Spain.\\nFAU - TomÃ¡s, MarÃ\\xada\\nAU  - TomÃ¡s M\\nAUID- ORCID: 0000-0003-4501-0387\\nAD  - Microbiology Department-A CoruÃ±a University Hospital (CHUAC), 15006 A CoruÃ±a, Spain.\\nAD  - A CoruÃ±a Biomedical Research Institute (INIBIC), University of A CoruÃ±a (UDC), 15006  A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) on  Behalf of the Spanish Society of Infectious Diseases and Clinical Microbiology  (SEIMC), 28003 Madrid, Spain.\\nAD  - Spanish Network for Research in Infectious Diseases (REIPI), 41071 Seville, Spain.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210622\\nTA  - Antibiotics (Basel)\\nJT  - Antibiotics (Basel, Switzerland)\\nJID - 101637404\\nPMC - PMC8300728\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - ESKAPE pathogens\\nOT  - treatment\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2021/07/03 06:00\\nMHDA- 2021/07/03 06:01\\nCRDT- 2021/07/02 01:28\\nPHST- 2021/05/24 00:00 [received]\\nPHST- 2021/06/18 00:00 [revised]\\nPHST- 2021/06/19 00:00 [accepted]\\nPHST- 2021/07/02 01:28 [entrez]\\nPHST- 2021/07/03 06:00 [pubmed]\\nPHST- 2021/07/03 06:01 [medline]\\nAID - antibiotics10070756 [pii]\\nAID - antibiotics-10-00756 [pii]\\nAID - 10.3390/antibiotics10070756 [doi]\\nPST - epublish\\nSO  - Antibiotics (Basel). 2021 Jun 22;10(7):756. doi: 10.3390/antibiotics10070756.\\n\\nPMID- 32008085\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200710\\nLR  - 20200710\\nIS  - 1420-9071 (Electronic)\\nIS  - 1420-682X (Linking)\\nVI  - 77\\nIP  - 14\\nDP  - 2020 Jul\\nTI  - EMT signaling: potential contribution of CRISPR/Cas gene editing.\\nPG  - 2701-2722\\nLID - 10.1007/s00018-020-03449-3 [doi]\\nAB  - Epithelial to mesenchymal transition (EMT) is a complex plastic and reversible  cellular process that has critical roles in diverse physiological and pathological  phenomena. EMT is involved in embryonic development, organogenesis and tissue  repair, as well as in fibrosis, cancer metastasis and drug resistance. In recent  years, the ability to edit the genome using the clustered regularly interspaced  palindromic repeats (CRISPR) and associated protein (Cas) system has greatly  contributed to identify or validate critical genes in pathway signaling. This review  delineates the complex EMT networks and discusses recent studies that have used  CRISPR/Cas technology to further advance our understanding of the EMT process.\\nFAU - Mohammadinejad, Reza\\nAU  - Mohammadinejad R\\nAD  - Pharmaceutics Research Center, Institute of Neuropharmacology, Kerman University of  Medical Sciences, Kerman, Iran.\\nFAU - Biagioni, Alessio\\nAU  - Biagioni A\\nAD  - Section of Experimental Pathology and Oncology, Department of Experimental and  Clinical Biomedical Sciences \"Mario Serio\", University of Florence, Florence, Italy.\\nFAU - Arunkumar, Ganesan\\nAU  - Arunkumar G\\nAD  - Center for Cancer Research, National Cancer Institute, National Institutes of  Health, Bethesda, MD, USA.\\nFAU - Shapiro, Rebecca\\nAU  - Shapiro R\\nAD  - Department of Molecular and Cellular Biology, University of Guelph, Guelph, ON,  Canada.\\nFAU - Chang, Kun-Che\\nAU  - Chang KC\\nAD  - Department of Ophthalmology, School of Medicine, Byers Eye Institute, Stanford  University, Palo Alto, CA, 94303, USA.\\nFAU - Sedeeq, Mohammed\\nAU  - Sedeeq M\\nAD  - Division of Pharmacy, College of Health and Medicine, University of Tasmania,  Hobart, TAS, Australia.\\nFAU - Taiyab, Aftab\\nAU  - Taiyab A\\nAD  - Department of Pathology and Molecular Medicine, McMaster University, Hamilton, ON,  L8S 4L8, Canada.\\nFAU - Hashemabadi, Mohammad\\nAU  - Hashemabadi M\\nAD  - Department of Biology, Faculty of Sciences, Shahid Bahonar University, Kerman, Iran.\\nAD  - Department of Genetics, Faculty of Biological Sciences, Tarbiat Modares University,  Tehran, Iran.\\nFAU - Pardakhty, Abbas\\nAU  - Pardakhty A\\nAD  - Neuroscience Research Center, Institute of Neuropharmacology, Kerman University of  Medical Sciences, Kerman, Iran.\\nFAU - Mandegary, Ali\\nAU  - Mandegary A\\nAD  - Physiology Research Center, Institute of Neuropharmacology and Department of  Toxicology & Pharmacology, Kerman University of Medical Sciences, Kerman, Iran.\\nFAU - Thiery, Jean-Paul\\nAU  - Thiery JP\\nAD  - Guangzhou Regenerative Medicine and Health, Guangdong Laboratory, Guangzhou, China.\\nFAU - Aref, Amir Reza\\nAU  - Aref AR\\nAD  - Department of Medical Oncology, Belfer Center for Applied Cancer Science,  Dana-Farber Cancer Institute, Harvard Medical School, Boston, MA, 02215, USA.  Amir_aref@hms.harvard.edu.\\nFAU - Azimi, Iman\\nAU  - Azimi I\\nAUID- ORCID: 0000-0001-9477-9999\\nAD  - Division of Pharmacy, College of Health and Medicine, University of Tasmania,  Hobart, TAS, Australia. iman.azimi@utas.edu.au.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20200201\\nPL  - Switzerland\\nTA  - Cell Mol Life Sci\\nJT  - Cellular and molecular life sciences : CMLS\\nJID - 9705402\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Embryonic Development/genetics\\nMH  - Epithelial-Mesenchymal Transition/*genetics\\nMH  - Gene Editing/*methods\\nMH  - Humans\\nMH  - Organogenesis/genetics\\nMH  - Signal Transduction/genetics\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Cancer\\nOT  - Epithelialâ€“mesenchymal transition\\nOT  - Gene editing\\nEDAT- 2020/02/03 06:00\\nMHDA- 2020/07/11 06:00\\nCRDT- 2020/02/03 06:00\\nPHST- 2019/06/03 00:00 [received]\\nPHST- 2020/01/02 00:00 [accepted]\\nPHST- 2019/12/24 00:00 [revised]\\nPHST- 2020/02/03 06:00 [pubmed]\\nPHST- 2020/07/11 06:00 [medline]\\nPHST- 2020/02/03 06:00 [entrez]\\nAID - 10.1007/s00018-020-03449-3 [pii]\\nAID - 10.1007/s00018-020-03449-3 [doi]\\nPST - ppublish\\nSO  - Cell Mol Life Sci. 2020 Jul;77(14):2701-2722. doi: 10.1007/s00018-020-03449-3. Epub  2020 Feb 1.\\n\\nPMID- 31004423\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190523\\nLR  - 20190523\\nIS  - 2165-0497 (Electronic)\\nIS  - 2165-0497 (Linking)\\nVI  - 7\\nIP  - 2\\nDP  - 2019 Mar\\nTI  - Noncoding RNA.\\nLID - 10.1128/microbiolspec.GPP3-0038-2018 [doi]\\nAB  - Regulatory RNAs, present in many bacterial genomes and particularly in pathogenic  bacteria such as Staphylococcus aureus, control the expression of genes encoding  virulence factors or metabolic proteins. They are extremely diverse and include  noncoding RNAs (sRNA), antisense RNAs, and some 5\\' or 3\\' untranslated regions of  messenger RNAs that act as sensors for metabolites, tRNAs, or environmental  conditions (e.g., temperature, pH). In this review we focus on specific examples of  sRNAs of S. aureus that illustrate how numerous sRNAs and associated proteins are  embedded in complex networks of regulation. In addition, we discuss the CRISPR-Cas  systems defined as an RNA-interference-like mechanism, which also exist in  staphylococcal strains.\\nFAU - Desgranges, E\\nAU  - Desgranges E\\nAD  - UniversitÃ© de Strasbourg, CNRS, Architecture et RÃ©activitÃ© de l\\'ARN, F-67000  Strasbourg, France.\\nFAU - Marzi, S\\nAU  - Marzi S\\nAD  - UniversitÃ© de Strasbourg, CNRS, Architecture et RÃ©activitÃ© de l\\'ARN, F-67000  Strasbourg, France.\\nFAU - Moreau, K\\nAU  - Moreau K\\nAD  - CIRI, International Center for Infectiology Research, Inserm, UniversitÃ© Claude  Bernard Lyon 1, CNRS, UMR5308, Ã‰cole Normale SupÃ©rieure de Lyon, Hospices Civils de  Lyon, University of Lyon, F-69008, Lyon, France.\\nFAU - Romby, P\\nAU  - Romby P\\nAD  - UniversitÃ© de Strasbourg, CNRS, Architecture et RÃ©activitÃ© de l\\'ARN, F-67000  Strasbourg, France.\\nFAU - Caldelari, I\\nAU  - Caldelari I\\nAD  - UniversitÃ© de Strasbourg, CNRS, Architecture et RÃ©activitÃ© de l\\'ARN, F-67000  Strasbourg, France.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nPL  - United States\\nTA  - Microbiol Spectr\\nJT  - Microbiology spectrum\\nJID - 101634614\\nRN  - 0 (RNA, Bacterial)\\nRN  - 0 (RNA, Small Untranslated)\\nSB  - IM\\nMH  - Animals\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - Genome, Bacterial\\nMH  - Humans\\nMH  - RNA, Bacterial/genetics/*metabolism\\nMH  - RNA, Small Untranslated/*metabolism\\nMH  - Staphylococcal Infections/microbiology\\nMH  - Staphylococcus aureus/*genetics/metabolism\\nEDAT- 2019/04/21 06:00\\nMHDA- 2019/05/24 06:00\\nCRDT- 2019/04/21 06:00\\nPHST- 2019/04/21 06:00 [entrez]\\nPHST- 2019/04/21 06:00 [pubmed]\\nPHST- 2019/05/24 06:00 [medline]\\nAID - 10.1128/microbiolspec.GPP3-0038-2018 [doi]\\nPST - ppublish\\nSO  - Microbiol Spectr. 2019 Mar;7(2). doi: 10.1128/microbiolspec.GPP3-0038-2018.\\n\\nPMID- 32794766\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210810\\nLR  - 20210810\\nIS  - 2161-5063 (Electronic)\\nIS  - 2161-5063 (Linking)\\nVI  - 9\\nIP  - 9\\nDP  - 2020 Sep 18\\nTI  - Applications of CRISPR in a Microbial Cell Factory: From Genome Reconstruction to  Metabolic Network Reprogramming.\\nPG  - 2228-2238\\nLID - 10.1021/acssynbio.0c00349 [doi]\\nAB  - The well-designed microbial cell factory finds wide applications in chemical,  pharmaceutical, and food industries due to its sustainable and environmentally  friendly features. Recently, the clustered regularly interspaced short palindromic  repeats (CRISPR) and CRISPR-associated proteins (CRISPR-Cas) systems have been  developed into powerful tools to perform genome editing and transcriptional  regulation in prokaryotic and eukaryotic cells. Accordingly, these tools are useful  to build microbial cell factories not only by reconstructing the genome but also by  reprogramming the metabolic network. In this review, we summarize the recent  significant headway and potential uses of the CRISPR technology in the construction  of efficient microbial cell factories. Moreover, the future perspectives on the  improvement and upgradation of CRISPR-based tools are also discussed.\\nFAU - Wu, Yaokang\\nAU  - Wu Y\\nAUID- ORCID: 0000-0003-2415-5739\\nAD  - Key Laboratory of Carbohydrate Chemistry and Biotechnology, Ministry of Education,  Jiangnan University, Wuxi, 214122, China.\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nFAU - Liu, Yanfeng\\nAU  - Liu Y\\nAD  - Key Laboratory of Carbohydrate Chemistry and Biotechnology, Ministry of Education,  Jiangnan University, Wuxi, 214122, China.\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nFAU - Lv, Xueqin\\nAU  - Lv X\\nAD  - Key Laboratory of Carbohydrate Chemistry and Biotechnology, Ministry of Education,  Jiangnan University, Wuxi, 214122, China.\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nFAU - Li, Jianghua\\nAU  - Li J\\nAD  - Key Laboratory of Carbohydrate Chemistry and Biotechnology, Ministry of Education,  Jiangnan University, Wuxi, 214122, China.\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nFAU - Du, Guocheng\\nAU  - Du G\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nFAU - Liu, Long\\nAU  - Liu L\\nAUID- ORCID: 0000-0002-9679-9130\\nAD  - Key Laboratory of Carbohydrate Chemistry and Biotechnology, Ministry of Education,  Jiangnan University, Wuxi, 214122, China.\\nAD  - Science Center for Future Foods, Jiangnan University, Wuxi, 214122, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20200831\\nPL  - United States\\nTA  - ACS Synth Biol\\nJT  - ACS synthetic biology\\nJID - 101575075\\nSB  - IM\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Escherichia coli/genetics/metabolism\\nMH  - Gene Editing\\nMH  - *Genome\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/*genetics\\nMH  - Saccharomyces cerevisiae/genetics/metabolism\\nOTO - NOTNLM\\nOT  - *CRISPR\\nOT  - *genome editing\\nOT  - *metabolic engineering\\nOT  - *microbial cell factory\\nOT  - *synthetic biology\\nOT  - *transcriptional regulation\\nEDAT- 2020/08/17 06:00\\nMHDA- 2021/08/11 06:00\\nCRDT- 2020/08/16 06:00\\nPHST- 2020/08/17 06:00 [pubmed]\\nPHST- 2021/08/11 06:00 [medline]\\nPHST- 2020/08/16 06:00 [entrez]\\nAID - 10.1021/acssynbio.0c00349 [doi]\\nPST - ppublish\\nSO  - ACS Synth Biol. 2020 Sep 18;9(9):2228-2238. doi: 10.1021/acssynbio.0c00349. Epub  2020 Aug 31.\\n\\nPMID- 33077929\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210111\\nLR  - 20210519\\nIS  - 2397-334X (Electronic)\\nIS  - 2397-334X (Linking)\\nVI  - 4\\nIP  - 12\\nDP  - 2020 Dec\\nTI  - The network structure and eco-evolutionary dynamics of CRISPR-induced immune  diversification.\\nPG  - 1650-1660\\nLID - 10.1038/s41559-020-01312-z [doi]\\nAB  - As a heritable sequence-specific adaptive immune system, CRISPR-Cas is a powerful  force shaping strain diversity in host-virus systems. While the diversity of CRISPR  alleles has been explored, the associated structure and dynamics of host-virus  interactions have not. We explore the role of CRISPR in mediating the interplay  between host-virus interaction structure and eco-evolutionary dynamics in a  computational model and compare the results with three empirical datasets from  natural systems. We show that the structure of the networks describing who infects  whom and the degree to which strains are immune, are respectively modular  (containing groups of hosts and viruses that interact strongly) and weighted-nested  (specialist hosts are more susceptible to subsets of viruses that in turn also  infect the more generalist hosts with many spacers matching many viruses). The  dynamic interplay between these networks influences transitions between dynamical  regimes of virus diversification and host control. The three empirical systems  exhibit weighted-nested immunity networks, a pattern our theory shows is indicative  of hosts able to suppress virus diversification. Previously missing from studies of  microbial host-pathogen systems, the immunity network plays a key role in the  coevolutionary dynamics.\\nFAU - Pilosof, Shai\\nAU  - Pilosof S\\nAUID- ORCID: 0000-0003-0430-5568\\nAD  - Department of Life Sciences, Ben-Gurion University of the Negev, Beer-Sheva, Israel.\\nAD  - Department of Ecology and Evolution, University of Chicago, Chicago, IL, USA.\\nFAU - AlcalÃ¡-Corona, Sergio A\\nAU  - AlcalÃ¡-Corona SA\\nAUID- ORCID: 0000-0001-8643-9214\\nAD  - Department of Ecology and Evolution, University of Chicago, Chicago, IL, USA.\\nFAU - Wang, Tong\\nAU  - Wang T\\nAUID- ORCID: 0000-0001-9900-0307\\nAD  - Department of Physics, Loomis Laboratory of Physics, University of Illinois at  Urbana-Champaign, Urbana, IL, USA.\\nAD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at  Urbana-Champaign, Urbana, IL, USA.\\nFAU - Kim, Ted\\nAU  - Kim T\\nAD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at  Urbana-Champaign, Urbana, IL, USA.\\nAD  - Department of Microbiology, University of Illinois at Urbana-Champaign, Urbana, IL,  USA.\\nFAU - Maslov, Sergei\\nAU  - Maslov S\\nAUID- ORCID: 0000-0002-3701-492X\\nAD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at  Urbana-Champaign, Urbana, IL, USA.\\nAD  - Department of Bioengineering, University of Illinois at Urbana-Champaign, Urbana,  IL, USA.\\nFAU - Whitaker, Rachel\\nAU  - Whitaker R\\nAUID- ORCID: 0000-0003-2263-5798\\nAD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at  Urbana-Champaign, Urbana, IL, USA. rwhitaker@life.illinois.edu.\\nAD  - Department of Microbiology, University of Illinois at Urbana-Champaign, Urbana, IL,  USA. rwhitaker@life.illinois.edu.\\nFAU - Pascual, Mercedes\\nAU  - Pascual M\\nAUID- ORCID: 0000-0003-3575-7233\\nAD  - Department of Ecology and Evolution, University of Chicago, Chicago, IL, USA.  pascualmm@uchicago.edu.\\nAD  - Santa Fe Institute, Santa Fe, NM, USA. pascualmm@uchicago.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20201019\\nPL  - England\\nTA  - Nat Ecol Evol\\nJT  - Nature ecology & evolution\\nJID - 101698577\\nSB  - IM\\nMH  - *Biological Evolution\\nMH  - *Clustered Regularly Interspaced Short Palindromic Repeats\\nEDAT- 2020/10/21 06:00\\nMHDA- 2021/01/12 06:00\\nCRDT- 2020/10/20 06:23\\nPHST- 2020/02/02 00:00 [received]\\nPHST- 2020/08/14 00:00 [accepted]\\nPHST- 2020/10/21 06:00 [pubmed]\\nPHST- 2021/01/12 06:00 [medline]\\nPHST- 2020/10/20 06:23 [entrez]\\nAID - 10.1038/s41559-020-01312-z [pii]\\nAID - 10.1038/s41559-020-01312-z [doi]\\nPST - ppublish\\nSO  - Nat Ecol Evol. 2020 Dec;4(12):1650-1660. doi: 10.1038/s41559-020-01312-z. Epub 2020  Oct 19.\\n\\nPMID- 29335603\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190515\\nLR  - 20200903\\nIS  - 1546-1726 (Electronic)\\nIS  - 1097-6256 (Linking)\\nVI  - 21\\nIP  - 3\\nDP  - 2018 Mar\\nTI  - In vivo simultaneous transcriptional activation of multiple genes in the brain using  CRISPR-dCas9-activator transgenic mice.\\nPG  - 440-446\\nLID - 10.1038/s41593-017-0060-6 [doi]\\nAB  - Despite rapid progresses in the genome-editing field, in vivo simultaneous  overexpression of multiple genes remains challenging. We generated a transgenic  mouse using an improved dCas9 system that enables simultaneous and precise in vivo  transcriptional activation of multiple genes and long noncoding RNAs in the nervous  system. As proof of concept, we were able to use targeted activation of endogenous  neurogenic genes in these transgenic mice to directly and efficiently convert  astrocytes into functional neurons in vivo. This system provides a flexible and  rapid screening platform for studying complex gene networks and gain-of-function  phenotypes in the mammalian brain.\\nFAU - Zhou, Haibo\\nAU  - Zhou H\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Liu, Junlai\\nAU  - Liu J\\nAD  - School of Life Science and Technology, ShanghaiTech University, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nAD  - Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences,  Shanghai, China.\\nFAU - Zhou, Changyang\\nAU  - Zhou C\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Gao, Ni\\nAU  - Gao N\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Rao, Zhiping\\nAU  - Rao Z\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Li, He\\nAU  - Li H\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Hu, Xinde\\nAU  - Hu X\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Li, Changlin\\nAU  - Li C\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Yao, Xuan\\nAU  - Yao X\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Shen, Xiaowen\\nAU  - Shen X\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Sun, Yidi\\nAU  - Sun Y\\nAD  - Key Lab of Computational Biology, CAS-MPG Partner Institute for Computational  Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences,  Shanghai, China.\\nFAU - Wei, Yu\\nAU  - Wei Y\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Liu, Fei\\nAU  - Liu F\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Ying, Wenqin\\nAU  - Ying W\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Zhang, Junming\\nAU  - Zhang J\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Tang, Cheng\\nAU  - Tang C\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Zhang, Xu\\nAU  - Zhang X\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Xu, Huatai\\nAU  - Xu H\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Shi, Linyu\\nAU  - Shi L\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Cheng, Leping\\nAU  - Cheng L\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Huang, Pengyu\\nAU  - Huang P\\nAUID- ORCID: 0000-0001-8394-7280\\nAD  - School of Life Science and Technology, ShanghaiTech University, Shanghai, China.  huangpy@shanghaitech.edu.cn.\\nFAU - Yang, Hui\\nAU  - Yang H\\nAUID- ORCID: 0000-0003-3590-722X\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Center for  Excellence in Brain Science and Intelligence Technology, Shanghai Institutes for  Biological Sciences, Chinese Academy of Sciences, Shanghai, China.  huiyang@ion.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180115\\nPL  - United States\\nTA  - Nat Neurosci\\nJT  - Nature neuroscience\\nJID - 9809671\\nRN  - 0 (DNA-Binding Proteins)\\nRN  - 0 (Nerve Tissue Proteins)\\nRN  - 0 (NeuN protein, mouse)\\nRN  - 0 (Nuclear Proteins)\\nRN  - 0 (RNA, Long Noncoding)\\nSB  - IM\\nMH  - Animals\\nMH  - Astrocytes/physiology\\nMH  - Brain Chemistry/*genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - DNA-Binding Proteins\\nMH  - Female\\nMH  - Male\\nMH  - Mice\\nMH  - Mice, Transgenic\\nMH  - Nerve Tissue Proteins/metabolism\\nMH  - Nervous System/metabolism\\nMH  - Neurons/physiology\\nMH  - Nuclear Proteins/metabolism\\nMH  - Primary Cell Culture\\nMH  - RNA, Long Noncoding/genetics\\nMH  - Transcriptional Activation/*genetics\\nEDAT- 2018/01/18 06:00\\nMHDA- 2019/05/16 06:00\\nCRDT- 2018/01/17 06:00\\nPHST- 2017/04/05 00:00 [received]\\nPHST- 2017/12/07 00:00 [accepted]\\nPHST- 2018/01/18 06:00 [pubmed]\\nPHST- 2019/05/16 06:00 [medline]\\nPHST- 2018/01/17 06:00 [entrez]\\nAID - 10.1038/s41593-017-0060-6 [pii]\\nAID - 10.1038/s41593-017-0060-6 [doi]\\nPST - ppublish\\nSO  - Nat Neurosci. 2018 Mar;21(3):440-446. doi: 10.1038/s41593-017-0060-6. Epub 2018 Jan  15.\\n\\nPMID- 33152516\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211124\\nLR  - 20211124\\nIS  - 1096-7184 (Electronic)\\nIS  - 1096-7176 (Linking)\\nVI  - 63\\nDP  - 2021 Jan\\nTI  - CRISPR-based metabolic pathway engineering.\\nPG  - 148-159\\nLID - S1096-7176(20)30159-2 [pii]\\nLID - 10.1016/j.ymben.2020.10.004 [doi]\\nAB  - A highly effective metabolic pathway is the key for an efficient cell factory.  However, the engineered homologous or heterologous multi-gene pathway may be  unbalanced, inefficient and causing the accumulation of potentially toxic  intermediates. Therefore, pathways must be constructed optimally to minimize these  negative effects and maximize catalytic efficiency. With the development of CRISPR  technology, some of the problems of previous pathway engineering and genome editing  techniques were resolved, providing higher efficiency, lower cost, and easily  customizable targets. Moreover, CRISPR was demonstrated as robust and effective in  various organisms including both prokaryotes and eukaryotes. In recent years,  researchers in the field of metabolic engineering and synthetic biology have  exploited various CRISPR-based pathway engineering approaches, which are both  effective and convenient, as well as valuable from a theoretical standpoint. In this  review, we systematically summarize novel pathway engineering techniques and  strategies based on CRISPR nucleases system, CRISPR interference (CRISPRi), and  CRISPR activation (CRISPRa), including figures and descriptions for easy  understanding, with the aim to facilitate their broader application among fellow  researchers.\\nCI  - Copyright Â© 2020. Published by Elsevier Inc.\\nFAU - Zhao, Dongdong\\nAU  - Zhao D\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China.\\nFAU - Zhu, Xinna\\nAU  - Zhu X\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China.\\nFAU - Zhou, Hang\\nAU  - Zhou H\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China.\\nFAU - Sun, Naxin\\nAU  - Sun N\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China.\\nFAU - Wang, Ting\\nAU  - Wang T\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China; University of Chinese Academy of Sciences,  Beijing, 100049, China.\\nFAU - Bi, Changhao\\nAU  - Bi C\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China. Electronic address: bi_ch@tib.cas.cn.\\nFAU - Zhang, Xueli\\nAU  - Zhang X\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, China; Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of  Sciences, Tianjin, 300308, China. Electronic address: zhang_xl@tib.cas.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20201102\\nPL  - Belgium\\nTA  - Metab Eng\\nJT  - Metabolic engineering\\nJID - 9815657\\nSB  - IM\\nMH  - *CRISPR-Cas Systems/genetics\\nMH  - Gene Editing\\nMH  - *Metabolic Engineering\\nMH  - Metabolic Networks and Pathways/genetics\\nOTO - NOTNLM\\nOT  - *Crispr\\nOT  - *Genome editing\\nOT  - *Pathway engineering\\nEDAT- 2020/11/06 06:00\\nMHDA- 2021/11/25 06:00\\nCRDT- 2020/11/05 20:12\\nPHST- 2020/06/29 00:00 [received]\\nPHST- 2020/10/27 00:00 [revised]\\nPHST- 2020/10/28 00:00 [accepted]\\nPHST- 2020/11/06 06:00 [pubmed]\\nPHST- 2021/11/25 06:00 [medline]\\nPHST- 2020/11/05 20:12 [entrez]\\nAID - S1096-7176(20)30159-2 [pii]\\nAID - 10.1016/j.ymben.2020.10.004 [doi]\\nPST - ppublish\\nSO  - Metab Eng. 2021 Jan;63:148-159. doi: 10.1016/j.ymben.2020.10.004. Epub 2020 Nov 2.\\n\\nPMID- 30688657\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200224\\nLR  - 20200309\\nIS  - 1558-8238 (Electronic)\\nIS  - 0021-9738 (Print)\\nIS  - 0021-9738 (Linking)\\nVI  - 129\\nIP  - 3\\nDP  - 2019 Mar 1\\nTI  - Cullin5 deficiency promotes small-cell lung cancer metastasis by stabilizing  integrin Î²1.\\nPG  - 972-987\\nLID - 122779 [pii]\\nLID - 10.1172/JCI122779 [doi]\\nAB  - Metastasis is the dominant cause of patient death in small-cell lung cancer (SCLC),  and a better understanding of the molecular mechanisms underlying SCLC metastasis  may potentially improve clinical treatment. Through genome-scale screening for key  regulators of mouse Rb1-/- Trp53-/- SCLC metastasis using the pooled CRISPR/Cas9  library, we identified Cullin5 (CUL5) and suppressor of cytokine signaling 3  (SOCS3), two components of the Cullin-RING E3 ubiquitin ligase complex, as top  candidates. Mechanistically, the deficiency of CUL5 or SOCS3 disrupted the  functional formation of the E3 ligase complex and prevented the degradation of  integrin Î²1, which stabilized integrin Î²1 and activated downstream focal adhesion  kinase/SRC (FAK/SRC) signaling and eventually drove SCLC metastasis. Low expression  levels of CUL5 and SOCS3 were significantly associated with high integrin Î²1 levels  and poor prognosis in a large cohort of 128 clinical patients with SCLC. Moreover,  the CUL5-deficient SCLCs were vulnerable to the treatment of the FDA-approved SRC  inhibitor dasatinib. Collectively, this work identifies the essential role of CUL5-  and SOCS3-mediated integrin Î²1 turnover in controlling SCLC metastasis, which might  have therapeutic implications.\\nFAU - Zhao, Gaoxiang\\nAU  - Zhao G\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Gong, Liyan\\nAU  - Gong L\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Su, Dan\\nAU  - Su D\\nAD  - Department of Pathology, Zhejiang Cancer Hospital, Hangzhou, Zhejiang, China.\\nFAU - Jin, Yujuan\\nAU  - Jin Y\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Guo, Chenchen\\nAU  - Guo C\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Yue, Meiting\\nAU  - Yue M\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Yao, Shun\\nAU  - Yao S\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Qin, Zhen\\nAU  - Qin Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Ye, Yi\\nAU  - Ye Y\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nFAU - Tang, Ying\\nAU  - Tang Y\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Wu, Qibiao\\nAU  - Wu Q\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Zhang, Jian\\nAU  - Zhang J\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Cui, Binghai\\nAU  - Cui B\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Ding, Qiurong\\nAU  - Ding Q\\nAD  - CAS Key Laboratory of Nutrition, Metabolism and Food Safety, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, Chinese Academy  of Sciences, Shanghai, China.\\nFAU - Huang, Hsinyi\\nAU  - Huang H\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Hu, Liang\\nAU  - Hu L\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Chen, Yuting\\nAU  - Chen Y\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nFAU - Zhang, Peiyuan\\nAU  - Zhang P\\nAD  - The Key Laboratory of Stem Cell Biology, Institute of Health Sciences, Shanghai  Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Hu, Guohong\\nAU  - Hu G\\nAD  - The Key Laboratory of Stem Cell Biology, Institute of Health Sciences, Shanghai  Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Chen, Luonan\\nAU  - Chen L\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nFAU - Wong, Kwok-Kin\\nAU  - Wong KK\\nAD  - Laura and Isaac Perlmutter Cancer Center, NYU Langone Medical Center, New York, New  York, USA.\\nFAU - Gao, Daming\\nAU  - Gao D\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Ji, Hongbin\\nAU  - Ji H\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences; University of Chinese  Academy of Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190128\\nTA  - J Clin Invest\\nJT  - The Journal of clinical investigation\\nJID - 7802877\\nRN  - 0 (CUL5 protein, human)\\nRN  - 0 (Cullin Proteins)\\nRN  - 0 (Integrin beta1)\\nRN  - 0 (Neoplasm Proteins)\\nRN  - 0 (cullin5 protein, mouse)\\nSB  - IM\\nMH  - Animals\\nMH  - Cullin Proteins/*genetics/metabolism\\nMH  - HEK293 Cells\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - *Integrin beta1/genetics/metabolism\\nMH  - *Lung Neoplasms/genetics/metabolism/pathology\\nMH  - Male\\nMH  - Mice\\nMH  - Mice, Inbred BALB C\\nMH  - Mice, Knockout\\nMH  - Mice, Nude\\nMH  - Neoplasm Metastasis\\nMH  - *Neoplasm Proteins/genetics/metabolism\\nMH  - *Neoplasms, Experimental/genetics/metabolism/pathology\\nMH  - Protein Stability\\nMH  - Signal Transduction/genetics\\nMH  - *Small Cell Lung Carcinoma/genetics/metabolism/pathology\\nPMC - PMC6391098\\nOTO - NOTNLM\\nOT  - *Cancer\\nOT  - *Lung cancer\\nOT  - *Oncology\\nOT  - *Ubiquitin-proteosome system\\nCOIS- Conflict of interest: The authors have declared that no conflict of interest exists.\\nEDAT- 2019/01/29 06:00\\nMHDA- 2020/02/25 06:00\\nCRDT- 2019/01/29 06:00\\nPHST- 2018/06/11 00:00 [received]\\nPHST- 2018/11/30 00:00 [accepted]\\nPHST- 2019/01/29 06:00 [pubmed]\\nPHST- 2020/02/25 06:00 [medline]\\nPHST- 2019/01/29 06:00 [entrez]\\nAID - 122779 [pii]\\nAID - 10.1172/JCI122779 [doi]\\nPST - ppublish\\nSO  - J Clin Invest. 2019 Mar 1;129(3):972-987. doi: 10.1172/JCI122779. Epub 2019 Jan 28.\\n\\nPMID- 29950558\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190111\\nLR  - 20190627\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 9\\nIP  - 1\\nDP  - 2018 Jun 27\\nTI  - Synthetic CRISPR-Cas gene activators for transcriptional reprogramming in bacteria.\\nPG  - 2489\\nLID - 10.1038/s41467-018-04901-6 [doi]\\nLID - 2489\\nAB  - Methods to regulate gene expression programs in bacterial cells are limited by the  absence of effective gene activators. To address this challenge, we have developed  synthetic bacterial transcriptional activators in E. coli by linking activation  domains to programmable CRISPR-Cas DNA binding domains. Effective gene activation  requires target sites situated in a narrow region just upstream of the transcription  start site, in sharp contrast to the relatively flexible target site requirements  for gene activation in eukaryotic cells. Together with existing tools for CRISPRi  gene repression, these bacterial activators enable programmable control over  multiple genes with simultaneous activation and repression. Further, the entire gene  expression program can be switched on by inducing expression of the CRISPR-Cas  system. This work will provide a foundation for engineering synthetic bacterial  cellular devices with applications including diagnostics, therapeutics, and  industrial biosynthesis.\\nFAU - Dong, Chen\\nAU  - Dong C\\nAD  - Department of Chemistry, University of Washington, Seattle, WA, 98195, USA.\\nFAU - Fontana, Jason\\nAU  - Fontana J\\nAD  - Molecular Engineering & Sciences Institute, University of Washington, Seattle, WA,  98195, USA.\\nFAU - Patel, Anika\\nAU  - Patel A\\nAD  - Department of Chemistry, University of Washington, Seattle, WA, 98195, USA.\\nFAU - Carothers, James M\\nAU  - Carothers JM\\nAUID- ORCID: 0000-0001-6728-7833\\nAD  - Molecular Engineering & Sciences Institute, University of Washington, Seattle, WA,  98195, USA. jcaroth@uw.edu.\\nAD  - Department of Chemical Engineering, University of Washington, Seattle, WA, 98195,  USA. jcaroth@uw.edu.\\nAD  - Center for Synthetic Biology, University of Washington, Seattle, WA, 98195, USA.  jcaroth@uw.edu.\\nFAU - Zalatan, Jesse G\\nAU  - Zalatan JG\\nAUID- ORCID: 0000-0002-1458-0654\\nAD  - Department of Chemistry, University of Washington, Seattle, WA, 98195, USA.  zalatan@uw.edu.\\nAD  - Molecular Engineering & Sciences Institute, University of Washington, Seattle, WA,  98195, USA. zalatan@uw.edu.\\nAD  - Center for Synthetic Biology, University of Washington, Seattle, WA, 98195, USA.  zalatan@uw.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180627\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nSB  - IM\\nEIN - Nat Commun. 2018 Oct 15;9(1):4318. PMID: 30323295\\nMH  - *CRISPR-Cas Systems\\nMH  - Escherichia coli/*genetics/metabolism\\nMH  - Gene Editing\\nMH  - Gene Knockout Techniques\\nMH  - Genes, Synthetic/*genetics\\nMH  - Genome, Bacterial/*genetics\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Transcription Initiation Site\\nMH  - Transcriptional Activation/genetics\\nPMC - PMC6021436\\nCOIS- The authors declare no competing interests.\\nEDAT- 2018/06/29 06:00\\nMHDA- 2019/01/12 06:00\\nCRDT- 2018/06/29 06:00\\nPHST- 2018/02/05 00:00 [received]\\nPHST- 2018/06/01 00:00 [accepted]\\nPHST- 2018/06/29 06:00 [entrez]\\nPHST- 2018/06/29 06:00 [pubmed]\\nPHST- 2019/01/12 06:00 [medline]\\nAID - 10.1038/s41467-018-04901-6 [pii]\\nAID - 4901 [pii]\\nAID - 10.1038/s41467-018-04901-6 [doi]\\nPST - epublish\\nSO  - Nat Commun. 2018 Jun 27;9(1):2489. doi: 10.1038/s41467-018-04901-6.\\n\\nPMID- 33307189\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211124\\nLR  - 20211124\\nIS  - 1096-7184 (Electronic)\\nIS  - 1096-7176 (Linking)\\nVI  - 63\\nDP  - 2021 Jan\\nTI  - CRISPR-derived genome editing technologies for metabolic engineering.\\nPG  - 141-147\\nLID - S1096-7176(20)30186-5 [pii]\\nLID - 10.1016/j.ymben.2020.12.002 [doi]\\nAB  - In metabolic engineering, genome editing tools make it much easier to discover and  evaluate relevant genes and pathways and construct strains. Clustered regularly  interspaced palindromic repeats (CRISPR)-associated (Cas) systems now have become  the first choice for genome engineering in many organisms includingindustrially  relevant ones. Targeted DNA cleavage by CRISPR-Cas provides variousgenome  engineering modes such as indels, replacements, large deletions, knock-in and  chromosomal rearrangements, while host-dependent differences in repair pathways need  to be considered. The versatility of the CRISPR system has given rise to derivative  technologies that complement nuclease-based editing, which causes cytotoxicity  especially in microorganisms. Deaminase-mediated base editing installs targeted  point mutations with much less toxicity. CRISPRi and CRISPRa can temporarily control  gene expression without changing the genomic sequence. Multiplex, combinatorial and  large scale editing are made possible by streamlined design and construction of gRNA  libraries to further accelerates comprehensive discovery, evaluation and building of  metabolic pathways. This review summarizes the technical basis and recent advances  in CRISPR-related genome editing tools applied for metabolic engineering purposes,  with representative examples of industrially relevant eukaryotic and prokaryotic  organisms.\\nCI  - Copyright Â© 2020. Published by Elsevier Inc.\\nFAU - Nishida, Keiji\\nAU  - Nishida K\\nAD  - Engineering Biology Research Center, Kobe University, Japan; Graduate School of  Science, Technology and Innovation, Kobe University, Japan.\\nFAU - Kondo, Akihiko\\nAU  - Kondo A\\nAD  - Engineering Biology Research Center, Kobe University, Japan; Graduate School of  Science, Technology and Innovation, Kobe University, Japan. Electronic address:  akondo@kobe-u.ac.jp.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20201208\\nPL  - Belgium\\nTA  - Metab Eng\\nJT  - Metabolic engineering\\nJID - 9815657\\nSB  - IM\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - *Gene Editing\\nMH  - Genome\\nMH  - *Metabolic Engineering\\nMH  - Metabolic Networks and Pathways\\nOTO - NOTNLM\\nOT  - *Base editing\\nOT  - *CRISPR\\nOT  - *Genome editing\\nOT  - *Metabolic engineering\\nOT  - *Recombineering\\nEDAT- 2020/12/12 06:00\\nMHDA- 2021/11/25 06:00\\nCRDT- 2020/12/11 20:09\\nPHST- 2020/10/06 00:00 [received]\\nPHST- 2020/12/02 00:00 [revised]\\nPHST- 2020/12/03 00:00 [accepted]\\nPHST- 2020/12/12 06:00 [pubmed]\\nPHST- 2021/11/25 06:00 [medline]\\nPHST- 2020/12/11 20:09 [entrez]\\nAID - S1096-7176(20)30186-5 [pii]\\nAID - 10.1016/j.ymben.2020.12.002 [doi]\\nPST - ppublish\\nSO  - Metab Eng. 2021 Jan;63:141-147. doi: 10.1016/j.ymben.2020.12.002. Epub 2020 Dec 8.\\n\\nPMID- 27148329\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20160505\\nLR  - 20201001\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Electronic)\\nIS  - 1664-462X (Linking)\\nVI  - 7\\nDP  - 2016\\nTI  - The CRISPR/Cas Genome-Editing Tool: Application in Improvement of Crops.\\nPG  - 506\\nLID - 10.3389/fpls.2016.00506 [doi]\\nLID - 506\\nAB  - The Clustered Regularly Interspaced Short Palindromic Repeats associated Cas9/sgRNA  system is a novel targeted genome-editing technique derived from bacterial immune  system. It is an inexpensive, easy, most user friendly and rapidly adopted genome  editing tool transforming to revolutionary paradigm. This technique enables precise  genomic modifications in many different organisms and tissues. Cas9 protein is an  RNA guided endonuclease utilized for creating targeted double-stranded breaks with  only a short RNA sequence to confer recognition of the target in animals and plants.  Development of genetically edited (GE) crops similar to those developed by  conventional or mutation breeding using this potential technique makes it a  promising and extremely versatile tool for providing sustainable productive  agriculture for better feeding of rapidly growing population in a changing climate.  The emerging areas of research for the genome editing in plants include  interrogating gene function, rewiring the regulatory signaling networks and sgRNA  library for high-throughput loss-of-function screening. In this review, we have  described the broad applicability of the Cas9 nuclease mediated targeted plant  genome editing for development of designer crops. The regulatory uncertainty and  social acceptance of plant breeding by Cas9 genome editing have also been described.  With this powerful and innovative technique the designer GE non-GM plants could  further advance climate resilient and sustainable agriculture in the future and  maximizing yield by combating abiotic and biotic stresses.\\nFAU - Khatodia, Surender\\nAU  - Khatodia S\\nAD  - Amity Institute of Biotechnology, Amity University Haryana Gurgaon, India.\\nFAU - Bhatotia, Kirti\\nAU  - Bhatotia K\\nAD  - Amity Institute of Biotechnology, Amity University Haryana Gurgaon, India.\\nFAU - Passricha, Nishat\\nAU  - Passricha N\\nAD  - Plant Molecular Biology Group, International Centre for Genetic Engineering and  Biotechnology New Delhi, India.\\nFAU - Khurana, S M P\\nAU  - Khurana SM\\nAD  - Amity Institute of Biotechnology, Amity University Haryana Gurgaon, India.\\nFAU - Tuteja, Narendra\\nAU  - Tuteja N\\nAD  - Plant Molecular Biology Group, International Centre for Genetic Engineering and  Biotechnology New Delhi, India; Amity Institute of Microbial Technology, Amity  UniversityNoida, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20160419\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC4835450\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Cas9/sgRNA\\nOT  - GE crops\\nOT  - NPBTs\\nOT  - RGENs\\nOT  - dCas9\\nOT  - plant genome editing\\nEDAT- 2016/05/06 06:00\\nMHDA- 2016/05/06 06:01\\nCRDT- 2016/05/06 06:00\\nPHST- 2015/01/21 00:00 [received]\\nPHST- 2016/03/30 00:00 [accepted]\\nPHST- 2016/05/06 06:00 [entrez]\\nPHST- 2016/05/06 06:00 [pubmed]\\nPHST- 2016/05/06 06:01 [medline]\\nAID - 10.3389/fpls.2016.00506 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2016 Apr 19;7:506. doi: 10.3389/fpls.2016.00506. eCollection 2016.\\n\\nPMID- 31147819\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200723\\nLR  - 20200928\\nIS  - 2629-3277 (Electronic)\\nIS  - 2629-3277 (Linking)\\nVI  - 15\\nIP  - 4\\nDP  - 2019 Aug\\nTI  - Enhancing the Therapeutic Potential of Mesenchymal Stem Cells with the CRISPR-Cas  System.\\nPG  - 463-473\\nLID - 10.1007/s12015-019-09897-0 [doi]\\nAB  - Mesenchymal stem cells (MSCs), also known as multipotent mesenchymal stromal stem  cells, are found in the perivascular space of several tissues. These cells have been  subject of intense research in the last decade due to their low teratogenicity, as  well as their ability to differentiate into mature cells and to secrete  immunomodulatory and trophic factors. However, they usually promote only a modest  benefit when transplanted in experimental disease models, one of the limitations for  their clinical application. The CRISPR-Cas system, in turn, is highlighted as a  simple and effective tool for genetic engineering. This system was tested in  clinical trials over a relatively short period of time after establishing its  applicability to the edition of the mammalian cell genome. Similar to the research  evolution in MSCs, the CRISPR-Cas system demonstrated inconsistencies that limited  its clinical application. In this review, we outline the evolution of MSC research  and its applicability, and the progress of the CRISPR-Cas system from its discovery  to the most recent clinical trials. We also propose perspectives on how the  CRISPR-Cas system may improve the therapeutic potential of MSCs, making it more  beneficial and long lasting.\\nFAU - Filho, Daniel Mendes\\nAU  - Filho DM\\nAD  - Department of Physiology, Ribeirao Preto Medical School, University of Sao Paulo,  Ribeirao Preto, Sao Paulo, Brazil.\\nFAU - de Carvalho Ribeiro, PatrÃ\\xadcia\\nAU  - de Carvalho Ribeiro P\\nAD  - Laboratory of Immunology and Experimental Transplantation, SÃ£o JosÃ© do Rio Preto  Medical School, SÃ£o JosÃ© do Rio Preto, SÃ£o Paulo, Brazil.\\nAD  - Division of Thoracic Surgery, Department of Surgery, Massachusetts General Hospital,  Boston, MA, USA.\\nFAU - Oliveira, Lucas Felipe\\nAU  - Oliveira LF\\nAD  - Department of Physiology, Biological and Natural Sciences Institute, Triangulo  Mineiro Federal University, Uberaba, Minas Gerais, Brazil.\\nAD  - National Institute of Science and Technology for Regenerative Medicine  (INCT-REGENERA-CNPq), Rio de Janeiro, RJ, Brazil.\\nAD  - Minas Gerais Network for Tissue Engineering and Cell Therapy (REMETTECFAPEMIG), Belo  Horizonte, MG, Brazil.\\nFAU - Dos Santos, Ana Luiza Romero Terra\\nAU  - Dos Santos ALRT\\nAD  - Department of Physiology, Ribeirao Preto Medical School, University of Sao Paulo,  Ribeirao Preto, Sao Paulo, Brazil.\\nFAU - Parreira, Ricardo Cambraia\\nAU  - Parreira RC\\nAUID- ORCID: 0000-0002-2769-7958\\nAD  - Department of Pharmacology, Biological Sciences Institute, Goias Federal University,  Goiania, Goias, Brazil. rcparreira@ufg.br.\\nFAU - Pinto, Mauro Cunha Xavier\\nAU  - Pinto MCX\\nAD  - Department of Pharmacology, Biological Sciences Institute, Goias Federal University,  Goiania, Goias, Brazil.\\nFAU - Resende, Rodrigo Ribeiro\\nAU  - Resende RR\\nAD  - Department of Biochemistry and Immunology, Federal University of Minas Gerais, Belo  Horizonte, MG, Brazil.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nPL  - United States\\nTA  - Stem Cell Rev Rep\\nJT  - Stem cell reviews and reports\\nJID - 101752767\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems\\nMH  - Disease Models, Animal\\nMH  - *Gene Editing\\nMH  - Humans\\nMH  - *Immunomodulation\\nMH  - Mesenchymal Stem Cells/*immunology\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *Cell therapy\\nOT  - *Genetic engineering\\nOT  - *Mesenchymal stem cells\\nOT  - *Mesenchymal stromal cells\\nEDAT- 2019/05/31 06:00\\nMHDA- 2020/07/24 06:00\\nCRDT- 2019/06/01 06:00\\nPHST- 2019/05/31 06:00 [pubmed]\\nPHST- 2020/07/24 06:00 [medline]\\nPHST- 2019/06/01 06:00 [entrez]\\nAID - 10.1007/s12015-019-09897-0 [pii]\\nAID - 10.1007/s12015-019-09897-0 [doi]\\nPST - ppublish\\nSO  - Stem Cell Rev Rep. 2019 Aug;15(4):463-473. doi: 10.1007/s12015-019-09897-0.\\n\\nPMID- 23628424\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150407\\nLR  - 20211021\\nIS  - 1474-760X (Electronic)\\nIS  - 1465-6906 (Print)\\nIS  - 1474-7596 (Linking)\\nVI  - 14\\nIP  - 4\\nDP  - 2013 Apr 29\\nTI  - CRISPR-Cas systems target a diverse collection of invasive mobile genetic elements  in human microbiomes.\\nPG  - R40\\nLID - 10.1186/gb-2013-14-4-r40 [doi]\\nAB  - BACKGROUND: Bacteria and archaea develop immunity against invading genomes by  incorporating pieces of the invaders\\' sequences, called spacers, into a clustered  regularly interspaced short palindromic repeats (CRISPR) locus between repeats,  forming arrays of repeat-spacer units. When spacers are expressed, they direct  CRISPR-associated (Cas) proteins to silence complementary invading DNA. In order to  characterize the invaders of human microbiomes, we use spacers from CRISPR arrays  that we had previously assembled from shotgun metagenomic datasets, and identify  contigs that contain these spacers\\' targets. RESULTS: We discover 95,000 contigs  that are putative invasive mobile genetic elements, some targeted by hundreds of  CRISPR spacers. We find that oral sites in healthy human populations have a much  greater variety of mobile genetic elements than stool samples. Mobile genetic  elements carry genes encoding diverse functions: only 7% of the mobile genetic  elements are similar to known phages or plasmids, although a much greater proportion  contain phage- or plasmid-related genes. A small number of contigs share similarity  with known integrative and conjugative elements, providing the first examples of  CRISPR defenses against this class of element. We provide detailed analyses of a few  large mobile genetic elements of various types, and a relative abundance analysis of  mobile genetic elements and putative hosts, exploring the dynamic activities of  mobile genetic elements in human microbiomes. A joint analysis of mobile genetic  elements and CRISPRs shows that protospacer-adjacent motifs drive their interaction  network; however, some CRISPR-Cas systems target mobile genetic elements lacking  motifs. CONCLUSIONS: We identify a large collection of invasive mobile genetic  elements in human microbiomes, an important resource for further study of the  interaction between the CRISPR-Cas immune system and invaders.\\nFAU - Zhang, Quan\\nAU  - Zhang Q\\nFAU - Rho, Mina\\nAU  - Rho M\\nFAU - Tang, Haixu\\nAU  - Tang H\\nFAU - Doak, Thomas G\\nAU  - Doak TG\\nFAU - Ye, Yuzhen\\nAU  - Ye Y\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20130429\\nTA  - Genome Biol\\nJT  - Genome biology\\nJID - 100960660\\nSB  - IM\\nMH  - Bacteria/genetics\\nMH  - *CRISPR-Cas Systems\\nMH  - Genes, Microbial\\nMH  - Humans\\nMH  - *Interspersed Repetitive Sequences\\nMH  - Microbiota/*genetics\\nMH  - Prophages/genetics\\nPMC - PMC4053933\\nEDAT- 2013/05/01 06:00\\nMHDA- 2015/04/08 06:00\\nCRDT- 2013/05/01 06:00\\nPHST- 2013/02/07 00:00 [received]\\nPHST- 2013/04/29 00:00 [accepted]\\nPHST- 2013/05/01 06:00 [entrez]\\nPHST- 2013/05/01 06:00 [pubmed]\\nPHST- 2015/04/08 06:00 [medline]\\nAID - gb-2013-14-4-r40 [pii]\\nAID - 10.1186/gb-2013-14-4-r40 [doi]\\nPST - epublish\\nSO  - Genome Biol. 2013 Apr 29;14(4):R40. doi: 10.1186/gb-2013-14-4-r40.\\n\\nPMID- 33514753\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210219\\nLR  - 20210219\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 12\\nIP  - 1\\nDP  - 2021 Jan 29\\nTI  - In-situ generation of large numbers of genetic combinations for metabolic  reprogramming via CRISPR-guided base editing.\\nPG  - 678\\nLID - 10.1038/s41467-021-21003-y [doi]\\nLID - 678\\nAB  - Reprogramming complex cellular metabolism requires simultaneous regulation of  multigene expression. Ex-situ cloning-based methods are commonly used, but the  target gene number and combinatorial library size are severely limited by cloning  and transformation efficiencies. In-situ methods such as multiplex automated genome  engineering (MAGE) depends on high-efficiency transformation and incorporation of  heterologous DNA donors, which are limited to few microorganisms. Here, we describe  a Base Editor-Targeted and Template-free Expression Regulation (BETTER) method for  simultaneously diversifying multigene expression. BETTER repurposes CRISPR-guided  base editors and in-situ generates large numbers of genetic combinations of diverse  ribosome binding sites, 5\\' untranslated regions, or promoters, without library  construction, transformation, and incorporation of DNA donors. We apply BETTER to  simultaneously regulate expression of up to ten genes in industrial and model  microorganisms Corynebacterium glutamicum and Bacillus subtilis. Variants with  improved xylose catabolism, glycerol catabolism, or lycopene biosynthesis are  respectively obtained. This technology will be useful for large-scale fine-tuning of  multigene expression in both genetically tractable and intractable microorganisms.\\nFAU - Wang, Yu\\nAU  - Wang Y\\nAUID- ORCID: 0000-0002-0957-9245\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China. wang_y@tib.cas.cn.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.  wang_y@tib.cas.cn.\\nAD  - University of Chinese Academy of Sciences, Beijing, China. wang_y@tib.cas.cn.\\nFAU - Cheng, Haijiao\\nAU  - Cheng H\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Liu, Yang\\nAU  - Liu Y\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Liu, Ye\\nAU  - Liu Y\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Wen, Xiao\\nAU  - Wen X\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei,  China.\\nFAU - Zhang, Kun\\nAU  - Zhang K\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Ni, Xiaomeng\\nAU  - Ni X\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Gao, Ning\\nAU  - Gao N\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Fan, Liwen\\nAU  - Fan L\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei,  China.\\nFAU - Zhang, Zhihui\\nAU  - Zhang Z\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Liu, Jiao\\nAU  - Liu J\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Chen, Jiuzhou\\nAU  - Chen J\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Wang, Lixian\\nAU  - Wang L\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Guo, Yanmei\\nAU  - Guo Y\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nFAU - Zheng, Ping\\nAU  - Zheng P\\nAUID- ORCID: 0000-0001-9434-9892\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China. zheng_p@tib.cas.cn.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.  zheng_p@tib.cas.cn.\\nAD  - University of Chinese Academy of Sciences, Beijing, China. zheng_p@tib.cas.cn.\\nFAU - Wang, Meng\\nAU  - Wang M\\nAUID- ORCID: 0000-0001-7648-9955\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China. wangmeng@tib.cas.cn.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.  wangmeng@tib.cas.cn.\\nAD  - University of Chinese Academy of Sciences, Beijing, China. wangmeng@tib.cas.cn.\\nFAU - Sun, Jibin\\nAU  - Sun J\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, China.\\nFAU - Ma, Yanhe\\nAU  - Ma Y\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Tianjin Institute of Industrial  Biotechnology, Chinese Academy of Sciences, Tianjin, China.\\nAD  - National Technology Innovation Center of Synthetic Biology, Tianjin, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210129\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (DNA, Bacterial)\\nRN  - A1TA934AKO (Xylose)\\nRN  - PDC6A3C0OX (Glycerol)\\nRN  - SB0N2N0WV6 (Lycopene)\\nSB  - IM\\nMH  - Bacillus subtilis/genetics/metabolism\\nMH  - Bacterial Proteins/genetics/metabolism\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Corynebacterium glutamicum/genetics/metabolism\\nMH  - DNA, Bacterial/genetics\\nMH  - Escherichia coli/genetics\\nMH  - Gene Editing/*methods\\nMH  - Genes, Bacterial/genetics\\nMH  - Glycerol/metabolism\\nMH  - Industrial Microbiology/*methods\\nMH  - Lycopene/metabolism\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Multigene Family/*genetics\\nMH  - Transformation, Bacterial\\nMH  - Xylose/metabolism\\nPMC - PMC7846839\\nCOIS- The authors declare no competing interests.\\nEDAT- 2021/01/31 06:00\\nMHDA- 2021/02/20 06:00\\nCRDT- 2021/01/30 05:28\\nPHST- 2020/04/30 00:00 [received]\\nPHST- 2021/01/07 00:00 [accepted]\\nPHST- 2021/01/30 05:28 [entrez]\\nPHST- 2021/01/31 06:00 [pubmed]\\nPHST- 2021/02/20 06:00 [medline]\\nAID - 10.1038/s41467-021-21003-y [pii]\\nAID - 21003 [pii]\\nAID - 10.1038/s41467-021-21003-y [doi]\\nPST - epublish\\nSO  - Nat Commun. 2021 Jan 29;12(1):678. doi: 10.1038/s41467-021-21003-y.\\n\\nPMID- 32249828\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200812\\nLR  - 20211204\\nIS  - 1546-1696 (Electronic)\\nIS  - 1087-0156 (Linking)\\nVI  - 38\\nIP  - 5\\nDP  - 2020 May\\nTI  - Genetic interaction mapping and exon-resolution functional genomics with a hybrid  Cas9-Cas12a platform.\\nPG  - 638-648\\nLID - 10.1038/s41587-020-0437-z [doi]\\nAB  - Systematic mapping of genetic interactions (GIs) and interrogation of the functions  of sizable genomic segments in mammalian cells represent important goals of  biomedical research. To advance these goals, we present a CRISPR (clustered  regularly interspaced short palindromic repeats)-based screening system for  combinatorial genetic manipulation that employs coexpression of CRISPR-associated  nucleases 9 and 12a (Cas9 and Cas12a) and machine-learning-optimized libraries of  hybrid Cas9-Cas12a guide RNAs. This system, named Cas Hybrid for Multiplexed Editing  and screening Applications (CHyMErA), outperforms genetic screens using Cas9 or  Cas12a editing alone. Application of CHyMErA to the ablation of mammalian paralog  gene pairs reveals extensive GIs and uncovers phenotypes normally masked by  functional redundancy. Application of CHyMErA in a chemogenetic interaction screen  identifies genes that impact cell growth in response to mTOR pathway inhibition.  Moreover, by systematically targeting thousands of alternative splicing events,  CHyMErA identifies exons underlying human cell line fitness. CHyMErA thus represents  an effective screening approach for GI mapping and the functional analysis of  sizable genomic regions, such as alternative exons.\\nFAU - Gonatopoulos-Pournatzis, Thomas\\nAU  - Gonatopoulos-Pournatzis T\\nAUID- ORCID: 0000-0003-2669-2570\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.  t.gonatopoulos@gmail.com.\\nFAU - Aregger, Michael\\nAU  - Aregger M\\nAUID- ORCID: 0000-0001-6443-9870\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Brown, Kevin R\\nAU  - Brown KR\\nAUID- ORCID: 0000-0002-5514-2538\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Farhangmehr, Shaghayegh\\nAU  - Farhangmehr S\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Braunschweig, Ulrich\\nAU  - Braunschweig U\\nAUID- ORCID: 0000-0001-6653-0926\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Ward, Henry N\\nAU  - Ward HN\\nAUID- ORCID: 0000-0002-9869-9790\\nAD  - Bioinformatics and Computational Biology Graduate Program, University of Minnesota,  Minneapolis, MN, USA.\\nFAU - Ha, Kevin C H\\nAU  - Ha KCH\\nAUID- ORCID: 0000-0001-5746-4365\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Weiss, Alexander\\nAU  - Weiss A\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Billmann, Maximilian\\nAU  - Billmann M\\nAUID- ORCID: 0000-0002-6556-9594\\nAD  - Department of Computer Science and Engineering, University of Minnesota,  Minneapolis, MN, USA.\\nFAU - Durbic, Tanja\\nAU  - Durbic T\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.\\nFAU - Myers, Chad L\\nAU  - Myers CL\\nAUID- ORCID: 0000-0002-1026-5972\\nAD  - Bioinformatics and Computational Biology Graduate Program, University of Minnesota,  Minneapolis, MN, USA.\\nAD  - Department of Computer Science and Engineering, University of Minnesota,  Minneapolis, MN, USA.\\nFAU - Blencowe, Benjamin J\\nAU  - Blencowe BJ\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.  b.blencowe@utoronto.ca.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada.  b.blencowe@utoronto.ca.\\nFAU - Moffat, Jason\\nAU  - Moffat J\\nAUID- ORCID: 0000-0002-5663-8586\\nAD  - Donnelly Centre, University of Toronto, Toronto, Ontario, Canada.  j.moffat@utoronto.ca.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada.  j.moffat@utoronto.ca.\\nAD  - Institute for Biomedical Engineering, University of Toronto, Toronto, Ontario,  Canada. j.moffat@utoronto.ca.\\nLA  - eng\\nGR  - MOP-142375/CIHR/Canada\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20200316\\nPL  - United States\\nTA  - Nat Biotechnol\\nJT  - Nature biotechnology\\nJID - 9604648\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - EC 2.7.11.1 (TOR Serine-Threonine Kinases)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nRN  - EC 3.1.- (Cas12a protein)\\nRN  - EC 3.1.- (Endodeoxyribonucleases)\\nSB  - IM\\nCIN - Nat Rev Genet. 2020 May;21(5):273. PMID: 32235908\\nMH  - Alternative Splicing\\nMH  - Animals\\nMH  - Bacterial Proteins/*metabolism\\nMH  - CRISPR-Associated Protein 9/*metabolism\\nMH  - CRISPR-Associated Proteins/*metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Cell Line\\nMH  - Endodeoxyribonucleases/*metabolism\\nMH  - Gene Editing/*methods\\nMH  - *Gene Regulatory Networks\\nMH  - Genetic Fitness\\nMH  - Humans\\nMH  - Machine Learning\\nMH  - Male\\nMH  - Mice\\nMH  - Signal Transduction\\nMH  - TOR Serine-Threonine Kinases/metabolism\\nEDAT- 2020/04/07 06:00\\nMHDA- 2020/08/13 06:00\\nCRDT- 2020/04/07 06:00\\nPHST- 2019/05/26 00:00 [received]\\nPHST- 2020/01/27 00:00 [accepted]\\nPHST- 2020/04/07 06:00 [pubmed]\\nPHST- 2020/08/13 06:00 [medline]\\nPHST- 2020/04/07 06:00 [entrez]\\nAID - 10.1038/s41587-020-0437-z [pii]\\nAID - 10.1038/s41587-020-0437-z [doi]\\nPST - ppublish\\nSO  - Nat Biotechnol. 2020 May;38(5):638-648. doi: 10.1038/s41587-020-0437-z. Epub 2020  Mar 16.\\n\\nPMID- 32646968\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210111\\nLR  - 20211204\\nIS  - 1538-7445 (Electronic)\\nIS  - 0008-5472 (Print)\\nIS  - 0008-5472 (Linking)\\nVI  - 80\\nIP  - 17\\nDP  - 2020 Sep 1\\nTI  - Epigenetic CRISPR Screens Identify Npm1 as a Therapeutic Vulnerability in Non-Small  Cell Lung Cancer.\\nPG  - 3556-3567\\nLID - 10.1158/0008-5472.CAN-19-3782 [doi]\\nAB  - Despite advancements in treatment options, the overall cure and survival rates for  non-small cell lung cancers (NSCLC) remain low. While small-molecule inhibitors of  epigenetic regulators have recently emerged as promising cancer therapeutics, their  application in patients with NSCLC is limited. To exploit epigenetic regulators as  novel therapeutic targets in NSCLC, we performed pooled epigenome-wide CRISPR  knockout screens in vitro and in vivo and identified the histone chaperone  nucleophosmin 1 (Npm1) as a potential therapeutic target. Genetic ablation of Npm1  significantly attenuated tumor progression in vitro and in vivo. Furthermore,  KRAS-mutant cancer cells were more addicted to NPM1 expression. Genetic ablation of  Npm1 rewired the balance of metabolism in cancer cells from predominant aerobic  glycolysis to oxidative phosphorylation and reduced the population of  tumor-propagating cells. Overall, our results support NPM1 as a therapeutic  vulnerability in NSCLC. SIGNIFICANCE: Epigenome-wide CRISPR knockout screens  identify NPM1 as a novel metabolic vulnerability and demonstrate that targeting NPM1  is a new therapeutic opportunity for patients with NSCLC.\\nCI  - Â©2020 American Association for Cancer Research.\\nFAU - Li, Fei\\nAU  - Li F\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Ng, Wai-Lung\\nAU  - Ng WL\\nAUID- ORCID: 0000-0003-2892-6318\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nAD  - Department of Medicine, Harvard Medical School, Boston, Massachusetts.\\nFAU - Luster, Troy A\\nAU  - Luster TA\\nAUID- ORCID: 0000-0002-7396-0171\\nAD  - Belfer Center for Applied Cancer Science, Dana-Farber Cancer Institute, Boston,  Massachusetts.\\nFAU - Hu, Hai\\nAU  - Hu H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Sviderskiy, Vladislav O\\nAU  - Sviderskiy VO\\nAD  - Department of Pathology, New York University School of Medicine, New York, New York.\\nFAU - Dowling, CatrÃ\\xadona M\\nAU  - Dowling CM\\nAUID- ORCID: 0000-0001-7887-9332\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Hollinshead, Kate E R\\nAU  - Hollinshead KER\\nAD  - Department of Radiation Oncology, Perlmutter Cancer Center, New York University  School of Medicine, New York, New York.\\nFAU - Zouitine, Paula\\nAU  - Zouitine P\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nFAU - Zhang, Hua\\nAU  - Zhang H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Huang, Qingyuan\\nAU  - Huang Q\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Ranieri, Michela\\nAU  - Ranieri M\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Wang, Wei\\nAU  - Wang W\\nAD  - S. Arthur Localio Laboratory, Department of Surgery, New York University School of  Medicine, New York, New York.\\nFAU - Fang, Zhaoyuan\\nAU  - Fang Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Chen, Ting\\nAU  - Chen T\\nAUID- ORCID: 0000-0003-1725-7956\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Deng, Jiehui\\nAU  - Deng J\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Zhao, Kai\\nAU  - Zhao K\\nAD  - School of Biomedical Sciences, Faculty of Medicine, The Chinese University of Hong  Kong, Sha Tin, Hong Kong SAR, China.\\nFAU - So, Hon-Cheong\\nAU  - So HC\\nAD  - School of Biomedical Sciences, Faculty of Medicine, The Chinese University of Hong  Kong, Sha Tin, Hong Kong SAR, China.\\nAD  - Department of Psychiatry, The Chinese University of Hong Kong, Sha Tin, Hong Kong  SAR, China.\\nFAU - Khodadadi-Jamayran, Alireza\\nAU  - Khodadadi-Jamayran A\\nAUID- ORCID: 0000-0003-2495-7504\\nAD  - Applied Bioinformatics Laboratories and Genome Technology Center, Division of  Advanced Research Technologies, New York University Grossman School of Medicine, NYU  Langone Health, New York, New York.\\nFAU - Xu, Mousheng\\nAU  - Xu M\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.\\nAD  - Department of Medicine, Harvard Medical School, Boston, Massachusetts.\\nFAU - Karatza, Angeliki\\nAU  - Karatza A\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Pyon, Val\\nAU  - Pyon V\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Li, Shuai\\nAU  - Li S\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Pan, Yuanwang\\nAU  - Pan Y\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Labbe, Kristen\\nAU  - Labbe K\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Almonte, Christina\\nAU  - Almonte C\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Poirier, John T\\nAU  - Poirier JT\\nAUID- ORCID: 0000-0001-9795-5644\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York.\\nFAU - Miller, George\\nAU  - Miller G\\nAD  - S. Arthur Localio Laboratory, Department of Surgery, New York University School of  Medicine, New York, New York.\\nFAU - Possemato, Richard\\nAU  - Possemato R\\nAUID- ORCID: 0000-0002-2401-0030\\nAD  - Department of Pathology, New York University School of Medicine, New York, New York.\\nFAU - Qi, Jun\\nAU  - Qi J\\nAUID- ORCID: 0000-0002-1461-3356\\nAD  - Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts.  Kwok-Kin.Wong@nyulangone.org Jun_Qi@dfci.harvard.edu.\\nAD  - Department of Medicine, Harvard Medical School, Boston, Massachusetts.\\nFAU - Wong, Kwok-Kin\\nAU  - Wong KK\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, New York. Kwok-Kin.Wong@nyulangone.org  Jun_Qi@dfci.harvard.edu.\\nLA  - eng\\nGR  - R01 GM132491/GM/NIGMS NIH HHS/United States\\nGR  - R01 CA214948/CA/NCI NIH HHS/United States\\nGR  - P01 CA154303/CA/NCI NIH HHS/United States\\nGR  - R01 CA222218/CA/NCI NIH HHS/United States\\nGR  - R01 CA205150/CA/NCI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20200709\\nTA  - Cancer Res\\nJT  - Cancer research\\nJID - 2984705R\\nRN  - 0 (NPM1 protein, human)\\nRN  - 0 (Npm1 protein, mouse)\\nRN  - 0 (Nuclear Proteins)\\nRN  - 117896-08-9 (Nucleophosmin)\\nSB  - IM\\nMH  - Animals\\nMH  - *Carcinoma, Non-Small-Cell Lung/genetics/metabolism/pathology\\nMH  - Cell Line, Tumor\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Epigenesis, Genetic\\nMH  - *Genetic Techniques\\nMH  - Heterografts\\nMH  - Humans\\nMH  - *Lung Neoplasms/genetics/metabolism/pathology\\nMH  - Mice\\nMH  - Nuclear Proteins/genetics/*metabolism\\nMH  - Nucleophosmin\\nPMC - PMC7493834\\nMID - NIHMS1611581\\nCOIS- COMPETING INTERESTS No potential conflicts of interest were disclosed by the other  authors.\\nEDAT- 2020/07/11 06:00\\nMHDA- 2021/01/12 06:00\\nCRDT- 2020/07/11 06:00\\nPHST- 2019/12/02 00:00 [received]\\nPHST- 2020/04/03 00:00 [revised]\\nPHST- 2020/07/06 00:00 [accepted]\\nPHST- 2020/07/11 06:00 [pubmed]\\nPHST- 2021/01/12 06:00 [medline]\\nPHST- 2020/07/11 06:00 [entrez]\\nAID - 0008-5472.CAN-19-3782 [pii]\\nAID - 10.1158/0008-5472.CAN-19-3782 [doi]\\nPST - ppublish\\nSO  - Cancer Res. 2020 Sep 1;80(17):3556-3567. doi: 10.1158/0008-5472.CAN-19-3782. Epub  2020 Jul 9.\\n\\nPMID- 33081350\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20201105\\nLR  - 20201218\\nIS  - 1999-4915 (Electronic)\\nIS  - 1999-4915 (Linking)\\nVI  - 12\\nIP  - 10\\nDP  - 2020 Oct 16\\nTI  - Viral Related Tools against SARS-CoV-2.\\nLID - 10.3390/v12101172 [doi]\\nLID - 1172\\nAB  - At the end of 2019, a new disease appeared and spread all over the world, the  COVID-19, produced by the coronavirus SARS-CoV-2. As a consequence of this worldwide  health crisis, the scientific community began to redirect their knowledge and  resources to fight against it. Here we summarize the recent research on viruses  employed as therapy and diagnostic of COVID-19: (i) viral-vector vaccines both in  clinical trials and pre-clinical phases; (ii) the use of bacteriophages to find  antibodies specific to this virus and some studies of how to use the bacteriophages  themselves as a treatment against viral diseases; and finally, (iii) the use of  CRISPR-Cas technology both to obtain a fast precise diagnose of the patient and also  the possible use of this technology as a cure.\\nFAU - Fernandez-Garcia, Laura\\nAU  - Fernandez-Garcia L\\nAUID- ORCID: 0000-0002-8531-6105\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - Pacios, Olga\\nAU  - Pacios O\\nAUID- ORCID: 0000-0002-4476-856X\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - GonzÃ¡lez-Bardanca, MÃ³nica\\nAU  - GonzÃ¡lez-Bardanca M\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - Blasco, Lucia\\nAU  - Blasco L\\nAUID- ORCID: 0000-0002-4039-4142\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - Bleriot, InÃ©s\\nAU  - Bleriot I\\nAUID- ORCID: 0000-0002-1846-4693\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - Ambroa, AntÃ³n\\nAU  - Ambroa A\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - LÃ³pez, MarÃ\\xada\\nAU  - LÃ³pez M\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nFAU - Bou, German\\nAU  - Bou G\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nAD  - Spanish Network for the Research in Infectious Diseases (REIPI), 41071 Sevilla,  Spain.\\nFAU - TomÃ¡s, Maria\\nAU  - TomÃ¡s M\\nAUID- ORCID: 0000-0003-4501-0387\\nAD  - Microbiology Department-Research Institute Biomedical A CoruÃ±a (INIBIC), Hospital A  CoruÃ±a (CHUAC), University of A CoruÃ±a (UDC), 15006 A CoruÃ±a, Spain.\\nAD  - Study Group on Mechanisms of Action and Resistance to Antimicrobials (GEMARA) of  Spanish Society of Infectious Diseases and Clinical Microbiology (SEIMC), 28003  Madrid, Spain.\\nAD  - Spanish Network for the Research in Infectious Diseases (REIPI), 41071 Sevilla,  Spain.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20201016\\nTA  - Viruses\\nJT  - Viruses\\nJID - 101509722\\nRN  - 0 (Antibodies, Viral)\\nRN  - 0 (Antiviral Agents)\\nRN  - 0 (Viral Vaccines)\\nSB  - IM\\nMH  - Antibodies, Viral/immunology/therapeutic use\\nMH  - Antiviral Agents/therapeutic use\\nMH  - Bacteriophages\\nMH  - Betacoronavirus/isolation & purification\\nMH  - COVID-19\\nMH  - CRISPR-Cas Systems\\nMH  - Coronavirus Infections/*therapy/*virology\\nMH  - Humans\\nMH  - Pandemics\\nMH  - Phage Therapy\\nMH  - Pneumonia, Viral/*therapy/*virology\\nMH  - SARS-CoV-2\\nMH  - Viral Vaccines/immunology\\nPMC - PMC7589879\\nOTO - NOTNLM\\nOT  - *COVID-19\\nOT  - *CRISPR\\nOT  - *SARS-CoV-2\\nOT  - *diagnosis\\nOT  - *phages\\nOT  - *prevention\\nOT  - *treatment\\nOT  - *viruses\\nCOIS- All authors declare no conflict of interest.\\nEDAT- 2020/10/22 06:00\\nMHDA- 2020/11/06 06:00\\nCRDT- 2020/10/21 01:03\\nPHST- 2020/09/18 00:00 [received]\\nPHST- 2020/10/14 00:00 [revised]\\nPHST- 2020/10/15 00:00 [accepted]\\nPHST- 2020/10/21 01:03 [entrez]\\nPHST- 2020/10/22 06:00 [pubmed]\\nPHST- 2020/11/06 06:00 [medline]\\nAID - v12101172 [pii]\\nAID - viruses-12-01172 [pii]\\nAID - 10.3390/v12101172 [doi]\\nPST - epublish\\nSO  - Viruses. 2020 Oct 16;12(10):1172. doi: 10.3390/v12101172.\\n\\nPMID- 31089700\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200603\\nLR  - 20200603\\nIS  - 1574-6968 (Electronic)\\nIS  - 0378-1097 (Print)\\nIS  - 0378-1097 (Linking)\\nVI  - 366\\nIP  - 8\\nDP  - 2019 Apr 1\\nTI  - Unexpected connections between type VI-B CRISPR-Cas systems, bacterial natural  competence, ubiquitin signaling network and DNA modification through a distinct  family of membrane proteins.\\nLID - 10.1093/femsle/fnz088 [doi]\\nLID - fnz088\\nAB  - In addition to core Cas proteins, CRISPR-Cas loci often encode ancillary proteins  that modulate the activity of the respective effectors in interference. Subtype  VI-B1 CRISPR-Cas systems encode the Csx27 protein that down-regulates the activity  of Cas13b when the type VI-B locus is expressed in Escherichia coli. We show that  Csx27 belongs to an expansive family of proteins that contain four predicted  transmembrane helices and are typically encoded in predicted operons with components  of the bacterial natural transformation machinery, multidomain proteins that consist  of components of the ubiquitin signaling system and proteins containing the  ligand-binding WYL domain and a helix-turn-helix domain. The Csx27 family proteins  are predicted to form membrane channels for ssDNA that might comprise the core of a  putative novel, Ub-regulated system for DNA uptake and, possibly, degradation. In  addition to these associations, a distinct subfamily of the Csx27 family appears to  be a part of a novel, membrane-associated system for DNA modification. In  Bacteroidetes, subtype VI-B1 systems might degrade nascent transcripts of foreign  DNA in conjunction with its uptake by the bacterial cell. These predictions suggest  several experimental directions for the study of type VI CRISPR-Cas systems and  distinct mechanisms of foreign DNA uptake and degradation in bacteria.\\nCI  - Published by Oxford University Press on behalf of FEMS 2019.\\nFAU - Makarova, Kira S\\nAU  - Makarova KS\\nAD  - National Center for Biotechnology Information, National Library of Medicine, 8600  Rockville pike, Bethesda, MD 20894, USA.\\nFAU - Gao, Linyi\\nAU  - Gao L\\nAD  - Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA.\\nAD  - Department of Biological Engineering, Massachusetts Institute of Technology, 77  Massachusetts Ave., Cambridge, MA 02139, USA.\\nFAU - Zhang, Feng\\nAU  - Zhang F\\nAD  - Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA.\\nAD  - Department of Biological Engineering, Massachusetts Institute of Technology, 77  Massachusetts Ave., Cambridge, MA 02139, USA.\\nAD  - McGovern Institute for Brain Research, Massachusetts Institute of Technology, 77  Massachusetts Ave., Cambridge, MA 02139, USA.\\nAD  - Howard Hughes Medical Institute, 77 Massachusetts Ave., Cambridge, MA 02139, USA.\\nAD  - Department of Brain and Cognitive Sciences, Massachusetts Institute of Technology,  77 Massachusetts Ave., Cambridge, MA 02139, USA.\\nFAU - Koonin, Eugene V\\nAU  - Koonin EV\\nAD  - National Center for Biotechnology Information, National Library of Medicine, 8600  Rockville pike, Bethesda, MD 20894, USA.\\nLA  - eng\\nGR  - DP1 HL141201/HL/NHLBI NIH HHS/United States\\nGR  - R01 HG009761/HG/NHGRI NIH HHS/United States\\nGR  - R01 MH110049/MH/NIMH NIH HHS/United States\\nGR  - HHMI/Howard Hughes Medical Institute/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, P.H.S.\\nTA  - FEMS Microbiol Lett\\nJT  - FEMS microbiology letters\\nJID - 7705721\\nRN  - 0 (DNA, Bacterial)\\nRN  - 0 (Escherichia coli Proteins)\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Ubiquitin)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - DNA, Bacterial/*chemistry\\nMH  - Escherichia coli/*genetics/metabolism\\nMH  - Escherichia coli Proteins/*genetics/metabolism\\nMH  - Genome, Bacterial\\nMH  - Genomics\\nMH  - Membrane Proteins/*genetics/metabolism\\nMH  - *Signal Transduction\\nMH  - Ubiquitin/*metabolism\\nPMC - PMC6700684\\nOTO - NOTNLM\\nOT  - *CRISPR ancillary proteins\\nOT  - *Csx27\\nOT  - *DNA modification\\nOT  - *membrane proteins\\nOT  - *natural competence\\nOT  - *type VI CRISPR-Cas\\nOT  - *ubiquitin network\\nEDAT- 2019/05/16 06:00\\nMHDA- 2020/06/04 06:00\\nCRDT- 2019/05/16 06:00\\nPHST- 2019/02/11 00:00 [received]\\nPHST- 2019/04/24 00:00 [accepted]\\nPHST- 2019/05/16 06:00 [entrez]\\nPHST- 2019/05/16 06:00 [pubmed]\\nPHST- 2020/06/04 06:00 [medline]\\nAID - 5479882 [pii]\\nAID - fnz088 [pii]\\nAID - 10.1093/femsle/fnz088 [doi]\\nPST - ppublish\\nSO  - FEMS Microbiol Lett. 2019 Apr 1;366(8):fnz088. doi: 10.1093/femsle/fnz088.\\n\\nPMID- 26635829\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20151204\\nLR  - 20200929\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Electronic)\\nIS  - 1664-462X (Linking)\\nVI  - 6\\nDP  - 2015\\nTI  - Targeting Non-Coding RNAs in Plants with the CRISPR-Cas Technology is a Challenge  yet Worth Accepting.\\nPG  - 1001\\nLID - 10.3389/fpls.2015.01001 [doi]\\nLID - 1001\\nAB  - Non-coding RNAs (ncRNAs) have emerged as versatile master regulator of biological  functions in recent years. MicroRNAs (miRNAs) are small endogenous ncRNAs of 18-24  nucleotides in length that originates from long self-complementary precursors.  Besides their direct involvement in developmental processes, plant miRNAs play key  roles in gene regulatory networks and varied biological processes. Alternatively,  long ncRNAs (lncRNAs) are a large and diverse class of transcribed ncRNAs whose  length exceed that of 200 nucleotides. Plant lncRNAs are transcribed by different  RNA polymerases, showing diverse structural features. Plant lncRNAs also are  important regulators of gene expression in diverse biological processes. There has  been a breakthrough in the technology of genome editing, the CRISPR-Cas9 (clustered  regulatory interspaced short palindromic repeats/CRISPR-associated protein 9)  technology, in the last decade. CRISPR loci are transcribed into ncRNA and  eventually form a functional complex with Cas9 and further guide the complex to  cleave complementary invading DNA. The CRISPR-Cas technology has been successfully  applied in model plants such as Arabidopsis and tobacco and important crops like  wheat, maize, and rice. However, all these studies are focused on protein coding  genes. Information about targeting non-coding genes is scarce. Hitherto, the  CRISPR-Cas technology has been exclusively used in vertebrate systems to engineer  miRNA/lncRNAs, but it is still relatively unexplored in plants. While briefing  miRNAs, lncRNAs and applications of the CRISPR-Cas technology in human and animals,  this review essentially elaborates several strategies to overcome the challenges of  applying the CRISPR-Cas technology in editing ncRNAs in plants and the future  perspective of this field.\\nFAU - Basak, Jolly\\nAU  - Basak J\\nAD  - Department of Biotechnology, Visva-Bharati University Santiniketan, India.\\nFAU - Nithin, Chandran\\nAU  - Nithin C\\nAD  - Computational Structural Biology Lab, Department of Biotechnology, Indian Institute  of Technology Kharagpur Kharagpur, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20151119\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC4652605\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - NHEJ\\nOT  - homologous recombination\\nOT  - lncRNA\\nOT  - miRNA\\nOT  - off-target mutation\\nEDAT- 2015/12/05 06:00\\nMHDA- 2015/12/05 06:01\\nCRDT- 2015/12/05 06:00\\nPHST- 2015/08/26 00:00 [received]\\nPHST- 2015/10/30 00:00 [accepted]\\nPHST- 2015/12/05 06:00 [entrez]\\nPHST- 2015/12/05 06:00 [pubmed]\\nPHST- 2015/12/05 06:01 [medline]\\nAID - 10.3389/fpls.2015.01001 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2015 Nov 19;6:1001. doi: 10.3389/fpls.2015.01001. eCollection 2015.\\n\\nPMID- 31349743\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200930\\nIS  - 2306-5354 (Print)\\nIS  - 2306-5354 (Electronic)\\nIS  - 2306-5354 (Linking)\\nVI  - 6\\nIP  - 3\\nDP  - 2019 Jul 25\\nTI  - WeReview: CRISPR Tools-Live Repository of Computational Tools for Assisting  CRISPR/Cas Experiments.\\nLID - 10.3390/bioengineering6030063 [doi]\\nLID - 63\\nAB  - Computational tools are essential in the process of designing a CRISPR/Cas  experiment for the targeted modification of an organism\\'s genome. Among other  functionalities, these tools facilitate the design of a guide-RNA (gRNA) for a given  nuclease that maximizes its binding to the intended genomic site, while avoiding  binding to undesired sites with similar sequences in the genome of the organism of  interest (off-targets). Due to the popularity of this methodology and the rapid pace  at which it evolves and changes, new computational tools show up constantly. This  rapid turnover, together with the intrinsic high death-rate of bioinformatics tools,  mean that many of the published tools become unavailable at some point.  Consequently, the traditional ways to inform the community about the landscape of  available tools, i.e., reviews in the scientific literature, are not adequate for  this fast-moving field. To overcome these limitations, we have developed \"WeReview:  CRISPR Tools,\" a live, on-line, user-updatable repository of computational tools to  assist researchers in designing CRISPR/Cas experiments. In its web site users can  find an updated comprehensive list of tools and search for those fulfilling their  specific needs, as well as proposing modifications to the data associated with the  tools or the incorporation of new ones.\\nFAU - Torres-Perez, Rafael\\nAU  - Torres-Perez R\\nAD  - National Centre for Biotechnology (CNB-CSIC). c/ Darwin 3, 28049 Madrid, Spain.\\nFAU - Garcia-Martin, Juan A\\nAU  - Garcia-Martin JA\\nAD  - National Centre for Biotechnology (CNB-CSIC). c/ Darwin 3, 28049 Madrid, Spain.\\nFAU - Montoliu, Lluis\\nAU  - Montoliu L\\nAD  - National Centre for Biotechnology (CNB-CSIC). c/ Darwin 3, 28049 Madrid, Spain.\\nAD  - Rare Diseases Networking Biomedical Research Centre (CIBERER-ISCIII), 28049 Madrid,  Spain.\\nFAU - Oliveros, Juan C\\nAU  - Oliveros JC\\nAD  - National Centre for Biotechnology (CNB-CSIC). c/ Darwin 3, 28049 Madrid, Spain.  oliveros@cnb.csic.es.\\nFAU - Pazos, Florencio\\nAU  - Pazos F\\nAD  - National Centre for Biotechnology (CNB-CSIC). c/ Darwin 3, 28049 Madrid, Spain.  pazos@cnb.csic.es.\\nLA  - eng\\nGR  - SAF2016-78041-C2-2-R/Ministerio de Ciencia y TecnologÃ\\xada/\\nPT  - Journal Article\\nDEP - 20190725\\nTA  - Bioengineering (Basel)\\nJT  - Bioengineering (Basel, Switzerland)\\nJID - 101676056\\nPMC - PMC6783853\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - computational tool\\nOT  - genome editing\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2019/07/28 06:00\\nMHDA- 2019/07/28 06:01\\nCRDT- 2019/07/28 06:00\\nPHST- 2019/07/09 00:00 [received]\\nPHST- 2019/07/17 00:00 [revised]\\nPHST- 2019/07/24 00:00 [accepted]\\nPHST- 2019/07/28 06:00 [entrez]\\nPHST- 2019/07/28 06:00 [pubmed]\\nPHST- 2019/07/28 06:01 [medline]\\nAID - bioengineering6030063 [pii]\\nAID - bioengineering-06-00063 [pii]\\nAID - 10.3390/bioengineering6030063 [doi]\\nPST - epublish\\nSO  - Bioengineering (Basel). 2019 Jul 25;6(3):63. doi: 10.3390/bioengineering6030063.\\n\\nPMID- 32994292\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20211026\\nIS  - 2379-5077 (Print)\\nIS  - 2379-5077 (Electronic)\\nIS  - 2379-5077 (Linking)\\nVI  - 5\\nIP  - 5\\nDP  - 2020 Sep 29\\nTI  - Virulence of the Pathogen Porphyromonas gingivalis Is Controlled by the CRISPR-Cas  Protein Cas3.\\nLID - 10.1128/mSystems.00852-20 [doi]\\nLID - e00852-20\\nAB  - The CRISPR (clustered regularly interspaced short palindromic repeat)-Cas system is  a unique genomic entity that provides prokaryotic cells with adaptive and heritable  immunity. Initial studies identified CRISPRs as central elements used by bacteria to  protect against foreign nucleic acids; however, emerging evidence points to CRISPR  involvement in bacterial virulence. The present study aimed to identify the  participation of one CRISPR-Cas protein, Cas3, in the virulence of the oral pathogen  Porphyromonas gingivalis, an organism highly associated with periodontitis. Our  results show that compared to the wild type, a mutant with a deletion of the Cas3  gene, an essential nuclease part of the class 1 type I CRISPR-Cas system, increased  the virulence of P. gingivalis In vitro infection modeling revealed only mildly  enhanced production of proinflammatory cytokines by THP-1 cells when infected with  the mutant strain. Dual transcriptome sequencing (RNA-seq) analysis of infected  THP-1 cells showed an increase in expression of genes associated with pathogenesis  in response to Î”cas3 mutant infection, with the target of Cas3 activities in  neutrophil chemotaxis and gene silencing. The importance of cas3 in controlling  virulence was corroborated in a Galleria mellonella infection model, where the  presence of the Î”cas3 mutant resulted in a statistically significant increase in  mortality of G. mellonella A time-series analysis of transcription patterning during  infection showed that G. mellonella elicited very different immune responses to the  wild-type and the Î”cas3 mutant strains and revealed a rearrangement of association  in coexpression networks. Together, these observations show for the first time that  Cas3 plays a significant role in regulating the virulence of P. gingivalis  IMPORTANCE Porphyromonas gingivalis is a key pathogen of periodontitis, a  polymicrobial disease characterized by a chronic inflammation that destroys the  tissues supporting the teeth. Thus, understanding the virulence potential of P.  gingivalis is essential to maintaining a healthy oral microbiome. In nonoral  organisms, CRISPR-Cas systems have been shown to modulate a variety of microbial  processes, including protection from exogenous nucleic acids, and, more recently,  have been implicated in bacterial virulence. Previously, our clinical findings  identified activation of the CRISPR-Cas system in patient samples at the transition  to disease; however, the mechanism of contribution to disease remained unknown. The  importance of the present study resides in that it is becoming increasingly clear  that CRISPR-associated proteins have broader functions than initially thought and  that those functions now include their role in the virulence of periodontal  pathogens. Studying a P. gingivalis cas3 mutant, we demonstrate that at least one of  the CRISPR-Cas systems is involved in the regulation of virulence during infection.\\nCI  - Copyright Â© 2020 Solbiati et al.\\nFAU - Solbiati, Jose\\nAU  - Solbiati J\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA.\\nFAU - Duran-Pinedo, Ana\\nAU  - Duran-Pinedo A\\nAUID- ORCID: 0000-0001-5648-6597\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA.\\nFAU - Godoy Rocha, Fernanda\\nAU  - Godoy Rocha F\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA.\\nFAU - Gibson, Frank C 3rd\\nAU  - Gibson FC 3rd\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA.\\nFAU - Frias-Lopez, Jorge\\nAU  - Frias-Lopez J\\nAUID- ORCID: 0000-0002-2097-3171\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA jfrias-lopez@dental.ufl.edu.\\nLA  - eng\\nGR  - R01 DE021553/DE/NIDCR NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20200929\\nTA  - mSystems\\nJT  - mSystems\\nJID - 101680636\\nPMC - PMC7527141\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - Porphyromonas gingivalis\\nOT  - pathogenesis\\nOT  - periodontal disease\\nOT  - virulence\\nEDAT- 2020/10/01 06:00\\nMHDA- 2020/10/01 06:01\\nCRDT- 2020/09/30 06:06\\nPHST- 2020/09/30 06:06 [entrez]\\nPHST- 2020/10/01 06:00 [pubmed]\\nPHST- 2020/10/01 06:01 [medline]\\nAID - 5/5/e00852-20 [pii]\\nAID - mSystems00852-20 [pii]\\nAID - 10.1128/mSystems.00852-20 [doi]\\nPST - epublish\\nSO  - mSystems. 2020 Sep 29;5(5):e00852-20. doi: 10.1128/mSystems.00852-20.\\n\\nPMID- 35154288\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20220216\\nIS  - 1664-8021 (Print)\\nIS  - 1664-8021 (Electronic)\\nIS  - 1664-8021 (Linking)\\nVI  - 13\\nDP  - 2022\\nTI  - The Basis and Promise of Programmable RNA Editing and Modification.\\nPG  - 834413\\nLID - 10.3389/fgene.2022.834413 [doi]\\nLID - 834413\\nAB  - One key advantage of RNA over genomic editing is its temporary effects. Aside from  current use of DNA-targeting CRISPR-Cas9, the more recently discovered CRISPR-Cas13  has been explored as a means of editing due to its RNA-targeting capabilities.  Specifically, there has been a recent interest in identifying and functionally  characterizing biochemical RNA modifications, which has spurred a new field of  research known as \"epitranscriptomics\". As one of the most frequently occurring  transcriptome modifications, N6-methyladenosine (m6A) has generated much interest.  The presence of m6A modifications is under the tight control of a series of  regulators, and the ability of fusing these proteins or demethylases to  catalytically inactive CRISPR proteins have resulted in a new wave of programmable  RNA methylation tools. In addition, studies have been conducted to develop different  CRISPR/Cas and base editor systems capable of more efficient editing, and some have  explored the effects of in vivo editing for certain diseases. As well, the  application of CRISPR and base editors for screening shows promise in revealing the  phenotypic outcomes from m6A modification, many of which are linked to  physiological, and pathological effects. Thus, the therapeutic potential of  CRISPR/Cas and base editors for not only m6A related, but other RNA and DNA related  disease has also garnered insight. In this review, we summarize/discuss the recent  findings on RNA editing with CRISPR, base editors and non-CRISPR related tools and  offer a perspective regarding future applications for basic and clinical research.\\nCI  - Copyright Â© 2022 Lo, Xu, Soares and He.\\nFAU - Lo, Nicholas\\nAU  - Lo N\\nAD  - Princess Margaret Cancer Center, University Health Network, Toronto, ON, Canada.\\nFAU - Xu, Xin\\nAU  - Xu X\\nAD  - Princess Margaret Cancer Center, University Health Network, Toronto, ON, Canada.\\nFAU - Soares, Fraser\\nAU  - Soares F\\nAD  - Princess Margaret Cancer Center, University Health Network, Toronto, ON, Canada.\\nFAU - He, Housheng Hansen\\nAU  - He HH\\nAD  - Princess Margaret Cancer Center, University Health Network, Toronto, ON, Canada.\\nAD  - Department of Medical Biophysics, University of Toronto, Toronto, ON, Canada.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20220128\\nTA  - Front Genet\\nJT  - Frontiers in genetics\\nJID - 101560621\\nPMC - PMC8831800\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - M6A\\nOT  - RNA editing\\nOT  - base editing\\nOT  - epitranscriptome\\nCOIS- The authors declare that the research was conducted in the absence of any commercial  or financial relationships that could be construed as a potential conflict of  interest.\\nEDAT- 2022/02/15 06:00\\nMHDA- 2022/02/15 06:01\\nCRDT- 2022/02/14 05:30\\nPHST- 2021/12/13 00:00 [received]\\nPHST- 2022/01/07 00:00 [accepted]\\nPHST- 2022/02/14 05:30 [entrez]\\nPHST- 2022/02/15 06:00 [pubmed]\\nPHST- 2022/02/15 06:01 [medline]\\nAID - 834413 [pii]\\nAID - 10.3389/fgene.2022.834413 [doi]\\nPST - epublish\\nSO  - Front Genet. 2022 Jan 28;13:834413. doi: 10.3389/fgene.2022.834413. eCollection  2022.\\n\\nPMID- 31177615\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200117\\nLR  - 20211204\\nIS  - 1755-0998 (Electronic)\\nIS  - 1755-098X (Linking)\\nVI  - 19\\nIP  - 5\\nDP  - 2019 Sep\\nTI  - The application of CRISPR-Cas for single species identification from environmental  DNA.\\nPG  - 1106-1114\\nLID - 10.1111/1755-0998.13045 [doi]\\nAB  - We report the first application of CRISPR-Cas technology to single species detection  from environmental DNA (eDNA). Organisms shed and excrete DNA into their environment  such as in skin cells and faeces, referred to as environmental DNA (eDNA). Utilising  eDNA allows noninvasive monitoring with increased specificity and sensitivity.  Current methods primarily employ PCR-based techniques to detect a given species from  eDNA samples, posing a logistical challenge for on-site monitoring and potential  adaptation to biosensor devices. We have developed an alternative method; coupling  isothermal amplification to a CRISPR-Cas12a detection system. This utilises the  collateral cleavage activity of Cas12a, a ribonuclease guided by a highly specific  single CRISPR RNA. We used the target species Salmo salar as a proof-of-concept test  of the specificity of the assay among closely related species and to show the assay  is successful at a single temperature of 37Â°C with signal detection at 535Â\\xa0nM. The  specific assay, detects at attomolar sensitivity with rapid detection rates  (<2.5Â\\xa0hr). This approach simplifies the challenge of building a biosensor device for  rapid target species detection in the field and can be easily adapted to detect any  species from eDNA samples from a variety of sources enhancing the capabilities of  eDNA as a tool for monitoring biodiversity.\\nCI  - Â© 2019 John Wiley & Sons Ltd.\\nFAU - Williams, Molly-Ann\\nAU  - Williams MA\\nAUID- ORCID: 0000-0002-7795-8001\\nAD  - School of Biotechnology, Dublin City University, Dublin, Ireland.\\nAD  - DCU Water Institute, Dublin City University, Dublin, Ireland.\\nFAU - O\\'Grady, Joyce\\nAU  - O\\'Grady J\\nAD  - DCU Water Institute, Dublin City University, Dublin, Ireland.\\nAD  - School of Chemical Sciences, Dublin City University, Dublin, Ireland.\\nFAU - Ball, Bernard\\nAU  - Ball B\\nAD  - Area 52 Research Group, School of Biology and Environmental Science/Earth Institute,  University College Dublin, Dublin, Ireland.\\nFAU - Carlsson, Jens\\nAU  - Carlsson J\\nAD  - Area 52 Research Group, School of Biology and Environmental Science/Earth Institute,  University College Dublin, Dublin, Ireland.\\nFAU - de Eyto, Elvira\\nAU  - de Eyto E\\nAD  - Marine Institute, Co. Mayo, Ireland.\\nFAU - McGinnity, Philip\\nAU  - McGinnity P\\nAD  - Marine Institute, Co. Mayo, Ireland.\\nAD  - School of Biological, Earth & Environmental Sciences, University College Cork, Cork,  Ireland.\\nFAU - Jennings, Eleanor\\nAU  - Jennings E\\nAD  - Centre for Freshwater Studies and Department of Applied Sciences, Dundalk Institute  of Technology, Dundalk, Ireland.\\nFAU - Regan, Fiona\\nAU  - Regan F\\nAD  - DCU Water Institute, Dublin City University, Dublin, Ireland.\\nAD  - School of Chemical Sciences, Dublin City University, Dublin, Ireland.\\nFAU - Parle-McDermott, Anne\\nAU  - Parle-McDermott A\\nAUID- ORCID: 0000-0002-1533-3209\\nAD  - School of Biotechnology, Dublin City University, Dublin, Ireland.\\nAD  - DCU Water Institute, Dublin City University, Dublin, Ireland.\\nLA  - eng\\nGR  - BEYOND 2020 PBA/FS/16/02/Irish Marine Institute as part of the Burrishoole Ecosystem  Observatory Network 2020:/\\nPT  - Journal Article\\nDEP - 20190718\\nPL  - England\\nTA  - Mol Ecol Resour\\nJT  - Molecular ecology resources\\nJID - 101465604\\nRN  - 0 (DNA, Environmental)\\nSB  - IM\\nCIN - Mol Ecol Resour. 2019 Sep;19(5):1103-1105. PMID: 31560843\\nMH  - Animals\\nMH  - *Biota\\nMH  - *Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - DNA, Environmental/analysis/*genetics\\nMH  - Gene Editing/*methods\\nMH  - Nucleic Acid Amplification Techniques/*methods\\nMH  - Salmo salar/*classification/*genetics\\nMH  - Sensitivity and Specificity\\nMH  - Temperature\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - biosensor\\nOT  - eDNA\\nOT  - environmental\\nOT  - freshwater\\nOT  - salmon\\nEDAT- 2019/06/10 06:00\\nMHDA- 2020/01/18 06:00\\nCRDT- 2019/06/10 06:00\\nPHST- 2019/02/23 00:00 [received]\\nPHST- 2019/05/30 00:00 [revised]\\nPHST- 2019/05/31 00:00 [accepted]\\nPHST- 2019/06/10 06:00 [pubmed]\\nPHST- 2020/01/18 06:00 [medline]\\nPHST- 2019/06/10 06:00 [entrez]\\nAID - 10.1111/1755-0998.13045 [doi]\\nPST - ppublish\\nSO  - Mol Ecol Resour. 2019 Sep;19(5):1106-1114. doi: 10.1111/1755-0998.13045. Epub 2019  Jul 18.\\n\\nPMID- 33398095\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210422\\nLR  - 20210803\\nIS  - 2058-5276 (Electronic)\\nIS  - 2058-5276 (Linking)\\nVI  - 6\\nIP  - 2\\nDP  - 2021 Feb\\nTI  - The Rcs stress response inversely controls surface and CRISPR-Cas adaptive immunity  to discriminate plasmids and phages.\\nPG  - 162-172\\nLID - 10.1038/s41564-020-00822-7 [doi]\\nAB  - Bacteria harbour multiple innate defences and adaptive CRISPR-Cas systems that  provide immunity against bacteriophages and mobile genetic elements. Although some  bacteria modulate defences in response to population density, stress and metabolic  state, a lack of high-throughput methods to systematically reveal regulators has  hampered efforts to understand when and how immune strategies are deployed. We  developed a robust approach called SorTn-seq, which combines saturation transposon  mutagenesis, fluorescence-activated cell sorting and deep sequencing to characterize  regulatory networks controlling CRISPR-Cas immunity in Serratia sp. ATCC 39006. We  applied our technology to assess csm gene expression for ~300,000 mutants and  uncovered multiple pathways regulating type III-A CRISPR-Cas expression. Mutation of  igaA or mdoG activated the Rcs outer-membrane stress response, eliciting  cell-surface-based innate immunity against diverse phages via the transcriptional  regulators RcsB and RcsA. Activation of this Rcs phosphorelay concomitantly  attenuated adaptive immunity by three distinct type I and III CRISPR-Cas systems.  Rcs-mediated repression of CRISPR-Cas defence enabled increased acquisition and  retention of plasmids. Dual downregulation of cell-surface receptors and adaptive  immunity in response to stress by the Rcs pathway enables protection from phage  infection without preventing the uptake of plasmids that may harbour beneficial  traits.\\nFAU - Smith, Leah M\\nAU  - Smith LM\\nAUID- ORCID: 0000-0001-8223-8407\\nAD  - Department of Microbiology and Immunology, University of Otago, Dunedin, New  Zealand.\\nFAU - Jackson, Simon A\\nAU  - Jackson SA\\nAD  - Department of Microbiology and Immunology, University of Otago, Dunedin, New  Zealand.\\nAD  - Genetics Otago, University of Otago, Dunedin, New Zealand.\\nFAU - Malone, Lucia M\\nAU  - Malone LM\\nAUID- ORCID: 0000-0002-3584-5084\\nAD  - Department of Microbiology and Immunology, University of Otago, Dunedin, New  Zealand.\\nFAU - Ussher, James E\\nAU  - Ussher JE\\nAUID- ORCID: 0000-0001-9222-7680\\nAD  - Department of Microbiology and Immunology, University of Otago, Dunedin, New  Zealand.\\nFAU - Gardner, Paul P\\nAU  - Gardner PP\\nAD  - Genetics Otago, University of Otago, Dunedin, New Zealand.\\nAD  - Department of Biochemistry, University of Otago, Dunedin, New Zealand.\\nAD  - Bio-Protection Research Centre, University of Otago, Dunedin, New Zealand.\\nFAU - Fineran, Peter C\\nAU  - Fineran PC\\nAUID- ORCID: 0000-0002-4639-6704\\nAD  - Department of Microbiology and Immunology, University of Otago, Dunedin, New  Zealand. peter.fineran@otago.ac.nz.\\nAD  - Genetics Otago, University of Otago, Dunedin, New Zealand.  peter.fineran@otago.ac.nz.\\nAD  - Bio-Protection Research Centre, University of Otago, Dunedin, New Zealand.  peter.fineran@otago.ac.nz.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210104\\nPL  - England\\nTA  - Nat Microbiol\\nJT  - Nature microbiology\\nJID - 101674869\\nRN  - 0 (Bacterial Proteins)\\nSB  - IM\\nCIN - Nat Microbiol. 2021 Feb;6(2):149-150. PMID: 33510433\\nMH  - Bacterial Proteins/genetics/*physiology\\nMH  - Bacteriophages/genetics/*physiology\\nMH  - CRISPR-Cas Systems/genetics/*physiology\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics/physiology\\nMH  - Flow Cytometry\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - High-Throughput Nucleotide Sequencing\\nMH  - Mutagenesis\\nMH  - Plasmids/genetics/physiology\\nMH  - Serratia/*physiology/*virology\\nMH  - Stress, Physiological/genetics\\nEDAT- 2021/01/06 06:00\\nMHDA- 2021/04/23 06:00\\nCRDT- 2021/01/05 06:21\\nPHST- 2020/08/30 00:00 [received]\\nPHST- 2020/10/23 00:00 [accepted]\\nPHST- 2021/01/06 06:00 [pubmed]\\nPHST- 2021/04/23 06:00 [medline]\\nPHST- 2021/01/05 06:21 [entrez]\\nAID - 10.1038/s41564-020-00822-7 [pii]\\nAID - 10.1038/s41564-020-00822-7 [doi]\\nPST - ppublish\\nSO  - Nat Microbiol. 2021 Feb;6(2):162-172. doi: 10.1038/s41564-020-00822-7. Epub 2021 Jan  4.\\n\\nPMID- 33897053\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210605\\nIS  - 0026-265X (Print)\\nIS  - 1095-9149 (Electronic)\\nIS  - 0026-265X (Linking)\\nVI  - 167\\nDP  - 2021 Aug\\nTI  - Current state of diagnostic, screening and surveillance testing methods for COVID-19  from an analytical chemistry point of view.\\nPG  - 106305\\nLID - 10.1016/j.microc.2021.106305 [doi]\\nAB  - Since December 2019, we have been in the battlefield with a new threat to the  humanity known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). In  this review, we describe the four main methods used for diagnosis, screening and/or  surveillance of SARS-CoV-2: Real-time reverse transcription polymerase chain  reaction (RT-PCR); chest computed tomography (CT); and different complementary  alternatives developed in order to obtain rapid results, antigen and antibody  detection. All of them compare the highlighting advantages and disadvantages from an  analytical point of view. The gold standard method in terms of sensitivity and  specificity is the RT-PCR. The different modifications propose to make it more rapid  and applicable at point of care (POC) are also presented and discussed. CT images  are limited to central hospitals. However, being combined with RT-PCR is the most  robust and accurate way to confirm COVID-19 infection. Antibody tests, although  unable to provide reliable results on the status of the infection, are suitable for  carrying out maximum screening of the population in order to know the immune  capacity. More recently, antigen tests, less sensitive than RT-PCR, have been  authorized to determine in a quicker way whether the patient is infected at the time  of analysis and without the need of specific instruments.\\nCI  - Â© 2021 Elsevier B.V. All rights reserved.\\nFAU - MartÃ\\xadn, Julia\\nAU  - MartÃ\\xadn J\\nAD  - Departamento de QuÃ\\xadmica AnalÃ\\xadtica, Escuela PolitÃ©cnica Superior, Universidad de  Sevilla, C/ Virgen de Ãfrica 7, Sevilla E-41011, Spain.\\nFAU - Tena, Noelia\\nAU  - Tena N\\nAD  - Departamento de QuÃ\\xadmica AnalÃ\\xadtica, Facultad de Farmacia, Universidad de Sevilla,  Prof. GarcÃ\\xada GonzÃ¡lez, 2, Sevilla 41012, Spain.\\nFAU - Asuero, Agustin G\\nAU  - Asuero AG\\nAD  - Departamento de QuÃ\\xadmica AnalÃ\\xadtica, Facultad de Farmacia, Universidad de Sevilla,  Prof. GarcÃ\\xada GonzÃ¡lez, 2, Sevilla 41012, Spain.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210419\\nTA  - Microchem J\\nJT  - Microchemical journal : devoted to the application of microtechniques in all  branches of science\\nJID - 0376647\\nPMC - PMC8054532\\nOTO - NOTNLM\\nOT  - 2019-nCoV, 2019 novel coronavirus\\nOT  - ACE2, Angiotensin-Converting Enzyme 2\\nOT  - AI, Artificial Intelligence\\nOT  - ALP, Alkaline Phosphatase\\nOT  - ASOs, Antisense Oligonucleotides\\nOT  - Antigen and antibody tests\\nOT  - AuNIs, Gold Nanoislands\\nOT  - AuNPs, Gold Nanoparticles\\nOT  - BSL, Biosecurity Level\\nOT  - CAP, College of American Pathologists\\nOT  - CCD, Charge-Coupled Device\\nOT  - CG, Colloidal Gold\\nOT  - CGIA, Colloidal Gold Immunochromatographic Assay\\nOT  - CLIA, Chemiluminescence Enzyme Immunoassay\\nOT  - CLIA, Clinical Laboratory Improvement Amendments\\nOT  - COVID-19\\nOT  - COVID-19, Coronavirus disease-19\\nOT  - CRISPR, Clustered Regularly Interspaced Short Palindromic Repeats\\nOT  - CT, Chest Computed Tomography\\nOT  - Cas, CRISPR Associate Protein\\nOT  - China CDC, Chinese Center for Disease Control and Prevention\\nOT  - Ct, Cycle Threshold\\nOT  - DETECTR, SARS-CoV-2 DNA Endonuclease-Targeted CRISPR Trans Reporter\\nOT  - DNA, Dexosyrosyribonucleic Acid\\nOT  - E, Envelope protein\\nOT  - ELISA, Enzyme Linked Immunosorbent Assay\\nOT  - EMA, European Medicines Agency\\nOT  - EUA, Emergence Use Authorization\\nOT  - FDA, Food and Drug Administration\\nOT  - FET, Field-Effect Transistor\\nOT  - GISAID, Global Initiative on Sharing All Influenza Data\\nOT  - GeneBank, Genetic sequence data base of the National Institute of Health\\nOT  - ICTV, International Committee on Taxonomy of Viruses\\nOT  - IgA, Immunoglobulins A\\nOT  - IgG, Immunoglobulins G\\nOT  - IgM, Immunoglobulins M\\nOT  - IoMT, Internet of Medical Things\\nOT  - IoT, Internet of Things\\nOT  - LFIA, Lateral Flow Immunochromatographic Assays\\nOT  - LOC, Lab-on-a-Chip\\nOT  - LOD, Limit of detection\\nOT  - LSPR, Localized Surface Plasmon Resonance\\nOT  - M, Membrane protein\\nOT  - MERS-CoV, Middle East Respiratory Syndrome Coronavirus\\nOT  - MNP, Magnetic Nanoparticle\\nOT  - MS, Mass spectrometry\\nOT  - N, Nucleocapsid protein\\nOT  - NER, Naked Eye Readout\\nOT  - NGM, Next Generation Molecular\\nOT  - NGS, Next Generation Sequencing\\nOT  - NIH, National Institute of Health\\nOT  - NSPs, Nonstructural Proteins\\nOT  - Net, Neural Network\\nOT  - ORF, Open Reading Frame\\nOT  - OSN, One Step Single-tube Nested\\nOT  - PDMS, Polydimethylsiloxane\\nOT  - POC, Point of Care\\nOT  - PPT, Plasmonic Photothermal\\nOT  - QD, Quantum Dot\\nOT  - R0, Basic reproductive number\\nOT  - RBD, Receptor-binding domain\\nOT  - RNA, Ribonucleic Acid\\nOT  - RNaseH, Ribonuclease H\\nOT  - RT, Reverse Transcriptase\\nOT  - RT-LAMP, Reverse Transcription Loop-Mediated Isothermal Amplification\\nOT  - RT-PCR, Real-Time Reverse Transcription Polymerase Chain Reaction\\nOT  - RT-PCR, chest computerized tomography\\nOT  - RdRp, RNA-Dependent RNA Polymerase\\nOT  - S, Spike protein\\nOT  - SARS-CoV-2\\nOT  - SARS-CoV-2, Severe Acute Respiratory Syndrome Coronavirus 2\\nOT  - SERS, Surface Enhanced Raman Spectroscopy\\nOT  - SHERLOCK, Specific High Sensitivity Enzymatic Reporter UnLOCKing\\nOT  - STOPCovid, SHERLOCK Testing on One Pot\\nOT  - SVM, Support Vector Machine\\nOT  - SiO2@Ag, Complete silver nanoparticle shell coated on silica core\\nOT  - US CDC, US Centers for Disease Control and Prevention\\nOT  - VOC, Variant of Concern\\nOT  - VTM, Viral Transport Medium\\nOT  - WGS, Whole Genome Sequencing\\nOT  - WHO, World Health Organization\\nOT  - aM, Attomolar\\nOT  - dNTPs, Nucleotides\\nOT  - dPCR, Digital PCR\\nOT  - ddPCR, Droplet digital PCR\\nOT  - fM, Femtomolar\\nOT  - m-RNA, Messenger Ribonucleic Acid\\nOT  - nM, Nanomolar\\nOT  - pM, Picomolar\\nOT  - pfu, Plaque-forming unit\\nOT  - rN, Recombinant nucleocapsid protein antigen\\nOT  - rS, Recombinant Spike protein antigen\\nOT  - ssRNA, Single-Stranded Positive-Sense RNA\\nCOIS- The authors declare that they have no known competing financial interests or  personal relationships that could have appeared to influence the work reported in  this paper.\\nEDAT- 2021/04/27 06:00\\nMHDA- 2021/04/27 06:01\\nCRDT- 2021/04/26 05:42\\nPHST- 2021/02/10 00:00 [received]\\nPHST- 2021/04/12 00:00 [revised]\\nPHST- 2021/04/14 00:00 [accepted]\\nPHST- 2021/04/27 06:00 [pubmed]\\nPHST- 2021/04/27 06:01 [medline]\\nPHST- 2021/04/26 05:42 [entrez]\\nAID - S0026-265X(21)00389-1 [pii]\\nAID - 106305 [pii]\\nAID - 10.1016/j.microc.2021.106305 [doi]\\nPST - ppublish\\nSO  - Microchem J. 2021 Aug;167:106305. doi: 10.1016/j.microc.2021.106305. Epub 2021 Apr  19.\\n\\nPMID- 31238294\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200207\\nLR  - 20200207\\nIS  - 1879-0429 (Electronic)\\nIS  - 0958-1669 (Linking)\\nVI  - 59\\nDP  - 2019 Oct\\nTI  - Genetic and metabolic advances in the engineering of cyanobacteria.\\nPG  - 150-156\\nLID - S0958-1669(18)30150-2 [pii]\\nLID - 10.1016/j.copbio.2019.05.012 [doi]\\nAB  - Cyanobacteria are a group of photosynthetic microorganisms with high commercial  potential. They can utilize sunlight directly to convert carbon dioxide or even  nitrogen into a variety of industrially relevant chemicals. However, commercial  platforms for the renewable and sustainable production of chemicals have yet to be  demonstrated for cyanobacteria. Diverse strategies have therefore been employed in  recent years to improve the production yields and efficiency of target chemicals.  These include the use of CRISPR/Cas systems for mutant selection, synthetic RNA  elements for controlling transcription, metabolic network modelling for  understanding pathway fluxes, enzyme engineering, improving growth rates,  alleviating product toxicity and microbial consortia. More elaborate strategies for  engineering cyanobacteria, however, are still very much required if we are to meet  the grand challenge of employing cyanobacteria as photosynthetic workhorses for  large-scale industrial applications.\\nCI  - Copyright Â© 2019 Elsevier Ltd. All rights reserved.\\nFAU - Vijay, Dhanya\\nAU  - Vijay D\\nAD  - Department of Chemistry, United Arab Emirates University, Al-Ain, United Arab  Emirates.\\nFAU - Akhtar, M Kalim\\nAU  - Akhtar MK\\nAD  - Department of Chemistry, United Arab Emirates University, Al-Ain, United Arab  Emirates. Electronic address: mk.akhtar@uaeu.ac.ae.\\nFAU - Hess, Wolfgang R\\nAU  - Hess WR\\nAD  - Genetics and Experimental Bioinformatics, Institute of Biology 3, Faculty of  Biology, University of Freiburg, Germany; Freiburg Institute for Advanced Studies,  University of Freiburg, Germany. Electronic address:  wolfgang.hess@biologie.uni-freiburg.de.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20190622\\nPL  - England\\nTA  - Curr Opin Biotechnol\\nJT  - Current opinion in biotechnology\\nJID - 9100492\\nRN  - 0 (Biofuels)\\nRN  - 142M471B3J (Carbon Dioxide)\\nSB  - IM\\nMH  - Biofuels\\nMH  - CRISPR-Cas Systems\\nMH  - Carbon Dioxide\\nMH  - *Cyanobacteria\\nMH  - Genetic Engineering\\nMH  - Metabolic Engineering\\nMH  - Photosynthesis\\nEDAT- 2019/06/27 06:00\\nMHDA- 2020/02/08 06:00\\nCRDT- 2019/06/26 06:00\\nPHST- 2018/12/15 00:00 [received]\\nPHST- 2019/05/16 00:00 [revised]\\nPHST- 2019/05/22 00:00 [accepted]\\nPHST- 2019/06/27 06:00 [pubmed]\\nPHST- 2020/02/08 06:00 [medline]\\nPHST- 2019/06/26 06:00 [entrez]\\nAID - S0958-1669(18)30150-2 [pii]\\nAID - 10.1016/j.copbio.2019.05.012 [doi]\\nPST - ppublish\\nSO  - Curr Opin Biotechnol. 2019 Oct;59:150-156. doi: 10.1016/j.copbio.2019.05.012. Epub  2019 Jun 22.\\n\\nPMID- 28985507\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20171017\\nLR  - 20181113\\nIS  - 1097-4164 (Electronic)\\nIS  - 1097-2765 (Print)\\nIS  - 1097-2765 (Linking)\\nVI  - 68\\nIP  - 1\\nDP  - 2017 Oct 5\\nTI  - Randomized CRISPR-Cas Transcriptional Perturbation Screening Reveals Protective  Genes against Alpha-Synuclein Toxicity.\\nPG  - 247-257.e5\\nLID - S1097-2765(17)30665-2 [pii]\\nLID - 10.1016/j.molcel.2017.09.014 [doi]\\nAB  - The genome-wide perturbation of transcriptional networks with CRISPR-Cas technology  has primarily involved systematic and targeted gene modulation. Here, we developed  PRISM (Perturbing Regulatory Interactions by Synthetic Modulators), a screening  platform that uses randomized CRISPR-Cas transcription factors (crisprTFs) to  globally perturb transcriptional networks. By applying PRISM to a yeast model of  Parkinson\\'s disease (PD), we identified guide RNAs (gRNAs) that modulate  transcriptional networks and protect cells from alpha-synuclein (Î±Syn) toxicity. One  gRNA identified in this screen outperformed the most protective suppressors of Î±Syn  toxicity reported previously, highlighting PRISM\\'s ability to identify modulators of  important phenotypes. Gene expression profiling revealed genes differentially  modulated by this strong protective gRNA that rescued yeast from Î±Syn toxicity when  overexpressed. Human homologs of top-ranked hits protected against Î±Syn-induced cell  death in a human neuronal PD model. Thus, high-throughput and unbiased perturbation  of transcriptional networks via randomized crisprTFs can reveal complex biological  phenotypes and effective disease modulators.\\nCI  - Copyright Â© 2017. Published by Elsevier Inc.\\nFAU - Chen, Ying-Chou\\nAU  - Chen YC\\nAD  - Synthetic Biology Group, MIT Synthetic Biology Center, Massachusetts Institute of  Technology, Cambridge, MA 02139, USA; Research Laboratory of Electronics,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\nFAU - Farzadfard, Fahim\\nAU  - Farzadfard F\\nAD  - Synthetic Biology Group, MIT Synthetic Biology Center, Massachusetts Institute of  Technology, Cambridge, MA 02139, USA; Research Laboratory of Electronics,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA; MIT Microbiology  Program, Massachusetts Institute of Technology, Cambridge, MA 02139, USA; Department  of Biological Engineering and Electrical Engineering & Computer Science,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\nFAU - Gharaei, Nava\\nAU  - Gharaei N\\nAD  - MCO Graduate Program, Department of Molecular and Cellular Biology, Harvard  University, Cambridge, MA 02138, USA.\\nFAU - Chen, William C W\\nAU  - Chen WCW\\nAD  - Synthetic Biology Group, MIT Synthetic Biology Center, Massachusetts Institute of  Technology, Cambridge, MA 02139, USA; Research Laboratory of Electronics,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\nFAU - Cao, Jicong\\nAU  - Cao J\\nAD  - Synthetic Biology Group, MIT Synthetic Biology Center, Massachusetts Institute of  Technology, Cambridge, MA 02139, USA; Research Laboratory of Electronics,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\nFAU - Lu, Timothy K\\nAU  - Lu TK\\nAD  - Synthetic Biology Group, MIT Synthetic Biology Center, Massachusetts Institute of  Technology, Cambridge, MA 02139, USA; Research Laboratory of Electronics,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA; MIT Microbiology  Program, Massachusetts Institute of Technology, Cambridge, MA 02139, USA; Department  of Biological Engineering and Electrical Engineering & Computer Science,  Massachusetts Institute of Technology, Cambridge, MA 02139, USA. Electronic address:  timlu@mit.edu.\\nLA  - eng\\nGR  - P30 DK043351/DK/NIDDK NIH HHS/United States\\nGR  - P50 GM098792/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nTA  - Mol Cell\\nJT  - Molecular cell\\nJID - 9802571\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Transcription Factors)\\nRN  - 0 (alpha-Synuclein)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - Cell Line, Tumor\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Gene Expression Profiling\\nMH  - Gene Expression Regulation\\nMH  - *Gene Regulatory Networks\\nMH  - High-Throughput Screening Assays\\nMH  - Humans\\nMH  - Models, Biological\\nMH  - Neurons/metabolism/pathology\\nMH  - Parkinson Disease/genetics/metabolism/pathology\\nMH  - Phenotype\\nMH  - RNA, Guide/*genetics/metabolism\\nMH  - Saccharomyces cerevisiae/genetics/metabolism\\nMH  - Transcription Factors/*genetics/metabolism\\nMH  - *Transcription, Genetic\\nMH  - Transgenes\\nMH  - alpha-Synuclein/antagonists & inhibitors/*genetics/metabolism\\nPMC - PMC5702536\\nMID - NIHMS919242\\nOTO - NOTNLM\\nOT  - CRISPR screening\\nOT  - PRISM\\nOT  - Parkinsonâ€™s disease\\nOT  - Perturbing Regulatory Interactions by Synthetic Modulators\\nOT  - alpha-synuclein\\nOT  - global transcriptional perturbations\\nOT  - randomized gRNA library\\nOT  - synthetic transcription factors\\nEDAT- 2017/10/07 06:00\\nMHDA- 2017/10/19 06:00\\nCRDT- 2017/10/07 06:00\\nPHST- 2016/12/07 00:00 [received]\\nPHST- 2017/06/05 00:00 [revised]\\nPHST- 2017/09/12 00:00 [accepted]\\nPHST- 2017/10/07 06:00 [entrez]\\nPHST- 2017/10/07 06:00 [pubmed]\\nPHST- 2017/10/19 06:00 [medline]\\nAID - S1097-2765(17)30665-2 [pii]\\nAID - 10.1016/j.molcel.2017.09.014 [doi]\\nPST - ppublish\\nSO  - Mol Cell. 2017 Oct 5;68(1):247-257.e5. doi: 10.1016/j.molcel.2017.09.014.\\n\\nPMID- 35157372\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220216\\nLR  - 20220216\\nIS  - 2691-1299 (Electronic)\\nIS  - 2691-1299 (Linking)\\nVI  - 2\\nIP  - 2\\nDP  - 2022 Feb\\nTI  - CRISPR-Act3.0-Based Highly Efficient Multiplexed Gene Activation in Plants.\\nPG  - e365\\nLID - 10.1002/cpz1.365 [doi]\\nAB  - CRISPR/Cas (clustered regularly interspaced short palindromic  repeats/CRISPR-associated protein)-mediated genome editing has revolutionized  fundamental research and plant breeding. Beyond gene editing, CRISPR/Cas systems  have been repurposed as a platform for programmable transcriptional regulation.  Catalytically inactive Cas variants (dCas), when fused with transcriptional  activation domains, allow for specific activation of any target gene in the genome  without inducing DNA double-strand breaks. CRISPR activation enables simultaneous  activation of multiple genes, holding great promise in the identification of gene  regulatory networks and rewiring of metabolic pathways. Here, we describe a simple  protocol for constructing a dCas9-mediated multiplexed gene activation system based  on the CRISPR-Act3.0 system. The resulting vectors are tested in rice protoplasts. Â©  2022 Wiley Periodicals LLC. Basic Protocol 1: sgRNA design and construction of  CRISPR-Act3.0 vectors for multiplexed gene activation Basic Protocol 2: Determining  the activation efficiency of CRISPR-Act3.0 vectors using rice protoplasts.\\nCI  - Â© 2022 Wiley Periodicals LLC.\\nFAU - Pan, Changtian\\nAU  - Pan C\\nAD  - Department of Plant Science and Landscape Architecture, University of Maryland,  College Park, Maryland.\\nFAU - Qi, Yiping\\nAU  - Qi Y\\nAD  - Department of Plant Science and Landscape Architecture, University of Maryland,  College Park, Maryland.\\nAD  - Institute for Bioscience and Biotechnology Research, University of Maryland,  Rockville, Maryland.\\nLA  - eng\\nGR  - IOS-1758745/National Science Foundation/\\nGR  - IOS-2029889/National Science Foundation/\\nPT  - Journal Article\\nPL  - United States\\nTA  - Curr Protoc\\nJT  - Current protocols\\nJID - 101773894\\nSB  - IM\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - *Gene Editing\\nMH  - *Plant Breeding\\nMH  - Plants/genetics\\nMH  - Transcriptional Activation\\nOTO - NOTNLM\\nOT  - CRISPR-Act3.0\\nOT  - dCas9\\nOT  - multiplexed gene activation\\nOT  - rice protoplast system\\nOT  - sgRNA design\\nOT  - tRNA\\nEDAT- 2022/02/15 06:00\\nMHDA- 2022/02/17 06:00\\nCRDT- 2022/02/14 12:19\\nPHST- 2022/02/14 12:19 [entrez]\\nPHST- 2022/02/15 06:00 [pubmed]\\nPHST- 2022/02/17 06:00 [medline]\\nAID - 10.1002/cpz1.365 [doi]\\nPST - ppublish\\nSO  - Curr Protoc. 2022 Feb;2(2):e365. doi: 10.1002/cpz1.365.\\n\\nPMID- 25619936\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151124\\nLR  - 20191210\\nIS  - 1879-1301 (Electronic)\\nIS  - 1074-5521 (Linking)\\nVI  - 22\\nIP  - 2\\nDP  - 2015 Feb 19\\nTI  - CRISPR-Cas9-based photoactivatable transcription system.\\nPG  - 169-74\\nLID - S1074-5521(14)00459-1 [pii]\\nLID - 10.1016/j.chembiol.2014.12.011 [doi]\\nAB  - Targeted endogenous gene activation is necessary for understanding complex gene  networks and has great potential in medical and industrial applications. The  CRISPR-Cas system offers simple and powerful tools for this purpose. However, these  CRISPR-Cas-based tools for activating user-defined genes are unable to offer precise  temporal control of gene expression, despite the fact that many biological phenomena  are regulated by highly dynamic patterns of gene expression. Here we created a  light-inducible, user-defined, endogenous gene activation system based on  CRISPR-Cas9. We demonstrated that thisÂ\\xa0CRISPR-Cas9-based transcription system can  allowÂ\\xa0rapid and reversible targeted gene activation by light. In addition, using  this system, we have exemplified photoactivation of multiple user-defined endogenous  genes in mammalian cells. The present CRISPR-Cas9-based transcription system offers  simple and versatile approaches for precise endogenous gene activation in basic  biological research and biotechnology applications.\\nCI  - Copyright Â© 2015 Elsevier Ltd. All rights reserved.\\nFAU - Nihongaki, Yuta\\nAU  - Nihongaki Y\\nAD  - Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba,  Meguro-ku, Tokyo 153-8902, Japan.\\nFAU - Yamamoto, Shun\\nAU  - Yamamoto S\\nAD  - Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba,  Meguro-ku, Tokyo 153-8902, Japan.\\nFAU - Kawano, Fuun\\nAU  - Kawano F\\nAD  - Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba,  Meguro-ku, Tokyo 153-8902, Japan.\\nFAU - Suzuki, Hideyuki\\nAU  - Suzuki H\\nAD  - Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba,  Meguro-ku, Tokyo 153-8902, Japan.\\nFAU - Sato, Moritoshi\\nAU  - Sato M\\nAD  - Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba,  Meguro-ku, Tokyo 153-8902, Japan. Electronic address: cmsato@mail.ecc.u-tokyo.ac.jp.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20150122\\nPL  - United States\\nTA  - Chem Biol\\nJT  - Chemistry & biology\\nJID - 9500160\\nRN  - 0 (ASCL1 protein, human)\\nRN  - 0 (Basic Helix-Loop-Helix Transcription Factors)\\nSB  - IM\\nMH  - Animals\\nMH  - Basic Helix-Loop-Helix Transcription Factors/genetics/metabolism\\nMH  - COS Cells\\nMH  - *CRISPR-Cas Systems\\nMH  - Chlorocebus aethiops\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - HEK293 Cells\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Light\\nMH  - Luminescent Measurements\\nMH  - Promoter Regions, Genetic\\nMH  - Transcription Initiation, Genetic\\nEDAT- 2015/01/27 06:00\\nMHDA- 2015/12/15 06:00\\nCRDT- 2015/01/27 06:00\\nPHST- 2014/10/14 00:00 [received]\\nPHST- 2014/12/04 00:00 [revised]\\nPHST- 2014/12/10 00:00 [accepted]\\nPHST- 2015/01/27 06:00 [entrez]\\nPHST- 2015/01/27 06:00 [pubmed]\\nPHST- 2015/12/15 06:00 [medline]\\nAID - S1074-5521(14)00459-1 [pii]\\nAID - 10.1016/j.chembiol.2014.12.011 [doi]\\nPST - ppublish\\nSO  - Chem Biol. 2015 Feb 19;22(2):169-74. doi: 10.1016/j.chembiol.2014.12.011. Epub 2015  Jan 22.\\n\\nPMID- 20580935\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20100823\\nLR  - 20100712\\nIS  - 1095-9513 (Electronic)\\nIS  - 1055-7903 (Linking)\\nVI  - 56\\nIP  - 3\\nDP  - 2010 Sep\\nTI  - Comparative network clustering of direct repeats (DRs) and cas genes confirms the  possibility of the horizontal transfer of CRISPR locus among bacteria.\\nPG  - 878-87\\nLID - 10.1016/j.ympev.2010.05.020 [doi]\\nAB  - CRISPRs are a diverse family of DNA repeat sequences that are widely distributed  among archaea and bacteria. The CRISPR locus is usually composed of three key  elements; direct repeats (DRs), spacer sequences and the cas genes. Although recent  studies have suggested that spacers may be of extrachromosomal origin, the  evolutionary origin of the other two elements of the CRISPR locus has remained  unresolved. With the aim to elucidate the evolutionary origin and association of DRs  and cas genes of the CRISPR locus, a comparative analysis of the evolutionary  network clusters of DRs, cas1 and 16S rRNA genes sequences from 100 different  bacteria was conducted. Significant matches of DR and cas1 gene clades imply that  these CRISPR components are evolutionary closely linked and potentially evolving  simultaneously as a whole locus. On the contrary, the prominent discordance between  the CASS (DR and cas1) clades and the 16S rRNA clusters indicates that CRISPR locus  has been transferred horizontally as a complete package. Sequence analysis also  revealed that DR and cas1 genes are coevolving under analogous evolutionary  pressure. This atypical evolutionary pattern also signifies the possibility of  horizontal transfer event of CRISPR locus.\\nCI  - Copyright 2010 Elsevier Inc. All rights reserved.\\nFAU - Chakraborty, Sajib\\nAU  - Chakraborty S\\nAD  - Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka 1000,  Bangladesh. sajib@univdhaka.edu\\nFAU - Snijders, Ambrosius P\\nAU  - Snijders AP\\nFAU - Chakravorty, Rajib\\nAU  - Chakravorty R\\nFAU - Ahmed, Musaddeque\\nAU  - Ahmed M\\nFAU - Tarek, Ashek Md\\nAU  - Tarek AM\\nFAU - Hossain, M Anwar\\nAU  - Hossain MA\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20100523\\nPL  - United States\\nTA  - Mol Phylogenet Evol\\nJT  - Molecular phylogenetics and evolution\\nJID - 9304400\\nRN  - 0 (RNA, Bacterial)\\nRN  - 0 (RNA, Ribosomal, 16S)\\nSB  - IM\\nMH  - Bacteria/classification/*genetics\\nMH  - Cluster Analysis\\nMH  - *Evolution, Molecular\\nMH  - *Gene Transfer, Horizontal\\nMH  - Genes, Bacterial\\nMH  - Phylogeny\\nMH  - RNA, Bacterial/genetics\\nMH  - RNA, Ribosomal, 16S/genetics\\nMH  - *Repetitive Sequences, Nucleic Acid\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, RNA\\nEDAT- 2010/06/29 06:00\\nMHDA- 2010/08/24 06:00\\nCRDT- 2010/06/29 06:00\\nPHST- 2009/09/09 00:00 [received]\\nPHST- 2010/05/16 00:00 [revised]\\nPHST- 2010/05/19 00:00 [accepted]\\nPHST- 2010/06/29 06:00 [entrez]\\nPHST- 2010/06/29 06:00 [pubmed]\\nPHST- 2010/08/24 06:00 [medline]\\nAID - S1055-7903(10)00239-3 [pii]\\nAID - 10.1016/j.ympev.2010.05.020 [doi]\\nPST - ppublish\\nSO  - Mol Phylogenet Evol. 2010 Sep;56(3):878-87. doi: 10.1016/j.ympev.2010.05.020. Epub  2010 May 23.\\n\\nPMID- 33251765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210825\\nLR  - 20210825\\nIS  - 1744-4292 (Electronic)\\nIS  - 1744-4292 (Linking)\\nVI  - 16\\nIP  - 11\\nDP  - 2020 Nov\\nTI  - CIGAR-seq, a CRISPR/Cas-based method for unbiased screening of novel mRNA  modification regulators.\\nPG  - e10025\\nLID - 10.15252/msb.202010025 [doi]\\nLID - e10025\\nAB  - Cellular RNA is decorated with over 170 types of chemical modifications. Many  modifications in mRNA, including m(6) A and m(5) C, have been associated with  critical cellular functions under physiological and/or pathological conditions. To  understand the biological functions of these modifications, it is vital to identify  the regulators that modulate the modification rate. However, a high-throughput  method for unbiased screening of these regulators is so far lacking. Here, we report  such a method combining pooled CRISPR screen and reporters with RNA modification  readout, termed CRISPR integrated gRNA and reporter sequencing (CIGAR-seq). Using  CIGAR-seq, we discovered NSUN6 as a novel mRNA m(5) C methyltransferase. Subsequent  mRNA bisulfite sequencing in HAP1 cells without or with NSUN6 and/or NSUN2 knockout  showed that NSUN6 and NSUN2 worked on non-overlapping subsets of mRNA m(5) C sites  and together contributed to almost all the m(5) C modification in mRNA. Finally,  using m(1) A as an example, we demonstrated that CIGAR-seq can be easily adapted for  identifying regulators of other mRNA modification.\\nCI  - Â© 2020 The Authors. Published under the terms of the CC BY 4.0 license.\\nFAU - Fang, Liang\\nAU  - Fang L\\nAUID- ORCID: 0000-0003-4502-1756\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nAD  - Academy for Advanced Interdisciplinary Studies, Southern University of Science and  Technology, Shenzhen, Guangdong, China.\\nFAU - Wang, Wen\\nAU  - Wang W\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nAD  - Harbin Institute of Technology, Harbin, Heilongjiang, China.\\nFAU - Li, Guipeng\\nAU  - Li G\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nAD  - Academy for Advanced Interdisciplinary Studies, Southern University of Science and  Technology, Shenzhen, Guangdong, China.\\nFAU - Zhang, Li\\nAU  - Zhang L\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Li, Jun\\nAU  - Li J\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Gan, Diwen\\nAU  - Gan D\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Yang, Jiao\\nAU  - Yang J\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Tang, Yisen\\nAU  - Tang Y\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Ding, Zewen\\nAU  - Ding Z\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Zhang, Min\\nAU  - Zhang M\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Zhang, Wenhao\\nAU  - Zhang W\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Deng, Daqi\\nAU  - Deng D\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Song, Zhengyu\\nAU  - Song Z\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Zhu, Qionghua\\nAU  - Zhu Q\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Cui, Huanhuan\\nAU  - Cui H\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nAD  - Academy for Advanced Interdisciplinary Studies, Southern University of Science and  Technology, Shenzhen, Guangdong, China.\\nFAU - Hu, Yuhui\\nAU  - Hu Y\\nAUID- ORCID: 0000-0002-5210-5301\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nFAU - Chen, Wei\\nAU  - Chen W\\nAUID- ORCID: 0000-0003-3263-1627\\nAD  - Department of Biology, Southern University of Science and Technology, Shenzhen,  Guangdong, China.\\nAD  - Academy for Advanced Interdisciplinary Studies, Southern University of Science and  Technology, Shenzhen, Guangdong, China.\\nLA  - eng\\nSI  - GEO/GSE157368\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Mol Syst Biol\\nJT  - Molecular systems biology\\nJID - 101235389\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (RNA, Messenger)\\nRN  - EC 2.1.1.- (Methyltransferases)\\nRN  - EC 2.1.1.- (NSUN6 protein, human)\\nRN  - EC 2.1.1.- (tRNA Methyltransferases)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Cells, Cultured\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - Gene Expression Regulation\\nMH  - Gene Regulatory Networks\\nMH  - Genetic Vectors/genetics\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Methylation\\nMH  - Methyltransferases/*genetics/metabolism\\nMH  - RNA Processing, Post-Transcriptional/*genetics\\nMH  - RNA, Guide/genetics\\nMH  - RNA, Messenger/analysis/*metabolism\\nMH  - Sequence Analysis, RNA/*methods\\nMH  - tRNA Methyltransferases/genetics\\nPMC - PMC7701898\\nOTO - NOTNLM\\nOT  - *CIGAR-seq\\nOT  - *NSUN6\\nOT  - *m5C modification\\nOT  - *mRNA modification\\nOT  - *pooled CRISPR screen\\nCOIS- The authors declare that they have no conflict of interest.\\nEDAT- 2020/12/01 06:00\\nMHDA- 2021/08/26 06:00\\nCRDT- 2020/11/30 05:59\\nPHST- 2020/09/25 00:00 [received]\\nPHST- 2020/11/04 00:00 [revised]\\nPHST- 2020/11/04 00:00 [accepted]\\nPHST- 2020/11/30 05:59 [entrez]\\nPHST- 2020/12/01 06:00 [pubmed]\\nPHST- 2021/08/26 06:00 [medline]\\nAID - MSB202010025 [pii]\\nAID - 10.15252/msb.202010025 [doi]\\nPST - ppublish\\nSO  - Mol Syst Biol. 2020 Nov;16(11):e10025. doi: 10.15252/msb.202010025.\\n\\nPMID- 33478513\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210913\\nLR  - 20210913\\nIS  - 1475-2859 (Electronic)\\nIS  - 1475-2859 (Linking)\\nVI  - 20\\nIP  - 1\\nDP  - 2021 Jan 21\\nTI  - Genome-based engineering of ligninolytic enzymes in fungi.\\nPG  - 20\\nLID - 10.1186/s12934-021-01510-9 [doi]\\nLID - 20\\nAB  - BACKGROUND: Many fungi grow as saprobic organisms and obtain nutrients from a wide  range of dead organic materials. Among saprobes, fungal species that grow on wood or  in polluted environments have evolved prolific mechanisms for the production of  degrading compounds, such as ligninolytic enzymes. These enzymes include arrays of  intense redox-potential oxidoreductase, such as laccase, catalase, and peroxidases.  The ability to produce ligninolytic enzymes makes a variety of fungal species  suitable for application in many industries, including the production of biofuels  and antibiotics, bioremediation, and biomedical application as biosensors. However,  fungal ligninolytic enzymes are produced naturally in small quantities that may not  meet the industrial or market demands. Over the last decade, combined synthetic  biology and computational designs have yielded significant results in enhancing the  synthesis of natural compounds in fungi. In this review, we gave insights into  different protein engineering methods, including rational, semi-rational, and  directed evolution approaches that have been employed to enhance the production of  some important ligninolytic enzymes in fungi. We described the role of metabolic  pathway engineering to optimize the synthesis of chemical compounds of interest in  various fields. We highlighted synthetic biology novel techniques for biosynthetic  gene cluster (BGC) activation in fungo and heterologous reconstruction of BGC in  microbial cells. We also discussed in detail some recombinant ligninolytic enzymes  that have been successfully enhanced and expressed in different heterologous hosts.  Finally, we described recent advance in CRISPR (Clustered Regularly Interspaced  Short Palindromic Repeats)-Cas (CRISPR associated) protein systems as the most  promising biotechnology for large-scale production of ligninolytic enzymes. SHORT  CONCLUSION: Aggregation, expression, and regulation of ligninolytic enzymes in fungi  require very complex procedures with many interfering factors. Synthetic and  computational biology strategies, as explained in this review, are powerful tools  that can be combined to solve these puzzles. These integrated strategies can lead to  the production of enzymes with special abilities, such as wide substrate  specifications, thermo-stability, tolerance to long time storage, and stability in  different substrate conditions, such as pH and nutrients.\\nFAU - Asemoloye, Michael Dare\\nAU  - Asemoloye MD\\nAD  - School of Pharmaceutical Science and Technology, Tianjin University, Nankai  District, 92 Weijin Road, Tianjin, 300072, China.\\nFAU - Marchisio, Mario Andrea\\nAU  - Marchisio MA\\nAD  - School of Pharmaceutical Science and Technology, Tianjin University, Nankai  District, 92 Weijin Road, Tianjin, 300072, China. mamarchisio@yahoo.com.\\nFAU - Gupta, Vijai Kumar\\nAU  - Gupta VK\\nAD  - Biorefining and Advanced Materials Research Center, Scotland\\'s Rural College (SRUC),  Kings Buildings, West Mains Road, Edinburgh, EH9 3JG, UK.\\nFAU - Pecoraro, Lorenzo\\nAU  - Pecoraro L\\nAUID- ORCID: 0000-0002-3234-4698\\nAD  - School of Pharmaceutical Science and Technology, Tianjin University, Nankai  District, 92 Weijin Road, Tianjin, 300072, China. lorenzo.pecoraro@gmail.com.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210121\\nTA  - Microb Cell Fact\\nJT  - Microbial cell factories\\nJID - 101139812\\nRN  - 9005-53-2 (Lignin)\\nRN  - EC 1.10.3.2 (Laccase)\\nRN  - EC 1.11.1.- (Peroxidases)\\nRN  - EC 1.11.1.- (lignin peroxidase)\\nRN  - EC 1.11.1.13 (manganese peroxidase)\\nSB  - IM\\nMH  - Cell Wall/genetics/metabolism\\nMH  - Fungi/enzymology/*genetics\\nMH  - Genetic Engineering/*methods\\nMH  - Genome, Fungal/*genetics\\nMH  - Laccase/*genetics/metabolism\\nMH  - Lignin/metabolism\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Peroxidases/*genetics/metabolism\\nMH  - Synthetic Biology/methods\\nPMC - PMC7819241\\nOTO - NOTNLM\\nOT  - Biosynthetic pathways\\nOT  - CRISPR-cas\\nOT  - Fungal secretome\\nOT  - Fungi\\nOT  - Gene editing\\nOT  - Heterologous protein expression\\nOT  - Ligninolytic enzymes\\nOT  - Synthetic promoters\\nOT  - Transcription activation\\nCOIS- Authors declare no competing interest on this manuscript.\\nEDAT- 2021/01/23 06:00\\nMHDA- 2021/09/14 06:00\\nCRDT- 2021/01/22 05:45\\nPHST- 2020/11/17 00:00 [received]\\nPHST- 2021/01/07 00:00 [accepted]\\nPHST- 2021/01/22 05:45 [entrez]\\nPHST- 2021/01/23 06:00 [pubmed]\\nPHST- 2021/09/14 06:00 [medline]\\nAID - 10.1186/s12934-021-01510-9 [pii]\\nAID - 1510 [pii]\\nAID - 10.1186/s12934-021-01510-9 [doi]\\nPST - epublish\\nSO  - Microb Cell Fact. 2021 Jan 21;20(1):20. doi: 10.1186/s12934-021-01510-9.\\n\\nPMID- 29599458\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190829\\nLR  - 20190829\\nIS  - 1740-1534 (Electronic)\\nIS  - 1740-1526 (Linking)\\nVI  - 16\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - CRISPR-based genomic tools for the manipulation of genetically intractable  microorganisms.\\nPG  - 333-339\\nLID - 10.1038/s41579-018-0002-7 [doi]\\nAB  - Genetic manipulation of microorganisms has been crucial in understanding their  biology, yet for many microbial species, robust tools for comprehensive genetic  analysis were lacking until the advent of CRISPR-Cas-based gene editing techniques.  In this Progress article, we discuss advances in CRISPR-based techniques for the  genetic analysis of genetically intractable microorganisms, with an emphasis on  mycobacteria, fungi and parasites. We discuss how CRISPR-based analyses in these  organisms have enabled the discovery of novel gene functions, the investigation of  genetic interaction networks and the identification of virulence factors.\\nFAU - Shapiro, Rebecca S\\nAU  - Shapiro RS\\nAD  - Department of Biological Engineering, Institute for Medical Engineering and Science,  Synthetic Biology Center, MIT, Cambridge, MA, USA.\\nAD  - Broad Institute of MIT and Harvard, Cambridge, MA, USA.\\nAD  - Department of Molecular and Cellular Biology, University of Guelph, Guelph, Ontario,  Canada.\\nFAU - Chavez, Alejandro\\nAU  - Chavez A\\nAD  - Department of Pathology and Cell Biology, Columbia University College of Physicians  and Surgeons, New York, NY, USA.\\nFAU - Collins, James J\\nAU  - Collins JJ\\nAD  - Department of Biological Engineering, Institute for Medical Engineering and Science,  Synthetic Biology Center, MIT, Cambridge, MA, USA. jimjc@mit.edu.\\nAD  - Broad Institute of MIT and Harvard, Cambridge, MA, USA. jimjc@mit.edu.\\nAD  - Wyss Institute for Biologically Inspired Engineering, Harvard University, Boston,  MA, USA. jimjc@mit.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPT  - Review\\nPL  - England\\nTA  - Nat Rev Microbiol\\nJT  - Nature reviews. Microbiology\\nJID - 101190261\\nRN  - 0 (DNA, Bacterial)\\nSB  - IM\\nMH  - Bacteria/*genetics\\nMH  - *CRISPR-Cas Systems\\nMH  - DNA, Bacterial/genetics\\nMH  - Genetic Engineering\\nMH  - *Genome, Bacterial\\nMH  - Genomics/*methods\\nEDAT- 2018/03/31 06:00\\nMHDA- 2019/08/30 06:00\\nCRDT- 2018/03/31 06:00\\nPHST- 2018/03/31 06:00 [pubmed]\\nPHST- 2019/08/30 06:00 [medline]\\nPHST- 2018/03/31 06:00 [entrez]\\nAID - 10.1038/s41579-018-0002-7 [pii]\\nAID - 10.1038/s41579-018-0002-7 [doi]\\nPST - ppublish\\nSO  - Nat Rev Microbiol. 2018 Jun;16(6):333-339. doi: 10.1038/s41579-018-0002-7.\\n\\nPMID- 31926765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200317\\nLR  - 20200317\\nIS  - 1878-4372 (Electronic)\\nIS  - 1360-1385 (Linking)\\nVI  - 25\\nIP  - 3\\nDP  - 2020 Mar\\nTI  - Revisiting the Role of Master Regulators in Tomato Ripening.\\nPG  - 291-301\\nLID - S1360-1385(19)30305-X [pii]\\nLID - 10.1016/j.tplants.2019.11.005 [doi]\\nAB  - The study of transcriptional regulation of tomato ripening has been led by  spontaneous mutations in transcription factor (TF) genes that completely inhibit  normal ripening, suggesting that they are \\'master regulators\\'. Studies using  CRISPR/Cas9 mutagenesis to produce knockouts of the underlying genes indicate a  different picture, suggesting that the regulation is more robust than previously  thought. This requires us to revisit our model of the regulation of ripening and  replace it with one involving a network of partially redundant components. At the  same time, the fast rise of CRISPR/Cas mutagenesis, resulting in unexpectedly weak  phenotypes, compared with knockdown technology, suggests that compensatory  mechanisms may obscure protein functions. This emphasises the need for assessment of  these mechanisms in plants and for the careful design of mutagenesis experiments.\\nCI  - Crown Copyright Â© 2019. Published by Elsevier Ltd. All rights reserved.\\nFAU - Wang, Rufang\\nAU  - Wang R\\nAD  - Laboratory of Molecular Biology, Wageningen University & Research, Wageningen, The  Netherlands.\\nFAU - Angenent, Gerco C\\nAU  - Angenent GC\\nAD  - Laboratory of Molecular Biology, Wageningen University & Research, Wageningen, The  Netherlands; Bioscience, Wageningen University & Research, Wageningen, The  Netherlands.\\nFAU - Seymour, Graham\\nAU  - Seymour G\\nAD  - School of Biosciences, University of Nottingham, Sutton Bonington, Loughborough, UK.\\nFAU - de Maagd, Ruud A\\nAU  - de Maagd RA\\nAD  - Bioscience, Wageningen University & Research, Wageningen, The Netherlands.  Electronic address: ruud.demaagd@wur.nl.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20200108\\nPL  - England\\nTA  - Trends Plant Sci\\nJT  - Trends in plant science\\nJID - 9890299\\nRN  - 0 (Plant Proteins)\\nSB  - IM\\nMH  - Fruit\\nMH  - Gene Expression Regulation, Plant\\nMH  - *Lycopersicon esculentum\\nMH  - Phenotype\\nMH  - Plant Proteins\\nOTO - NOTNLM\\nOT  - *CRISPR- mutagenesis\\nOT  - *gain-of-function\\nOT  - *mutants\\nOT  - *ripening\\nOT  - *tomato\\nOT  - *transcription factors\\nEDAT- 2020/01/14 06:00\\nMHDA- 2020/03/18 06:00\\nCRDT- 2020/01/14 06:00\\nPHST- 2019/08/13 00:00 [received]\\nPHST- 2019/11/14 00:00 [revised]\\nPHST- 2019/11/18 00:00 [accepted]\\nPHST- 2020/01/14 06:00 [pubmed]\\nPHST- 2020/03/18 06:00 [medline]\\nPHST- 2020/01/14 06:00 [entrez]\\nAID - S1360-1385(19)30305-X [pii]\\nAID - 10.1016/j.tplants.2019.11.005 [doi]\\nPST - ppublish\\nSO  - Trends Plant Sci. 2020 Mar;25(3):291-301. doi: 10.1016/j.tplants.2019.11.005. Epub  2020 Jan 8.\\n\\nPMID- 30520167\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190415\\nLR  - 20210109\\nIS  - 1365-2184 (Electronic)\\nIS  - 0960-7722 (Print)\\nIS  - 0960-7722 (Linking)\\nVI  - 52\\nIP  - 2\\nDP  - 2019 Mar\\nTI  - A revolutionary tool: CRISPR technology plays an important role in construction of  intelligentized gene circuits.\\nPG  - e12552\\nLID - 10.1111/cpr.12552 [doi]\\nLID - e12552\\nAB  - With the development of synthetic biology, synthetic gene circuits have shown great  applied potential in medicine, biology, and as commodity chemicals. An ultimate  challenge in the construction of gene circuits is the lack of effective,  programmable, secure and sequence-specific gene editing tools. The clustered  regularly interspaced short palindromic repeat (CRISPR) system, a CRISPR-associated  RNA-guided endonuclease Cas9 (CRISPR-associated protein 9)-targeted genome editing  tool, has recently been applied in engineering gene circuits for its unique  properties-operability, high efficiency and programmability. The traditional  single-targeted therapy cannot effectively distinguish tumour cells from normal  cells, and gene therapy for single targets has poor anti-tumour effects, which  severely limits the application of gene therapy. Currently, the design of gene  circuits using tumour-specific targets based on CRISPR/Cas systems provides a new  way for precision cancer therapy. Hence, the application of intelligentized gene  circuits based on CRISPR technology effectively guarantees the safety, efficiency  and specificity of cancer therapy. Here, we assessed the use of synthetic gene  circuits and if the CRISPR system could be used, especially artificial  switch-inducible Cas9, to more effectively target and treat tumour cells. Moreover,  we also discussed recent advances, prospectives and underlying challenges in  CRISPR-based gene circuit development.\\nCI  - Â© 2018 The Authors. Cell Proliferation Published by John Wiley & Sons Ltd.\\nFAU - Zhou, Qun\\nAU  - Zhou Q\\nAUID- ORCID: 0000-0003-2524-1452\\nAD  - Department of Urology, Shenzhen Second People\\'s Hospital, Clinical Medicine College  of Anhui Medical University, Shenzhen, China.\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Zhan, Hengji\\nAU  - Zhan H\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Liao, Xinhui\\nAU  - Liao X\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Fang, Lan\\nAU  - Fang L\\nAD  - Department of Urology, Shenzhen Second People\\'s Hospital, Clinical Medicine College  of Anhui Medical University, Shenzhen, China.\\nFAU - Liu, Yuhan\\nAU  - Liu Y\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Xie, Haibiao\\nAU  - Xie H\\nAUID- ORCID: 0000-0001-6729-8479\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Yang, Kang\\nAU  - Yang K\\nAUID- ORCID: 0000-0002-1544-0477\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Gao, Qunjun\\nAU  - Gao Q\\nAUID- ORCID: 0000-0002-6524-1230\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Ding, Mengting\\nAU  - Ding M\\nAUID- ORCID: 0000-0001-9156-0457\\nAD  - Department of Urology, Shenzhen Second People\\'s Hospital, Clinical Medicine College  of Anhui Medical University, Shenzhen, China.\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Cai, Zhiming\\nAU  - Cai Z\\nAD  - Department of Urology, Shenzhen Second People\\'s Hospital, Clinical Medicine College  of Anhui Medical University, Shenzhen, China.\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Huang, Weiren\\nAU  - Huang W\\nAD  - Department of Urology, Shenzhen Second People\\'s Hospital, Clinical Medicine College  of Anhui Medical University, Shenzhen, China.\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nFAU - Liu, Yuchen\\nAU  - Liu Y\\nAD  - Department of Urology, the First Affiliated Hospital of Shenzhen University,  Shenzhen, China.\\nLA  - eng\\nGR  - 81772737/National Natural Science Foundation of China/\\nGR  - 2017A020215120/Guangdong Provincial Department of Science and Technology/\\nGR  - 2016031638/High Level University\\'s Medical Discipline Construction project/\\nGR  - JCYJ20170413161749433/Shenzhen Municipal Government of China/\\nGR  - JSGG20160301161836370/Shenzhen Municipal Government of China/\\nGR  - 2014CB745201/National Key Basic Research Program of China (973 Program)/\\nGR  - SZSM201412018/Sanming Project of Shenzhen Health and Family Planning Commission/\\nGR  - SZSM201512037/Sanming Project of Shenzhen Health and Family Planning Commission/\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20181205\\nTA  - Cell Prolif\\nJT  - Cell proliferation\\nJID - 9105195\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Gene Editing/*methods\\nMH  - *Gene Regulatory Networks\\nMH  - *Genes, Synthetic\\nMH  - Genetic Therapy/methods\\nMH  - Humans\\nMH  - Neoplasms/genetics/therapy\\nPMC - PMC6496519\\nOTO - NOTNLM\\nOT  - CRISPR technology\\nOT  - artificial\\nOT  - gene circuits\\nOT  - gene editing\\nOT  - intelligentized\\nCOIS- All authors declare that there is no conflict of interest.\\nEDAT- 2018/12/07 06:00\\nMHDA- 2019/04/16 06:00\\nCRDT- 2018/12/07 06:00\\nPHST- 2018/06/26 00:00 [received]\\nPHST- 2018/08/28 00:00 [revised]\\nPHST- 2018/08/30 00:00 [accepted]\\nPHST- 2018/12/07 06:00 [pubmed]\\nPHST- 2019/04/16 06:00 [medline]\\nPHST- 2018/12/07 06:00 [entrez]\\nAID - CPR12552 [pii]\\nAID - 10.1111/cpr.12552 [doi]\\nPST - ppublish\\nSO  - Cell Prolif. 2019 Mar;52(2):e12552. doi: 10.1111/cpr.12552. Epub 2018 Dec 5.\\n\\nPMID- 28448738\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180227\\nLR  - 20181113\\nIS  - 1949-1042 (Electronic)\\nIS  - 1949-1034 (Print)\\nIS  - 1949-1034 (Linking)\\nVI  - 8\\nIP  - 3\\nDP  - 2017 May 4\\nTI  - Site-specific recruitment of epigenetic factors with a modular CRISPR/Cas system.\\nPG  - 279-286\\nLID - 10.1080/19491034.2017.1292194 [doi]\\nAB  - Dissecting the complex network of epigenetic modifications requires tools that  combine precise recognition of DNA sequences with the capability to modify  epigenetic marks. The CRISPR/Cas system has been proven to be a valuable addition to  existing methodologies that fulfill these tasks. So far, sequence-specific editing  of epigenetic modifications such as DNA methylation and histone posttranslational  modifications relied on direct fusions of enzymatically inactivated Cas9 (dCas9)  with epigenetic effectors. Here, we report a novel, modular system that facilitates  the recruitment of any GFP-tagged protein to desired genomic loci. By fusing dCas9  to a GFP-binding nanobody (GBP) we demonstrate that prevalent epigenetic  modifications at mouse major satellite repeats can be erased or set de novo by  recruiting GFP-coupled catalytic domains of TET1 and DNMT3A, respectively.  Furthermore, we construct an inducible expression system that enables a temporally  controlled expression of both GBP-dCas9 and the effector protein. Thus, our approach  further expands the CRISPR/Cas toolbox for site-specific manipulation of epigenetic  modifications with a modular and easy-to-use system.\\nFAU - Anton, Tobias\\nAU  - Anton T\\nAD  - a Department of Biology II and Center for Integrated Protein Science Munich (CIPSM)  , LMU Munich , Martinsried , Germany.\\nFAU - Bultmann, Sebastian\\nAU  - Bultmann S\\nAD  - a Department of Biology II and Center for Integrated Protein Science Munich (CIPSM)  , LMU Munich , Martinsried , Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170223\\nTA  - Nucleus\\nJT  - Nucleus (Austin, Tex.)\\nJID - 101518322\\nRN  - 147336-22-9 (Green Fluorescent Proteins)\\nRN  - EC 2.1.1.- (DNA Modification Methylases)\\nRN  - N12000U13O (Doxycycline)\\nSB  - IM\\nMH  - Animals\\nMH  - Base Sequence\\nMH  - Binding Sites\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Cell Line\\nMH  - DNA Modification Methylases/metabolism\\nMH  - Doxycycline/pharmacology\\nMH  - *Epigenesis, Genetic\\nMH  - Green Fluorescent Proteins/genetics\\nMH  - Mice\\nMH  - Promoter Regions, Genetic/genetics\\nMH  - Substrate Specificity\\nPMC - PMC5499900\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas\\nOT  - *DNA de novo methylation\\nOT  - *DNA demethylation\\nOT  - *GBP\\nOT  - *GFP\\nOT  - *Nanobody\\nEDAT- 2017/04/28 06:00\\nMHDA- 2018/02/28 06:00\\nCRDT- 2017/04/28 06:00\\nPHST- 2017/04/28 06:00 [pubmed]\\nPHST- 2018/02/28 06:00 [medline]\\nPHST- 2017/04/28 06:00 [entrez]\\nAID - 1292194 [pii]\\nAID - 10.1080/19491034.2017.1292194 [doi]\\nPST - ppublish\\nSO  - Nucleus. 2017 May 4;8(3):279-286. doi: 10.1080/19491034.2017.1292194. Epub 2017 Feb  23.\\n\\nPMID- 25524111\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150831\\nLR  - 20141219\\nIS  - 2161-5063 (Electronic)\\nIS  - 2161-5063 (Linking)\\nVI  - 3\\nIP  - 12\\nDP  - 2014 Dec 19\\nTI  - Multistage regulator based on tandem promoters and CRISPR/Cas.\\nPG  - 1007-10\\nLID - 10.1021/sb500029n [doi]\\nAB  - Accurately controlling expression of target genes between several designed levels is  essential for low-noise gene network and dynamic range of gene expression. However,  such manipulations have been hard to achieve due to technical limitations. Based on  tandem promoters and CRISPR (clustered regularly interspaced short palindromic  repeats)/Cas (CRISPR-associated) system, we constructed a multistage regulator that  could stably regulate the expression of the reporter gene on three levels, with more  than 2-fold difference between each of them. Our findings provide novel insights  into constructing a more powerful gene regulation system.\\nFAU - Jia, Hangxing\\nAU  - Jia H\\nAD  - College of Life Sciences, Wuhan University , Wuhan, Hubei 430072, China.\\nFAU - Liang, Tong\\nAU  - Liang T\\nFAU - Wang, Zhaoning\\nAU  - Wang Z\\nFAU - He, Zhaoren\\nAU  - He Z\\nFAU - Liu, Yang\\nAU  - Liu Y\\nFAU - Yang, Lei\\nAU  - Yang L\\nFAU - Zeng, Yan\\nAU  - Zeng Y\\nFAU - Liu, Shaopeng\\nAU  - Liu S\\nFAU - Tang, Linyi\\nAU  - Tang L\\nFAU - Wang, Jianbo\\nAU  - Wang J\\nFAU - Chen, Yu\\nAU  - Chen Y\\nFAU - Xie, Zhixiong\\nAU  - Xie Z\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - United States\\nTA  - ACS Synth Biol\\nJT  - ACS synthetic biology\\nJID - 101575075\\nSB  - IM\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Escherichia coli\\nMH  - Gene Expression Regulation/*genetics\\nMH  - Genetic Engineering\\nMH  - Promoter Regions, Genetic/*genetics\\nMH  - Synthetic Biology/*methods\\nEDAT- 2014/12/20 06:00\\nMHDA- 2015/09/01 06:00\\nCRDT- 2014/12/20 06:00\\nPHST- 2014/12/20 06:00 [entrez]\\nPHST- 2014/12/20 06:00 [pubmed]\\nPHST- 2015/09/01 06:00 [medline]\\nAID - 10.1021/sb500029n [doi]\\nPST - ppublish\\nSO  - ACS Synth Biol. 2014 Dec 19;3(12):1007-10. doi: 10.1021/sb500029n.\\n\\nPMID- 32651924\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210315\\nLR  - 20210315\\nIS  - 1940-6029 (Electronic)\\nIS  - 1064-3745 (Linking)\\nVI  - 2173\\nDP  - 2020\\nTI  - Optogenetics and CRISPR: A New Relationship Built to Last.\\nPG  - 261-281\\nLID - 10.1007/978-1-0716-0755-8_18 [doi]\\nAB  - Since the breakthrough discoveries that CRISPR-Cas9 nucleases can be easily  programmed and employed to induce targeted double-strand breaks in mammalian cells,  the gene editing field has grown exponentially. Today, CRISPR technologies based on  engineered class II CRISPR effectors facilitate targeted modification of genes and  RNA transcripts. Moreover, catalytically impaired CRISPR-Cas variants can be  employed as programmable DNA binding domains and used to recruit effector proteins,  such as transcriptional regulators, epigenetic modifiers or base-modifying enzymes,  to selected genomic loci. The juxtaposition of CRISPR and optogenetics enables  spatiotemporally confined and highly dynamic genome perturbations in living cells  and animals and holds unprecedented potential for biology and biomedicine.Here, we  provide an overview of the state-of-the-art methods for light-control of CRISPR  effectors. We will detail the plethora of exciting applications enabled by these  systems, including spatially confined genome editing, timed activation of endogenous  genes, as well as remote control of chromatin-chromatin interactions. Finally, we  will discuss limitations of current optogenetic CRISPR tools and point out routes  for future innovation in this emerging field.\\nFAU - Mathony, Jan\\nAU  - Mathony J\\nAD  - Synthetic Biology Group, BioQuant Center, University of Heidelberg, Heidelberg,  Germany.\\nAD  - Digital Health Center, Berlin Institute of Health (BIH) and CharitÃ©, Berlin,  Germany.\\nFAU - Hoffmann, Mareike D\\nAU  - Hoffmann MD\\nAD  - Synthetic Biology Group, BioQuant Center, University of Heidelberg, Heidelberg,  Germany.\\nAD  - Division of Chromatin Networks, German Cancer Research Center (DKFZ), Heidelberg,  Germany.\\nFAU - Niopek, Dominik\\nAU  - Niopek D\\nAD  - Synthetic Biology Group, BioQuant Center, University of Heidelberg, Heidelberg,  Germany. dominik.niopek@bioquant.uni-heidelberg.de.\\nAD  - Health Data Science Unit, Heidelberg University Hospital and Medical Faculty of  Heidelberg University, Heidelberg, Germany.  dominik.niopek@bioquant.uni-heidelberg.de.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - United States\\nTA  - Methods Mol Biol\\nJT  - Methods in molecular biology (Clifton, N.J.)\\nJID - 9214969\\nSB  - IM\\nMH  - Animals\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Gene Editing/methods\\nMH  - Humans\\nMH  - Optogenetics/*methods\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *Genome editing\\nOT  - *Optogenetics\\nOT  - *Photoreceptor\\nOT  - *anti-CRISPR\\nEDAT- 2020/07/12 06:00\\nMHDA- 2021/03/16 06:00\\nCRDT- 2020/07/12 06:00\\nPHST- 2020/07/12 06:00 [entrez]\\nPHST- 2020/07/12 06:00 [pubmed]\\nPHST- 2021/03/16 06:00 [medline]\\nAID - 10.1007/978-1-0716-0755-8_18 [doi]\\nPST - ppublish\\nSO  - Methods Mol Biol. 2020;2173:261-281. doi: 10.1007/978-1-0716-0755-8_18.\\n\\nPMID- 34521192\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211004\\nLR  - 20211004\\nIS  - 1520-6882 (Electronic)\\nIS  - 0003-2700 (Linking)\\nVI  - 93\\nIP  - 38\\nDP  - 2021 Sep 28\\nTI  - Versatile CRISPR-Cas12a-Based Biosensing Platform Modulated with Programmable  Entropy-Driven Dynamic DNA Networks.\\nPG  - 12881-12888\\nLID - 10.1021/acs.analchem.1c01597 [doi]\\nAB  - In addition to their roles as revolutionary genome engineering tools, CRISPR-Cas  systems are also highly promising candidates in the construction of biosensing  systems and diagnostic devices, which have attracted significant attention recently.  However, the CRISPR-Cas system cannot be directly applied in the sensing of  non-nucleic acid targets, and the needs of synthesizing and storing different  vulnerable guide RNA for different targets also increase the application and storage  costs of relevant biosensing systems, and therefore restrict their widespread  applications. To tackle these barriers, in this work, a versatile  CRISPR-Cas12a-based biosensing platform was developed through the introduction of an  enzyme-free and robust DNA reaction network, the entropy-driven dynamic DNA network.  By programming the sequences of the system, the entropy-driven catalysis-based  dynamic DNA network can respond to different types of targets, such as nucleic acids  or proteins, and then activate the CRISPR-Cas12a to generate amplified signals. As a  proof of concept, both nucleic acid targets (a DNA target with random sequence, T,  and an RNA target, microRNA-21 (miR-21)) and a non-nucleic acid target (a protein  target, thrombin) were chosen as model analytes to address the feasibility of the  designed sensing platform, with detection limits at the pM level for the nucleic  acid analytes (7.4 pM for the DNA target T and 25.5 pM for miR-21) and 0.4 nM for  thrombin. In addition, the detection of miR-21 or thrombin in human serum samples  further demonstrated the applicability of the proposed biosensing platform in real  sample analysis.\\nFAU - Wang, Chunyan\\nAU  - Wang C\\nAD  - College of Chemistry, Research Center for Analytical Sciences, and Tianjin Key  Laboratory of Molecular Recognition and Biosensing, Nankai University, Tianjin  300071, P. R. China.\\nFAU - Han, Cuiyan\\nAU  - Han C\\nAD  - College of Chemistry, Research Center for Analytical Sciences, and Tianjin Key  Laboratory of Molecular Recognition and Biosensing, Nankai University, Tianjin  300071, P. R. China.\\nFAU - Du, Xiaoxue\\nAU  - Du X\\nAD  - College of Chemistry, Research Center for Analytical Sciences, and Tianjin Key  Laboratory of Molecular Recognition and Biosensing, Nankai University, Tianjin  300071, P. R. China.\\nFAU - Guo, Weiwei\\nAU  - Guo W\\nAUID- ORCID: 0000-0002-1472-6633\\nAD  - College of Chemistry, Research Center for Analytical Sciences, and Tianjin Key  Laboratory of Molecular Recognition and Biosensing, Nankai University, Tianjin  300071, P. R. China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210914\\nPL  - United States\\nTA  - Anal Chem\\nJT  - Analytical chemistry\\nJID - 0370536\\nRN  - 0 (Nucleic Acids)\\nRN  - 9007-49-2 (DNA)\\nSB  - IM\\nMH  - *Biosensing Techniques\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - DNA/genetics\\nMH  - Entropy\\nMH  - Humans\\nMH  - *Nucleic Acids\\nEDAT- 2021/09/16 06:00\\nMHDA- 2021/10/05 06:00\\nCRDT- 2021/09/15 05:35\\nPHST- 2021/09/16 06:00 [pubmed]\\nPHST- 2021/10/05 06:00 [medline]\\nPHST- 2021/09/15 05:35 [entrez]\\nAID - 10.1021/acs.analchem.1c01597 [doi]\\nPST - ppublish\\nSO  - Anal Chem. 2021 Sep 28;93(38):12881-12888. doi: 10.1021/acs.analchem.1c01597. Epub  2021 Sep 14.\\n\\nPMID- 26707540\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20161024\\nLR  - 20200930\\nIS  - 1096-7184 (Electronic)\\nIS  - 1096-7176 (Linking)\\nVI  - 34\\nDP  - 2016 Mar\\nTI  - CRISPR/Cas9 advances engineering of microbial cell factories.\\nPG  - 44-59\\nLID - S1096-7176(15)00159-7 [pii]\\nLID - 10.1016/j.ymben.2015.12.003 [doi]\\nAB  - One of the key drivers for successful metabolic engineering in microbes is the  efficacy by which genomes can be edited. As such there are many methods to choose  from when aiming to modify genomes, especially those of model organisms like yeast  and bacteria. In recent years, clustered regularly interspaced palindromic repeats  (CRISPR) and its associated proteins (Cas) have become the method of choice for  precision genome engineering in many organisms due to their orthogonality,  versatility and efficacy. Here we review the strategies adopted for implementation  of RNA-guided CRISPR/Cas9 genome editing with special emphasis on their application  for metabolic engineering of yeast and bacteria. Also, examples of how  nuclease-deficient Cas9 has been applied for RNA-guided transcriptional regulation  of target genes will be reviewed, as well as tools available for computer-aided  design of guide-RNAs will be highlighted. Finally, this review will provide a  perspective on the immediate challenges and opportunities foreseen by the use of  CRISPR/Cas9 genome engineering and regulation in the context of metabolic  engineering.\\nCI  - Copyright Â© 2015 International Metabolic Engineering Society. All rights reserved.\\nFAU - JakoÄiÅ«nas, Tadas\\nAU  - JakoÄiÅ«nas T\\nAD  - The Novo Nordisk Foundation Center for Biosustainability, Technical University of  Denmark, Denmark.\\nFAU - Jensen, Michael K\\nAU  - Jensen MK\\nAD  - The Novo Nordisk Foundation Center for Biosustainability, Technical University of  Denmark, Denmark. Electronic address: mije@biosustain.dtu.dk.\\nFAU - Keasling, Jay D\\nAU  - Keasling JD\\nAD  - The Novo Nordisk Foundation Center for Biosustainability, Technical University of  Denmark, Denmark; Joint BioEnergy Institute, Emeryville, CA, USA; Biological Systems  & Engineering Division, Lawrence Berkeley National Laboratory, Berkeley, CA, USA;  Department of Chemical and Biomolecular Engineering & Department of Bioengineering  University of California, Berkeley, CA, USA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20151217\\nPL  - Belgium\\nTA  - Metab Eng\\nJT  - Metabolic engineering\\nJID - 9815657\\nRN  - 0 (CRISPR-Associated Proteins)\\nSB  - IM\\nMH  - CRISPR-Associated Proteins/*genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Genetic Enhancement/methods\\nMH  - Genome, Microbial/*genetics\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/*genetics\\nOTO - NOTNLM\\nOT  - Bacteria\\nOT  - CRISPR/Cas9\\nOT  - Genome editing\\nOT  - Metabolic engineering\\nOT  - Recombineering\\nOT  - Yeast\\nEDAT- 2015/12/29 06:00\\nMHDA- 2016/10/25 06:00\\nCRDT- 2015/12/29 06:00\\nPHST- 2015/09/23 00:00 [received]\\nPHST- 2015/11/29 00:00 [revised]\\nPHST- 2015/12/10 00:00 [accepted]\\nPHST- 2015/12/29 06:00 [entrez]\\nPHST- 2015/12/29 06:00 [pubmed]\\nPHST- 2016/10/25 06:00 [medline]\\nAID - S1096-7176(15)00159-7 [pii]\\nAID - 10.1016/j.ymben.2015.12.003 [doi]\\nPST - ppublish\\nSO  - Metab Eng. 2016 Mar;34:44-59. doi: 10.1016/j.ymben.2015.12.003. Epub 2015 Dec 17.\\n\\nPMID- 26322736\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160602\\nLR  - 20181202\\nIS  - 1879-0429 (Electronic)\\nIS  - 0958-1669 (Linking)\\nVI  - 36\\nDP  - 2015 Dec\\nTI  - Programmable genetic circuits for pathway engineering.\\nPG  - 115-21\\nLID - S0958-1669(15)00106-8 [pii]\\nLID - 10.1016/j.copbio.2015.08.007 [doi]\\nAB  - Synthetic biology has the potential to provide decisive advances in genetic control  of metabolic pathways. However, there are several challenges that synthetic  biologists must overcome before this vision becomes a reality. First, a library of  diverse and well-characterized sensors, such as metabolite-sensing or  condition-sensing promoters, must be constructed. Second, robust programmable  circuits that link input conditions with a specific gene regulation response must be  developed. Finally, multi-gene targeting strategies must be integrated with  metabolically relevant sensors and complex, robust logic. Achievements in each of  these areas, which employ the CRISPR/Cas system, in silico modeling, and dynamic  sensor-regulators, among other tools, provide a strong basis for future research.  Overall, the future for synthetic biology approaches in metabolic engineering holds  immense promise.\\nCI  - Copyright Â© 2015 Elsevier Ltd. All rights reserved.\\nFAU - Hoynes-O\\'Connor, Allison\\nAU  - Hoynes-O\\'Connor A\\nAD  - Energy, Environmental and Chemical Engineering, Washington University in St. Louis,  St. Louis, MO 63130, USA.\\nFAU - Moon, Tae Seok\\nAU  - Moon TS\\nAD  - Energy, Environmental and Chemical Engineering, Washington University in St. Louis,  St. Louis, MO 63130, USA. Electronic address: tsmoon@wustl.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPT  - Review\\nDEP - 20150829\\nPL  - England\\nTA  - Curr Opin Biotechnol\\nJT  - Current opinion in biotechnology\\nJID - 9100492\\nSB  - IM\\nMH  - Gene Expression Regulation\\nMH  - *Gene Regulatory Networks\\nMH  - Gene Targeting\\nMH  - Metabolic Engineering/*methods\\nMH  - *Metabolic Networks and Pathways/genetics\\nMH  - Synthetic Biology/methods\\nEDAT- 2015/09/01 06:00\\nMHDA- 2016/06/03 06:00\\nCRDT- 2015/09/01 06:00\\nPHST- 2015/05/23 00:00 [received]\\nPHST- 2015/08/04 00:00 [revised]\\nPHST- 2015/08/09 00:00 [accepted]\\nPHST- 2015/09/01 06:00 [entrez]\\nPHST- 2015/09/01 06:00 [pubmed]\\nPHST- 2016/06/03 06:00 [medline]\\nAID - S0958-1669(15)00106-8 [pii]\\nAID - 10.1016/j.copbio.2015.08.007 [doi]\\nPST - ppublish\\nSO  - Curr Opin Biotechnol. 2015 Dec;36:115-21. doi: 10.1016/j.copbio.2015.08.007. Epub  2015 Aug 29.\\n\\nPMID- 34446084\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220105\\nLR  - 20220105\\nIS  - 1756-9966 (Electronic)\\nIS  - 0392-9078 (Print)\\nIS  - 0392-9078 (Linking)\\nVI  - 40\\nIP  - 1\\nDP  - 2021 Aug 26\\nTI  - CRISPR/Cas9 revitalizes adoptive T-cell therapy for cancer immunotherapy.\\nPG  - 269\\nLID - 10.1186/s13046-021-02076-5 [doi]\\nLID - 269\\nAB  - Cancer immunotherapy has gained attention as the supreme therapeutic modality for  the treatment of various malignancies. Adoptive T-cell therapy (ACT) is one of the  most distinctive modalities of this therapeutic approach, which seeks to harness the  potential of combating cancer cells by using autologous or allogenic tumor-specific  T-cells. However, a plethora of circumstances must be optimized to produce  functional, durable, and efficient T-cells. Recently, the potential of ACT has been  further realized by the introduction of novel gene-editing platforms such as the  CRISPR/Cas9 system; this technique has been utilized to create T-cells furnished  with recombinant T-cell receptor (TCR) or chimeric antigen receptor (CAR) that have  precise tumor antigen recognition, minimal side effects and treatment-related  toxicities, robust proliferation and cytotoxicity, and nominal exhaustion. Here, we  aim to review and categorize the recent breakthroughs of genetically modified  TCR/CAR T-cells through CRISPR/Cas9 technology and address the pearls and pitfalls  of each method. In addition, we investigate the latest ongoing clinical trials that  are applying CRISPR-associated TCR/CAR T-cells for the treatment of cancers.\\nCI  - Â© 2021. The Author(s).\\nFAU - Ghaffari, Sasan\\nAU  - Ghaffari S\\nAD  - Cancer Immunology Project (CIP), Universal Scientific Education and Research Network  (USERN), Tehran, Iran.\\nAD  - Department of Hematology, School of Allied Medical Sciences, Tehran University of  Medical Sciences, Tehran, Iran.\\nFAU - Khalili, Nastaran\\nAU  - Khalili N\\nAD  - Cancer Immunology Project (CIP), Universal Scientific Education and Research Network  (USERN), Tehran, Iran.\\nFAU - Rezaei, Nima\\nAU  - Rezaei N\\nAD  - Research Center for Immunodeficiencies, Children\\'s Medical Center, Tehran University  of Medical Sciences, Dr. Qarib St, Keshavarz Blvd, Tehran, 14194, Iran.  rezaei_nima@yahoo.com.\\nAD  - Department of Immunology, School of Medicine, Tehran University of Medical Sciences,  Tehran, Iran. rezaei_nima@yahoo.com.\\nAD  - Network of Immunity in Infection, Malignancy and Autoimmunity (NIIMA), Universal  Scientific Education and Research Network (USERN), Stockholm, Sweden.  rezaei_nima@yahoo.com.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210826\\nTA  - J Exp Clin Cancer Res\\nJT  - Journal of experimental & clinical cancer research : CR\\nJID - 8308647\\nRN  - 0 (Receptors, Antigen, T-Cell)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - Gene Editing/methods\\nMH  - Humans\\nMH  - Immunotherapy, Adoptive/*methods\\nMH  - Neoplasms/immunology/*therapy\\nMH  - Receptors, Antigen, T-Cell/immunology\\nMH  - T-Lymphocytes/*immunology\\nPMC - PMC8390258\\nOTO - NOTNLM\\nOT  - Adoptive immunotherapy\\nOT  - CRISPR/Cas-9\\nOT  - Cancer\\nOT  - Cas enzymes\\nOT  - Clustered regularly interspaced short palindromic repeats\\nOT  - Gene editing\\nOT  - T-lymphocytes\\nCOIS- The authors declare that they have no competing interests.\\nEDAT- 2021/08/28 06:00\\nMHDA- 2022/01/06 06:00\\nCRDT- 2021/08/27 05:42\\nPHST- 2021/06/09 00:00 [received]\\nPHST- 2021/08/16 00:00 [accepted]\\nPHST- 2021/08/27 05:42 [entrez]\\nPHST- 2021/08/28 06:00 [pubmed]\\nPHST- 2022/01/06 06:00 [medline]\\nAID - 10.1186/s13046-021-02076-5 [pii]\\nAID - 2076 [pii]\\nAID - 10.1186/s13046-021-02076-5 [doi]\\nPST - epublish\\nSO  - J Exp Clin Cancer Res. 2021 Aug 26;40(1):269. doi: 10.1186/s13046-021-02076-5.\\n\\nPMID- 34694813\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220125\\nLR  - 20220125\\nIS  - 1520-5207 (Electronic)\\nIS  - 1520-5207 (Linking)\\nVI  - 125\\nIP  - 43\\nDP  - 2021 Nov 4\\nTI  - Divalent-Metal-Ion Selectivity of the CRISPR-Cas System-Associated Cas1 Protein:  Insights from Classical Molecular Dynamics Simulations and Electronic Structure  Calculations.\\nPG  - 11943-11954\\nLID - 10.1021/acs.jpcb.1c07744 [doi]\\nAB  - CRISPR-associated protein 1 (Cas1) is a universally conserved essential  metalloenzyme of the clustered regularly interspaced short palindromic repeat  (CRISPR) immune system of prokaryotes (bacteria, archaea) that can cut and integrate  a part of viral DNA to its host genome with the help of other proteins. The  integrated DNA acts as a memory of viral infection, which can be transcribed to RNA  and stop future infection by recognition (based on the RNA/DNA complementarity  principle) followed by protein-mediated degradation of the viral DNA. It has been  proposed that the presence of a single manganese (Mn(2+)) ion in a conserved  divalent-metal-ion binding pocket (key residues: E190, H254, D265, D268) of Cas1 is  crucial for its function. Cas1-mediated DNA degradation was proposed to be hindered  by metal substitution, metal chelation, or mutation of the binding pocket residues.  Cas1 is active toward dsDNA degradation with both Mn(2+) and Mg(2+). X-ray  structures of Cas1 revealed an intricate atomic interaction network of the  divalent-metal-ion binding pocket and opened up the possibility of modeling related  metal ions (viz., Mg(2+), Ca(2+)) in the binding pocket of wild-type (WT) and  mutated Cas1 proteins for computational analysis, which includes (1) quantitative  estimation of the energetics of the divalent-metal-ion preference and (2) exploring  the structural and dynamical aspects of the protein in response to  divalent-metal-ion substitution or amino acid mutation. Using the X-ray structure of  the Cas1 protein from Pseudomonas aeruginosa as a template (PDB 3GOD), we performed  (âˆ¼2.23 Î¼s) classical molecular dynamics (MD) simulations to compare structural and  dynamical differences between Mg(2+)- and Ca(2+)-bound binding pockets of wild-type  (WT) and mutant (E190A, H254A, D265A, D268A) Cas1. Furthermore, reduced binding  pocket models were generated from X-ray and molecular dynamics (MD) trajectories,  and the resulting structures were subjected to quantum chemical calculations.  Results suggest that Cas1 prefers Mg(2+) binding relative to Ca(2+) and the  preference is the strongest for WT and the weakest for the D268A mutant. Quantum  chemical calculations indicate that Mn(2+) is the most preferred relative to both  Mg(2+) and Ca(2+) in the wild-type and mutant Cas1. Substitution of Mg(2+) by Ca(2+)  does not alter the interaction network between Cas1 and the divalent metal ion but  increases the wetness of the binding pocket by introducing a single water molecule  in the first coordination shell of the latter. The strength of metal-ion preference  (Mg(2+) versus Ca(2+)) seems to be dependent on the solvent accessibility of the  divalent-metal-ion binding pocket, strongest for wild-type Cas1 (in which the  metal-ion binding pocket is dry, which includes two water molecules) and the weakest  for the D268A mutant (in which the metal-ion binding pocket is wet, which includes  four water molecules).\\nFAU - Kumar, Abhishek\\nAU  - Kumar A\\nAD  - Department of Biosciences and Bioengineering, Indian Institute of Technology  Guwahati, Guwahati 781039, Assam, India.\\nFAU - Satpati, Priyadarshi\\nAU  - Satpati P\\nAUID- ORCID: 0000-0002-0391-3580\\nAD  - Department of Biosciences and Bioengineering, Indian Institute of Technology  Guwahati, Guwahati 781039, Assam, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20211025\\nPL  - United States\\nTA  - J Phys Chem B\\nJT  - The journal of physical chemistry. B\\nJID - 101157530\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - 0 (Ions)\\nSB  - IM\\nMH  - *CRISPR-Associated Proteins/genetics/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Electronics\\nMH  - Ions\\nMH  - Molecular Dynamics Simulation\\nEDAT- 2021/10/26 06:00\\nMHDA- 2022/01/27 06:00\\nCRDT- 2021/10/25 17:11\\nPHST- 2021/10/26 06:00 [pubmed]\\nPHST- 2022/01/27 06:00 [medline]\\nPHST- 2021/10/25 17:11 [entrez]\\nAID - 10.1021/acs.jpcb.1c07744 [doi]\\nPST - ppublish\\nSO  - J Phys Chem B. 2021 Nov 4;125(43):11943-11954. doi: 10.1021/acs.jpcb.1c07744. Epub  2021 Oct 25.\\n\\nPMID- 30853940\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 1664-302X (Print)\\nIS  - 1664-302X (Electronic)\\nIS  - 1664-302X (Linking)\\nVI  - 10\\nDP  - 2019\\nTI  - Gene Editing and Systems Biology Tools for Pesticide Bioremediation: A Review.\\nPG  - 87\\nLID - 10.3389/fmicb.2019.00087 [doi]\\nLID - 87\\nAB  - Bioremediation is the degradation potential of microorganisms to dissimilate the  complex chemical compounds from the surrounding environment. The genetics and  biochemistry of biodegradation processes in datasets opened the way of systems  biology. Systemic biology aid the study of interacting parts involved in the system.  The significant keys of system biology are biodegradation network, computational  biology, and omics approaches. Biodegradation network consists of all the databases  and datasets which aid in assisting the degradation and deterioration potential of  microorganisms for bioremediation processes. This review deciphers the  bio-degradation network, i.e., the databases and datasets (UM-BBD, PAN, PTID, etc.)  aiding in assisting the degradation and deterioration potential of microorganisms  for bioremediation processes, computational biology and multi omics approaches like  metagenomics, genomics, transcriptomics, proteomics, and metabolomics for the  efficient functional gene mining and their validation for bioremediation  experiments. Besides, the present review also describes the gene editing tools like  CRISPR Cas, TALEN, and ZFNs which can possibly make design microbe with functional  gene of interest for degradation of particular recalcitrant for improved  bioremediation.\\nFAU - Jaiswal, Shweta\\nAU  - Jaiswal S\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nFAU - Singh, Dileep Kumar\\nAU  - Singh DK\\nAD  - Soil Microbial Ecology and Environmental Toxicology Laboratory, Department of  Zoology, University of Delhi, New Delhi, India.\\nFAU - Shukla, Pratyoosh\\nAU  - Shukla P\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20190213\\nTA  - Front Microbiol\\nJT  - Frontiers in microbiology\\nJID - 101548977\\nPMC - PMC6396717\\nOTO - NOTNLM\\nOT  - bioremediation\\nOT  - gene editing\\nOT  - metabolic network\\nOT  - metabolomics\\nOT  - pollutant\\nOT  - systems biology\\nOT  - xenobiotics\\nEDAT- 2019/03/12 06:00\\nMHDA- 2019/03/12 06:01\\nCRDT- 2019/03/12 06:00\\nPHST- 2018/08/26 00:00 [received]\\nPHST- 2019/01/16 00:00 [accepted]\\nPHST- 2019/03/12 06:00 [entrez]\\nPHST- 2019/03/12 06:00 [pubmed]\\nPHST- 2019/03/12 06:01 [medline]\\nAID - 10.3389/fmicb.2019.00087 [doi]\\nPST - epublish\\nSO  - Front Microbiol. 2019 Feb 13;10:87. doi: 10.3389/fmicb.2019.00087. eCollection 2019.\\n\\nPMID- 33900064\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211220\\nLR  - 20211220\\nIS  - 2161-5063 (Electronic)\\nIS  - 2161-5063 (Linking)\\nVI  - 10\\nIP  - 5\\nDP  - 2021 May 21\\nTI  - CRISPR-Mediated Strand Displacement Logic Circuits with Toehold-Free DNA.\\nPG  - 950-956\\nLID - 10.1021/acssynbio.0c00649 [doi]\\nAB  - DNA nanotechnology, and DNA computing in particular, has grown extensively over the  past decade to end with a variety of functional stable structures and dynamic  circuits. However, the use as designer elements of regular DNA pieces, perfectly  complementary double strands, has remained elusive. Here, we report the exploitation  of CRISPR-Cas systems to engineer logic circuits based on isothermal strand  displacement that perform with toehold-free double-stranded DNA. We designed and  implemented molecular converters for signal detection and amplification, showing  good interoperability between enzymatic and nonenzymatic processes. Overall, these  results contribute to enlarge the repertoire of substrates and reactions (hardware)  for DNA computing.\\nFAU - Montagud-MartÃ\\xadnez, Roser\\nAU  - Montagud-MartÃ\\xadnez R\\nAD  - I2SysBio, CSIC - Universitat ValÃ¨ncia, Cat. AgustÃ\\xadn Escardino 9, 46980 Paterna,  Spain.\\nFAU - Heras-HernÃ¡ndez, MarÃ\\xada\\nAU  - Heras-HernÃ¡ndez M\\nAD  - I2SysBio, CSIC - Universitat ValÃ¨ncia, Cat. AgustÃ\\xadn Escardino 9, 46980 Paterna,  Spain.\\nFAU - Goiriz, Lucas\\nAU  - Goiriz L\\nAD  - I2SysBio, CSIC - Universitat ValÃ¨ncia, Cat. AgustÃ\\xadn Escardino 9, 46980 Paterna,  Spain.\\nFAU - DarÃ²s, JosÃ©-Antonio\\nAU  - DarÃ²s JA\\nAUID- ORCID: 0000-0002-6535-2889\\nAD  - IBMCP, CSIC - Universitat PolitÃ¨cnica ValÃ¨ncia, Av. Naranjos s/n, 46022 Valencia,  Spain.\\nFAU - Rodrigo, Guillermo\\nAU  - Rodrigo G\\nAUID- ORCID: 0000-0002-1871-9617\\nAD  - I2SysBio, CSIC - Universitat ValÃ¨ncia, Cat. AgustÃ\\xadn Escardino 9, 46980 Paterna,  Spain.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210426\\nTA  - ACS Synth Biol\\nJT  - ACS synthetic biology\\nJID - 101575075\\nRN  - 0 (DNA, Single-Stranded)\\nRN  - 0 (RNA, Guide)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nRN  - EC 3.1.26.4 (Ribonuclease H)\\nRN  - EC 3.1.27.5 (Ribonuclease, Pancreatic)\\nRN  - EC 3.4.21.64 (Endopeptidase K)\\nSB  - IM\\nMH  - CRISPR-Associated Protein 9/genetics\\nMH  - *CRISPR-Cas Systems\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - *Computers, Molecular\\nMH  - DNA, Single-Stranded/*genetics\\nMH  - Endopeptidase K/genetics\\nMH  - *Gene Regulatory Networks\\nMH  - Genetic Engineering/*methods\\nMH  - Nanotechnology/methods\\nMH  - RNA, Guide/*genetics\\nMH  - Ribonuclease H/genetics\\nMH  - Ribonuclease, Pancreatic/genetics\\nMH  - Streptococcus pyogenes/genetics\\nMH  - Transcription, Genetic/genetics\\nPMC - PMC8489798\\nOTO - NOTNLM\\nOT  - *DNA nanotechnology\\nOT  - *biological computing\\nOT  - *synthetic biology\\nCOIS- The authors declare no competing financial interest.\\nEDAT- 2021/04/27 06:00\\nMHDA- 2021/12/21 06:00\\nCRDT- 2021/04/26 12:10\\nPHST- 2021/04/27 06:00 [pubmed]\\nPHST- 2021/12/21 06:00 [medline]\\nPHST- 2021/04/26 12:10 [entrez]\\nAID - 10.1021/acssynbio.0c00649 [doi]\\nPST - ppublish\\nSO  - ACS Synth Biol. 2021 May 21;10(5):950-956. doi: 10.1021/acssynbio.0c00649. Epub 2021  Apr 26.\\n\\nPMID- 30643127\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190218\\nLR  - 20200915\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 10\\nIP  - 1\\nDP  - 2019 Jan 14\\nTI  - Anti-CRISPR-mediated control of gene editing and synthetic circuits in eukaryotic  cells.\\nPG  - 194\\nLID - 10.1038/s41467-018-08158-x [doi]\\nLID - 194\\nAB  - Repurposed CRISPR-Cas molecules provide a useful tool set for broad applications of  genomic editing and regulation of gene expression in prokaryotes and eukaryotes.  Recent discovery of phage-derived proteins, anti-CRISPRs, which serve to abrogate  natural CRISPR anti-phage activity, potentially expands the ability to build  synthetic CRISPR-mediated circuits. Here, we characterize a panel of anti-CRISPR  molecules for expanded applications to counteract CRISPR-mediated gene activation  and repression of reporter and endogenous genes in various cell types. We  demonstrate that cells pre-engineered with anti-CRISPR molecules become resistant to  gene editing, thus providing a means to generate \"write-protected\" cells that  prevent future gene editing. We further show that anti-CRISPRs can be used to  control CRISPR-based gene regulation circuits, including implementation of a pulse  generator circuit in mammalian cells. Our work suggests that anti-CRISPR proteins  should serve as widely applicable tools for synthetic systems regulating the  behavior of eukaryotic cells.\\nFAU - Nakamura, Muneaki\\nAU  - Nakamura M\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Srinivasan, Prashanth\\nAU  - Srinivasan P\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Chavez, Michael\\nAU  - Chavez M\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Carter, Matthew A\\nAU  - Carter MA\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Dominguez, Antonia A\\nAU  - Dominguez AA\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.\\nFAU - La Russa, Marie\\nAU  - La Russa M\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Lau, Matthew B\\nAU  - Lau MB\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - International Christian School, 1 On Muk Ln, Sha Tin, 999077, Hong Kong SAR, China.\\nFAU - Abbott, Timothy R\\nAU  - Abbott TR\\nAUID- ORCID: 0000-0001-7404-811X\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Xu, Xiaoshu\\nAU  - Xu X\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Zhao, Dehua\\nAU  - Zhao D\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Gao, Yuchen\\nAU  - Gao Y\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nAD  - Cancer Biology Program, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Kipniss, Nathan H\\nAU  - Kipniss NH\\nAUID- ORCID: 0000-0001-9864-942X\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Smolke, Christina D\\nAU  - Smolke CD\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.\\nFAU - Bondy-Denomy, Joseph\\nAU  - Bondy-Denomy J\\nAD  - Department of Microbiology and Immunology, University of California, San Francisco,  CA, 94158, USA.\\nAD  - Quantitative Biosciences Institute, University of California, San Francisco, CA,  94158, USA.\\nFAU - Qi, Lei S\\nAU  - Qi LS\\nAUID- ORCID: 0000-0002-3965-3223\\nAD  - Department of Bioengineering, Stanford University, Stanford, CA, 94305, USA.  stanley.qi@stanford.edu.\\nAD  - Department of Chemical and Systems Biology, Stanford University, Stanford, CA,  94305, USA. stanley.qi@stanford.edu.\\nAD  - Stanford ChEM-H, Stanford University, Stanford, CA, 94305, USA.  stanley.qi@stanford.edu.\\nLA  - eng\\nGR  - T32 GM008568/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Video-Audio Media\\nDEP - 20190114\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nSB  - IM\\nCIN - Nat Methods. 2019 Mar;16(3):218. PMID: 30814697\\nMH  - Bacteriophages/*genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Cell Culture Techniques\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - Eukaryotic Cells\\nMH  - Gene Editing/*methods\\nMH  - Gene Regulatory Networks/*genetics\\nMH  - Genetic Vectors/genetics\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Induced Pluripotent Stem Cells\\nMH  - Intravital Microscopy/methods\\nMH  - Lentivirus/genetics\\nMH  - Microscopy, Fluorescence/methods\\nMH  - Time-Lapse Imaging/methods\\nMH  - Transduction, Genetic/methods\\nMH  - Transfection/methods\\nPMC - PMC6331597\\nCOIS- A provisional patent has been filed with M.N. and L.S.Q. as inventors on the method  of using anti-CRISPR molecules for constructing gene editing and expression circuits  (US Provisional Application No. 62/777,652). J.B.-D. is a scientific advisory board  member of SNIPR Biome and Excision Biotherapeutics.Â\\xa0All remaining authors declare no  competing interests.\\nEDAT- 2019/01/16 06:00\\nMHDA- 2019/02/20 06:00\\nCRDT- 2019/01/16 06:00\\nPHST- 2018/03/13 00:00 [received]\\nPHST- 2018/12/20 00:00 [accepted]\\nPHST- 2019/01/16 06:00 [entrez]\\nPHST- 2019/01/16 06:00 [pubmed]\\nPHST- 2019/02/20 06:00 [medline]\\nAID - 10.1038/s41467-018-08158-x [pii]\\nAID - 8158 [pii]\\nAID - 10.1038/s41467-018-08158-x [doi]\\nPST - epublish\\nSO  - Nat Commun. 2019 Jan 14;10(1):194. doi: 10.1038/s41467-018-08158-x.\\n\\nPMID- 31992616\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200608\\nLR  - 20200608\\nIS  - 2150-7511 (Electronic)\\nVI  - 11\\nIP  - 1\\nDP  - 2020 Jan 28\\nTI  - A CRISPR-Assisted Nonhomologous End-Joining Strategy for Efficient Genome Editing in  Mycobacterium tuberculosis.\\nLID - 10.1128/mBio.02364-19 [doi]\\nLID - e02364-19\\nAB  - New tools for genetic manipulation of Mycobacterium tuberculosis are needed for the  development of new drug regimens and vaccines aimed at curing tuberculosis  infections. Clustered regularly interspaced short palindromic repeat  (CRISPR)-CRISPR-associated protein (Cas) systems generate a highly specific  double-strand break at the target site that can be repaired via nonhomologous end  joining (NHEJ), resulting in the desired genome alteration. In this study, we first  improved the NHEJ repair pathway and developed a CRISPR-Cas-mediated genome-editing  method that allowed us to generate markerless deletion in Mycobacterium smegmatis,  Mycobacterium marinum, and M. tuberculosis Then, we demonstrated that this system  could efficiently achieve simultaneous generation of double mutations and  large-scale genetic mutations in M. tuberculosis Finally, we showed that the  strategy we developed can also be used to facilitate genome editing in Escherichia  coli IMPORTANCE The global health impact of M. tuberculosis necessitates the  development of new genetic tools for its manipulation, to facilitate the  identification and characterization of novel drug targets and vaccine candidates.  Clustered regularly interspaced short palindromic repeat (CRISPR)-CRISPR-associated  protein (Cas) genome editing has proven to be a powerful genetic tool in various  organisms; to date, however, attempts to use this approach in M. tuberculosis have  failed. Here, we describe a genome-editing tool based on CRISPR cleavage and the  nonhomologous end-joining (NHEJ) repair pathway that can efficiently generate  deletion mutants in M. tuberculosis More importantly, this system can generate  simultaneous double mutations and large-scale genetic mutations in this species. We  anticipate that this CRISPR-NHEJ-assisted genome-editing system will be broadly  useful for research on mycobacteria, vaccine development, and drug target profiling.\\nCI  - Copyright Â© 2020 Yan et al.\\nFAU - Yan, Mei-Yi\\nAU  - Yan MY\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China.\\nFAU - Li, Si-Shang\\nAU  - Li SS\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China.\\nFAU - Ding, Xin-Yuan\\nAU  - Ding XY\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China.\\nFAU - Guo, Xiao-Peng\\nAU  - Guo XP\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China.\\nFAU - Jin, Qi\\nAU  - Jin Q\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China.\\nFAU - Sun, Yi-Cheng\\nAU  - Sun YC\\nAD  - MOH Key Laboratory of Systems Biology of Pathogens, Institute of Pathogen Biology,  and Center for Tuberculosis Research, Chinese Academy of Medical Sciences and Peking  Union Medical College, Beijing, China sunyc@ipbcams.ac.cn.\\nAD  - Sanming Project of Medicine in Shenzhen on Construction of Novel Systematic Network  against Tuberculosis, National Clinical Research Center for Infectious Diseases,  Shenzhen Third People\\'s Hospital, Southern University of Science and Technology,  Shenzhen, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200128\\nTA  - mBio\\nJT  - mBio\\nJID - 101519231\\nRN  - 0 (Bacterial Toxins)\\nRN  - 0 (RNA, Guide)\\nRN  - EC 2.7.7.- (Rec A Recombinases)\\nSB  - IM\\nMH  - Bacterial Toxins/genetics/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - *Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - *DNA End-Joining Repair\\nMH  - *Gene Editing\\nMH  - *Genome, Bacterial\\nMH  - Models, Biological\\nMH  - Mycobacterium smegmatis/genetics\\nMH  - Mycobacterium tuberculosis/*genetics/metabolism\\nMH  - Protein Binding\\nMH  - RNA, Guide\\nMH  - Rec A Recombinases/metabolism\\nPMC - PMC6989103\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas system\\nOT  - *Mycobacterium marinum\\nOT  - *Mycobacterium smegmatis\\nOT  - *Mycobacterium tuberculosis\\nOT  - *genome editing\\nOT  - *nonhomologous end joining\\nEDAT- 2020/01/30 06:00\\nMHDA- 2020/06/09 06:00\\nCRDT- 2020/01/30 06:00\\nPHST- 2020/01/30 06:00 [entrez]\\nPHST- 2020/01/30 06:00 [pubmed]\\nPHST- 2020/06/09 06:00 [medline]\\nAID - mBio.02364-19 [pii]\\nAID - mBio02364-19 [pii]\\nAID - 10.1128/mBio.02364-19 [doi]\\nPST - epublish\\nSO  - mBio. 2020 Jan 28;11(1):e02364-19. doi: 10.1128/mBio.02364-19.\\n\\nPMID- 25422271\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150209\\nLR  - 20181113\\nIS  - 1744-4292 (Electronic)\\nIS  - 1744-4292 (Print)\\nIS  - 1744-4292 (Linking)\\nVI  - 10\\nIP  - 11\\nDP  - 2014 Nov 24\\nTI  - Multi-input CRISPR/Cas genetic circuits that interface host regulatory networks.\\nPG  - 763\\nLID - 10.15252/msb.20145735 [doi]\\nLID - 763\\nAB  - Genetic circuits require many regulatory parts in order to implement signal  processing or execute algorithms in cells. A potentially scalable approach is to use  dCas9, which employs small guide RNAs (sgRNAs) to repress genetic loci via the  programmability of RNA:DNA base pairing. To this end, we use dCas9 and designed  sgRNAs to build transcriptional logic gates and connect them to perform computation  in living cells. We constructed a set of NOT gates by designing five synthetic  Escherichia coli Ïƒ70 promoters that are repressed by corresponding sgRNAs, and these  interactions do not exhibit crosstalk between each other. These sgRNAs exhibit high  on-target repression (56- to 440-fold) and negligible off-target interactions (<  1.3-fold). These gates were connected to build larger circuits, including the  Boolean-complete NOR gate and a 3-gate circuit consisting of four layered sgRNAs.  The synthetic circuits were connected to the native E. coli regulatory network by  designing output sgRNAs to target an E. coli transcription factor (malT). This  converts the output of a synthetic circuit to a switch in cellular phenotype (sugar  utilization, chemotaxis, phage resistance).\\nCI  - Â© 2014 The Authors. Published under the terms of the CC BY 4.0 license.\\nFAU - Nielsen, Alec A K\\nAU  - Nielsen AA\\nAD  - Synthetic Biology Center, Department of Biological Engineering, Massachusetts  Institute of Technology, Cambridge, MA, USA.\\nFAU - Voigt, Christopher A\\nAU  - Voigt CA\\nAD  - Synthetic Biology Center, Department of Biological Engineering, Massachusetts  Institute of Technology, Cambridge, MA, USA cavoigt@gmail.com.\\nLA  - eng\\nGR  - 27302C0028/ES/NIEHS NIH HHS/United States\\nGR  - P50 GMO98792/PHS HHS/United States\\nGR  - P50 GM098792/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM095765/GM/NIGMS NIH HHS/United States\\nGR  - GM095765/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20141124\\nTA  - Mol Syst Biol\\nJT  - Molecular systems biology\\nJID - 101235389\\nRN  - 0 (Escherichia coli Proteins)\\nSB  - IM\\nMH  - Base Pairing\\nMH  - *CRISPR-Cas Systems\\nMH  - Escherichia coli/*genetics\\nMH  - Escherichia coli Proteins/*genetics\\nMH  - *Gene Regulatory Networks\\nMH  - Models, Biological\\nPMC - PMC4299604\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - TALE\\nOT  - TetR homologue\\nOT  - genetic compiler\\nOT  - synthetic biology\\nEDAT- 2014/11/26 06:00\\nMHDA- 2015/02/11 06:00\\nCRDT- 2014/11/26 06:00\\nPHST- 2014/11/26 06:00 [entrez]\\nPHST- 2014/11/26 06:00 [pubmed]\\nPHST- 2015/02/11 06:00 [medline]\\nAID - 10.15252/msb.20145735 [doi]\\nPST - epublish\\nSO  - Mol Syst Biol. 2014 Nov 24;10(11):763. doi: 10.15252/msb.20145735.\\n\\nPMID- 30793202\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191112\\nLR  - 20200309\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 47\\nIP  - 6\\nDP  - 2019 Apr 8\\nTI  - Does co-transcriptional regulation of alternative splicing mediate plant stress  responses?\\nPG  - 2716-2726\\nLID - 10.1093/nar/gkz121 [doi]\\nAB  - Plants display exquisite control over gene expression to elicit appropriate  responses under normal and stress conditions. Alternative splicing (AS) of  pre-mRNAs, a process that generates two or more transcripts from multi-exon genes,  adds another layer of regulation to fine-tune condition-specific gene expression in  animals and plants. However, exactly how plants control splice isoform ratios and  the timing of this regulation in response to environmental signals remains elusive.  In mammals, recent evidence indicate that epigenetic and epitranscriptome changes,  such as DNA methylation, chromatin modifications and RNA methylation, regulate RNA  polymerase II processivity, co-transcriptional splicing, and stability and  translation efficiency of splice isoforms. In plants, the role of epigenetic  modifications in regulating transcription rate and mRNA abundance under stress is  beginning to emerge. However, the mechanisms by which epigenetic and  epitranscriptomic modifications regulate AS and translation efficiency require  further research. Dynamic changes in the chromatin landscape in response to stress  may provide a scaffold around which gene expression, AS and translation are  orchestrated. Finally, we discuss CRISPR/Cas-based strategies for engineering  chromatin architecture to manipulate AS patterns (or splice isoforms levels) to  obtain insight into the epigenetic regulation of AS.\\nCI  - Â© The Author(s) 2019. Published by Oxford University Press on behalf of Nucleic  Acids Research.\\nFAU - Jabre, Ibtissam\\nAU  - Jabre I\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nFAU - Reddy, Anireddy S N\\nAU  - Reddy ASN\\nAD  - Department of Biology and Program in Cell and Molecular Biology, Colorado State  University, Fort Collins, CO 80523-1878, USA.\\nFAU - Kalyna, Maria\\nAU  - Kalyna M\\nAD  - Department of Applied Genetics and Cell Biology, University of Natural Resources and  Life Sciences - BOKU, Muthgasse 18, 1190 Vienna, Austria.\\nFAU - Chaudhary, Saurabh\\nAU  - Chaudhary S\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nFAU - Khokhar, Waqas\\nAU  - Khokhar W\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nFAU - Byrne, Lee J\\nAU  - Byrne LJ\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nFAU - Wilson, Cornelia M\\nAU  - Wilson CM\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nFAU - Syed, Naeem H\\nAU  - Syed NH\\nAD  - School of Human and Life Sciences, Canterbury Christ Church University, Canterbury,  CT1 1QU, UK.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPT  - Review\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Animals\\nMH  - Arabidopsis/genetics\\nMH  - DNA Methylation/genetics\\nMH  - Epigenesis, Genetic/physiology\\nMH  - *Gene Expression Regulation, Plant\\nMH  - Gene Regulatory Networks/genetics\\nMH  - Humans\\nMH  - Stress, Physiological/*genetics\\nMH  - Transcription, Genetic/genetics\\nPMC - PMC6451118\\nEDAT- 2019/02/23 06:00\\nMHDA- 2019/11/13 06:00\\nCRDT- 2019/02/23 06:00\\nPHST- 2019/02/13 00:00 [accepted]\\nPHST- 2019/02/11 00:00 [revised]\\nPHST- 2018/11/09 00:00 [received]\\nPHST- 2019/02/23 06:00 [pubmed]\\nPHST- 2019/11/13 06:00 [medline]\\nPHST- 2019/02/23 06:00 [entrez]\\nAID - 5356938 [pii]\\nAID - gkz121 [pii]\\nAID - 10.1093/nar/gkz121 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2019 Apr 8;47(6):2716-2726. doi: 10.1093/nar/gkz121.\\n\\nPMID- 33956155\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210625\\nLR  - 20210625\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 49\\nIP  - 10\\nDP  - 2021 Jun 4\\nTI  - Minimized combinatorial CRISPR screens identify genetic interactions in autophagy.\\nPG  - 5684-5704\\nLID - 10.1093/nar/gkab309 [doi]\\nAB  - Combinatorial CRISPR-Cas screens have advanced the mapping of genetic interactions,  but their experimental scale limits the number of targetable gene combinations.  Here, we describe 3Cs multiplexing, a rapid and scalable method to generate highly  diverse and uniformly distributed combinatorial CRISPR libraries. We demonstrate  that the library distribution skew is the critical determinant of its required  screening coverage. By circumventing iterative cloning of PCR-amplified  oligonucleotides, 3Cs multiplexing facilitates the generation of combinatorial  CRISPR libraries with low distribution skews. We show that combinatorial 3Cs  libraries can be screened with minimal coverages, reducing associated efforts and  costs at least 10-fold. We apply a 3Cs multiplexing library targeting 12,736  autophagy gene combinations with 247,032 paired gRNAs in viability and  reporter-based enrichment screens. In the viability screen, we identify, among  others, the synthetic lethal WDR45B-PIK3R4 and the proliferation-enhancing  ATG7-KEAP1 genetic interactions. In the reporter-based screen, we identify over  1,570 essential genetic interactions for autophagy flux, including interactions  among paralogous genes, namely ATG2A-ATG2B, GABARAP-MAP1LC3B and GABARAP-GABARAPL2.  However, we only observe few genetic interactions within paralogous gene families of  more than two members, indicating functional compensation between them. This work  establishes 3Cs multiplexing as a platform for genetic interaction screens at scale.\\nCI  - Â© The Author(s) 2021. Published by Oxford University Press on behalf of Nucleic  Acids Research.\\nFAU - Diehl, Valentina\\nAU  - Diehl V\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Wegner, Martin\\nAU  - Wegner M\\nAUID- ORCID: 0000-0001-6403-3926\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Grumati, Paolo\\nAU  - Grumati P\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Husnjak, Koraljka\\nAU  - Husnjak K\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Schaubeck, Simone\\nAU  - Schaubeck S\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Gubas, Andrea\\nAU  - Gubas A\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Shah, Varun Jayeshkumar\\nAU  - Shah VJ\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Polat, Ibrahim H\\nAU  - Polat IH\\nAD  - Department of Medicine, Hematology/Oncology, University Hospital, Goethe University,  60590 Frankfurt am Main, Germany.\\nFAU - Langschied, Felix\\nAU  - Langschied F\\nAD  - Applied Bioinformatics Group, Institute of Cell Biology and Neuroscience, Goethe  University Frankfurt, 60438 Frankfurt am Main, Germany.\\nFAU - Prieto-Garcia, Cristian\\nAU  - Prieto-Garcia C\\nAUID- ORCID: 0000-0003-1732-2535\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - MÃ¼ller, Konstantin\\nAU  - MÃ¼ller K\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Kalousi, Alkmini\\nAU  - Kalousi A\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nFAU - Ebersberger, Ingo\\nAU  - Ebersberger I\\nAUID- ORCID: 0000-0001-8187-9253\\nAD  - Applied Bioinformatics Group, Institute of Cell Biology and Neuroscience, Goethe  University Frankfurt, 60438 Frankfurt am Main, Germany.\\nAD  - Senckenberg Biodiversity and Climate Research Centre (S-BIK-F), Frankfurt am Main,  Germany.\\nAD  - LOEWE Centre for Translational Biodiversity Genomics (TBG), Frankfurt am Main,  Germany.\\nFAU - Brandts, Christian H\\nAU  - Brandts CH\\nAD  - Department of Medicine, Hematology/Oncology, University Hospital, Goethe University,  60590 Frankfurt am Main, Germany.\\nAD  - Frankfurt Cancer Institute, 60596 Frankfurt am Main, Germany.\\nAD  - University Cancer Center Frankfurt (UCT), University Hospital, Goethe University,  Frankfurt, Germany.\\nFAU - Dikic, Ivan\\nAU  - Dikic I\\nAUID- ORCID: 0000-0001-8156-9511\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nAD  - Frankfurt Cancer Institute, 60596 Frankfurt am Main, Germany.\\nAD  - Cardio-Pulmonary Institute, 60590 Frankfurt am Main, Germany.\\nAD  - Buchmann Institute for Molecular Life Sciences, Goethe University Frankfurt, 60438  Frankfurt am Main, Germany.\\nFAU - Kaulich, Manuel\\nAU  - Kaulich M\\nAUID- ORCID: 0000-0002-9528-8822\\nAD  - Institute of Biochemistry II, Faculty of Medicine, Goethe University Frankfurt,  Theodor-Stern-Kai 7, 60590 Frankfurt am Main, Germany.\\nAD  - Frankfurt Cancer Institute, 60596 Frankfurt am Main, Germany.\\nAD  - Cardio-Pulmonary Institute, 60590 Frankfurt am Main, Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nRN  - 0 (ATG2B protein, human)\\nRN  - 0 (Autophagy-Related Proteins)\\nRN  - 0 (KEAP1 protein, human)\\nRN  - 0 (Kelch-Like ECH-Associated Protein 1)\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Autophagy/*genetics\\nMH  - Autophagy-Related Proteins/genetics/metabolism\\nMH  - *CRISPR-Cas Systems\\nMH  - Carcinoma, Squamous Cell/mortality\\nMH  - Cell Proliferation/genetics\\nMH  - Cell Survival/genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Databases, Genetic\\nMH  - Gene Knockout Techniques/*methods\\nMH  - Gene Library\\nMH  - Gene Regulatory Networks/*genetics\\nMH  - Genes, Essential\\nMH  - HEK293 Cells\\nMH  - High-Throughput Nucleotide Sequencing\\nMH  - Humans\\nMH  - Kaplan-Meier Estimate\\nMH  - Kelch-Like ECH-Associated Protein 1/genetics/metabolism\\nMH  - Lung Neoplasms/mortality\\nMH  - Models, Genetic\\nMH  - RNA, Guide\\nMH  - RNA-Seq\\nMH  - Vesicular Transport Proteins/genetics/metabolism\\nPMC - PMC8191801\\nEDAT- 2021/05/07 06:00\\nMHDA- 2021/06/29 06:00\\nCRDT- 2021/05/06 13:05\\nPHST- 2021/04/14 00:00 [accepted]\\nPHST- 2021/04/01 00:00 [revised]\\nPHST- 2021/02/16 00:00 [received]\\nPHST- 2021/05/07 06:00 [pubmed]\\nPHST- 2021/06/29 06:00 [medline]\\nPHST- 2021/05/06 13:05 [entrez]\\nAID - 6270805 [pii]\\nAID - gkab309 [pii]\\nAID - 10.1093/nar/gkab309 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2021 Jun 4;49(10):5684-5704. doi: 10.1093/nar/gkab309.\\n\\nPMID- 34069056\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210624\\nLR  - 20210624\\nIS  - 1660-4601 (Electronic)\\nIS  - 1661-7827 (Print)\\nIS  - 1660-4601 (Linking)\\nVI  - 18\\nIP  - 10\\nDP  - 2021 May 14\\nTI  - Extremophiles, a Nifty Tool to Face Environmental Pollution: From Exploitation of  Metabolism to Genome Engineering.\\nLID - 10.3390/ijerph18105228 [doi]\\nLID - 5228\\nAB  - Extremophiles are microorganisms that populate habitats considered inhospitable from  an anthropocentric point of view and are able to tolerate harsh conditions such as  high temperatures, extreme pHs, high concentrations of salts, toxic organic  substances, and/or heavy metals. These microorganisms have been broadly studied in  the last 30 years and represent precious sources of biomolecules and bioprocesses  for many biotechnological applications; in this context, scientific efforts have  been focused on the employment of extremophilic microbes and their metabolic  pathways to develop biomonitoring and bioremediation strategies to face  environmental pollution, as well as to improve biorefineries for the conversion of  biomasses into various chemical compounds. This review gives an overview on the  peculiar metabolic features of certain extremophilic microorganisms, with a main  focus on thermophiles, which make them attractive for biotechnological applications  in the field of environmental remediation; moreover, it sheds light on updated  genetic systems (also those based on the CRISPR-Cas tool), which expand the  potentialities of these microorganisms to be genetically manipulated for various  biotechnological purposes.\\nFAU - Gallo, Giovanni\\nAU  - Gallo G\\nAUID- ORCID: 0000-0001-6846-6526\\nAD  - Department of Biology, University of Naples Federico II, Via Cinthia 21, 80126  Napoli, Italy.\\nAD  - Consiglio Nazionale delle Ricerche CNR, Institute of Polymers, Composites and  Biomaterials (IPCB), Via Campi Flegrei, 34, 80078 Pozzuoli, Italy.\\nFAU - Puopolo, Rosanna\\nAU  - Puopolo R\\nAUID- ORCID: 0000-0002-7640-5384\\nAD  - Department of Biology, University of Naples Federico II, Via Cinthia 21, 80126  Napoli, Italy.\\nFAU - Carbonaro, Miriam\\nAU  - Carbonaro M\\nAD  - Department of Biology, University of Naples Federico II, Via Cinthia 21, 80126  Napoli, Italy.\\nFAU - Maresca, Emanuela\\nAU  - Maresca E\\nAD  - Department of Biology, University of Naples Federico II, Via Cinthia 21, 80126  Napoli, Italy.\\nFAU - Fiorentino, Gabriella\\nAU  - Fiorentino G\\nAUID- ORCID: 0000-0002-4089-3822\\nAD  - Department of Biology, University of Naples Federico II, Via Cinthia 21, 80126  Napoli, Italy.\\nAD  - Consiglio Nazionale delle Ricerche CNR, Institute of Polymers, Composites and  Biomaterials (IPCB), Via Campi Flegrei, 34, 80078 Pozzuoli, Italy.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20210514\\nTA  - Int J Environ Res Public Health\\nJT  - International journal of environmental research and public health\\nJID - 101238455\\nRN  - 0 (Metals, Heavy)\\nSB  - IM\\nMH  - Biodegradation, Environmental\\nMH  - Ecosystem\\nMH  - *Extremophiles/genetics\\nMH  - Metabolic Networks and Pathways\\nMH  - *Metals, Heavy\\nPMC - PMC8157027\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *aromatic-compounds\\nOT  - *bioremediation\\nOT  - *biosensors\\nOT  - *environmental pollution\\nOT  - *extremophiles\\nOT  - *genome-engineering\\nOT  - *heavy-metal resistance\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2021/06/03 06:00\\nMHDA- 2021/06/25 06:00\\nCRDT- 2021/06/02 01:24\\nPHST- 2021/04/16 00:00 [received]\\nPHST- 2021/05/06 00:00 [revised]\\nPHST- 2021/05/09 00:00 [accepted]\\nPHST- 2021/06/02 01:24 [entrez]\\nPHST- 2021/06/03 06:00 [pubmed]\\nPHST- 2021/06/25 06:00 [medline]\\nAID - ijerph18105228 [pii]\\nAID - ijerph-18-05228 [pii]\\nAID - 10.3390/ijerph18105228 [doi]\\nPST - epublish\\nSO  - Int J Environ Res Public Health. 2021 May 14;18(10):5228. doi:  10.3390/ijerph18105228.\\n\\nPMID- 30773816\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200413\\nLR  - 20200413\\nIS  - 1462-2920 (Electronic)\\nIS  - 1462-2912 (Print)\\nIS  - 1462-2912 (Linking)\\nVI  - 21\\nIP  - 6\\nDP  - 2019 Jun\\nTI  - Integrated mobile genetic elements in Thaumarchaeota.\\nPG  - 2056-2078\\nLID - 10.1111/1462-2920.14564 [doi]\\nAB  - To explore the diversity of mobile genetic elements (MGE) associated with archaea of  the phylum Thaumarchaeota, we exploited the property of most MGE to integrate into  the genomes of their hosts. Integrated MGE (iMGE) were identified in 20  thaumarchaeal genomes amounting to 2 Mbp of mobile thaumarchaeal DNA. These iMGE  group into five major classes: (i) proviruses, (ii) casposons, (iii) insertion  sequence-like transposons, (iv) integrative-conjugative elements and (v) cryptic  integrated elements. The majority of the iMGE belong to the latter category and  might represent novel families of viruses or plasmids. The identified proviruses are  related to tailed viruses of the order Caudovirales and to tailless icosahedral  viruses with the double jelly-roll capsid proteins. The thaumarchaeal iMGE are all  connected within a gene sharing network, highlighting pervasive gene exchange  between MGE occupying the same ecological niche. The thaumarchaeal mobilome carries  multiple auxiliary metabolic genes, including multicopper oxidases and ammonia  monooxygenase subunit C (AmoC), and stress response genes, such as those for  universal stress response proteins (UspA). Thus, iMGE might make important  contributions to the fitness and adaptation of their hosts. We identified several  iMGE carrying type I-B CRISPR-Cas systems and spacers matching other thaumarchaeal  iMGE, suggesting antagonistic interactions between coexisting MGE and symbiotic  relationships with the ir archaeal hosts.\\nCI  - Â© 2019 The Authors. Environmental Microbiology published by Society for Applied  Microbiology and John Wiley & Sons Ltd.\\nFAU - Krupovic, Mart\\nAU  - Krupovic M\\nAUID- ORCID: 0000-0001-5486-0098\\nAD  - Institut Pasteur, UnitÃ© Biologie MolÃ©culaire du GÃ¨ne chez les ExtrÃªmophiles, 75015,  Paris, France.\\nFAU - Makarova, Kira S\\nAU  - Makarova KS\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  Bethesda, MD 20894, USA.\\nFAU - Wolf, Yuri I\\nAU  - Wolf YI\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  Bethesda, MD 20894, USA.\\nFAU - Medvedeva, Sofia\\nAU  - Medvedeva S\\nAD  - Institut Pasteur, UnitÃ© Biologie MolÃ©culaire du GÃ¨ne chez les ExtrÃªmophiles, 75015,  Paris, France.\\nAD  - Center of Life Sciences, Skolkovo Institute of Science and Technology, Skolkovo,  Russia.\\nAD  - Sorbonne UniversitÃ©, CollÃ¨ge doctoral, 75005, Paris, France.\\nFAU - Prangishvili, David\\nAU  - Prangishvili D\\nAUID- ORCID: 0000-0002-0812-9699\\nAD  - Institut Pasteur, UnitÃ© Biologie MolÃ©culaire du GÃ¨ne chez les ExtrÃªmophiles, 75015,  Paris, France.\\nFAU - Forterre, Patrick\\nAU  - Forterre P\\nAD  - Institut Pasteur, UnitÃ© Biologie MolÃ©culaire du GÃ¨ne chez les ExtrÃªmophiles, 75015,  Paris, France.\\nAD  - Institute for Integrative Biology of the Cell (I2BC), CEA, CNRS, Univ. Paris- Sud,  UniversitÃ© Paris-Saclay, Gif-sur-Yvette cedex, Paris, France.\\nFAU - Koonin, Eugene V\\nAU  - Koonin EV\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  Bethesda, MD 20894, USA.\\nLA  - eng\\nGR  - ANR-17-CE15-0005-01/Agence Nationale de la Recherche/International\\nGR  - UE 340440/H2020 European Research Council/International\\nGR  - 685778/Horizon 2020 Framework Programme/International\\nGR  - intramural program/U.S. National Library of Medicine/International\\nGR  - RSF 14-14-00988/Campus France/International\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190318\\nTA  - Environ Microbiol\\nJT  - Environmental microbiology\\nJID - 100883692\\nRN  - 0 (Archaeal Proteins)\\nRN  - 0 (DNA Transposable Elements)\\nRN  - EC 1.- (Oxidoreductases)\\nRN  - EC 1.7.3.- (ammonia monooxygenase)\\nSB  - IM\\nMH  - Archaea/classification/enzymology/*genetics/virology\\nMH  - Archaeal Proteins/genetics/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - *DNA Transposable Elements\\nMH  - Oxidoreductases/genetics/metabolism\\nMH  - Phylogeny\\nMH  - Plasmids/genetics/metabolism\\nPMC - PMC6563490\\nEDAT- 2019/02/19 06:00\\nMHDA- 2020/04/14 06:00\\nCRDT- 2019/02/19 06:00\\nPHST- 2018/11/30 00:00 [received]\\nPHST- 2019/02/10 00:00 [revised]\\nPHST- 2019/02/13 00:00 [accepted]\\nPHST- 2019/02/19 06:00 [pubmed]\\nPHST- 2020/04/14 06:00 [medline]\\nPHST- 2019/02/19 06:00 [entrez]\\nAID - EMI14564 [pii]\\nAID - 10.1111/1462-2920.14564 [doi]\\nPST - ppublish\\nSO  - Environ Microbiol. 2019 Jun;21(6):2056-2078. doi: 10.1111/1462-2920.14564. Epub 2019  Mar 18.\\n\\nPMID- 32281426\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210112\\nLR  - 20210112\\nIS  - 1744-7593 (Electronic)\\nIS  - 1742-5247 (Linking)\\nVI  - 17\\nIP  - 6\\nDP  - 2020 Jun\\nTI  - Delivery of genome editing tools: A promising strategy for HPV-related cervical  malignancy therapy.\\nPG  - 753-766\\nLID - 10.1080/17425247.2020.1747429 [doi]\\nAB  - INTRODUCTION: Persistent high-risk human papillomavirus infection is the main cause  of various types of cancer especially cervical cancer. The E6 and E7 oncoproteins of  HPV play critical roles in promoting carcinogenesis and cancer cell growth. As a  result, E6 and E7 oncogenes are considered as promising therapeutic targets for  cervical cancer. Recently, the development of genome-editing technologies including  transcription activator-like effector nucleases (TALEN), meganucleases (MNs), zinc  finger nucleases (ZFN), and more importantly clustered regularly interspaced short  palindromic repeat-CRISPR-associated protein (CRISPR-Cas) has sparked a revolution  in the cervical cancer-targeted therapy. However, due to immunogenicity, off-target  effect, renal clearance, guide RNA (gRNA) nuclease degradation, and difficult direct  transportation into the cytoplasm and nucleus, the safe and effective delivery is  considered as the Achilles\\' heel of this robust strategy. AREAS COVERED: In this  review, we discuss cutting-edge available strategies for in vivo delivery of  genome-editing technologies for HPV-induced cervical cancer therapy. Moreover, the  combination of genome-editing tools and other therapies has been fully discussed.  EXPERT OPINION: The combination of nanoparticle-based delivery systems and  genome-editing tools is a promising powerful strategy for cervical cancer therapy.  The most significant limitations of this strategy that need to be focused on are low  efficiency and off-target events.\\nFAU - Aghamiri, Shahin\\nAU  - Aghamiri S\\nAD  - Student Research Committee, Department of Medical Biotechnology, School of Advanced  Technologies in Medicine, Shahid Beheshti University of Medical Sciences , Tehran,  Iran.\\nAD  - Cellular and Molecular Biology Research Center, Shahid Beheshti University of  Medical Sciences , Tehran, Iran.\\nFAU - Talaei, Sam\\nAU  - Talaei S\\nAD  - School of Pharmacy, Shahid Beheshti University of Medical Sciences , Tehran, Iran.\\nFAU - Roshanzamiri, Soheil\\nAU  - Roshanzamiri S\\nAD  - Department of Clinical Pharmacy, School of Pharmacy, Shahid Beheshti University of  Medical Sciences , Tehran, Iran.\\nFAU - Zandsalimi, Farshid\\nAU  - Zandsalimi F\\nAD  - Students\\' Scientific Research Center, Department of Molecular Medicine, School of  Advanced Technologies in Medicine, Tehran University of Medical Sciences , Tehran,  Iran.\\nFAU - Fazeli, Elnaz\\nAU  - Fazeli E\\nAD  - Department of Medical Genetics, School of Medicine, Shahid Beheshti University of  Medical Sciences , Tehran, Iran.\\nFAU - Aliyu, Mansur\\nAU  - Aliyu M\\nAD  - Department of Immunology, School of Public Health, Tehran University of Medical  Sciences, International Campus , Tehran, Iran.\\nFAU - Kheiry Avarvand, Omid\\nAU  - Kheiry Avarvand O\\nAD  - Student Research Committee, Faculty of Pharmacy, Ahvaz Jundishapur University of  Medical Sciences , Ahvaz, Iran.\\nFAU - Ebrahimi, Zahra\\nAU  - Ebrahimi Z\\nAD  - Department of Biotechnology, School of Advanced Technologies in Medicine, Shahid  Beheshti University of Medical Sciences , Tehran, Iran.\\nFAU - Keshavarz-Fathi, Mahsa\\nAU  - Keshavarz-Fathi M\\nAD  - Cancer Immunology Project (CIP), Universal Scientific Education & Research Network  (USERN) , Tehran, Iran.\\nAD  - School of Medicine, Tehran University of Medical Sciences , Tehran, Iran.\\nFAU - Ghanbarian, Hossein\\nAU  - Ghanbarian H\\nAD  - Cellular and Molecular Biology Research Center, Shahid Beheshti University of  Medical Sciences , Tehran, Iran.\\nAD  - Department of Biotechnology, School of Advanced Technologies in Medicine, Shahid  Beheshti University of Medical Sciences , Tehran, Iran.\\nAD  - Department of Tissue Engineering and Applied Cell Sciences, School of Advanced  Technologies in Medicine, Shahid Beheshti University of Medical Sciences , Tehran,  Iran.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20200413\\nPL  - England\\nTA  - Expert Opin Drug Deliv\\nJT  - Expert opinion on drug delivery\\nJID - 101228421\\nRN  - EC 3.1.- (Endonucleases)\\nRN  - EC 3.1.- (Transcription Activator-Like Effector Nucleases)\\nSB  - IM\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - Endonucleases/genetics\\nMH  - Female\\nMH  - *Gene Editing\\nMH  - Humans\\nMH  - Papillomavirus Infections/*complications/genetics\\nMH  - Transcription Activator-Like Effector Nucleases/genetics\\nMH  - Uterine Cervical Neoplasms/genetics/*virology\\nOTO - NOTNLM\\nOT  - *Drug delivery systems\\nOT  - *cervical cancer\\nOT  - *genome-editing tools\\nOT  - *nanoparticles\\nOT  - *viral vectors\\nEDAT- 2020/04/14 06:00\\nMHDA- 2021/01/13 06:00\\nCRDT- 2020/04/14 06:00\\nPHST- 2020/04/14 06:00 [pubmed]\\nPHST- 2021/01/13 06:00 [medline]\\nPHST- 2020/04/14 06:00 [entrez]\\nAID - 10.1080/17425247.2020.1747429 [doi]\\nPST - ppublish\\nSO  - Expert Opin Drug Deliv. 2020 Jun;17(6):753-766. doi: 10.1080/17425247.2020.1747429.  Epub 2020 Apr 13.\\n\\nPMID- 30102386\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190627\\nLR  - 20190627\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 46\\nIP  - 18\\nDP  - 2018 Oct 12\\nTI  - Molecular organization of the type II-A CRISPR adaptation module and its interaction  with Cas9 via Csn2.\\nPG  - 9805-9815\\nLID - 10.1093/nar/gky702 [doi]\\nAB  - Clustered regularly interspaced short palindromic repeats (CRISPRs) and  CRISPR-associated (Cas) proteins provide microbial adaptive immunity against  invading foreign nucleic acids. In type II-A CRISPR-Cas systems, the Cas1-Cas2  integrase complex and the subtype-specific Csn2 comprise the CRISPR adaptation  module, which cooperates with the Cas9 nuclease effector for spacer selection. Here,  we report the molecular organization of the Streptococcus pyogenes type II-A CRISPR  adaptation module and its interaction with Cas9 via Csn2. We determined the crystal  structure of S. pyogenes type II-A Cas2. Chromatographic and calorimetric analyses  revealed the stoichiometry and topology of the type II-A adaptation module composed  of Cas1, Cas2 and Csn2. We also demonstrated that Cas9 interacts with Csn2 in a  direct and stoichiometric manner. Our results reveal a network of molecular  interactions among type II-A Cas proteins and highlight the role of Csn2 in  coordinating Cas components involved in the adaptation and interference stages of  CRISPR-mediated immunity.\\nFAU - Ka, Donghyun\\nAU  - Ka D\\nAD  - Department of Agricultural Biotechnology, Seoul National University, Seoul 08826,  Korea.\\nFAU - Jang, Dong Man\\nAU  - Jang DM\\nAD  - Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National  University, Seoul 08826, Korea.\\nFAU - Han, Byung Woo\\nAU  - Han BW\\nAD  - Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National  University, Seoul 08826, Korea.\\nFAU - Bae, Euiyoung\\nAU  - Bae E\\nAD  - Department of Agricultural Biotechnology, Seoul National University, Seoul 08826,  Korea.\\nAD  - Research Institute of Agriculture and Life Sciences, Seoul National University,  Seoul 08826, Korea.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - 0 (Nucleic Acids)\\nRN  - EC 2.7.7.- (Integrases)\\nSB  - IM\\nMH  - Adaptation, Physiological/genetics\\nMH  - Adaptive Immunity/*genetics\\nMH  - CRISPR-Associated Proteins/*chemistry/genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Crystallography, X-Ray\\nMH  - Integrases/*chemistry/genetics\\nMH  - Nucleic Acids/chemistry/genetics/immunology\\nMH  - Streptococcus pyogenes/enzymology/genetics\\nPMC - PMC6182153\\nEDAT- 2018/08/14 06:00\\nMHDA- 2019/06/30 06:00\\nCRDT- 2018/08/14 06:00\\nPHST- 2018/05/30 00:00 [received]\\nPHST- 2018/07/24 00:00 [accepted]\\nPHST- 2018/08/14 06:00 [pubmed]\\nPHST- 2019/06/30 06:00 [medline]\\nPHST- 2018/08/14 06:00 [entrez]\\nAID - 5068242 [pii]\\nAID - gky702 [pii]\\nAID - 10.1093/nar/gky702 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2018 Oct 12;46(18):9805-9815. doi: 10.1093/nar/gky702.\\n\\nPMID- 32076642\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20201021\\nLR  - 20201021\\nIS  - 2375-2548 (Electronic)\\nIS  - 2375-2548 (Linking)\\nVI  - 6\\nIP  - 6\\nDP  - 2020 Feb\\nTI  - Coupling Cas9 to artificial inhibitory domains enhances CRISPR-Cas9 target  specificity.\\nPG  - eaay0187\\nLID - 10.1126/sciadv.aay0187 [doi]\\nLID - eaay0187\\nAB  - The limited target specificity of CRISPR-Cas nucleases poses a challenge with  respect to their application in research and therapy. Here, we present a simple and  original strategy to enhance the specificity of CRISPR-Cas9 genome editing by  coupling Cas9 to artificial inhibitory domains. Applying a combination of  mathematical modeling and experiments, we first determined how CRISPR-Cas9 activity  profiles relate to Cas9 specificity. We then used artificially weakened anti-CRISPR  (Acr) proteins either coexpressed with or directly fused to Cas9 to fine-tune its  activity toward selected levels, thereby achieving an effective kinetic insulation  of ON- and OFF-target editing events. We demonstrate highly specific genome editing  in mammalian cells using diverse single-guide RNAs prone to potent OFF-targeting.  Last, we show that our strategy is compatible with different modes of delivery,  including transient transfection and adeno-associated viral vectors. Together, we  provide a highly versatile approach to reduce CRISPR-Cas OFF-target effects via  kinetic insulation.\\nCI  - Copyright Â© 2020 The Authors, some rights reserved; exclusive licensee American  Association for the Advancement of Science. No claim to original U.S. Government  Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0  (CC BY-NC).\\nFAU - Aschenbrenner, Sabine\\nAU  - Aschenbrenner S\\nAUID- ORCID: 0000-0002-2260-3254\\nAD  - Synthetic Biology Group, Institute for Pharmacy and Molecular Biotechnology (IPMB)  and Center for Quantitative Analysis of Molecular and Cellular Biosystems  (BioQuant), University of Heidelberg, Heidelberg 69120, Germany.\\nAD  - Division of Chromatin Networks, German Cancer Research Center (DKFZ), Heidelberg  69120, Germany.\\nAD  - Digital Health Center, Berlin Institute of Health (BIH) and CharitÃ©, Berlin 10178,  Germany.\\nFAU - Kallenberger, Stefan M\\nAU  - Kallenberger SM\\nAUID- ORCID: 0000-0003-3783-0246\\nAD  - Digital Health Center, Berlin Institute of Health (BIH) and CharitÃ©, Berlin 10178,  Germany.\\nAD  - Health Data Science Unit, University Hospital Heidelberg, Heidelberg 69120, Germany.\\nFAU - Hoffmann, Mareike D\\nAU  - Hoffmann MD\\nAD  - Synthetic Biology Group, Institute for Pharmacy and Molecular Biotechnology (IPMB)  and Center for Quantitative Analysis of Molecular and Cellular Biosystems  (BioQuant), University of Heidelberg, Heidelberg 69120, Germany.\\nAD  - Division of Chromatin Networks, German Cancer Research Center (DKFZ), Heidelberg  69120, Germany.\\nFAU - Huck, Adrian\\nAU  - Huck A\\nAUID- ORCID: 0000-0003-1151-3891\\nAD  - Synthetic Biology Group, Institute for Pharmacy and Molecular Biotechnology (IPMB)  and Center for Quantitative Analysis of Molecular and Cellular Biosystems  (BioQuant), University of Heidelberg, Heidelberg 69120, Germany.\\nFAU - Eils, Roland\\nAU  - Eils R\\nAUID- ORCID: 0000-0002-0034-4036\\nAD  - Digital Health Center, Berlin Institute of Health (BIH) and CharitÃ©, Berlin 10178,  Germany.\\nAD  - Health Data Science Unit, University Hospital Heidelberg, Heidelberg 69120, Germany.\\nFAU - Niopek, Dominik\\nAU  - Niopek D\\nAUID- ORCID: 0000-0001-7479-530X\\nAD  - Synthetic Biology Group, Institute for Pharmacy and Molecular Biotechnology (IPMB)  and Center for Quantitative Analysis of Molecular and Cellular Biosystems  (BioQuant), University of Heidelberg, Heidelberg 69120, Germany.\\nAD  - Health Data Science Unit, University Hospital Heidelberg, Heidelberg 69120, Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200205\\nTA  - Sci Adv\\nJT  - Science advances\\nJID - 101653440\\nRN  - 0 (Recombinant Fusion Proteins)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nSB  - IM\\nMH  - CRISPR-Associated Protein 9/chemistry/*genetics/metabolism\\nMH  - *CRISPR-Cas Systems\\nMH  - Cell Line\\nMH  - Enzyme Activation\\nMH  - *Gene Editing\\nMH  - *Gene Targeting/methods\\nMH  - Genes, Reporter\\nMH  - Genetic Loci\\nMH  - Humans\\nMH  - Kinetics\\nMH  - Models, Theoretical\\nMH  - *Recombinant Fusion Proteins\\nMH  - Substrate Specificity\\nPMC - PMC7002122\\nEDAT- 2020/02/23 06:00\\nMHDA- 2020/10/22 06:00\\nCRDT- 2020/02/21 06:00\\nPHST- 2019/05/13 00:00 [received]\\nPHST- 2019/11/22 00:00 [accepted]\\nPHST- 2020/02/21 06:00 [entrez]\\nPHST- 2020/02/23 06:00 [pubmed]\\nPHST- 2020/10/22 06:00 [medline]\\nAID - aay0187 [pii]\\nAID - 10.1126/sciadv.aay0187 [doi]\\nPST - epublish\\nSO  - Sci Adv. 2020 Feb 5;6(6):eaay0187. doi: 10.1126/sciadv.aay0187. eCollection 2020  Feb.\\n\\nPMID- 28101863\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170619\\nLR  - 20170619\\nIS  - 0306-0225 (Print)\\nIS  - 0306-0225 (Linking)\\nVI  - 82\\nDP  - 2017\\nTI  - Lessons from Animal Models of Cytoplasmic Intermediate Filament Proteins.\\nPG  - 171-230\\nLID - 10.1007/978-3-319-49674-0_7 [doi]\\nAB  - Cytoplasmic intermediate filaments (IFs) represent a major cytoskeletal network  contributing to cell shape, adhesion and migration as well as to tissue resilience  and renewal in numerous bilaterians, including mammals. The observation that IFs are  dispensable in cultured mammalian cells, but cause tissue-specific, life-threatening  disorders, has pushed the need to investigate their function in vivo. In keeping  with human disease, the deletion or mutation of murine IF genes resulted in highly  specific pathologies. Epidermal keratins, together with desmin, are essential to  protect corresponding tissues against mechanical force but also participate in  stabilizing cell adhesion and in inflammatory signalling. Surprisingly, other IF  proteins contribute to tissue integrity to a much lesser extent than anticipated,  pointing towards their role in stress situations. In support, the overexpression of  small chaperones or the interference with inflammatory signalling in several  settings has been shown to rescue severe tissue pathologies that resulted from the  expression of mutant IF proteins. It stills remains an open issue whether the wide  range of IF disorders share similar pathomechanisms. Moreover, we lack an  understanding how IF proteins participate in signalling processes. Now, with a large  number of mouse models in hand, the next challenge will be to develop organotypic  cell culture models to dissect pathomechanisms at the molecular level, to employ  Crispr/Cas-mediated genome engineering to optimize models and, finally, to combine  available animal models with medicinal chemistry for the development of molecular  therapies.\\nFAU - Bouameur, Jamal-Eddine\\nAU  - Bouameur JE\\nAD  - SÃ¤chsischer Inkubator fÃ¼r Klinische Translation (SIKT) & Institute of Biology  Division of Cell and Developmental Biology, University of Leipzig,  Philipp-Rosenthal-StraÃŸe 55, 04103, Leipzig, Germany.  jamal-eddine.bouameur@uni-leipzig.de.\\nFAU - Magin, Thomas M\\nAU  - Magin TM\\nAD  - SÃ¤chsischer Inkubator fÃ¼r Klinische Translation (SIKT) & Institute of Biology  Division of Cell and Developmental Biology, University of Leipzig,  Philipp-Rosenthal-StraÃŸe 55, 04103, Leipzig, Germany. thomas.magin@uni-leipzig.de.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nPL  - United States\\nTA  - Subcell Biochem\\nJT  - Sub-cellular biochemistry\\nJID - 0316571\\nRN  - 0 (Intermediate Filament Proteins)\\nSB  - IM\\nMH  - Animals\\nMH  - Connective Tissue Diseases\\nMH  - Cytoplasm\\nMH  - Disease Models, Animal\\nMH  - Humans\\nMH  - *Intermediate Filament Proteins\\nOTO - NOTNLM\\nOT  - Animal models\\nOT  - Desmin\\nOT  - Epithelia\\nOT  - Filensin\\nOT  - GFAP\\nOT  - Intermediate filaments\\nOT  - Keratins\\nOT  - Nestin\\nOT  - Neurofilaments\\nOT  - Peripherin\\nOT  - Phakinin\\nOT  - Syncoilin\\nOT  - Synemin\\nOT  - Vimentin\\nOT  - Î±-internexin\\nEDAT- 2017/01/20 06:00\\nMHDA- 2017/06/20 06:00\\nCRDT- 2017/01/20 06:00\\nPHST- 2017/01/20 06:00 [entrez]\\nPHST- 2017/01/20 06:00 [pubmed]\\nPHST- 2017/06/20 06:00 [medline]\\nAID - 10.1007/978-3-319-49674-0_7 [doi]\\nPST - ppublish\\nSO  - Subcell Biochem. 2017;82:171-230. doi: 10.1007/978-3-319-49674-0_7.\\n\\nPMID- 30619231\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 1664-302X (Print)\\nIS  - 1664-302X (Electronic)\\nIS  - 1664-302X (Linking)\\nVI  - 9\\nDP  - 2018\\nTI  - Iron Regulation in Clostridioides difficile.\\nPG  - 3183\\nLID - 10.3389/fmicb.2018.03183 [doi]\\nLID - 3183\\nAB  - The response to iron limitation of several bacteria is regulated by the ferric  uptake regulator (Fur). The Fur-regulated transcriptional, translational and  metabolic networks of the Gram-positive, pathogen Clostridioides difficile were  investigated by a combined RNA sequencing, proteomic, metabolomic and electron  microscopy approach. At high iron conditions (15 Î¼M) the C. difficile fur mutant  displayed a growth deficiency compared to wild type C. difficile cells. Several iron  and siderophore transporter genes were induced by Fur during low iron (0.2 Î¼M)  conditions. The major adaptation to low iron conditions was observed for the central  energy metabolism. Most ferredoxin-dependent amino acid fermentations were  significantly down regulated (had, etf, acd, grd, trx, bdc, hbd). The substrates of  these pathways phenylalanine, leucine, glycine and some intermediates  (phenylpyruvate, 2-oxo-isocaproate, 3-hydroxy-butyryl-CoA, crotonyl-CoA)  accumulated, while end products like isocaproate and butyrate were found reduced.  Flavodoxin (fldX) formation and riboflavin biosynthesis (rib) were enhanced, most  likely to replace the missing ferredoxins. Proline reductase (prd), the  corresponding ion pumping RNF complex (rnf) and the reaction product 5-aminovalerate  were significantly enhanced. An ATP forming ATPase (atpCDGAHFEB) of the  F(0)F(1)-type was induced while the formation of a ATP-consuming, proton-pumping  V-type ATPase (atpDBAFCEKI) was decreased. The [Fe-S] enzyme-dependent pyruvate  formate lyase (pfl), formate dehydrogenase (fdh) and hydrogenase (hyd) branch of  glucose utilization and glycogen biosynthesis (glg) were significantly reduced,  leading to an accumulation of glucose and pyruvate. The formation of [Fe-S] enzyme  carbon monoxide dehydrogenase (coo) was inhibited. The fur mutant showed an  increased sensitivity to vancomycin and polymyxin B. An intensive remodeling of the  cell wall was observed, Polyamine biosynthesis (spe) was induced leading to an  accumulation of spermine, spermidine, and putrescine. The fur mutant lost most of  its flagella and motility. Finally, the CRISPR/Cas and a prophage encoding operon  were downregulated. Fur binding sites were found upstream of around 20 of the  regulated genes. Overall, adaptation to low iron conditions in C. difficile focused  on an increase of iron import, a significant replacement of iron requiring metabolic  pathways and the restructuring of the cell surface for protection during the complex  adaptation phase and was only partly directly regulated by Fur.\\nFAU - Berges, Mareike\\nAU  - Berges M\\nAD  - Braunschweig Integrated Centre of Systems Biology (BRICS), Technische UniversitÃ¤t  Braunschweig, Braunschweig, Germany.\\nFAU - Michel, Annika-Marisa\\nAU  - Michel AM\\nAD  - Braunschweig Integrated Centre of Systems Biology (BRICS), Technische UniversitÃ¤t  Braunschweig, Braunschweig, Germany.\\nFAU - Lassek, Christian\\nAU  - Lassek C\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - Nuss, Aaron M\\nAU  - Nuss AM\\nAD  - Department of Molecular Infection Biology, Helmholtz Centre for Infection Research  (HZI), Braunschweig, Germany.\\nFAU - Beckstette, Michael\\nAU  - Beckstette M\\nAD  - Department of Molecular Infection Biology, Helmholtz Centre for Infection Research  (HZI), Braunschweig, Germany.\\nFAU - Dersch, Petra\\nAU  - Dersch P\\nAD  - Department of Molecular Infection Biology, Helmholtz Centre for Infection Research  (HZI), Braunschweig, Germany.\\nFAU - Riedel, Katharina\\nAU  - Riedel K\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - Sievers, Susanne\\nAU  - Sievers S\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - Becher, DÃ¶rte\\nAU  - Becher D\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - Otto, Andreas\\nAU  - Otto A\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - MaaÃŸ, Sandra\\nAU  - MaaÃŸ S\\nAD  - Center for Functional Genomics of Microbes (CFGM), Institute of Microbiology,  University of Greifswald, Greifswald, Germany.\\nFAU - Rohde, Manfred\\nAU  - Rohde M\\nAD  - Central Facility for Microscopy, Helmholtz Centre for Infection Research (HZI),  Braunschweig, Germany.\\nFAU - Eckweiler, Denitsa\\nAU  - Eckweiler D\\nAD  - Braunschweig Integrated Centre of Systems Biology (BRICS), Technische UniversitÃ¤t  Braunschweig, Braunschweig, Germany.\\nFAU - Borrero-de AcuÃ±a, Jose M\\nAU  - Borrero-de AcuÃ±a JM\\nAD  - Institute of Microbiology, Technische UniversitÃ¤t Braunschweig, Braunschweig,  Germany.\\nFAU - Jahn, Martina\\nAU  - Jahn M\\nAD  - Institute of Microbiology, Technische UniversitÃ¤t Braunschweig, Braunschweig,  Germany.\\nFAU - Neumann-Schaal, Meina\\nAU  - Neumann-Schaal M\\nAD  - Leibniz-Institute DSMZ-German Collection of Microorganisms and Cell Cultures,  Braunschweig, Germany.\\nFAU - Jahn, Dieter\\nAU  - Jahn D\\nAD  - Braunschweig Integrated Centre of Systems Biology (BRICS), Technische UniversitÃ¤t  Braunschweig, Braunschweig, Germany.\\nAD  - Institute of Microbiology, Technische UniversitÃ¤t Braunschweig, Braunschweig,  Germany.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20181224\\nTA  - Front Microbiol\\nJT  - Frontiers in microbiology\\nJID - 101548977\\nPMC - PMC6311696\\nOTO - NOTNLM\\nOT  - Fur\\nOT  - cell wall\\nOT  - iron regulation\\nOT  - iron transport\\nOT  - metabolism\\nOT  - polyamine\\nEDAT- 2019/01/09 06:00\\nMHDA- 2019/01/09 06:01\\nCRDT- 2019/01/09 06:00\\nPHST- 2018/09/24 00:00 [received]\\nPHST- 2018/12/10 00:00 [accepted]\\nPHST- 2019/01/09 06:00 [entrez]\\nPHST- 2019/01/09 06:00 [pubmed]\\nPHST- 2019/01/09 06:01 [medline]\\nAID - 10.3389/fmicb.2018.03183 [doi]\\nPST - epublish\\nSO  - Front Microbiol. 2018 Dec 24;9:3183. doi: 10.3389/fmicb.2018.03183. eCollection  2018.\\n\\nPMID- 33139880\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210316\\nLR  - 20210914\\nIS  - 2058-5276 (Electronic)\\nIS  - 2058-5276 (Linking)\\nVI  - 6\\nIP  - 1\\nDP  - 2021 Jan\\nTI  - Roles of bacteriophages, plasmids and CRISPR immunity in microbial community  dynamics revealed using time-series integrated meta-omics.\\nPG  - 123-135\\nLID - 10.1038/s41564-020-00794-8 [doi]\\nAB  - Viruses and plasmids (invasive mobile genetic elements (iMGEs)) have important roles  in shaping microbial communities, but their dynamic interactions with CRISPR-based  immunity remain unresolved. We analysed generation-resolved iMGE-host dynamics  spanning one and a half years in a microbial consortium from a biological wastewater  treatment plant using integrated meta-omics. We identified 31 bacterial  metagenome-assembled genomes encoding complete CRISPR-Cas systems and their  corresponding iMGEs. CRISPR-targeted plasmids outnumbered their bacteriophage  counterparts by at least fivefold, highlighting the importance of CRISPR-mediated  defence against plasmids. Linear modelling of our time-series data revealed that the  variation in plasmid abundance over time explained more of the observed community  dynamics than phages. Community-scale CRISPR-based plasmid-host and phage-host  interaction networks revealed an increase in CRISPR-mediated interactions coinciding  with a decrease in the dominant \\'Candidatus Microthrix parvicella\\' population.  Protospacers were enriched in sequences targeting genes involved in the transmission  of iMGEs. Understanding the factors shaping the fitness of specific populations is  necessary to devise control strategies for undesirable species and to predict or  explain community-wide phenotypes.\\nFAU - MartÃ\\xadnez Arbas, Susana\\nAU  - MartÃ\\xadnez Arbas S\\nAUID- ORCID: 0000-0001-7309-8190\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Narayanasamy, Shaman\\nAU  - Narayanasamy S\\nAUID- ORCID: 0000-0002-5793-6235\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nAD  - Megeno S.A., Esch-sur-Alzette, Luxembourg.\\nFAU - Herold, Malte\\nAU  - Herold M\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Lebrun, Laura A\\nAU  - Lebrun LA\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Hoopmann, Michael R\\nAU  - Hoopmann MR\\nAUID- ORCID: 0000-0001-7029-7792\\nAD  - Institute for Systems Biology, Seattle, WA, USA.\\nFAU - Li, Sujun\\nAU  - Li S\\nAUID- ORCID: 0000-0001-5233-2005\\nAD  - School of Informatics, Computing and Engineering, Indiana University, Bloomington,  IN, USA.\\nFAU - Lam, Tony J\\nAU  - Lam TJ\\nAD  - School of Informatics, Computing and Engineering, Indiana University, Bloomington,  IN, USA.\\nFAU - Kunath, BenoÃ®t J\\nAU  - Kunath BJ\\nAUID- ORCID: 0000-0002-3356-8562\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Hicks, Nathan D\\nAU  - Hicks ND\\nAD  - TGen North, Flagstaff, AZ, USA.\\nAD  - Harvard T.H. Chan School of Public Health, Harvard University, Boston, MA, USA.\\nFAU - Liu, Cindy M\\nAU  - Liu CM\\nAUID- ORCID: 0000-0003-4537-173X\\nAD  - TGen North, Flagstaff, AZ, USA.\\nAD  - Department of Environmental and Occupational Health, Miken Institute School of  Public Health, George Washington University, Washington, DC, USA.\\nFAU - Price, Lance B\\nAU  - Price LB\\nAD  - TGen North, Flagstaff, AZ, USA.\\nAD  - Department of Environmental and Occupational Health, Miken Institute School of  Public Health, George Washington University, Washington, DC, USA.\\nFAU - Laczny, Cedric C\\nAU  - Laczny CC\\nAUID- ORCID: 0000-0002-1100-1282\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Gillece, John D\\nAU  - Gillece JD\\nAD  - TGen North, Flagstaff, AZ, USA.\\nFAU - Schupp, James M\\nAU  - Schupp JM\\nAD  - TGen North, Flagstaff, AZ, USA.\\nFAU - Keim, Paul S\\nAU  - Keim PS\\nAUID- ORCID: 0000-0003-0089-5820\\nAD  - TGen North, Flagstaff, AZ, USA.\\nAD  - The Pathogen and Microbiome Institute, Northern Arizona University, Flagstaff, AZ,  USA.\\nFAU - Moritz, Robert L\\nAU  - Moritz RL\\nAUID- ORCID: 0000-0002-3216-9447\\nAD  - Institute for Systems Biology, Seattle, WA, USA.\\nFAU - Faust, Karoline\\nAU  - Faust K\\nAUID- ORCID: 0000-0001-7129-2803\\nAD  - Laboratory of Molecular Bacteriology, KU Leuven, Leuven, Belgium.\\nFAU - Tang, Haixu\\nAU  - Tang H\\nAD  - School of Informatics, Computing and Engineering, Indiana University, Bloomington,  IN, USA.\\nFAU - Ye, Yuzhen\\nAU  - Ye Y\\nAD  - School of Informatics, Computing and Engineering, Indiana University, Bloomington,  IN, USA.\\nFAU - Skupin, Alexander\\nAU  - Skupin A\\nAUID- ORCID: 0000-0002-8955-8304\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nAD  - Department of Neuroscience, University of California, La Jolla, CA, USA.\\nFAU - May, Patrick\\nAU  - May P\\nAUID- ORCID: 0000-0001-8698-3770\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nFAU - Muller, Emilie E L\\nAU  - Muller EEL\\nAUID- ORCID: 0000-0002-3750-2966\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg.\\nAD  - Department of Microbiology, Genomics and the Environment, UMR 7156 UNISTRA-CNRS,  UniversitÃ© de Strasbourg, Strasbourg, France.\\nFAU - Wilmes, Paul\\nAU  - Wilmes P\\nAUID- ORCID: 0000-0002-6478-2924\\nAD  - Luxembourg Centre for Systems Biomedicine, University of Luxembourg,  Esch-sur-Alzette, Luxembourg. paul.wilmes@uni.lu.\\nAD  - Department of Life Sciences and Medicine, Faculty of Science, Technology and  Medicine, University of Luxembourg, Esch-sur-Alzette, Luxembourg.  paul.wilmes@uni.lu.\\nLA  - eng\\nGR  - R01 AI108888/AI/NIAID NIH HHS/United States\\nGR  - R01 AI143254/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20201102\\nTA  - Nat Microbiol\\nJT  - Nature microbiology\\nJID - 101674869\\nRN  - 0 (Sewage)\\nSB  - IM\\nMH  - Bacteria/*genetics/virology\\nMH  - Bacteriophages/*genetics\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/genetics\\nMH  - Genome, Bacterial/genetics\\nMH  - Metagenome/genetics\\nMH  - Microbial Consortia/genetics\\nMH  - Microbial Interactions/*genetics/physiology\\nMH  - Plasmids/*genetics\\nMH  - Sewage/microbiology\\nMH  - Water Purification\\nPMC - PMC7752763\\nCOIS- The authors declare no competing interests.\\nEDAT- 2020/11/04 06:00\\nMHDA- 2021/03/17 06:00\\nCRDT- 2020/11/03 05:45\\nPHST- 2019/12/08 00:00 [received]\\nPHST- 2020/09/11 00:00 [accepted]\\nPHST- 2020/11/04 06:00 [pubmed]\\nPHST- 2021/03/17 06:00 [medline]\\nPHST- 2020/11/03 05:45 [entrez]\\nAID - 10.1038/s41564-020-00794-8 [pii]\\nAID - 794 [pii]\\nAID - 10.1038/s41564-020-00794-8 [doi]\\nPST - ppublish\\nSO  - Nat Microbiol. 2021 Jan;6(1):123-135. doi: 10.1038/s41564-020-00794-8. Epub 2020 Nov  2.\\n\\nPMID- 30709766\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190814\\nLR  - 20190814\\nIS  - 1873-2976 (Electronic)\\nIS  - 0960-8524 (Linking)\\nVI  - 278\\nDP  - 2019 Apr\\nTI  - Thermozymes: Adaptive strategies and tools for their biotechnological applications.\\nPG  - 372-382\\nLID - S0960-8524(19)30108-7 [pii]\\nLID - 10.1016/j.biortech.2019.01.088 [doi]\\nAB  - In today\\'s scenario of global climate change, there is a colossal demand for  sustainable industrial processes and enzymes from thermophiles. Plausibly,  thermozymes are an important toolkit, as they are known to be polyextremophilic in  nature. Small genome size and diverse molecular conformational modifications have  been implicated in devising adaptive strategies. Besides, the utilization of  chemical technology and gene editing attributions according to mechanical  necessities are the additional key factor for efficacious bioprocess development.  Microbial thermozymes have been extensively used in waste management, biofuel, food,  paper, detergent, medicinal and pharmaceutical industries. To understand the  strength of enzymes at higher temperatures different models utilize X-ray structures  of thermostable proteins, machine learning calculations, neural networks, but  unified adaptive measures are yet to be totally comprehended. The present review  provides a recent updates on thermozymes and various interdisciplinary applications  including the aspects of thermophiles bioengineering utilizing synthetic biology and  gene editing tools.\\nCI  - Copyright Â© 2019 Elsevier Ltd. All rights reserved.\\nFAU - Kumar, Sumit\\nAU  - Kumar S\\nAD  - Enzyme and Microbial Biochemistry Laboratory, Department of Chemistry, Indian  Institute of Technology Delhi, New Delhi 110016, India.\\nFAU - Dangi, Arun K\\nAU  - Dangi AK\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nFAU - Shukla, Pratyoosh\\nAU  - Shukla P\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nFAU - Baishya, Debabrat\\nAU  - Baishya D\\nAD  - Department of Bioengineering and Technology, Institute of Science and Technology,  Gauhati University, Guwahati 781014, Assam, India.\\nFAU - Khare, Sunil K\\nAU  - Khare SK\\nAD  - Enzyme and Microbial Biochemistry Laboratory, Department of Chemistry, Indian  Institute of Technology Delhi, New Delhi 110016, India. Electronic address:  skhare@rocketmail.com.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20190122\\nPL  - England\\nTA  - Bioresour Technol\\nJT  - Bioresource technology\\nJID - 9889523\\nRN  - 0 (Biofuels)\\nMH  - Biofuels\\nMH  - *Biotechnology\\nMH  - Gene Editing\\nMH  - Hot Temperature\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - Gene editing\\nOT  - Synthetic biology\\nOT  - Thermozymes\\nOT  - Waste management\\nEDAT- 2019/02/03 06:00\\nMHDA- 2019/08/15 06:00\\nCRDT- 2019/02/03 06:00\\nPHST- 2018/11/14 00:00 [received]\\nPHST- 2019/01/19 00:00 [revised]\\nPHST- 2019/01/21 00:00 [accepted]\\nPHST- 2019/02/03 06:00 [pubmed]\\nPHST- 2019/08/15 06:00 [medline]\\nPHST- 2019/02/03 06:00 [entrez]\\nAID - S0960-8524(19)30108-7 [pii]\\nAID - 10.1016/j.biortech.2019.01.088 [doi]\\nPST - ppublish\\nSO  - Bioresour Technol. 2019 Apr;278:372-382. doi: 10.1016/j.biortech.2019.01.088. Epub  2019 Jan 22.\\n\\nPMID- 27540473\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20160819\\nLR  - 20181113\\nIS  - 2046-1402 (Print)\\nIS  - 2046-1402 (Electronic)\\nIS  - 2046-1402 (Linking)\\nVI  - 5\\nDP  - 2016\\nTI  - Mapping paths: new approaches to dissect eukaryotic signaling circuitry.\\nLID - F1000 Faculty Rev-1853 [pii]\\nLID - 10.12688/f1000research.8818.1 [doi]\\nAB  - Eukaryotic cells are precisely \"wired\" to coordinate changes in external and  intracellular signals with corresponding adjustments in the output of complex and  often interconnected signaling pathways. These pathways are critical in  understanding cellular growth and function, and several experimental trends are  emerging with applicability toward more fully describing the composition and  topology of eukaryotic signaling networks. In particular, recent studies have  implemented CRISPR/Cas-based screens in mouse and human cell lines for genes  involved in various cell growth and disease phenotypes. Proteomic methods using mass  spectrometry have enabled quantitative and dynamic profiling of protein  interactions, revealing previously undiscovered complexes and allele-specific  protein interactions. Methods for the single-cell study of protein localization and  gene expression have been integrated with computational analyses to provide insight  into cell signaling in yeast and metazoans. In this review, we present an overview  of exemplary studies using the above approaches, relevant for the analysis of cell  signaling and indeed, more broadly, for many modern biological applications.\\nFAU - Mutlu, Nebibe\\nAU  - Mutlu N\\nAD  - Department of Molecular, Cellular and Developmental Biology, University of Michigan,  Ann Arbor, MI, USA.\\nFAU - Kumar, Anuj\\nAU  - Kumar A\\nAD  - Department of Molecular, Cellular and Developmental Biology, University of Michigan,  Ann Arbor, MI, USA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20160727\\nTA  - F1000Res\\nJT  - F1000Research\\nJID - 101594320\\nPMC - PMC4965690\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - cell signaling\\nOT  - eukaryotic signaling networks\\nOT  - protein interactions\\nOT  - proteomics\\nOT  - signaling pathways\\nOT  - yeast\\nCOIS- Competing interests: The authors declare that they have no competing interests. No  competing interests were disclosed.\\nEDAT- 2016/08/20 06:00\\nMHDA- 2016/08/20 06:01\\nCRDT- 2016/08/20 06:00\\nPHST- 2016/07/21 00:00 [accepted]\\nPHST- 2016/08/20 06:00 [entrez]\\nPHST- 2016/08/20 06:00 [pubmed]\\nPHST- 2016/08/20 06:01 [medline]\\nAID - F1000 Faculty Rev-1853 [pii]\\nAID - 10.12688/f1000research.8818.1 [doi]\\nPST - epublish\\nSO  - F1000Res. 2016 Jul 27;5:F1000 Faculty Rev-1853. doi: 10.12688/f1000research.8818.1.  eCollection 2016.\\n\\nPMID- 30198342\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200706\\nLR  - 20200706\\nIS  - 1549-7801 (Electronic)\\nIS  - 0738-8551 (Linking)\\nVI  - 39\\nIP  - 1\\nDP  - 2019 Feb\\nTI  - Bioremediation through microbes: systems biology and metabolic engineering approach.\\nPG  - 79-98\\nLID - 10.1080/07388551.2018.1500997 [doi]\\nAB  - Today, environmental pollution is a serious problem, and bioremediation can play an  important role in cleaning contaminated sites. Remediation strategies, such as  chemical and physical approaches, are not enough to mitigate pollution problems  because of the continuous generation of novel recalcitrant pollutants due to  anthropogenic activities. Bioremediation using microbes is an eco-friendly and  socially acceptable alternative to conventional remediation approaches. Many  microbes with a bioremediation potential have been isolated and characterized but,  in many cases, cannot completely degrade the targeted pollutant or are ineffective  in situations with mixed wastes. This review envisages advances in systems biology  (SB), which enables the analysis of microbial behavior at a community level under  different environmental stresses. By applying a SB approach, crucial preliminary  information can be obtained for metabolic engineering (ME) of microbes for their  enhanced bioremediation capabilities. This review also highlights the integrated SB  and ME tools and techniques for bioremediation purposes.\\nFAU - Dangi, Arun Kumar\\nAU  - Dangi AK\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nFAU - Sharma, Babita\\nAU  - Sharma B\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nFAU - Hill, Russell T\\nAU  - Hill RT\\nAD  - Institute of Marine and Environmental Technology, University of Maryland Center for  Environmental Science, Baltimore, MD, USA.\\nFAU - Shukla, Pratyoosh\\nAU  - Shukla P\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University, Rohtak, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20180909\\nPL  - England\\nTA  - Crit Rev Biotechnol\\nJT  - Critical reviews in biotechnology\\nJID - 8505177\\nRN  - 0 (Environmental Pollutants)\\nSB  - IM\\nMH  - *Biodegradation, Environmental\\nMH  - Biotechnology\\nMH  - CRISPR-Cas Systems\\nMH  - Computational Biology\\nMH  - Environmental Pollutants\\nMH  - Environmental Pollution\\nMH  - Gene Editing/methods\\nMH  - Gene Expression Regulation\\nMH  - Genomics\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics/physiology\\nMH  - Microbiota/genetics/*physiology\\nMH  - Multigene Family\\nMH  - *Systems Biology\\nMH  - Transcriptome\\nOTO - NOTNLM\\nOT  - Bioremediation\\nOT  - CRISPR-Cas\\nOT  - gene editing\\nOT  - metabolic engineering\\nOT  - systems biology\\nEDAT- 2018/09/11 06:00\\nMHDA- 2020/07/07 06:00\\nCRDT- 2018/09/11 06:00\\nPHST- 2018/09/11 06:00 [pubmed]\\nPHST- 2020/07/07 06:00 [medline]\\nPHST- 2018/09/11 06:00 [entrez]\\nAID - 10.1080/07388551.2018.1500997 [doi]\\nPST - ppublish\\nSO  - Crit Rev Biotechnol. 2019 Feb;39(1):79-98. doi: 10.1080/07388551.2018.1500997. Epub  2018 Sep 9.\\n\\nPMID- 28793788\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180817\\nLR  - 20180817\\nIS  - 1549-7801 (Electronic)\\nIS  - 0738-8551 (Linking)\\nVI  - 38\\nIP  - 3\\nDP  - 2018 May\\nTI  - Improvements in algal lipid production: a systems biology and gene editing approach.\\nPG  - 369-385\\nLID - 10.1080/07388551.2017.1356803 [doi]\\nAB  - BACKGROUND: In the wake of rising energy demands, microalgae have emerged as  potential sources of sustainable and renewable carbon-neutral fuels, such as  bio-hydrogen and bio-oil. PURPOSE: For rational metabolic engineering, the  elucidation of metabolic pathways in fine detail and their manipulation according to  requirements is the key to exploiting the use of microalgae. Emergence of  site-specific nucleases have revolutionized applied research leading to  biotechnological gains. Genome engineering as well as modulation of the endogenous  genome with high precision using CRISPR systems is being gradually employed in  microalgal research. Further, to optimize and produce better algal platforms, use of  systems biology network analysis and integration of omics data is required. This  review discusses two important approaches: systems biology and gene editing  strategies used on microalgal systems with a focus on biofuel production and  sustainable solutions. It also emphasizes that the integration of such systems would  contribute and compliment applied research on microalgae. CONCLUSIONS: Recent  advances in microalgae are discussed, including systems biology, gene editing  approaches in lipid bio-synthesis, and antenna engineering. Lastly, it has been  attempted here to showcase how CRISPR/Cas systems are a better editing tool than  existing techniques that can be utilized for gene modulation and engineering during  biofuel production.\\nFAU - Banerjee, Avik\\nAU  - Banerjee A\\nAD  - a Department of Environmental Science and Engineering , Indian Institute of  Technology (Indian School of Mines) , Dhanbad , India.\\nFAU - Banerjee, Chiranjib\\nAU  - Banerjee C\\nAD  - a Department of Environmental Science and Engineering , Indian Institute of  Technology (Indian School of Mines) , Dhanbad , India.\\nFAU - Negi, Sangeeta\\nAU  - Negi S\\nAD  - b New Mexico Consortium , Los Alamos , NM , USA.\\nFAU - Chang, Jo-Shu\\nAU  - Chang JS\\nAD  - c Department of Chemical Engineering , National Cheng Kung University , Tainan ,  Taiwan.\\nFAU - Shukla, Pratyoosh\\nAU  - Shukla P\\nAD  - d Department of Microbiology, Enzyme Technology and Protein Bioinformatics  Laboratory , Maharshi Dayanand University , Rohtak , India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20170809\\nPL  - England\\nTA  - Crit Rev Biotechnol\\nJT  - Critical reviews in biotechnology\\nJID - 8505177\\nRN  - 0 (Lipids)\\nSB  - IM\\nMH  - Biosynthetic Pathways\\nMH  - Gene Editing/*methods\\nMH  - Genetic Engineering\\nMH  - Lipids/*biosynthesis\\nMH  - Microalgae/*metabolism\\nMH  - Systems Biology/*methods\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - Microalgae\\nOT  - biofuel\\nOT  - gene editing\\nOT  - sgRNA\\nOT  - systems biology\\nEDAT- 2017/08/11 06:00\\nMHDA- 2018/08/18 06:00\\nCRDT- 2017/08/11 06:00\\nPHST- 2017/08/11 06:00 [pubmed]\\nPHST- 2018/08/18 06:00 [medline]\\nPHST- 2017/08/11 06:00 [entrez]\\nAID - 10.1080/07388551.2017.1356803 [doi]\\nPST - ppublish\\nSO  - Crit Rev Biotechnol. 2018 May;38(3):369-385. doi: 10.1080/07388551.2017.1356803.  Epub 2017 Aug 9.\\n\\nPMID- 27507970\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20160810\\nLR  - 20201001\\nIS  - 1664-302X (Print)\\nIS  - 1664-302X (Electronic)\\nIS  - 1664-302X (Linking)\\nVI  - 7\\nDP  - 2016\\nTI  - Regulatory RNAs in the Less Studied Streptococcal Species: From Nomenclature to  Identification.\\nPG  - 1161\\nLID - 10.3389/fmicb.2016.01161 [doi]\\nLID - 1161\\nAB  - Streptococcal species are Gram-positive bacteria involved in severe and invasive  diseases in humans and animals. Although, this group includes different pathogenic  species involved in life-threatening infections for humans, it also includes  beneficial species, such as Streptococcus thermophilus, which is used in yogurt  production. In bacteria virulence factors are controlled by various regulatory  networks including regulatory RNAs. For clearness and to develop logical thinking,  we start this review with a revision of regulatory RNAs nomenclature. Previous  reviews are mostly dealing with Streptococcus pyogenes and Streptococcus pneumoniae  regulatory RNAs. We especially focused our analysis on regulatory RNAs in  Streptococcus agalactiae, Streptococcus mutans, Streptococcus thermophilus and other  less studied Streptococcus species. Although, S. agalactiae RNome remains largely  unknown, sRNAs (small RNAs) are supposed to mediate regulation during environmental  adaptation and host infection. In the case of S. mutans, sRNAs are suggested to be  involved in competence regulation, carbohydrate metabolism, and Toxin-Antitoxin  systems. A new category of miRNA-size small RNAs (msRNAs) was also identified for  the first time in this species. The analysis of S. thermophilus sRNome shows that  many sRNAs are associated to the bacterial immune system known as CRISPR-Cas system.  Only few of the other different Streptococcus species have been the subject of  studies pointed toward the characterization of regulatory RNAs. Finally,  understanding bacterial sRNome can constitute one step forward to the elaboration of  new strategies in therapy such as substitution of antibiotics in the management of  S. agalactiae neonatal infections, prevention of S. mutans dental caries or use of  S. thermophilus CRISPR-Cas system in genome editing applications.\\nFAU - Zorgani, Mohamed A\\nAU  - Zorgani MA\\nAD  - ISP, INRA, Equipe 5 \"BactÃ©ries et Risque Materno-foetal\", FacultÃ© de MÃ©decine, UMR  1282, UniversitÃ© FranÃ§ois Rabelais de Tours, Tours France.\\nFAU - Quentin, Roland\\nAU  - Quentin R\\nAD  - ISP, INRA, Equipe 5 \"BactÃ©ries et Risque Materno-foetal\", FacultÃ© de MÃ©decine, UMR  1282, UniversitÃ© FranÃ§ois Rabelais de Tours, ToursFrance; Service de BactÃ©riologie  Virologie et HygiÃ¨ne HospitaliÃ¨re, Centre Hospitalier RÃ©gional Universitaire de  Tours, ToursFrance.\\nFAU - Lartigue, Marie-FrÃ©dÃ©rique\\nAU  - Lartigue MF\\nAD  - ISP, INRA, Equipe 5 \"BactÃ©ries et Risque Materno-foetal\", FacultÃ© de MÃ©decine, UMR  1282, UniversitÃ© FranÃ§ois Rabelais de Tours, ToursFrance; Service de BactÃ©riologie  Virologie et HygiÃ¨ne HospitaliÃ¨re, Centre Hospitalier RÃ©gional Universitaire de  Tours, ToursFrance.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20160726\\nTA  - Front Microbiol\\nJT  - Frontiers in microbiology\\nJID - 101548977\\nPMC - PMC4960207\\nOTO - NOTNLM\\nOT  - Regulatory RNA nomenclature\\nOT  - Streptococcus agalactiae\\nOT  - Streptococcus mutans\\nOT  - Streptococcus thermophilus\\nOT  - non-coding RNA\\nOT  - small RNAs\\nEDAT- 2016/08/11 06:00\\nMHDA- 2016/08/11 06:01\\nCRDT- 2016/08/11 06:00\\nPHST- 2016/06/02 00:00 [received]\\nPHST- 2016/07/12 00:00 [accepted]\\nPHST- 2016/08/11 06:00 [entrez]\\nPHST- 2016/08/11 06:00 [pubmed]\\nPHST- 2016/08/11 06:01 [medline]\\nAID - 10.3389/fmicb.2016.01161 [doi]\\nPST - epublish\\nSO  - Front Microbiol. 2016 Jul 26;7:1161. doi: 10.3389/fmicb.2016.01161. eCollection  2016.\\n\\nPMID- 33137193\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210126\\nLR  - 20210126\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 49\\nIP  - D1\\nDP  - 2021 Jan 8\\nTI  - AcrHub: an integrative hub for investigating, predicting and mapping anti-CRISPR  proteins.\\nPG  - D630-D638\\nLID - 10.1093/nar/gkaa951 [doi]\\nAB  - Anti-CRISPR (Acr) proteins naturally inhibit CRISPR-Cas adaptive immune systems  across bacterial and archaeal domains of life. This emerging field has caused a  paradigm shift in the way we think about the CRISPR-Cas system, and promises a  number of useful applications from gene editing to phage therapy. As the number of  verified and predicted Acrs rapidly expands, few online resources have been  developed to deal with this wealth of information. To overcome this shortcoming, we  developed AcrHub, an integrative database to provide an all-in-one solution for  investigating, predicting and mapping Acr proteins. AcrHub catalogs 339  non-redundant experimentally validated Acrs and over 70 000 predicted Acrs extracted  from genome sequence data from a diverse range of prokaryotic organisms and their  viruses. It integrates state-of-the-art predictors to predict potential Acrs, and  incorporates three analytical modules: similarity analysis, phylogenetic analysis  and homology network analysis, to analyze their relationships with known Acrs. By  interconnecting all modules as a platform, AcrHub presents enriched and in-depth  analysis of known and potential Acrs and therefore provides new and exciting  insights into the future of Acr discovery and validation. AcrHub is freely available  at http://pacrispr.erc.monash.edu/AcrHub/.\\nCI  - Â© The Author(s) 2020. Published by Oxford University Press on behalf of Nucleic  Acids Research.\\nFAU - Wang, Jiawei\\nAU  - Wang J\\nAD  - Infection and Immunity Program, Biomedicine Discovery Institute and Department of  Microbiology, Monash University, VIC 3800, Australia.\\nFAU - Dai, Wei\\nAU  - Dai W\\nAD  - Infection and Immunity Program, Biomedicine Discovery Institute and Department of  Microbiology, Monash University, VIC 3800, Australia.\\nAD  - School of Computer Science and Information Security, Guilin University of Electronic  Technology, Guilin 541004, China.\\nFAU - Li, Jiahui\\nAU  - Li J\\nAD  - School of Computer Science and Information Security, Guilin University of Electronic  Technology, Guilin 541004, China.\\nFAU - Li, Qi\\nAU  - Li Q\\nAD  - School of Computer Science and Information Security, Guilin University of Electronic  Technology, Guilin 541004, China.\\nFAU - Xie, Ruopeng\\nAU  - Xie R\\nAD  - School of Computer Science and Information Security, Guilin University of Electronic  Technology, Guilin 541004, China.\\nFAU - Zhang, Yanju\\nAU  - Zhang Y\\nAD  - School of Computer Science and Information Security, Guilin University of Electronic  Technology, Guilin 541004, China.\\nFAU - Stubenrauch, Christopher\\nAU  - Stubenrauch C\\nAD  - Infection and Immunity Program, Biomedicine Discovery Institute and Department of  Microbiology, Monash University, VIC 3800, Australia.\\nFAU - Lithgow, Trevor\\nAU  - Lithgow T\\nAD  - Infection and Immunity Program, Biomedicine Discovery Institute and Department of  Microbiology, Monash University, VIC 3800, Australia.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Data Analysis\\nMH  - *Databases, Protein\\nMH  - Internet\\nPMC - PMC7779044\\nEDAT- 2020/11/03 06:00\\nMHDA- 2021/01/27 06:00\\nCRDT- 2020/11/02 17:14\\nPHST- 2020/10/08 00:00 [accepted]\\nPHST- 2020/09/29 00:00 [revised]\\nPHST- 2020/08/15 00:00 [received]\\nPHST- 2020/11/03 06:00 [pubmed]\\nPHST- 2021/01/27 06:00 [medline]\\nPHST- 2020/11/02 17:14 [entrez]\\nAID - 5952202 [pii]\\nAID - gkaa951 [pii]\\nAID - 10.1093/nar/gkaa951 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2021 Jan 8;49(D1):D630-D638. doi: 10.1093/nar/gkaa951.\\n\\nPMID- 31326272\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200129\\nLR  - 20200905\\nIS  - 1097-4164 (Electronic)\\nIS  - 1097-2765 (Print)\\nIS  - 1097-2765 (Linking)\\nVI  - 75\\nIP  - 5\\nDP  - 2019 Sep 5\\nTI  - Second Messenger cA(4) Formation within the Composite Csm1 Palm Pocket of Type III-A  CRISPR-Cas Csm Complex and Its Release Path.\\nPG  - 933-943.e6\\nLID - S1097-2765(19)30446-0 [pii]\\nLID - 10.1016/j.molcel.2019.06.013 [doi]\\nAB  - Target RNA binding to crRNA-bound type III-A CRISPR-Cas multi-subunit Csm  surveillance complexes activates cyclic-oligoadenylate (cA(n)) formation from ATP  subunits positioned within the composite pair of Palm domain pockets of the Csm1  subunit. The generated cA(n) second messenger in turn targets the CARF domain of  trans-acting RNase Csm6, triggering its HEPN domain-based RNase activity. We have  undertaken cryo-EM studies on multi-subunit Thermococcus onnurineus Csm effector  ternary complexes, as well as X-ray studies on Csm1-Csm4 cassette, both bound to  substrate (AMPPNP), intermediates (pppA(n)), and products (cA(n)), to decipher  mechanistic aspects of cA(n) formation and release. A network of intermolecular  hydrogen bond alignments accounts for the observed adenosine specificity, with  ligand positioning dictating formation of linear pppA(n) intermediates and  subsequent cA(n) formation by cyclization. We combine our structural results with  published functional studies to highlight mechanistic insights into the role of the  Csm effector complex in mediating the cA(n) signaling pathway.\\nCI  - Copyright Â© 2019 Elsevier Inc. All rights reserved.\\nFAU - Jia, Ning\\nAU  - Jia N\\nAD  - Structural Biology Program Memorial Sloan Kettering Cancer Center, New York, NY  10065, USA. Electronic address: jian@mskcc.org.\\nFAU - Jones, Roger\\nAU  - Jones R\\nAD  - Department of Chemistry and Chemical Biology Rutgers, The State University of New  Jersey, Piscataway, NJ 08854, USA.\\nFAU - Sukenick, George\\nAU  - Sukenick G\\nAD  - Chemical Biology Program Memorial Sloan Kettering Cancer Center, New York, NY 10065,  USA.\\nFAU - Patel, Dinshaw J\\nAU  - Patel DJ\\nAD  - Structural Biology Program Memorial Sloan Kettering Cancer Center, New York, NY  10065, USA. Electronic address: pateld@mskcc.org.\\nLA  - eng\\nGR  - P30 GM124165/GM/NIGMS NIH HHS/United States\\nGR  - R01 AI141507/AI/NIAID NIH HHS/United States\\nGR  - R01 GM129430/GM/NIGMS NIH HHS/United States\\nGR  - P30 CA008748/CA/NCI NIH HHS/United States\\nGR  - S10 RR029205/RR/NCRR NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20190717\\nTA  - Mol Cell\\nJT  - Molecular cell\\nJID - 9802571\\nRN  - 0 (Adenine Nucleotides)\\nRN  - 0 (Archaeal Proteins)\\nRN  - 0 (Oligoribonucleotides)\\nRN  - 61172-40-5 (2\\',5\\'-oligoadenylate)\\nRN  - EC 3.1.- (Ribonucleases)\\nSB  - IM\\nMH  - Adenine Nucleotides/*chemistry/metabolism\\nMH  - Archaeal Proteins/*chemistry/metabolism\\nMH  - *CRISPR-Cas Systems\\nMH  - Cryoelectron Microscopy\\nMH  - Oligoribonucleotides/*chemistry/metabolism\\nMH  - Ribonucleases/*chemistry/metabolism\\nMH  - *Second Messenger Systems\\nMH  - Thermococcus/*chemistry/metabolism/ultrastructure\\nPMC - PMC6731140\\nMID - NIHMS1532404\\nOTO - NOTNLM\\nOT  - *ATP conversion into pppA(n) intermediates and cA(n) products\\nOT  - *adenosine recognition specificity\\nOT  - *ligand positioning in composite Palm pocket\\nOT  - *mechanistic insights into cA(4) formation and release\\nOT  - *type III-A CRISPR-Cas Csm\\nCOIS- DECLARATION OF INTERESTS Roger Jones is a consultant to IFM Therapeutics.\\nEDAT- 2019/07/22 06:00\\nMHDA- 2020/01/30 06:00\\nCRDT- 2019/07/22 06:00\\nPHST- 2019/03/29 00:00 [received]\\nPHST- 2019/05/09 00:00 [revised]\\nPHST- 2019/06/09 00:00 [accepted]\\nPHST- 2019/07/22 06:00 [pubmed]\\nPHST- 2020/01/30 06:00 [medline]\\nPHST- 2019/07/22 06:00 [entrez]\\nAID - S1097-2765(19)30446-0 [pii]\\nAID - 10.1016/j.molcel.2019.06.013 [doi]\\nPST - ppublish\\nSO  - Mol Cell. 2019 Sep 5;75(5):933-943.e6. doi: 10.1016/j.molcel.2019.06.013. Epub 2019  Jul 17.\\n\\nPMID- 25477473\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150409\\nLR  - 20211203\\nIS  - 1522-1466 (Electronic)\\nIS  - 1931-857X (Print)\\nIS  - 1522-1466 (Linking)\\nVI  - 308\\nIP  - 4\\nDP  - 2015 Feb 15\\nTI  - Generation of WNK1 knockout cell lines by CRISPR/Cas-mediated genome editing.\\nPG  - F366-76\\nLID - 10.1152/ajprenal.00612.2014 [doi]\\nAB  - Sodium-coupled SLC12 cation chloride cotransporters play important roles in cell  volume and chloride homeostasis, epithelial fluid secretion, and renal tubular salt  reabsorption. These cotransporters are phosphorylated and activated indirectly by  With-No-Lysine (WNK) kinases through their downstream effector kinases, Ste20- and  SPS1-related proline alanine-rich kinase (SPAK) and oxidative stress-responsive  kinase 1 (OSR1). Multiple WNK kinases can coexist within a single cell type,  although their relative contributions to SPAK/OSR1 activation and salt transport  remain incompletely understood. Deletion of specific WNKs from cells that natively  express a functional WNK-SPAK/OSR1 network will help resolve these knowledge gaps.  Here, we outline a simple method to selectively knock out full-length WNK1  expression from mammalian cells using RNA-guided clustered regularly interspaced  short palindromic repeats/Cas9 endonucleases. Two clonal cell lines were generated  by using a single-guide RNA (sgRNA) targeting exon 1 of the WNK1 gene, which  produced indels that abolished WNK1 protein expression. Both cell lines exhibited  reduced endogenous WNK4 protein abundance, indicating that WNK1 is required for WNK4  stability. Consistent with an on-target effect, the reduced WNK4 abundance was  associated with increased expression of the KLHL3/cullin-3 E3 ubiquitin ligase  complex and was rescued by exogenous WNK1 overexpression. Although the morphology of  the knockout cells was indistinguishable from control, they exhibited low baseline  SPAK/OSR1 activity and failed to trigger regulatory volume increase after hypertonic  stress, confirming an essential role for WNK1 in cell volume regulation.  Collectively, our data show how this new, powerful, and accessible gene-editing  technology can be used to dissect and analyze WNK signaling networks.\\nFAU - Roy, Ankita\\nAU  - Roy A\\nAD  - Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania;\\nFAU - Goodman, Joshua H\\nAU  - Goodman JH\\nAD  - Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania;\\nFAU - Begum, Gulnaz\\nAU  - Begum G\\nAD  - Department of Neurology, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania;\\nFAU - Donnelly, Bridget F\\nAU  - Donnelly BF\\nAD  - Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania;\\nFAU - Pittman, Gabrielle\\nAU  - Pittman G\\nAD  - Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania;\\nFAU - Weinman, Edward J\\nAU  - Weinman EJ\\nAD  - Department of Medicine, University of Maryland Medical School, Baltimore, Maryland;  and.\\nFAU - Sun, Dandan\\nAU  - Sun D\\nAD  - Department of Neurology, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania; Veterans Affairs Pittsburgh Healthcare System, Pittsburgh,  Pennsylvania.\\nFAU - Subramanya, Arohan R\\nAU  - Subramanya AR\\nAD  - Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh,  Pennsylvania; Department of Cell Biology, University of Pittsburgh School of  Medicine, Pittsburgh, Pennsylvania; Veterans Affairs Pittsburgh Healthcare System,  Pittsburgh, Pennsylvania ars129@pitt.edu.\\nLA  - eng\\nGR  - R01 NS038118/NS/NINDS NIH HHS/United States\\nGR  - NS075995/NS/NINDS NIH HHS/United States\\nGR  - P30 DK079307/DK/NIDDK NIH HHS/United States\\nGR  - R01DK098145/DK/NIDDK NIH HHS/United States\\nGR  - P30DK79307/DK/NIDDK NIH HHS/United States\\nGR  - R01DK55881/DK/NIDDK NIH HHS/United States\\nGR  - R01 DK098145/DK/NIDDK NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20141204\\nTA  - Am J Physiol Renal Physiol\\nJT  - American journal of physiology. Renal physiology\\nJID - 100901990\\nRN  - 0 (Adaptor Proteins, Signal Transducing)\\nRN  - 0 (CUL3 protein, human)\\nRN  - 0 (Carrier Proteins)\\nRN  - 0 (Cullin Proteins)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (KLHL3 protein, human)\\nRN  - 0 (Microfilament Proteins)\\nRN  - 0 (Minor Histocompatibility Antigens)\\nRN  - EC 2.7.1.- (OXSR1 protein, human)\\nRN  - EC 2.7.11.1 (Protein Serine-Threonine Kinases)\\nRN  - EC 2.7.11.1 (STK39 protein, human)\\nRN  - EC 2.7.11.1 (WNK Lysine-Deficient Protein Kinase 1)\\nRN  - EC 2.7.11.1 (WNK1 protein, human)\\nRN  - EC 2.7.11.1 (WNK4 protein, human)\\nSB  - IM\\nMH  - Adaptor Proteins, Signal Transducing\\nMH  - *CRISPR-Cas Systems\\nMH  - Carrier Proteins/genetics/metabolism\\nMH  - Cell Shape\\nMH  - Cell Size\\nMH  - Cullin Proteins/genetics/metabolism\\nMH  - Down-Regulation\\nMH  - Exons\\nMH  - Gene Knockdown Techniques/*methods\\nMH  - *Genome, Human\\nMH  - Genotype\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Intracellular Signaling Peptides and Proteins/*deficiency/genetics\\nMH  - Microfilament Proteins\\nMH  - Minor Histocompatibility Antigens\\nMH  - Osmotic Pressure\\nMH  - Phenotype\\nMH  - Protein Serine-Threonine Kinases/*deficiency/genetics/metabolism\\nMH  - Protein Stability\\nMH  - Signal Transduction\\nMH  - Time Factors\\nMH  - Transfection\\nMH  - WNK Lysine-Deficient Protein Kinase 1\\nPMC - PMC4329490\\nOTO - NOTNLM\\nOT  - CRISPR/Cas system\\nOT  - SLC12 cotransporters\\nOT  - WNK1\\nOT  - genome editing\\nEDAT- 2014/12/06 06:00\\nMHDA- 2015/04/10 06:00\\nCRDT- 2014/12/06 06:00\\nPHST- 2014/12/06 06:00 [entrez]\\nPHST- 2014/12/06 06:00 [pubmed]\\nPHST- 2015/04/10 06:00 [medline]\\nAID - ajprenal.00612.2014 [pii]\\nAID - F-00612-2014 [pii]\\nAID - 10.1152/ajprenal.00612.2014 [doi]\\nPST - ppublish\\nSO  - Am J Physiol Renal Physiol. 2015 Feb 15;308(4):F366-76. doi:  10.1152/ajprenal.00612.2014. Epub 2014 Dec 4.\\n\\nPMID- 30173764\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190624\\nLR  - 20190624\\nIS  - 1557-7988 (Electronic)\\nIS  - 0076-6879 (Linking)\\nVI  - 608\\nDP  - 2018\\nTI  - Metabolic Engineering of Saccharomyces cerevisiae Using a Trifunctional CRISPR/Cas  System for Simultaneous Gene Activation, Interference, and Deletion.\\nPG  - 265-276\\nLID - S0076-6879(18)30144-7 [pii]\\nLID - 10.1016/bs.mie.2018.04.010 [doi]\\nAB  - Design and construction of an optimal microbial cell factory typically requires  overexpression, knockdown, and knockout of multiple gene targets. In this chapter,  we describe a combinatorial metabolic engineering strategy utilizing an orthogonal  trifunctional CRISPR system that combines transcriptional activation,  transcriptional interference, and gene deletion (CRISPR-AID) in the yeast  Saccharomyces cerevisiae. This strategy enables multiplexed perturbation of the  metabolic and regulatory networks in a modular, parallel, and high-throughput  manner. To implement this system, three orthogonal Cas proteins were utilized:  dLbCpf1 fused to a transcriptional activator, dSpCas9 fused to a transcriptional  repressor, and SaCas9 for gene deletion. Deletion was accomplished by the  introduction of a 28bp frame-shift mutation using a homology donor on the guide RNA  expression vector. This approach enables the application of metabolic engineering to  systematically optimize phenotypes of interest through a combination of gain-,  reduction-, and loss-of-function mutations. Finally, we describe the construction of  the CRISPR-AID system and its application toward engineering an example phenotype,  surface display of recombinant Trichoderma reesei endoglucanase II.\\nCI  - Â© 2018 Elsevier Inc. All rights reserved.\\nFAU - Schultz, Carl\\nAU  - Schultz C\\nAD  - Department of Chemical and Biomolecular Engineering, University of Illinois at  Urbana-Champaign, Urbana, IL, United States; Carl R. Woese Institute for Genomic  Biology, University of Illinois at Urbana-Champaign, Urbana, IL, United States.\\nFAU - Lian, Jiazhang\\nAU  - Lian J\\nAD  - Department of Chemical and Biomolecular Engineering, University of Illinois at  Urbana-Champaign, Urbana, IL, United States; Key Laboratory of Biomass Chemical  Engineering of Ministry of Education, College of Chemical and Biological  Engineering, Zhejiang University, Hangzhou, China.\\nFAU - Zhao, Huimin\\nAU  - Zhao H\\nAD  - Department of Chemical and Biomolecular Engineering, University of Illinois at  Urbana-Champaign, Urbana, IL, United States; Carl R. Woese Institute for Genomic  Biology, University of Illinois at Urbana-Champaign, Urbana, IL, United States.  Electronic address: zhao5@illinois.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20180508\\nPL  - United States\\nTA  - Methods Enzymol\\nJT  - Methods in enzymology\\nJID - 0212271\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Recombinant Proteins)\\nRN  - EC 3.2.1.- (endoglucanase 2)\\nRN  - EC 3.2.1.4 (Cellulase)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - Cellulase/genetics\\nMH  - Gene Deletion\\nMH  - Gene Editing/*methods\\nMH  - Metabolic Engineering/*methods\\nMH  - RNA, Guide/genetics\\nMH  - Recombinant Proteins/genetics\\nMH  - Saccharomyces cerevisiae/*genetics\\nMH  - Transcriptional Activation\\nMH  - Trichoderma/enzymology/genetics\\nOTO - NOTNLM\\nOT  - *CRISPR\\nOT  - *Genome engineering\\nOT  - *Metabolic engineering\\nOT  - *Saccharomyces cerevisiae\\nOT  - *Synthetic biology\\nEDAT- 2018/09/04 06:00\\nMHDA- 2019/06/25 06:00\\nCRDT- 2018/09/04 06:00\\nPHST- 2018/09/04 06:00 [entrez]\\nPHST- 2018/09/04 06:00 [pubmed]\\nPHST- 2019/06/25 06:00 [medline]\\nAID - S0076-6879(18)30144-7 [pii]\\nAID - 10.1016/bs.mie.2018.04.010 [doi]\\nPST - ppublish\\nSO  - Methods Enzymol. 2018;608:265-276. doi: 10.1016/bs.mie.2018.04.010. Epub 2018 May 8.\\n\\nPMID- 23977949\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20141208\\nLR  - 20211021\\nIS  - 2161-5063 (Electronic)\\nIS  - 2161-5063 (Print)\\nIS  - 2161-5063 (Linking)\\nVI  - 2\\nIP  - 10\\nDP  - 2013 Oct 18\\nTI  - Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas.\\nPG  - 604-13\\nLID - 10.1021/sb400081r [doi]\\nAB  - Transcriptional regulation is central to the complex behavior of natural biological  systems and synthetic gene circuits. Platforms for the scalable, tunable, and simple  modulation of transcription would enable new abilities to study natural systems and  implement artificial capabilities in living cells. Previous approaches to synthetic  transcriptional regulation have relied on engineering DNA-binding proteins, which  necessitate multistep processes for construction and optimization of function. Here,  we show that the CRISPR/Cas system of Streptococcus pyogenes can be programmed to  direct both activation and repression to natural and artificial eukaryotic promoters  through the simple engineering of guide RNAs with base-pairing complementarity to  target DNA sites. We demonstrate that the activity of CRISPR-based transcription  factors (crisprTFs) can be tuned by directing multiple crisprTFs to different  positions in natural promoters and by arraying multiple crisprTF-binding sites in  the context of synthetic promoters in yeast and human cells. Furthermore, externally  controllable regulatory modules can be engineered by layering gRNAs with small  molecule-responsive proteins. Additionally, single nucleotide substitutions within  promoters are sufficient to render them orthogonal with respect to the same  gRNA-guided crisprTF. We envision that CRISPR-based eukaryotic gene regulation will  enable the facile construction of scalable synthetic gene circuits and open up new  approaches for mapping natural gene networks and their effects on complex cellular  phenotypes.\\nFAU - Farzadfard, Fahim\\nAU  - Farzadfard F\\nAD  - Department of Electrical Engineering & Computer Science and Department of Biological  Engineering, Massachusetts Institute of Technology , 77 Massachusetts Avenue,  Cambridge, Massachusetts 02139, United States.\\nFAU - Perli, Samuel D\\nAU  - Perli SD\\nFAU - Lu, Timothy K\\nAU  - Lu TK\\nLA  - eng\\nGR  - DP2 OD008435/OD/NIH HHS/United States\\nGR  - 1DP2OD008435/OD/NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20130911\\nTA  - ACS Synth Biol\\nJT  - ACS synthetic biology\\nJID - 101575075\\nRN  - 0 (Transcription Factors)\\nSB  - IM\\nMH  - Base Sequence\\nMH  - *CRISPR-Cas Systems\\nMH  - Molecular Sequence Data\\nMH  - Plasmids\\nMH  - Promoter Regions, Genetic\\nMH  - Saccharomyces cerevisiae/genetics/metabolism\\nMH  - Streptococcus pyogenes/genetics/metabolism\\nMH  - Transcription Factors/genetics/*metabolism\\nPMC - PMC3805333\\nEDAT- 2013/08/28 06:00\\nMHDA- 2014/12/15 06:00\\nCRDT- 2013/08/28 06:00\\nPHST- 2013/08/28 06:00 [entrez]\\nPHST- 2013/08/28 06:00 [pubmed]\\nPHST- 2014/12/15 06:00 [medline]\\nAID - 10.1021/sb400081r [doi]\\nPST - ppublish\\nSO  - ACS Synth Biol. 2013 Oct 18;2(10):604-13. doi: 10.1021/sb400081r. Epub 2013 Sep 11.\\n\\nPMID- 28672161\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180516\\nLR  - 20191008\\nIS  - 1879-6265 (Electronic)\\nIS  - 1879-6257 (Print)\\nIS  - 1879-6257 (Linking)\\nVI  - 25\\nDP  - 2017 Aug\\nTI  - Polintons, virophages and transpovirons: a tangled web linking viruses, transposons  and immunity.\\nPG  - 7-15\\nLID - S1879-6257(17)30033-0 [pii]\\nLID - 10.1016/j.coviro.2017.06.008 [doi]\\nAB  - Virophages are satellite DNA viruses that depend for their replication on giant  viruses of the family Mimiviridae. An evolutionary relationship exists between the  virophages and Polintons, large self-synthesizing transposons that are wide spread  in the genomes of diverse eukaryotes. Most of the Polintons encode homologs of major  and minor icosahedral virus capsid proteins and accordingly are predicted to form  virions. Additionally, metagenome analysis has led to the discovery of an expansive  family of Polinton-like viruses (PLV) that are more distantly related to bona fide  Polintons and virophages. Another group of giant virus parasites includes small,  linear, double-stranded DNA elements called transpovirons. Recent in-depth  comparative genomic analysis has yielded evidence of the origin of the PLV and the  transpovirons from Polintons. Integration of virophage genomes into genomes of both  giant viruses and protists has been demonstrated. Furthermore, in an experimental  coinfection system that consisted of a protist host, a giant virus and an associated  virophage, the virophage integrated into the host genome and, after activation of  its expression by a superinfecting giant virus, served as an agent of adaptive  immunity. There is a striking analogy between this mechanism and the CRISPR-Cas  system of prokaryotic adaptive immunity. Taken together, these findings show that  Polintons, PLV, virophages and transpovirons form a dynamic network of integrating  mobile genetic elements that contribute to the cellular antivirus defense and  host-virus coevolution.\\nCI  - Published by Elsevier B.V.\\nFAU - Koonin, Eugene V\\nAU  - Koonin EV\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA. Electronic address:  koonin@ncbi.nlm.nih.gov.\\nFAU - Krupovic, Mart\\nAU  - Krupovic M\\nAD  - UnitÃ© Biologie MolÃ©culaire du GÃ¨ne chez les ExtrÃªmophiles, Department of  Microbiology, Institut Pasteur, Paris, France. Electronic address:  krupovic@pasteur.fr.\\nLA  - eng\\nGR  - Z01 LM000073-12/Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20170630\\nTA  - Curr Opin Virol\\nJT  - Current opinion in virology\\nJID - 101560941\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Viral)\\nSB  - IM\\nMH  - CRISPR-Cas Systems\\nMH  - DNA Replication\\nMH  - *DNA Transposable Elements\\nMH  - DNA Viruses/genetics/immunology/physiology\\nMH  - DNA, Viral\\nMH  - Eukaryota/virology\\nMH  - *Evolution, Molecular\\nMH  - Genome, Viral\\nMH  - Giant Viruses/genetics/immunology/pathogenicity/*physiology\\nMH  - Host-Pathogen Interactions\\nMH  - Metagenome\\nMH  - Phylogeny\\nMH  - *Virion/genetics/immunology\\nMH  - *Virophages/genetics/immunology/physiology\\nPMC - PMC5610638\\nMID - NIHMS889931\\nEDAT- 2017/07/04 06:00\\nMHDA- 2018/05/17 06:00\\nCRDT- 2017/07/04 06:00\\nPHST- 2017/04/20 00:00 [received]\\nPHST- 2017/05/30 00:00 [revised]\\nPHST- 2017/06/19 00:00 [accepted]\\nPHST- 2017/07/04 06:00 [pubmed]\\nPHST- 2018/05/17 06:00 [medline]\\nPHST- 2017/07/04 06:00 [entrez]\\nAID - S1879-6257(17)30033-0 [pii]\\nAID - 10.1016/j.coviro.2017.06.008 [doi]\\nPST - ppublish\\nSO  - Curr Opin Virol. 2017 Aug;25:7-15. doi: 10.1016/j.coviro.2017.06.008. Epub 2017 Jun  30.\\n\\nPMID- 34800362\\nOWN - NLM\\nSTAT- Publisher\\nLR  - 20211120\\nIS  - 2405-4720 (Electronic)\\nIS  - 2405-4712 (Linking)\\nDP  - 2021 Nov 17\\nTI  - Multi-layer CRISPRa/i circuits for dynamic genetic programs in cell-free and  bacterial systems.\\nLID - S2405-4712(21)00419-1 [pii]\\nLID - 10.1016/j.cels.2021.10.008 [doi]\\nAB  - CRISPR-Cas transcriptional circuits hold great promise as platforms for engineering  metabolic networks and information processing circuits. Historically, prokaryotic  CRISPR control systems have been limited to CRISPRi. Creating approaches to  integrate CRISPRa for transcriptional activation with existing CRISPRi-based systems  would greatly expand CRISPR circuit design space. Here, we develop design principles  for engineering prokaryotic CRISPRa/i genetic circuits with network topologies  specified by guide RNAs. We demonstrate that multi-layer CRISPRa/i cascades and  feedforward loops can operate through the regulated expression of guide RNAs in  cell-free expression systems and E.Â\\xa0coli. We show that CRISPRa/i circuits can  program complex functions by designing type 1 incoherent feedforward loops acting as  fold-change detectors and tunable pulse-generators. By investigating how component  characteristics relate to network properties such as depth, width, and speed, this  work establishes a framework for building scalable CRISPRa/i circuits as regulatory  programs in cell-free expression systems and bacterial hosts. A record of this  paper\\'s transparent peer review process is included in the supplemental information.\\nCI  - Copyright Â© 2021 Elsevier Inc. All rights reserved.\\nFAU - Tickman, Benjamin I\\nAU  - Tickman BI\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA.\\nFAU - Burbano, Diego Alba\\nAU  - Burbano DA\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA; Department of Chemical  Engineering, University of Washington, Seattle, WA 98195, USA.\\nFAU - Chavali, Venkata P\\nAU  - Chavali VP\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA.\\nFAU - Kiattisewee, Cholpisit\\nAU  - Kiattisewee C\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA.\\nFAU - Fontana, Jason\\nAU  - Fontana J\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA.\\nFAU - Khakimzhan, Aset\\nAU  - Khakimzhan A\\nAD  - School of Physics and Astronomy, University of Minnesota, Minneapolis, MN 55455,  USA.\\nFAU - Noireaux, Vincent\\nAU  - Noireaux V\\nAD  - School of Physics and Astronomy, University of Minnesota, Minneapolis, MN 55455,  USA.\\nFAU - Zalatan, Jesse G\\nAU  - Zalatan JG\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA; Department of Chemistry,  University of Washington, Seattle, WA 98195, USA. Electronic address:  zalatan@uw.edu.\\nFAU - Carothers, James M\\nAU  - Carothers JM\\nAD  - Molecular Engineering & Sciences Institute and Center for Synthetic Biology,  University of Washington, Seattle, WA 98195, USA; Department of Chemical  Engineering, University of Washington, Seattle, WA 98195, USA. Electronic address:  jcaroth@uw.edu.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20211117\\nPL  - United States\\nTA  - Cell Syst\\nJT  - Cell systems\\nJID - 101656080\\nSB  - IM\\nOTO - NOTNLM\\nOT  - CRISPRa\\nOT  - CRISPRi\\nOT  - E.Â\\xa0coli\\nOT  - I1-FFL\\nOT  - cell-free\\nOT  - expression dynamics\\nOT  - fold-change detection\\nOT  - transcriptional circuits\\nCOIS- Declaration of interests J.F. is a founder of Wayfinder Biosciences, and J.G.Z. and  J.M.C are members of the Wayfinder Biosciences scientific advisory board; all three  are listed as co-inventors of RNA technologies that are the subjects of patent  filings. The Noireaux laboratory receives research funds from Arbor Biosciences, a  distributor of the myTXTL cell-free protein synthesis kit.\\nEDAT- 2021/11/21 06:00\\nMHDA- 2021/11/21 06:00\\nCRDT- 2021/11/20 20:08\\nPHST- 2021/06/08 00:00 [received]\\nPHST- 2021/08/24 00:00 [revised]\\nPHST- 2021/10/26 00:00 [accepted]\\nPHST- 2021/11/20 20:08 [entrez]\\nPHST- 2021/11/21 06:00 [pubmed]\\nPHST- 2021/11/21 06:00 [medline]\\nAID - S2405-4712(21)00419-1 [pii]\\nAID - 10.1016/j.cels.2021.10.008 [doi]\\nPST - aheadofprint\\nSO  - Cell Syst. 2021 Nov 17:S2405-4712(21)00419-1. doi: 10.1016/j.cels.2021.10.008.\\n\\nPMID- 32241274\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210422\\nLR  - 20210422\\nIS  - 1755-8794 (Electronic)\\nIS  - 1755-8794 (Linking)\\nVI  - 13\\nIP  - Suppl 5\\nDP  - 2020 Apr 3\\nTI  - SCNrank: spectral clustering for network-based ranking to reveal potential drug  targets and its application in pancreatic ductal adenocarcinoma.\\nPG  - 50\\nLID - 10.1186/s12920-020-0681-6 [doi]\\nLID - 50\\nAB  - BACKGROUND: Pancreatic ductal adenocarcinoma (PDAC) is the most common pancreatic  malignancy. Due to its wide heterogeneity, PDAC acts aggressively and responds  poorly to most chemotherapies, causing an urgent need for the development of new  therapeutic strategies. Cell lines have been used as the foundation for drug  development and disease modeling. CRISPR-Cas9 plays a key role in every step-in drug  discovery: from target identification and validation to preclinical cancer cell  testing. Using cell-line models and CRISPR-Cas9 technology together make drug target  prediction feasible. However, there is still a large gap between predicted results  and actionable targets in real tumors. Biological network models provide great modus  to mimic genetic interactions in real biological systems, which can benefit gene  perturbation studies and potential target identification for treating PDAC.  Nevertheless, building a network model that takes cell-line data and CRISPR-Cas9  data as input to accurately predict potential targets that will respond well on real  tissue remains unsolved. METHODS: We developed a novel algorithm \\'Spectral  Clustering for Network-based target Ranking\\' (SCNrank) that systematically  integrates three types of data: expression profiles from tumor tissue, normal tissue  and cell-line PDAC; protein-protein interaction network (PPI); and CRISPR-Cas9 data  to prioritize potential drug targets for PDAC. The whole algorithm can be classified  into three steps: 1. using STRING PPI network skeleton, SCNrank constructs  tissue-specific networks with PDAC tumor and normal pancreas tissues from expression  profiles; 2. With the same network skeleton, SCNrank constructs cell-line-specific  networks using the cell-line PDAC expression profiles and CRISPR-Cas 9 data from  pancreatic cancer cell-lines; 3. SCNrank applies a novel spectral clustering  approach to reduce data dimension and generate gene clusters that carry common  features from both networks. Finally, SCNrank applies a scoring scheme called  \\'Target Influence score\\' (TI), which estimates a given target\\'s influence towards  the cluster it belongs to, for scoring and ranking each drug target. RESULTS: We  applied SCNrank to analyze 263 expression profiles, CRPSPR-Cas9 data from 22  different pancreatic cancer cell-lines and the STRING protein-protein interaction  (PPI) network. With SCNrank, we successfully constructed an integrated tissue PDAC  network and an integrated cell-line PDAC network, both of which contain 4414  selected genes that are overexpressed in tumor tissue samples. After clustering,  4414 genes are distributed into 198 clusters, which include 367 targets of FDA  approved drugs. These drug targets are all scored and ranked by their TI scores,  which we defined to measure their influence towards the network. We validated  top-ranked targets in three aspects: Firstly, mapping them onto the existing  clinical drug targets of PDAC to measure the concordance. Secondly, we performed  enrichment analysis to these drug targets and the clusters there are within, to  reveal functional associations between clusters and PDAC; Thirdly, we performed  survival analysis for the top-ranked targets to connect targets with clinical  outcomes. Survival analysis reveals that overexpression of three top-ranked genes,  PGK1, HMMR and POLE2, significantly increases the risk of death in PDAC patients.  CONCLUSION: SCNrank is an unbiased algorithm that systematically integrates multiple  types of omics data to do potential drug target selection and ranking. SCNrank shows  great capability in predicting drug targets for PDAC. Pancreatic cancer-associated  gene candidates predicted by our SCNrank approach have the potential to guide  genetics-based anti-pancreatic drug discovery.\\nFAU - Liu, Enze\\nAU  - Liu E\\nAD  - Department of BioHealth Informatics, School of Informatics and Computing, Indiana  University-Purdue University, Indianapolis, IN, 46202, USA.\\nFAU - Zhang, Zhuang Zhuang\\nAU  - Zhang ZZ\\nAD  - Department of Toxicology and Cancer Biology, College of Medicine, University of  Kentucky, Lexington, KY, 40536, USA.\\nFAU - Cheng, Xiaolin\\nAU  - Cheng X\\nAD  - College of Pharmacy, Division of Medicinal Chemistry and Pharmacognosy, the Ohio  State University, Columbus, OH, 43210, USA.\\nFAU - Liu, Xiaoqi\\nAU  - Liu X\\nAD  - Department of Toxicology and Cancer Biology, College of Medicine, University of  Kentucky, Lexington, KY, 40536, USA. Xiaoqi.Liu@uky.edu.\\nFAU - Cheng, Lijun\\nAU  - Cheng L\\nAD  - Department of Biomedical informatics, College of medicine, the Ohio State  University, Columbus, OH, 43210, USA. Lijun.cheng@osumc.edu.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200403\\nTA  - BMC Med Genomics\\nJT  - BMC medical genomics\\nJID - 101319628\\nRN  - 0 (Antineoplastic Agents)\\nRN  - 0 (Biomarkers, Tumor)\\nSB  - IM\\nMH  - *Algorithms\\nMH  - Antineoplastic Agents/*therapeutic use\\nMH  - Biomarkers, Tumor/*antagonists & inhibitors/genetics\\nMH  - Carcinoma, Pancreatic Ductal/drug therapy/genetics/pathology\\nMH  - Gene Expression Profiling\\nMH  - Gene Expression Regulation, Neoplastic/*drug effects\\nMH  - Gene Regulatory Networks/*drug effects\\nMH  - Humans\\nMH  - *Molecular Targeted Therapy\\nMH  - Pancreatic Neoplasms/*drug therapy/genetics/pathology\\nMH  - Prognosis\\nMH  - Protein Interaction Maps\\nMH  - Survival Rate\\nPMC - PMC7119297\\nOTO - NOTNLM\\nOT  - *Drug target ranking\\nOT  - *Integrated network\\nOT  - *Protein-protein interaction network\\nOT  - *Spectral clustering\\nCOIS- The authorsÂ\\xa0declareÂ\\xa0that they have no competing interest.\\nEDAT- 2020/04/04 06:00\\nMHDA- 2021/04/23 06:00\\nCRDT- 2020/04/04 06:00\\nPHST- 2020/04/04 06:00 [entrez]\\nPHST- 2020/04/04 06:00 [pubmed]\\nPHST- 2021/04/23 06:00 [medline]\\nAID - 10.1186/s12920-020-0681-6 [pii]\\nAID - 681 [pii]\\nAID - 10.1186/s12920-020-0681-6 [doi]\\nPST - epublish\\nSO  - BMC Med Genomics. 2020 Apr 3;13(Suppl 5):50. doi: 10.1186/s12920-020-0681-6.\\n\\nPMID- 33833286\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211108\\nLR  - 20211108\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 11\\nIP  - 1\\nDP  - 2021 Apr 8\\nTI  - Single molecule network analysis identifies structural changes to caveolae and  scaffolds due to mutation of the caveolin-1 scaffolding domain.\\nPG  - 7810\\nLID - 10.1038/s41598-021-86770-6 [doi]\\nLID - 7810\\nAB  - Caveolin-1 (CAV1), the caveolae coat protein, also associates with non-caveolar  scaffold domains. Single molecule localization microscopy (SMLM) network analysis  distinguishes caveolae and three scaffold domains, hemispherical S2 scaffolds and  smaller S1B and S1A scaffolds. The caveolin scaffolding domain (CSD) is a highly  conserved hydrophobic region that mediates interaction of CAV1 with multiple  effector molecules. F92A/V94A mutation disrupts CSD function, however the structural  impact of CSD mutation on caveolae or scaffolds remains unknown. Here, SMLM network  analysis quantitatively shows that expression of the CAV1 CSD F92A/V94A mutant in  CRISPR/Cas CAV1 knockout MDA-MB-231 breast cancer cells reduces the size and volume  and enhances the elongation of caveolae and scaffold domains, with more pronounced  effects on S2 and S1B scaffolds. Convex hull analysis of the outer surface of the  CAV1 point clouds confirms the size reduction of CSD mutant CAV1 blobs and shows  that CSD mutation reduces volume variation amongst S2 and S1B CAV1 blobs at  increasing shrink values, that may reflect retraction of the CAV1 N-terminus towards  the membrane, potentially preventing accessibility of the CSD. Detection of point  mutation-induced changes to CAV1 domains highlights the utility of SMLM network  analysis for mesoscale structural analysis of oligomers in their native environment.\\nFAU - Wong, Timothy H\\nAU  - Wong TH\\nAD  - Life Sciences Institute, Department of Cellular and Physiological Sciences,  University of British Columbia, Vancouver, BC, V6T 1Z3, Canada.\\nFAU - Khater, Ismail M\\nAU  - Khater IM\\nAD  - School of Computing Science, Simon Fraser University, Burnaby, BC, V5A 1S6, Canada.\\nFAU - Joshi, Bharat\\nAU  - Joshi B\\nAD  - Life Sciences Institute, Department of Cellular and Physiological Sciences,  University of British Columbia, Vancouver, BC, V6T 1Z3, Canada.\\nFAU - Shahsavari, Mona\\nAU  - Shahsavari M\\nAD  - School of Computing Science, Simon Fraser University, Burnaby, BC, V5A 1S6, Canada.\\nFAU - Hamarneh, Ghassan\\nAU  - Hamarneh G\\nAD  - School of Computing Science, Simon Fraser University, Burnaby, BC, V5A 1S6, Canada.  hamarneh@sfu.ca.\\nFAU - Nabi, Ivan R\\nAU  - Nabi IR\\nAD  - Life Sciences Institute, Department of Cellular and Physiological Sciences,  University of British Columbia, Vancouver, BC, V6T 1Z3, Canada. irnabi@mail.ubc.ca.\\nAD  - School of Biomedical Engineering, University of British Columbia, Vancouver, BC, V6T  1Z3, Canada. irnabi@mail.ubc.ca.\\nLA  - eng\\nGR  - PJT-159845/CIHR/Canada\\nGR  - PJT-156424/CIHR/Canada\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210408\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nRN  - 0 (CAV1 protein, human)\\nRN  - 0 (Caveolin 1)\\nSB  - IM\\nMH  - Caveolin 1/*chemistry\\nMH  - Cell Line\\nMH  - Humans\\nMH  - Mutation\\nMH  - Protein Conformation\\nMH  - Protein Domains/*genetics\\nPMC - PMC8032680\\nCOIS- The authors declare no competing interests.\\nEDAT- 2021/04/10 06:00\\nMHDA- 2021/11/09 06:00\\nCRDT- 2021/04/09 06:03\\nPHST- 2020/10/15 00:00 [received]\\nPHST- 2021/03/15 00:00 [accepted]\\nPHST- 2021/04/09 06:03 [entrez]\\nPHST- 2021/04/10 06:00 [pubmed]\\nPHST- 2021/11/09 06:00 [medline]\\nAID - 10.1038/s41598-021-86770-6 [pii]\\nAID - 86770 [pii]\\nAID - 10.1038/s41598-021-86770-6 [doi]\\nPST - epublish\\nSO  - Sci Rep. 2021 Apr 8;11(1):7810. doi: 10.1038/s41598-021-86770-6.\\n\\nPMID- 32294089\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200706\\nLR  - 20200706\\nIS  - 1932-6203 (Electronic)\\nIS  - 1932-6203 (Linking)\\nVI  - 15\\nIP  - 4\\nDP  - 2020\\nTI  - Mapping the coevolution, leadership and financing of research on viral vectors,  RNAi, CRISPR/Cas9 and other genomic editing technologies.\\nPG  - e0227593\\nLID - 10.1371/journal.pone.0227593 [doi]\\nLID - e0227593\\nAB  - Genomic editing technologies are developing rapidly, promising significant  developments for biomedicine, agriculture and other fields. In the present  investigation, we analyzed and compared the process of innovation for six genomic  technologies: viral vectors, RNAi, TALENs, meganucleases, ZFNs and CRISPR/Cas  including the profile of the main research institutions and their funders, to  understand how innovation evolved and what institutions influenced research  trajectories. A Web of Science search of papers on viral vectors RNAi, CRISPR/Cas,  TALENs, ZFNs and meganucleases was used to build a citation network of 16,746  papers. An analysis of network clustering combined with text mining was performed.  For viral vectors, a long-term process of incremental innovation was identified,  which was largely publicly funded in the United States and the European Union. The  trajectory of RNAi research included clusters related to the study of RNAi as a  biological phenomenon and its use in functional genomics, biomedicine and pest  control. A British philanthropic organization and a US pharmaceutical company played  a key role in the development of basic RNAi research and clinical application  respectively, in addition to government and academic institutions. In the case of  CRISPR/Cas research, basic science discoveries led to the technical improvements,  and these two in turn provided the information required for the development of  biomedical, agricultural, livestock and industrial applications. The trajectory of  CRISPR/Cas research exhibits a geopolitical division of the investigation efforts  between the US, as the main producer and funder of basic research and technical  improvements, and Chinese research institutions increasingly leading applied  research. Our results reflect a change in the model for financing science, with  reduced public financing for basic science and applied research on publicly funded  technological developments in the US, and the emergence of China as a scientific  superpower, with implications for the development of applications of genomic  technologies.\\nFAU - Fajardo-Ortiz, David\\nAU  - Fajardo-Ortiz D\\nAUID- ORCID: 0000-0002-7657-7514\\nAD  - Research System and Science Dynamics research area, Deutsche Zentrum fÃ¼r Hochschul-  und Wissenschaftsforschung (DZHW), Berlin, Germany.\\nFAU - Shattuck, Annie\\nAU  - Shattuck A\\nAD  - Department of Geography, Indiana University, Bloomington, Indiana, United States of  America.\\nFAU - Hornbostel, Stefan\\nAU  - Hornbostel S\\nAD  - Research System and Science Dynamics research area, Deutsche Zentrum fÃ¼r Hochschul-  und Wissenschaftsforschung (DZHW), Berlin, Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200415\\nTA  - PLoS One\\nJT  - PloS one\\nJID - 101285081\\nSB  - IM\\nMH  - Biomedical Research/economics/methods/organization & administration/*trends\\nMH  - Biomedical Technology/economics/methods/organization & administration/*trends\\nMH  - CRISPR-Cas Systems\\nMH  - China\\nMH  - Financing, Organized/economics/methods/*trends\\nMH  - Gene Editing/economics/methods/*trends\\nMH  - Genetic Vectors\\nMH  - Inventions/economics/*trends\\nMH  - Leadership\\nMH  - Politics\\nMH  - RNA Interference\\nMH  - United States\\nMH  - Viruses/genetics\\nPMC - PMC7159216\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2020/04/16 06:00\\nMHDA- 2020/07/07 06:00\\nCRDT- 2020/04/16 06:00\\nPHST- 2019/12/19 00:00 [received]\\nPHST- 2020/03/24 00:00 [accepted]\\nPHST- 2020/04/16 06:00 [entrez]\\nPHST- 2020/04/16 06:00 [pubmed]\\nPHST- 2020/07/07 06:00 [medline]\\nAID - PONE-D-19-35217 [pii]\\nAID - 10.1371/journal.pone.0227593 [doi]\\nPST - epublish\\nSO  - PLoS One. 2020 Apr 15;15(4):e0227593. doi: 10.1371/journal.pone.0227593. eCollection  2020.\\n\\nPMID- 34661427\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220215\\nLR  - 20220215\\nIS  - 2573-1602 (Electronic)\\nIS  - 2573-1599 (Linking)\\nVI  - 4\\nIP  - 5\\nDP  - 2021 Oct\\nTI  - Determination of Factors Driving the Genome Editing Field in the CRISPR Era Using  Bibliometrics.\\nPG  - 728-738\\nLID - 10.1089/crispr.2021.0001 [doi]\\nAB  - Over the past two decades, the discovery of CRISPR-Cas immune systems and the  repurposing of their effector nucleases as biotechnological tools have  revolutionized genome editing. The corresponding work has been captured by 90,000  authors representing 7,600 affiliations in 126 countries, who have published more  than 19,000 papers spanning medicine, agriculture, and biotechnology. Here, we use  tech mining and an integrated bibliometric and networks framework to investigate the  CRISPR literature over three time periods. The analysis identified seminal papers,  leading authors, influential journals, and rising applications and topics  interconnected through collaborative networks. A core set of foundational topics  gave rise to diverging avenues of research and applications, reflecting a bona fide  disruptive emerging technology. This analysis illustrates how bibliometrics can  identify key factors, decipher rising trends, and untangle emerging applications and  technologies that dynamically shape a morphing field, and provides insights into the  trajectory of genome editing.\\nFAU - Huang, Ying\\nAU  - Huang Y\\nAD  - Center for Studies of Information Resources, School of Information Management, Wuhan  University, Wuhan, P.R. China; Bioprocessing and Nutrition Sciences, North Carolina  State University, Raleigh, North Carolina, USA.\\nAD  - Center for Science, Technology and Education Assessment (CSTEA), Wuhan University,  Wuhan, P.R. China; Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh, North Carolina, USA.\\nAD  - Department of MSI, Centre for R&D Monitoring (ECOOM), KU Leuven, Leuven, Belgium;  Bioprocessing and Nutrition Sciences, North Carolina State University, Raleigh,  North Carolina, USA.\\nFAU - Zhang, Yi\\nAU  - Zhang Y\\nAD  - Australian Artificial Intelligence Institute, Faculty of Engineering and Information  Technology, University of Technology Sydney, Sydney, Australia; Bioprocessing and  Nutrition Sciences, North Carolina State University, Raleigh, North Carolina, USA.\\nFAU - Wu, Mengjia\\nAU  - Wu M\\nAD  - Australian Artificial Intelligence Institute, Faculty of Engineering and Information  Technology, University of Technology Sydney, Sydney, Australia; Bioprocessing and  Nutrition Sciences, North Carolina State University, Raleigh, North Carolina, USA.\\nFAU - Porter, Alan\\nAU  - Porter A\\nAD  - Search Technology, Inc., Norcross, Georgia, USA; Bioprocessing and Nutrition  Sciences, North Carolina State University, Raleigh, North Carolina, USA.\\nAD  - Program in Science, Technology and Innovation Policy, Georgia Institute of  Technology, Atlanta, Georgia, USA; and Bioprocessing and Nutrition Sciences, North  Carolina State University, Raleigh, North Carolina, USA.\\nFAU - Barrangou, Rodolphe\\nAU  - Barrangou R\\nAD  - Department of Food, Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh, North Carolina, USA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPL  - United States\\nTA  - CRISPR J\\nJT  - The CRISPR journal\\nJID - 101738191\\nSB  - IM\\nMH  - Animals\\nMH  - *Bibliometrics\\nMH  - *CRISPR-Cas Systems\\nMH  - Gene Editing/*trends\\nMH  - Humans\\nEDAT- 2021/10/19 06:00\\nMHDA- 2022/02/16 06:00\\nCRDT- 2021/10/18 12:18\\nPHST- 2021/10/18 12:18 [entrez]\\nPHST- 2021/10/19 06:00 [pubmed]\\nPHST- 2022/02/16 06:00 [medline]\\nAID - 10.1089/crispr.2021.0001 [doi]\\nPST - ppublish\\nSO  - CRISPR J. 2021 Oct;4(5):728-738. doi: 10.1089/crispr.2021.0001.\\n\\nPMID- 31451697\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191216\\nLR  - 20210110\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 10\\nIP  - 1\\nDP  - 2019 Aug 26\\nTI  - Engineered CRISPRa enables programmable eukaryote-like gene activation in bacteria.\\nPG  - 3693\\nLID - 10.1038/s41467-019-11479-0 [doi]\\nLID - 3693\\nAB  - Transcriptional regulation by nuclease-deficient CRISPR/Cas is a popular and  valuable tool for routine control of gene expression. CRISPR interference in  bacteria can be reliably achieved with high efficiencies. Yet, options for CRISPR  activation (CRISPRa) remained limited in flexibility and activity because they  relied on Ïƒ(70) promoters. Here we report a eukaryote-like bacterial CRISPRa system  based on Ïƒ(54)-dependent promoters, which supports long distance, and hence  multi-input regulation with high dynamic ranges. Our CRISPRa device can activate  Ïƒ(54)-dependent promoters with biotechnology relevance in non-model bacteria. It  also supports orthogonal gene regulation on multiple levels. Combining our CRISPRa  with dxCas9 further expands flexibility in DNA targeting, and boosts dynamic ranges  into regimes that enable construction of cascaded CRISPRa circuits.  Application-wise, we construct a reusable scanning platform for readily optimizing  metabolic pathways without library reconstructions. This eukaryote-like CRISPRa  system is therefore a powerful and versatile synthetic biology tool for diverse  research and industrial applications.\\nFAU - Liu, Yang\\nAU  - Liu Y\\nAD  - School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3FF, UK.\\nAD  - Centre for Synthetic and Systems Biology, University of Edinburgh, Edinburgh, EH9  3FF, UK.\\nFAU - Wan, Xinyi\\nAU  - Wan X\\nAD  - School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3FF, UK.\\nAD  - Centre for Synthetic and Systems Biology, University of Edinburgh, Edinburgh, EH9  3FF, UK.\\nFAU - Wang, Baojun\\nAU  - Wang B\\nAUID- ORCID: 0000-0002-4858-8937\\nAD  - School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3FF, UK.  baojun.wang@ed.ac.uk.\\nAD  - Centre for Synthetic and Systems Biology, University of Edinburgh, Edinburgh, EH9  3FF, UK. baojun.wang@ed.ac.uk.\\nLA  - eng\\nGR  - WT_/Wellcome Trust/United Kingdom\\nGR  - BB/N007212/1/BB_/Biotechnology and Biological Sciences Research Council/United  Kingdom\\nGR  - MR/S018875/1/MRC_/Medical Research Council/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190826\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (Escherichia coli Proteins)\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (rpoN protein, E coli)\\nRN  - EC 2.7.7.6 (RNA Polymerase Sigma 54)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Escherichia coli/*genetics/metabolism\\nMH  - Escherichia coli Proteins/*genetics\\nMH  - Gene Expression/genetics\\nMH  - Gene Expression Regulation/genetics\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Plasmids/genetics\\nMH  - Promoter Regions, Genetic/genetics\\nMH  - Protein Engineering/*methods\\nMH  - RNA Polymerase Sigma 54/*genetics\\nMH  - RNA, Guide/genetics\\nMH  - Transcriptional Activation/genetics\\nPMC - PMC6710252\\nCOIS- The authors declare no competing interests.\\nEDAT- 2019/08/28 06:00\\nMHDA- 2019/12/18 06:00\\nCRDT- 2019/08/28 06:00\\nPHST- 2019/01/10 00:00 [received]\\nPHST- 2019/07/16 00:00 [accepted]\\nPHST- 2019/08/28 06:00 [entrez]\\nPHST- 2019/08/28 06:00 [pubmed]\\nPHST- 2019/12/18 06:00 [medline]\\nAID - 10.1038/s41467-019-11479-0 [pii]\\nAID - 11479 [pii]\\nAID - 10.1038/s41467-019-11479-0 [doi]\\nPST - epublish\\nSO  - Nat Commun. 2019 Aug 26;10(1):3693. doi: 10.1038/s41467-019-11479-0.\\n\\nPMID- 28327544\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20181109\\nLR  - 20191210\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 8\\nDP  - 2017 Mar 22\\nTI  - FMNL formins boost lamellipodial force generation.\\nPG  - 14832\\nLID - 10.1038/ncomms14832 [doi]\\nLID - 14832\\nAB  - Migration frequently involves Rac-mediated protrusion of lamellipodia, formed by  Arp2/3 complex-dependent branching thought to be crucial for force generation and  stability of these networks. The formins FMNL2 and FMNL3 are Cdc42 effectors  targeting to the lamellipodium tip and shown here to nucleate and elongate actin  filaments with complementary activities in vitro. In migrating B16-F1 melanoma  cells, both formins contribute to the velocity of lamellipodium protrusion. Loss of  FMNL2/3 function in melanoma cells and fibroblasts reduces lamellipodial width,  actin filament density and -bundling, without changing patterns of Arp2/3 complex  incorporation. Strikingly, in melanoma cells, FMNL2/3 gene inactivation almost  completely abolishes protrusion forces exerted by lamellipodia and modifies their  ultrastructural organization. Consistently, CRISPR/Cas-mediated depletion of FMNL2/3  in fibroblasts reduces both migration and capability of cells to move against  viscous media. Together, we conclude that force generation in lamellipodia strongly  depends on FMNL formin activity, operating in addition to Arp2/3 complex-dependent  filament branching.\\nFAU - Kage, Frieda\\nAU  - Kage F\\nAD  - Division of Molecular Cell Biology, Zoological Institute, Technische UniversitÃ¤t  Braunschweig, Spielmannstrasse 7, 38106 Braunschweig, Germany.\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nFAU - Winterhoff, Moritz\\nAU  - Winterhoff M\\nAD  - Institute for Biophysical Chemistry, Hannover Medical School, Carl-Neuberg-Strasse  1, 30625 Hannover, Germany.\\nFAU - Dimchev, Vanessa\\nAU  - Dimchev V\\nAD  - Division of Molecular Cell Biology, Zoological Institute, Technische UniversitÃ¤t  Braunschweig, Spielmannstrasse 7, 38106 Braunschweig, Germany.\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nFAU - Mueller, Jan\\nAU  - Mueller J\\nAD  - Institute of Science and Technology Austria, Am Campus 1, 3400 Klosterneuburg,  Austria.\\nFAU - Thalheim, Tobias\\nAU  - Thalheim T\\nAD  - Soft Matter Physics Group, Institut fÃ¼r experimentelle Physik I, Leipzig University,  LinnÃ©straÃŸe 5, 04103 Leipzig, Germany.\\nFAU - Freise, Anika\\nAU  - Freise A\\nAD  - Division of Molecular Cell Biology, Zoological Institute, Technische UniversitÃ¤t  Braunschweig, Spielmannstrasse 7, 38106 Braunschweig, Germany.\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nFAU - BrÃ¼hmann, Stefan\\nAU  - BrÃ¼hmann S\\nAD  - Institute for Biophysical Chemistry, Hannover Medical School, Carl-Neuberg-Strasse  1, 30625 Hannover, Germany.\\nFAU - Kollasser, Jana\\nAU  - Kollasser J\\nAD  - Biomedical Institute, BRIC, University of Copenhagen, DK-2200 Copenhagen, Denmark.\\nAD  - Department of Cell Biology, Helmholtz Centre for Infection Research, Inhoffenstrasse  7, 38124 Braunschweig, Germany.\\nFAU - Block, Jennifer\\nAU  - Block J\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nFAU - Dimchev, Georgi\\nAU  - Dimchev G\\nAD  - Division of Molecular Cell Biology, Zoological Institute, Technische UniversitÃ¤t  Braunschweig, Spielmannstrasse 7, 38106 Braunschweig, Germany.\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nFAU - Geyer, Matthias\\nAU  - Geyer M\\nAD  - Institute of Innate Immunity, Department of Structural Immunology, University of  Bonn, Sigmund-Freud-Strasse 25, 53127 Bonn, Germany.\\nFAU - Schnittler, Hans-Joachim\\nAU  - Schnittler HJ\\nAD  - Institute of Anatomy and Vascular Biology, WestfÃ¤lische Wilhelms-UniversitÃ¤t  MÃ¼nster, Vesaliusweg 2-4, 48149 MÃ¼nster, Germany.\\nFAU - Brakebusch, Cord\\nAU  - Brakebusch C\\nAD  - Biomedical Institute, BRIC, University of Copenhagen, DK-2200 Copenhagen, Denmark.\\nFAU - Stradal, Theresia E B\\nAU  - Stradal TE\\nAD  - Department of Cell Biology, Helmholtz Centre for Infection Research, Inhoffenstrasse  7, 38124 Braunschweig, Germany.\\nFAU - Carlier, Marie-France\\nAU  - Carlier MF\\nAD  - Cytoskeleton Dynamics and Motility Group, Laboratoire d\\'Enzymologie et Biochimie  Structurales, Centre de Recherche de Gif, CNRS, Gif-sur-Yvette 91198, France.\\nFAU - Sixt, Michael\\nAU  - Sixt M\\nAD  - Institute of Science and Technology Austria, Am Campus 1, 3400 Klosterneuburg,  Austria.\\nFAU - KÃ¤s, Josef\\nAU  - KÃ¤s J\\nAD  - Soft Matter Physics Group, Institut fÃ¼r experimentelle Physik I, Leipzig University,  LinnÃ©straÃŸe 5, 04103 Leipzig, Germany.\\nFAU - Faix, Jan\\nAU  - Faix J\\nAD  - Institute for Biophysical Chemistry, Hannover Medical School, Carl-Neuberg-Strasse  1, 30625 Hannover, Germany.\\nFAU - Rottner, Klemens\\nAU  - Rottner K\\nAUID- ORCID: 0000-0003-4244-4198\\nAD  - Division of Molecular Cell Biology, Zoological Institute, Technische UniversitÃ¤t  Braunschweig, Spielmannstrasse 7, 38106 Braunschweig, Germany.\\nAD  - Molecular Cell Biology Group, Helmholtz Centre for Infection Research,  Inhoffenstrasse 7, 38124 Braunschweig, Germany.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170322\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (FMNL2 protein, mouse)\\nRN  - 0 (FMNL3 protein, mouse)\\nRN  - 0 (Formins)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Animals\\nMH  - Biomechanical Phenomena\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Cell Movement\\nMH  - Fibroblasts/metabolism\\nMH  - Formins\\nMH  - Gene Knockdown Techniques\\nMH  - Intracellular Signaling Peptides and Proteins/*metabolism\\nMH  - Melanoma, Experimental/pathology\\nMH  - Mice\\nMH  - Mice, Knockout\\nMH  - Models, Biological\\nMH  - NIH 3T3 Cells\\nMH  - Phenotype\\nMH  - Polymerization\\nMH  - Proteins/*metabolism\\nMH  - Pseudopodia/*metabolism/ultrastructure\\nMH  - RNA Interference\\nPMC - PMC5364437\\nCOIS- The authors declare no competing financial interests.\\nEDAT- 2017/03/23 06:00\\nMHDA- 2018/11/10 06:00\\nCRDT- 2017/03/23 06:00\\nPHST- 2017/01/12 00:00 [received]\\nPHST- 2017/02/03 00:00 [accepted]\\nPHST- 2017/03/23 06:00 [entrez]\\nPHST- 2017/03/23 06:00 [pubmed]\\nPHST- 2018/11/10 06:00 [medline]\\nAID - ncomms14832 [pii]\\nAID - 10.1038/ncomms14832 [doi]\\nPST - epublish\\nSO  - Nat Commun. 2017 Mar 22;8:14832. doi: 10.1038/ncomms14832.\\n\\nPMID- 22188759\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20120612\\nLR  - 20211021\\nIS  - 1745-6150 (Electronic)\\nIS  - 1745-6150 (Linking)\\nVI  - 6\\nIP  - 1\\nDP  - 2011 Dec 21\\nTI  - CRISPR loci reveal networks of gene exchange in archaea.\\nPG  - 65\\nLID - 10.1186/1745-6150-6-65 [doi]\\nAB  - BACKGROUND: CRISPR (Clustered, Regularly, Interspaced, Short, Palindromic Repeats)  loci provide prokaryotes with an adaptive immunity against viruses and other mobile  genetic elements. CRISPR arrays can be transcribed and processed into small crRNA  molecules, which are then used by the cell to target the foreign nucleic acid. Since  spacers are accumulated by active CRISPR/Cas systems, the sequences of these spacers  provide a record of the past \"infection history\" of the organism. RESULTS: Here we  analyzed all currently known spacers present in archaeal genomes and identified  their source by DNA similarity. While nearly 50% of archaeal spacers matched mobile  genetic elements, such as plasmids or viruses, several others matched chromosomal  genes of other organisms, primarily other archaea. Thus, networks of gene exchange  between archaeal species were revealed by the spacer analysis, including many cases  of inter-genus and inter-species gene transfer events. Spacers that recognize viral  sequences tend to be located further away from the leader sequence, implying that  there exists a selective pressure for their retention. CONCLUSIONS: CRISPR spacers  provide direct evidence for extensive gene exchange in archaea, especially within  genera, and support the current dogma where the primary role of the CRISPR/Cas  system is anti-viral and anti-plasmid defense. OPEN PEER REVIEW: This article was  reviewed by: Profs. W. Ford Doolittle, John van der Oost, Christa Schleper  (nominated by board member Prof. J Peter Gogarten).\\nFAU - Brodt, Avital\\nAU  - Brodt A\\nAD  - Department of Molecular Microbiology and Biotechnology, George S, Wise Faculty of  Life Sciences, Tel Aviv University, Ramat Aviv, 69978, Israel. urigo@tauex.tau.ac.il\\nFAU - Lurie-Weinberger, Mor N\\nAU  - Lurie-Weinberger MN\\nFAU - Gophna, Uri\\nAU  - Gophna U\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20111221\\nTA  - Biol Direct\\nJT  - Biology direct\\nJID - 101258412\\nSB  - IM\\nMH  - Archaea/*genetics\\nMH  - Base Sequence\\nMH  - *Gene Transfer, Horizontal\\nMH  - *Genetic Loci\\nMH  - *Genome, Archaeal\\nMH  - Inverted Repeat Sequences\\nPMC - PMC3285040\\nEDAT- 2011/12/23 06:00\\nMHDA- 2012/06/13 06:00\\nCRDT- 2011/12/23 06:00\\nPHST- 2011/08/30 00:00 [received]\\nPHST- 2011/12/21 00:00 [accepted]\\nPHST- 2011/12/23 06:00 [entrez]\\nPHST- 2011/12/23 06:00 [pubmed]\\nPHST- 2012/06/13 06:00 [medline]\\nAID - 1745-6150-6-65 [pii]\\nAID - 10.1186/1745-6150-6-65 [doi]\\nPST - epublish\\nSO  - Biol Direct. 2011 Dec 21;6(1):65. doi: 10.1186/1745-6150-6-65.\\n\\nPMID- 29537074\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190711\\nLR  - 20190711\\nIS  - 1097-0290 (Electronic)\\nIS  - 0006-3592 (Linking)\\nVI  - 115\\nIP  - 7\\nDP  - 2018 Jul\\nTI  - Combinatorial pathway engineering using type I-E CRISPR interference.\\nPG  - 1878-1883\\nLID - 10.1002/bit.26589 [doi]\\nAB  - Optimization of metabolic flux is a difficult and time-consuming process that often  involves changing the expression levels of multiple genes simultaneously. While some  pathways have a known rate limiting step, more complex metabolic networks can  require a trial-and-error approach of tuning the expression of multiple genes to  achieve a desired distribution of metabolic resources. Here we present an efficient  method for generating expression diversity on a combinatorial scale using CRISPR  interference. We use a modified native Escherichia coli Type I-E CRISPR-Cas system  and an iterative cloning strategy for construction of guide RNA arrays. This  approach allowed us to build a combinatorial gene expression library three orders of  magnitude larger than previous studies. In less than 1 month, we generated âˆ¼12,000  combinatorial gene expression variants that target six different genes and screened  these variants for increased malonyl-CoA flux and 3-hydroxypropionate (3HP)  production. We were able to identify a set of variants that exhibited a significant  increase in malonyl-CoA flux and up to a 98% increase in 3HP production. This  approach provides a fast and easy-to-implement strategy for engineering metabolic  pathway flux for development of industrially relevant strains, as well as  investigation of fundamental biological questions.\\nCI  - Â© 2018 Wiley Periodicals, Inc.\\nFAU - Tarasava, Katia\\nAU  - Tarasava K\\nAD  - Chemical and Biological Engineering Department, University of Colorado Boulder,  Boulder, Colorado.\\nFAU - Liu, Rongming\\nAU  - Liu R\\nAD  - Renewable and Sustainable Energy Institute (RASEI), University of Colorado Boulder,  Boulder, Colorado.\\nFAU - Garst, Andrew\\nAU  - Garst A\\nAD  - Muse Biotechnology, Boulder, Colorado.\\nFAU - Gill, Ryan T\\nAU  - Gill RT\\nAUID- ORCID: 0000-0002-4545-233X\\nAD  - Chemical and Biological Engineering Department, University of Colorado Boulder,  Boulder, Colorado.\\nAD  - Renewable and Sustainable Energy Institute (RASEI), University of Colorado Boulder,  Boulder, Colorado.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20180330\\nPL  - United States\\nTA  - Biotechnol Bioeng\\nJT  - Biotechnology and bioengineering\\nJID - 7502021\\nRN  - 33X04XA5AT (Lactic Acid)\\nRN  - 524-14-1 (Malonyl Coenzyme A)\\nRN  - C4ZF6XLD2X (hydracrylic acid)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - Escherichia coli/*genetics/*metabolism\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - Genetic Variation\\nMH  - Lactic Acid/*analogs & derivatives/metabolism\\nMH  - Malonyl Coenzyme A/metabolism\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/*genetics\\nMH  - Recombination, Genetic\\nOTO - NOTNLM\\nOT  - *3-hydroxypropionic acid\\nOT  - *CRISPR interference\\nOT  - *combinatorial gene expression\\nOT  - *malonyl-coA\\nOT  - *metabolic flux engineering\\nEDAT- 2018/03/15 06:00\\nMHDA- 2019/07/12 06:00\\nCRDT- 2018/03/15 06:00\\nPHST- 2017/11/24 00:00 [received]\\nPHST- 2018/02/08 00:00 [revised]\\nPHST- 2018/03/06 00:00 [accepted]\\nPHST- 2018/03/15 06:00 [pubmed]\\nPHST- 2019/07/12 06:00 [medline]\\nPHST- 2018/03/15 06:00 [entrez]\\nAID - 10.1002/bit.26589 [doi]\\nPST - ppublish\\nSO  - Biotechnol Bioeng. 2018 Jul;115(7):1878-1883. doi: 10.1002/bit.26589. Epub 2018 Mar  30.\\n\\nPMID- 32893479\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220131\\nLR  - 20220131\\nIS  - 1601-183X (Electronic)\\nIS  - 1601-1848 (Print)\\nIS  - 1601-183X (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2021 Feb\\nTI  - A long non-coding RNA (Lrap) modulates brain gene expression and levels of alcohol  consumption in rats.\\nPG  - e12698\\nLID - 10.1111/gbb.12698 [doi]\\nLID - e12698\\nAB  - LncRNAs are important regulators of quantitative and qualitative features of the  transcriptome. We have used QTL and other statistical analyses to identify a gene  coexpression module associated with alcohol consumption. The \"hub gene\" of this  module, Lrap (Long non-coding RNA for alcohol preference), was an unannotated  transcript resembling a lncRNA. We used partial correlation analyses to establish  that Lrap is a major contributor to the integrity of the coexpression module. Using  CRISPR/Cas9 technology, we disrupted an exon of Lrap in Wistar rats. Measures of  alcohol consumption in wild type, heterozygous and knockout rats showed that  disruption of Lrap produced increases in alcohol consumption/alcohol preference. The  disruption of Lrap also produced changes in expression of over 700 other  transcripts. Furthermore, it became apparent that Lrap may have a function in  alternative splicing of the affected transcripts. The GO category of \"Response to  Ethanol\" emerged as one of the top candidates in an enrichment analysis of the  differentially expressed transcripts. We validate the role of Lrap as a mediator of  alcohol consumption by rats, and also implicate Lrap as a modifier of the expression  and splicing of a large number of brain transcripts. A defined subset of these  transcripts significantly impacts alcohol consumption by rats (and possibly humans).  Our work shows the pleiotropic nature of non-coding elements of the genome, the  power of network analysis in identifying the critical elements influencing  phenotypes, and the fact that not all changes produced by genetic editing are  critical for the concomitant changes in phenotype.\\nCI  - Â© 2020 The Authors. Genes, Brain and Behavior published by International Behavioural  and Neural Genetics Society and John Wiley & Sons Ltd.\\nFAU - Saba, Laura M\\nAU  - Saba LM\\nAD  - Department of Pharmaceutical Sciences, Skaggs School of Pharmacy & Pharmaceutical  Sciences, University of Colorado Anschutz Medical Campus, Aurora, Colorado, USA.\\nFAU - Hoffman, Paula L\\nAU  - Hoffman PL\\nAD  - Department of Pharmaceutical Sciences, Skaggs School of Pharmacy & Pharmaceutical  Sciences, University of Colorado Anschutz Medical Campus, Aurora, Colorado, USA.\\nAD  - Department of Pharmacology, School of Medicine, University of Colorado Anschutz  Medical Campus, Aurora, Colorado, USA.\\nFAU - Homanics, Gregg E\\nAU  - Homanics GE\\nAD  - Departments of Anesthesiology, Neurobiology and Pharmacology & Chemical Biology,  University of Pittsburgh, Pittsburgh, Pennsylvania, USA.\\nFAU - Mahaffey, Spencer\\nAU  - Mahaffey S\\nAD  - Department of Pharmaceutical Sciences, Skaggs School of Pharmacy & Pharmaceutical  Sciences, University of Colorado Anschutz Medical Campus, Aurora, Colorado, USA.\\nFAU - Daulatabad, Swapna Vidhur\\nAU  - Daulatabad SV\\nAD  - Department of BioHealth Informatics, Indiana University School of Informatics and  Computing, Indiana University-Purdue University Indianapolis, Indianapolis, Indiana,  USA.\\nFAU - Janga, Sarath Chandra\\nAU  - Janga SC\\nAD  - Department of BioHealth Informatics, Indiana University School of Informatics and  Computing, Indiana University-Purdue University Indianapolis, Indianapolis, Indiana,  USA.\\nAD  - Department of Medical and Molecular Genetics, Indiana University School of Medicine,  Indianapolis, Indiana, USA.\\nAD  - Center for Computational Biology and Bioinformatics, Indiana University School of  Medicine, Indianapolis, Indiana, USA.\\nFAU - Tabakoff, Boris\\nAU  - Tabakoff B\\nAUID- ORCID: 0000-0001-5132-5030\\nAD  - Department of Pharmaceutical Sciences, Skaggs School of Pharmacy & Pharmaceutical  Sciences, University of Colorado Anschutz Medical Campus, Aurora, Colorado, USA.\\nLA  - eng\\nGR  - R37 AA010422/AA/NIAAA NIH HHS/United States\\nGR  - R24 AA013162/AA/NIAAA NIH HHS/United States\\nGR  - R01 GM123314/GM/NIGMS NIH HHS/United States\\nGR  - P30 DA044223/DA/NIDA NIH HHS/United States\\nGR  - U01 AA020889/AA/NIAAA NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Review\\nDEP - 20201019\\nTA  - Genes Brain Behav\\nJT  - Genes, brain, and behavior\\nJID - 101129617\\nRN  - 0 (RNA, Long Noncoding)\\nSB  - IM\\nMH  - Alcohol Drinking/*genetics/physiopathology\\nMH  - Animals\\nMH  - Brain/*metabolism\\nMH  - Quantitative Trait Loci\\nMH  - RNA, Long Noncoding/*genetics/metabolism\\nMH  - Rats\\nMH  - Rats, Wistar\\nMH  - Transcriptome\\nPMC - PMC7900948\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas\\nOT  - *brain RNA expression networks\\nOT  - *genetic modification\\nOT  - *long non-coding RNA\\nOT  - *predisposing factors\\nOT  - *quantitative genetics\\nOT  - *recombinant inbred rat strains\\nOT  - *systems genetics\\nOT  - *transcriptome\\nOT  - *voluntary alcohol consumption\\nEDAT- 2020/09/08 06:00\\nMHDA- 2022/02/01 06:00\\nCRDT- 2020/09/07 06:02\\nPHST- 2020/06/09 00:00 [received]\\nPHST- 2020/08/20 00:00 [revised]\\nPHST- 2020/09/01 00:00 [accepted]\\nPHST- 2020/09/08 06:00 [pubmed]\\nPHST- 2022/02/01 06:00 [medline]\\nPHST- 2020/09/07 06:02 [entrez]\\nAID - GBB12698 [pii]\\nAID - 10.1111/gbb.12698 [doi]\\nPST - ppublish\\nSO  - Genes Brain Behav. 2021 Feb;20(2):e12698. doi: 10.1111/gbb.12698. Epub 2020 Oct 19.\\n\\nPMID- 35135438\\nOWN - NLM\\nSTAT- Publisher\\nLR  - 20220209\\nIS  - 2164-5701 (Electronic)\\nIS  - 2164-5698 (Linking)\\nDP  - 2022 Feb 9\\nTI  - Genome editing techniques in plants: a comprehensive review and future prospects  toward zero hunger.\\nPG  - 1-15\\nLID - 10.1080/21645698.2021.2021724 [doi]\\nAB  - Promoting sustainable agriculture and improving nutrition are the main united nation  sustainable development goals by 2030. New technologies are requested to achieve  zero hunger, and genome editing technology is the most promising one. In the last  decade, genome editing using the CRISPR/Cas system has attracted researcher as a  safer and easy tool for genome editing in several living organisms. GE has  revolutionized the field of agriculture by improving biotic and abiotic stresses and  yield improvement. GE technologies were developed fast lately to avoid the obstacles  that face GM crops. GE technology, depending on site directed nuclease (SDN), is  divided into three categories according to the modification methods. Developing  transgenic-free edited plants without introducing foreign DNA meet the acceptance  and regulatory ratification of several countries. There are several ongoing efforts  from different countries are rapidly expanding to adopt the current technological  innovations. This review summarizes the different GE technologies and their  application as a way to help in ending hunger.\\nFAU - Abdallah, Naglaa A\\nAU  - Abdallah NA\\nAUID- ORCID: 0000-0002-9290-5944\\nAD  - Department of Genetics Faculty of Agriculture, Cairo University, Cairo, Egypt.\\nAD  - National Biotechnology Network of Expertise, ASRT, Egypt.\\nFAU - Hamwieh, Aladdin\\nAU  - Hamwieh A\\nAD  - Department of Biotechnology, International Centre for Agricultural Research in the  Dry Areas (ICARDA), Giza, Egypt.\\nFAU - Radwan, Khaled\\nAU  - Radwan K\\nAD  - National Biotechnology Network of Expertise, ASRT, Egypt.\\nAD  - Department of Biotechnology, Agricultural Genetic Engineering Research Institute  (AGERI), ARC, Giza, Egypt.\\nFAU - Fouad, Nourhan\\nAU  - Fouad N\\nAD  - Department of Biotechnology, International Centre for Agricultural Research in the  Dry Areas (ICARDA), Giza, Egypt.\\nFAU - Prakash, Channapatna\\nAU  - Prakash C\\nAD  - Tuskegee University, Tuskegee, AL, USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20220209\\nPL  - United States\\nTA  - GM Crops Food\\nJT  - GM crops & food\\nJID - 101572655\\nSB  - IM\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - base editing\\nOT  - crop improvenet\\nOT  - food security\\nOT  - prime editing\\nEDAT- 2022/02/10 06:00\\nMHDA- 2022/02/10 06:00\\nCRDT- 2022/02/09 05:38\\nPHST- 2022/02/09 05:38 [entrez]\\nPHST- 2022/02/10 06:00 [pubmed]\\nPHST- 2022/02/10 06:00 [medline]\\nAID - 10.1080/21645698.2021.2021724 [doi]\\nPST - aheadofprint\\nSO  - GM Crops Food. 2022 Feb 9:1-15. doi: 10.1080/21645698.2021.2021724.\\n\\nPMID- 30258941\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 2352-3042 (Electronic)\\nIS  - 2352-4820 (Print)\\nIS  - 2352-3042 (Linking)\\nVI  - 5\\nIP  - 2\\nDP  - 2018 Jun\\nTI  - In silico sgRNA tool design for CRISPR control of quorum sensing in Acinetobacter  species.\\nPG  - 123-129\\nLID - 10.1016/j.gendis.2018.03.004 [doi]\\nAB  - CRISPR genome editing utilizes Cas9 nuclease and single guide RNA (sgRNA), which  directs the nuclease to a specific site in the genome and makes a double-stranded  break (DSB). Design of sgRNA for CRISPR-Cas targeting, and to promote CRISPR  adaptation, uses a regulatory mechanism that ensures maximum CRISPR-Cas9 system  functions when a bacterial population is at highest risk of phage infection.  Acinetobacter baumannii is the most regularly identified gram-negative bacterium  infecting patients. Recent reports have demonstrated that the extent of diseases  caused by A. baumannii is expanding and, in a few cases, now surpasses the quantity  of infections caused by P. aeruginosa. Most Acinetobacter strains possess  biofilm-forming ability, which plays a major role in virulence and drug resistance.  Biofilm bacteria use quorum sensing, a cell-to-cell communication process, to  activate gene expression. Many genes are involved in biofilm formation and the  mechanism to disrupt the biofilm network is still not clearly understood. In this  study, we performed in silico gene editing to exploit the AbaI gene, responsible for  biofilm formation. The study explored different tools available for genome editing  to create gene knockouts, selecting the A. baumannii AbaI gene as a target.\\nFAU - Karlapudi, Abraham Peele\\nAU  - Karlapudi AP\\nAD  - Department of Biotechnology, Vignan\\'s Foundation for Science Technology and  Research, Vadlamudi, Guntur 522213, India.\\nFAU - T C, Venkateswarulu\\nAU  - T C V\\nAD  - Department of Biotechnology, Vignan\\'s Foundation for Science Technology and  Research, Vadlamudi, Guntur 522213, India.\\nFAU - Tammineedi, Jahnavi\\nAU  - Tammineedi J\\nAD  - Department of Biotechnology, Vignan\\'s Foundation for Science Technology and  Research, Vadlamudi, Guntur 522213, India.\\nFAU - Srirama, Krupanidhi\\nAU  - Srirama K\\nAD  - Department of Biotechnology, Vignan\\'s Foundation for Science Technology and  Research, Vadlamudi, Guntur 522213, India.\\nFAU - Kanumuri, Lohit\\nAU  - Kanumuri L\\nAD  - Department of Biotechnology, Vignan\\'s Foundation for Science Technology and  Research, Vadlamudi, Guntur 522213, India.\\nFAU - Prabhakar Kodali, Vidya\\nAU  - Prabhakar Kodali V\\nAD  - Department of Biotechnology, Vikrama Simhapuri University, Nellore, India.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20180411\\nTA  - Genes Dis\\nJT  - Genes & diseases\\nJID - 101635967\\nPMC - PMC6146548\\nOTO - NOTNLM\\nOT  - AbaI\\nOT  - Acinetobacter baumannii\\nOT  - CHOPCHOP\\nOT  - Crispr-cas9\\nOT  - sgRNA\\nEDAT- 2018/09/28 06:00\\nMHDA- 2018/09/28 06:01\\nCRDT- 2018/09/28 06:00\\nPHST- 2018/01/09 00:00 [received]\\nPHST- 2018/03/29 00:00 [accepted]\\nPHST- 2018/09/28 06:00 [entrez]\\nPHST- 2018/09/28 06:00 [pubmed]\\nPHST- 2018/09/28 06:01 [medline]\\nAID - S2352-3042(18)30056-4 [pii]\\nAID - 10.1016/j.gendis.2018.03.004 [doi]\\nPST - epublish\\nSO  - Genes Dis. 2018 Apr 11;5(2):123-129. doi: 10.1016/j.gendis.2018.03.004. eCollection  2018 Jun.\\n\\nPMID- 34517762\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211129\\nLR  - 20211129\\nIS  - 2150-7511 (Electronic)\\nVI  - 12\\nIP  - 5\\nDP  - 2021 Oct 26\\nTI  - MOV10 Helicase Interacts with Coronavirus Nucleocapsid Protein and Has Antiviral  Activity.\\nPG  - e0131621\\nLID - 10.1128/mBio.01316-21 [doi]\\nLID - e01316-21\\nAB  - Coronaviruses (CoVs) are emergent pathogens that may cause life-threatening  respiratory diseases in humans. Understanding of CoV-host interactions may help to  identify novel therapeutic targets. MOV10 is an RNA helicase involved in different  steps of cellular RNA metabolism. Both MOV10 antiviral and proviral activities have  been described in a limited number of viruses, but this protein has not been  previously associated with CoVs. We found that during Middle East respiratory  syndrome coronavirus (MERS-CoV) infection, MOV10 aggregated in cytoplasmic  structures colocalizing with viral nucleocapsid (N) protein. MOV10-N interaction was  confirmed by endogenous MOV10 coimmunoprecipitation, and the presence of other  cellular proteins was also detected in MOV10 complexes. MOV10 silencing  significantly increased both N protein accumulation and virus titer, with no changes  in the accumulation of viral RNAs. Moreover, MOV10 overexpression caused a 10-fold  decrease in viral titers. These data indicated that MOV10 has antiviral activity  during MERS-CoV infection. We postulated that this activity could be mediated by  viral RNA sequestration, and in fact, RNA immunoprecipitation data showed the  presence of viral RNAs in the MOV10 cytoplasmic complexes. Expression of wild-type  MOV10 or of a MOV10 mutant without helicase activity in MOV10 knockout cell lines,  developed by CRISPR-Cas technology, indicated that the helicase activity of MOV10  was required for its antiviral effect. Interestingly MOV10-N interaction was  conserved in other mildly or highly pathogenic human CoVs, including the recently  emerged severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), although MOV10  antiviral activity was found only in highly pathogenic CoVs, suggesting a potential  role of MOV10 in the modulation of human CoVs pathogenesis. IMPORTANCE Coronaviruses  (CoVs) are emerging pathogens causing life-threatening diseases in humans. Knowledge  of virus-host interactions and viral subversion mechanisms of host pathways is  required for the development of effective countermeasures against CoVs. The  interaction between cellular RNA helicase MOV10 and nucleocapsid (N) protein from  several human CoVs is shown. Using MERS-CoV as a model, we demonstrate that MOV10  has antiviral function, requiring its helicase activity, most likely mediated by  viral RNA sequestration in cytoplasmic ribonucleoprotein structures. Furthermore, we  found that MOV10 antiviral activity may act only in highly pathogenic human CoVs,  suggesting a role for MOV10 in modulating CoVs pathogenesis. The present study  uncovers a complex network of viral and cellular RNAs and proteins interaction  modulating the antiviral response against CoVs.\\nFAU - Wang, Li\\nAU  - Wang L\\nAD  - Department of Molecular and Cell Biology, National Center of Biotechnology,  Universidad AutÃ³noma de Madrid, Madrid, Spain.\\nFAU - Sola, Isabel\\nAU  - Sola I\\nAD  - Department of Molecular and Cell Biology, National Center of Biotechnology,  Universidad AutÃ³noma de Madrid, Madrid, Spain.\\nFAU - Enjuanes, Luis\\nAU  - Enjuanes L\\nAUID- ORCID: 0000-0002-0854-0226\\nAD  - Department of Molecular and Cell Biology, National Center of Biotechnology,  Universidad AutÃ³noma de Madrid, Madrid, Spain.\\nFAU - ZuÃ±iga, Sonia\\nAU  - ZuÃ±iga S\\nAD  - Department of Molecular and Cell Biology, National Center of Biotechnology,  Universidad AutÃ³noma de Madrid, Madrid, Spain.\\nLA  - eng\\nGR  - P01 AI060699/AI/NIAID NIH HHS/United States\\nGR  - 202020E043/MEC | Consejo Superior de Investigaciones CientÃ\\xadficas (CSIC)/\\nGR  - 2P01AI060699/HHS | National Institutes of Health (NIH)/\\nGR  - IMI_JU_115760/European Commission (EC)/\\nGR  - 202020E079/Government of Spain/\\nGR  - PID2019-107001RB-I00/Government of Spain/\\nGR  - BIO2016-75549-R/Government of Spain/\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210914\\nTA  - mBio\\nJT  - mBio\\nJID - 101519231\\nRN  - 0 (Antiviral Agents)\\nRN  - 0 (Coronavirus Nucleocapsid Proteins)\\nRN  - 0 (RNA, Viral)\\nRN  - EC 3.6.4.13 (RNA Helicases)\\nSB  - IM\\nMH  - Animals\\nMH  - Antiviral Agents/*pharmacology\\nMH  - Blotting, Western\\nMH  - Cell Line, Tumor\\nMH  - Chlorocebus aethiops\\nMH  - Coronavirus Nucleocapsid Proteins/*metabolism\\nMH  - Fluorescent Antibody Technique\\nMH  - Humans\\nMH  - Immunoprecipitation\\nMH  - RNA Helicases/*metabolism/*pharmacology\\nMH  - RNA, Viral/metabolism\\nMH  - Vero Cells\\nMH  - Virus Replication/drug effects\\nPMC - PMC8546642\\nOTO - NOTNLM\\nOT  - *antiviral response\\nOT  - *coronavirus\\nOT  - *highly pathogenic coronavirus\\nOT  - *nucleocapsid protein\\nOT  - *virus-host interactions\\nEDAT- 2021/09/15 06:00\\nMHDA- 2021/11/30 06:00\\nCRDT- 2021/09/14 05:37\\nPHST- 2021/09/15 06:00 [pubmed]\\nPHST- 2021/11/30 06:00 [medline]\\nPHST- 2021/09/14 05:37 [entrez]\\nAID - mBio01316-21 [pii]\\nAID - 10.1128/mBio.01316-21 [doi]\\nPST - ppublish\\nSO  - mBio. 2021 Oct 26;12(5):e0131621. doi: 10.1128/mBio.01316-21. Epub 2021 Sep 14.\\n\\nPMID- 28574055\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180416\\nLR  - 20190718\\nIS  - 1748-7838 (Electronic)\\nIS  - 1001-0602 (Print)\\nIS  - 1001-0602 (Linking)\\nVI  - 27\\nIP  - 7\\nDP  - 2017 Jul\\nTI  - Alternate binding modes of anti-CRISPR viral suppressors AcrF1/2 to Csy surveillance  complex revealed by cryo-EM structures.\\nPG  - 853-864\\nLID - 10.1038/cr.2017.79 [doi]\\nAB  - Bacteriophages encode anti-CRISPR suppressors to counteract the CRISPR/Cas immunity  of their bacterial hosts, thus facilitating their survival and replication. Previous  studies have shown that two phage-encoded anti-CRISPR proteins, AcrF1 and AcrF2,  suppress the type I-F CRISPR/Cas system of Pseudomonas aeruginosa by preventing  target DNA recognition by the Csy surveillance complex, but the precise underlying  mechanism was unknown. Here we present the structure of AcrF1/2 bound to the Csy  complex determined by cryo-EM single-particle reconstruction. By structural  analysis, we found that AcrF1 inhibits target DNA recognition of the Csy complex by  interfering with base pairing between the DNA target strand and crRNA spacer. In  addition, multiple copies of AcrF1 bind to the Csy complex with different modes when  working individually or cooperating with AcrF2, which might exclude target DNA  binding through different mechanisms. Together with previous reports, we provide a  comprehensive working scenario for the two anti-CRISPR suppressors, AcrF1 and AcrF2,  which silence CRISPR/Cas immunity by targeting the Csy surveillance complex.\\nFAU - Peng, Ruchao\\nAU  - Peng R\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 101408, China.\\nFAU - Xu, Ying\\nAU  - Xu Y\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei, Anhui  230026, China.\\nFAU - Zhu, Tengfei\\nAU  - Zhu T\\nAD  - University of Chinese Academy of Sciences, Beijing 101408, China.\\nAD  - Research Network of Immunity and Health (RNIH), Beijing Institute of Life Science,  Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Li, Ningning\\nAU  - Li N\\nAD  - State Key Laboratory of Membrane Biology, Peking-Tsinghua Center for Life Sciences,  School of Life Sciences, Peking University, Beijing 100871, China.\\nFAU - Qi, Jianxun\\nAU  - Qi J\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 101408, China.\\nFAU - Chai, Yan\\nAU  - Chai Y\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Wu, Min\\nAU  - Wu M\\nAD  - Department of Biomedical Sciences, School of Medicine and Health Sciences,  University of North Dakota, Grand Forks, ND 58203, USA.\\nFAU - Zhang, Xinzheng\\nAU  - Zhang X\\nAD  - National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy  of Sciences, Beijing 100101, China.\\nFAU - Shi, Yi\\nAU  - Shi Y\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 101408, China.\\nAD  - Shenzhen Key Laboratory of Pathogen and Immunity, Shenzhen Third People\\'s Hospital,  Shenzhen, Guangdong 518112, China.\\nAD  - Center for Influenza Research and Early-warning (CASCIRE), Chinese Academy of  Sciences, Beijing 100101, China.\\nFAU - Wang, Peiyi\\nAU  - Wang P\\nAD  - Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.\\nFAU - Wang, Jiawei\\nAU  - Wang J\\nAD  - State Key Laboratory of Membrane Biology, School of Life Sciences, Tsinghua  University, Beijing 100084, China.\\nFAU - Gao, Ning\\nAU  - Gao N\\nAD  - State Key Laboratory of Membrane Biology, Peking-Tsinghua Center for Life Sciences,  School of Life Sciences, Peking University, Beijing 100871, China.\\nFAU - Gao, George Fu\\nAU  - Gao GF\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 101408, China.\\nAD  - Research Network of Immunity and Health (RNIH), Beijing Institute of Life Science,  Chinese Academy of Sciences, Beijing 100101, China.\\nAD  - Shenzhen Key Laboratory of Pathogen and Immunity, Shenzhen Third People\\'s Hospital,  Shenzhen, Guangdong 518112, China.\\nAD  - Center for Influenza Research and Early-warning (CASCIRE), Chinese Academy of  Sciences, Beijing 100101, China.\\nAD  - National Institute for Viral Disease Control and Prevention, Chinese Center for  Disease Control and Prevention (China CDC), Beijing 102206, China.\\nLA  - eng\\nGR  - R01 AI109317/AI/NIAID NIH HHS/United States\\nGR  - R01 AI138203/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20170602\\nTA  - Cell Res\\nJT  - Cell research\\nJID - 9425763\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - 0 (DNA, Bacterial)\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (RNA, Bacterial)\\nRN  - 0 (Viral Proteins)\\nRN  - EC 3.4.- (Endopeptidases)\\nRN  - EC 3.4.- (TEV protease)\\nRN  - Tobacco etch virus\\nSB  - IM\\nMH  - Bacterial Proteins/*chemistry/genetics/ultrastructure\\nMH  - Binding Sites\\nMH  - CRISPR-Associated Proteins/*chemistry/genetics/ultrastructure\\nMH  - *CRISPR-Cas Systems\\nMH  - Cryoelectron Microscopy\\nMH  - DNA, Bacterial/metabolism\\nMH  - Endopeptidases/metabolism\\nMH  - Membrane Proteins/*chemistry/genetics/ultrastructure\\nMH  - Models, Molecular\\nMH  - Potyvirus\\nMH  - Pseudomonas Phages\\nMH  - Pseudomonas aeruginosa/*virology\\nMH  - RNA, Bacterial/metabolism\\nMH  - Viral Proteins/metabolism\\nPMC - PMC5518991\\nEDAT- 2017/06/03 06:00\\nMHDA- 2018/04/17 06:00\\nCRDT- 2017/06/03 06:00\\nPHST- 2017/04/08 00:00 [received]\\nPHST- 2017/05/07 00:00 [revised]\\nPHST- 2017/05/08 00:00 [accepted]\\nPHST- 2017/06/03 06:00 [pubmed]\\nPHST- 2018/04/17 06:00 [medline]\\nPHST- 2017/06/03 06:00 [entrez]\\nAID - cr201779 [pii]\\nAID - 10.1038/cr.2017.79 [doi]\\nPST - ppublish\\nSO  - Cell Res. 2017 Jul;27(7):853-864. doi: 10.1038/cr.2017.79. Epub 2017 Jun 2.\\n\\nPMID- 29580291\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190827\\nLR  - 20190827\\nIS  - 1474-760X (Electronic)\\nIS  - 1474-7596 (Print)\\nIS  - 1474-7596 (Linking)\\nVI  - 19\\nIP  - 1\\nDP  - 2018 Mar 26\\nTI  - Pulling the genome in opposite directions to dissect gene networks.\\nPG  - 42\\nLID - 10.1186/s13059-018-1425-1 [doi]\\nLID - 42\\nAB  - Orthogonal CRISPR-Cas systems have been integrated into combinatorial screens to  decipher complex genetic relationships in two recent studies.\\nFAU - Gersbach, Charles A\\nAU  - Gersbach CA\\nAD  - Department of Biomedical Engineering, Center for Genomic and Computational Biology,  Duke University, Durham, NC, USA. charles.gersbach@duke.edu.\\nAD  - Department of Orthopaedic Surgery, Duke University Medical Center, Durham, NC, USA.  charles.gersbach@duke.edu.\\nFAU - Barrangou, Rodolphe\\nAU  - Barrangou R\\nAD  - Department of Food, Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh, NC, 27695, USA. rbarran@ncsu.edu.\\nLA  - eng\\nGR  - R01 DA036865/DA/NIDA NIH HHS/United States\\nGR  - R21 DA041878/DA/NIDA NIH HHS/United States\\nGR  - U01 HG007900/HG/NHGRI NIH HHS/United States\\nGR  - R01DA036865, U01HG007900, R21DA041878/Foundation for the National Institutes of  Health/International\\nGR  - UM1 HG009428/HG/NHGRI NIH HHS/United States\\nPT  - Comment\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180326\\nTA  - Genome Biol\\nJT  - Genome biology\\nJID - 100960660\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nSB  - IM\\nCON - Nat Biotechnol. 2018 Feb;36(2):179-189. PMID: 29251726\\nCON - Nat Biotechnol. 2018 Feb;36(2):170-178. PMID: 29334369\\nMH  - *CRISPR-Associated Protein 9\\nMH  - CRISPR-Cas Systems\\nMH  - *Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Gene Regulatory Networks\\nMH  - Genome\\nPMC - PMC5870483\\nCOIS- COMPETING INTERESTS: CAG is a co-founder and advisor to Element Genomics and Locus  Biosciences, and an advisor to Sarepta Therapeutics. RB is a co-founder and advisor  to Locus Biosciences and Intellia Therapeutics and advisor to Caribou Biosciences.  PUBLISHERâ€™S NOTE: Springer Nature remains neutral with regard to jurisdictional  claims in published maps and institutional affiliations.\\nEDAT- 2018/03/28 06:00\\nMHDA- 2019/08/28 06:00\\nCRDT- 2018/03/28 06:00\\nPHST- 2018/03/28 06:00 [entrez]\\nPHST- 2018/03/28 06:00 [pubmed]\\nPHST- 2019/08/28 06:00 [medline]\\nAID - 10.1186/s13059-018-1425-1 [pii]\\nAID - 1425 [pii]\\nAID - 10.1186/s13059-018-1425-1 [doi]\\nPST - epublish\\nSO  - Genome Biol. 2018 Mar 26;19(1):42. doi: 10.1186/s13059-018-1425-1.\\n\\nPMID- 33468685\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210910\\nLR  - 20210910\\nIS  - 2150-7511 (Electronic)\\nVI  - 12\\nIP  - 1\\nDP  - 2021 Jan 19\\nTI  - CRNKL1 Is a Highly Selective Regulator of Intron-Retaining HIV-1 and Cellular mRNAs.\\nLID - 10.1128/mBio.02525-20 [doi]\\nLID - e02525-20\\nAB  - The HIV-1 Rev protein is a nuclear export factor for unspliced and incompletely  spliced HIV-1 RNAs. Without Rev, these intron-retaining RNAs are trapped in the  nucleus. A genome-wide screen identified nine proteins of the spliceosome, which all  enhanced expression from the HIV-1 unspliced RNA after CRISPR/Cas knockdown.  Depletion of DHX38, WDR70, and four proteins of the Prp19-associated complex (ISY1,  BUD31, XAB2, and CRNKL1) resulted in a more than 20-fold enhancement of unspliced  HIV-1 RNA levels in the cytoplasm. Targeting of CRNKL1, DHX38, and BUD31 affected  nuclear export efficiencies of the HIV-1 unspliced RNA to a much larger extent than  splicing. Transcriptomic analyses further revealed that CRNKL1 also suppresses  cytoplasmic levels of a subset of cellular mRNAs, including some with selectively  retained introns. Thus, CRNKL1-dependent nuclear retention is a novel cellular  mechanism for the regulation of cytoplasmic levels of intron-retaining HIV-1 mRNAs,  which HIV-1 may have harnessed to direct its complex splicing pattern.IMPORTANCE To  regulate its complex splicing pattern, HIV-1 uses the adaptor protein Rev to shuttle  unspliced or partially spliced mRNA from the nucleus to the cytoplasm. In the  absence of Rev, these RNAs are retained in the nucleus, but it is unclear why. Here  we identify cellular proteins whose depletion enhances cytoplasmic levels of the  HIV-1 unspliced RNA. Depletion of one of them, CRNKL1, also increases cytoplasmic  levels of a subset of intron-retaining cellular mRNA, suggesting that  CRNKL1-dependent nuclear retention may be a basic cellular mechanism exploited by  HIV-1.\\nCI  - Copyright Â© 2021 Xiao et al.\\nFAU - Xiao, Han\\nAU  - Xiao H\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Wyler, Emanuel\\nAU  - Wyler E\\nAD  - Berlin Institute for Medical Systems Biology, Max-DelbrÃ¼ck-Center for Molecular  Medicine in the Helmholtz Association, Berlin, Germany.\\nFAU - Milek, Miha\\nAU  - Milek M\\nAD  - Berlin Institute for Medical Systems Biology, Max-DelbrÃ¼ck-Center for Molecular  Medicine in the Helmholtz Association, Berlin, Germany.\\nFAU - Grewe, Bastian\\nAU  - Grewe B\\nAD  - Department of Molecular and Medical Virology, Ruhr-University, Bochum, Germany.\\nFAU - Kirchner, Philipp\\nAU  - Kirchner P\\nAD  - Institute of Human Genetics, University Hospital Erlangen, Friedrich-Alexander  UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Ekici, Arif\\nAU  - Ekici A\\nAD  - Institute of Human Genetics, University Hospital Erlangen, Friedrich-Alexander  UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Silva, Ana Beatriz Oliveira Villela\\nAU  - Silva ABOV\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Jungnickl, Doris\\nAU  - Jungnickl D\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Full, Florian\\nAU  - Full F\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Thomas, Marco\\nAU  - Thomas M\\nAUID- ORCID: 0000-0003-4820-2712\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Landthaler, Markus\\nAU  - Landthaler M\\nAD  - Berlin Institute for Medical Systems Biology, Max-DelbrÃ¼ck-Center for Molecular  Medicine in the Helmholtz Association, Berlin, Germany.\\nAD  - IRI Life Sciences, Institute fÃ¼r Biologie, Humboldt UniversitÃ¤t zu Berlin, Berlin,  Germany.\\nFAU - Ensser, Armin\\nAU  - Ensser A\\nAUID- ORCID: 0000-0001-8873-3863\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany.\\nFAU - Ãœberla, Klaus\\nAU  - Ãœberla K\\nAUID- ORCID: 0000-0001-7822-3835\\nAD  - Institute of Clinical and Molecular Virology, University Hospital Erlangen,  Friedrich-Alexander UniversitÃ¤t Erlangen-NÃ¼rnberg, Erlangen, Germany  klaus.ueberla@fau.de.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210119\\nTA  - mBio\\nJT  - mBio\\nJID - 101519231\\nRN  - 0 (BUD31 protein, human)\\nRN  - 0 (CRNKL1 protein, human)\\nRN  - 0 (ISY1 protein, human)\\nRN  - 0 (Nuclear Proteins)\\nRN  - 0 (RNA Splicing Factors)\\nRN  - 0 (RNA, Messenger)\\nRN  - 0 (RNA, Small Interfering)\\nRN  - 0 (RNA, Viral)\\nRN  - 0 (XAB2 protein, human)\\nRN  - 0 (rev Gene Products, Human Immunodeficiency Virus)\\nRN  - 0 (rev protein, Human Immunodeficiency Virus-1)\\nRN  - EC 3.6.1.- (DHX38 protein, human)\\nRN  - EC 3.6.4.13 (DEAD-box RNA Helicases)\\nSB  - IM\\nMH  - Cell Nucleus/genetics/virology\\nMH  - Cytosol/metabolism/virology\\nMH  - DEAD-box RNA Helicases/genetics/metabolism\\nMH  - Exons\\nMH  - Gene Regulatory Networks\\nMH  - HIV-1/*genetics/metabolism\\nMH  - Host-Pathogen Interactions/genetics\\nMH  - Humans\\nMH  - Introns\\nMH  - Jurkat Cells\\nMH  - Nuclear Proteins/antagonists & inhibitors/*genetics/metabolism\\nMH  - Protein Interaction Mapping\\nMH  - *RNA Splicing\\nMH  - RNA Splicing Factors/genetics/metabolism\\nMH  - RNA, Messenger/*genetics/metabolism\\nMH  - RNA, Small Interfering/genetics/metabolism\\nMH  - RNA, Viral/*genetics/metabolism\\nMH  - Spliceosomes/genetics/metabolism\\nMH  - Transcriptome\\nMH  - rev Gene Products, Human Immunodeficiency Virus/*genetics/metabolism\\nPMC - PMC7845644\\nOTO - NOTNLM\\nOT  - *CRNKL1\\nOT  - *RNA splicing\\nOT  - *association\\nOT  - *human immunodeficiency virus\\nOT  - *intron-retaining RNA\\nOT  - *nuclear retention\\nEDAT- 2021/01/21 06:00\\nMHDA- 2021/09/11 06:00\\nCRDT- 2021/01/20 05:44\\nPHST- 2021/01/20 05:44 [entrez]\\nPHST- 2021/01/21 06:00 [pubmed]\\nPHST- 2021/09/11 06:00 [medline]\\nAID - mBio.02525-20 [pii]\\nAID - mBio02525-20 [pii]\\nAID - 10.1128/mBio.02525-20 [doi]\\nPST - epublish\\nSO  - mBio. 2021 Jan 19;12(1):e02525-20. doi: 10.1128/mBio.02525-20.\\n\\nPMID- 32839482\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210107\\nLR  - 20210824\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 10\\nIP  - 1\\nDP  - 2020 Aug 24\\nTI  - Genome editing with the donor plasmid equipped with synthetic crRNA-target sequence.\\nPG  - 14120\\nLID - 10.1038/s41598-020-70804-6 [doi]\\nLID - 14120\\nAB  - CRISPR/Cas-mediated genome editing is a powerful tool for generating genetically  mutated cells and organisms. Linearisation of donor cassettes with this system has  been shown to facilitate both transgene donor insertion and targeted knock-in. Here,  we developed a donor plasmid that we name pCriMGET (plasmid of synthetic CRISPR  coded RNA target sequence-equipped donor plasmid-mediated gene targeting), in which  an off-target free synthetic CRISPR coded RNA-target sequence (syn-crRNA-TS) is  incorporated with a multi-cloning site, where a donor cassette can be inserted. With  co-expression of Cas9 and the syn-crRNA-TS guide RNA (gRNA), pCriMGET provides a  linearised donor cassette in vivo, thereby promoting the transgene donor insertion  and targeted knock-in. When co-injected with Cas9 protein and gRNA into murine  zygotes, pCriMGET yielded around 20% transgene insertion in embryos. This method  also achieved more than 25% in-frame knock-in at the mouse Tbx3 gene locus without  predicted insertion-deletion mutations using a transgene donor with 400-bp homology  arms. pCriMGET is therefore useful as a versatile CRISPR/Cas9-cleavable donor  plasmid for efficient integration and targeted knock-in of exogenous DNA in mice.\\nFAU - Ishibashi, Riki\\nAU  - Ishibashi R\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan. rishibas@infront.kyoto-u.ac.jp.\\nAD  - Department of Mammalian Regulatory Networks, Graduate School of Biostudies, Kyoto  University, Sakyo-ku, Kyoto, 606-8502, Japan. rishibas@infront.kyoto-u.ac.jp.\\nFAU - Abe, Kota\\nAU  - Abe K\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan.\\nAD  - Department of Mammalian Regulatory Networks, Graduate School of Biostudies, Kyoto  University, Sakyo-ku, Kyoto, 606-8502, Japan.\\nFAU - Ido, Nanami\\nAU  - Ido N\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan.\\nAD  - Department of Mammalian Regulatory Networks, Graduate School of Biostudies, Kyoto  University, Sakyo-ku, Kyoto, 606-8502, Japan.\\nFAU - Kitano, Satsuki\\nAU  - Kitano S\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan.\\nFAU - Miyachi, Hitoshi\\nAU  - Miyachi H\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan.\\nFAU - Toyoshima, Fumiko\\nAU  - Toyoshima F\\nAD  - Department of Biosystems Science, Institute for Frontier and Medical Sciences, Kyoto  University, Sakyo-ku, Kyoto, 606-8507, Japan. ftoyoshi@infront.kyoto-u.ac.jp.\\nAD  - Department of Mammalian Regulatory Networks, Graduate School of Biostudies, Kyoto  University, Sakyo-ku, Kyoto, 606-8502, Japan. ftoyoshi@infront.kyoto-u.ac.jp.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200824\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (T-Box Domain Proteins)\\nRN  - 0 (Tbx3 protein, mouse)\\nSB  - IM\\nMH  - Animals\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Female\\nMH  - Gene Editing/*methods\\nMH  - Gene Knock-In Techniques/*methods\\nMH  - Gene Targeting\\nMH  - Genetic Engineering/methods\\nMH  - Genome/genetics\\nMH  - HEK293 Cells\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - INDEL Mutation/genetics\\nMH  - Mice\\nMH  - Mice, Inbred C57BL\\nMH  - Mice, Inbred ICR\\nMH  - Mice, Transgenic\\nMH  - Plasmids/*genetics\\nMH  - RNA, Guide/*genetics\\nMH  - T-Box Domain Proteins/*genetics\\nMH  - Transgenes/genetics\\nPMC - PMC7445171\\nCOIS- The authors declare no competing interests.\\nEDAT- 2020/08/26 06:00\\nMHDA- 2021/01/08 06:00\\nCRDT- 2020/08/26 06:00\\nPHST- 2020/03/18 00:00 [received]\\nPHST- 2020/08/03 00:00 [accepted]\\nPHST- 2020/08/26 06:00 [entrez]\\nPHST- 2020/08/26 06:00 [pubmed]\\nPHST- 2021/01/08 06:00 [medline]\\nAID - 10.1038/s41598-020-70804-6 [pii]\\nAID - 70804 [pii]\\nAID - 10.1038/s41598-020-70804-6 [doi]\\nPST - epublish\\nSO  - Sci Rep. 2020 Aug 24;10(1):14120. doi: 10.1038/s41598-020-70804-6.\\n\\nPMID- 33523177\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20211021\\nIS  - 2397-8554 (Print)\\nIS  - 2397-8562 (Electronic)\\nIS  - 2397-8554 (Linking)\\nVI  - 3\\nIP  - 5\\nDP  - 2019 Nov 11\\nTI  - Synthetic bionanotechnology: synthetic biology finds a toehold in nanotechnology.\\nPG  - 507-516\\nLID - 10.1042/ETLS20190100 [doi]\\nAB  - Enabled by its central role in the molecular networks that govern cell function, RNA  has been widely used for constructing components used in biological circuits for  synthetic biology. Nucleic acid nanotechnology, which exploits predictable nucleic  acid interactions to implement programmable molecular systems, has seen remarkable  advances in in vitro nanoscale self-assembly and molecular computation, enabling the  production of complex nanostructures and DNA-based neural networks. Living cells  genetically engineered to execute nucleic acid nanotechnology programs thus have  outstanding potential to significantly extend the current limits of synthetic  biology. This perspective discusses the recent developments and future challenges in  the field of synthetic bionanotechnology. Thus far, researchers in this emerging  area have implemented dozens of programmable RNA nanodevices that provide precise  control over gene expression at the transcriptional and translational levels and  through CRISPR/Cas effectors. Moreover, they have employed synthetic self-assembling  RNA networks in engineered bacteria to carry out computations featuring up to a  dozen inputs and to substantially enhance the rate of chemical synthesis. Continued  advancement of the field will benefit from improved in vivo strategies for  streamlining nucleic acid network synthesis and new approaches for enhancing network  function. As the field matures and the complexity gap between in vitro and in vivo  systems narrows, synthetic bionanotechnology promises to have diverse potential  applications ranging from intracellular circuits that detect and treat disease to  synthetic enzymatic pathways that efficiently produce novel drug molecules.\\nCI  - Â© 2019 The Author(s).\\nFAU - Green, Alexander A\\nAU  - Green AA\\nAD  - Biodesign Center for Molecular Design and Biomimetics at the Biodesign Institute and  School of Molecular Sciences, Arizona State University, Tempe, Arizona 85287, U.S.A.\\nLA  - eng\\nGR  - DP2 GM126892/GM/NIGMS NIH HHS/United States\\nGR  - R21 AI136571/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nTA  - Emerg Top Life Sci\\nJT  - Emerging topics in life sciences\\nJID - 101706399\\nSB  - IM\\nPMC - PMC7288988\\nMID - NIHMS1636292\\nOTO - NOTNLM\\nOT  - RNA\\nOT  - nucleic acid nanotechnology\\nOT  - synthetic biological circuits\\nCOIS- A.A.G. is an inventor on several patents and provisional patent applications  describing genetically encoded sensors for molecular logic and nucleic acid  detection.\\nEDAT- 2019/11/11 00:00\\nMHDA- 2019/11/11 00:00\\nCRDT- 2021/02/01 12:18\\nPHST- 2019/07/22 00:00 [received]\\nPHST- 2019/09/30 00:00 [revised]\\nPHST- 2019/10/07 00:00 [accepted]\\nPHST- 2021/02/01 12:18 [entrez]\\nPHST- 2019/11/11 00:00 [pubmed]\\nPHST- 2019/11/11 00:00 [medline]\\nAID - 220913 [pii]\\nAID - ETLS-3-507 [pii]\\nAID - 10.1042/ETLS20190100 [doi]\\nPST - ppublish\\nSO  - Emerg Top Life Sci. 2019 Nov 11;3(5):507-516. doi: 10.1042/ETLS20190100.\\n\\nPMID- 28479163\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180312\\nLR  - 20180312\\nIS  - 1873-4863 (Electronic)\\nIS  - 0168-1656 (Linking)\\nVI  - 252\\nDP  - 2017 Jun 20\\nTI  - Cell-penetrating peptides (CPPs): From delivery of nucleic acids and antigens to  transduction of engineered nucleases for application in transgenesis.\\nPG  - 15-26\\nLID - S0168-1656(17)30203-1 [pii]\\nLID - 10.1016/j.jbiotec.2017.05.002 [doi]\\nAB  - Cell-penetrating peptides (CPPs) have been studied for their capacity to translocate  across the lipid membrane of several cell types. In membrane translocation, these  peptides can remarkably transport biologically active hydrophilic molecules, such as  pharmaceuticals, nucleic acids (DNA and RNA) and even high-molecular-weight  proteins, Fig. 3 into the cell cytoplasm and organelles. The development of CPPs as  transduction agents includes the modification of gene and protein expression, the  reprogramming and differentiation of induced pluripotent stem cells and the  preparation of cellular vaccines. A relatively recent field of CPP application is  the transduction of plasmid DNA vectors and CPP-fusion proteins to modify genomes  and introduce new traits in cells and organisms. CPP-mediated transduction of  components for genome editing is an advantageous alternative to viral DNA vectors.  Engineered site-specific nucleases, such as Cre recombinase, ZFN, TALENs and CRISPR  associated protein (Cas), have been coupled to CPPs, and the fused proteins have  been used to permeate targeted cells and tissues. The functionally active fusion  CPP-nucleases subsequently home to the nucleus, incise genomic DNA at specific sites  and induce repair and recombination. This review has the objective of discussing  CPPs and elucidating the prospective use of CPP-mediated transduction technology,  particularly in genome modification and transgenesis.\\nCI  - Copyright Â© 2017 Elsevier B.V. All rights reserved.\\nFAU - RÃ¡dis-Baptista, Gandhi\\nAU  - RÃ¡dis-Baptista G\\nAD  - Laboratory of Biochemistry and Biotechnology, Institute for Marine Science, Federal  University of CearÃ¡, Fortaleza-CE, 60.165-081, Brazil. Electronic address:  gandhi.radis@ufc.br.\\nFAU - Campelo, Iana S\\nAU  - Campelo IS\\nAD  - Laboratory of Physiology and Control of Reproduction, Faculty of Veterinary, State  University of CearÃ¡, Fortaleza-CE, 60.714-903, Brazil.\\nFAU - Morlighem, Jean-Ã‰tienne R L\\nAU  - Morlighem JRL\\nAD  - Laboratory of Biochemistry and Biotechnology, Institute for Marine Science, Federal  University of CearÃ¡, Fortaleza-CE, 60.165-081, Brazil; Northeast Biotechnology  Network (RENORBIO), Post-graduation program in Biotechnology, Federal University of  CearÃ¡, Fortaleza, CE, 60.455-900, Brazil.\\nFAU - Melo, Luciana M\\nAU  - Melo LM\\nAD  - Laboratory of Physiology and Control of Reproduction, Faculty of Veterinary, State  University of CearÃ¡, Fortaleza-CE, 60.714-903, Brazil.\\nFAU - Freitas, Vicente J F\\nAU  - Freitas VJF\\nAD  - Laboratory of Physiology and Control of Reproduction, Faculty of Veterinary, State  University of CearÃ¡, Fortaleza-CE, 60.714-903, Brazil. Electronic address:  vicente.freitas@uece.br.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20170504\\nPL  - Netherlands\\nTA  - J Biotechnol\\nJT  - Journal of biotechnology\\nJID - 8411927\\nRN  - 0 (Antigens)\\nRN  - 0 (Cell-Penetrating Peptides)\\nRN  - 0 (Enzymes)\\nRN  - 0 (Nucleic Acids)\\nSB  - IM\\nMH  - Animals\\nMH  - Antigens/*administration & dosage\\nMH  - Cell-Penetrating Peptides/*administration & dosage\\nMH  - Enzymes/genetics\\nMH  - Gene Transfer Techniques\\nMH  - Genetic Engineering\\nMH  - Humans\\nMH  - Nucleic Acids/*administration & dosage\\nOTO - NOTNLM\\nOT  - CPP-mediated transduction\\nOT  - CPP-nuclease fusion proteins\\nOT  - Cationic peptides\\nOT  - Cell-penetrating peptides (CPPs)\\nOT  - Engineered site-directed nucleases (ESDNs)\\nOT  - Transgenesis\\nEDAT- 2017/05/10 06:00\\nMHDA- 2018/03/13 06:00\\nCRDT- 2017/05/09 06:00\\nPHST- 2016/12/06 00:00 [received]\\nPHST- 2017/04/22 00:00 [revised]\\nPHST- 2017/05/03 00:00 [accepted]\\nPHST- 2017/05/10 06:00 [pubmed]\\nPHST- 2018/03/13 06:00 [medline]\\nPHST- 2017/05/09 06:00 [entrez]\\nAID - S0168-1656(17)30203-1 [pii]\\nAID - 10.1016/j.jbiotec.2017.05.002 [doi]\\nPST - ppublish\\nSO  - J Biotechnol. 2017 Jun 20;252:15-26. doi: 10.1016/j.jbiotec.2017.05.002. Epub 2017  May 4.\\n\\nPMID- 27462456\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20160727\\nLR  - 20200930\\nIS  - 2056-5968 (Print)\\nIS  - 2056-5968 (Electronic)\\nIS  - 2056-5968 (Linking)\\nVI  - 2\\nDP  - 2016\\nTI  - A CRISPR-based approach for targeted DNA demethylation.\\nPG  - 16009\\nLID - 10.1038/celldisc.2016.9 [doi]\\nAB  - In mammalian cells, DNA methylation critically regulates gene expression and thus  has pivotal roles in myriad of physiological and pathological processes. Here we  report a novel method for targeted DNA demethylation using the widely used clustered  regularly interspaced short palindromic repeat (CRISPR)-Cas system. Initially,  modified single guide RNAs (sgRNAs) (sgRNA2.0) were constructed by inserting two  copies of bacteriophage MS2 RNA elements into the conventional sgRNAs, which would  facilitate the tethering of the Tet1 catalytic domain (Tet-CD), in fusion with dCas9  or MS2 coat proteins, to the targeted gene loci. Subsequently, such system was shown  to significantly upregulate transcription of the target genes, including RANKL,  MAGEB2 or MMP2, which was in close correlation to DNA demethylation of their  neighboring CpGs in the promoters. In addition, the dCas9/sgRNA2.0-directed  demethylation system appeared to afford efficient demethylation of the target genes  with tenuous off-target effects. Applications of this system would not only help us  understand mechanistically how DNA methylation might regulate gene expression in  specific contexts, but also enable control of gene expression and functionality with  potential clinical benefits.\\nFAU - Xu, Xingxing\\nAU  - Xu X\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences , Shanghai, China.\\nFAU - Tao, Yonghui\\nAU  - Tao Y\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences, Shanghai, China; University of Chinese Academy of Sciences, Shanghai,  China.\\nFAU - Gao, Xiaobo\\nAU  - Gao X\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences, Shanghai, China; University of Chinese Academy of Sciences, Shanghai,  China.\\nFAU - Zhang, Lei\\nAU  - Zhang L\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences , Shanghai, China.\\nFAU - Li, Xufang\\nAU  - Li X\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences , Shanghai, China.\\nFAU - Zou, Weiguo\\nAU  - Zou W\\nAD  - State Key Laboratory of Cell Biology , Shanghai, China.\\nFAU - Ruan, Kangcheng\\nAU  - Ruan K\\nAD  - State Key Laboratory of Molecular Biology; Institute of Biochemistry and Cell  Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences ,  Shanghai, China.\\nFAU - Wang, Feng\\nAU  - Wang F\\nAD  - Departments of Oral and Craniofacial Implant and Oral and Maxillofacial Surgery,  School of Medicine, the Ninth People\\'s Hospital Affiliated with Shanghai Jiao Tong  University , Shanghai, China.\\nFAU - Xu, Guo-Liang\\nAU  - Xu GL\\nAD  - State Key Laboratory of Molecular Biology; Institute of Biochemistry and Cell  Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences ,  Shanghai, China.\\nFAU - Hu, Ronggui\\nAU  - Hu R\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network; Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences , Shanghai, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20160503\\nTA  - Cell Discov\\nJT  - Cell discovery\\nJID - 101661034\\nPMC - PMC4853773\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - RANKL\\nOT  - demethylation\\nEDAT- 2016/07/28 06:00\\nMHDA- 2016/07/28 06:01\\nCRDT- 2016/07/28 06:00\\nPHST- 2015/12/21 00:00 [received]\\nPHST- 2016/02/29 00:00 [accepted]\\nPHST- 2016/07/28 06:00 [entrez]\\nPHST- 2016/07/28 06:00 [pubmed]\\nPHST- 2016/07/28 06:01 [medline]\\nAID - celldisc20169 [pii]\\nAID - 10.1038/celldisc.2016.9 [doi]\\nPST - epublish\\nSO  - Cell Discov. 2016 May 3;2:16009. doi: 10.1038/celldisc.2016.9. eCollection 2016.\\n\\nPMID- 30583622\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190730\\nLR  - 20190730\\nIS  - 1001-5302 (Print)\\nIS  - 1001-5302 (Linking)\\nVI  - 43\\nIP  - 21\\nDP  - 2018 Nov\\nTI  - [Directing construction of CRISPR/Cas9 vector of SmPAL1 in Salvia miltiorrhiza by  target efficiency detection in vitro].\\nPG  - 4226-4230\\nLID - 10.19540/j.cnki.cjcmm.20180726.007 [doi]\\nAB  - To construct CRISPR/Cas9 vectors for the editing of SmPAL1 in the phenylpropane  metabolic pathway of Salvia miltiorrhiza, CIRSPR/Cas9 target sites of SmPAL1 were  designed by online software. Its target efficiencies were detected in vitro by  enzyme digestion and sequences with highly efficiency were constructed into  CRISPR/Cas9 vectors. Three possible CRISPR target sequences (SmPAL1-g1, SmPAL1-g2,  SmPAL1-g3) were designed and the enzyme digestion efficiencies were 53.3%, 76.6% and  10.0%. SmPAL1-g1 and SmPAL1-g2 were constructed into vector VK005-03 named as  VK005-03-g1 and VK005-03-g2. The results of sequencing showed that the two  CRISPR/Cas target sequences were all constructed into VK005-03. Here we first laid  the foundation for the study of SmPAL1 and provided an effective strategy for the  screening of sgRNA.\\nCI  - CopyrightÂ© by the Chinese Pharmaceutical Association.\\nFAU - Qiu, Jing-Ren\\nAU  - Qiu JR\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Su, Yue-Kai\\nAU  - Su YK\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Song, Zhen-Qiao\\nAU  - Song ZQ\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Fang, Xin-Sheng\\nAU  - Fang XS\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Li, Jing-Yu\\nAU  - Li JY\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Zhang, Jin\\nAU  - Zhang J\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nFAU - Wang, Jian-Hua\\nAU  - Wang JH\\nAD  - College of Agronomy, Shandong Agricultural University, Taian 271018, China.\\nLA  - chi\\nPT  - Journal Article\\nPL  - China\\nTA  - Zhongguo Zhong Yao Za Zhi\\nJT  - Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi = China journal of Chinese  materia medica\\nJID - 8913656\\nMH  - *CRISPR-Cas Systems\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Metabolic Networks and Pathways/*genetics\\nMH  - Salvia miltiorrhiza/chemistry/*genetics\\nOTO - NOTNLM\\nOT  - CRISPR/Cas9\\nOT  - Salvia miltiorrhiza\\nOT  - gene editing\\nOT  - phenylalanine ammonia-lyase\\nOT  - target efficienty detection\\nCOIS- The authors of this article and the planning committee members and staff have no  relevant financial relationships with commercial interests to disclose.\\nEDAT- 2018/12/26 06:00\\nMHDA- 2019/07/31 06:00\\nCRDT- 2018/12/26 06:00\\nPHST- 2018/04/17 00:00 [received]\\nPHST- 2018/12/26 06:00 [entrez]\\nPHST- 2018/12/26 06:00 [pubmed]\\nPHST- 2019/07/31 06:00 [medline]\\nAID - 10.19540/j.cnki.cjcmm.20180726.007 [doi]\\nPST - ppublish\\nSO  - Zhongguo Zhong Yao Za Zhi. 2018 Nov;43(21):4226-4230. doi:  10.19540/j.cnki.cjcmm.20180726.007.\\n\\nPMID- 34360856\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210826\\nLR  - 20210826\\nIS  - 1422-0067 (Electronic)\\nIS  - 1422-0067 (Linking)\\nVI  - 22\\nIP  - 15\\nDP  - 2021 Jul 28\\nTI  - Integrating Omics and Gene Editing Tools for Rapid Improvement of Traditional Food  Plants for Diversified and Sustainable Food Security.\\nLID - 10.3390/ijms22158093 [doi]\\nLID - 8093\\nAB  - Indigenous communities across the globe, especially in rural areas, consume locally  available plants known as Traditional Food Plants (TFPs) for their nutritional and  health-related needs. Recent research shows that many TFPs are highly nutritious as  they contain health beneficial metabolites, vitamins, mineral elements and other  nutrients. Excessive reliance on the mainstream staple crops has its own  disadvantages. Traditional food plants are nowadays considered important crops of  the future and can act as supplementary foods for the burgeoning global population.  They can also act as emergency foods in situations such as COVID-19 and in times of  other pandemics. The current situation necessitates locally available alternative  nutritious TFPs for sustainable food production. To increase the cultivation or  improve the traits in TFPs, it is essential to understand the molecular basis of the  genes that regulate some important traits such as nutritional components and  resilience to biotic and abiotic stresses. The integrated use of modern omics and  gene editing technologies provide great opportunities to better understand the  genetic and molecular basis of superior nutrient content, climate-resilient traits  and adaptation to local agroclimatic zones. Recently, realizing the importance and  benefits of TFPs, scientists have shown interest in the prospection and sequencing  of TFPs for their improvements, cultivation and mainstreaming. Integrated omics such  as genomics, transcriptomics, proteomics, metabolomics and ionomics are successfully  used in plants and have provided a comprehensive understanding of  gene-protein-metabolite networks. Combined use of omics and editing tools has led to  successful editing of beneficial traits in several TFPs. This suggests that there is  ample scope for improvement of TFPs for sustainable food production. In this  article, we highlight the importance, scope and progress towards improvement of TFPs  for valuable traits by integrated use of omics and gene editing techniques.\\nFAU - Kumar, Ajay\\nAU  - Kumar A\\nAUID- ORCID: 0000-0001-7436-5000\\nAD  - Department of Plant Science, Central University of Kerala, Kasaragod 671316, Kerala,  India.\\nFAU - Anju, Thattantavide\\nAU  - Anju T\\nAUID- ORCID: 0000-0001-6427-5642\\nAD  - Department of Plant Science, Central University of Kerala, Kasaragod 671316, Kerala,  India.\\nFAU - Kumar, Sushil\\nAU  - Kumar S\\nAD  - Department of Botany, Govt. Degree College, Kishtwar 182204, Jammu and Kashmir,  India.\\nFAU - Chhapekar, Sushil Satish\\nAU  - Chhapekar SS\\nAUID- ORCID: 0000-0002-9802-884X\\nAD  - Molecular Genetics & Genomics Laboratory, Department of Horticulture, Chungnam  National University, Daejeon 34134, Korea.\\nFAU - Sreedharan, Sajana\\nAU  - Sreedharan S\\nAD  - Department of Plant Science, Central University of Kerala, Kasaragod 671316, Kerala,  India.\\nFAU - Singh, Sonam\\nAU  - Singh S\\nAD  - Molecular Genetics & Genomics Laboratory, Department of Horticulture, Chungnam  National University, Daejeon 34134, Korea.\\nFAU - Choi, Su Ryun\\nAU  - Choi SR\\nAD  - Molecular Genetics & Genomics Laboratory, Department of Horticulture, Chungnam  National University, Daejeon 34134, Korea.\\nFAU - Ramchiary, Nirala\\nAU  - Ramchiary N\\nAUID- ORCID: 0000-0002-8297-0523\\nAD  - School of Life Sciences, Jawaharlal Nehru University, New Delhi 110067, Delhi,  India.\\nFAU - Lim, Yong Pyo\\nAU  - Lim YP\\nAD  - Molecular Genetics & Genomics Laboratory, Department of Horticulture, Chungnam  National University, Daejeon 34134, Korea.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210728\\nTA  - Int J Mol Sci\\nJT  - International journal of molecular sciences\\nJID - 101092791\\nSB  - IM\\nMH  - Food Security/*methods\\nMH  - Gene Editing\\nMH  - Genomics/methods\\nMH  - Humans\\nMH  - Metabolomics\\nMH  - Plants, Edible/chemistry/*genetics/*metabolism\\nMH  - Proteomics\\nPMC - PMC8348985\\nOTO - NOTNLM\\nOT  - COVID-19\\nOT  - CRISPR/Cas\\nOT  - climate change\\nOT  - food security\\nOT  - gene editing\\nOT  - omics\\nOT  - traditional food plants\\nOT  - translational genomics\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2021/08/08 06:00\\nMHDA- 2021/08/27 06:00\\nCRDT- 2021/08/07 01:08\\nPHST- 2021/06/12 00:00 [received]\\nPHST- 2021/07/21 00:00 [revised]\\nPHST- 2021/07/23 00:00 [accepted]\\nPHST- 2021/08/07 01:08 [entrez]\\nPHST- 2021/08/08 06:00 [pubmed]\\nPHST- 2021/08/27 06:00 [medline]\\nAID - ijms22158093 [pii]\\nAID - ijms-22-08093 [pii]\\nAID - 10.3390/ijms22158093 [doi]\\nPST - epublish\\nSO  - Int J Mol Sci. 2021 Jul 28;22(15):8093. doi: 10.3390/ijms22158093.\\n\\nPMID- 33959107\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210508\\nIS  - 1664-302X (Print)\\nIS  - 1664-302X (Electronic)\\nIS  - 1664-302X (Linking)\\nVI  - 12\\nDP  - 2021\\nTI  - Expanding Diversity of Firmicutes Single-Strand Annealing Proteins: A Putative Role  of Bacteriophage-Host Arms Race.\\nPG  - 644622\\nLID - 10.3389/fmicb.2021.644622 [doi]\\nLID - 644622\\nAB  - Bacteriophage-encoded single strand annealing proteins (SSAPs) are recombinases  which can substitute the classical, bacterial RecA and manage the DNA metabolism at  different steps of phage propagation. SSAPs have been shown to efficiently promote  recombination between short and rather divergent DNA sequences and were exploited  for in vivo genetic engineering mainly in Gram-negative bacteria. In opposition to  the conserved and almost universal bacterial RecA protein, SSAPs display great  sequence diversity. The importance for SSAPs in phage biology and phage-bacteria  evolution is underlined by their role as key players in events of horizontal gene  transfer (HGT). All of the above provoke a constant interest for the identification  and study of new phage recombinase proteins in vivo, in vitro as well as in silico.  Despite this, a huge body of putative ssap genes escapes conventional  classification, as they are not properly annotated. In this work, we performed a  wide-scale identification, classification and analysis of SSAPs encoded by the  Firmicutes bacteria and their phages. By using sequence similarity network and gene  context analyses, we created a new high quality dataset of phage-related SSAPs,  substantially increasing the number of annotated SSAPs. We classified the identified  SSAPs into seven distinct families, namely RecA, Gp2.5, RecT/RedÎ², Erf, Rad52/22,  Sak3, and Sak4, organized into three superfamilies. Analysis of the relationships  between the revealed protein clusters led us to recognize Sak3-like proteins as a  new distinct SSAP family. Our analysis showed an irregular phylogenetic distribution  of ssap genes among different bacterial phyla and specific phages, which can be  explained by the high rates of ssap HGT. We propose that the evolution of phage  recombinases could be tightly linked to the dissemination of bacterial  phage-resistance mechanisms (e.g., abortive infection and CRISPR/Cas systems)  targeting ssap genes and be a part of the constant phage-bacteria arms race.\\nCI  - Copyright Â© 2021 Steczkiewicz, Prestel, Bidnenko and Szczepankowska.\\nFAU - Steczkiewicz, Kamil\\nAU  - Steczkiewicz K\\nAD  - Institute of Biochemistry and Biophysics PAS, Warsaw, Poland.\\nFAU - Prestel, Eric\\nAU  - Prestel E\\nAD  - Micalis Institute, INRAE, AgroParisTech, UniversitÃ© Paris-Saclay, Jouy-en-Josas,  France.\\nFAU - Bidnenko, Elena\\nAU  - Bidnenko E\\nAD  - Micalis Institute, INRAE, AgroParisTech, UniversitÃ© Paris-Saclay, Jouy-en-Josas,  France.\\nFAU - Szczepankowska, Agnieszka K\\nAU  - Szczepankowska AK\\nAD  - Institute of Biochemistry and Biophysics PAS, Warsaw, Poland.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210420\\nTA  - Front Microbiol\\nJT  - Frontiers in microbiology\\nJID - 101548977\\nPMC - PMC8093625\\nOTO - NOTNLM\\nOT  - CLANS\\nOT  - CRISPR/cas\\nOT  - Firmicutes bacteriophages\\nOT  - Sak3/DUF1071\\nOT  - abortive intection\\nOT  - phage recombinase\\nOT  - phage-bacteria arms race\\nOT  - single strand annealing proteins (SSAP)\\nCOIS- The authors declare that the research was conducted in the absence of any commercial  or financial relationships that could be construed as a potential conflict of  interest.\\nEDAT- 2021/05/08 06:00\\nMHDA- 2021/05/08 06:01\\nCRDT- 2021/05/07 07:10\\nPHST- 2021/01/19 00:00 [received]\\nPHST- 2021/03/25 00:00 [accepted]\\nPHST- 2021/05/07 07:10 [entrez]\\nPHST- 2021/05/08 06:00 [pubmed]\\nPHST- 2021/05/08 06:01 [medline]\\nAID - 10.3389/fmicb.2021.644622 [doi]\\nPST - epublish\\nSO  - Front Microbiol. 2021 Apr 20;12:644622. doi: 10.3389/fmicb.2021.644622. eCollection  2021.\\n\\nPMID- 33897769\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210428\\nIS  - 1664-8021 (Print)\\nIS  - 1664-8021 (Electronic)\\nIS  - 1664-8021 (Linking)\\nVI  - 12\\nDP  - 2021\\nTI  - Proteome and Phosphoproteome Analyses Reveal the Kinase Regulatory Network Involved  in Glycogen Synthesis Kinase 3Î².\\nPG  - 657140\\nLID - 10.3389/fgene.2021.657140 [doi]\\nLID - 657140\\nAB  - Diabetic nephropathy is the most common chronic kidney disease in the world and the  main cause of end-stage renal disease (ESRD). The structural integrity of podocytes  is fundamental to the normal function of the glomerulus, and the role of glycogen  synthase kinase 3Î² (GSK-3Î²) in podocytes is complicated. A thorough understanding of  GSK-3Î² is crucial to understand the mechanism of diabetic nephropathy. To analyze  the roles of GSK-3Î² in podocytes, GSK-3Î² knockdown lentivirus by clustered regularly  interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein (Cas)9 was  applied to establish stable cell lines. Mass spectrometry was utilized to search for  differentially expressed proteins. Consequently, we found 34 proteins with higher  levels and 115 proteins with lower levels in GSk-3Î² knockdown cells than in control  cells and identified 581 phosphosites with higher phosphorylation levels and 288  phosphosites with lower phosphorylation levels. We performed functional enrichment  analysis of these proteins and phosphorylated proteins based on public databases.  Enrichment analysis revealed that GSK-3Î² participates in the spliceosome, Hippo  signaling pathway, actin binding, structural molecule activity, and other pathways.  Then, we used motif analysis of phosphate sites to determine 89 conserved motifs  based on 1,068 phosphoserine (pS) sites and 15 conserved motifs in view of 104  phosphothreonine (pT) sites. Additionally, protein-protein interaction network  analysis was carried out using the STRING database. Cytoscape\\'s add-on Molecular  Complex Detection (MCODE) was used to analyze key and core protein groups. In  quantitative differential protein analysis, four MCODEs were obtained, and 22 MCODEs  were obtained in the analysis of the phosphoproteome of differentially expressed  proteins. Finally, we analyzed the kinase regulatory network in podocytes after  GSK-3Î² knockdown and identified 299 protein kinases and 3,460 significantly changed  phosphorylation modification sites on 1,574 proteins. These results will be valuable  for further research on GSK-3Î².\\nCI  - Copyright Â© 2021 Hu, Fang, Wang and Zhou.\\nFAU - Hu, Mingyang\\nAU  - Hu M\\nAD  - The First Affiliated Hospital of Zhengzhou University, Zhengzhou, China.\\nAD  - Zhengzhou University School of Medical Sciences, Zhengzhou, China.\\nFAU - Fang, Jiuyuan\\nAU  - Fang J\\nAD  - School of Basic Medical Sciences, Zhengzhou University, Zhengzhou, China.\\nFAU - Wang, Huijuan\\nAU  - Wang H\\nAD  - The First Affiliated Hospital of Zhengzhou University, Zhengzhou, China.\\nAD  - Zhengzhou University School of Medical Sciences, Zhengzhou, China.\\nFAU - Zhou, Sijie\\nAU  - Zhou S\\nAD  - The First Affiliated Hospital of Zhengzhou University, Zhengzhou, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210407\\nTA  - Front Genet\\nJT  - Frontiers in genetics\\nJID - 101560621\\nPMC - PMC8059835\\nOTO - NOTNLM\\nOT  - GSK-3Î²\\nOT  - diabetic nephropathy\\nOT  - phosphoproteome\\nOT  - podocytes\\nOT  - proteome\\nCOIS- The authors declare that the research was conducted in the absence of any commercial  or financial relationships that could be construed as a potential conflict of  interest.\\nEDAT- 2021/04/27 06:00\\nMHDA- 2021/04/27 06:01\\nCRDT- 2021/04/26 05:50\\nPHST- 2021/01/22 00:00 [received]\\nPHST- 2021/03/04 00:00 [accepted]\\nPHST- 2021/04/26 05:50 [entrez]\\nPHST- 2021/04/27 06:00 [pubmed]\\nPHST- 2021/04/27 06:01 [medline]\\nAID - 10.3389/fgene.2021.657140 [doi]\\nPST - epublish\\nSO  - Front Genet. 2021 Apr 7;12:657140. doi: 10.3389/fgene.2021.657140. eCollection 2021.\\n\\nPMID- 34202963\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210705\\nIS  - 2079-7737 (Print)\\nIS  - 2079-7737 (Electronic)\\nIS  - 2079-7737 (Linking)\\nVI  - 10\\nIP  - 6\\nDP  - 2021 Jun 20\\nTI  - Analysing Parallel Strategies to Alter the Host Specificity of Bacteriophage T7.\\nLID - 10.3390/biology10060556 [doi]\\nLID - 556\\nAB  - The recognition and binding of host bacteria by bacteriophages is most often enabled  by a highly specific receptor-ligand type of interaction, with the receptor-binding  proteins (RBPs) of phages being the primary determinants of host specificity.  Specifically modifying the RBPs could alter or extend the host range of phages  otherwise exhibiting desired phenotypic properties. This study employed two  different strategies to reprogram T7 phages ordinarily infecting commensal K12  Escherichia coli strains to infect pathogen-associated K1-capsule-expressing  strains. The strategies were based on either plasmid-based homologous recombination  or bacteriophage recombineering using electroporated DNA (BRED). Our work pursued  the construction of two genetic designs: one replacing the gp17 gene of T7, the  other replacing gp11, gp12, and gp17 of T7 with their K1F counterparts. Both  strategies displayed successful integration of the K1F sequences into the T7 genome,  detected by PCR screening. Multiple methods were utilised to select or enrich for  chimeric phages incorporating the K1F gp17 alone, including trxA, host-specificity,  and CRISPR-Cas-based selection. Irrespective of the selection method, the above  strategy yielded poorly reproducible phage propagation on the new host, indicating  that the chimeric phage was less fit than the wild type and could not promote  continual autonomous reproduction. Chimeric phages obtained from BRED incorporating  gp11-12 and gp17, however, all displayed infection in a 2-stage pattern, indicating  the presence of both K1F and T7 phenotypes. This study shows that BRED can be used  as a tool to quickly access the potential of new RBP constructs without the need to  engineer sustainably replicating phages. Additionally, we show that solely  repurposing the primary RBP is, in some cases, insufficient to produce a viable  chimeric phage.\\nFAU - Avramucz, Ãkos\\nAU  - Avramucz Ã\\nAD  - Synthetic and Systems Biology Unit, Biological Research Centre, EÃ¶tvÃ¶s LorÃ¡nd  Research Network (ELKH), 6726 Szeged, Hungary.\\nAD  - Doctoral School in Biology, University of Szeged, 6726 Szeged, Hungary.\\nFAU - MÃ¸ller-Olsen, Christian\\nAU  - MÃ¸ller-Olsen C\\nAD  - School of Life Sciences, University of Warwick, Coventry CV4 7AL, UK.\\nFAU - Grigonyte, Aurelija M\\nAU  - Grigonyte AM\\nAD  - School of Life Sciences, University of Warwick, Coventry CV4 7AL, UK.\\nFAU - Paramalingam, Yanahan\\nAU  - Paramalingam Y\\nAD  - School of Life Sciences, University of Warwick, Coventry CV4 7AL, UK.\\nFAU - Millard, Andrew\\nAU  - Millard A\\nAD  - Department Genetics and Genome Biology, University of Leicester, Leicester LE1 7RH,  UK.\\nFAU - Sagona, Antonia P\\nAU  - Sagona AP\\nAUID- ORCID: 0000-0002-0386-2322\\nAD  - School of Life Sciences, University of Warwick, Coventry CV4 7AL, UK.\\nAD  - Warwick Integrative Synthetic Biology Centre, University of Warwick, Coventry CV4  7AL, UK.\\nFAU - FehÃ©r, TamÃ¡s\\nAU  - FehÃ©r T\\nAUID- ORCID: 0000-0001-9318-3640\\nAD  - Synthetic and Systems Biology Unit, Biological Research Centre, EÃ¶tvÃ¶s LorÃ¡nd  Research Network (ELKH), 6726 Szeged, Hungary.\\nLA  - eng\\nGR  - Grant No. K119298/National Research, Development, and Innovation Office of Hungary  (NKFIH)/\\nGR  - ref. BB/N011872/1/Biological Sciences Research Council (BBSRC) Future Leader  Fellowship/\\nGR  - BB/M01116X/1/Biotechnology and Biological Sciences Research Council (BBSRC) and  University of Warwick funded Midlands Integrative Biosciences Training Partnership  (MIBTP)/\\nPT  - Journal Article\\nDEP - 20210620\\nTA  - Biology (Basel)\\nJT  - Biology\\nJID - 101587988\\nPMC - PMC8234382\\nOTO - NOTNLM\\nOT  - BRED\\nOT  - RBP\\nOT  - bacteriophage\\nOT  - host range\\nOT  - host specificity\\nOT  - phage\\nOT  - tail fibres\\nCOIS- The authors declare no conflict of interests.\\nEDAT- 2021/07/03 06:00\\nMHDA- 2021/07/03 06:01\\nCRDT- 2021/07/02 01:15\\nPHST- 2021/05/20 00:00 [received]\\nPHST- 2021/06/14 00:00 [revised]\\nPHST- 2021/06/16 00:00 [accepted]\\nPHST- 2021/07/02 01:15 [entrez]\\nPHST- 2021/07/03 06:00 [pubmed]\\nPHST- 2021/07/03 06:01 [medline]\\nAID - biology10060556 [pii]\\nAID - biology-10-00556 [pii]\\nAID - 10.3390/biology10060556 [doi]\\nPST - epublish\\nSO  - Biology (Basel). 2021 Jun 20;10(6):556. doi: 10.3390/biology10060556.\\n\\nPMID- 29685154\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180924\\nLR  - 20181114\\nIS  - 1475-2859 (Electronic)\\nIS  - 1475-2859 (Linking)\\nVI  - 17\\nIP  - 1\\nDP  - 2018 Apr 23\\nTI  - A RecET-assisted CRISPR-Cas9 genome editing in Corynebacterium glutamicum.\\nPG  - 63\\nLID - 10.1186/s12934-018-0910-2 [doi]\\nLID - 63\\nAB  - BACKGROUND: Extensive modification of genome is an efficient manner to regulate the  metabolic network for producing target metabolites or non-native products using  Corynebacterium glutamicum as a cell factory. Genome editing approaches by means of  homologous recombination and counter-selection markers are laborious and time  consuming due to multiple round manipulations and low editing efficiencies. The  current two-plasmid-based CRISPR-Cas9 editing methods generate false positives due  to the potential instability of Cas9 on the plasmid, and require a high  transformation efficiency for co-occurrence of two plasmids transformation. RESULTS:  Here, we developed a RecET-assisted CRISPR-Cas9 genome editing method using a  chromosome-borne Cas9-RecET and a single plasmid harboring sgRNA and repair  templates. The inducible expression of chromosomal RecET promoted the frequencies of  homologous recombination, and increased the efficiency for gene deletion. Due to the  high transformation efficiency of a single plasmid, this method enabled 10- and  20-kb region deletion, 2.5-, 5.7- and 7.5-kb expression cassette insertion and  precise site-specific mutation, suggesting a versatility of this method. Deletion of  argR and farR regulators as well as site-directed mutation of argB and pgi genes  generated the mutant capable of accumulating L-arginine, indicating the stability of  chromosome-borne Cas9 for iterative genome editing. Using this method, the  model-predicted target genes were modified to redirect metabolic flux towards  1,2-propanediol biosynthetic pathway. The final engineered strain produced  6.75â€‰Â±â€‰0.46Â\\xa0g/L of 1,2-propanediol that is the highest titer reported in C.  glutamicum. Furthermore, this method is available for Corynebacterium pekinense  1.563, suggesting its universal applicability in other Corynebacterium species.  CONCLUSIONS: The RecET-assisted CRISPR-Cas9 genome editing method will facilitate  engineering of metabolic networks for the synthesis of interested bio-based products  from renewable biomass using Corynebacterium species as cell factories.\\nFAU - Wang, Bo\\nAU  - Wang B\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Hu, Qitiao\\nAU  - Hu Q\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Zhang, Yu\\nAU  - Zhang Y\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Shi, Ruilin\\nAU  - Shi R\\nAD  - Beijing Zhongke Eppen Biotech Co., Ltd, Beijing, 100085, China.\\nFAU - Chai, Xin\\nAU  - Chai X\\nAD  - Beijing Zhongke Eppen Biotech Co., Ltd, Beijing, 100085, China.\\nFAU - Liu, Zhe\\nAU  - Liu Z\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Shang, Xiuling\\nAU  - Shang X\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.\\nFAU - Zhang, Yun\\nAU  - Zhang Y\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China.  zhangyun@im.ac.cn.\\nFAU - Wen, Tingyi\\nAU  - Wen T\\nAUID- ORCID: 0000-0003-2583-7088\\nAD  - CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of  Microbiology, Chinese Academy of Sciences, Beijing, 100101, China. wenty@im.ac.cn.\\nAD  - Beijing Zhongke Eppen Biotech Co., Ltd, Beijing, 100085, China. wenty@im.ac.cn.\\nAD  - Savaid Medical School, University of Chinese Academy of Sciences, Beijing, 100049,  China. wenty@im.ac.cn.\\nLA  - eng\\nGR  - KFJ-STS-QYZD-047/the Science and Technology Service Network Initiative of the  Chinese Academy of Sciences/\\nGR  - KFJ-EW-STS-078/the Science and Technology Service Network Initiative of the Chinese  Academy of Sciences/\\nGR  - 31100074/National Natural Science Foundation of China/\\nPT  - Journal Article\\nDEP - 20180423\\nTA  - Microb Cell Fact\\nJT  - Microbial cell factories\\nJID - 101139812\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Corynebacterium glutamicum/*metabolism\\nMH  - Gene Editing/*methods\\nPMC - PMC5913818\\nOTO - NOTNLM\\nOT  - 1,2-Propanediol\\nOT  - CRISPRâ€“Cas9\\nOT  - Corynebacterium glutamicum\\nOT  - Genome editing\\nOT  - Transformation efficiency\\nOT  - recET-encoded recombinase\\nEDAT- 2018/04/25 06:00\\nMHDA- 2018/09/25 06:00\\nCRDT- 2018/04/25 06:00\\nPHST- 2018/01/16 00:00 [received]\\nPHST- 2018/04/13 00:00 [accepted]\\nPHST- 2018/04/25 06:00 [entrez]\\nPHST- 2018/04/25 06:00 [pubmed]\\nPHST- 2018/09/25 06:00 [medline]\\nAID - 10.1186/s12934-018-0910-2 [pii]\\nAID - 910 [pii]\\nAID - 10.1186/s12934-018-0910-2 [doi]\\nPST - epublish\\nSO  - Microb Cell Fact. 2018 Apr 23;17(1):63. doi: 10.1186/s12934-018-0910-2.\\n\\nPMID- 30173772\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190624\\nLR  - 20190624\\nIS  - 1557-7988 (Electronic)\\nIS  - 0076-6879 (Linking)\\nVI  - 608\\nDP  - 2018\\nTI  - Modular Pathway Rewiring of Yeast for Amino Acid Production.\\nPG  - 417-439\\nLID - S0076-6879(18)30243-X [pii]\\nLID - 10.1016/bs.mie.2018.06.009 [doi]\\nAB  - Amino acids find various applications in biotechnology in view of their importance  in the food, feed, pharmaceutical, and personal care industries as nutrients,  additives, and drugs, respectively. For the large-scale production of amino acids,  microbial cell factories are widely used and the development of amino acid-producing  strains has mainly focused on prokaryotes Corynebacterium glutamicum and Escherichia  coli. However, the eukaryote Saccharomyces cerevisiae is becoming an even more  appealing microbial host for production of amino acids and derivatives because of  its superior molecular and physiological features, such as amenable to genetic  engineering and high tolerance to harsh conditions. To transform S. cerevisiae into  an industrial amino acid production platform, the highly coordinated and multiple  layers regulation in its amino acid metabolism should be relieved and reconstituted  to optimize the metabolic flux toward synthesis of target products. This chapter  describes principles, strategies, and applications of modular pathway rewiring in  yeast using the engineering of l-ornithine metabolism as a paradigm. Additionally,  detailed protocols for in vitro module construction and CRISPR/Cas-mediated pathway  assembly are provided.\\nCI  - Â© 2018 Elsevier Inc. All rights reserved.\\nFAU - Liu, Quanli\\nAU  - Liu Q\\nAD  - Department of Biology and Biological Engineering, Chalmers University of Technology,  Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers  University of Technology, Gothenburg, Sweden.\\nFAU - Yu, Tao\\nAU  - Yu T\\nAD  - Department of Biology and Biological Engineering, Chalmers University of Technology,  Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers  University of Technology, Gothenburg, Sweden.\\nFAU - Campbell, Kate\\nAU  - Campbell K\\nAD  - Department of Biology and Biological Engineering, Chalmers University of Technology,  Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers  University of Technology, Gothenburg, Sweden.\\nFAU - Nielsen, Jens\\nAU  - Nielsen J\\nAD  - Department of Biology and Biological Engineering, Chalmers University of Technology,  Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers  University of Technology, Gothenburg, Sweden; Novo Nordisk Foundation Center for  Biosustainability, Technical University of Denmark, Kongens Lyngby, Denmark.  Electronic address: nielsenj@chalmers.se.\\nFAU - Chen, Yun\\nAU  - Chen Y\\nAD  - Department of Biology and Biological Engineering, Chalmers University of Technology,  Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers  University of Technology, Gothenburg, Sweden.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180731\\nPL  - United States\\nTA  - Methods Enzymol\\nJT  - Methods in enzymology\\nJID - 0212271\\nRN  - 0 (Amino Acids)\\nRN  - E524N2IXA3 (Ornithine)\\nSB  - IM\\nMH  - Amino Acids/genetics/*metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Industrial Microbiology/*methods\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Flux Analysis/methods\\nMH  - Metabolic Networks and Pathways\\nMH  - Ornithine/genetics/metabolism\\nMH  - Saccharomyces cerevisiae/genetics/*metabolism\\nOTO - NOTNLM\\nOT  - *Amino acids\\nOT  - *Metabolic flux\\nOT  - *Modular pathway engineering\\nOT  - *Saccharomyces cerevisiae\\nOT  - *l-Ornithine\\nEDAT- 2018/09/04 06:00\\nMHDA- 2019/06/25 06:00\\nCRDT- 2018/09/04 06:00\\nPHST- 2018/09/04 06:00 [entrez]\\nPHST- 2018/09/04 06:00 [pubmed]\\nPHST- 2019/06/25 06:00 [medline]\\nAID - S0076-6879(18)30243-X [pii]\\nAID - 10.1016/bs.mie.2018.06.009 [doi]\\nPST - ppublish\\nSO  - Methods Enzymol. 2018;608:417-439. doi: 10.1016/bs.mie.2018.06.009. Epub 2018 Jul  31.\\n\\nPMID- 27725824\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200930\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Electronic)\\nIS  - 1664-462X (Linking)\\nVI  - 7\\nDP  - 2016\\nTI  - Recent Developments in Systems Biology and Metabolic Engineering of Plant-Microbe  Interactions.\\nPG  - 1421\\nLID - 1421\\nAB  - Microorganisms play a crucial role in the sustainability of the various ecosystems.  The characterization of various interactions between microorganisms and other biotic  factors is a necessary footstep to understand the association and functions of  microbial communities. Among the different microbial interactions in an ecosystem,  plant-microbe interaction plays an important role to balance the ecosystem. The  present review explores plant-microbe interactions using gene editing and system  biology tools toward the comprehension in improvement of plant traits. Further,  system biology tools like FBA (flux balance analysis), OptKnock, and  constraint-based modeling helps in understanding such interactions as a whole. In  addition, various gene editing tools have been summarized and a strategy has been  hypothesized for the development of disease free plants. Furthermore, we have tried  to summarize the predictions through data retrieved from various types of sources  such as high throughput sequencing data (e.g., single nucleotide polymorphism  detection, RNA-seq, proteomics) and metabolic models have been reconstructed from  such sequences for species communities. It is well known fact that systems biology  approaches and modeling of biological networks will enable us to learn the insight  of such network and will also help further in understanding these interactions.\\nFAU - Kumar, Vishal\\nAU  - Kumar V\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University Rohtak, India.\\nFAU - Baweja, Mehak\\nAU  - Baweja M\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University Rohtak, India.\\nFAU - Singh, Puneet K\\nAU  - Singh PK\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University Rohtak, India.\\nFAU - Shukla, Pratyoosh\\nAU  - Shukla P\\nAD  - Enzyme Technology and Protein Bioinformatics Laboratory, Department of Microbiology,  Maharshi Dayanand University Rohtak, India.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20160926\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC5035732\\nOTO - NOTNLM\\nOT  - CRISPR-Cas\\nOT  - gene editing\\nOT  - plantâ€“microbe interactions\\nOT  - signaling\\nOT  - systems biology\\nEDAT- 2016/10/12 06:00\\nMHDA- 2016/10/12 06:01\\nCRDT- 2016/10/12 06:00\\nPHST- 2016/07/21 00:00 [received]\\nPHST- 2016/09/06 00:00 [accepted]\\nPHST- 2016/10/12 06:00 [entrez]\\nPHST- 2016/10/12 06:00 [pubmed]\\nPHST- 2016/10/12 06:01 [medline]\\nAID - 10.3389/fpls.2016.01421 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2016 Sep 26;7:1421. doi: 10.3389/fpls.2016.01421. eCollection 2016.\\n\\nPMID- 31195957\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190730\\nLR  - 20200225\\nIS  - 1471-2105 (Electronic)\\nIS  - 1471-2105 (Linking)\\nVI  - 20\\nIP  - 1\\nDP  - 2019 Jun 13\\nTI  - Prediction of activity and specificity of CRISPR-Cpf1 using convolutional deep  learning neural networks.\\nPG  - 332\\nLID - 10.1186/s12859-019-2939-6 [doi]\\nLID - 332\\nAB  - BACKGROUND: CRISPR-Cpf1 has recently been reported as another RNA-guided  endonuclease of class 2 CRISPR-Cas system, which expands the molecular biology  toolkit for genome editing. However, most of the online tools and applications to  date have been developed primarily for the Cas9. There are a limited number of tools  available for the Cpf1. RESULTS: We present DeepCpf1, a deep convolution neural  networks (CNN) approach to predict Cpf1 guide RNAs on-target activity and off-target  effects using their matched and mismatched DNA sequences. Trained on published data  sets, DeepCpf1 is superior to other machine learning algorithms and reliably  predicts the most efficient and less off-target effects guide RNAs for a given gene.  Combined with a permutation importance analysis, the key features of guide RNA  sequences are identified, which determine the activity and specificity of genome  editing. CONCLUSIONS: DeepCpf1 can significantly improve the accuracy of Cpf1-based  genome editing and facilitates the generation of optimized guide RNAs libraries.\\nFAU - Luo, Jiesi\\nAU  - Luo J\\nAUID- ORCID: 0000-0002-1199-7024\\nAD  - Department of Pharmacology, Key Laboratory for Aging and Regenerative Medicine,  School of Pharmacy, Southwest Medical University, Luzhou, Sichuan, China.  ljs@swmu.edu.cn.\\nAD  - Center for Bioinformatics and Systems Biology and Department of Radiology, Wake  Forest School of Medicine, Winston-Salem, NC, 27157, USA. ljs@swmu.edu.cn.\\nFAU - Chen, Wei\\nAU  - Chen W\\nAD  - Center for Bioinformatics and Systems Biology and Department of Radiology, Wake  Forest School of Medicine, Winston-Salem, NC, 27157, USA.\\nFAU - Xue, Li\\nAU  - Xue L\\nAD  - School of Public Health, Southwest Medical University, Luzhou, Sichuan, China.\\nFAU - Tang, Bin\\nAU  - Tang B\\nAD  - Basic Medical College of Southwest Medical University, Luzhou, Sichuan, China.  bt@swmu.edu.cn.\\nLA  - eng\\nGR  - 1U01CA166886/Foundation for the National Institutes of Health/\\nGR  - No. 21803045/National Natural Science Foundation of China/\\nPT  - Journal Article\\nDEP - 20190613\\nTA  - BMC Bioinformatics\\nJT  - BMC bioinformatics\\nJID - 100965194\\nRN  - 0 (RNA, Guide)\\nRN  - EC 3.1.- (Endonucleases)\\nMH  - Algorithms\\nMH  - Base Sequence\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - *Deep Learning\\nMH  - Endonucleases/*metabolism\\nMH  - *Neural Networks, Computer\\nMH  - RNA, Guide/genetics\\nPMC - PMC6567654\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Deep learning\\nOT  - Guide RNAs design\\nCOIS- The authors declare that they have no competing interests.\\nEDAT- 2019/06/15 06:00\\nMHDA- 2019/07/31 06:00\\nCRDT- 2019/06/15 06:00\\nPHST- 2018/02/13 00:00 [received]\\nPHST- 2019/06/07 00:00 [accepted]\\nPHST- 2019/06/15 06:00 [entrez]\\nPHST- 2019/06/15 06:00 [pubmed]\\nPHST- 2019/07/31 06:00 [medline]\\nAID - 10.1186/s12859-019-2939-6 [pii]\\nAID - 2939 [pii]\\nAID - 10.1186/s12859-019-2939-6 [doi]\\nPST - epublish\\nSO  - BMC Bioinformatics. 2019 Jun 13;20(1):332. doi: 10.1186/s12859-019-2939-6.\\n\\nPMID- 30356064\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190102\\nLR  - 20200713\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 9\\nIP  - 1\\nDP  - 2018 Oct 24\\nTI  - SOX17 regulates uterine epithelial-stromal cross-talk acting via a distal enhancer  upstream of Ihh.\\nPG  - 4421\\nLID - 10.1038/s41467-018-06652-w [doi]\\nLID - 4421\\nAB  - Mammalian pregnancy depends on the ability of the uterus to support embryo  implantation. Previous studies reveal the Sox17 gene as a downstream target of the  Pgr-Gata2-dependent transcription network that directs genomic actions in the  uterine endometrium receptive for embryo implantation. Here, we report that ablating  Sox17 in the uterine epithelium impairs leukemia inhibitory factor (LIF) and Indian  hedgehog homolog (IHH) signaling, leading to failure of embryo implantation. In vivo  deletion of the SOX17-binding region 19â€‰kb upstream of the Ihh locus by CRISPR-Cas  technology reduces Ihh expression specifically in the uterus and alters proper  endometrial epithelial-stromal interactions, thereby impairing pregnancy. This  SOX17-binding interval is also bound by GATA2, FOXA2, and PGR. This cluster of  transcription factor binding is common in 737 uterine genes and may represent a key  regulatory element essential for uterine epithelial gene expression.\\nFAU - Wang, Xiaoqiu\\nAU  - Wang X\\nAUID- ORCID: 0000-0001-9660-528X\\nAD  - Reproductive and Development Biology Laboratory, National Institute of Environmental  Health Sciences, Research Triangle Park, NC, USA.\\nAD  - Department of Animal Science, North Carolina State University, Raleigh, NC, USA.\\nFAU - Li, Xilong\\nAU  - Li X\\nAD  - Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston,  TX, USA.\\nAD  - Feed Research Institute, Chinese Academy of Agricultural Sciences, Beijing, China.\\nFAU - Wang, Tianyuan\\nAU  - Wang T\\nAD  - Integrative Bioinformatics Support Group, National Institute of Environmental Health  Sciences, Research Triangle Park, NC, USA.\\nFAU - Wu, San-Pin\\nAU  - Wu SP\\nAD  - Reproductive and Development Biology Laboratory, National Institute of Environmental  Health Sciences, Research Triangle Park, NC, USA.\\nFAU - Jeong, Jae-Wook\\nAU  - Jeong JW\\nAD  - Department of Obstetrics and Gynecology and Reproductive Biology, Michigan State  University, Grand Rapids, MI, USA.\\nFAU - Kim, Tae Hoon\\nAU  - Kim TH\\nAD  - Department of Obstetrics and Gynecology and Reproductive Biology, Michigan State  University, Grand Rapids, MI, USA.\\nFAU - Young, Steven L\\nAU  - Young SL\\nAD  - Department of Obstetrics and Gynecology, University of North Carolina, Chapel Hill,  NC, USA.\\nFAU - Lessey, Bruce A\\nAU  - Lessey BA\\nAD  - Deptartment of Obstetrics and Gynecology, University of South Carolina School of  Medicine, Greenville, SC, USA.\\nFAU - Lanz, Rainer B\\nAU  - Lanz RB\\nAD  - Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston,  TX, USA.\\nFAU - Lydon, John P\\nAU  - Lydon JP\\nAD  - Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston,  TX, USA.\\nFAU - DeMayo, Francesco J\\nAU  - DeMayo FJ\\nAD  - Reproductive and Development Biology Laboratory, National Institute of Environmental  Health Sciences, Research Triangle Park, NC, USA. francesco.demayo@nih.gov.\\nLA  - eng\\nGR  - R01 HD042311/HD/NICHD NIH HHS/United States\\nGR  - R01 HD067721/HD/NICHD NIH HHS/United States\\nGR  - HD-042311/U.S. Department of Health &amp; Human Services | NIH | Eunice Kennedy  Shriver National Institute of Child Health and Human Development  (NICHD)/International\\nGR  - Z1AES103311-01/U.S. Department of Health &amp; Human Services | NIH | National  Institute of Environmental Health Sciences (NIEHS)/International\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20181024\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (Foxa2 protein, mouse)\\nRN  - 0 (GATA2 Transcription Factor)\\nRN  - 0 (Gata2 protein, mouse)\\nRN  - 0 (HMGB Proteins)\\nRN  - 0 (Hedgehog Proteins)\\nRN  - 0 (Leukemia Inhibitory Factor)\\nRN  - 0 (SOXF Transcription Factors)\\nRN  - 0 (Sox17 protein, mouse)\\nRN  - 135845-92-0 (Hepatocyte Nuclear Factor 3-beta)\\nSB  - IM\\nMH  - Animals\\nMH  - CRISPR-Cas Systems/genetics/physiology\\nMH  - Endometrium/metabolism\\nMH  - Female\\nMH  - GATA2 Transcription Factor/genetics/metabolism\\nMH  - HMGB Proteins/genetics/*metabolism\\nMH  - Hedgehog Proteins/genetics/metabolism\\nMH  - Hepatocyte Nuclear Factor 3-beta/genetics/metabolism\\nMH  - Leukemia Inhibitory Factor/genetics/metabolism\\nMH  - Mice\\nMH  - SOXF Transcription Factors/genetics/*metabolism\\nMH  - Transcriptome/genetics\\nMH  - Uterus/*metabolism\\nPMC - PMC6200785\\nCOIS- The authors declare no competing interests.\\nEDAT- 2018/10/26 06:00\\nMHDA- 2019/01/03 06:00\\nCRDT- 2018/10/26 06:00\\nPHST- 2018/03/06 00:00 [received]\\nPHST- 2018/08/31 00:00 [accepted]\\nPHST- 2018/10/26 06:00 [entrez]\\nPHST- 2018/10/26 06:00 [pubmed]\\nPHST- 2019/01/03 06:00 [medline]\\nAID - 10.1038/s41467-018-06652-w [pii]\\nAID - 6652 [pii]\\nAID - 10.1038/s41467-018-06652-w [doi]\\nPST - epublish\\nSO  - Nat Commun. 2018 Oct 24;9(1):4421. doi: 10.1038/s41467-018-06652-w.\\n\\nPMID- 21139194\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110321\\nLR  - 20211020\\nIS  - 1744-3091 (Electronic)\\nIS  - 1744-3091 (Linking)\\nVI  - 66\\nIP  - Pt 12\\nDP  - 2010 Dec 1\\nTI  - Structure of a CRISPR-associated protein Cas2 from Desulfovibrio vulgaris.\\nPG  - 1552-6\\nLID - 10.1107/S1744309110039801 [doi]\\nAB  - CRISPRs (clustered regularly interspaced short palindromic repeats) provide bacteria  and archaea with RNA-guided acquired immunity to invasive DNAs. CRISPR-associated  (Cas) proteins carry out the immune effector functions. Cas2 is a universal  component of the CRISPR system. Here, a 1.35â€…Ã… resolution crystal structure of Cas2  from the bacterium Desulfovibrio vulgaris (DvuCas2) isÂ\\xa0reported. DvuCas2 is a  homodimer, with each protomer consisting of an N-terminal Î²Î±Î²Î²Î±Î² ferredoxin fold  (amino acids 1-78) to which is appended a C-terminal segment (amino acids 79-102)  that includes a short 3(10)-helix and a fifth Î²-strand. The Î²5 strands align with  the Î²4 strands of the opposite protomers, resulting in two five-stranded  antiparallel Î²-sheets that form a sandwich at the dimer interface. The DvuCas2 dimer  is stabilized by a distinctive network of hydrophilic cross-protomer side-chain  interactions.\\nFAU - Samai, Poulami\\nAU  - Samai P\\nAD  - Molecular Biology Program, Sloan-Kettering Institute for Cancer Research, USA.\\nFAU - Smith, Paul\\nAU  - Smith P\\nFAU - Shuman, Stewart\\nAU  - Shuman S\\nLA  - eng\\nSI  - PDB/3OQ2\\nGR  - R01 GM063611/GM/NIGMS NIH HHS/United States\\nGR  - GM63611/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20101116\\nTA  - Acta Crystallogr Sect F Struct Biol Cryst Commun\\nJT  - Acta crystallographica. Section F, Structural biology and crystallization  communications\\nJID - 101226117\\nRN  - 0 (Bacterial Proteins)\\nRN  - EC 3.1.4.- (Phosphoric Diester Hydrolases)\\nSB  - IM\\nMH  - Amino Acid Sequence\\nMH  - Bacterial Proteins/*chemistry\\nMH  - Catalytic Domain\\nMH  - Crystallography, X-Ray\\nMH  - Desulfovibrio vulgaris/*chemistry\\nMH  - Molecular Sequence Data\\nMH  - Phosphoric Diester Hydrolases/metabolism\\nMH  - Protein Multimerization\\nMH  - Protein Structure, Secondary\\nMH  - *Repetitive Sequences, Amino Acid\\nMH  - Sequence Homology, Amino Acid\\nPMC - PMC2998353\\nEDAT- 2010/12/09 06:00\\nMHDA- 2011/03/22 06:00\\nCRDT- 2010/12/09 06:00\\nPHST- 2010/09/08 00:00 [received]\\nPHST- 2010/10/05 00:00 [accepted]\\nPHST- 2010/12/09 06:00 [entrez]\\nPHST- 2010/12/09 06:00 [pubmed]\\nPHST- 2011/03/22 06:00 [medline]\\nAID - S1744309110039801 [pii]\\nAID - sw5037 [pii]\\nAID - 10.1107/S1744309110039801 [doi]\\nPST - ppublish\\nSO  - Acta Crystallogr Sect F Struct Biol Cryst Commun. 2010 Dec 1;66(Pt 12):1552-6. doi:  10.1107/S1744309110039801. Epub 2010 Nov 16.\\n\\nPMID- 33421147\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210531\\nLR  - 20210531\\nIS  - 1399-3054 (Electronic)\\nIS  - 0031-9317 (Linking)\\nVI  - 172\\nIP  - 2\\nDP  - 2021 Jun\\nTI  - A manipulative interplay between positive and negative regulators of phytohormones:  A way forward for improving drought tolerance in plants.\\nPG  - 1269-1290\\nLID - 10.1111/ppl.13325 [doi]\\nAB  - Among different abiotic stresses, drought stress is the leading cause of impaired  plant growth and low productivity worldwide. It is therefore essential to understand  the process of drought tolerance in plants and thus to enhance drought resistance.  Accumulating evidence indicates that phytohormones are essential signaling molecules  that regulate diverse processes of plant growth and development under drought  stress. Plants can often respond to drought stress through a cascade of  phytohormones signaling as a means of plant growth regulation. Understanding  biosynthesis pathways and regulatory crosstalk involved in these vital compounds  could pave the way for improving plant drought tolerance while maintaining overall  plant health. In recent years, the identification of phytohormones related key  regulatory genes and their manipulation through state-of-the-art genome engineering  tools have helped to improve drought tolerance plants. To date, several genes linked  to phytohormones signaling networks, biosynthesis, and metabolism have been  described as a promising contender for engineering drought tolerance. Recent  advances in functional genomics have shown that enhanced expression of positive  regulators involved in hormone biosynthesis could better equip plants against  drought stress. Similarly, knocking down negative regulators of phytohormone  biosynthesis can also be very effective to negate the negative effects of drought on  plants. This review explained how manipulating positive and negative regulators of  phytohormone signaling could be improvised to develop future crop varieties  exhibiting higher drought tolerance. In addition, we also discuss the role of a  promising genome editing tool, CRISPR/Cas9, on phytohormone mediated plant growth  regulation for tackling drought stress.\\nCI  - Â© 2021 Scandinavian Plant Physiology Society.\\nFAU - Mubarik, Muhammad Salman\\nAU  - Mubarik MS\\nAUID- ORCID: 0000-0001-6522-4660\\nAD  - Centre of Agricultural Biochemistry and Biotechnology (CABB), University of  Agriculture, Faisalabad, Pakistan.\\nAD  - Center for Advanced Studies in Agriculture and Food Security (CAS-AFS), University  of Agriculture, Faisalabad, Pakistan.\\nFAU - Khan, Sultan Habibullah\\nAU  - Khan SH\\nAUID- ORCID: 0000-0002-3200-5151\\nAD  - Centre of Agricultural Biochemistry and Biotechnology (CABB), University of  Agriculture, Faisalabad, Pakistan.\\nAD  - Center for Advanced Studies in Agriculture and Food Security (CAS-AFS), University  of Agriculture, Faisalabad, Pakistan.\\nFAU - Sajjad, Muhammad\\nAU  - Sajjad M\\nAUID- ORCID: 0000-0002-1036-2510\\nAD  - Department of Biosciences, COMSATS University Islamabad (CUI), Islamabad, Pakistan.\\nFAU - Raza, Ali\\nAU  - Raza A\\nAUID- ORCID: 0000-0002-5120-2791\\nAD  - Key Laboratory of Biology and Genetic Improvement of Oil Crops, Oil Crops Research  Institute, Chinese Academy of Agricultural Sciences (CAAS), Wuhan, China.\\nFAU - Hafeez, Muhammad Bilal\\nAU  - Hafeez MB\\nAUID- ORCID: 0000-0001-6719-6667\\nAD  - College of Agronomy, Northwest A&F University, Yangling, China.\\nFAU - Yasmeen, Tahira\\nAU  - Yasmeen T\\nAUID- ORCID: 0000-0002-8854-1148\\nAD  - Department of Environmental Sciences and Engineering, Government College University  Faisalabad, Faisalabad, Pakistan.\\nFAU - Rizwan, Muhammad\\nAU  - Rizwan M\\nAUID- ORCID: 0000-0002-3513-2041\\nAD  - Department of Environmental Sciences and Engineering, Government College University  Faisalabad, Faisalabad, Pakistan.\\nFAU - Ali, Shafaqat\\nAU  - Ali S\\nAUID- ORCID: 0000-0003-0272-7097\\nAD  - Department of Environmental Sciences and Engineering, Government College University  Faisalabad, Faisalabad, Pakistan.\\nFAU - Arif, Muhammad Saleem\\nAU  - Arif MS\\nAUID- ORCID: 0000-0003-1247-0539\\nAD  - Department of Environmental Sciences and Engineering, Government College University  Faisalabad, Faisalabad, Pakistan.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210202\\nPL  - Denmark\\nTA  - Physiol Plant\\nJT  - Physiologia plantarum\\nJID - 1256322\\nRN  - 0 (Plant Growth Regulators)\\nSB  - IM\\nMH  - *Droughts\\nMH  - Plant Development\\nMH  - *Plant Growth Regulators\\nMH  - Plants/genetics\\nMH  - Stress, Physiological\\nEDAT- 2021/01/10 06:00\\nMHDA- 2021/06/01 06:00\\nCRDT- 2021/01/09 17:09\\nPHST- 2020/11/20 00:00 [revised]\\nPHST- 2020/08/31 00:00 [received]\\nPHST- 2020/12/23 00:00 [accepted]\\nPHST- 2021/01/10 06:00 [pubmed]\\nPHST- 2021/06/01 06:00 [medline]\\nPHST- 2021/01/09 17:09 [entrez]\\nAID - 10.1111/ppl.13325 [doi]\\nPST - ppublish\\nSO  - Physiol Plant. 2021 Jun;172(2):1269-1290. doi: 10.1111/ppl.13325. Epub 2021 Feb 2.\\n\\nPMID- 31042076\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200327\\nLR  - 20220129\\nIS  - 1522-1504 (Electronic)\\nIS  - 1040-0605 (Print)\\nIS  - 1040-0605 (Linking)\\nVI  - 317\\nIP  - 2\\nDP  - 2019 Aug 1\\nTI  - Extrinsic acidosis suppresses glycolysis and migration while increasing network  formation in pulmonary microvascular endothelial cells.\\nPG  - L188-L201\\nLID - 10.1152/ajplung.00544.2018 [doi]\\nAB  - Acidosis is common among critically ill patients, but current approaches to correct  pH do not improve disease outcomes. During systemic acidosis, cells are either  passively exposed to extracellular acidosis that other cells have generated  (extrinsic acidosis) or they are exposed to acid that they generate and export into  the extracellular space (intrinsic acidosis). Although endothelial repair following  intrinsic acidosis has been studied, the impact of extrinsic acidosis on migration  and angiogenesis is unclear. We hypothesized that extrinsic acidosis inhibits  metabolism and migration but promotes capillary-like network formation in pulmonary  microvascular endothelial cells (PMVECs). Extrinsic acidosis was modeled by  titrating media pH. Two types of intrinsic acidosis were compared, including  increasing cellular metabolism by chemically inhibiting carbonic anhydrases (CAs) IX  and XII (SLC-0111) and with hypoxia. PMVECs maintained baseline intracellular pH for  24 h with both extrinsic and intrinsic acidosis. Whole cell CA IX protein expression  was decreased by extrinsic acidosis but not affected by hypoxia. When extracellular  pH was equally acidic, extrinsic acidosis suppressed glycolysis, whereas intrinsic  acidosis did not. Extrinsic acidosis suppressed migration, but increased Matrigel  network master junction and total segment length. CRISPR-Cas9 CA IX knockout PMVECs  revealed an independent role of CA IX in promoting glycolysis, as loss of CA IX  alone was accompanied by decreased hexokinase I and pyruvate dehydrogenase E1Î±  expression and decreasing migration. 2-deoxy-d-glucose had no effect on migration  but profoundly inhibited network formation and increased N-cadherin expression.  Thus, we report that while extrinsic acidosis suppresses endothelial glycolysis and  migration, it promotes network formation.\\nFAU - Lee, Ji Young\\nAU  - Lee JY\\nAUID- ORCID: 0000-0002-8005-2446\\nAD  - Departments of Physiology and Cell Biology, University of South Alabama, Mobile,  Alabama.\\nAD  - Department of Internal Medicine, University of South Alabama, Mobile, Alabama.\\nAD  - Division of Pulmonary and Critical Care Medicine, University of South Alabama,  Mobile, Alabama.\\nAD  - Center for Lung Biology, University of South Alabama, Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - Onanyan, Mher\\nAU  - Onanyan M\\nAD  - Department of Internal Medicine, University of South Alabama, Mobile, Alabama.\\nAD  - Division of Pulmonary and Critical Care Medicine, University of South Alabama,  Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - Garrison, Ian\\nAU  - Garrison I\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - White, Roderica\\nAU  - White R\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nAD  - Center for Healthy Communities, University of South Alabama, Mobile, Alabama.\\nFAU - Crook, Maura\\nAU  - Crook M\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nAD  - Office of Diversity and Inclusion, University of South Alabama, Mobile, Alabama.\\nFAU - Alexeyev, Mikhail F\\nAU  - Alexeyev MF\\nAD  - Departments of Physiology and Cell Biology, University of South Alabama, Mobile,  Alabama.\\nAD  - Center for Lung Biology, University of South Alabama, Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - Kozhukhar, Natalya\\nAU  - Kozhukhar N\\nAD  - Departments of Physiology and Cell Biology, University of South Alabama, Mobile,  Alabama.\\nAD  - Center for Lung Biology, University of South Alabama, Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - Pastukh, Viktoriya\\nAU  - Pastukh V\\nAD  - Departments of Physiology and Cell Biology, University of South Alabama, Mobile,  Alabama.\\nAD  - Center for Lung Biology, University of South Alabama, Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nFAU - Swenson, Erik R\\nAU  - Swenson ER\\nAD  - Medical Service, VA Puget Sound Health Care System, University of Washington,  Seattle, Washington.\\nFAU - Supuran, Claudiu T\\nAU  - Supuran CT\\nAD  - Neurofarba Department, University of Florence, Florence, Italy.\\nFAU - Stevens, Troy\\nAU  - Stevens T\\nAD  - Departments of Physiology and Cell Biology, University of South Alabama, Mobile,  Alabama.\\nAD  - Department of Internal Medicine, University of South Alabama, Mobile, Alabama.\\nAD  - Center for Lung Biology, University of South Alabama, Mobile, Alabama.\\nAD  - College of Medicine, University of South Alabama, Mobile, Alabama.\\nLA  - eng\\nGR  - 18CDA34080151/AHA_/American Heart Association-American Stroke Association/United  States\\nGR  - P01 HL066299/HL/NHLBI NIH HHS/United States\\nGR  - R37 HL060024/HL/NHLBI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190501\\nTA  - Am J Physiol Lung Cell Mol Physiol\\nJT  - American journal of physiology. Lung cellular and molecular physiology\\nJID - 100901229\\nRN  - 0 (Phenylurea Compounds)\\nRN  - 0 (SLC-0111)\\nRN  - 0 (Sulfonamides)\\nRN  - EC 4.2.1.1 (Carbonic Anhydrases)\\nSB  - IM\\nMH  - Acidosis/drug therapy\\nMH  - Animals\\nMH  - Carbonic Anhydrases/drug effects/metabolism\\nMH  - Endothelial Cells/*drug effects/metabolism\\nMH  - Extracellular Space/metabolism\\nMH  - Glycolysis/*drug effects\\nMH  - Hydrogen-Ion Concentration/drug effects\\nMH  - Hypoxia/drug therapy/metabolism\\nMH  - Lung/drug effects/metabolism\\nMH  - Male\\nMH  - Microvessels/*drug effects\\nMH  - Phenylurea Compounds/*pharmacology\\nMH  - Rats, Sprague-Dawley\\nMH  - Sulfonamides/*pharmacology\\nPMC - PMC6734381\\nOTO - NOTNLM\\nOT  - *2-deoxy--glucose (2DG)\\nOT  - *Matrigel\\nOT  - *N-cadherin\\nOT  - *SLC-0111\\nOT  - *carbonic anhydrase IX (CA IX)\\nCOIS- No conflicts of interest, financial or otherwise, are declared by the authors.\\nEDAT- 2019/05/02 06:00\\nMHDA- 2020/03/28 06:00\\nCRDT- 2019/05/02 06:00\\nPHST- 2019/05/02 06:00 [pubmed]\\nPHST- 2020/03/28 06:00 [medline]\\nPHST- 2019/05/02 06:00 [entrez]\\nAID - L-00544-2018 [pii]\\nAID - 10.1152/ajplung.00544.2018 [doi]\\nPST - ppublish\\nSO  - Am J Physiol Lung Cell Mol Physiol. 2019 Aug 1;317(2):L188-L201. doi:  10.1152/ajplung.00544.2018. Epub 2019 May 1.\\n\\nPMID- 34378391\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210923\\nLR  - 20210923\\nIS  - 1520-5851 (Electronic)\\nIS  - 0013-936X (Linking)\\nVI  - 55\\nIP  - 17\\nDP  - 2021 Sep 7\\nTI  - Enhanced Bioreduction of Radionuclides by Driving Microbial Extracellular Electron  Pumping with an Engineered CRISPR Platform.\\nPG  - 11997-12008\\nLID - 10.1021/acs.est.1c03713 [doi]\\nAB  - Dissimilatory metal-reducing bacteria (DMRB) with extracellular electron transfer  (EET) capability show great potential in bioremediating the subsurface environments  contaminated by uranium through bioreduction and precipitation of hexavalent uranium  [U(VI)]. However, the low EET efficiency of DMRB remains a bottleneck for their  applications. Herein, we develop an engineered CRISPR platform to drive the  extracellular electron pumping of Shewanella oneidensis, a representative DMRB  species widely present in aquatic environments. The CRISPR platform allows for  highly efficient and multiplex genome editing and rapid platform elimination  post-editing in S. oneidensis. Enabled by such a platform, a genomic promoter  engineering strategy (GPS) for genome-widely engineering the EET-encoding gene  network was established. The production of electron conductive Mtr complex,  synthesis of electron shuttle flavin, and generation of NADH as intracellular  electron carrier are globally optimized and promoted, leading to a significantly  enhanced EET ability. Applied to U(VI) bioreduction, the edited strains achieve up  to 3.62-fold higher reduction capacity over the control. Our work endows DMRB with  an enhanced ability to remediate the radionuclides-contaminated environments and  provides a gene editing approach to handle the growing environmental challenges of  radionuclide contaminations.\\nFAU - Fan, Yang-Yang\\nAU  - Fan YY\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Tang, Qiang\\nAU  - Tang Q\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Li, Feng-He\\nAU  - Li FH\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Sun, Hong\\nAU  - Sun H\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei  230026, China.\\nFAU - Min, Di\\nAU  - Min D\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Wu, Jing-Hang\\nAU  - Wu JH\\nAUID- ORCID: 0000-0002-8087-8440\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Li, Yang\\nAU  - Li Y\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei  230026, China.\\nFAU - Li, Wen-Wei\\nAU  - Li WW\\nAUID- ORCID: 0000-0001-9280-0045\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nFAU - Yu, Han-Qing\\nAU  - Yu HQ\\nAUID- ORCID: 0000-0001-5247-6244\\nAD  - CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental  Science and Technology, University of Science and Technology of China, Hefei 230026,  China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210811\\nPL  - United States\\nTA  - Environ Sci Technol\\nJT  - Environmental science & technology\\nJID - 0213155\\nRN  - 4OC371KSTK (Uranium)\\nRN  - Shewanella oneidensis\\nSB  - IM\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Electron Transport\\nMH  - Electrons\\nMH  - *Shewanella/genetics\\nMH  - *Uranium\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas9\\nOT  - *gene editing\\nOT  - *genome-wide\\nOT  - *genomic promoter engineering strategy (GPS)\\nOT  - *uranium bioreduction\\nEDAT- 2021/08/12 06:00\\nMHDA- 2021/09/24 06:00\\nCRDT- 2021/08/11 08:43\\nPHST- 2021/08/12 06:00 [pubmed]\\nPHST- 2021/09/24 06:00 [medline]\\nPHST- 2021/08/11 08:43 [entrez]\\nAID - 10.1021/acs.est.1c03713 [doi]\\nPST - ppublish\\nSO  - Environ Sci Technol. 2021 Sep 7;55(17):11997-12008. doi: 10.1021/acs.est.1c03713.  Epub 2021 Aug 11.\\n\\nPMID- 31343404\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200225\\nLR  - 20201029\\nIS  - 2050-084X (Electronic)\\nIS  - 2050-084X (Linking)\\nVI  - 8\\nDP  - 2019 Jul 25\\nTI  - Identification of an emphysema-associated genetic variant near TGFB2 with regulatory  effects in lung fibroblasts.\\nLID - 10.7554/eLife.42720 [doi]\\nLID - e42720\\nAB  - Murine studies have linked TGF-Î² signaling to emphysema, and human genome-wide  association studies (GWAS) studies of lung function and COPD have identified  associated regions near genes in the TGF-Î² superfamily. However, the functional  regulatory mechanisms at these loci have not been identified. We performed the  largest GWAS of emphysema patterns to date, identifying 10 GWAS loci including an  association peak spanning a 200 kb region downstream from TGFB2. Integrative  analysis of publicly available eQTL, DNaseI, and chromatin conformation data  identified a putative functional variant, rs1690789, that may regulate TGFB2  expression in human fibroblasts. Using chromatin conformation capture, we confirmed  that the region containing rs1690789 contacts the TGFB2 promoter in fibroblasts, and  CRISPR/Cas-9 targeted deletion of a ~ 100 bp region containing rs1690789 resulted in  decreased TGFB2 expression in primary human lung fibroblasts. These data provide  novel mechanistic evidence linking genetic variation affecting the TGF-Î² pathway to  emphysema in humans.\\nCI  - Â© 2019, Parker et al.\\nFAU - Parker, Margaret M\\nAU  - Parker MM\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Hao, Yuan\\nAU  - Hao Y\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Guo, Feng\\nAU  - Guo F\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Pham, Betty\\nAU  - Pham B\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Chase, Robert\\nAU  - Chase R\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Platig, John\\nAU  - Platig J\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Cho, Michael H\\nAU  - Cho MH\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nAD  - Division of Pulmonary and Critical Care Medicine, Brigham and Women\\'s Hospital,  Boston, United States.\\nFAU - Hersh, Craig P\\nAU  - Hersh CP\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nAD  - Division of Pulmonary and Critical Care Medicine, Brigham and Women\\'s Hospital,  Boston, United States.\\nFAU - Thannickal, Victor J\\nAU  - Thannickal VJ\\nAD  - Division of Pulmonary, Allergy and Critical Care, Department of Medicine, School of  Medicine, University of Alabama at Birmingham, Birmingham, United States.\\nFAU - Crapo, James\\nAU  - Crapo J\\nAD  - Division of Pulmonary, Critical Care and Sleep Medicine, National Jewish Health,  Denver, United States.\\nFAU - Washko, George\\nAU  - Washko G\\nAD  - Division of Pulmonary and Critical Care Medicine, Brigham and Women\\'s Hospital,  Boston, United States.\\nFAU - Randell, Scott H\\nAU  - Randell SH\\nAD  - Marsico Lung Institute, The University of North Carolina at Chapel Hill, Chapel  Hill, United States.\\nFAU - Silverman, Edwin K\\nAU  - Silverman EK\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nAD  - Division of Pulmonary and Critical Care Medicine, Brigham and Women\\'s Hospital,  Boston, United States.\\nFAU - San JosÃ© EstÃ©par, RaÃºl\\nAU  - San JosÃ© EstÃ©par R\\nAD  - Applied Chest Imaging Laboratory, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Zhou, Xiaobo\\nAU  - Zhou X\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nFAU - Castaldi, Peter J\\nAU  - Castaldi PJ\\nAUID- ORCID: 0000-0001-9920-4713\\nAD  - Channing Division of Network Medicine, Brigham and Women\\'s Hospital, Boston, United  States.\\nAD  - Division of General Internal Medicine and Primary Care, Brigham and Women\\'s  Hospital, Boston, United States.\\nLA  - eng\\nSI  - dbGaP/phs000765.v3.p2\\nGR  - R01 HL113264/HL/NHLBI NIH HHS/United States\\nGR  - R01 HL130512/HL/NHLBI NIH HHS/United States\\nGR  - U01 HL089897/HL/NHLBI NIH HHS/United States\\nGR  - R01 HL116931/HL/NHLBI NIH HHS/United States\\nGR  - R01 HL126596/HL/NHLBI NIH HHS/United States\\nGR  - U01 HL089856/HL/NHLBI NIH HHS/United States\\nGR  - R01HL089897/HB/NHLBI NIH HHS/United States\\nGR  - R01HL113264/HB/NHLBI NIH HHS/United States\\nGR  - P01HL114501/HB/NHLBI NIH HHS/United States\\nGR  - R01 HL124233/HL/NHLBI NIH HHS/United States\\nGR  - R01 HL125583/HL/NHLBI NIH HHS/United States\\nGR  - P01 HL132825/HL/NHLBI NIH HHS/United States\\nGR  - P01 HL114501/HL/NHLBI NIH HHS/United States\\nGR  - P01105339/HB/NHLBI NIH HHS/United States\\nGR  - R01HL089856/HB/NHLBI NIH HHS/United States\\nGR  - T32 HL007427/HL/NHLBI NIH HHS/United States\\nGR  - P30 DK065988/DK/NIDDK NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20190725\\nTA  - Elife\\nJT  - eLife\\nJID - 101579614\\nRN  - 0 (TGFB2 protein, human)\\nRN  - 0 (Transforming Growth Factor beta2)\\nSB  - IM\\nMH  - Aged\\nMH  - Aged, 80 and over\\nMH  - Emphysema/*genetics\\nMH  - Fibroblasts/*physiology\\nMH  - *Gene Expression Regulation\\nMH  - *Genetic Loci\\nMH  - *Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Humans\\nMH  - Lung/*pathology\\nMH  - Middle Aged\\nMH  - Transforming Growth Factor beta2/*biosynthesis/genetics\\nMH  - United States\\nPMC - PMC6693893\\nOTO - NOTNLM\\nOT  - *COPD\\nOT  - *TGF-beta signaling\\nOT  - *bioinformatics\\nOT  - *emphysema\\nOT  - *genetics\\nOT  - *genome-wide association study\\nOT  - *genomics\\nOT  - *human\\nOT  - *human biology\\nOT  - *medicine\\nOT  - *transcriptomics\\nCOIS- MP, YH, FG, BP, RC, JP, VT, JC, XZ No competing interests declared, MC reports  grants from GSK and personal fees from Genentech, CH reports personal fees from  Mylan, personal fees from AstraZeneca, Concert Pharmaceuticals, 23andMe, grants from  Novartis, and Boehringer-Ingelheim, GW reports grants and other support from  Boehringer Ingelheim, PulmonX, BTG Interventional Medicine, Janssen Pharmaceuticals  and GSK, SR reports receiving personal fees from Amgen, ES received honoraria from  Novartis for Continuing Medical Education Seminars and grant and travel support from  GlaxoSmithKline, RS reports personal fees from Boehringer Ingelheim, Eolo Medical  and Toshiba, PC has received research support and consulting fees from GSK and  Novartis\\nEDAT- 2019/07/26 06:00\\nMHDA- 2020/02/26 06:00\\nCRDT- 2019/07/26 06:00\\nPHST- 2018/10/09 00:00 [received]\\nPHST- 2019/07/25 00:00 [accepted]\\nPHST- 2019/07/26 06:00 [pubmed]\\nPHST- 2020/02/26 06:00 [medline]\\nPHST- 2019/07/26 06:00 [entrez]\\nAID - 42720 [pii]\\nAID - 10.7554/eLife.42720 [doi]\\nPST - epublish\\nSO  - Elife. 2019 Jul 25;8:e42720. doi: 10.7554/eLife.42720.\\n\\nPMID- 35054418\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20220128\\nIS  - 2075-1729 (Print)\\nIS  - 2075-1729 (Electronic)\\nIS  - 2075-1729 (Linking)\\nVI  - 12\\nIP  - 1\\nDP  - 2021 Dec 24\\nTI  - An Interplay between Viruses and Bacteria Associated with the White Sea Sponges  Revealed by Metagenomics.\\nLID - 10.3390/life12010025 [doi]\\nLID - 25\\nAB  - Sponges are remarkable holobionts harboring extremely diverse microbial and viral  communities. However, the interactions between the components within holobionts and  between a holobiont and environment are largely unknown, especially for polar  organisms. To investigate possible interactions within and between sponge-associated  communities, we probed the microbiomes and viromes of cold-water sympatric sponges  Isodictya palmata (n = 2), Halichondria panicea (n = 3), and Halichondria sitiens (n  = 3) by 16S and shotgun metagenomics. We showed that the bacterial and viral  communities associated with these White Sea sponges are species-specific and  different from the surrounding water. Extensive mining of bacterial antiphage  defense systems in the metagenomes revealed a variety of defense mechanisms. The  abundance of defense systems was comparable in the metagenomes of the sponges and  the surrounding water, thus distinguishing the White Sea sponges from those  inhabiting the tropical seas. We developed a network-based approach for the combined  analysis of CRISPR-spacers and protospacers. Using this approach, we showed that the  virus-host interactions within the sponge-associated community are typically more  abundant (three out of four interactions studied) than the inter-community  interactions. Additionally, we detected the occurrence of viral exchanges between  the communities. Our work provides the first insight into the metagenomics of the  three cold-water sponge species from the White Sea and paves the way for a  comprehensive analysis of the interactions between microbial communities and  associated viruses.\\nFAU - Rusanova, Anastasiia\\nAU  - Rusanova A\\nAD  - Institute of Gene Biology, Russian Academy of Sciences, 119334 Moscow, Russia.\\nFAU - Fedorchuk, Victor\\nAU  - Fedorchuk V\\nAD  - The Faculty of Geology, Lomonosov Moscow State University, 119234 Moscow, Russia.\\nFAU - Toshchakov, Stepan\\nAU  - Toshchakov S\\nAD  - Kurchatov Center for Genome Research, National Research Center \"Kurchatov  Institute\", 123182 Moscow, Russia.\\nFAU - Dubiley, Svetlana\\nAU  - Dubiley S\\nAD  - Institute of Gene Biology, Russian Academy of Sciences, 119334 Moscow, Russia.\\nAD  - Skolkovo Institute of Science and Technology, 121205 Moscow, Russia.\\nFAU - Sutormin, Dmitry\\nAU  - Sutormin D\\nAD  - Institute of Gene Biology, Russian Academy of Sciences, 119334 Moscow, Russia.\\nAD  - Skolkovo Institute of Science and Technology, 121205 Moscow, Russia.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20211224\\nTA  - Life (Basel)\\nJT  - Life (Basel, Switzerland)\\nJID - 101580444\\nPMC - PMC8777954\\nOTO - NOTNLM\\nOT  - 16S metagenomics\\nOT  - CRISPR-Cas\\nOT  - bacterial antiphage defense system\\nOT  - metavirome\\nOT  - phage-host interaction\\nOT  - shot-gun metagenomics\\nOT  - sponge-associated communities\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2022/01/22 06:00\\nMHDA- 2022/01/22 06:01\\nCRDT- 2022/01/21 01:08\\nPHST- 2021/11/12 00:00 [received]\\nPHST- 2021/12/20 00:00 [revised]\\nPHST- 2021/12/21 00:00 [accepted]\\nPHST- 2022/01/21 01:08 [entrez]\\nPHST- 2022/01/22 06:00 [pubmed]\\nPHST- 2022/01/22 06:01 [medline]\\nAID - life12010025 [pii]\\nAID - life-12-00025 [pii]\\nAID - 10.3390/life12010025 [doi]\\nPST - epublish\\nSO  - Life (Basel). 2021 Dec 24;12(1):25. doi: 10.3390/life12010025.\\n\\nPMID- 29416123\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190118\\nLR  - 20190401\\nIS  - 1751-7370 (Electronic)\\nIS  - 1751-7362 (Print)\\nIS  - 1751-7362 (Linking)\\nVI  - 12\\nIP  - 4\\nDP  - 2018 Apr\\nTI  - Lysogeny is prevalent and widely distributed in the murine gut microbiota.\\nPG  - 1127-1141\\nLID - 10.1038/s41396-018-0061-9 [doi]\\nAB  - Bacteriophages are central members and potential modulators of the gut microbiome;  however, the ecological and evolutionary relationships of gut bacteria and phages  are poorly understood. Here we investigated the abundance and diversity of lysogenic  bacteria (lysogens) in the bacterial community of C57BL/6J mice by detecting  integrated prophages in genomes reconstructed from the metagenome of commensal  bacteria. For the activities of lysogens and prophages, we compared the prophage  genomes with the metagenome of free phages. The majority of commensal bacteria in  different taxa were identified as lysogens. More lysogens were found among  Firmicutes and Proteobacteria, than among Bacteroidetes and Actinobacteria. The  prophage genomes shared high sequence similarity with the metagenome of free phages,  indicating that most lysogens appeared to be active, and that prophages are  spontaneously induced as active phages; dietary interventions changed the  composition of the induced prophages. By contrast, CRISPR-Cas systems were present  in few commensal bacteria, and were rarely active against gut phages. The structure  of the bacteria-phage infection networks was \"nested-modular\", with modularity  emerging across taxonomic scales, indicating that temperate phage features have  developed over a long phylogenetic timescale. We concluded that phage generalists  contribute to the prevalence of lysogeny in the gut ecosystem.\\nFAU - Kim, Min-Soo\\nAU  - Kim MS\\nAD  - Department of Biology and Department of Life and Nanopharmaceutical Sciences, Kyung  Hee University, 26 Kyungheedae-ro, Dongdaemun-gu, Seoul, 02447, Republic of Korea.\\nFAU - Bae, Jin-Woo\\nAU  - Bae JW\\nAUID- ORCID: 0000-0001-6433-5270\\nAD  - Department of Biology and Department of Life and Nanopharmaceutical Sciences, Kyung  Hee University, 26 Kyungheedae-ro, Dongdaemun-gu, Seoul, 02447, Republic of Korea.  baejw@khu.ac.kr.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180207\\nTA  - ISME J\\nJT  - The ISME journal\\nJID - 101301086\\nSB  - IM\\nMH  - Animals\\nMH  - Bacteria/classification/genetics/isolation & purification\\nMH  - Bacteriophages/genetics\\nMH  - CRISPR-Cas Systems\\nMH  - *Gastrointestinal Microbiome\\nMH  - *Lysogeny\\nMH  - Male\\nMH  - Metagenome\\nMH  - Mice, Inbred C57BL\\nMH  - Phylogeny\\nMH  - Prophages/genetics\\nMH  - Virus Activation\\nPMC - PMC5864201\\nCOIS- The authors declare that they have no conflict of interest.\\nEDAT- 2018/02/09 06:00\\nMHDA- 2019/01/19 06:00\\nCRDT- 2018/02/09 06:00\\nPHST- 2017/08/30 00:00 [received]\\nPHST- 2017/11/23 00:00 [accepted]\\nPHST- 2017/11/14 00:00 [revised]\\nPHST- 2018/02/09 06:00 [pubmed]\\nPHST- 2019/01/19 06:00 [medline]\\nPHST- 2018/02/09 06:00 [entrez]\\nAID - 10.1038/s41396-018-0061-9 [pii]\\nAID - 61 [pii]\\nAID - 10.1038/s41396-018-0061-9 [doi]\\nPST - ppublish\\nSO  - ISME J. 2018 Apr;12(4):1127-1141. doi: 10.1038/s41396-018-0061-9. Epub 2018 Feb 7.\\n\\nPMID- 27455460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170629\\nLR  - 20201209\\nIS  - 1545-9985 (Electronic)\\nIS  - 1545-9985 (Linking)\\nVI  - 23\\nIP  - 9\\nDP  - 2016 Sep\\nTI  - Structural basis of Cas3 inhibition by the bacteriophage protein AcrF3.\\nPG  - 868-70\\nLID - 10.1038/nsmb.3269 [doi]\\nAB  - Bacteriophages express proteins that inactivate the CRISPR-Cas bacterial immune  system. Here we report the crystal structure of the anti-CRISPR protein AcrF3 in  complex with Pseudomonas aeruginosa Cas3 (PaCas3). AcrF3 forms a homodimer that  locks PaCas3 in an ADP-bound form, blocks the entrance of the DNA-binding tunnel in  the helicase domain, and masks the linker region and C-terminal domain of PaCas3,  thereby preventing recruitment by Cascade and inhibiting the type I-F CRISPR-Cas  system.\\nFAU - Wang, Xiaofei\\nAU  - Wang X\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Yao, Deqiang\\nAU  - Yao D\\nAD  - National Center for Protein Science Shanghai, Institute of Biochemistry and Cell  Biology, Shanghai Institutes for Biological Sciences, Shanghai Science Research  Center, Chinese Academy of Sciences, Shanghai, China.\\nFAU - Xu, Jin-Gen\\nAU  - Xu JG\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Li, A-Rong\\nAU  - Li AR\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Xu, Jianpo\\nAU  - Xu J\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Fu, Panhan\\nAU  - Fu P\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Zhou, Yan\\nAU  - Zhou Y\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nFAU - Zhu, Yongqun\\nAU  - Zhu Y\\nAD  - Life Sciences Institute and Innovation Center for Cell Signaling Network, Zhejiang  University, Hangzhou, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20160725\\nPL  - United States\\nTA  - Nat Struct Mol Biol\\nJT  - Nature structural & molecular biology\\nJID - 101186374\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (CRISPR-Associated Proteins)\\nRN  - 0 (Viral Proteins)\\nSB  - IM\\nMH  - Bacterial Proteins/*chemistry\\nMH  - Bacteriophages/*physiology\\nMH  - CRISPR-Associated Proteins/*chemistry\\nMH  - Catalytic Domain\\nMH  - Crystallography, X-Ray\\nMH  - Hydrogen Bonding\\nMH  - Hydrophobic and Hydrophilic Interactions\\nMH  - Models, Molecular\\nMH  - Protein Conformation, alpha-Helical\\nMH  - Protein Interaction Domains and Motifs\\nMH  - Protein Structure, Quaternary\\nMH  - Pseudomonas aeruginosa/*virology\\nMH  - Viral Proteins/*chemistry\\nEDAT- 2016/07/28 06:00\\nMHDA- 2017/07/01 06:00\\nCRDT- 2016/07/26 06:00\\nPHST- 2016/04/25 00:00 [received]\\nPHST- 2016/07/04 00:00 [accepted]\\nPHST- 2016/07/26 06:00 [entrez]\\nPHST- 2016/07/28 06:00 [pubmed]\\nPHST- 2017/07/01 06:00 [medline]\\nAID - nsmb.3269 [pii]\\nAID - 10.1038/nsmb.3269 [doi]\\nPST - ppublish\\nSO  - Nat Struct Mol Biol. 2016 Sep;23(9):868-70. doi: 10.1038/nsmb.3269. Epub 2016 Jul  25.\\n\\nPMID- 33958793\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210907\\nLR  - 20211005\\nIS  - 1552-4469 (Electronic)\\nIS  - 1552-4450 (Linking)\\nVI  - 17\\nIP  - 8\\nDP  - 2021 Aug\\nTI  - Circularly permuted LOV2 as a modular photoswitch for optogenetic engineering.\\nPG  - 915-923\\nLID - 10.1038/s41589-021-00792-9 [doi]\\nAB  - Plant-based photosensors, such as the light-oxygen-voltage sensing domain 2 (LOV2)  from oat phototropin 1, can be modularly wired into cell signaling networks to  remotely control protein activity and physiological processes. However, the  applicability of LOV2 is hampered by the limited choice of available caging surfaces  and its preference to accommodate the effector domains downstream of the C-terminal  JÎ± helix. Here, we engineered a set of LOV2 circular permutants (cpLOV2) with  additional caging capabilities, thereby expanding the repertoire of genetically  encoded photoswitches to accelerate the design of optogenetic devices. We  demonstrate the use of cpLOV2-based optogenetic tools to reversibly gate ion  channels, antagonize CRISPR-Cas9-mediated genome engineering, control protein  subcellular localization, reprogram transcriptional outputs, elicit cell suicide and  generate photoactivatable chimeric antigen receptor T cells for inducible tumor cell  killing. Our approach is widely applicable for engineering other photoreceptors to  meet the growing need of optogenetic tools tailored for biomedical and  biotechnological applications.\\nCI  - Â© 2021. The Author(s), under exclusive licence to Springer Nature America, Inc.\\nFAU - He, Lian\\nAU  - He L\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nFAU - Tan, Peng\\nAU  - Tan P\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nAD  - Klarman Cell Observatory, Broad Institute of MIT and Harvard, Cambridge, MA, USA.\\nFAU - Zhu, Lei\\nAU  - Zhu L\\nAUID- ORCID: 0000-0001-5003-9059\\nAD  - High Magnetic Field Laboratory, CAS Key Laboratory of High Magnetic Field and Ion  Beam Physical Biology, Hefei Institutes of Physical Science, Chinese Academy of  Sciences, Hefei, Anhui, China.\\nFAU - Huang, Kai\\nAU  - Huang K\\nAD  - Department of Biochemistry and Molecular Pharmacology, University of Massachusetts  Medical School, Worcester, Worcester, MA, USA.\\nFAU - Nguyen, Nhung T\\nAU  - Nguyen NT\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nFAU - Wang, Rui\\nAU  - Wang R\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nFAU - Guo, Lei\\nAU  - Guo L\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nFAU - Li, Ling\\nAU  - Li L\\nAD  - High Magnetic Field Laboratory, CAS Key Laboratory of High Magnetic Field and Ion  Beam Physical Biology, Hefei Institutes of Physical Science, Chinese Academy of  Sciences, Hefei, Anhui, China.\\nFAU - Yang, Yuhan\\nAU  - Yang Y\\nAUID- ORCID: 0000-0003-1715-2345\\nAD  - Center for Epigenetics and Disease Prevention, Institute of Biosciences and  Technology, College of Medicine, Texas A&M University, Houston, TX, USA.\\nFAU - Huang, Zixian\\nAU  - Huang Z\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA.\\nFAU - Huang, Yun\\nAU  - Huang Y\\nAUID- ORCID: 0000-0001-5950-9168\\nAD  - Center for Epigenetics and Disease Prevention, Institute of Biosciences and  Technology, College of Medicine, Texas A&M University, Houston, TX, USA.  yun.huang@tamu.edu.\\nFAU - Han, Gang\\nAU  - Han G\\nAUID- ORCID: 0000-0002-2300-5862\\nAD  - Department of Biochemistry and Molecular Pharmacology, University of Massachusetts  Medical School, Worcester, Worcester, MA, USA. Gang.Han@umassmed.edu.\\nFAU - Wang, Junfeng\\nAU  - Wang J\\nAUID- ORCID: 0000-0002-9608-6851\\nAD  - High Magnetic Field Laboratory, CAS Key Laboratory of High Magnetic Field and Ion  Beam Physical Biology, Hefei Institutes of Physical Science, Chinese Academy of  Sciences, Hefei, Anhui, China. junfeng@hmfl.ac.cn.\\nAD  - Institute of Physical Science and Information Technology, Anhui University, Hefei,  Anhui, China. junfeng@hmfl.ac.cn.\\nFAU - Zhou, Yubin\\nAU  - Zhou Y\\nAUID- ORCID: 0000-0001-7962-0517\\nAD  - Center for Translational Cancer Research, Institute of Biosciences and Technology,  Texas A&M University, Houston, TX, USA. yubinzhou@tamu.edu.\\nAD  - Department of Translational Medical Sciences, College of Medicine, Texas A&M  University, Houston, TX, USA. yubinzhou@tamu.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210506\\nPL  - United States\\nTA  - Nat Chem Biol\\nJT  - Nature chemical biology\\nJID - 101231976\\nRN  - 0 (Arabidopsis Proteins)\\nRN  - 0 (DNA-Binding Proteins)\\nRN  - 0 (LOV2 protein, Arabidopsis)\\nSB  - IM\\nMH  - Animals\\nMH  - Arabidopsis Proteins/chemistry/*genetics/metabolism\\nMH  - Cells, Cultured\\nMH  - DNA-Binding Proteins/chemistry/*genetics/metabolism\\nMH  - Female\\nMH  - *Genetic Engineering\\nMH  - Humans\\nMH  - Mice\\nMH  - Mice, Inbred NOD\\nMH  - Mice, Transgenic\\nMH  - *Optogenetics\\nMH  - Photochemical Processes\\nEDAT- 2021/05/08 06:00\\nMHDA- 2021/09/08 06:00\\nCRDT- 2021/05/07 07:06\\nPHST- 2020/04/14 00:00 [received]\\nPHST- 2021/04/02 00:00 [accepted]\\nPHST- 2021/05/08 06:00 [pubmed]\\nPHST- 2021/09/08 06:00 [medline]\\nPHST- 2021/05/07 07:06 [entrez]\\nAID - 10.1038/s41589-021-00792-9 [pii]\\nAID - 10.1038/s41589-021-00792-9 [doi]\\nPST - ppublish\\nSO  - Nat Chem Biol. 2021 Aug;17(8):915-923. doi: 10.1038/s41589-021-00792-9. Epub 2021  May 6.\\n\\nPMID- 30266728\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191004\\nLR  - 20191007\\nIS  - 1098-5336 (Electronic)\\nIS  - 0099-2240 (Print)\\nIS  - 0099-2240 (Linking)\\nVI  - 84\\nIP  - 23\\nDP  - 2018 Dec 1\\nTI  - Transcriptional and Functional Analysis of Bifidobacterium animalis subsp. lactis  Exposure to Tetracycline.\\nLID - 10.1128/AEM.01999-18 [doi]\\nLID - e01999-18\\nAB  - Commercial probiotic bacteria must be tested for acquired antibiotic resistance  elements to avoid potential transfer to pathogens. The European Food Safety  Authority recommends testing resistance using microdilution culture techniques  previously used to establish inhibitory thresholds for the Bifidobacterium genus.  Many Bifidobacterium animalis subsp. lactis strains exhibit increased resistance to  tetracycline, historically attributed to the ribosomal protection gene tet(W).  However, some strains that harbor genetically identical tet(W) genes show various  inhibition levels, suggesting that other genetic elements also contribute to  observed differences. Here, we adapted several molecular assays to confirm the  inhibition of B. animalis subsp. lactis strains Bl-04 and HN019 and employed RNA  sequencing to assess the transcriptional differences related to genomic  polymorphisms. We detected specific stress responses to the antibiotic by  correlating ATP concentration to number of viable genome copies from droplet digital  PCR and found that the bacteria were still metabolically active in high drug  concentrations. Transcriptional analyses revealed that several polymorphic regions,  particularly a novel multidrug efflux transporter, were differentially expressed  between the strains in each experimental condition, likely having phenotypic  effects. We also found that the tet(W) gene was upregulated only during  subinhibitory tetracycline concentrations, while two novel tetracycline resistance  genes were upregulated at high concentrations. Furthermore, many genes involved in  amino acid metabolism and transporter function were upregulated, while genes for  complex carbohydrate utilization, protein metabolism, and clustered regularly  interspaced short palindromic repeat(s) (CRISPR)-Cas systems were downregulated.  These results provide high-throughput means for assessing antibiotic resistances of  two highly related probiotic strains and determine the genetic network that  contributes to the global tetracycline response.IMPORTANCE Bifidobacterium animalis  subsp. lactis is widely used in human food and dietary supplements. Although well  documented to be safe, B. animalis subsp. lactis strains must not contain  transferable antibiotic resistance elements. Many B. animalis subsp. lactis strains  have different resistance measurements despite being genetically similar, and the  reasons for this are not well understood. In the current study, we sought to examine  how genomic differences between two closely related industrial B. animalis subsp.  lactis strains contribute to different resistance levels. This will lead to a better  understanding of resistance, identify future targets for analysis of  transferability, and expand our understanding of tetracycline resistance in  bacteria.\\nCI  - Copyright Â© 2018 Morovic et al.\\nFAU - Morovic, Wesley\\nAU  - Morovic W\\nAUID- ORCID: 0000-0001-8869-3670\\nAD  - Genomics & Microbiome Science, DuPont Nutrition & Health, Madison, Wisconsin, USA  wesley.morovic@dupont.com rbarran@ncsu.edu.\\nFAU - Roos, Paige\\nAU  - Roos P\\nAD  - Genomics Laboratory, DuPont Pioneer, Johnston, Iowa, USA.\\nFAU - Zabel, Bryan\\nAU  - Zabel B\\nAD  - Genomics & Microbiome Science, DuPont Nutrition & Health, Madison, Wisconsin, USA.\\nFAU - Hidalgo-Cantabrana, Claudio\\nAU  - Hidalgo-Cantabrana C\\nAD  - Department of Food, Processing and Nutrition Sciences, North Carolina State  University, Raleigh, North Carolina, USA.\\nFAU - Kiefer, Anthony\\nAU  - Kiefer A\\nAD  - Probiotic Development, DuPont Nutrition & Health, Madison, Wisconsin, USA.\\nFAU - Barrangou, Rodolphe\\nAU  - Barrangou R\\nAUID- ORCID: 0000-0002-0648-3504\\nAD  - Department of Food, Processing and Nutrition Sciences, North Carolina State  University, Raleigh, North Carolina, USA wesley.morovic@dupont.com rbarran@ncsu.edu.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20181115\\nTA  - Appl Environ Microbiol\\nJT  - Applied and environmental microbiology\\nJID - 7605801\\nRN  - 0 (Anti-Bacterial Agents)\\nRN  - 0 (Bacterial Proteins)\\nRN  - F8VB5M810T (Tetracycline)\\nSB  - IM\\nMH  - Anti-Bacterial Agents/*pharmacology\\nMH  - Bacterial Proteins/genetics/metabolism\\nMH  - Bifidobacterium animalis/*drug effects/*genetics/metabolism\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/drug effects\\nMH  - Gene Expression Regulation, Bacterial/drug effects\\nMH  - Genome, Bacterial/drug effects\\nMH  - Tetracycline/*pharmacology\\nMH  - Tetracycline Resistance\\nMH  - Transcription, Genetic/*drug effects\\nPMC - PMC6238047\\nOTO - NOTNLM\\nOT  - *ATP luminescence\\nOT  - *Bifidobacterium animalis subsp. lactis\\nOT  - *RNA sequencing\\nOT  - *antibiotic resistance\\nOT  - *droplet digital PCR\\nOT  - *genomics\\nOT  - *tetracycline\\nEDAT- 2018/09/30 06:00\\nMHDA- 2019/10/08 06:00\\nCRDT- 2018/09/30 06:00\\nPHST- 2018/08/14 00:00 [received]\\nPHST- 2018/09/25 00:00 [accepted]\\nPHST- 2018/09/30 06:00 [pubmed]\\nPHST- 2019/10/08 06:00 [medline]\\nPHST- 2018/09/30 06:00 [entrez]\\nAID - AEM.01999-18 [pii]\\nAID - 01999-18 [pii]\\nAID - 10.1128/AEM.01999-18 [doi]\\nPST - epublish\\nSO  - Appl Environ Microbiol. 2018 Nov 15;84(23):e01999-18. doi: 10.1128/AEM.01999-18.  Print 2018 Dec 1.\\n\\nPMID- 27694915\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180925\\nLR  - 20181113\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 7\\nDP  - 2016 Oct 3\\nTI  - Integration and exchange of split dCas9 domains for transcriptional controls in  mammalian cells.\\nPG  - 13056\\nLID - 10.1038/ncomms13056 [doi]\\nLID - 13056\\nAB  - Programmable and precise regulation of dCas9 functions in response to multiple  molecular signals by using synthetic gene circuits will expand the application of  the CRISPR-Cas technology. However, the application of CRISPR-Cas therapeutic  circuits is still challenging due to the restrictive cargo size of existing viral  delivery vehicles. Here, we construct logic AND circuits by integrating multiple  split dCas9 domains, which is useful to reduce the size of synthetic circuits. In  addition, we engineer sensory switches by exchanging split dCas9 domains, allowing  differential regulations on one gene, or activating two different genes in response  to cell-type specific microRNAs. Therefore, we provide a valuable split-dCas9  toolkit to engineer complex transcription controls, which may inspire new biomedical  applications.\\nFAU - Ma, Dacheng\\nAU  - Ma D\\nAD  - MOE Key Laboratory of Bioinformatics and Bioinformatics Division, Center for  Synthetic and System Biology, Department of Automation, Tsinghua National Lab for  Information Science and Technology, Tsinghua University, Beijing 100084, China.\\nFAU - Peng, Shuguang\\nAU  - Peng S\\nAD  - MOE Key Laboratory of Bioinformatics and Bioinformatics Division, Center for  Synthetic and System Biology, Department of Automation, Tsinghua National Lab for  Information Science and Technology, Tsinghua University, Beijing 100084, China.\\nFAU - Xie, Zhen\\nAU  - Xie Z\\nAD  - MOE Key Laboratory of Bioinformatics and Bioinformatics Division, Center for  Synthetic and System Biology, Department of Automation, Tsinghua National Lab for  Information Science and Technology, Tsinghua University, Beijing 100084, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20161003\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (RNA, Small Interfering)\\nRN  - 0 (Transcription Factors)\\nSB  - IM\\nMH  - *CRISPR-Cas Systems\\nMH  - *Gene Expression Regulation\\nMH  - Gene Regulatory Networks\\nMH  - HEK293 Cells\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Microscopy, Fluorescence\\nMH  - Nucleotide Motifs\\nMH  - Protein Domains\\nMH  - RNA, Small Interfering/metabolism\\nMH  - Streptococcus pyogenes/metabolism\\nMH  - Transcription Factors/*metabolism\\nMH  - Transcription, Genetic\\nMH  - Transcriptional Activation\\nPMC - PMC5063958\\nCOIS- Z.X., D.M. and S.P. have filed a patent application to State Intellectual Property  Office of China based on the findings in this work (patent number: 201610341363.0).  The remaining authors declare no competing financial interests.\\nEDAT- 2016/10/04 06:00\\nMHDA- 2018/09/27 06:00\\nCRDT- 2016/10/04 06:00\\nPHST- 2016/02/09 00:00 [received]\\nPHST- 2016/08/30 00:00 [accepted]\\nPHST- 2016/10/04 06:00 [entrez]\\nPHST- 2016/10/04 06:00 [pubmed]\\nPHST- 2018/09/27 06:00 [medline]\\nAID - ncomms13056 [pii]\\nAID - 10.1038/ncomms13056 [doi]\\nPST - epublish\\nSO  - Nat Commun. 2016 Oct 3;7:13056. doi: 10.1038/ncomms13056.\\n\\nPMID- 35119931\\nOWN - NLM\\nSTAT- In-Data-Review\\nLR  - 20220207\\nIS  - 2375-2548 (Electronic)\\nIS  - 2375-2548 (Linking)\\nVI  - 8\\nIP  - 5\\nDP  - 2022 Feb 4\\nTI  - Loss of TSC1/TSC2 sensitizes immune checkpoint blockade in non-small cell lung  cancer.\\nPG  - eabi9533\\nLID - 10.1126/sciadv.abi9533 [doi]\\nLID - eabi9533\\nAB  - Tuberous sclerosis complex subunit 1 (TSC1) and 2 (TSC2) are frequently mutated in  non-small cell lung cancer (NSCLC), however, their effects on antitumor immunity  remained unexplored. A CRISPR screening in murine Kras(G12D)/Trp53(-/-) (KP) model  identified Tsc1 and Tsc2 as potent regulators of programmed cell death ligand 1  (Pd-l1) expression in vitro and sensitivity to anti-programmed cell death receptor 1  (PD-1) treatment in vivo. TSC1 or TSC2 knockout (KO) promoted the transcriptional  and membrane expression of PD-L1 in cell lines. TSC2-deficient tumors manifested an  inflamed microenvironment in patient samples and The Cancer Genome Atlas dataset. In  syngeneic murine models, KP-Tsc2-KO tumors showed notable response to anti-PD-1  antibody treatment, but Tsc2-wild-type tumors did not. Patients with  TSC1/TSC2-mutant NSCLC receiving immune checkpoint blockade (ICB) had increased  durable clinical benefit and survival. Collectively, TSC1/TSC2 loss defines a  distinct subtype of NSCLC characterized as inflamed tumor microenvironment and  superior sensitivity to ICB.\\nFAU - Huang, Qingyuan\\nAU  - Huang Q\\nAUID- ORCID: 0000-0003-4389-6188\\nAD  - Department of Thoracic Surgery and State Key Laboratory of Genetic Engineering,  Fudan University Shanghai Cancer Center, Shanghai, China.\\nAD  - Institute of Thoracic Oncology, Fudan University, Shanghai, China.\\nFAU - Li, Fei\\nAU  - Li F\\nAD  - Department of Pathology, School of Basic Medical Sciences, Fudan University,  Shanghai, China.\\nFAU - Hu, Hai\\nAU  - Hu H\\nAUID- ORCID: 0000-0002-6985-8959\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Fang, Zhaoyuan\\nAU  - Fang Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Gao, Zhendong\\nAU  - Gao Z\\nAD  - Department of Thoracic Surgery and State Key Laboratory of Genetic Engineering,  Fudan University Shanghai Cancer Center, Shanghai, China.\\nAD  - Institute of Thoracic Oncology, Fudan University, Shanghai, China.\\nFAU - Xia, Guozhan\\nAU  - Xia G\\nAUID- ORCID: 0000-0003-4632-0512\\nAD  - Department of Thoracic Surgery and State Key Laboratory of Genetic Engineering,  Fudan University Shanghai Cancer Center, Shanghai, China.\\nAD  - Institute of Thoracic Oncology, Fudan University, Shanghai, China.\\nFAU - Ng, Wai-Lung\\nAU  - Ng WL\\nAUID- ORCID: 0000-0003-2892-6318\\nAD  - School of Pharmacy, Faculty of Medicine, The Chinese University of Hong Kong, Sha  Tin, Hong Kong SAR, China.\\nFAU - Khodadadi-Jamayran, Alireza\\nAU  - Khodadadi-Jamayran A\\nAUID- ORCID: 0000-0003-2495-7504\\nAD  - Applied Bioinformatics Laboratories and Genome Technology Center, Division of  Advanced Research Technologies, New York University Langone Medical Center, New  York, NY, USA.\\nFAU - Chen, Ting\\nAU  - Chen T\\nAUID- ORCID: 0000-0003-1725-7956\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Deng, Jiehui\\nAU  - Deng J\\nAUID- ORCID: 0000-0001-7986-8643\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Zhang, Hua\\nAU  - Zhang H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Almonte, Christina\\nAU  - Almonte C\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Labbe, Kristen\\nAU  - Labbe K\\nAUID- ORCID: 0000-0001-6731-5849\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Han, Han\\nAU  - Han H\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Geng, Ke\\nAU  - Geng K\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Tang, Sittinon\\nAU  - Tang S\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nFAU - Freeman, Gordon J\\nAU  - Freeman GJ\\nAUID- ORCID: 0000-0002-7210-5616\\nAD  - Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA, USA.\\nAD  - Department of Medicine, Brigham and Women\\'s Hospital, Harvard Medical School,  Boston, MA, USA.\\nFAU - Li, Yuan\\nAU  - Li Y\\nAUID- ORCID: 0000-0001-5651-819X\\nAD  - Department of Pathology, Fudan University Shanghai Cancer Center, Shanghai, China.\\nFAU - Chen, Haiquan\\nAU  - Chen H\\nAUID- ORCID: 0000-0001-7846-257X\\nAD  - Department of Thoracic Surgery and State Key Laboratory of Genetic Engineering,  Fudan University Shanghai Cancer Center, Shanghai, China.\\nAD  - Institute of Thoracic Oncology, Fudan University, Shanghai, China.\\nFAU - Wong, Kwok-Kin\\nAU  - Wong KK\\nAUID- ORCID: 0000-0001-6323-235X\\nAD  - Laura and Isaac Perlmutter Cancer Center, New York University Grossman School of  Medicine, NYU Langone Health, New York, NY, USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20220204\\nTA  - Sci Adv\\nJT  - Science advances\\nJID - 101653440\\nSB  - IM\\nPMC - PMC8816329\\nEDAT- 2022/02/05 06:00\\nMHDA- 2022/02/05 06:00\\nCRDT- 2022/02/04 17:10\\nPHST- 2022/02/04 17:10 [entrez]\\nPHST- 2022/02/05 06:00 [pubmed]\\nPHST- 2022/02/05 06:00 [medline]\\nAID - abi9533 [pii]\\nAID - 10.1126/sciadv.abi9533 [doi]\\nPST - ppublish\\nSO  - Sci Adv. 2022 Feb 4;8(5):eabi9533. doi: 10.1126/sciadv.abi9533. Epub 2022 Feb 4.\\n\\nPMID- 30346516\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190925\\nLR  - 20190925\\nIS  - 1574-6968 (Electronic)\\nIS  - 0378-1097 (Linking)\\nVI  - 365\\nIP  - 23\\nDP  - 2018 Dec 1\\nTI  - Genome analysis of lactic acid bacterial strains selected as potential starters for  traditional Slovakian bryndza cheese.\\nLID - 10.1093/femsle/fny257 [doi]\\nAB  - Genomes of 21 strains of lactic acid bacteria isolated from Slovakian traditional  cheeses were sequenced on an Illumina MiSeq platform. Subsequently, they were  analysed regarding taxonomic classification, presence of genes encoding defence  systems, antibiotic resistance and production of biogenic amines. Thirteen strains  were found to carry genes encoding at least one bacteriocin, 18 carried genes  encoding at least one restriction-modification system, all strains carried 1-6  prophages and 9 strains had CRISPR-Cas systems. CRISPR-Cas type II-A was the most  common, containing 0-24 spacers. Only 10% spacers were found to be homological to  known bacteriophage or plasmid sequences in databases. Two Enterococcus faecium  strains and a Lactococcus lactis strain carried antibiotic resistance genes. Genes  encoding for ornithine decarboxylase were detected in four strains and genes  encoding for agmatine deiminase were detected in four strains. Lactobacillus  paraplantarum 251 L appeared to be the most interesting strain, as it contained  genes encoding for two bacteriocins, a restriction-modification system, two  CRISPR-Cas systems, four prophages and no genes connected with antibiotic resistance  or production of biogenic amines.\\nFAU - MarkuskovÃ¡, Barbora\\nAU  - MarkuskovÃ¡ B\\nAD  - Department of Molecular Biology, Faculty of Natural Sciences, Comenius University,  Ilkovicova 6, 84215 Bratislava, Slovakia.\\nFAU - LichvarikovÃ¡, Aneta\\nAU  - LichvarikovÃ¡ A\\nAD  - Department of Molecular Biology, Faculty of Natural Sciences, Comenius University,  Ilkovicova 6, 84215 Bratislava, Slovakia.\\nFAU - Szemes, TomÃ¡Å¡\\nAU  - Szemes T\\nAD  - Department of Molecular Biology, Faculty of Natural Sciences, Comenius University,  IlkoviÄova 6, 84215 Bratislava, Slovakia.\\nAD  - Comenius University Science Park, IlkoviÄova 8, 84104 Bratislava, Slovakia.\\nFAU - KorenovÃ¡, Janka\\nAU  - KorenovÃ¡ J\\nAD  - Food Research Institute, National Agricultural and Food Centre, PriemyselnÃ¡ 4, 82475  Bratislava, Slovakia.\\nFAU - Kuchta, TomÃ¡Å¡\\nAU  - Kuchta T\\nAD  - Food Research Institute, National Agricultural and Food Centre, PriemyselnÃ¡ 4, 82475  Bratislava, Slovakia.\\nFAU - DrahovskÃ¡, Hana\\nAU  - DrahovskÃ¡ H\\nAD  - Department of Molecular Biology, Faculty of Natural Sciences, Comenius University,  Ilkovicova 6, 84215 Bratislava, Slovakia.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - England\\nTA  - FEMS Microbiol Lett\\nJT  - FEMS microbiology letters\\nJID - 7705721\\nRN  - 0 (Bacteriocins)\\nRN  - 0 (DNA Restriction-Modification Enzymes)\\nRN  - EC 3.- (Hydrolases)\\nRN  - EC 3.5.3.12 (agmatine deiminase)\\nRN  - EC 4.1.1.17 (Ornithine Decarboxylase)\\nSB  - IM\\nMH  - Bacteriocins/genetics\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Cheese/*microbiology\\nMH  - Computational Biology\\nMH  - DNA Restriction-Modification Enzymes/genetics\\nMH  - Drug Resistance, Bacterial\\nMH  - Genes, Bacterial\\nMH  - *Genome, Bacterial\\nMH  - High-Throughput Nucleotide Sequencing\\nMH  - Hydrolases/genetics\\nMH  - Lactobacillales/classification/*genetics/isolation & purification\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Molecular Sequence Annotation\\nMH  - Ornithine Decarboxylase/genetics\\nMH  - Prophages/genetics\\nMH  - Slovakia\\nEDAT- 2018/10/23 06:00\\nMHDA- 2019/09/26 06:00\\nCRDT- 2018/10/23 06:00\\nPHST- 2018/07/24 00:00 [received]\\nPHST- 2018/10/19 00:00 [accepted]\\nPHST- 2018/10/23 06:00 [pubmed]\\nPHST- 2019/09/26 06:00 [medline]\\nPHST- 2018/10/23 06:00 [entrez]\\nAID - 5140145 [pii]\\nAID - 10.1093/femsle/fny257 [doi]\\nPST - ppublish\\nSO  - FEMS Microbiol Lett. 2018 Dec 1;365(23). doi: 10.1093/femsle/fny257.\\n\\nPMID- 30293782\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190415\\nLR  - 20190415\\nIS  - 1097-4164 (Electronic)\\nIS  - 1097-2765 (Linking)\\nVI  - 72\\nIP  - 2\\nDP  - 2018 Oct 18\\nTI  - Genetic Modulation of RNA Splicing with a CRISPR-Guided Cytidine Deaminase.\\nPG  - 380-394.e7\\nLID - S1097-2765(18)30741-X [pii]\\nLID - 10.1016/j.molcel.2018.09.002 [doi]\\nAB  - RNA splicing is a critical mechanism by which to modify transcriptome, and its  dysregulation is the underlying cause of many human diseases. It remains  challenging, however, to genetically modulate a splicing event in its native  context. Here, we demonstrate that a CRISPR-guided cytidine deaminase (i.e.,  targeted-AID mediated mutagenesis [TAM]) can efficiently modulate various forms of  mRNA splicing. By converting invariant guanines to adenines at either 5\\' or 3\\'  splice sites (SS), TAM induces exon skipping, activation of alternative SS,  switching between mutually exclusive exons, or targeted intron retention.  Conversely, TAM promotes downstream exon inclusion by mutating cytidines into  thymines at the polypyrimidine tract. Applying this approach, we genetically  restored the open reading frame and dystrophin function of a mutant DMD gene in  patient-derived induced pluripotent stem cells (iPSCs). Thus, the CRISPR-guided  cytidine deaminase provides a versatile genetic platform to modulate RNA splicing  and to correct mutations associated with aberrant splicing in human diseases.\\nCI  - Copyright Â© 2018 Elsevier Inc. All rights reserved.\\nFAU - Yuan, Juanjuan\\nAU  - Yuan J\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Ma, Yunqing\\nAU  - Ma Y\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Huang, Tao\\nAU  - Huang T\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Chen, Yanhao\\nAU  - Chen Y\\nAD  - CAS Key Laboratory of Nutrition, Metabolism, and Food Safety, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Peng, Yuanzheng\\nAU  - Peng Y\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Li, Bing\\nAU  - Li B\\nAD  - Department of Cellular and Molecular Medicine, University of California at San  Diego, La Jolla, CA 92037, USA.\\nFAU - Li, Jia\\nAU  - Li J\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Zhang, Yuchen\\nAU  - Zhang Y\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Song, Bing\\nAU  - Song B\\nAD  - Key Laboratory for Major Obstetric Diseases of Guangdong Province, Key Laboratory of  Reproduction and Genetics of Guangdong Higher Education Institutes, The Third  Affiliated Hospital of Guangzhou Medical University, Guangzhou, Guangdong 510510,  China.\\nFAU - Sun, Xiaofang\\nAU  - Sun X\\nAD  - Key Laboratory for Major Obstetric Diseases of Guangdong Province, Key Laboratory of  Reproduction and Genetics of Guangdong Higher Education Institutes, The Third  Affiliated Hospital of Guangzhou Medical University, Guangzhou, Guangdong 510510,  China.\\nFAU - Ding, Qiurong\\nAU  - Ding Q\\nAD  - CAS Key Laboratory of Nutrition, Metabolism, and Food Safety, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Song, Yan\\nAU  - Song Y\\nAD  - Department of Cellular and Molecular Medicine, University of California at San  Diego, La Jolla, CA 92037, USA. Electronic address: yasong@ucsd.edu.\\nFAU - Chang, Xing\\nAU  - Chang X\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, China.  Electronic address: changxing@sibs.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20181004\\nPL  - United States\\nTA  - Mol Cell\\nJT  - Molecular cell\\nJID - 9802571\\nRN  - 0 (Dystrophin)\\nRN  - 0 (RNA Splice Sites)\\nRN  - EC 3.5.4.5 (Cytidine Deaminase)\\nSB  - IM\\nMH  - Amino Acid Sequence\\nMH  - Animals\\nMH  - Cell Line\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Cytidine Deaminase/*genetics\\nMH  - Dystrophin/genetics\\nMH  - Exons/genetics\\nMH  - Gene Regulatory Networks\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Introns/genetics\\nMH  - Mice\\nMH  - Open Reading Frames/genetics\\nMH  - RNA Splice Sites/genetics\\nMH  - RNA Splicing/*genetics\\nOTO - NOTNLM\\nOT  - *AID\\nOT  - *DMD\\nOT  - *base editing\\nOT  - *exon skipping\\nOT  - *gene therapy\\nOT  - *polypyrimidine tract\\nOT  - *splice site\\nOT  - *splicing\\nOT  - *targeted AID-mediated mutagenesis\\nEDAT- 2018/10/09 06:00\\nMHDA- 2019/04/16 06:00\\nCRDT- 2018/10/09 06:00\\nPHST- 2018/05/03 00:00 [received]\\nPHST- 2018/07/26 00:00 [revised]\\nPHST- 2018/08/31 00:00 [accepted]\\nPHST- 2018/10/09 06:00 [pubmed]\\nPHST- 2019/04/16 06:00 [medline]\\nPHST- 2018/10/09 06:00 [entrez]\\nAID - S1097-2765(18)30741-X [pii]\\nAID - 10.1016/j.molcel.2018.09.002 [doi]\\nPST - ppublish\\nSO  - Mol Cell. 2018 Oct 18;72(2):380-394.e7. doi: 10.1016/j.molcel.2018.09.002. Epub 2018  Oct 4.\\n\\nPMID- 34197615\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210809\\nLR  - 20210809\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 49\\nIP  - 13\\nDP  - 2021 Jul 21\\nTI  - Recombination machinery engineering facilitates metabolic engineering of the  industrial yeast PichiaÂ\\xa0pastoris.\\nPG  - 7791-7805\\nLID - 10.1093/nar/gkab535 [doi]\\nAB  - The industrial yeast Pichia pastoris has been harnessed extensively for production  of proteins, and it is attracting attention as a chassis cell factory for production  of chemicals. However, the lack of synthetic biology tools makes it challenging in  rewiring P. pastoris metabolism. We here extensively engineered the recombination  machinery by establishing a CRISPR-Cas9 based genome editing platform, which  improved the homologous recombination (HR) efficiency by more than 54 times, in  particular, enhanced the simultaneously assembly of multiple fragments by 13.5  times. We also found that the key HR-relating gene RAD52 of P. pastoris was largely  repressed in compared to that of Saccharomyces cerevisiae. This gene editing system  enabled efficient seamless gene disruption, genome integration and multiple gene  assembly with positive rates of 68-90%. With this efficient genome editing platform,  we characterized 46 potential genome integration sites and 18 promoters at different  growth conditions. This library of neutral sites and promoters enabled two-factorial  regulation of gene expression and metabolic pathways and resulted in a 30-fold range  of fatty alcohol production (12.6-380 mg/l). The expanding genetic toolbox will  facilitate extensive rewiring of P. pastoris for chemical production, and also shed  light on engineering of other non-conventional yeasts.\\nCI  - Â© The Author(s) 2021. Published by Oxford University Press on behalf of Nucleic  Acids Research.\\nFAU - Cai, Peng\\nAU  - Cai P\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - School of Bioengineering, Dalian University of Technology, Dalian 116024,Â\\xa0China.\\nFAU - Duan, Xingpeng\\nAU  - Duan X\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - Dalian Key Laboratory of Energy Biotechnology, Dalian Institute of Chemical Physics,  Chinese Academy of Sciences, Dalian 116023, China.\\nFAU - Wu, Xiaoyan\\nAU  - Wu X\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - Dalian Key Laboratory of Energy Biotechnology, Dalian Institute of Chemical Physics,  Chinese Academy of Sciences, Dalian 116023, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 100049, China.\\nFAU - Gao, Linhui\\nAU  - Gao L\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - Dalian Key Laboratory of Energy Biotechnology, Dalian Institute of Chemical Physics,  Chinese Academy of Sciences, Dalian 116023, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 100049, China.\\nFAU - Ye, Min\\nAU  - Ye M\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - Dalian Key Laboratory of Energy Biotechnology, Dalian Institute of Chemical Physics,  Chinese Academy of Sciences, Dalian 116023, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 100049, China.\\nFAU - Zhou, Yongjin J\\nAU  - Zhou YJ\\nAD  - Division of Biotechnology, Dalian Institute of Chemical Physics, Chinese Academy of  Sciences, Dalian 116023, China.\\nAD  - CAS Key Laboratory of Separation Science for Analytical Chemistry, Dalian Institute  of Chemical Physics, Chinese Academy of Sciences, Dalian 116023, China.\\nAD  - Dalian Key Laboratory of Energy Biotechnology, Dalian Institute of Chemical Physics,  Chinese Academy of Sciences, Dalian 116023, China.\\nAD  - Laboratory of Synthetic Biology for Biocataysis, Dalian Institute of Chemical  Physics, Chinese Academy of Sciences, Dalian 116023, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nRN  - 0 (Fatty Alcohols)\\nRN  - Komagataella pastoris\\nSB  - IM\\nMH  - CRISPR-Cas Systems\\nMH  - Fatty Alcohols/metabolism\\nMH  - Gene Editing\\nMH  - Gene Expression\\nMH  - *Homologous Recombination\\nMH  - *Metabolic Engineering\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Promoter Regions, Genetic\\nMH  - Saccharomycetales/*genetics/metabolism\\nPMC - PMC8287956\\nEDAT- 2021/07/02 06:00\\nMHDA- 2021/08/10 06:00\\nCRDT- 2021/07/01 17:27\\nPHST- 2021/06/13 00:00 [accepted]\\nPHST- 2021/04/28 00:00 [revised]\\nPHST- 2021/01/01 00:00 [received]\\nPHST- 2021/07/02 06:00 [pubmed]\\nPHST- 2021/08/10 06:00 [medline]\\nPHST- 2021/07/01 17:27 [entrez]\\nAID - 6312753 [pii]\\nAID - gkab535 [pii]\\nAID - 10.1093/nar/gkab535 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2021 Jul 21;49(13):7791-7805. doi: 10.1093/nar/gkab535.\\n\\nPMID- 30329048\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190621\\nLR  - 20200225\\nIS  - 1460-2083 (Electronic)\\nIS  - 0964-6906 (Print)\\nIS  - 0964-6906 (Linking)\\nVI  - 28\\nIP  - 4\\nDP  - 2019 Feb 15\\nTI  - CRISPR/Cas9-mediated disruption of SHANK3 in monkey leads to drug-treatable  autism-like symptoms.\\nPG  - 561-571\\nLID - 10.1093/hmg/ddy367 [doi]\\nAB  - Monogenic mutations in the SHANK3 gene, which encodes a postsynaptic scaffold  protein, play a causative role in autism spectrum disorder (ASD). Although a number  of mouse models with Shank3 mutations have been valuable for investigating the  pathogenesis of ASD, species-dependent differences in behaviors and brain structures  post considerable challenges to use small animals to model ASD and to translate  experimental therapeutics to the clinic. We have used clustered regularly  interspersed short palindromic repeat/CRISPR-associated nuclease 9 to generate a  cynomolgus monkey model by disrupting SHANK3 at exons 6 and 12. Analysis of the live  mutant monkey revealed the core behavioral abnormalities of ASD, including impaired  social interaction and repetitive behaviors, and reduced brain network activities  detected by positron-emission computed tomography (PET). Importantly, these abnormal  behaviors and brain activities were alleviated by the antidepressant fluoxetine  treatment. Our findings provide the first demonstration that the genetically  modified non-human primate can be used for translational research of therapeutics  for ASD.\\nFAU - Tu, Zhuchi\\nAU  - Tu Z\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Zhao, Hui\\nAU  - Zhao H\\nAD  - State Key Laboratory of Molecular Developmental Biology, CAS Center for Excellence  in Brain Science and Intelligence Technology, Institute of Genetics and  Developmental Biology, Chinese Academy of Sciences, Beijing, China.\\nAD  - Key Laboratory of Regenerative Biology, South China Institute for Stem Cell, Biology  and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health, Chinese  Academy of Sciences, Guangzhou, China.\\nFAU - Li, Bang\\nAU  - Li B\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Yan, Sen\\nAU  - Yan S\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Wang, Lu\\nAU  - Wang L\\nAD  - Department of Nuclear Medicine and PET/CT-MRI Center, the First Affiliated Hospital  of Jinan University & Institute of Molecular and Functional Imaging, Jinan  University, Guangzhou China.\\nFAU - Tang, Yongjin\\nAU  - Tang Y\\nAD  - Department of Nuclear Medicine and PET/CT-MRI Center, the First Affiliated Hospital  of Jinan University & Institute of Molecular and Functional Imaging, Jinan  University, Guangzhou China.\\nFAU - Li, Zhujun\\nAU  - Li Z\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Bai, Dazhang\\nAU  - Bai D\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Li, Caijuan\\nAU  - Li C\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Lin, Yingqi\\nAU  - Lin Y\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Li, Yuefeng\\nAU  - Li Y\\nAD  - Guangdong Landau Biotechnology Co. Ltd., Guangzhou, China.\\nFAU - Liu, Jianrong\\nAU  - Liu J\\nAD  - Yuanxi Biotech Inc., Guangzhou China.\\nFAU - Xu, Hao\\nAU  - Xu H\\nAD  - Department of Nuclear Medicine and PET/CT-MRI Center, the First Affiliated Hospital  of Jinan University & Institute of Molecular and Functional Imaging, Jinan  University, Guangzhou China.\\nFAU - Guo, Xiangyu\\nAU  - Guo X\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nFAU - Jiang, Yong-Hui\\nAU  - Jiang YH\\nAD  - Department of Pediatrics and Department of Neurobiology, Duke University, Durham,  NC, USA.\\nFAU - Zhang, Yong Q\\nAU  - Zhang YQ\\nAD  - State Key Laboratory of Molecular Developmental Biology, CAS Center for Excellence  in Brain Science and Intelligence Technology, Institute of Genetics and  Developmental Biology, Chinese Academy of Sciences, Beijing, China.\\nFAU - Li, Xiao-Jiang\\nAU  - Li XJ\\nAD  - Guangdong-Hongkong-Macau Institute of CNS Regeneration, Ministry of Education CNS  Regeneration Collaborative Joint Laboratory, Jinan University, Guangzhou, China.\\nAD  - Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA.\\nLA  - eng\\nGR  - R01 NS036232/NS/NINDS NIH HHS/United States\\nGR  - R01 NS101701/NS/NINDS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Hum Mol Genet\\nJT  - Human molecular genetics\\nJID - 9208958\\nRN  - 0 (Nerve Tissue Proteins)\\nRN  - 0 (SHANK3 protein, human)\\nRN  - 01K63SUP8D (Fluoxetine)\\nSB  - IM\\nMH  - Animals\\nMH  - Autism Spectrum Disorder/diagnostic imaging/*drug therapy/genetics/pathology\\nMH  - Behavior, Animal/drug effects\\nMH  - Brain/diagnostic imaging/*drug effects/pathology\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Disease Models, Animal\\nMH  - Exons\\nMH  - Fluoxetine/*administration & dosage\\nMH  - Humans\\nMH  - Interpersonal Relations\\nMH  - Macaca fascicularis/genetics\\nMH  - Mice\\nMH  - Mutation\\nMH  - Nerve Tissue Proteins/*genetics\\nPMC - PMC6489410\\nEDAT- 2018/10/18 06:00\\nMHDA- 2019/06/22 06:00\\nCRDT- 2018/10/18 06:00\\nPHST- 2018/08/31 00:00 [received]\\nPHST- 2018/10/12 00:00 [accepted]\\nPHST- 2018/10/18 06:00 [pubmed]\\nPHST- 2019/06/22 06:00 [medline]\\nPHST- 2018/10/18 06:00 [entrez]\\nAID - 5133462 [pii]\\nAID - ddy367 [pii]\\nAID - 10.1093/hmg/ddy367 [doi]\\nPST - ppublish\\nSO  - Hum Mol Genet. 2019 Feb 15;28(4):561-571. doi: 10.1093/hmg/ddy367.\\n\\nPMID- 31279547\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191120\\nLR  - 20191120\\nIS  - 1525-3198 (Electronic)\\nIS  - 0022-0302 (Linking)\\nVI  - 102\\nIP  - 9\\nDP  - 2019 Sep\\nTI  - Short communication: Transcriptional response to a large genomic island deletion in  the dairy starter culture Streptococcus thermophilus.\\nPG  - 7800-7806\\nLID - S0022-0302(19)30572-7 [pii]\\nLID - 10.3168/jds.2019-16397 [doi]\\nAB  - Streptococcus thermophilus is a lactic acid bacterium widely used in the syntrophic  fermentation of milk into yogurt and cheese. Streptococcus thermophilus has adapted  to ferment milk primarily through reductive genome evolution but also through  acquisition of genes conferring proto-cooperation with Lactobacillus bulgaricus and  efficient metabolism of milk macronutrients. Genomic analysis of Strep. thermophilus  strains suggests that mobile genetic elements have contributed to genomic evolution  through horizontal gene transfer and genomic plasticity. We previously used the  endogenous type II CRISPR-Cas [clustered regularly interspaced short palindromic  repeats (CRISPR) with CRISPR-associated sequences (Cas)] system in Strep.  thermophilus to isolate derivatives lacking the chromosomal mobile genetic element  and expandable island that display decreased fitness under routine culturing  conditions. Of note, the Lac operon and Leloir pathway genes were deleted in the  largest expendable genomic island (102 kbp), rendering the strain incapable of  acidifying milk. However, the removal of other open reading frames in the same  island had unclear effects on the fitness and regulatory networks of Strep.  thermophilus. To uncover the physiological basis for the observed phenotypic changes  and underlying regulatory networks affected by deletion of the 102-kbp genomic  island in Strep. thermophilus, we analyzed the transcriptome of the mutant that  lacked âˆ¼5% of its genome. In addition to the loss of transcripts encoded by the  deleted material, we detected a total of 56 genes that were differentially  expressed, primarily encompassing 10 select operons. Several predicted metabolic  pathways were affected, including amino acid and purine metabolism, oligopeptide  transport, and iron transport. Collectively, these results suggest that deletion of  a 102-kb genomic island in Strep. thermophilus influences compensatory transcription  of starvation stress response genes and metabolic pathways involved in important  niche-related adaptation.\\nCI  - Copyright Â© 2019 American Dairy Science Association. Published by Elsevier Inc. All  rights reserved.\\nFAU - Selle, Kurt\\nAU  - Selle K\\nAD  - Department of Food, Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh 27695; Functional Genomics Graduate Program, North Carolina  State University, Raleigh 27695. Electronic address: kmselle@ncsu.edu.\\nFAU - Andersen, Joakim M\\nAU  - Andersen JM\\nAD  - Department of Food, Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh 27695.\\nFAU - Barrangou, Rodolphe\\nAU  - Barrangou R\\nAD  - Department of Food, Bioprocessing and Nutrition Sciences, North Carolina State  University, Raleigh 27695; Functional Genomics Graduate Program, North Carolina  State University, Raleigh 27695. Electronic address: rbarran@ncsu.edu.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20190703\\nPL  - United States\\nTA  - J Dairy Sci\\nJT  - Journal of dairy science\\nJID - 2985126R\\nRN  - 33X04XA5AT (Lactic Acid)\\nSB  - IM\\nMH  - Animals\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Fermentation\\nMH  - Gene Deletion\\nMH  - Gene Expression Regulation, Bacterial/physiology\\nMH  - *Genomic Islands\\nMH  - Genomics\\nMH  - Lactic Acid/metabolism\\nMH  - Lactobacillus delbrueckii/metabolism\\nMH  - Milk/*microbiology\\nMH  - Streptococcus thermophilus/genetics/*metabolism\\nMH  - Transcriptome\\nMH  - Yogurt/*microbiology\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Streptococcus thermophilus\\nOT  - fermentation\\nOT  - transcriptome\\nEDAT- 2019/07/08 06:00\\nMHDA- 2019/11/21 06:00\\nCRDT- 2019/07/08 06:00\\nPHST- 2019/01/29 00:00 [received]\\nPHST- 2019/05/02 00:00 [accepted]\\nPHST- 2019/07/08 06:00 [pubmed]\\nPHST- 2019/11/21 06:00 [medline]\\nPHST- 2019/07/08 06:00 [entrez]\\nAID - S0022-0302(19)30572-7 [pii]\\nAID - 10.3168/jds.2019-16397 [doi]\\nPST - ppublish\\nSO  - J Dairy Sci. 2019 Sep;102(9):7800-7806. doi: 10.3168/jds.2019-16397. Epub 2019 Jul  3.\\n\\nPMID- 26590262\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160621\\nLR  - 20181113\\nIS  - 1362-4962 (Electronic)\\nIS  - 0305-1048 (Print)\\nIS  - 0305-1048 (Linking)\\nVI  - 43\\nIP  - 22\\nDP  - 2015 Dec 15\\nTI  - Comparative genomic analyses reveal a vast, novel network of nucleotide-centric  systems in biological conflicts, immunity and signaling.\\nPG  - 10633-54\\nLID - 10.1093/nar/gkv1267 [doi]\\nAB  - Cyclic di- and linear oligo-nucleotide signals activate defenses against invasive  nucleic acids in animal immunity; however, their evolutionary antecedents are poorly  understood. Using comparative genomics, sequence and structure analysis, we  uncovered a vast network of systems defined by conserved prokaryotic  gene-neighborhoods, which encode enzymes generating such nucleotides or  alternatively processing them to yield potential signaling molecules. The  nucleotide-generating enzymes include several clades of the DNA-polymerase Î²-like  superfamily (including Vibrio cholerae DncV), a minimal version of the CRISPR  polymerase and DisA-like cyclic-di-AMP synthetases. Nucleotide-binding/processing  domains include TIR domains and members of a superfamily prototyped by Smf/DprA  proteins and base (cytokinin)-releasing LOG enzymes. They are combined in conserved  gene-neighborhoods with genes for a plethora of protein superfamilies, which we  predict to function as nucleotide-sensors and effectors targeting nucleic acids,  proteins or membranes (pore-forming agents). These systems are sometimes combined  with other biological conflict-systems such as restriction-modification and  CRISPR/Cas. Interestingly, several are coupled in mutually exclusive neighborhoods  with either a prokaryotic ubiquitin-system or a HORMA domain-PCH2-like AAA+ ATPase  dyad. The latter are potential precursors of equivalent proteins in eukaryotic  chromosome dynamics. Further, components from these nucleotide-centric systems have  been utilized in several other systems including a novel diversity-generating system  with a reverse transcriptase. We also found the Smf/DprA/LOG domain from these  systems to be recruited as a predicted nucleotide-binding domain in eukaryotic TRPM  channels. These findings point to evolutionary and mechanistic links, which bring  together CRISPR/Cas, animal interferon-induced immunity, and several other systems  that combine nucleic-acid-sensing and nucleotide-dependent signaling.\\nCI  - Published by Oxford University Press on behalf of Nucleic Acids Research 2015. This  work is written by (a) US Government employee(s) and is in the public domain in the  US.\\nFAU - Burroughs, A Maxwell\\nAU  - Burroughs AM\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA.\\nFAU - Zhang, Dapeng\\nAU  - Zhang D\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA.\\nFAU - SchÃ¤ffer, Daniel E\\nAU  - SchÃ¤ffer DE\\nAD  - Montgomery Blair High School, Magnet Program, Silver Spring, MD 20901, USA.\\nFAU - Iyer, Lakshminarayan M\\nAU  - Iyer LM\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA.\\nFAU - Aravind, L\\nAU  - Aravind L\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, MD 20894, USA aravind@ncbi.nlm.nih.gov.\\nLA  - eng\\nGR  - Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20151120\\nTA  - Nucleic Acids Res\\nJT  - Nucleic acids research\\nJID - 0411011\\nRN  - 0 (Archaeal Proteins)\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (DNA Restriction-Modification Enzymes)\\nRN  - 0 (Membrane Transport Proteins)\\nRN  - 0 (Nucleotides)\\nRN  - 0 (Ubiquitins)\\nRN  - EC 2.7.7.- (Nucleotidyltransferases)\\nSB  - IM\\nMH  - Archaea/enzymology\\nMH  - Archaeal Proteins/chemistry/metabolism\\nMH  - Bacteria/enzymology/metabolism\\nMH  - Bacterial Proteins/chemistry/metabolism\\nMH  - Biological Evolution\\nMH  - CRISPR-Cas Systems\\nMH  - DNA Restriction-Modification Enzymes/metabolism\\nMH  - Eukaryota/enzymology/metabolism\\nMH  - Gene Regulatory Networks\\nMH  - Genomics\\nMH  - Ion Transport\\nMH  - Membrane Transport Proteins/metabolism\\nMH  - Nucleotides/*metabolism\\nMH  - Nucleotidyltransferases/metabolism\\nMH  - Protein Structure, Tertiary\\nMH  - *Signal Transduction\\nMH  - Ubiquitins/metabolism\\nPMC - PMC4678834\\nEDAT- 2015/11/22 06:00\\nMHDA- 2016/06/22 06:00\\nCRDT- 2015/11/22 06:00\\nPHST- 2015/11/04 00:00 [accepted]\\nPHST- 2015/09/11 00:00 [received]\\nPHST- 2015/11/22 06:00 [entrez]\\nPHST- 2015/11/22 06:00 [pubmed]\\nPHST- 2016/06/22 06:00 [medline]\\nAID - gkv1267 [pii]\\nAID - 10.1093/nar/gkv1267 [doi]\\nPST - ppublish\\nSO  - Nucleic Acids Res. 2015 Dec 15;43(22):10633-54. doi: 10.1093/nar/gkv1267. Epub 2015  Nov 20.\\n\\nPMID- 31444197\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200804\\nLR  - 20200804\\nIS  - 1098-5336 (Electronic)\\nIS  - 0099-2240 (Print)\\nIS  - 0099-2240 (Linking)\\nVI  - 85\\nIP  - 21\\nDP  - 2019 Nov 1\\nTI  - CRISPR/Cas9-Assisted Seamless Genome Editing in Lactobacillus plantarum and Its  Application in N-Acetylglucosamine Production.\\nLID - 10.1128/AEM.01367-19 [doi]\\nLID - e01367-19\\nAB  - Lactobacillus plantarum is a potential starter and health-promoting probiotic  bacterium. Effective, precise, and diverse genome editing of Lactobacillus plantarum  without introducing exogenous genes or plasmids is of great importance. In this  study, CRISPR/Cas9-assisted double-stranded DNA (dsDNA) and single-stranded DNA  (ssDNA) recombineering was established in L. plantarum WCFS1 to seamlessly edit the  genome, including gene knockouts, insertions, and point mutations. To optimize our  editing method, phosphorothioate modification was used to improve the dsDNA  insertion, and adenine-specific methyltransferase was used to improve the ssDNA  recombination efficiency. These strategies were applied to engineer L. plantarum  WCFS1 toward producing N-acetylglucosamine (GlcNAc). nagB was truncated to eliminate  the reverse reaction of fructose-6-phosphate (F6P) to glucosamine 6-phosphate  (GlcN-6P). Riboswitch replacement and point mutation in glmS1 were introduced to  relieve feedback repression. The resulting strain produced 797.3â€‰mg/liter GlcNAc  without introducing exogenous genes or plasmids. This strategy may contribute to the  available methods for precise and diverse genetic engineering in lactic acid  bacteria and boost strain engineering for more applications.IMPORTANCE  CRISPR/Cas9-assisted recombineering is restricted in lactic acid bacteria because of  the lack of available antibiotics and vectors. In this study, a seamless genome  editing method was carried out in Lactobacillus plantarum using CRISPR/Cas9-assisted  double-stranded DNA (dsDNA) and single-stranded DNA (ssDNA) recombineering, and  recombination efficiency was effectively improved by endogenous adenine-specific  methyltransferase overexpression. L. plantarum WCFS1 produced 797.3â€‰mg/liter  N-acetylglucosamine (GlcNAc) through reinforcement of the GlcNAc pathway, without  introducing exogenous genes or plasmids. This seamless editing strategy, combined  with the potential exogenous GlcNAc-producing pathway, makes this strain an  attractive candidate for industrial use in the future.\\nCI  - Copyright Â© 2019 American Society for Microbiology.\\nFAU - Zhou, Ding\\nAU  - Zhou D\\nAD  - National Glycoengineering Research Center, Shandong University, Qingdao, People\\'s  Republic of China.\\nFAU - Jiang, Zhennan\\nAU  - Jiang Z\\nAD  - State Key Laboratory of Microbial Technology, Shandong University, Qingdao, People\\'s  Republic of China.\\nFAU - Pang, Qingxiao\\nAU  - Pang Q\\nAD  - State Key Laboratory of Microbial Technology, Shandong University, Qingdao, People\\'s  Republic of China.\\nFAU - Zhu, Yuan\\nAU  - Zhu Y\\nAD  - National Glycoengineering Research Center, Shandong University, Qingdao, People\\'s  Republic of China.\\nFAU - Wang, Qian\\nAU  - Wang Q\\nAD  - National Glycoengineering Research Center, Shandong University, Qingdao, People\\'s  Republic of China qiqi20011983@gmail.com.\\nAD  - State Key Laboratory of Microbial Technology, Shandong University, Qingdao, People\\'s  Republic of China.\\nFAU - Qi, Qingsheng\\nAU  - Qi Q\\nAD  - State Key Laboratory of Microbial Technology, Shandong University, Qingdao, People\\'s  Republic of China.\\nAD  - CAS Key Lab of Biobased Materials, Qingdao Institute of Bioenergy and Bioprocess  Technology, Chinese Academy of Sciences, Qingdao, People\\'s Republic of China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20191016\\nTA  - Appl Environ Microbiol\\nJT  - Applied and environmental microbiology\\nJID - 7605801\\nRN  - 0 (DNA, Single-Stranded)\\nRN  - 9007-49-2 (DNA)\\nRN  - V956696549 (Acetylglucosamine)\\nSB  - IM\\nMH  - Acetylglucosamine/*biosynthesis\\nMH  - *CRISPR-Cas Systems\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - DNA\\nMH  - DNA, Single-Stranded\\nMH  - Gene Editing/*methods\\nMH  - Gene Knockout Techniques\\nMH  - Genes, Bacterial/genetics\\nMH  - Genetic Engineering\\nMH  - Genome, Bacterial\\nMH  - Lactobacillus plantarum/*genetics\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Recombination, Genetic\\nPMC - PMC6803305\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas9\\nOT  - *Lactobacillus plantarum WCFS1\\nOT  - *N-acetylglucosamine\\nOT  - *metabolic engineering\\nOT  - *recombineering\\nEDAT- 2019/08/25 06:00\\nMHDA- 2020/08/05 06:00\\nCRDT- 2019/08/25 06:00\\nPHST- 2019/06/18 00:00 [received]\\nPHST- 2019/08/14 00:00 [accepted]\\nPHST- 2019/08/25 06:00 [pubmed]\\nPHST- 2020/08/05 06:00 [medline]\\nPHST- 2019/08/25 06:00 [entrez]\\nAID - AEM.01367-19 [pii]\\nAID - 01367-19 [pii]\\nAID - 10.1128/AEM.01367-19 [doi]\\nPST - epublish\\nSO  - Appl Environ Microbiol. 2019 Oct 16;85(21):e01367-19. doi: 10.1128/AEM.01367-19.  Print 2019 Nov 1.\\n\\nPMID- 33399232\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210812\\nLR  - 20210812\\nIS  - 1097-4644 (Electronic)\\nIS  - 0730-2312 (Linking)\\nVI  - 122\\nIP  - 3-4\\nDP  - 2021 Apr\\nTI  - HnRNPA2B1 promotes the proliferation of breast cancer MCF-7 cells via the STAT3  pathway.\\nPG  - 472-484\\nLID - 10.1002/jcb.29875 [doi]\\nAB  - HnRNPA2/B1 is highly expressed in many tumors. However, the role of hnRNPA2/B1 in  breast cancer isÂ\\xa0not clear. In this study, we found the proliferation rate was  decreased after knockout of hnRNPA2/B1 by CRISPR-CAS9 in MCF-7 cells, as  demonstrated by the reduced expression of CDK4 and p-AKT, and the increased  expression of P27. Besides this, the western blot results showed that knockout of  hnRNPA2/B1 increased the rate of apoptosis and declined autophagy. ByÂ\\xa0in vivo assay,  we found that knockout of hnRNPA2/B1 suppressed tumor growth in a xenograft mouse  model. Immunohistochemical staining results confirmed knockout of hnRNPA2/B1  impaired tumor angiogenesis, as illustrated by downregulated expression of VEGF-A.  BesidesÂ\\xa0this,Â\\xa0interacting proteins with hnRNPA2/B1 were identified by mass  spectrometry and the PPI network was constructed. GO analysis suggests that the  Interacting proteins are mainly enriched in the Wnt signaling pathway, tumor  necrosis factor-mediated signaling pathway, translation, and so on. We then  identified hnRNPA2/B1 interacted with signal transducer and activator of  transcription 3 (STAT3), as supported by the colocalization of hnRNPA2/B1 and STAT3.  Meanwhile, knockout of hnRNPA2/B1 inhibited the phosphorylation of STAT3.  Collectively, our results demonstrate that hnRNPA2/B1 promotes tumor cell growth in  vitro and in vivo by activating the STAT3 pathway, regulating apoptosis and  autophagy.\\nCI  - Â© 2020 Wiley Periodicals LLC.\\nFAU - Gao, Li-Bin\\nAU  - Gao LB\\nAUID- ORCID: 0000-0001-9625-5208\\nAD  - Cancer Research Center, School of Medicine, Xiamen University, Xiamen, China.\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, University of Chinese Academy of Sciences, Shanghai, 200031,  China.\\nFAU - Zhu, Xin-Le\\nAU  - Zhu XL\\nAD  - Cancer Research Center, School of Medicine, Xiamen University, Xiamen, China.\\nFAU - Shi, Jing-Xian\\nAU  - Shi JX\\nAD  - Cancer Research Center, School of Medicine, Xiamen University, Xiamen, China.\\nFAU - Yang, Ling\\nAU  - Yang L\\nAD  - Cancer Research Center, School of Medicine, Xiamen University, Xiamen, China.\\nFAU - Xu, Zhen-Qiang\\nAU  - Xu ZQ\\nAD  - Department of Urology, Zhangzhou Affiliated Hospital of Fujian Medical University,  Zhangzhou, China.\\nFAU - Shi, Song-Lin\\nAU  - Shi SL\\nAUID- ORCID: 0000-0003-1213-0550\\nAD  - Cancer Research Center, School of Medicine, Xiamen University, Xiamen, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210105\\nPL  - United States\\nTA  - J Cell Biochem\\nJT  - Journal of cellular biochemistry\\nJID - 8205768\\nRN  - 0 (Heterogeneous-Nuclear Ribonucleoprotein Group A-B)\\nRN  - 0 (STAT3 Transcription Factor)\\nSB  - IM\\nMH  - Animals\\nMH  - Apoptosis/physiology\\nMH  - Breast Neoplasms/*metabolism\\nMH  - Cell Line, Tumor\\nMH  - Cell Proliferation/physiology\\nMH  - Female\\nMH  - Gene Expression Regulation, Neoplastic/physiology\\nMH  - Heterogeneous-Nuclear Ribonucleoprotein Group A-B/*metabolism\\nMH  - Humans\\nMH  - Immunohistochemistry\\nMH  - MCF-7 Cells\\nMH  - STAT3 Transcription Factor/genetics/metabolism\\nMH  - Signal Transduction/physiology\\nOTO - NOTNLM\\nOT  - *HnRNPA2/B1\\nOT  - *STAT3 pathway\\nOT  - *breast cancer\\nOT  - *tumor angiogenesis\\nEDAT- 2021/01/06 06:00\\nMHDA- 2021/08/13 06:00\\nCRDT- 2021/01/05 08:38\\nPHST- 2020/08/13 00:00 [received]\\nPHST- 2020/11/11 00:00 [revised]\\nPHST- 2020/11/16 00:00 [accepted]\\nPHST- 2021/01/06 06:00 [pubmed]\\nPHST- 2021/08/13 06:00 [medline]\\nPHST- 2021/01/05 08:38 [entrez]\\nAID - 10.1002/jcb.29875 [doi]\\nPST - ppublish\\nSO  - J Cell Biochem. 2021 Apr;122(3-4):472-484. doi: 10.1002/jcb.29875. Epub 2021 Jan 5.\\n\\nPMID- 33981145\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210514\\nIS  - 1178-2390 (Print)\\nIS  - 1178-2390 (Electronic)\\nIS  - 1178-2390 (Linking)\\nVI  - 14\\nDP  - 2021\\nTI  - Attitudes of Indonesian Medical Doctors and Medical Students Toward Genome Editing.\\nPG  - 1017-1027\\nLID - 10.2147/JMDH.S303881 [doi]\\nAB  - PURPOSE: This study aimed to measure the attitudes of Indonesian medical doctors and  students toward new technology in genome editing. MATERIALS AND METHODS: Online  questionnaires regarding attitudes toward genome editing on health and non-health  conditions, both in somatic cells and embryo, were distributed through researcher  networks, email and social media specific to medical doctors and students. The data  of 1055 valid questionnaires were processed; descriptive and association analyses  between sociodemographic factors and attitudes toward genome editing were performed.  Email in-depth interview was performed to explore the respondents\\' answers. RESULTS:  The results showed that Indonesian medical doctors\\' and students\\' knowledge of  genome editing was limited and correlated with gender, place of residence, religion,  education, marital status, childbearing and experience abroad. More than half of  respondents supported genome editing for the treatment of fatal and debilitating  diseases both in somatic cells and embryos, implying their consent to edited gene  inheritance. However, this approval decreased when applied to non-health-related  aspects, such as physical appearance, intelligence and strength. Factors affecting  their attitudes toward genome editing included their status as medical doctors or  students, gender, age, education, religion, economic status and place of residence.  CONCLUSION: Increasing knowledge and awareness of Indonesian medical doctors and  students regarding genome editing is important. Even though its application in  health-related matter was supported by a majority of the respondents, discussion  from ethical and religious perspectives is necessary to ensure the acceptance.\\nCI  - Â© 2021 Izzah et al.\\nFAU - Izzah, Safira Nur\\nAU  - Izzah SN\\nAD  - Faculty of Medicine, Universitas Airlangga, Surabaya, Indonesia.\\nFAU - Setyanto, Dimas\\nAU  - Setyanto D\\nAD  - Faculty of Medicine, Universitas Airlangga, Surabaya, Indonesia.\\nFAU - Hasanatuludhhiyah, Nurina\\nAU  - Hasanatuludhhiyah N\\nAD  - Division of Pharmacology and Therapy, Department of Anatomy, Histology, and  Pharmacology, Faculty of Medicine, Universitas Airlangga, Surabaya, Indonesia.\\nAD  - Airlangga Research Group for Translational Medicine and Therapeutic, Universitas  Airlangga, Surabaya, Indonesia.\\nFAU - Indiastuti, Danti Nur\\nAU  - Indiastuti DN\\nAD  - Division of Pharmacology and Therapy, Department of Anatomy, Histology, and  Pharmacology, Faculty of Medicine, Universitas Airlangga, Surabaya, Indonesia.\\nAD  - Airlangga Research Group for Translational Medicine and Therapeutic, Universitas  Airlangga, Surabaya, Indonesia.\\nFAU - Nasution, Zamal\\nAU  - Nasution Z\\nAD  - Institute for Population and Social Research, Mahidol University, Bangkok, Thailand.\\nFAU - d\\'Arqom, Annette\\nAU  - d\\'Arqom A\\nAD  - Division of Pharmacology and Therapy, Department of Anatomy, Histology, and  Pharmacology, Faculty of Medicine, Universitas Airlangga, Surabaya, Indonesia.\\nAD  - Airlangga Research Group for Translational Medicine and Therapeutic, Universitas  Airlangga, Surabaya, Indonesia.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210504\\nTA  - J Multidiscip Healthc\\nJT  - Journal of multidisciplinary healthcare\\nJID - 101512691\\nPMC - PMC8106925\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - developing country\\nOT  - good health and well-being\\nCOIS- The authors report no conflicts of interest in this work.\\nEDAT- 2021/05/14 06:00\\nMHDA- 2021/05/14 06:01\\nCRDT- 2021/05/13 06:23\\nPHST- 2021/01/27 00:00 [received]\\nPHST- 2021/03/29 00:00 [accepted]\\nPHST- 2021/05/13 06:23 [entrez]\\nPHST- 2021/05/14 06:00 [pubmed]\\nPHST- 2021/05/14 06:01 [medline]\\nAID - 303881 [pii]\\nAID - 10.2147/JMDH.S303881 [doi]\\nPST - epublish\\nSO  - J Multidiscip Healthc. 2021 May 4;14:1017-1027. doi: 10.2147/JMDH.S303881.  eCollection 2021.\\n\\nPMID- 31852806\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200605\\nLR  - 20200605\\nIS  - 2379-5042 (Electronic)\\nIS  - 2379-5042 (Linking)\\nVI  - 4\\nIP  - 6\\nDP  - 2019 Dec 18\\nTI  - Genome-Resolved Metagenomics Extends the Environmental Distribution of the  Verrucomicrobia Phylum to the Deep Terrestrial Subsurface.\\nLID - 10.1128/mSphere.00613-19 [doi]\\nLID - e00613-19\\nAB  - Bacteria of the phylum Verrucomicrobia are prevalent and are particularly common in  soil and freshwater environments. Their cosmopolitan distribution and reported  capacity for polysaccharide degradation suggests members of Verrucomicrobia are  important contributors to carbon cycling across Earth\\'s ecosystems. Despite their  prevalence, the Verrucomicrobia are underrepresented in isolate collections and  genome databases; consequently, their ecophysiological roles may not be fully  realized. Here, we expand genomic sampling of the Verrucomicrobia phylum by  describing a novel genus, \"Candidatus Marcellius,\" belonging to the order Opitutales  \"Ca. Marcellius\" was recovered from a shale-derived produced fluid metagenome  collected 313â€‰days after hydraulic fracturing, the deepest environment from which a  member of the Verrucomicrobia has been recovered to date. We uncover genomic  attributes that may explain the capacity of this organism to inhabit a shale gas  well, including the potential for utilization of organic polymers common in  hydraulic fracturing fluids, nitrogen fixation, adaptation to high salinities, and  adaptive immunity via CRISPR-Cas. To illuminate the phylogenetic and environmental  distribution of these metabolic and adaptive traits across the Verrucomicrobia  phylum, we performed a comparative genomic analysis of 31 publicly available, nearly  complete Verrucomicrobia genomes. Our genomic findings extend the environmental  distribution of the Verrucomicrobia 2.3 kilometers into the terrestrial subsurface.  Moreover, we reveal traits widely encoded across members of the Verrucomicrobia,  including the capacity to degrade hemicellulose and to adapt to physical and  biological environmental perturbations, thereby contributing to the expansive  habitat range reported for this phylum.IMPORTANCE The Verrucomicrobia phylum of  bacteria is widespread in many different ecosystems; however, its role in microbial  communities remains poorly understood. Verrucomicrobia are often low-abundance  community members, yet previous research suggests they play a major role in organic  carbon degradation. While Verrucomicrobia remain poorly represented in culture  collections, numerous genomes have been reconstructed from metagenomic data sets in  recent years. The study of genomes from across the phylum allows for an extensive  assessment of their potential ecosystem roles. The significance of this work is (i)  the recovery of a novel genus of Verrucomicrobia from 2.3â€‰km in the subsurface with  the ability to withstand the extreme conditions that characterize this environment,  and (ii) the most extensive assessment of ecophysiological traits encoded by  Verrucomicrobia genomes to date. We show that members of this phylum are specialist  organic polymer degraders that can withstand a wider range of environmental  conditions than previously thought.\\nCI  - Copyright Â© 2019 Nixon et al.\\nFAU - Nixon, Sophie L\\nAU  - Nixon SL\\nAUID- ORCID: 0000-0003-2914-8386\\nAD  - Department of Earth and Environmental Sciences, University of Manchester,  Manchester, United Kingdom sophie.nixon@manchester.ac.uk kwrighton@gmail.com.\\nFAU - Daly, Rebecca A\\nAU  - Daly RA\\nAD  - Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA.\\nFAU - Borton, Mikayla A\\nAU  - Borton MA\\nAD  - Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA.\\nFAU - Solden, Lindsey M\\nAU  - Solden LM\\nAD  - Centre for Microbial Pathogenesis, The Research Institute at Nationwide Children\\'s  Hospital, Columbus, Ohio, USA.\\nFAU - Welch, Susan A\\nAU  - Welch SA\\nAD  - School of Earth Sciences, The Ohio State University, Columbus, Ohio, USA.\\nFAU - Cole, David R\\nAU  - Cole DR\\nAD  - School of Earth Sciences, The Ohio State University, Columbus, Ohio, USA.\\nFAU - Mouser, Paula J\\nAU  - Mouser PJ\\nAUID- ORCID: 0000-0003-2316-0915\\nAD  - College of Engineering and Physical Sciences, University of New Hampshire, Durham,  New Hampshire, USA.\\nFAU - Wilkins, Michael J\\nAU  - Wilkins MJ\\nAD  - Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA.\\nFAU - Wrighton, Kelly C\\nAU  - Wrighton KC\\nAD  - Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA  sophie.nixon@manchester.ac.uk kwrighton@gmail.com.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20191218\\nTA  - mSphere\\nJT  - mSphere\\nJID - 101674533\\nSB  - IM\\nMH  - Computational Biology\\nMH  - Genes, Bacterial\\nMH  - *Genome, Bacterial\\nMH  - Genomics\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Metagenomics/*methods\\nMH  - *Soil Microbiology\\nMH  - Verrucomicrobia/*classification/*genetics\\nPMC - PMC6920513\\nOTO - NOTNLM\\nOT  - *glycoside hydrolases\\nOT  - *hydraulic fracturing\\nOT  - *hypersaline\\nOT  - *shale\\nOT  - *viruses\\nEDAT- 2019/12/20 06:00\\nMHDA- 2020/06/06 06:00\\nCRDT- 2019/12/20 06:00\\nPHST- 2019/12/20 06:00 [entrez]\\nPHST- 2019/12/20 06:00 [pubmed]\\nPHST- 2020/06/06 06:00 [medline]\\nAID - 4/6/e00613-19 [pii]\\nAID - mSphere00613-19 [pii]\\nAID - 10.1128/mSphere.00613-19 [doi]\\nPST - epublish\\nSO  - mSphere. 2019 Dec 18;4(6):e00613-19. doi: 10.1128/mSphere.00613-19.\\n\\nPMID- 33620493\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220217\\nLR  - 20220217\\nIS  - 1532-2548 (Electronic)\\nIS  - 0032-0889 (Print)\\nIS  - 0032-0889 (Linking)\\nVI  - 186\\nIP  - 1\\nDP  - 2021 May 27\\nTI  - OsmiR396/growth regulating factor modulate rice grain size through direct regulation  of embryo-specific miR408.\\nPG  - 519-533\\nLID - 10.1093/plphys/kiab084 [doi]\\nAB  - microRNAs (miRNAs) are promising targets for crop improvement of complex  agricultural traits. Coordinated activity between/among different miRNAs may  fine-tune specific developmental processes in diverse organisms. Grain size is a  main factor determining rice (Oryza sativa L.) crop yield, but the network of miRNAs  influencing this trait remains uncharacterized. Here we show that sequestering  OsmiR396 through target mimicry (MIM396) can substantially increase grain size in  several japonica and indica rice subspecies and in plants with excessive tillers and  a high panicle density. Thus, OsmiR396 has a major role related to the regulation of  rice grain size. The grain shape of Growth Regulating Factor8  (OsGRF8)-overexpressing transgenic plants was most similar to that of MIM396 plants,  suggesting OsGRF8 is a major mediator of OsmiR396 in grain size regulation. A miRNA  microarray analysis revealed changes to the expression of many miRNAs, including  OsmiR408, in the MIM396 plants. Analyses of gene expression patterns and functions  indicated OsmiR408 is an embryo-specific miRNA that positively regulates grain size.  Silencing OsmiR408 expression (miR408KO) using CRISPR technology resulted in small  grains. Moreover, we revealed the direct regulatory effects of OsGRF8 on OsMIR408  expression. A genetic analysis further showed that the large-grain phenotype of  MIM396 plants could be complemented by miR408KO. Also, several hormone signaling  pathways might be involved in the OsmiR396/GRF-meditated grain size regulation. Our  findings suggest that genetic regulatory networks comprising various miRNAs, such as  OsmiR396 and OsmiR408, may be crucial for controlling rice grain size. Furthermore,  the OsmiR396/GRF module may be important for breeding new high-yielding rice  varieties.\\nCI  - Â© The Author(s) 2021. Published by Oxford University Press on behalf of American  Society of Plant Biologists.\\nFAU - Yang, Xiaofang\\nAU  - Yang X\\nAD  - Key Laboratory of Insect Developmental and Evolutionary Biology, CAS Center for  Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology and  Ecology, Chinese Academy of Sciences, Shanghai 200032, China.\\nAD  - University of Chinese Academy of Sciences, Shanghai 200032, China.\\nFAU - Zhao, Xiaoling\\nAU  - Zhao X\\nAUID- ORCID: 0000-0002-6275-7418\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai 200032, China.\\nFAU - Dai, Zhengyan\\nAU  - Dai Z\\nAD  - Key Laboratory of Insect Developmental and Evolutionary Biology, CAS Center for  Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology and  Ecology, Chinese Academy of Sciences, Shanghai 200032, China.\\nFAU - Ma, Feilong\\nAU  - Ma F\\nAD  - Key Laboratory of Insect Developmental and Evolutionary Biology, CAS Center for  Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology and  Ecology, Chinese Academy of Sciences, Shanghai 200032, China.\\nAD  - University of Chinese Academy of Sciences, Shanghai 200032, China.\\nFAU - Miao, Xuexia\\nAU  - Miao X\\nAUID- ORCID: 0000-0003-3244-0126\\nAD  - Key Laboratory of Insect Developmental and Evolutionary Biology, CAS Center for  Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology and  Ecology, Chinese Academy of Sciences, Shanghai 200032, China.\\nFAU - Shi, Zhenying\\nAU  - Shi Z\\nAUID- ORCID: 0000-0002-8441-2162\\nAD  - Key Laboratory of Insect Developmental and Evolutionary Biology, CAS Center for  Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology and  Ecology, Chinese Academy of Sciences, Shanghai 200032, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Plant Physiol\\nJT  - Plant physiology\\nJID - 0401224\\nRN  - 0 (MicroRNAs)\\nRN  - 0 (Plant Growth Regulators)\\nRN  - 0 (Plant Proteins)\\nRN  - 0 (RNA, Plant)\\nSB  - IM\\nMH  - Edible Grain/genetics/*growth & development\\nMH  - MicroRNAs/*metabolism\\nMH  - Oryza/*genetics/growth & development\\nMH  - Plant Growth Regulators/*metabolism\\nMH  - Plant Proteins/*genetics/metabolism\\nMH  - RNA, Plant/*metabolism\\nPMC - PMC8154042\\nEDAT- 2021/02/24 06:00\\nMHDA- 2021/02/24 06:00\\nCRDT- 2021/02/23 12:15\\nPHST- 2020/10/26 00:00 [received]\\nPHST- 2021/02/08 00:00 [accepted]\\nPHST- 2021/02/24 06:00 [pubmed]\\nPHST- 2021/02/24 06:00 [medline]\\nPHST- 2021/02/23 12:15 [entrez]\\nAID - 6146992 [pii]\\nAID - kiab084 [pii]\\nAID - 10.1093/plphys/kiab084 [doi]\\nPST - ppublish\\nSO  - Plant Physiol. 2021 May 27;186(1):519-533. doi: 10.1093/plphys/kiab084.\\n\\nPMID- 26797494\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20161227\\nLR  - 20161230\\nIS  - 2161-5063 (Electronic)\\nIS  - 2161-5063 (Linking)\\nVI  - 5\\nIP  - 4\\nDP  - 2016 Apr 15\\nTI  - Molecular Titration Promotes Oscillations and Bistability in Minimal Network Models  with Monomeric Regulators.\\nPG  - 321-33\\nLID - 10.1021/acssynbio.5b00176 [doi]\\nAB  - Molecular titration is emerging as an important biochemical interaction mechanism  within synthetic devices built with nucleic acids and the CRISPR/Cas system. We show  that molecular titration in the context of feedback circuits is a suitable mechanism  to enhance the emergence of oscillations and bistable behaviors. We consider  biomolecular modules that can be inhibited or activated by input monomeric  regulators; the regulators compete with constitutive titrating species to determine  the activity of their target. By tuning the titration rate and the concentration of  titrating species, it is possible to modulate the delay and convergence speed of the  transient response, and the steepness and dead zone of the stationary response of  the modules. These phenomena favor the occurrence of oscillations when modules are  interconnected to create a negative feedback loop; bistability is favored in a  positive feedback interconnection. Numerical simulations are supported by  mathematical analysis showing that the capacity of the closed loop systems to  exhibit oscillations or bistability is structural.\\nFAU - Cuba Samaniego, Christian\\nAU  - Cuba Samaniego C\\nAD  - Mechanical Engineering, University of California at Riverside , Riverside,  California 92521, United States.\\nFAU - Giordano, Giulia\\nAU  - Giordano G\\nAD  - Mathematics and Computer Science, University of Udine , 33100 Udine, Italy.\\nFAU - Kim, Jongmin\\nAU  - Kim J\\nAD  - Wyss Institute for Biologically Inspired Engineering, Harvard University , 3  Blackfan Circle, Boston, Massachusetts 02115, United States.\\nFAU - Blanchini, Franco\\nAU  - Blanchini F\\nAD  - Mathematics and Computer Science, University of Udine , 33100 Udine, Italy.\\nFAU - Franco, Elisa\\nAU  - Franco E\\nAD  - Mechanical Engineering, University of California at Riverside , Riverside,  California 92521, United States.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20160219\\nPL  - United States\\nTA  - ACS Synth Biol\\nJT  - ACS synthetic biology\\nJID - 101575075\\nSB  - IM\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Gene Regulatory Networks/genetics\\nMH  - *Models, Biological\\nOTO - NOTNLM\\nOT  - RNA\\nOT  - bistability\\nOT  - delays\\nOT  - monomeric regulator\\nOT  - oscillations\\nOT  - synthetic biology\\nOT  - titration\\nEDAT- 2016/01/23 06:00\\nMHDA- 2016/12/28 06:00\\nCRDT- 2016/01/23 06:00\\nPHST- 2016/01/23 06:00 [entrez]\\nPHST- 2016/01/23 06:00 [pubmed]\\nPHST- 2016/12/28 06:00 [medline]\\nAID - 10.1021/acssynbio.5b00176 [doi]\\nPST - ppublish\\nSO  - ACS Synth Biol. 2016 Apr 15;5(4):321-33. doi: 10.1021/acssynbio.5b00176. Epub 2016  Feb 19.\\n\\nPMID- 30473704\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 1664-8021 (Print)\\nIS  - 1664-8021 (Electronic)\\nIS  - 1664-8021 (Linking)\\nVI  - 9\\nDP  - 2018\\nTI  - FGF10 and the Mystery of Duodenal Atresia in Humans.\\nPG  - 530\\nLID - 10.3389/fgene.2018.00530 [doi]\\nLID - 530\\nAB  - Background: Duodenal atresia (DA) is a congenital obstruction of the duodenum, which  affects 1 in 7000 pregnancies and requires major surgery in the 1st days of life.  Three morphological DA types are described. In humans, the association between DA  and Down syndrome suggests an underlying, albeit elusive, genetic etiology. In mice,  interruption of fibroblast growth factor 10 (Fgf10) gene signaling results in DA in  30-50% of embryos, supporting a genetic etiology. This study aims to validate the  spectrum of DA in two novel strains of Fgf10 knock-out mice, in preparation for  future and translational research. Methods: Two novel CRISPR Fgf10 knock-out mouse  strains were derived and embryos generated by heterozygous plug-mating. E15.5-E19.5  embryos were genotyped with respect to Fgf10 and micro-dissected to determine the  presence and type of DA. Results: One twenty seven embryos (32 wild-type, 34  heterozygous, 61 null) were analyzed. No wild-type or heterozygous embryos had DA.  However, 74% of Fgf10 null embryos had DA (49% type 1, 18% type 2, and 33% type 3).  Conclusion: Our CRISPR-derived strains showed higher penetrance of DA due to  single-gene deletion of Fgf10 in mice than previously reported. Further, the DA type  distribution in these mice more closely reiterated that observed in humans. Future  experiments will document RNA and protein expression of FGF10 and its key downstream  signaling targets in normal and atretic duodenum. This includes exploitation of  modern, high-fidelity developmental tools, e.g., Fgf10 (flox/+)-tomato(flox/flox)  mice.\\nFAU - Teague, Warwick J\\nAU  - Teague WJ\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nAD  - Department of Paediatrics, The University of Melbourne, Melbourne, VIC, Australia.\\nAD  - Discipline of Surgery, Sydney Medical School, The University of Sydney, Sydney, NSW,  Australia.\\nAD  - Department of Paediatric Surgery, The Royal Children\\'s Hospital, Melbourne, VIC,  Australia.\\nFAU - Jones, Matthew L M\\nAU  - Jones MLM\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nAD  - Discipline of Surgery, Sydney Medical School, The University of Sydney, Sydney, NSW,  Australia.\\nAD  - Department of Paediatric Surgery, The Royal Children\\'s Hospital, Melbourne, VIC,  Australia.\\nFAU - Hawkey, Leanne\\nAU  - Hawkey L\\nAD  - Australian Phenomics Network, Department of Anatomy and Developmental Biology,  Monash University, Melbourne, VIC, Australia.\\nFAU - Smyth, Ian M\\nAU  - Smyth IM\\nAD  - Australian Phenomics Network, Department of Anatomy and Developmental Biology,  Monash University, Melbourne, VIC, Australia.\\nAD  - Department of Anatomy and Developmental Biology, Monash Biomedicine Discovery  Institute, Monash University, Melbourne, VIC, Australia.\\nAD  - Department of Biochemistry and Molecular Biology, Monash Biomedicine Discovery  Institute, Monash University, Melbourne, VIC, Australia.\\nFAU - Catubig, Angelique\\nAU  - Catubig A\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nFAU - King, Sebastian K\\nAU  - King SK\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nAD  - Department of Paediatrics, The University of Melbourne, Melbourne, VIC, Australia.\\nAD  - Department of Paediatric Surgery, The Royal Children\\'s Hospital, Melbourne, VIC,  Australia.\\nAD  - Department of Gastroenterology and Clinical Nutrition, The Royal Children\\'s  Hospital, Melbourne, VIC, Australia.\\nFAU - Sarila, Gulcan\\nAU  - Sarila G\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nFAU - Li, Ruili\\nAU  - Li R\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nFAU - Hutson, John M\\nAU  - Hutson JM\\nAD  - F. Douglas Stephens Surgical Research Laboratory, Murdoch Children\\'s Research  Institute, Melbourne, VIC, Australia.\\nAD  - Department of Paediatrics, The University of Melbourne, Melbourne, VIC, Australia.\\nAD  - Department of Urology, The Royal Children\\'s Hospital, Melbourne, VIC, Australia.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20181109\\nTA  - Front Genet\\nJT  - Frontiers in genetics\\nJID - 101560621\\nPMC - PMC6238159\\nOTO - NOTNLM\\nOT  - CRISPR-Cas systems\\nOT  - animal\\nOT  - congenital intestinal atresia\\nOT  - duodenal obstruction\\nOT  - fibroblast growth factor 10\\nOT  - models\\nOT  - morphogenesis\\nEDAT- 2018/11/27 06:00\\nMHDA- 2018/11/27 06:01\\nCRDT- 2018/11/27 06:00\\nPHST- 2018/08/24 00:00 [received]\\nPHST- 2018/10/22 00:00 [accepted]\\nPHST- 2018/11/27 06:00 [entrez]\\nPHST- 2018/11/27 06:00 [pubmed]\\nPHST- 2018/11/27 06:01 [medline]\\nAID - 10.3389/fgene.2018.00530 [doi]\\nPST - epublish\\nSO  - Front Genet. 2018 Nov 9;9:530. doi: 10.3389/fgene.2018.00530. eCollection 2018.\\n\\nPMID- 28765632\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190212\\nLR  - 20191220\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 7\\nIP  - 1\\nDP  - 2017 Aug 1\\nTI  - Efficient increase of É£-aminobutyric acid (GABA) content in tomato fruits by  targeted mutagenesis.\\nPG  - 7057\\nLID - 10.1038/s41598-017-06400-y [doi]\\nLID - 7057\\nAB  - Î³-Aminobutyric acid (GABA) is a non-proteinogenic amino acid that has hypotensive  effects. Tomato (Solanum lycopersicum L.) is among the most widely cultivated and  consumed vegetables in the world and contains higher levels of GABA than other major  crops. Increasing these levels can further enhance the blood pressure-lowering  function of tomato fruit. Glutamate decarboxylase (GAD) is a key enzyme in GABA  biosynthesis; it has a C-terminal autoinhibitory domain that regulates enzymatic  function, and deleting this domain increases GAD activity. The tomato genome has  five GAD genes (SlGAD1-5), of which two (SlGAD2 and SlGAD3) are expressed during  tomato fruit development. To increase GABA content in tomato, we deleted the  autoinhibitory domain of SlGAD2 and SlGAD3 using clustered regularly interspaced  short palindromic repeats (CRISPR)/CRISPR-associated protein (Cas)9 technology.  Introducing a stop codon immediately before the autoinhibitory domain increased GABA  accumulation by 7 to 15 fold while having variable effects on plant and fruit size  and yield. This is the first study describing the application of the CRISPR/Cas9  system to increase GABA content in tomato fruits. Our findings provide a basis for  the improvement of other types of crop by CRISPR/Cas9-based genetic modification.\\nFAU - Nonaka, Satoko\\nAU  - Nonaka S\\nAD  - Faculty of Life and Environmental Sciences, University of Tsukuba, 1-1-1 Tennodai,  Tsukuba, Ibaraki, 305-8572, Japan.\\nAD  - Gene Research Center, University of Tsukuba, 1-1-1 Tennodai, Tsukuba, Ibaraki,  305-8572, Japan.\\nFAU - Arai, Chikako\\nAU  - Arai C\\nAD  - Graduate School of Life and Environmental Sciences, University of Tsukuba, 1-1-1  Tennodai, Tsukuba, Ibaraki, 305-8572, Japan.\\nFAU - Takayama, Mariko\\nAU  - Takayama M\\nAD  - Faculty of Life and Environmental Sciences, University of Tsukuba, 1-1-1 Tennodai,  Tsukuba, Ibaraki, 305-8572, Japan.\\nFAU - Matsukura, Chiaki\\nAU  - Matsukura C\\nAD  - Faculty of Life and Environmental Sciences, University of Tsukuba, 1-1-1 Tennodai,  Tsukuba, Ibaraki, 305-8572, Japan.\\nAD  - Gene Research Center, University of Tsukuba, 1-1-1 Tennodai, Tsukuba, Ibaraki,  305-8572, Japan.\\nFAU - Ezura, Hiroshi\\nAU  - Ezura H\\nAD  - Faculty of Life and Environmental Sciences, University of Tsukuba, 1-1-1 Tennodai,  Tsukuba, Ibaraki, 305-8572, Japan. ezura.hiroshi.fa@u.tsukuba.ac.jp.\\nAD  - Gene Research Center, University of Tsukuba, 1-1-1 Tennodai, Tsukuba, Ibaraki,  305-8572, Japan. ezura.hiroshi.fa@u.tsukuba.ac.jp.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170801\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nRN  - 56-12-2 (gamma-Aminobutyric Acid)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nRN  - EC 4.1.1.15 (Glutamate Decarboxylase)\\nRN  - EC 4.1.1.15 (glutamate decarboxylase 1)\\nRN  - EC 4.1.1.15 (glutamate decarboxylase 2)\\nSB  - IM\\nEIN - Sci Rep. 2019 Dec 19;9(1):19822. PMID: 31852913\\nMH  - CRISPR-Associated Protein 9\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Fruit/enzymology/genetics/metabolism\\nMH  - Gene Editing\\nMH  - Glutamate Decarboxylase/genetics/*metabolism\\nMH  - Lycopersicon esculentum/enzymology/genetics/*metabolism\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - *Mutagenesis\\nMH  - *Sequence Deletion\\nMH  - gamma-Aminobutyric Acid/*metabolism\\nPMC - PMC5539196\\nCOIS- The authors declare that they have no competing interests.\\nEDAT- 2017/08/03 06:00\\nMHDA- 2019/02/13 06:00\\nCRDT- 2017/08/03 06:00\\nPHST- 2017/03/22 00:00 [received]\\nPHST- 2017/06/09 00:00 [accepted]\\nPHST- 2017/08/03 06:00 [entrez]\\nPHST- 2017/08/03 06:00 [pubmed]\\nPHST- 2019/02/13 06:00 [medline]\\nAID - 10.1038/s41598-017-06400-y [pii]\\nAID - 6400 [pii]\\nAID - 10.1038/s41598-017-06400-y [doi]\\nPST - epublish\\nSO  - Sci Rep. 2017 Aug 1;7(1):7057. doi: 10.1038/s41598-017-06400-y.\\n\\nPMID- 27479602\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170822\\nLR  - 20201119\\nIS  - 1932-6203 (Electronic)\\nIS  - 1932-6203 (Linking)\\nVI  - 11\\nIP  - 8\\nDP  - 2016\\nTI  - Mic13 Is Essential for Formation of Crista Junctions in Mammalian Cells.\\nPG  - e0160258\\nLID - 10.1371/journal.pone.0160258 [doi]\\nLID - e0160258\\nAB  - Mitochondrial cristae are connected to the inner boundary membrane via crista  junctions which are implicated in the regulation of oxidative phosphorylation,  apoptosis, and import of lipids and proteins. The MICOS complex determines formation  of crista junctions. We performed complexome profiling and identified Mic13, also  termed Qil1, as a subunit of the MICOS complex. We show that MIC13 is an inner  membrane protein physically interacting with MIC60, a central subunit of the MICOS  complex. Using the CRISPR/Cas method we generated the first cell line deleted for  MIC13. These knockout cells show a complete loss of crista junctions demonstrating  that MIC13 is strictly required for the formation of crista junctions. MIC13 is  required for the assembly of MIC10, MIC26, and MIC27 into the MICOS complex.  However, it is not needed for the formation of the MIC60/MIC19/MIC25 subcomplex  suggesting that the latter is not sufficient for crista junction formation. MIC13 is  also dispensable for assembly of respiratory chain complexes and for maintaining  mitochondrial network morphology. Still, lack of MIC13 resulted in a moderate  reduction of mitochondrial respiration. In summary, we show that MIC13 has a  fundamental role in crista junction formation and that assembly of respiratory chain  supercomplexes is independent of mitochondrial cristae shape.\\nFAU - Anand, Ruchika\\nAU  - Anand R\\nAD  - Institute of Biochemistry and Molecular Biology I, Heinrich Heine University,  Medical Faculty, UniversitÃ¤tsstr. 1, 40225, DÃ¼sseldorf, Germany.\\nFAU - Strecker, Valentina\\nAU  - Strecker V\\nAD  - Functional Proteomics, SFB 815 Core Unit, Faculty of Medicine, Goethe-University,  Frankfurt am Main, Germany.\\nFAU - Urbach, Jennifer\\nAU  - Urbach J\\nAD  - Institute of Biochemistry and Molecular Biology I, Heinrich Heine University,  Medical Faculty, UniversitÃ¤tsstr. 1, 40225, DÃ¼sseldorf, Germany.\\nFAU - Wittig, Ilka\\nAU  - Wittig I\\nAD  - Functional Proteomics, SFB 815 Core Unit, Faculty of Medicine, Goethe-University,  Frankfurt am Main, Germany.\\nAD  - Cluster of Excellence \"Macromolecular Complexes\", Goethe University, Frankfurt am  Main, Germany.\\nAD  - German Center of Cardiovascular Research (DZHK), Partner site RheinMain, Frankfurt,  Germany.\\nFAU - Reichert, Andreas S\\nAU  - Reichert AS\\nAD  - Institute of Biochemistry and Molecular Biology I, Heinrich Heine University,  Medical Faculty, UniversitÃ¤tsstr. 1, 40225, DÃ¼sseldorf, Germany.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20160801\\nTA  - PLoS One\\nJT  - PloS one\\nJID - 101285081\\nRN  - 0 (MICOS13 protein, human)\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Mitochondrial Proteins)\\nRN  - 0 (RNA, Small Interfering)\\nSB  - IM\\nMH  - Amino Acid Sequence\\nMH  - Animals\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - HEK293 Cells\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Membrane Proteins/chemistry/deficiency/genetics/*metabolism\\nMH  - Microscopy, Electron\\nMH  - Microscopy, Fluorescence\\nMH  - Mitochondrial Membranes/*metabolism\\nMH  - Mitochondrial Proteins/chemistry/deficiency/genetics/*metabolism\\nMH  - Molecular Sequence Data\\nMH  - Oxidative Phosphorylation\\nMH  - RNA Interference\\nMH  - RNA, Small Interfering/metabolism\\nMH  - Sequence Alignment\\nPMC - PMC4968808\\nCOIS- Competing Interests: The authors have declared that no competing interests exist.\\nEDAT- 2016/08/02 06:00\\nMHDA- 2017/08/23 06:00\\nCRDT- 2016/08/02 06:00\\nPHST- 2016/04/25 00:00 [received]\\nPHST- 2016/07/15 00:00 [accepted]\\nPHST- 2016/08/02 06:00 [entrez]\\nPHST- 2016/08/02 06:00 [pubmed]\\nPHST- 2017/08/23 06:00 [medline]\\nAID - PONE-D-16-16695 [pii]\\nAID - 10.1371/journal.pone.0160258 [doi]\\nPST - epublish\\nSO  - PLoS One. 2016 Aug 1;11(8):e0160258. doi: 10.1371/journal.pone.0160258. eCollection  2016.\\n\\nPMID- 34691834\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20211026\\nIS  - 2053-714X (Electronic)\\nIS  - 2095-5138 (Print)\\nIS  - 2053-714X (Linking)\\nVI  - 6\\nIP  - 1\\nDP  - 2019 Jan\\nTI  - BMAL1 knockout macaque monkeys display reduced sleep and psychiatric disorders.\\nPG  - 87-100\\nLID - 10.1093/nsr/nwz002 [doi]\\nAB  - Circadian disruption is a risk factor for metabolic, psychiatric and age-related  disorders, and non-human primate models could help to develop therapeutic  treatments. Here, we report the generation of BMAL1 knockout cynomolgus monkeys for  circadian-related disorders by CRISPR/Cas9 editing of monkey embryos. These monkeys  showed higher nocturnal locomotion and reduced sleep, which was further exacerbated  by a constant light regimen. Physiological circadian disruption was reflected by the  markedly dampened and arrhythmic blood hormonal levels. Furthermore, BMAL1-deficient  monkeys exhibited anxiety and depression, consistent with their stably elevated  blood cortisol, and defective sensory processing in auditory oddball tests found in  schizophrenia patients. Ablation of BMAL1 up-regulated transcriptional programs  toward inflammatory and stress responses, with transcription networks associated  with human sleep deprivation, major depressive disorders, and aging. Thus, BMAL1  knockout monkeys are potentially useful for studying the physiological consequences  of circadian disturbance, and for developing therapies for circadian and psychiatric  disorders.\\nCI  - Â© The Author(s) 2019. Published by Oxford University Press on behalf of China  Science Publishing & Media Ltd.\\nFAU - Qiu, Peiyuan\\nAU  - Qiu P\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Jiang, Jian\\nAU  - Jiang J\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Dynamic Brain Signal Analysis Facility, Institute of Neuroscience, Chinese Academy  of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Liu, Zhen\\nAU  - Liu Z\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Cai, Yijun\\nAU  - Cai Y\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Huang, Tao\\nAU  - Huang T\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Wang, Yan\\nAU  - Wang Y\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Liu, Qiming\\nAU  - Liu Q\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Nie, Yanhong\\nAU  - Nie Y\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Liu, Fang\\nAU  - Liu F\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049,  China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Cheng, Jiumu\\nAU  - Cheng J\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Li, Qing\\nAU  - Li Q\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Tang, Yun-Chi\\nAU  - Tang YC\\nAD  - CAS Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of  Nutrition and Health, Shanghai Institutes for Biological Sciences, University of  Chinese Academy of Sciences, Shanghai 200031, China.\\nFAU - Poo, Mu-Ming\\nAU  - Poo MM\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Sun, Qiang\\nAU  - Sun Q\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nFAU - Chang, Hung-Chun\\nAU  - Chang HC\\nAUID- ORCID: 0000-0003-0030-8323\\nAD  - Institute of Neuroscience, State Key Laboratory of Neuroscience, CAS Key Laboratory  of Primate Neurobiology, CAS Center for Excellence in Brain Science and Intelligence  Technology, Chinese Academy of Sciences, Shanghai 200031, China.\\nAD  - Dynamic Brain Signal Analysis Facility, Institute of Neuroscience, Chinese Academy  of Sciences, Shanghai 200031, China.\\nAD  - Shanghai Research Center for Brain Science and Brain-inspired Technology, Shanghai  200031, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20190124\\nTA  - Natl Sci Rev\\nJT  - National science review\\nJID - 101633095\\nCIN - doi: 10.1093/nsr/nwz011\\nPMC - PMC8291534\\nOTO - NOTNLM\\nOT  - BMAL1\\nOT  - aging\\nOT  - circadian rhythms\\nOT  - macaque monkey\\nOT  - psychosis\\nOT  - sleep disruption\\nEDAT- 2019/01/01 00:00\\nMHDA- 2019/01/01 00:01\\nCRDT- 2021/10/25 06:33\\nPHST- 2018/12/03 00:00 [received]\\nPHST- 2018/12/30 00:00 [revised]\\nPHST- 2019/01/05 00:00 [accepted]\\nPHST- 2021/10/25 06:33 [entrez]\\nPHST- 2019/01/01 00:00 [pubmed]\\nPHST- 2019/01/01 00:01 [medline]\\nAID - nwz002 [pii]\\nAID - 10.1093/nsr/nwz002 [doi]\\nPST - ppublish\\nSO  - Natl Sci Rev. 2019 Jan;6(1):87-100. doi: 10.1093/nsr/nwz002. Epub 2019 Jan 24.\\n\\nPMID- 33752238\\nOWN - NLM\\nSTAT- In-Process\\nLR  - 20220212\\nIS  - 1532-2548 (Electronic)\\nIS  - 0032-0889 (Print)\\nIS  - 0032-0889 (Linking)\\nVI  - 186\\nIP  - 3\\nDP  - 2021 Jul 6\\nTI  - GRAS-domain transcription factor PAT1 regulates jasmonic acid biosynthesis in grape  cold stress response.\\nPG  - 1660-1678\\nLID - 10.1093/plphys/kiab142 [doi]\\nAB  - Cultivated grapevine (Vitis) is a highly valued horticultural crop, and cold stress  affects its growth and productivity. Wild Amur grape (Vitis amurensis) PAT1  (Phytochrome A signal transduction 1, VaPAT1) is induced by low temperature, and  ectopic expression of VaPAT1 enhances cold tolerance in Arabidopsis (Arabidopsis  thaliana). However, little is known about the molecular mechanism of VaPAT1 during  the cold stress response in grapevine. Here, we confirmed the overexpression of  VaPAT1 in transformed grape calli enhanced cold tolerance. Yeast two-hybrid and  bimolecular fluorescence complementation assays highlighted an interaction between  VaPAT1 with INDETERMINATE-DOMAIN 3 (VaIDD3). A role of VaIDD3 in cold tolerance was  also indicated. Transcriptome analysis revealed VaPAT1 and VaIDD3 overexpression and  cold treatment coordinately modulate the expression of stress-related genes  including lipoxygenase 3 (LOX3), a gene encoding a key jasmonate biosynthesis  enzyme. Co-expression network analysis indicated LOX3 might be a downstream target  of VaPAT1. Both electrophoretic mobility shift and dual luciferase reporter assays  showed the VaPAT1-IDD3 complex binds to the IDD-box (AGACAAA) in the VaLOX3 promoter  to activate its expression. Overexpression of both VaPAT1 and VaIDD3 increased the  transcription of VaLOX3 and JA levels in transgenic grape calli. Conversely,  VaPAT1-SRDX (dominant repression) and CRISPR/Cas9-mediated mutagenesis of PAT1-ED  causing the loss of the C-terminus in grape calli dramatically prohibited the  accumulation of VaLOX3 and JA levels during cold treatment. Together, these findings  point to a pivotal role of VaPAT1 in the cold stress response in grape by regulating  JA biosynthesis.\\nCI  - Â© American Society of Plant Biologists 2021. All rights reserved. For permissions,  please email: journals.permissions@oup.com.\\nFAU - Wang, Zemin\\nAU  - Wang Z\\nAUID- ORCID: 0000-0001-7936-5229\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 10049, China.\\nFAU - Wong, Darren Chern Jan\\nAU  - Wong DCJ\\nAD  - Department of Ecology and Evolution, Research School of Biology, Australian National  University, Acton, ACT 2601, Australia.\\nFAU - Wang, Yi\\nAU  - Wang Y\\nAUID- ORCID: 0000-0002-4534-8946\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 10049, China.\\nFAU - Xu, Guangzhao\\nAU  - Xu G\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 10049, China.\\nFAU - Ren, Chong\\nAU  - Ren C\\nAUID- ORCID: 0000-0001-9286-8031\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nFAU - Liu, Yanfei\\nAU  - Liu Y\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 10049, China.\\nFAU - Kuang, Yangfu\\nAU  - Kuang Y\\nAUID- ORCID: 0000-0002-6460-5522\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - University of Chinese Academy of Sciences, Beijing 10049, China.\\nFAU - Fan, Peige\\nAU  - Fan P\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nAD  - China Wine Industry Technology Institute, Yinchuan 750021, China.\\nFAU - Li, Shaohua\\nAU  - Li S\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nFAU - Xin, Haiping\\nAU  - Xin H\\nAD  - Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan  Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China.\\nAD  - Center of Economic Botany, Core Botanical Gardens, Chinese academy of Sciences,  Wuhan 430074, China.\\nFAU - Liang, Zhenchang\\nAU  - Liang Z\\nAUID- ORCID: 0000-0002-4644-4454\\nAD  - Beijing Key Laboratory of Grape Science and Enology, and CAS Key Laboratory of Plant  Resources, Institute of Botany, Innovation Academy for Seed Design, the Chinese  Academy of Science, Beijing 100093, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nTA  - Plant Physiol\\nJT  - Plant physiology\\nJID - 0401224\\nSB  - IM\\nPMC - PMC8260143\\nEDAT- 2021/03/23 06:00\\nMHDA- 2021/03/23 06:00\\nPMCR- 2022/03/22\\nCRDT- 2021/03/22 20:29\\nPHST- 2020/09/09 00:00 [received]\\nPHST- 2021/03/01 00:00 [accepted]\\nPHST- 2022/03/22 00:00 [pmc-release]\\nPHST- 2021/03/23 06:00 [pubmed]\\nPHST- 2021/03/23 06:00 [medline]\\nPHST- 2021/03/22 20:29 [entrez]\\nAID - 6180079 [pii]\\nAID - kiab142 [pii]\\nAID - 10.1093/plphys/kiab142 [doi]\\nPST - ppublish\\nSO  - Plant Physiol. 2021 Jul 6;186(3):1660-1678. doi: 10.1093/plphys/kiab142.\\n\\nPMID- 31315906\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200224\\nLR  - 20200309\\nIS  - 1549-5469 (Electronic)\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 29\\nIP  - 9\\nDP  - 2019 Sep\\nTI  - Profiling the long noncoding RNA interaction network in the regulatory elements of  target genes by chromatin in situ reverse transcription sequencing.\\nPG  - 1521-1532\\nLID - 10.1101/gr.244996.118 [doi]\\nAB  - Long noncoding RNAs (lncRNAs) can regulate the activity of target genes by  participating in the organization of chromatin architecture. We have devised a  \"chromatin-RNA in situ reverse transcription sequencing\" (CRIST-seq) approach to  profile the lncRNA interaction network in gene regulatory elements by combining the  simplicity of RNA biotin labeling with the specificity of the CRISPR/Cas9 system.  Using gene-specific gRNAs, we describe a pluripotency-specific lncRNA interacting  network in the promoters of Sox2 and Pou5f1, two critical stem cell factors that are  required for the maintenance of pluripotency. The promoter-interacting lncRNAs were  specifically activated during reprogramming into pluripotency. Knockdown of these  lncRNAs caused the stem cells to exit from pluripotency. In contrast, overexpression  of the pluripotency-associated lncRNA activated the promoters of core stem cell  factor genes and enhanced fibroblast reprogramming into pluripotency. These  CRIST-seq data suggest that the Sox2 and Pou5f1 promoters are organized within a  unique lncRNA interaction network that determines the fate of pluripotency during  reprogramming. This CRIST approach may be broadly used to map lncRNA interaction  networks at target loci across the genome.\\nCI  - Â© 2019 Zhang et al.; Published by Cold Spring Harbor Laboratory Press.\\nFAU - Zhang, Shilin\\nAU  - Zhang S\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Wang, Yichen\\nAU  - Wang Y\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Jia, Lin\\nAU  - Jia L\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Wen, Xue\\nAU  - Wen X\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Du, Zhonghua\\nAU  - Du Z\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Wang, Cong\\nAU  - Wang C\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nAD  - Stanford University Medical School, VA Palo Alto Health Care System, Palo Alto,  California 94304, USA.\\nFAU - Hao, Yajing\\nAU  - Hao Y\\nAD  - CAS Key Laboratory of RNA Biology, Institute of Biophysics, Chinese Academy of  Sciences, Beijing 100101, P.R. China.\\nFAU - Yu, Dehai\\nAU  - Yu D\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Zhou, Lei\\nAU  - Zhou L\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Chen, Naifei\\nAU  - Chen N\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Chen, Jingcheng\\nAU  - Chen J\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nAD  - Stanford University Medical School, VA Palo Alto Health Care System, Palo Alto,  California 94304, USA.\\nFAU - Chen, Huiling\\nAU  - Chen H\\nAD  - Department of Endocrinology, Xiangya Hospital, Central South University, Changsha,  Hunan 410008, P.R. China.\\nFAU - Zhang, Hui\\nAU  - Zhang H\\nAD  - Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences,  Guangzhou, Guangdong 510530, P.R. China.\\nFAU - Celik, Ilkay\\nAU  - Celik I\\nAD  - Stanford University Medical School, VA Palo Alto Health Care System, Palo Alto,  California 94304, USA.\\nFAU - GÃ¼lsoy, GÃ¼nhan\\nAU  - GÃ¼lsoy G\\nAD  - Google Incorporated, Mountain View, California 94043, USA.\\nFAU - Luo, Jianjun\\nAU  - Luo J\\nAD  - CAS Key Laboratory of RNA Biology, Institute of Biophysics, Chinese Academy of  Sciences, Beijing 100101, P.R. China.\\nFAU - Qin, Baoming\\nAU  - Qin B\\nAD  - Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences,  Guangzhou, Guangdong 510530, P.R. China.\\nFAU - Cui, Xueling\\nAU  - Cui X\\nAD  - Department of Immunology, College of Basic Medical Sciences, Jilin University,  Changchun, Jilin 130021, P.R. China.\\nFAU - Liu, Zhonghui\\nAU  - Liu Z\\nAD  - Department of Immunology, College of Basic Medical Sciences, Jilin University,  Changchun, Jilin 130021, P.R. China.\\nFAU - Zhang, Songling\\nAU  - Zhang S\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Esteban, Miguel A\\nAU  - Esteban MA\\nAD  - Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences,  Guangzhou, Guangdong 510530, P.R. China.\\nFAU - Ay, Ferhat\\nAU  - Ay F\\nAD  - La Jolla Institute for Allergy and Immunology, La Jolla, California 92037, USA.\\nFAU - Xu, Wei\\nAU  - Xu W\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Chen, Runsheng\\nAU  - Chen R\\nAD  - CAS Key Laboratory of RNA Biology, Institute of Biophysics, Chinese Academy of  Sciences, Beijing 100101, P.R. China.\\nFAU - Li, Wei\\nAU  - Li W\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nFAU - Hoffman, Andrew R\\nAU  - Hoffman AR\\nAD  - Stanford University Medical School, VA Palo Alto Health Care System, Palo Alto,  California 94304, USA.\\nFAU - Hu, Ji-Fan\\nAU  - Hu JF\\nAUID- ORCID: 0000-0002-2174-0361\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nAD  - Stanford University Medical School, VA Palo Alto Health Care System, Palo Alto,  California 94304, USA.\\nFAU - Cui, Jiuwei\\nAU  - Cui J\\nAD  - Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education,  Cancer Center, The First Hospital of Jilin University, Changchun, Jilin 130061, P.R.  China.\\nLA  - eng\\nGR  - I01 BX002905/BX/BLRD VA/United States\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20190717\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (Chromatin)\\nRN  - 0 (Octamer Transcription Factor-3)\\nRN  - 0 (Pou5f1 protein, mouse)\\nRN  - 0 (RNA, Long Noncoding)\\nRN  - 0 (SOXB1 Transcription Factors)\\nRN  - 0 (Sox2 protein, mouse)\\nSB  - IM\\nMH  - Animals\\nMH  - CRISPR-Cas Systems\\nMH  - Cell Line\\nMH  - Cellular Reprogramming\\nMH  - Chromatin/*genetics\\nMH  - Fibroblasts/cytology/metabolism\\nMH  - Mice\\nMH  - Octamer Transcription Factor-3/*genetics\\nMH  - Pluripotent Stem Cells/cytology/metabolism\\nMH  - Promoter Regions, Genetic\\nMH  - RNA, Long Noncoding/*genetics\\nMH  - Regulatory Sequences, Nucleic Acid\\nMH  - SOXB1 Transcription Factors/*genetics\\nMH  - Sequence Analysis, RNA/*methods\\nPMC - PMC6724666\\nEDAT- 2019/07/19 06:00\\nMHDA- 2020/02/25 06:00\\nCRDT- 2019/07/19 06:00\\nPHST- 2018/10/10 00:00 [received]\\nPHST- 2019/07/10 00:00 [accepted]\\nPHST- 2019/07/19 06:00 [pubmed]\\nPHST- 2020/02/25 06:00 [medline]\\nPHST- 2019/07/19 06:00 [entrez]\\nAID - gr.244996.118 [pii]\\nAID - 10.1101/gr.244996.118 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2019 Sep;29(9):1521-1532. doi: 10.1101/gr.244996.118. Epub 2019 Jul 17.\\n\\nPMID- 29433437\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180820\\nLR  - 20181113\\nIS  - 1471-2164 (Electronic)\\nIS  - 1471-2164 (Linking)\\nVI  - 19\\nIP  - 1\\nDP  - 2018 Feb 13\\nTI  - Genomic insights into Staphylococcus equorum KS1039 as a potential starter culture  for the fermentation of high-salt foods.\\nPG  - 136\\nLID - 10.1186/s12864-018-4532-1 [doi]\\nLID - 136\\nAB  - BACKGROUND: Our previous comparative genomic analysis of Staphylococcus equorum  KS1039 with five S. equorum strains illuminated the genomic basis of its safety and  salt tolerance. However, a comprehensive picture of the cellular components and  metabolic pathways involved in the degradation of macromolecules and development of  sensory properties has not been obtained for S. equorum. Therefore, in this study,  we examined the general metabolism of S. equorum based on information obtained from  published complete genome sequences of six S. equorum strains isolated from  different niches. Additionally, the utility of strain KS1039 as a starter culture  for high-salt food fermentations was examined. RESULTS: All six S. equorum strains  contained genes involved in glycolysis, the tricarboxylic acid cycle, and amino acid  metabolic pathways, as well as color development. Moreover, the strains had the  potential to produce acetoin, butanediol, and branched chain fatty acids, all of  which are important flavor compounds. None of the strains contained decarboxylase  genes, which are required for histamine and tyramine production. Strain KS1039  contained bacteriocin and CRISPR/Cas gene clusters, and experimental results  suggested that these genes were functional in vitro. CONCLUSIONS: The comparative  genomic analysis carried out herein provides important information on the usefulness  of S. equorum KS1039 as a starter culture for the fermentation of high-salt foods in  terms of safety, salt tolerance, bacteriocin production, and foreign plasmid  restriction.\\nFAU - Lee, Jong-Hoon\\nAU  - Lee JH\\nAD  - Department of Food Science and Biotechnology, Kyonggi University, Suwon, 16227,  Republic of Korea.\\nFAU - Heo, Sojeong\\nAU  - Heo S\\nAD  - Department of Food Science and Biotechnology, Kyonggi University, Suwon, 16227,  Republic of Korea.\\nAD  - Department of Food and Nutrition, Dongduk Women\\'s University, Seoul, 02748, Republic  of Korea.\\nFAU - Jeong, Do-Won\\nAU  - Jeong DW\\nAUID- ORCID: 0000-0001-6586-748X\\nAD  - Department of Food and Nutrition, Dongduk Women\\'s University, Seoul, 02748, Republic  of Korea. jeongdw@dongduk.ac.kr.\\nLA  - eng\\nGR  - NRF-2016R1D1A1B01011421/National Research Foundation of Korea/International\\nGR  - NRF 2016R1D1A1B03930239/National Research Foundation of Korea/International\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180213\\nTA  - BMC Genomics\\nJT  - BMC genomics\\nJID - 100965258\\nRN  - 0 (Bacteriocins)\\nRN  - 0 (Flavoring Agents)\\nSB  - IM\\nMH  - Bacteriocins/genetics/metabolism\\nMH  - Energy Metabolism/genetics\\nMH  - *Fermentation\\nMH  - Flavoring Agents/metabolism\\nMH  - Food Microbiology/*methods\\nMH  - Genes, Bacterial/genetics\\nMH  - Genome, Bacterial/*genetics\\nMH  - Genomics/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Species Specificity\\nMH  - Staphylococcus/classification/*genetics/metabolism\\nPMC - PMC5810056\\nOTO - NOTNLM\\nOT  - *Bacteriocin\\nOT  - *CRISPR/Cas system\\nOT  - *Fermentation\\nOT  - *Jeotgal\\nOT  - *Staphylococcus equorum\\nOT  - *Starter\\nCOIS- ETHICS APPROVAL AND CONSENT TO PARTICIPATE: Not applicable. No human or animal  material was directly used in this study. CONSENT FOR PUBLICATION: Not applicable.  COMPETING INTERESTS: The authors declare that they have no competing interests.  PUBLISHERâ€™S NOTE: Springer Nature remains neutral with regard to jurisdictional  claims in published maps and institutional affiliations.\\nEDAT- 2018/02/13 06:00\\nMHDA- 2018/08/21 06:00\\nCRDT- 2018/02/14 06:00\\nPHST- 2017/05/04 00:00 [received]\\nPHST- 2018/02/05 00:00 [accepted]\\nPHST- 2018/02/14 06:00 [entrez]\\nPHST- 2018/02/13 06:00 [pubmed]\\nPHST- 2018/08/21 06:00 [medline]\\nAID - 10.1186/s12864-018-4532-1 [pii]\\nAID - 4532 [pii]\\nAID - 10.1186/s12864-018-4532-1 [doi]\\nPST - epublish\\nSO  - BMC Genomics. 2018 Feb 13;19(1):136. doi: 10.1186/s12864-018-4532-1.\\n\\nPMID- 27520814\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170922\\nLR  - 20181113\\nIS  - 1098-5336 (Electronic)\\nIS  - 0099-2240 (Print)\\nIS  - 0099-2240 (Linking)\\nVI  - 82\\nIP  - 20\\nDP  - 2016 Oct 15\\nTI  - Understanding the Streptococcus mutans Cid/Lrg System through CidB Function.\\nPG  - 6189-6203\\nAB  - The Streptococcus mutans lrgAB and cidAB operons have been previously described as a  potential model system to dissect the complexity of biofilm development and  virulence of S. mutans Herein, we have attempted to further characterize the Cid/Lrg  system by focusing on CidB, which has been shown to be critical for the ability of  S. mutans to survive and persist in a nonpreferred oxygen-enriched condition. We  have found that the expression level of cidB is critical to oxidative stress  tolerance of S. mutans, most likely by impacting lrg expression. Intriguingly, the  impaired aerobic growth phenotype of the cidB mutant could be restored by the  additional loss of either CidA or LrgA. Growth-dependent expression of cid and lrg  was demonstrated to be tightly under the control of both CcpA and the VicKR  two-component system (TCS), regulators known to play an essential role in  controlling major catabolic pathways and cell envelope homeostasis, respectively.  RNA sequencing (RNA-Seq) analysis revealed that mutation of cidB resulted in global  gene expression changes, comprising major domains of central metabolism and  virulence processes, particularly in those involved with oxidative stress  resistance. Loss of CidB also significantly changed the expression of genes related  to genomic islands (GI) TnSmu1 and TnSmu2, the CRISPR (clustered regularly  interspaced short palindromic repeats)-Cas system, and toxin-antitoxin (T/A)  modules. Taken together, these data show that CidB impinges on the stress response,  as well as the fundamental cellular physiology of S. mutans, and further suggest a  potential link between Cid/Lrg-mediated cellular processes, S. mutans pathogenicity,  and possible programmed growth arrest and cell death mechanisms. IMPORTANCE: The  ability of Streptococcus mutans to survive a variety of harmful or stressful  conditions and to emerge as a numerically significant member of stable oral biofilm  communities are essential elements for its persistence and cariogenicity. In this  study, the homologous cidAB and lrgAB operons, previously identified as being highly  balanced and coordinated during S. mutans aerobic growth, were further characterized  through the functional and transcriptomic analysis of CidB. Precise control of CidB  levels is shown to impact the expression of lrg, oxidative stress tolerance, major  metabolic domains, and the molecular modules linked to cell death and lysis. This  study advances our understanding of the Cid/Lrg system as a key player in the  integration of complex environmental signals (such as oxidative stress) into the  regulatory networks that modulate S. mutans virulence and cell homeostasis.\\nCI  - Copyright Â© 2016, American Society for Microbiology. All Rights Reserved.\\nFAU - Ahn, Sang-Joon\\nAU  - Ahn SJ\\nAD  - Department of Oral Biology, College of Dentistry, University of Florida,  Gainesville, Florida, USA sahn@dental.ufl.edu.\\nFAU - Rice, Kelly C\\nAU  - Rice KC\\nAD  - Department of Microbiology and Cell Science, Institute of Food and Agricultural  Sciences, University of Florida, Gainesville, Florida, USA.\\nLA  - eng\\nGR  - R01 DE025237/DE/NIDCR NIH HHS/United States\\nGR  - R03 DE023604/DE/NIDCR NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20160930\\nTA  - Appl Environ Microbiol\\nJT  - Applied and environmental microbiology\\nJID - 7605801\\nRN  - 0 (Bacterial Proteins)\\nSB  - IM\\nMH  - Bacterial Proteins/genetics/*metabolism\\nMH  - Gene Expression Regulation, Bacterial\\nMH  - Operon\\nMH  - Oxidative Stress\\nMH  - Streptococcus mutans/genetics/*metabolism\\nMH  - Virulence\\nPMC - PMC5068163\\nEDAT- 2016/08/16 06:00\\nMHDA- 2017/09/25 06:00\\nCRDT- 2016/08/14 06:00\\nPHST- 2016/05/16 00:00 [received]\\nPHST- 2016/08/05 00:00 [accepted]\\nPHST- 2016/08/16 06:00 [pubmed]\\nPHST- 2017/09/25 06:00 [medline]\\nPHST- 2016/08/14 06:00 [entrez]\\nAID - AEM.01499-16 [pii]\\nAID - 01499-16 [pii]\\nAID - 10.1128/AEM.01499-16 [doi]\\nPST - epublish\\nSO  - Appl Environ Microbiol. 2016 Sep 30;82(20):6189-6203. doi: 10.1128/AEM.01499-16.  Print 2016 Oct 15.\\n\\nPMID- 28559295\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170726\\nLR  - 20181113\\nIS  - 1098-5530 (Electronic)\\nIS  - 0021-9193 (Print)\\nIS  - 0021-9193 (Linking)\\nVI  - 199\\nIP  - 15\\nDP  - 2017 Aug 1\\nTI  - Polyvalent Proteins, a Pervasive Theme in the Intergenomic Biological Conflicts of  Bacteriophages and Conjugative Elements.\\nLID - 10.1128/JB.00245-17 [doi]\\nLID - e00245-17\\nAB  - Intense biological conflicts between prokaryotic genomes and their genomic parasites  have resulted in an arms race in terms of the molecular \"weaponry\" deployed on both  sides. Using a recursive computational approach, we uncovered a remarkable class of  multidomain proteins with 2 to 15 domains in the same polypeptide deployed by  viruses and plasmids in such conflicts. Domain architectures and genomic contexts  indicate that they are part of a widespread conflict strategy involving proteins  injected into the host cell along with parasite DNA during the earliest phase of  infection. Their unique feature is the combination of domains with highly disparate  biochemical activities in the same polypeptide; accordingly, we term them polyvalent  proteins. Of the 131 domains in polyvalent proteins, a large fraction are enzymatic  domains predicted to modify proteins, target nucleic acids, alter nucleotide  signaling/metabolism, and attack peptidoglycan or cytoskeletal components. They  further contain nucleic acid-binding domains, virion structural domains, and 40  novel uncharacterized domains. Analysis of their architectural network reveals both  pervasive common themes and specialized strategies for conjugative elements and  plasmids or (pro)phages. The themes include likely processing of multidomain  polypeptides by zincin-like metallopeptidases and mechanisms to counter restriction  or CRISPR/Cas systems and jump-start transcription or replication. DNA-binding  domains acquired by eukaryotes from such systems have been reused in  XPC/RAD4-dependent DNA repair and mitochondrial genome replication in  kinetoplastids. Characterization of the novel domains discovered here, such as  RNases and peptidases, are likely to aid in the development of new reagents and  elucidation of the spread of antibiotic resistance.IMPORTANCE This is the first  report of the widespread presence of large proteins, termed polyvalent proteins,  predicted to be transmitted by genomic parasites such as conjugative elements,  plasmids, and phages during the initial phase of infection along with their DNA.  They are typified by the presence of multiple domains with disparate activities  combined in the same protein. While some of these domains are predicted to assist  the invasive element in replication, transcription, or protection of their DNA,  several are likely to target various host defense systems or modify the host to  favor the parasite\\'s life cycle. Notably, DNA-binding domains from these systems  have been transferred to eukaryotes, where they have been incorporated into DNA  repair and mitochondrial genome replication systems.\\nCI  - Copyright Â© 2017 Iyer et al.\\nFAU - Iyer, Lakshminarayan M\\nAU  - Iyer LM\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, Maryland, USA.\\nFAU - Burroughs, A Maxwell\\nAU  - Burroughs AM\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, Maryland, USA.\\nFAU - Anand, Swadha\\nAU  - Anand S\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, Maryland, USA.\\nFAU - de Souza, Robson F\\nAU  - de Souza RF\\nAD  - Microbiology Department, Biomedical Sciences Institute, University of Sao Paulo,  Cidade UniversitÃ¡ria, SÃ£o Paulo, Brazil.\\nFAU - Aravind, L\\nAU  - Aravind L\\nAD  - National Center for Biotechnology Information, National Library of Medicine,  National Institutes of Health, Bethesda, Maryland, USA aravind@ncbi.nlm.nih.gov.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20170711\\nTA  - J Bacteriol\\nJT  - Journal of bacteriology\\nJID - 2985120R\\nRN  - 0 (Peptides)\\nSB  - IM\\nMH  - Bacteriophages/*genetics\\nMH  - Computational Biology\\nMH  - Evolution, Molecular\\nMH  - Peptides/*genetics/*metabolism\\nMH  - *Plasmids\\nMH  - Protein Domains\\nPMC - PMC5512222\\nOTO - NOTNLM\\nOT  - *DNA replication\\nOT  - *DNA-binding proteins\\nOT  - *RNases\\nOT  - *antirestriction\\nOT  - *bacteriophages\\nOT  - *biological conflicts\\nOT  - *effectors\\nOT  - *metallopeptidase\\nOT  - *plasmids\\nOT  - *transcription\\nEDAT- 2017/06/01 06:00\\nMHDA- 2017/07/27 06:00\\nCRDT- 2017/06/01 06:00\\nPHST- 2017/04/04 00:00 [received]\\nPHST- 2017/05/17 00:00 [accepted]\\nPHST- 2017/06/01 06:00 [pubmed]\\nPHST- 2017/07/27 06:00 [medline]\\nPHST- 2017/06/01 06:00 [entrez]\\nAID - JB.00245-17 [pii]\\nAID - 00245-17 [pii]\\nAID - 10.1128/JB.00245-17 [doi]\\nPST - epublish\\nSO  - J Bacteriol. 2017 Jul 11;199(15):e00245-17. doi: 10.1128/JB.00245-17. Print 2017 Aug  1.\\n\\nPMID- 33265669\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201207\\nIS  - 1099-4300 (Electronic)\\nIS  - 1099-4300 (Linking)\\nVI  - 20\\nIP  - 8\\nDP  - 2018 Aug 7\\nTI  - Entropic Stabilization of Cas4 Protein SSO0001 Predicted with Popcoen.\\nLID - 10.3390/e20080580 [doi]\\nLID - 580\\nAB  - Popcoen is a method for configurational entropy estimation of proteins based on  machine-learning. Entropy is predicted with an artificial neural network which was  trained on simulation trajectories of a large set of representative proteins.  Popcoen is extremely fast compared to other approaches based on the sampling of a  multitude of microstates. Consequently, Popcoen can be incorporated into a large  class of protein software which currently neglects configurational entropy for  performance reasons. Here, we apply Popcoen to various conformations of the Cas4  protein SSO0001 of Sulfolobus solfataricus, a protein that assembles to a decamer of  known toroidal shape. We provide numerical evidence that the native state (NAT) of a  SSO0001 monomer has a similar structure to the protomers of the oligomer, where NAT  of the monomer is stabilized mainly entropically. Due to its large amount of  configurational entropy, NAT has lower free energy than alternative conformations of  very low enthalpy and solvation free-energy. Hence, SSO0001 serves as an example  case where neglecting configurational entropy leads to incorrect conclusion. Our  results imply that no refolding of the subunits is required during oligomerization  which suggests that configurational entropy is employed by nature to largely enhance  the rate of assembly.\\nFAU - Goethe, Martin\\nAU  - Goethe M\\nAUID- ORCID: 0000-0002-5826-2180\\nAD  - Department of Condensed Matter Physics, University of Barcelona, Carrer MartÃ\\xad i  FranquÃ¨s 1, 08028 Barcelona, Spain.\\nAD  - Department of Inorganic and Organic Chemistry, University of Barcelona, Carrer MartÃ\\xad  i FranquÃ¨s 1, 08028 Barcelona, Spain.\\nFAU - Fita, Ignacio\\nAU  - Fita I\\nAD  - Molecular Biology Institute of Barcelona (IBMB-CSIC, Maria de Maeztu Unit of  Excellence), Carrer Baldiri Reixac 4-8, 08028 Barcelona, Spain.\\nFAU - Rubi, J Miguel\\nAU  - Rubi JM\\nAD  - Department of Condensed Matter Physics, University of Barcelona, Carrer MartÃ\\xad i  FranquÃ¨s 1, 08028 Barcelona, Spain.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20180807\\nTA  - Entropy (Basel)\\nJT  - Entropy (Basel, Switzerland)\\nJID - 101243874\\nPMC - PMC7513108\\nOTO - NOTNLM\\nOT  - CRISPR/Cas system\\nOT  - configurational entropy\\nOT  - entropic stabilization\\nOT  - entropy estimation\\nOT  - oligomerization\\nOT  - protein conformations\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2018/08/07 00:00\\nMHDA- 2018/08/07 00:01\\nCRDT- 2020/12/03 01:04\\nPHST- 2018/06/28 00:00 [received]\\nPHST- 2018/07/27 00:00 [revised]\\nPHST- 2018/07/28 00:00 [accepted]\\nPHST- 2020/12/03 01:04 [entrez]\\nPHST- 2018/08/07 00:00 [pubmed]\\nPHST- 2018/08/07 00:01 [medline]\\nAID - e20080580 [pii]\\nAID - entropy-20-00580 [pii]\\nAID - 10.3390/e20080580 [doi]\\nPST - epublish\\nSO  - Entropy (Basel). 2018 Aug 7;20(8):580. doi: 10.3390/e20080580.\\n\\nPMID- 29663663\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20181022\\nLR  - 20181022\\nIS  - 1860-7314 (Electronic)\\nIS  - 1860-6768 (Linking)\\nVI  - 13\\nIP  - 9\\nDP  - 2018 Sep\\nTI  - Modular Ligation Extension of Guide RNA Operons (LEGO) for Multiplexed dCas9  Regulation of Metabolic Pathways in Saccharomyces cerevisiae.\\nPG  - e1700582\\nLID - 10.1002/biot.201700582 [doi]\\nAB  - Metabolic engineering typically utilizes a suboptimal step-wise gene target  optimization approach to parse a highly connected and regulated cellular metabolism.  While the endonuclease-null CRISPR/Cas system has enabled gene expression  perturbations without genetic modification, it has been mostly limited to small sets  of gene targets in eukaryotes due to inefficient methods to assemble and express  large sgRNA operons. In this work, we develop a TEF1p-tRNA expression system and  demonstrate that the use of tRNAs as splicing elements flanking sgRNAs provides  higher efficiency than both Pol III and ribozyme-based expression across a variety  of single sgRNA and multiplexed contexts. Next, we devise and validate a scheme to  allow modular construction of tRNA-sgRNA (TST) operons using an iterative Type IIs  digestion/ligation extension approach, termed CRISPR-Ligation Extension of sgRNA  Operons (LEGO). This approach enables facile construction of large TST operons. We  demonstrate this utility by constructing a metabolic rewiring prototype for  2,3-butanediol production in 2 distinct yeast strain backgrounds. These results  demonstrate that our approach can act as a surrogate for traditional genetic  modification on a much shorter design-cycle timescale.\\nCI  - Â© 2018 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\\nFAU - Deaner, Matthew\\nAU  - Deaner M\\nAD  - McKetta Department of Chemical Engineering, The University of Texas at Austin, 200 E  Dean Keeton St. Stop C0400, Austin, TX 78712.\\nFAU - Holzman, Allison\\nAU  - Holzman A\\nAD  - Institute for Cellular and Molecular Biology, The University of Texas at Austin,  2500 Speedway Avenue, Austin, TX 78712.\\nFAU - Alper, Hal S\\nAU  - Alper HS\\nAD  - McKetta Department of Chemical Engineering, The University of Texas at Austin, 200 E  Dean Keeton St. Stop C0400, Austin, TX 78712.\\nAD  - Institute for Cellular and Molecular Biology, The University of Texas at Austin,  2500 Speedway Avenue, Austin, TX 78712.\\nLA  - eng\\nGR  - FA9550-14-1-0089/Air Force Office of Scientific Research/\\nPT  - Journal Article\\nDEP - 20180429\\nPL  - Germany\\nTA  - Biotechnol J\\nJT  - Biotechnology journal\\nJID - 101265833\\nRN  - 0 (RNA, Guide)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/*genetics\\nMH  - Operon/genetics\\nMH  - RNA, Guide/*genetics\\nMH  - Saccharomyces cerevisiae/*genetics/metabolism\\nOTO - NOTNLM\\nOT  - CRISPR/dCas9\\nOT  - Type IIs synthesis\\nOT  - gene multiplexing\\nOT  - metabolic engineering\\nOT  - tRNA splicing\\nEDAT- 2018/04/18 06:00\\nMHDA- 2018/10/23 06:00\\nCRDT- 2018/04/18 06:00\\nPHST- 2018/02/27 00:00 [received]\\nPHST- 2018/03/23 00:00 [revised]\\nPHST- 2018/04/18 06:00 [pubmed]\\nPHST- 2018/10/23 06:00 [medline]\\nPHST- 2018/04/18 06:00 [entrez]\\nAID - 10.1002/biot.201700582 [doi]\\nPST - ppublish\\nSO  - Biotechnol J. 2018 Sep;13(9):e1700582. doi: 10.1002/biot.201700582. Epub 2018 Apr  29.\\n\\nPMID- 24360884\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20141029\\nLR  - 20160518\\nIS  - 1875-9777 (Electronic)\\nIS  - 1875-9777 (Linking)\\nVI  - 14\\nIP  - 3\\nDP  - 2014 Mar 6\\nTI  - Genetic modification and screening in rat using haploid embryonic stem cells.\\nPG  - 404-14\\nLID - S1934-5909(13)00507-9 [pii]\\nLID - 10.1016/j.stem.2013.11.016 [doi]\\nAB  - The rat is an important animal model in biomedical research, but practical  limitations to genetic manipulation have restricted the application of genetic  analysis. Here we report the derivation of rat androgenetic haploid embryonic stem  cells (RahESCs) as a tool to facilitate such studies. Our approach is based on  removal of the maternal pronucleus from zygotes to generate androgenetic embryos  followed by derivation of ESCs. The resulting RahESCs have 21 chromosomes, express  pluripotency markers, differentiate into three germ layer cells, and contribute to  the germline. Homozygous mutations can be introduced by both large-scale gene  trapping and precise gene targeting via homologous recombination or the CRISPR-Cas  system. RahESCs can also produce fertile rats after intracytoplasmic injection into  oocytes and are therefore able to transmit genetic modifications to offspring.  Overall, RahESCs represent a practical tool for functional genetic studies and  production of transgenic lines in rat.\\nCI  - Copyright Â© 2014 Elsevier Inc. All rights reserved.\\nFAU - Li, Wei\\nAU  - Li W\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Li, Xin\\nAU  - Li X\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; College of Life Science, Northeast Agricultural  University of China, Harbin 150030, China.\\nFAU - Li, Tianda\\nAU  - Li T\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Jiang, Ming-Gui\\nAU  - Jiang MG\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China;  College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, China.\\nFAU - Wan, Haifeng\\nAU  - Wan H\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Luo, Guan-Zheng\\nAU  - Luo GZ\\nAD  - Key Laboratory of Genetic Network Biology, Institute of Genetics and Developmental  Biology, Chinese Academy of Sciences, BeijingÂ\\xa0100101, China.\\nFAU - Feng, Chunjing\\nAU  - Feng C\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China;  University of Chinese Academy of Sciences, Beijing 100049, China.\\nFAU - Cui, Xiaolong\\nAU  - Cui X\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Genetic Network Biology,  Institute of Genetics and Developmental Biology, Chinese Academy of Sciences,  BeijingÂ\\xa0100101, China; University of Chinese Academy of Sciences, Beijing 100049,  China.\\nFAU - Teng, Fei\\nAU  - Teng F\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China;  University of Chinese Academy of Sciences, Beijing 100049, China.\\nFAU - Yuan, Yan\\nAU  - Yuan Y\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Laboratory of Reproductive Medicine, Department  of Histology and Embryology, Nanjing Medical University, Nanjing 210029, China.\\nFAU - Zhou, Quan\\nAU  - Zhou Q\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Laboratory of Reproductive Medicine, Department  of Histology and Embryology, Nanjing Medical University, Nanjing 210029, China.\\nFAU - Gu, Qi\\nAU  - Gu Q\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; College of Life Science, Northeast Agricultural  University of China, Harbin 150030, China.\\nFAU - Shuai, Ling\\nAU  - Shuai L\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; College of Life Science, Northeast Agricultural  University of China, Harbin 150030, China.\\nFAU - Sha, Jiahao\\nAU  - Sha J\\nAD  - Laboratory of Reproductive Medicine, Department of Histology and Embryology, Nanjing  Medical University, Nanjing 210029, China.\\nFAU - Xiao, Yamei\\nAU  - Xiao Y\\nAD  - College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, China.\\nFAU - Wang, Liu\\nAU  - Wang L\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Liu, Zhonghua\\nAU  - Liu Z\\nAD  - College of Life Science, Northeast Agricultural University of China, Harbin 150030,  China.\\nFAU - Wang, Xiu-Jie\\nAU  - Wang XJ\\nAD  - Key Laboratory of Genetic Network Biology, Institute of Genetics and Developmental  Biology, Chinese Academy of Sciences, BeijingÂ\\xa0100101, China.\\nFAU - Zhao, Xiao-Yang\\nAU  - Zhao XY\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.\\nFAU - Zhou, Qi\\nAU  - Zhou Q\\nAD  - State Key Laboratory of Reproductive Biology, Institute of Zoology, Chinese Academy  of Sciences, Beijing 100101, China; Key Laboratory of Translational Stem Cell  Research, Institute of Zoology, Chinese Academy of Sciences, Beijing 100101, China.  Electronic address: qzhou@ioz.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20131219\\nPL  - United States\\nTA  - Cell Stem Cell\\nJT  - Cell stem cell\\nJID - 101311472\\nRN  - 0 (Biomarkers)\\nRN  - 0 (DNA Transposable Elements)\\nSB  - IM\\nCIN - Cell Stem Cell. 2014 Mar 6;14(3):265-7. PMID: 24607398\\nMH  - Animals\\nMH  - Base Sequence\\nMH  - Biomarkers/metabolism\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - Cell Differentiation/genetics\\nMH  - DNA Transposable Elements/genetics\\nMH  - Embryo, Mammalian/metabolism\\nMH  - Embryonic Stem Cells/cytology/*metabolism\\nMH  - Gene Expression Regulation\\nMH  - Gene Targeting\\nMH  - *Genetic Testing\\nMH  - Genome\\nMH  - *Haploidy\\nMH  - Homologous Recombination/genetics\\nMH  - Homozygote\\nMH  - Male\\nMH  - Mice\\nMH  - Molecular Sequence Data\\nMH  - Mutation/genetics\\nMH  - Pluripotent Stem Cells/cytology/metabolism\\nMH  - Rats\\nMH  - Sperm Injections, Intracytoplasmic\\nEDAT- 2013/12/24 06:00\\nMHDA- 2014/10/30 06:00\\nCRDT- 2013/12/24 06:00\\nPHST- 2013/06/05 00:00 [received]\\nPHST- 2013/08/19 00:00 [revised]\\nPHST- 2013/11/18 00:00 [accepted]\\nPHST- 2013/12/24 06:00 [entrez]\\nPHST- 2013/12/24 06:00 [pubmed]\\nPHST- 2014/10/30 06:00 [medline]\\nAID - S1934-5909(13)00507-9 [pii]\\nAID - 10.1016/j.stem.2013.11.016 [doi]\\nPST - ppublish\\nSO  - Cell Stem Cell. 2014 Mar 6;14(3):404-14. doi: 10.1016/j.stem.2013.11.016. Epub 2013  Dec 19.\\n\\nPMID- 31112734\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200622\\nLR  - 20210717\\nIS  - 1090-2422 (Electronic)\\nIS  - 0014-4827 (Linking)\\nVI  - 382\\nIP  - 1\\nDP  - 2019 Sep 1\\nTI  - Fam208a orchestrates interaction protein network essential for early embryonic  development and cell division.\\nPG  - 111437\\nLID - S0014-4827(19)30259-9 [pii]\\nLID - 10.1016/j.yexcr.2019.05.018 [doi]\\nAB  - Maintenance of genome stability is essential for every living cell as genetic  information is repeatedly challenged during DNA replication in each cell division  event. Errors, defects, delays, and mistakes that arise during mitosis or meiosis  lead to an activation of DNA repair processes and in case of their failure,  programmed cell death, i.e. apoptosis, could be initiated. Fam208a is a protein  whose importance in heterochromatin maintenance has been described recently. In this  work, we describe the crucial role of Fam208a in sustaining the genome stability  during the cellular division. The targeted depletion of Fam208a in mice using  CRISPR/Cas9 leads to embryonic lethality before E12.5. We also used the siRNA  approach to downregulate Fam208a in zygotes to avoid the influence of maternal RNA  in the early stages of development. This early downregulation increased arresting of  the embryonal development at the two-cell stage and occurrence of multipolar  spindles formation. To investigate this further, we used the yeast two-hybrid (Y2H)  system and identified new putative interaction partners Gpsm2, Amn1, Eml1, Svil, and  Itgb3bp. Their co-expression with Fam208a was assessed by qRT-PCR profiling and in  situ hybridisation [1] in multiple murine tissues. Based on these results we  proposed that Fam208a functions within the HUSH complex by interaction with Mphosph8  as these proteins are not only able to physically interact but also co-localise. We  are bringing new evidence that Fam208a is multi-interacting protein affecting genome  stability on the level of cell division at the earliest stages of development and  also by interaction with methylation complex in adult tissues. In addition to its  epigenetic functions, Fam208a appears to have an additional role in zygotic  division, possibly via interaction with newly identified putative partners Gpsm2,  Amn1, Eml1, Svil, and Itgb3bp.\\nCI  - Copyright Â© 2019 The Authors. Published by Elsevier Inc. All rights reserved.\\nFAU - Gresakova, Veronika\\nAU  - Gresakova V\\nAD  - Laboratory of Transgenic Models of Diseases, Institute of Molecular Genetics of the  ASCR, v.v.i, PrÅ¯myslova 595, 252 50, Vestec, Czech Republic; Palacky University in  Olomouc, Faculty of Medicine and Dentistry, HnÄ›votÃ\\xadnskÃ¡ 3775 15, Olomouc, Czech  Republic. Electronic address: veronika.gresakova@img.cas.cz.\\nFAU - Novosadova, Vendula\\nAU  - Novosadova V\\nAD  - Czech Centre of Phenogenomics, Institute of Molecular Genetics, ASCR v.v.i,  PrÅ¯myslova 595, 252 50, Vestec, Czech Republic. Electronic address:  vendula.novosadova@img.cas.cz.\\nFAU - Prochazkova, Michaela\\nAU  - Prochazkova M\\nAD  - Czech Centre of Phenogenomics, Institute of Molecular Genetics, ASCR v.v.i,  PrÅ¯myslova 595, 252 50, Vestec, Czech Republic. Electronic address:  michaela.prochazkova@img.cas.cz.\\nFAU - Bhargava, Shohag\\nAU  - Bhargava S\\nAD  - Laboratory of Transgenic Models of Diseases, Institute of Molecular Genetics of the  ASCR, v.v.i, PrÅ¯myslova 595, 252 50, Vestec, Czech Republic. Electronic address:  shohag.bhargava@img.cas.cz.\\nFAU - Jenickova, Irena\\nAU  - Jenickova I\\nAD  - Czech Centre of Phenogenomics, Institute of Molecular Genetics, ASCR v.v.i,  PrÅ¯myslova 595, 252 50, Vestec, Czech Republic. Electronic address:  irena.jenickova@img.cas.cz.\\nFAU - Prochazka, Jan\\nAU  - Prochazka J\\nAD  - Laboratory of Transgenic Models of Diseases, Institute of Molecular Genetics of the  ASCR, v.v.i, PrÅ¯myslova 595, 252 50, Vestec, Czech Republic; Czech Centre of  Phenogenomics, Institute of Molecular Genetics, ASCR v.v.i, PrÅ¯myslova 595, 252 50,  Vestec, Czech Republic. Electronic address: jan.prochazka@img.cas.cz.\\nFAU - Sedlacek, Radislav\\nAU  - Sedlacek R\\nAD  - Laboratory of Transgenic Models of Diseases, Institute of Molecular Genetics of the  ASCR, v.v.i, PrÅ¯myslova 595, 252 50, Vestec, Czech Republic; Czech Centre of  Phenogenomics, Institute of Molecular Genetics, ASCR v.v.i, PrÅ¯myslova 595, 252 50,  Vestec, Czech Republic. Electronic address: radislav.sedlacek@img.cas.cz.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190518\\nPL  - United States\\nTA  - Exp Cell Res\\nJT  - Experimental cell research\\nJID - 0373226\\nRN  - 0 (Fam208a protein, mouse)\\nRN  - 0 (MPHOSPH8 protein, human)\\nRN  - 0 (Mphosph8 protein, mouse)\\nRN  - 0 (Multiprotein Complexes)\\nRN  - 0 (Nuclear Proteins)\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (RNA, Small Interfering)\\nRN  - 0 (TASOR protein, human)\\nSB  - IM\\nEIN - Exp Cell Res. 2021 Aug 15;405(2):112718. PMID: 34272044\\nMH  - Animals\\nMH  - CRISPR-Cas Systems\\nMH  - Cell Division/genetics/physiology\\nMH  - Embryonic Development/genetics/*physiology\\nMH  - *Gene Expression Regulation, Developmental\\nMH  - Genes, Lethal\\nMH  - Genomic Instability\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Mice\\nMH  - Mice, Inbred C57BL\\nMH  - Mice, Knockout\\nMH  - Multiprotein Complexes\\nMH  - Nuclear Proteins/*physiology\\nMH  - Phosphoproteins/metabolism\\nMH  - RNA Interference\\nMH  - RNA, Small Interfering/genetics/pharmacology\\nMH  - Spindle Apparatus/*metabolism\\nMH  - Zygote/metabolism\\nOTO - NOTNLM\\nOT  - *Fam208a\\nOT  - *Genome stability\\nOT  - *HUSH\\nOT  - *Multipolar spindle apparatus\\nEDAT- 2019/05/22 06:00\\nMHDA- 2020/06/23 06:00\\nCRDT- 2019/05/22 06:00\\nPHST- 2019/04/02 00:00 [received]\\nPHST- 2019/05/11 00:00 [revised]\\nPHST- 2019/05/14 00:00 [accepted]\\nPHST- 2019/05/22 06:00 [pubmed]\\nPHST- 2020/06/23 06:00 [medline]\\nPHST- 2019/05/22 06:00 [entrez]\\nAID - S0014-4827(19)30259-9 [pii]\\nAID - 10.1016/j.yexcr.2019.05.018 [doi]\\nPST - ppublish\\nSO  - Exp Cell Res. 2019 Sep 1;382(1):111437. doi: 10.1016/j.yexcr.2019.05.018. Epub 2019  May 18.\\n\\nPMID- 31235524\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200316\\nLR  - 20210205\\nIS  - 1083-351X (Electronic)\\nIS  - 0021-9258 (Print)\\nIS  - 0021-9258 (Linking)\\nVI  - 294\\nIP  - 31\\nDP  - 2019 Aug 2\\nTI  - WNT-3A-induced Î²-catenin signaling does not require signaling through heterotrimeric  G proteins.\\nPG  - 11677-11684\\nLID - 10.1074/jbc.AC119.009412 [doi]\\nAB  - The network of Wingless/Int-1 (WNT)-induced signaling pathways includes  Î²-catenin-dependent and -independent pathways. Î²-Catenin regulates T cell  factor/lymphoid enhancer-binding factor (TCF/LEF)-mediated gene transcription, and  in response to WNTs, Î²-catenin signaling is initiated through engagement of a  Frizzled (FZD)/LDL receptor-related protein 5/6 (LRP5/6) receptor complex. FZDs are  G protein-coupled receptors, but the question of whether heterotrimeric G proteins  are involved in WNT/Î²-catenin signaling remains unanswered. Here, we investigate  whether acute activation of WNT/Î²-catenin signaling by purified WNT-3A requires  functional signaling through heterotrimeric G proteins. Using genome editing, we  ablated expression of G(s)/G(olf)/G(q)/G(11)/G(12)/G(13)/G(z) in HEK293 (Î”G7) cells,  leaving the expression of pertussis toxin (PTX)-sensitive G(i/o) proteins unchanged,  to assess whether WNT-3A activates WNT/Î²-catenin signaling in WT and Î”G7 cells  devoid of functional G protein signaling. We monitored WNT-3A-induced activation by  detection of phosphorylation of LDL receptor-related protein 6 (LRP6),  electrophoretic mobility shift of the phosphoprotein Dishevelled (DVL), Î²-catenin  stabilization and dephosphorylation, and TCF-dependent transcription. We found that  purified, recombinant WNT-3A efficiently induces WNT/Î²-catenin signaling in Î”G7  cells in both the absence and presence of G(i/o)-blocking PTX. Furthermore, cells  completely devoid of G protein expression, so called GÎ±-depleted HEK293 cells,  maintain responsiveness to WNT-3A with regard to the hallmarks of WNT/Î²-catenin  signaling. These findings corroborate the concept that heterotrimeric G proteins are  not required for this FZD- and DVL-mediated signaling branch. Our observations agree  with previous results arguing for FZD conformation-dependent functional selectivity  between DVL and heterotrimeric G proteins. In conclusion, WNT/Î²-catenin signaling  through FZDs does not require the involvement of heterotrimeric G proteins.\\nCI  - Â© 2019 Bowin et al.\\nFAU - Bowin, Carl-Fredrik\\nAU  - Bowin CF\\nAUID- ORCID: 0000-0001-9090-9493\\nAD  - Section for Receptor Biology and Signaling, Department of Physiology and  Pharmacology, Karolinska Institutet, Biomedicum (6D), SolnavÃ¤gen 9, SE-17165  Stockholm, Sweden.\\nFAU - Inoue, Asuka\\nAU  - Inoue A\\nAD  - Graduate School of Pharmaceutical Sciences, Tohoku University, Sendai, Miyagi  980-8578, Japan.\\nFAU - Schulte, Gunnar\\nAU  - Schulte G\\nAUID- ORCID: 0000-0002-2700-7013\\nAD  - Section for Receptor Biology and Signaling, Department of Physiology and  Pharmacology, Karolinska Institutet, Biomedicum (6D), SolnavÃ¤gen 9, SE-17165  Stockholm, Sweden gunnar.schulte@ki.se.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190624\\nTA  - J Biol Chem\\nJT  - The Journal of biological chemistry\\nJID - 2985121R\\nRN  - 0 (Dishevelled Proteins)\\nRN  - 0 (LRP6 protein, human)\\nRN  - 0 (Low Density Lipoprotein Receptor-Related Protein-6)\\nRN  - 0 (Recombinant Proteins)\\nRN  - 0 (TCF Transcription Factors)\\nRN  - 0 (Wnt3A Protein)\\nRN  - 0 (beta Catenin)\\nRN  - EC 2.4.2.31 (Pertussis Toxin)\\nRN  - EC 3.6.5.1 (Heterotrimeric GTP-Binding Proteins)\\nSB  - IM\\nMH  - Dishevelled Proteins\\nMH  - Gene Editing\\nMH  - HEK293 Cells\\nMH  - Heterotrimeric GTP-Binding Proteins/genetics/*metabolism\\nMH  - Humans\\nMH  - Low Density Lipoprotein Receptor-Related Protein-6/metabolism\\nMH  - Pertussis Toxin/pharmacology\\nMH  - Phosphorylation\\nMH  - Recombinant Proteins/biosynthesis/isolation & purification/pharmacology\\nMH  - TCF Transcription Factors/metabolism\\nMH  - Wnt Signaling Pathway/*drug effects\\nMH  - Wnt3A Protein/genetics/metabolism/*pharmacology\\nMH  - beta Catenin/metabolism\\nPMC - PMC6682745\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas\\nOT  - *Frizzled (FZD)\\nOT  - *G protein\\nOT  - *G protein-coupled receptor (GPCR)\\nOT  - *WNT pathway\\nOT  - *WNT signaling\\nOT  - *WNT-3A\\nOT  - *beta-catenin\\nOT  - *cell signaling\\nOT  - *conformational selection\\nCOIS- The authors declare that they have no conflicts of interest with the contents of  this article\\nEDAT- 2019/06/27 06:00\\nMHDA- 2020/03/17 06:00\\nCRDT- 2019/06/26 06:00\\nPHST- 2019/05/21 00:00 [received]\\nPHST- 2019/06/17 00:00 [revised]\\nPHST- 2019/06/27 06:00 [pubmed]\\nPHST- 2020/03/17 06:00 [medline]\\nPHST- 2019/06/26 06:00 [entrez]\\nAID - S0021-9258(20)30211-8 [pii]\\nAID - AC119.009412 [pii]\\nAID - 10.1074/jbc.AC119.009412 [doi]\\nPST - ppublish\\nSO  - J Biol Chem. 2019 Aug 2;294(31):11677-11684. doi: 10.1074/jbc.AC119.009412. Epub  2019 Jun 24.\\n\\nPMID- 31223339\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200929\\nIS  - 1754-6834 (Print)\\nIS  - 1754-6834 (Electronic)\\nIS  - 1754-6834 (Linking)\\nVI  - 12\\nDP  - 2019\\nTI  - A quantitative image analysis pipeline for the characterization of filamentous  fungal morphologies as a tool to uncover targets for morphology engineering: a case  study using aplD in Aspergillus niger.\\nPG  - 149\\nLID - 10.1186/s13068-019-1473-0 [doi]\\nLID - 149\\nAB  - BACKGROUND: Fungal fermentation is used to produce a diverse repertoire of enzymes,  chemicals, and drugs for various industries. During submerged cultivation,  filamentous fungi form a range of macromorphologies, including dispersed mycelia,  clumped aggregates, or pellets, which have critical implications for rheological  aspects during fermentation, gas/nutrient transfer, and, thus, product titres. An  important component of strain engineering efforts is the ability to quantitatively  assess fungal growth phenotypes, which will drive novel leads for morphologically  optimized production strains. RESULTS: In this study, we developed an automated  image analysis pipeline to quantify the morphology of pelleted and dispersed growth  (MPD) which rapidly and reproducibly measures dispersed and pelleted  macromorphologies from any submerged fungal culture. It (i) enables capture and  analysis of several hundred images per user/day, (ii) is designed to quantitatively  assess heterogeneous cultures consisting of dispersed and pelleted forms, (iii)  gives a quantitative measurement of culture heterogeneity, (iv) automatically  generates key Euclidian parameters for individual fungal structures including  particle diameter, aspect ratio, area, and solidity, which are also assembled into a  previously described dimensionless morphology number MN, (v) has an in-built quality  control check which enables end-users to easily confirm the accuracy of the  automated calls, and (vi) is easily adaptable to user-specified magnifications and  macromorphological definitions. To concomitantly provide proof of principle for the  utility of this image analysis pipeline, and provide new leads for morphologically  optimized fungal strains, we generated a morphological mutant in the cell factory  Aspergillus niger based on CRISPR-Cas technology. First, we interrogated a  previously published co-expression networks for A. niger to identify a putative  gamma-adaptin encoding gene (aplD) that was predicted to play a role in endosome  cargo trafficking. Gene editing was used to generate a conditional aplD expression  mutant under control of the titratable Tet-on system. Reduced aplD expression caused  a hyperbranched growth phenotype and diverse defects in pellet formation with a  putative increase in protein secretion. This possible protein hypersecretion  phenotype could be correlated with increased dispersed mycelia, and both decreased  pellet diameter and MN. CONCLUSION: The MPD image analysis pipeline is a simple,  rapid, and flexible approach to quantify diverse fungal morphologies. As an  exemplar, we have demonstrated that the putative endosomal transport gene aplD plays  a crucial role in A. niger filamentous growth and pellet formation during submerged  culture. This suggests that endocytic components are underexplored targets for  engineering fungal cell factories.\\nFAU - Cairns, Timothy C\\nAU  - Cairns TC\\nAUID- ORCID: 0000-0001-7106-224X\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nFAU - Feurstein, Claudia\\nAU  - Feurstein C\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 3Department of Applied and Molecular Microbiology, Institute of Biotechnology,  Technische UniversitÃ¤t Berlin, 13355 Berlin, Germany. ISNI: 0000 0001 2292 8254.  GRID: grid.6734.6\\nFAU - Zheng, Xiaomei\\nAU  - Zheng X\\nAUID- ORCID: 0000-0001-9136-0666\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 4University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Zheng, Ping\\nAU  - Zheng P\\nAUID- ORCID: 0000-0001-9434-9892\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 4University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Sun, Jibin\\nAU  - Sun J\\nAUID- ORCID: 0000-0002-0208-504X\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 4University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Meyer, Vera\\nAU  - Meyer V\\nAUID- ORCID: 0000-0002-2298-2258\\nAD  - 1Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 2Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308 People\\'s Republic of China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 3Department of Applied and Molecular Microbiology, Institute of Biotechnology,  Technische UniversitÃ¤t Berlin, 13355 Berlin, Germany. ISNI: 0000 0001 2292 8254.  GRID: grid.6734.6\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20190615\\nTA  - Biotechnol Biofuels\\nJT  - Biotechnology for biofuels\\nJID - 101316935\\nPMC - PMC6570962\\nOTO - NOTNLM\\nOT  - Aspergillus niger\\nOT  - CRISPR\\nOT  - Dispersed mycelia\\nOT  - Endocytosis\\nOT  - Image analysis\\nOT  - Morphology engineering\\nOT  - Pellet\\nOT  - Polar growth\\nOT  - Protein secretion\\nOT  - Tet-on\\nCOIS- Competing interestsThe authors declare that they have no competing interests.\\nEDAT- 2019/06/22 06:00\\nMHDA- 2019/06/22 06:01\\nCRDT- 2019/06/22 06:00\\nPHST- 2019/04/10 00:00 [received]\\nPHST- 2019/05/20 00:00 [accepted]\\nPHST- 2019/06/22 06:00 [entrez]\\nPHST- 2019/06/22 06:00 [pubmed]\\nPHST- 2019/06/22 06:01 [medline]\\nAID - 1473 [pii]\\nAID - 10.1186/s13068-019-1473-0 [doi]\\nPST - epublish\\nSO  - Biotechnol Biofuels. 2019 Jun 15;12:149. doi: 10.1186/s13068-019-1473-0. eCollection  2019.\\n\\nPMID- 32977294\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210621\\nLR  - 20210621\\nIS  - 1876-7753 (Electronic)\\nIS  - 1873-5061 (Linking)\\nVI  - 48\\nDP  - 2020 Oct\\nTI  - Generation of a tdTomato-GAD67 reporter human epilepsia mutation induced pluripotent  stem cell line, USTCi001-A-2, using CRISPR/Cas9 editing.\\nPG  - 102003\\nLID - S1873-5061(20)30304-4 [pii]\\nLID - 10.1016/j.scr.2020.102003 [doi]\\nAB  - Dravet syndrome is an epileptic encephalopathy largely due to haploinsufficiency of  the voltage-gated sodium channel Nav1.1 that is expressed primarily in GABAergic  neurons. In order to distinguish the different subtypes, we used gene editing to  introduce tdTomato gene into the genome of iPSCs to label the GABAergic neurons in  the differentiated neuronal networks. The gene-edited cell line demonstrates normal  karyotype, expresses the main pluripotency markers, and shows the presence of  differentiation into the three embryonic germ layers in teratomas.\\nCI  - Copyright Â© 2020. Published by Elsevier B.V.\\nFAU - Zhao, Huifang\\nAU  - Zhao H\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei,  China; CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key  Laboratory of Stem Cell and Regenerative Medicine, Guangzhou Institutes of  Biomedicine and Health, Chinese Academy of Sciences, Guangzhou 510530, China;  Guangzhou Regenerative Medicine and Health Guangdong Laboratory, 510005 Guangzhou,  China.\\nFAU - He, Lang\\nAU  - He L\\nAD  - Department of Neurology, Xiangya Hospital, Central South University, Changsha,  Hunan, China.\\nFAU - Huang, Hualin\\nAU  - Huang H\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; University of Chinese Academy  of Sciences, Beijing 100049, China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Li, Shuai\\nAU  - Li S\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; University of Chinese Academy  of Sciences, Beijing 100049, China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Cheng, Na\\nAU  - Cheng N\\nAD  - Department of Anatomy and Neurobiology, Xiangya School of Medicine, Central South  University, Changsha, Hunan, China.\\nFAU - Tang, Feng\\nAU  - Tang F\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; University of Chinese Academy  of Sciences, Beijing 100049, China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Han, Xiaobo\\nAU  - Han X\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; University of Chinese Academy  of Sciences, Beijing 100049, China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Lin, Zuoxian\\nAU  - Lin Z\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; Guangzhou Regenerative  Medicine and Health Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Huang, Rongqi\\nAU  - Huang R\\nAD  - CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of  Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,  Chinese Academy of Sciences, Guangzhou 510530, China; University of Chinese Academy  of Sciences, Beijing 100049, China; Guangzhou Regenerative Medicine and Health  Guangdong Laboratory, 510005 Guangzhou, China.\\nFAU - Zhou, Peng\\nAU  - Zhou P\\nAD  - Department of Anatomy and Neurobiology, Xiangya School of Medicine, Central South  University, Changsha, Hunan, China.\\nFAU - Deng, Sihao\\nAU  - Deng S\\nAD  - Department of Anatomy and Neurobiology, Xiangya School of Medicine, Central South  University, Changsha, Hunan, China.\\nFAU - Huang, Jufang\\nAU  - Huang J\\nAD  - Department of Anatomy and Neurobiology, Xiangya School of Medicine, Central South  University, Changsha, Hunan, China.\\nFAU - Li, Zhiyuan\\nAU  - Li Z\\nAD  - School of Life Sciences, University of Science and Technology of China, Hefei,  China; CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key  Laboratory of Stem Cell and Regenerative Medicine, Guangzhou Institutes of  Biomedicine and Health, Chinese Academy of Sciences, Guangzhou 510530, China; CAS  Key Laboratory of Regenerative Biology, Joint School of Life Sciences, Guangzhou  Institutes of Biomedicine and Health, Chinese Academy of Sciences, Guangzhou,  510530, China; Guangzhou Medical University, Guangzhou 511436, China; Department of  Anatomy and Neurobiology, Xiangya School of Medicine, Central South University,  Changsha, Hunan, China; University of Chinese Academy of Sciences, Beijing 100049,  China; Guangzhou Regenerative Medicine and Health Guangdong Laboratory, 510005  Guangzhou, China. Electronic address: li_zhiyuan@gibh.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200917\\nPL  - England\\nTA  - Stem Cell Res\\nJT  - Stem cell research\\nJID - 101316957\\nRN  - 0 (Luminescent Proteins)\\nRN  - 0 (red fluorescent protein)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/genetics\\nMH  - *Epilepsies, Myoclonic/genetics\\nMH  - Humans\\nMH  - *Induced Pluripotent Stem Cells\\nMH  - Luminescent Proteins\\nMH  - Mutation/genetics\\nCOIS- Declaration of Competing Interest The authors declare that they have no known  competing financial interests or personal relationships that could have appeared to  influence the work reported in this paper.\\nEDAT- 2020/09/26 06:00\\nMHDA- 2021/06/22 06:00\\nCRDT- 2020/09/25 20:20\\nPHST- 2020/06/24 00:00 [received]\\nPHST- 2020/08/23 00:00 [revised]\\nPHST- 2020/09/12 00:00 [accepted]\\nPHST- 2020/09/26 06:00 [pubmed]\\nPHST- 2021/06/22 06:00 [medline]\\nPHST- 2020/09/25 20:20 [entrez]\\nAID - S1873-5061(20)30304-4 [pii]\\nAID - 10.1016/j.scr.2020.102003 [doi]\\nPST - ppublish\\nSO  - Stem Cell Res. 2020 Oct;48:102003. doi: 10.1016/j.scr.2020.102003. Epub 2020 Sep 17.\\n\\nPMID- 27538728\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20171107\\nLR  - 20180820\\nIS  - 1365-313X (Electronic)\\nIS  - 0960-7412 (Linking)\\nVI  - 88\\nIP  - 6\\nDP  - 2016 Dec\\nTI  - Genome editing of model oleaginous microalgae Nannochloropsis spp. by CRISPR/Cas9.\\nPG  - 1071-1081\\nLID - 10.1111/tpj.13307 [doi]\\nAB  - Microalgae are promising feedstock for biofuels yet mechanistic probing of their  cellular network and industrial strain development have been hindered by lack of  genome-editing tools. Nannochloropsis spp. are emerging model microalgae for  scalable oil production and carbon sequestration. Here we established a  CRISPR/Cas9-based precise genome-editing approach for the industrial oleaginous  microalga Nannochloropsis oceanica, using nitrate reductase (NR; g7988) as example.  A new screening procedure that compares between restriction enzyme-digested nested  PCR (nPCR) products derived from enzyme-digested and not-digested genomic DNA of  transformant pools was developed to quickly, yet reliably, detect genome-engineered  mutants. Deep sequencing of nPCR products directly amplified from pooled genomic DNA  revealed over an 1% proportion of 5-bp deletion mutants and a lower frequency of  12-bp deletion mutants, with both types of editing precisely located at the targeted  site. The isolated mutants, in which precise deletion of five bases caused a  frameshift in NR translation, grow normally under NH(4) Cl but fail to grow under  NaNO(3) , and thus represent a valuable chassis strain for transgenic-strain  development. This demonstration of CRISPR/Cas9-based genome editing in industrial  microalgae opens many doors for microalgae-based biotechnological applications.\\nCI  - Â© 2016 The Authors The Plant Journal Â© 2016 John Wiley & Sons Ltd.\\nFAU - Wang, Qintao\\nAU  - Wang Q\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Lu, Yandu\\nAU  - Lu Y\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nAD  - State Key Laboratory of Marine Resource Utilization in South China Sea, College of  Oceanology, Hainan University, 570228, Haikou, Hainan, China.\\nFAU - Xin, Yi\\nAU  - Xin Y\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nFAU - Wei, Li\\nAU  - Wei L\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nFAU - Huang, Shi\\nAU  - Huang S\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nFAU - Xu, Jian\\nAU  - Xu J\\nAD  - Single-Cell Center, CAS Key Laboratory of Biofuels and Shandong Key Laboratory of  Energy Genetics, Qingdao Institute of BioEnergy and Bioprocess Technology, Chinese  Academy of Sciences, Qingdao, Shandong, 266101, China.\\nAD  - University of Chinese Academy of Sciences, Beijing, 100049, China.\\nLA  - eng\\nSI  - GENBANK/KX446948\\nSI  - GENBANK/KX446949\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20161018\\nPL  - England\\nTA  - Plant J\\nJT  - The Plant journal : for cell and molecular biology\\nJID - 9207397\\nRN  - EC 1.7.99.4 (Nitrate Reductase)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Gene Editing/*methods\\nMH  - Microalgae/*genetics\\nMH  - Nitrate Reductase/genetics/metabolism\\nMH  - Polymerase Chain Reaction\\nOTO - NOTNLM\\nOT  - *Nannochloropsis spp.\\nOT  - *clustered regularly interspaced short palindromic repeat-Cas system\\nOT  - *genome editing\\nOT  - *nitrate reductase\\nOT  - *oleaginous microalgae\\nOT  - *technical advance\\nEDAT- 2016/10/19 06:00\\nMHDA- 2017/11/08 06:00\\nCRDT- 2016/08/20 06:00\\nPHST- 2016/04/08 00:00 [received]\\nPHST- 2016/08/04 00:00 [revised]\\nPHST- 2016/08/05 00:00 [accepted]\\nPHST- 2016/10/19 06:00 [pubmed]\\nPHST- 2017/11/08 06:00 [medline]\\nPHST- 2016/08/20 06:00 [entrez]\\nAID - 10.1111/tpj.13307 [doi]\\nPST - ppublish\\nSO  - Plant J. 2016 Dec;88(6):1071-1081. doi: 10.1111/tpj.13307. Epub 2016 Oct 18.\\n\\nPMID- 32572597\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210806\\nLR  - 20210806\\nIS  - 1432-0533 (Electronic)\\nIS  - 0001-6322 (Linking)\\nVI  - 140\\nIP  - 3\\nDP  - 2020 Sep\\nTI  - The KBTBD6/7-DRD2 axis regulates pituitary adenoma sensitivity to dopamine agonist  treatment.\\nPG  - 377-396\\nLID - 10.1007/s00401-020-02180-4 [doi]\\nAB  - Pituitary adenoma (PA) is one of the most common intracranial tumors, and  approximately 40% of all PAs are prolactinomas. Dopamine agonists (DAs), such as  cabergoline (CAB), have been successfully used in the treatment of prolactinomas.  The expression of dopamine type 2 receptor (DRD2) determines the therapeutic effect  of DAs, but the molecular mechanisms of DRD2 regulation are not fully understood. In  this study, we first demonstrated that DRD2 underwent proteasome-mediated  degradation. We further employed the yeast two-hybrid system and identified kelch  repeat and BTB (POZ) domain containing 7 (KBTBD7), a substrate adaptor for the  CUL3-RING ubiquitin (Ub) ligase complex, as a DRD2-interacting protein. KBTBD6/7  directly interacted with, and ubiquitinated DRD2 at five ubiquitination sites (K221,  K226, K241, K251, and K258). CAB, a high-affinity DRD2 agonist, induced DRD2  internalization, and cytoplasmic DRD2 was degraded via ubiquitination under the  control of KBTBD6/7, the activity of which attenuated CAB-mediated inhibition of the  AKT/mTOR pathway. KBTBD7 knockout (KO) mice were generated using the CRISPR-Cas9  technique, in which the static level of DRD2 protein was elevated in the pituitary  gland, thalamus, and heart, compared to that of WT mice. Consistently, the  expression of KBTBD6/7 was negatively correlated with that of DRD2 in human  pituitary tumors. Moreover, KBTBD7 was highly expressed in dopamine-resistant  prolactinomas, but at low levels in dopamine-sensitive prolactinomas. Knockdown of  KBTBD6/7 sensitized MMQ cells and primary pituitary tumor cells to CAB treatment.  Conversely, KBTBD7 overexpression increased CAB resistance of estrogen-induced in  situ rat prolactinoma model. Together, our findings have uncovered the novel  mechanism of DRD2 protein degradation and shown that the KBTBD6/7-DRD2 axis  regulates PA sensitivity to DA treatment. KBTBD6/7 may thus become a promising  therapeutic target for pituitary tumors.\\nFAU - Liu, Yan Ting\\nAU  - Liu YT\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Liu, Fang\\nAU  - Liu F\\nAD  - State Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, 200031, China.\\nAD  - Clinical Research Lab Center, Guizhou Provincial People\\'s Hospital, Guizhou, 550002,  China.\\nFAU - Cao, Lei\\nAU  - Cao L\\nAD  - Department of Neurosurgery, Beijing Tiantan Hospital, Capital Medical University,  Beijing, 100070, China.\\nFAU - Xue, Li\\nAU  - Xue L\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Gu, Wei Ting\\nAU  - Gu WT\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Zheng, Yong Zhi\\nAU  - Zheng YZ\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nAD  - Department of Neurosurgery, First Affiliated Hospital of Wenzhou Medical University,  Wenzhou, 325000, China.\\nFAU - Tang, Hao\\nAU  - Tang H\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Wang, Yu\\nAU  - Wang Y\\nAD  - Department of Neurosurgery, Renji Hospital, Shanghai Jiao Tong University School of  Medicine, Shanghai, 200127, China.\\nFAU - Yao, Hong\\nAU  - Yao H\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Zhang, Yong\\nAU  - Zhang Y\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nFAU - Xie, Wan Qun\\nAU  - Xie WQ\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nAD  - Department of Neurosurgery, First Affiliated Hospital of Wenzhou Medical University,  Wenzhou, 325000, China.\\nFAU - Ren, Bo Han\\nAU  - Ren BH\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.\\nAD  - Department of Neurosurgery, First Affiliated Hospital of Wenzhou Medical University,  Wenzhou, 325000, China.\\nFAU - Xiao, Zhuo Hui\\nAU  - Xiao ZH\\nAD  - State Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, 200031, China.\\nFAU - Nie, Ying Jie\\nAU  - Nie YJ\\nAD  - Clinical Research Lab Center, Guizhou Provincial People\\'s Hospital, Guizhou, 550002,  China.\\nFAU - Hu, Ronggui\\nAU  - Hu R\\nAD  - State Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, 200031, China. coryhu00@gmail.com.\\nAD  - School of Life Science, Hangzhou Institute for Advanced Study, University of Chinese  Academy of Sciences, Hangzhou, 310024, China. coryhu00@gmail.com.\\nAD  - Guangdong Provincial Key Laboratory of Brain Connectome and Behavior, CAS Key  Laboratory of Brain Connectome and Manipulation, the Brain Cognition and Brain  Disease Institute (BCBDI), Shenzhen Institutes of Advanced Technology, Chinese  Academy of Sciences, Shenzhen-Hong Kong Institute of Brain Science-Shenzhen  Fundamental Research Institutions, Shenzhen, 518055, China. coryhu00@gmail.com.\\nFAU - Wu, Zhe Bao\\nAU  - Wu ZB\\nAUID- ORCID: 0000-0002-1611-8228\\nAD  - Department of Neurosurgery, Center of Pituitary Tumor, Ruijin Hospital, Shanghai  Jiao Tong University School of Medicine, Shanghai, 200025, China.  zhebaowu@aliyun.com.\\nAD  - Department of Neurosurgery, First Affiliated Hospital of Wenzhou Medical University,  Wenzhou, 325000, China. zhebaowu@aliyun.com.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200622\\nPL  - Germany\\nTA  - Acta Neuropathol\\nJT  - Acta neuropathologica\\nJID - 0412041\\nRN  - 0 (DRD2 protein, human)\\nRN  - 0 (DRD2 protein, mouse)\\nRN  - 0 (Dopamine Agonists)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (KBTBD7 protein, human)\\nRN  - 0 (Kbtbd7 protein, mouse)\\nRN  - 0 (Receptors, Dopamine D2)\\nRN  - VTD58H1Z2X (Dopamine)\\nSB  - IM\\nMH  - Adenoma/*drug therapy/metabolism\\nMH  - Animals\\nMH  - Dopamine/metabolism\\nMH  - Dopamine Agonists/*therapeutic use\\nMH  - Humans\\nMH  - Intracellular Signaling Peptides and Proteins/drug effects/metabolism\\nMH  - Mice, Knockout\\nMH  - Pituitary Gland/*drug effects/pathology\\nMH  - Pituitary Neoplasms/*drug therapy/metabolism\\nMH  - Prolactinoma/drug therapy/metabolism/pathology\\nMH  - Receptors, Dopamine D2/drug effects/metabolism\\nOTO - NOTNLM\\nOT  - *DRD2\\nOT  - *KBTBD6\\nOT  - *KBTBD7\\nOT  - *Pituitary tumor\\nOT  - *Ubiquitin degradation\\nEDAT- 2020/06/24 06:00\\nMHDA- 2021/08/07 06:00\\nCRDT- 2020/06/24 06:00\\nPHST- 2020/03/08 00:00 [received]\\nPHST- 2020/06/12 00:00 [accepted]\\nPHST- 2020/06/12 00:00 [revised]\\nPHST- 2020/06/24 06:00 [pubmed]\\nPHST- 2021/08/07 06:00 [medline]\\nPHST- 2020/06/24 06:00 [entrez]\\nAID - 10.1007/s00401-020-02180-4 [pii]\\nAID - 10.1007/s00401-020-02180-4 [doi]\\nPST - ppublish\\nSO  - Acta Neuropathol. 2020 Sep;140(3):377-396. doi: 10.1007/s00401-020-02180-4. Epub  2020 Jun 22.\\n\\nPMID- 29632194\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180814\\nLR  - 20210109\\nIS  - 1091-6490 (Electronic)\\nIS  - 0027-8424 (Print)\\nIS  - 0027-8424 (Linking)\\nVI  - 115\\nIP  - 17\\nDP  - 2018 Apr 24\\nTI  - In vivo CRISPR screening unveils histone demethylase UTX as an important epigenetic  regulator in lung tumorigenesis.\\nPG  - E3978-E3986\\nLID - 10.1073/pnas.1716589115 [doi]\\nAB  - Lung cancer is the leading cause of cancer-related death worldwide. Inactivation of  tumor suppressor genes (TSGs) promotes lung cancer malignant progression. Here, we  take advantage of the clustered regularly interspaced short palindromic repeats  (CRISPR)/Cas9-mediated somatic gene knockout in a Kras(G12D/+) mouse model to  identify bona fide TSGs. From individual knockout of 55 potential TSGs, we identify  five genes, including Utx, Ptip, Acp5, Acacb, and Clu, whose knockout significantly  promotes lung tumorigenesis. These candidate genes are frequently down-regulated in  human lung cancer specimens and significantly associated with survival in patients  with lung cancer. Through crossing the conditional Utx knockout allele to the  Kras(G12D/+) mouse model, we further find that Utx deletion dramatically promotes  lung cancer progression. The tumor-promotive effect of Utx knockout in vivo is  mainly mediated through an increase of the EZH2 level, which up-regulates the  H3K27me3 level. Moreover, the Utx-knockout lung tumors are preferentially sensitive  to EZH2 inhibitor treatment. Collectively, our study provides a systematic screening  of TSGs in vivo and identifies UTX as an important epigenetic regulator in lung  tumorigenesis.\\nFAU - Wu, Qibiao\\nAU  - Wu Q\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Tian, Yahui\\nAU  - Tian Y\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China.\\nFAU - Zhang, Jian\\nAU  - Zhang J\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Tong, Xinyuan\\nAU  - Tong X\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Huang, Hsinyi\\nAU  - Huang H\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Li, Shuai\\nAU  - Li S\\nAD  - Laura & Isaac Perlmutter Cancer Center, NYU Langone Medical Center, New York, NY  10016.\\nFAU - Zhao, Hong\\nAU  - Zhao H\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China.\\nFAU - Tang, Ying\\nAU  - Tang Y\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Yuan, Chongze\\nAU  - Yuan C\\nAD  - Department of Thoracic Surgery, Fudan University Shanghai Cancer Center, Shanghai  200031, China.\\nAD  - Department of Oncology, Shanghai Medical College, Fudan University, Shanghai 200031,  China.\\nFAU - Wang, Kun\\nAU  - Wang K\\nAD  - Key Laboratory of Molecular Imaging, Institute of Automation, Chinese Academy of  Sciences, Beijing 100190, China.\\nFAU - Fang, Zhaoyuan\\nAU  - Fang Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Gao, Lei\\nAU  - Gao L\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China.\\nFAU - Hu, Xin\\nAU  - Hu X\\nAD  - The University of Texas Graduate School in Biomedical Sciences, Houston, TX 77030.\\nAD  - Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center,  Houston, TX 77030.\\nFAU - Li, Fuming\\nAU  - Li F\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Qin, Zhen\\nAU  - Qin Z\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Yao, Shun\\nAU  - Yao S\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Chen, Ting\\nAU  - Chen T\\nAD  - Laura & Isaac Perlmutter Cancer Center, NYU Langone Medical Center, New York, NY  10016.\\nFAU - Chen, Haiquan\\nAU  - Chen H\\nAD  - Department of Thoracic Surgery, Fudan University Shanghai Cancer Center, Shanghai  200031, China.\\nAD  - Department of Oncology, Shanghai Medical College, Fudan University, Shanghai 200031,  China.\\nFAU - Zhang, Gong\\nAU  - Zhang G\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China.\\nFAU - Liu, Wanting\\nAU  - Liu W\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China.\\nFAU - Sun, Yihua\\nAU  - Sun Y\\nAD  - Department of Thoracic Surgery, Fudan University Shanghai Cancer Center, Shanghai  200031, China.\\nAD  - Department of Oncology, Shanghai Medical College, Fudan University, Shanghai 200031,  China.\\nFAU - Chen, Luonan\\nAU  - Chen L\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China.\\nFAU - Wong, Kwok-Kin\\nAU  - Wong KK\\nAD  - Laura & Isaac Perlmutter Cancer Center, NYU Langone Medical Center, New York, NY  10016.\\nFAU - Ge, Kai\\nAU  - Ge K\\nAUID- ORCID: 0000-0002-7442-5138\\nAD  - Laboratory of Endocrinology and Receptor Biology, National Institute of Diabetes and  Digestive and Kidney Diseases, National Institutes of Health, Bethesda, MD 20892.\\nFAU - Chen, Liang\\nAU  - Chen L\\nAD  - Institute of Life and Health Engineering, Jinan University, Guangzhou, Guangdong  510632, China; chenliang@nibs.ac.cn hbji@sibcb.ac.cn.\\nFAU - Ji, Hongbin\\nAU  - Ji H\\nAD  - State Key Laboratory of Cell Biology, Innovation Center for Cell Signaling Network,  CAS Center for Excellence in Molecular Cell Science, Shanghai Institute of  Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai 200031, China; chenliang@nibs.ac.cn hbji@sibcb.ac.cn.\\nAD  - School of Life Science and Technology, ShanghaiTech University, Shanghai 200120,  China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180409\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nRN  - 0 (Tumor Suppressor Proteins)\\nRN  - EC 1.14.11.- (Histone Demethylases)\\nRN  - EC 1.14.11.- (Utx protein, mouse)\\nRN  - EC 3.6.5.2 (Hras protein, mouse)\\nRN  - EC 3.6.5.2 (Proto-Oncogene Proteins p21(ras))\\nSB  - IM\\nMH  - Animals\\nMH  - *CRISPR-Cas Systems\\nMH  - Cell Transformation, Neoplastic/genetics/*metabolism/pathology\\nMH  - *Epigenesis, Genetic\\nMH  - *Gene Expression Regulation, Neoplastic\\nMH  - Histone Demethylases/genetics/*metabolism\\nMH  - Humans\\nMH  - Lung Neoplasms/genetics/*metabolism/pathology\\nMH  - Mice\\nMH  - Mice, Knockout\\nMH  - Neoplasms, Experimental/genetics/*metabolism/pathology\\nMH  - Proto-Oncogene Proteins p21(ras)/genetics/metabolism\\nMH  - Tumor Suppressor Proteins/genetics/metabolism\\nPMC - PMC5924887\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas9 in vivo knockout\\nOT  - *EZH2 inhibitor\\nOT  - *UTX\\nOT  - *non-small cell lung cancer\\nOT  - *tumor suppressor genes\\nCOIS- The authors declare no conflict of interest.\\nEDAT- 2018/04/11 06:00\\nMHDA- 2018/08/15 06:00\\nCRDT- 2018/04/11 06:00\\nPHST- 2018/04/11 06:00 [pubmed]\\nPHST- 2018/08/15 06:00 [medline]\\nPHST- 2018/04/11 06:00 [entrez]\\nAID - 1716589115 [pii]\\nAID - 201716589 [pii]\\nAID - 10.1073/pnas.1716589115 [doi]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2018 Apr 24;115(17):E3978-E3986. doi:  10.1073/pnas.1716589115. Epub 2018 Apr 9.\\n\\nPMID- 26283688\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151215\\nLR  - 20181113\\nIS  - 1469-9001 (Electronic)\\nIS  - 1355-8382 (Print)\\nIS  - 1355-8382 (Linking)\\nVI  - 21\\nIP  - 10\\nDP  - 2015 Oct\\nTI  - piRNAs derived from ancient viral processed pseudogenes as transgenerational  sequence-specific immune memory in mammals.\\nPG  - 1691-703\\nLID - 10.1261/rna.052092.115 [doi]\\nAB  - Endogenous bornavirus-like nucleoprotein elements (EBLNs) are sequences within  vertebrate genomes derived from reverse transcription and integration of ancient  bornaviral nucleoprotein mRNA via the host retrotransposon machinery. While species  with EBLNs appear relatively resistant to bornaviral disease, the nature of this  association is unclear. We hypothesized that EBLNs could give rise to antiviral  interfering RNA in the form of PIWI-interacting RNAs (piRNAs), a class of small RNA  known to silence transposons but not exogenous viruses. We found that in both  rodents and primates, which acquired their EBLNs independently some 25-40 million  years ago, EBLNs are present within piRNA-generating regions of the genome far more  often than expected by chance alone (â„™ = 8 Ã— 10(-3)-6 Ã— 10(-8)). Three of the seven  human EBLNs fall within annotated piRNA clusters and two marmoset EBLNs give rise to  bona fide piRNAs. In both rats and mice, at least two of the five EBLNs give rise to  abundant piRNAs in the male gonad. While no EBLNs are syntenic between rodent and  primate, some of the piRNA clusters containing EBLNs are; thus we deduce that EBLNs  were integrated into existing piRNA clusters. All true piRNAs derived from EBLNs are  antisense relative to the proposed ancient bornaviral nucleoprotein mRNA. These  observations are consistent with a role for EBLN-derived piRNA-like RNAs in  interfering with ancient bornaviral infection. They raise the hypothesis that  retrotransposon-dependent virus-to-host gene flow could engender RNA-mediated,  sequence-specific antiviral immune memory in metazoans analogous to the CRISPR/Cas  system in prokaryotes.\\nCI  - Â© 2015 Parrish et al.; Published by Cold Spring Harbor Laboratory Press for the RNA  Society.\\nFAU - Parrish, Nicholas F\\nAU  - Parrish NF\\nAD  - Department of Viral Oncology, Institute for Virus Research, Kyoto University, Kyoto  606-8507, Japan.\\nFAU - Fujino, Kan\\nAU  - Fujino K\\nAD  - Department of Viral Oncology, Institute for Virus Research, Kyoto University, Kyoto  606-8507, Japan.\\nFAU - Shiromoto, Yusuke\\nAU  - Shiromoto Y\\nAD  - Department of Pathology, Medical School and Graduate School of Frontier Biosciences,  Osaka University, Osaka 565-0871, Japan.\\nFAU - Iwasaki, Yuka W\\nAU  - Iwasaki YW\\nAD  - Department of Molecular Biology, Keio University School of Medicine, Tokyo 160-8582,  Japan.\\nFAU - Ha, Hongseok\\nAU  - Ha H\\nAD  - Department of Genetics, Rutgers, The State University of New Jersey, Piscataway, New  Jersey 08854, USA.\\nFAU - Xing, Jinchuan\\nAU  - Xing J\\nAD  - Department of Genetics, Rutgers, The State University of New Jersey, Piscataway, New  Jersey 08854, USA.\\nFAU - Makino, Akiko\\nAU  - Makino A\\nAD  - Department of Viral Oncology, Institute for Virus Research, Kyoto University, Kyoto  606-8507, Japan Center for Emerging Virus Research, Institute for Virus Research,  Kyoto University, Kyoto 606-8507, Japan.\\nFAU - Kuramochi-Miyagawa, Satomi\\nAU  - Kuramochi-Miyagawa S\\nAD  - Department of Pathology, Medical School and Graduate School of Frontier Biosciences,  Osaka University, Osaka 565-0871, Japan.\\nFAU - Nakano, Toru\\nAU  - Nakano T\\nAD  - Department of Pathology, Medical School and Graduate School of Frontier Biosciences,  Osaka University, Osaka 565-0871, Japan.\\nFAU - Siomi, Haruhiko\\nAU  - Siomi H\\nAD  - Department of Molecular Biology, Keio University School of Medicine, Tokyo 160-8582,  Japan.\\nFAU - Honda, Tomoyuki\\nAU  - Honda T\\nAD  - Department of Viral Oncology, Institute for Virus Research, Kyoto University, Kyoto  606-8507, Japan Department of Tumor Viruses, Graduate School of Medicine, Kyoto  University, Kyoto 606-8507, Japan.\\nFAU - Tomonaga, Keizo\\nAU  - Tomonaga K\\nAD  - Department of Viral Oncology, Institute for Virus Research, Kyoto University, Kyoto  606-8507, Japan Department of Tumor Viruses, Graduate School of Medicine, Kyoto  University, Kyoto 606-8507, Japan Department of Mammalian Regulatory Network,  Graduate School of Biostudies, Kyoto University, Kyoto 606-8507, Japan.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20150817\\nTA  - RNA\\nJT  - RNA (New York, N.Y.)\\nJID - 9509184\\nRN  - 0 (RNA, Small Interfering)\\nSB  - IM\\nMH  - Animals\\nMH  - Immunologic Memory/*physiology\\nMH  - Mammals\\nMH  - Primates\\nMH  - *Pseudogenes\\nMH  - RNA, Small Interfering/*physiology\\nMH  - Rats\\nPMC - PMC4574747\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - endogenous viral elements\\nOT  - paleovirology\\nOT  - piRNA\\nOT  - retrotransposon\\nEDAT- 2015/08/19 06:00\\nMHDA- 2015/12/17 06:00\\nCRDT- 2015/08/19 06:00\\nPHST- 2015/04/16 00:00 [received]\\nPHST- 2015/07/08 00:00 [accepted]\\nPHST- 2015/08/19 06:00 [entrez]\\nPHST- 2015/08/19 06:00 [pubmed]\\nPHST- 2015/12/17 06:00 [medline]\\nAID - rna.052092.115 [pii]\\nAID - RA [pii]\\nAID - 10.1261/rna.052092.115 [doi]\\nPST - ppublish\\nSO  - RNA. 2015 Oct;21(10):1691-703. doi: 10.1261/rna.052092.115. Epub 2015 Aug 17.\\n\\nPMID- 31569754\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 2076-0817 (Print)\\nIS  - 2076-0817 (Electronic)\\nIS  - 2076-0817 (Linking)\\nVI  - 8\\nIP  - 4\\nDP  - 2019 Sep 27\\nTI  - Genome Mining and Comparative Pathogenomic Analysis of An Endemic  Methicillin-Resistant Staphylococcus Aureus (MRSA) Clone,  ST612-CC8-t1257-SCCmec_IVd(2B), Isolated in South Africa.\\nLID - 10.3390/pathogens8040166 [doi]\\nLID - 166\\nAB  - This study undertook genome mining and comparative genomics to gain genetic insights  into the dominance of the methicillin-resistant Staphylococcus aureus (MRSA) endemic  clone ST612-CC8-t1257-SCCmec_IVd(2B), obtained from the poultry food chain in South  Africa. Functional annotation of the genome revealed a vast array of similar central  metabolic, cellular and biochemical networks within the endemic clone crucial for  its survival in the microbial community. In-silico analysis of the clone revealed  the possession of uniform defense systems, restriction-modification system (type I  and IV), accessory gene regulator (type I), arginine catabolic mobile element (type  II), and type 1 clustered, regularly interspaced, short palindromic repeat  (CRISPR)Cas array (N = 7 Â± 1), which offer protection against exogenous attacks. The  estimated pathogenic potential predicted a higher probability (average P(score) â‰ˆ  0.927) of the clone being pathogenic to its host. The clone carried a battery of  putative virulence determinants whose expression are critical for establishing  infection. However, there was a slight difference in their possession of adherence  factors (biofilm operon system) and toxins (hemolysins and enterotoxins). Further  analysis revealed a conserved environmental tolerance and persistence mechanisms  related to stress (oxidative and osmotic), heat shock, sporulation, bacteriocins,  and detoxification, which enable it to withstand lethal threats and contribute to  its success in diverse ecological niches. Phylogenomic analysis with close sister  lineages revealed that the clone was closely related to the MRSA isolate SHV713 from  Australia. The results of this bioinformatic analysis provide valuable insights into  the biology of this endemic clone.\\nFAU - Amoako, Daniel Gyamfi\\nAU  - Amoako DG\\nAD  - Infection Genomics and Applied Bioinformatics Division, Antimicrobial Research Unit,  College of Health Sciences, University of KwaZulu-Natal, Durban 4000, South Africa.  amoakod@gmail.com.\\nAD  - Biomedical Resource Unit, School of Laboratory Medicine and Medical Sciences,  College of Health Sciences, University of KwaZulu-Natal; Durban 4000, South Africa.  amoakod@gmail.com.\\nFAU - Somboro, Anou M\\nAU  - Somboro AM\\nAUID- ORCID: 0000-0002-1085-7740\\nAD  - Biomedical Resource Unit, School of Laboratory Medicine and Medical Sciences,  College of Health Sciences, University of KwaZulu-Natal; Durban 4000, South Africa.  anou.somboro@gmail.com.\\nAD  - Antimicrobial Research Unit, College of Health Sciences, University of  KwaZulu-Natal, Durban 4000, South Africa. anou.somboro@gmail.com.\\nFAU - Abia, Akebe Luther King\\nAU  - Abia ALK\\nAUID- ORCID: 0000-0002-5194-2810\\nAD  - Antimicrobial Research Unit, College of Health Sciences, University of  KwaZulu-Natal, Durban 4000, South Africa. lutherkinga@yahoo.fr.\\nFAU - Allam, Mushal\\nAU  - Allam M\\nAUID- ORCID: 0000-0002-9875-6716\\nAD  - Sequencing Core Facility, National Institute for Communicable Diseases, National  Health Laboratory Service, Johannesburg 2131, South Africa. MushalA@nicd.ac.za.\\nFAU - Ismail, Arshad\\nAU  - Ismail A\\nAUID- ORCID: 0000-0003-4672-5915\\nAD  - Sequencing Core Facility, National Institute for Communicable Diseases, National  Health Laboratory Service, Johannesburg 2131, South Africa. arshadi@nicd.ac.za.\\nFAU - Bester, Linda A\\nAU  - Bester LA\\nAD  - Biomedical Resource Unit, School of Laboratory Medicine and Medical Sciences,  College of Health Sciences, University of KwaZulu-Natal; Durban 4000, South Africa.  besterL@ukzn.ac.za.\\nFAU - Essack, Sabiha Y\\nAU  - Essack SY\\nAD  - Antimicrobial Research Unit, College of Health Sciences, University of  KwaZulu-Natal, Durban 4000, South Africa. essacks@ukzn.ac.za.\\nLA  - eng\\nGR  - 204517/WHO_/World Health Organization/International\\nGR  - 98342/South African National Research Foundation/\\nPT  - Journal Article\\nDEP - 20190927\\nTA  - Pathogens\\nJT  - Pathogens (Basel, Switzerland)\\nJID - 101596317\\nPMC - PMC6963616\\nOTO - NOTNLM\\nOT  - MRSA\\nOT  - South Africa\\nOT  - comparative genomics\\nOT  - defense system\\nOT  - endemic clone\\nOT  - poultry\\nOT  - tolerance mechanisms\\nOT  - virulence factors\\nCOIS- Professor Essack Sabiha is the chairperson of the Global Respiratory Infection  Partnership sponsored by an unrestricted educational grant from Reckitt and  Benckiser. All other authors declare that they have no competing interests regarding  the publication of this paper. The funders had no role in the design of the study,  in the collection, analyses, or interpretation of data, in the writing of the  manuscript, or in the decision to publish the results.\\nEDAT- 2019/10/02 06:00\\nMHDA- 2019/10/02 06:01\\nCRDT- 2019/10/02 06:00\\nPHST- 2019/07/29 00:00 [received]\\nPHST- 2019/09/16 00:00 [revised]\\nPHST- 2019/09/17 00:00 [accepted]\\nPHST- 2019/10/02 06:00 [entrez]\\nPHST- 2019/10/02 06:00 [pubmed]\\nPHST- 2019/10/02 06:01 [medline]\\nAID - pathogens8040166 [pii]\\nAID - pathogens-08-00166 [pii]\\nAID - 10.3390/pathogens8040166 [doi]\\nPST - epublish\\nSO  - Pathogens. 2019 Sep 27;8(4):166. doi: 10.3390/pathogens8040166.\\n\\nPMID- 30745359\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191014\\nLR  - 20210314\\nIS  - 1083-351X (Electronic)\\nIS  - 0021-9258 (Print)\\nIS  - 0021-9258 (Linking)\\nVI  - 294\\nIP  - 15\\nDP  - 2019 Apr 12\\nTI  - Rational design of a heterotrimeric G protein Î± subunit with artificial inhibitor  sensitivity.\\nPG  - 5747-5758\\nLID - S0021-9258(20)36643-6 [pii]\\nLID - 10.1074/jbc.RA118.007250 [doi]\\nAB  - Transmembrane signals initiated by a range of extracellular stimuli converge on  members of the Gq family of heterotrimeric G proteins, which relay these signals in  target cells. Gq family G proteins comprise Gq, G11, G14, and G16, which upon  activation mediate their cellular effects via inositol lipid-dependent and  -independent signaling to control fundamental processes in mammalian physiology. To  date, highly specific inhibition of Gq/11/14 signaling can be achieved only with  FR900359 (FR) and YM-254890 (YM), two naturally occurring cyclic depsipeptides. To  further development of FR or YM mimics for other GÎ± subunits, we here set out to  rationally design GÎ±16 proteins with artificial FR/YM sensitivity by introducing an  engineered depsipeptide-binding site. Thereby we permit control of G16 function  through ligands that are inactive on the WT protein. Using CRISPR/Cas9-generated  GÎ±q/GÎ±11-null cells and loss- and gain-of-function mutagenesis along with label-free  whole-cell biosensing, we determined the molecular coordinates for FR/YM inhibition  of Gq and transplanted these to FR/YM-insensitive G16. Intriguingly, despite having  close structural similarity, FR and YM yielded biologically distinct activities: it  was more difficult to perturb Gq inhibition by FR and easier to install FR  inhibition onto G16 than perturb or install inhibition with YM. A unique hydrophobic  network utilized by FR accounted for these unexpected discrepancies. Our results  suggest that non-Gq/11/14 proteins should be amenable to inhibition by FR  scaffold-based inhibitors, provided that these inhibitors mimic the interaction of  FR with GÎ± proteins harboring engineered FR-binding sites.\\nCI  - Â© 2019 Malfacini et al.\\nFAU - Malfacini, Davide\\nAU  - Malfacini D\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.\\nFAU - Patt, Julian\\nAU  - Patt J\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.\\nFAU - Annala, Suvi\\nAU  - Annala S\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.\\nFAU - HarpsÃ¸e, Kasper\\nAU  - HarpsÃ¸e K\\nAD  - Department of Drug Design and Pharmacology, University of Copenhagen,  Universitetsparken 2, 2100 Copenhagen, Denmark.\\nFAU - Eryilmaz, Funda\\nAU  - Eryilmaz F\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.\\nFAU - Reher, Raphael\\nAU  - Reher R\\nAD  - Institute for Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn,  Germany.\\nFAU - CrÃ¼semann, Max\\nAU  - CrÃ¼semann M\\nAD  - Institute for Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn,  Germany.\\nFAU - Hanke, Wiebke\\nAU  - Hanke W\\nAD  - Institute for Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn,  Germany.\\nFAU - Zhang, Hang\\nAU  - Zhang H\\nAD  - Department of Drug Design and Pharmacology, University of Copenhagen,  Universitetsparken 2, 2100 Copenhagen, Denmark.\\nFAU - Tietze, Daniel\\nAU  - Tietze D\\nAD  - Eduard Zintl Institute of Inorganic and Physical Chemistry, Technische UniversitÃ¤t  Darmstadt, 64287 Darmstadt, Germany.\\nFAU - Gloriam, David E\\nAU  - Gloriam DE\\nAD  - Department of Drug Design and Pharmacology, University of Copenhagen,  Universitetsparken 2, 2100 Copenhagen, Denmark.\\nFAU - BrÃ¤uner-Osborne, Hans\\nAU  - BrÃ¤uner-Osborne H\\nAD  - Department of Drug Design and Pharmacology, University of Copenhagen,  Universitetsparken 2, 2100 Copenhagen, Denmark.\\nFAU - StrÃ¸mgaard, Kristian\\nAU  - StrÃ¸mgaard K\\nAD  - Department of Drug Design and Pharmacology, University of Copenhagen,  Universitetsparken 2, 2100 Copenhagen, Denmark.\\nFAU - KÃ¶nig, Gabriele M\\nAU  - KÃ¶nig GM\\nAD  - Institute for Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn,  Germany.\\nFAU - Inoue, Asuka\\nAU  - Inoue A\\nAD  - Graduate School of Pharmaceutical Sciences, Tohoku University, 6-3 Aoba, Aramaki,  Aoba-ku, Sendai, Miyagi 980-8578, Japan.\\nFAU - Gomeza, Jesus\\nAU  - Gomeza J\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.\\nFAU - Kostenis, Evi\\nAU  - Kostenis E\\nAD  - From the Molecular, Cellular and Pharmacobiology Section, Institute for  Pharmaceutical Biology, University of Bonn, Nussallee 6, 53115 Bonn, Germany.  Electronic address: kostenis@uni-bonn.de.\\nLA  - eng\\nSI  - PDB/3AH8\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190211\\nTA  - J Biol Chem\\nJT  - The Journal of biological chemistry\\nJID - 2985121R\\nRN  - 0 (Depsipeptides)\\nRN  - 0 (Enzyme Inhibitors)\\nRN  - 0 (FR900359)\\nRN  - 0 (GTP-Binding Protein alpha Subunits)\\nRN  - 0 (Peptides, Cyclic)\\nRN  - 0 (YM-254890)\\nSB  - IM\\nMH  - Animals\\nMH  - CRISPR-Cas Systems\\nMH  - Depsipeptides/*pharmacology\\nMH  - Enzyme Inhibitors/*pharmacology\\nMH  - *GTP-Binding Protein alpha Subunits/antagonists & inhibitors/genetics/metabolism\\nMH  - HEK293 Cells\\nMH  - Humans\\nMH  - Hydrophobic and Hydrophilic Interactions\\nMH  - Mice\\nMH  - Peptides, Cyclic/*pharmacology\\nMH  - *Protein Engineering\\nPMC - PMC6463727\\nOTO - NOTNLM\\nOT  - *CRISPR/Cas\\nOT  - *DMR\\nOT  - *FR900359\\nOT  - *G protein\\nOT  - *G protein-coupled receptor (GPCR)\\nOT  - *YM-254890\\nOT  - *label-free\\nOT  - *pharmacology\\nOT  - *signal transduction\\nCOIS- The authors declare that they have no conflicts of interest with the contents of  this article\\nEDAT- 2019/02/13 06:00\\nMHDA- 2019/10/15 06:00\\nCRDT- 2019/02/13 06:00\\nPHST- 2018/12/20 00:00 [received]\\nPHST- 2019/01/28 00:00 [revised]\\nPHST- 2019/02/13 06:00 [pubmed]\\nPHST- 2019/10/15 06:00 [medline]\\nPHST- 2019/02/13 06:00 [entrez]\\nAID - S0021-9258(20)36643-6 [pii]\\nAID - RA118.007250 [pii]\\nAID - 10.1074/jbc.RA118.007250 [doi]\\nPST - ppublish\\nSO  - J Biol Chem. 2019 Apr 12;294(15):5747-5758. doi: 10.1074/jbc.RA118.007250. Epub 2019  Feb 11.\\n\\nPMID- 24940621\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150629\\nLR  - 20211021\\nIS  - 1932-6203 (Electronic)\\nIS  - 1932-6203 (Linking)\\nVI  - 9\\nIP  - 6\\nDP  - 2014\\nTI  - Comparative genome analysis reveals metabolic versatility and environmental  adaptations of Sulfobacillus thermosulfidooxidans strain ST.\\nPG  - e99417\\nLID - 10.1371/journal.pone.0099417 [doi]\\nLID - e99417\\nAB  - The genus Sulfobacillus is a cohort of mildly thermophilic or thermotolerant  acidophiles within the phylum Firmicutes and requires extremely acidic environments  and hypersalinity for optimal growth. However, our understanding of them is still  preliminary partly because few genome sequences are available. Here, the draft  genome of Sulfobacillus thermosulfidooxidans strain ST was deciphered to obtain a  comprehensive insight into the genetic content and to understand the cellular  mechanisms necessary for its survival. Furthermore, the expressions of key genes  related with iron and sulfur oxidation were verified by semi-quantitative RT-PCR  analysis. The draft genome sequence of Sulfobacillus thermosulfidooxidans strain ST,  which encodes 3225 predicted coding genes on a total length of 3,333,554 bp and a  48.35% G+C, revealed the high degree of heterogeneity with other Sulfobacillus  species. The presence of numerous transposases, genomic islands and complete  CRISPR/Cas defence systems testifies to its dynamic evolution consistent with the  genome heterogeneity. As expected, S. thermosulfidooxidans encodes a suit of  conserved enzymes required for the oxidation of inorganic sulfur compounds (ISCs).  The model of sulfur oxidation in S. thermosulfidooxidans was proposed, which showed  some different characteristics from the sulfur oxidation of Gram-negative A.  ferrooxidans. Sulfur oxygenase reductase and heterodisulfide reductase were  suggested to play important roles in the sulfur oxidation. Although the iron  oxidation ability was observed, some key proteins cannot be identified in S.  thermosulfidooxidans. Unexpectedly, a predicted sulfocyanin is proposed to transfer  electrons in the iron oxidation. Furthermore, its carbon metabolism is rather  flexible, can perform the transformation of pentose through the oxidative and  non-oxidative pentose phosphate pathways and has the ability to take up small  organic compounds. It encodes a multitude of heavy metal resistance systems to adapt  the heavy metal-containing environments.\\nFAU - Guo, Xue\\nAU  - Guo X\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Yin, Huaqun\\nAU  - Yin H\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Liang, Yili\\nAU  - Liang Y\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Hu, Qi\\nAU  - Hu Q\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Zhou, Xishu\\nAU  - Zhou X\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Xiao, Yunhua\\nAU  - Xiao Y\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China.\\nFAU - Ma, Liyuan\\nAU  - Ma L\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China.\\nFAU - Zhang, Xian\\nAU  - Zhang X\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China.\\nFAU - Qiu, Guanzhou\\nAU  - Qiu G\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nFAU - Liu, Xueduan\\nAU  - Liu X\\nAD  - School of Minerals Processing and Bioengineering, Central South University,  Changsha, China; Key Laboratory of Biometallurgy of Ministry of Education, Central  South University, Changsha, China.\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20140618\\nTA  - PLoS One\\nJT  - PloS one\\nJID - 101285081\\nSB  - IM\\nMH  - *Adaptation, Physiological\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats\\nMH  - Energy Metabolism/genetics\\nMH  - *Genome, Archaeal\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Molecular Sequence Annotation\\nMH  - Phylogeny\\nMH  - Sequence Analysis, DNA\\nMH  - Sulfolobus/*genetics/metabolism\\nPMC - PMC4062416\\nCOIS- Competing Interests: The authors have declared that no competing interests exist.\\nEDAT- 2014/06/19 06:00\\nMHDA- 2015/06/30 06:00\\nCRDT- 2014/06/19 06:00\\nPHST- 2013/12/23 00:00 [received]\\nPHST- 2014/05/14 00:00 [accepted]\\nPHST- 2014/06/19 06:00 [entrez]\\nPHST- 2014/06/19 06:00 [pubmed]\\nPHST- 2015/06/30 06:00 [medline]\\nAID - PONE-D-13-54225 [pii]\\nAID - 10.1371/journal.pone.0099417 [doi]\\nPST - epublish\\nSO  - PLoS One. 2014 Jun 18;9(6):e99417. doi: 10.1371/journal.pone.0099417. eCollection  2014.\\n\\nPMID- 24324032\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20140919\\nLR  - 20200826\\nIS  - 1465-2080 (Electronic)\\nIS  - 1350-0872 (Linking)\\nVI  - 160\\nIP  - Pt 2\\nDP  - 2014 Feb\\nTI  - Single-cell genomics reveals metabolic strategies for microbial growth and survival  in an oligotrophic aquifer.\\nPG  - 362-72\\nLID - 10.1099/mic.0.073965-0 [doi]\\nAB  - Bacteria from the genus Pedobacter are a major component of microbial assemblages at  Hanford Site (a largely decommissioned nuclear production complex) in eastern  Washington state, USA, and have been shown to change significantly in abundance in  response to the subsurface intrusion of Columbia River water. Here we employed  single-cell genomics techniques to shed light on the physiological niche of these  micro-organisms. Analysis of four Pedobacter single amplified genomes (SAGs) from  Hanford Site sediments revealed a chemoheterotrophic lifestyle, with the potential  to exist under both aerobic and microaerophilic conditions via expression of both  aa3-type and cbb3-type cytochrome c oxidases. These SAGs encoded a wide range of  both intra- and extracellular carbohydrate-active enzymes, potentially enabling the  degradation of recalcitrant substrates such as xylan and chitin, and the utilization  of more labile sugars such as mannose and fucose. Coupled to these enzymes, a  diversity of transporters and sugar-binding molecules were involved in the uptake of  carbon from the extracellular local environment. The SAGs were enriched in  TonB-dependent receptors, which play a key role in uptake of substrates resulting  from degradation of recalcitrant carbon. Clustered Regularly Interspaced Short  Palindromic Repeat (CRISPR)-Cas mechanisms for resisting viral infections were  identified in all SAGs. These data demonstrate the potential mechanisms utilized for  persistence by heterotrophic micro-organisms in a carbon-limited aquifer, and hint  at potential linkages between observed Pedobacter abundance shifts within the 300  Area (in the south-eastern corner of the site) subsurface and biogeochemical shifts  associated with Columbia River water intrusion.\\nFAU - Wilkins, Michael J\\nAU  - Wilkins MJ\\nAD  - Pacific Northwest National Laboratory, Richland, WA 99354, USA.\\nFAU - Kennedy, David W\\nAU  - Kennedy DW\\nFAU - Castelle, Cindy J\\nAU  - Castelle CJ\\nFAU - Field, Erin K\\nAU  - Field EK\\nFAU - Stepanauskas, Ramunas\\nAU  - Stepanauskas R\\nFAU - Fredrickson, James K\\nAU  - Fredrickson JK\\nFAU - Konopka, Allan E\\nAU  - Konopka AE\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20131209\\nPL  - England\\nTA  - Microbiology (Reading)\\nJT  - Microbiology (Reading, England)\\nJID - 9430468\\nRN  - 7440-44-0 (Carbon)\\nSB  - IM\\nMH  - Aerobiosis\\nMH  - Carbohydrate Metabolism\\nMH  - Carbon/metabolism\\nMH  - Energy Metabolism\\nMH  - *Genome, Bacterial\\nMH  - Groundwater/*microbiology\\nMH  - Heterotrophic Processes\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Pedobacter/*genetics/*growth & development\\nMH  - Washington\\nEDAT- 2013/12/11 06:00\\nMHDA- 2014/09/23 06:00\\nCRDT- 2013/12/11 06:00\\nPHST- 2013/12/11 06:00 [entrez]\\nPHST- 2013/12/11 06:00 [pubmed]\\nPHST- 2014/09/23 06:00 [medline]\\nAID - mic.0.073965-0 [pii]\\nAID - 10.1099/mic.0.073965-0 [doi]\\nPST - ppublish\\nSO  - Microbiology (Reading). 2014 Feb;160(Pt 2):362-72. doi: 10.1099/mic.0.073965-0. Epub  2013 Dec 9.\\n\\nPMID- 27345842\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170829\\nLR  - 20171126\\nIS  - 1758-2229 (Electronic)\\nIS  - 1758-2229 (Linking)\\nVI  - 8\\nIP  - 4\\nDP  - 2016 Aug\\nTI  - Analysis of defence systems and a conjugative IncP-1 plasmid in the marine  polyaromatic hydrocarbons-degrading bacterium Cycloclasticus sp. 78-ME.\\nPG  - 508-19\\nLID - 10.1111/1758-2229.12424 [doi]\\nAB  - Marine prokaryotes have evolved a broad repertoire of defence systems to protect  their genomes from lateral gene transfer including innate or acquired immune systems  and infection-induced programmed cell suicide and dormancy. Here we report on the  analysis of multiple defence systems present in the genome of the strain  Cycloclasticus sp. 78-ME isolated from petroleum deposits of the tanker \\'Amoco  Milford Haven\\'. Cycloclasticus are ubiquitous bacteria globally important in  polyaromatic hydrocarbons degradation in marine environments. Two \\'defence islands\\'  were identified in 78-ME genome: the first harbouring CRISPR-Cas with  toxin-antitoxin system, while the second was composed by an array of genes for  toxin-antitoxin and restriction-modification proteins. Among all identified spacers  of CRISPR-Cas system only seven spacers match sequences of phages and plasmids.  Furthermore, a conjugative plasmid p7ME01, which belongs to a new IncP-1Î¸ ancestral  archetype without any accessory mobile elements was found in 78-ME. Our results  provide the context to the co-occurrence of diverse defence mechanisms in the genome  of Cycloclasticus sp. 78-ME, which protect the genome of this highly specialized  PAH-degrader. This study contributes to the further understanding of complex  networks established in petroleum-based microbial communities.\\nCI  - Â© 2016 Society for Applied Microbiology and John Wiley & Sons Ltd.\\nFAU - Yakimov, Michail M\\nAU  - Yakimov MM\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nFAU - Crisafi, Francesca\\nAU  - Crisafi F\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nFAU - Messina, Enzo\\nAU  - Messina E\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nFAU - Smedile, Francesco\\nAU  - Smedile F\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nFAU - Lopatina, Anna\\nAU  - Lopatina A\\nAD  - Institute of Molecular Genetics and Gene Biology, Russian Academy of Sciences,  Moscow, Russia.\\nFAU - Denaro, Renata\\nAU  - Denaro R\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nFAU - Pieper, Dietmar H\\nAU  - Pieper DH\\nAD  - Microbial Interactions and Processes Research Group, HZI - Helmholtz Centre for  Infection Research, InhoffenstraÃŸe 7, Braunschweig, D-38124, Germany.\\nFAU - Golyshin, Peter N\\nAU  - Golyshin PN\\nAD  - School of Biological Sciences, Bangor University, ECW Bldg Deiniol Rd, Bangor,  Gwynedd, LL57 2UW, United Kingdom.\\nFAU - Giuliano, Laura\\nAU  - Giuliano L\\nAD  - Institute for Coastal Marine Environment, CNR, Spianata S. Raineri 86, Messina,  98122, Italy.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20160627\\nPL  - United States\\nTA  - Environ Microbiol Rep\\nJT  - Environmental microbiology reports\\nJID - 101499207\\nRN  - 0 (DNA Restriction-Modification Enzymes)\\nRN  - 0 (Hydrocarbons)\\nSB  - IM\\nMH  - Biotransformation\\nMH  - *CRISPR-Cas Systems\\nMH  - *DNA Restriction-Modification Enzymes\\nMH  - Genes, Bacterial\\nMH  - Genomic Islands\\nMH  - Hydrocarbons/*metabolism\\nMH  - Piscirickettsiaceae/*genetics/*metabolism\\nMH  - Plasmids/*analysis/*classification\\nMH  - Seawater/microbiology\\nEDAT- 2016/06/28 06:00\\nMHDA- 2017/08/30 06:00\\nCRDT- 2016/06/28 06:00\\nPHST- 2016/03/04 00:00 [received]\\nPHST- 2016/04/24 00:00 [accepted]\\nPHST- 2016/06/28 06:00 [entrez]\\nPHST- 2016/06/28 06:00 [pubmed]\\nPHST- 2017/08/30 06:00 [medline]\\nAID - 10.1111/1758-2229.12424 [doi]\\nPST - ppublish\\nSO  - Environ Microbiol Rep. 2016 Aug;8(4):508-19. doi: 10.1111/1758-2229.12424. Epub 2016  Jun 27.\\n\\nPMID- 25055974\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150407\\nLR  - 20211021\\nIS  - 2045-8827 (Electronic)\\nIS  - 2045-8827 (Print)\\nIS  - 2045-8827 (Linking)\\nVI  - 3\\nIP  - 4\\nDP  - 2014 Aug\\nTI  - Genome sequence of the model sulfate reducer Desulfovibrio gigas: a comparative  analysis within the Desulfovibrio genus.\\nPG  - 513-30\\nLID - 10.1002/mbo3.184 [doi]\\nAB  - Desulfovibrio gigas is a model organism of sulfate-reducing bacteria of which energy  metabolism and stress response have been extensively studied. The complete genomic  context of this organism was however, not yet available. The sequencing of the D.  gigas genome provides insights into the integrated network of energy conserving  complexes and structures present in this bacterium. Comparison with genomes of other  Desulfovibrio spp. reveals the presence of two different CRISPR/Cas systems in D.  gigas. Phylogenetic analysis using conserved protein sequences (encoded by rpoB and  gyrB) indicates two main groups of Desulfovibrio spp, being D. gigas more closely  related to D. vulgaris and D. desulfuricans strains. Gene duplications were found  such as those encoding fumarate reductase, formate dehydrogenase, and superoxide  dismutase. Complexes not yet described within Desulfovibrio genus were identified:  Mnh complex, a v-type ATP-synthase as well as genes encoding the MinCDE system that  could be responsible for the larger size of D. gigas when compared to other members  of the genus. A low number of hydrogenases and the absence of the codh/acs and pfl  genes, both present in D. vulgaris strains, indicate that intermediate cycling  mechanisms may contribute substantially less to the energy gain in D. gigas compared  to other Desulfovibrio spp. This might be compensated by the presence of other  unique genomic arrangements of complexes such as the Rnf and the Hdr/Flox, or by the  presence of NAD(P)H related complexes, like the Nuo, NfnAB or Mnh.\\nCI  - Â© 2014 The Authors. MicrobiologyOpen published by John Wiley & Sons Ltd.\\nFAU - Morais-Silva, Fabio O\\nAU  - Morais-Silva FO\\nAD  - Instituto de Tecnologia QuÃ³mica e BiolÃ³gica - Antonio Xavier, Universidade Nova de  Lisboa (ITQB-UNL), Av. da RepÃºblica - EstaÃ§Ã£o AgronÃ³mica Nacional, 2780-157, Oeiras,  Portugal.\\nFAU - Rezende, Antonio Mauro\\nAU  - Rezende AM\\nFAU - Pimentel, Catarina\\nAU  - Pimentel C\\nFAU - Santos, Catia I\\nAU  - Santos CI\\nFAU - Clemente, Carla\\nAU  - Clemente C\\nFAU - Varela-Raposo, Ana\\nAU  - Varela-Raposo A\\nFAU - Resende, Daniela M\\nAU  - Resende DM\\nFAU - da Silva, Sofia M\\nAU  - da Silva SM\\nFAU - de Oliveira, Luciana MÃ¡rcia\\nAU  - de Oliveira LM\\nFAU - Matos, Marcia\\nAU  - Matos M\\nFAU - Costa, Daniela A\\nAU  - Costa DA\\nFAU - Flores, Orfeu\\nAU  - Flores O\\nFAU - Ruiz, JerÃ³nimo C\\nAU  - Ruiz JC\\nFAU - Rodrigues-Pousada, Claudina\\nAU  - Rodrigues-Pousada C\\nLA  - eng\\nSI  - GENBANK/CP006585\\nSI  - GENBANK/CP006586\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20140723\\nTA  - Microbiologyopen\\nJT  - MicrobiologyOpen\\nJID - 101588314\\nRN  - 0 (Bacterial Proteins)\\nRN  - 0 (DNA, Bacterial)\\nSB  - IM\\nMH  - Bacterial Proteins/genetics\\nMH  - Cluster Analysis\\nMH  - Conserved Sequence\\nMH  - DNA, Bacterial/*chemistry/*genetics\\nMH  - Desulfovibrio gigas/*genetics\\nMH  - Genetic Variation\\nMH  - *Genome, Bacterial\\nMH  - Molecular Sequence Data\\nMH  - Phylogeny\\nMH  - *Sequence Analysis, DNA\\nPMC - PMC4287179\\nOTO - NOTNLM\\nOT  - Analysis\\nOT  - Desulfovibrio gigas\\nOT  - comparative genomics\\nOT  - genome\\nEDAT- 2014/07/25 06:00\\nMHDA- 2015/04/08 06:00\\nCRDT- 2014/07/25 06:00\\nPHST- 2014/03/20 00:00 [received]\\nPHST- 2014/04/30 00:00 [revised]\\nPHST- 2014/05/15 00:00 [accepted]\\nPHST- 2014/07/25 06:00 [entrez]\\nPHST- 2014/07/25 06:00 [pubmed]\\nPHST- 2015/04/08 06:00 [medline]\\nAID - 10.1002/mbo3.184 [doi]\\nPST - ppublish\\nSO  - Microbiologyopen. 2014 Aug;3(4):513-30. doi: 10.1002/mbo3.184. Epub 2014 Jul 23.\\n\\nPMID- 27066031\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20160412\\nLR  - 20201001\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Electronic)\\nIS  - 1664-462X (Linking)\\nVI  - 7\\nDP  - 2016\\nTI  - Reassessment of the Four Yield-related Genes Gn1a, DEP1, GS3, and IPA1 in Rice Using  a CRISPR/Cas9 System.\\nPG  - 377\\nLID - 10.3389/fpls.2016.00377 [doi]\\nLID - 377\\nAB  - Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-associated (Cas)  systems have been successfully used as efficient tools for genome editing in a  variety of species. We used the CRISPR/Cas9 system to mutate the Gn1a  (Os01g0197700), DEP1 (Os09g0441900), GS3 (Os03g0407400), and IPA1 (Os08g0509600)  genes of rice cultivar Zhonghua 11, genes which have been reported to function as  regulators of grain number, panicle architecture, grain size and plant architecture,  respectively. Analysis of the phenotypes and frequencies of edited genes in the  first generation of transformed plants (T0) showed that the CRISPR/Cas9 system was  highly efficient in inducing targeted gene editing, with the desired genes being  edited in 42.5% (Gn1a), 67.5% (DEP1), 57.5% (GS3), and 27.5% (IPA1) of the  transformed plants. The T2 generation of the gn1a, dep1, and gs3 mutants featured  enhanced grain number, dense erect panicles, and larger grain size, respectively.  Furthermore, semi-dwarf, and grain with long awn, phenotypes were observed in dep1  and gs3 mutants, respectively. The ipa1 mutants showed two contrasting phenotypes,  having either fewer tillers or more tillers, depending on the changes induced in the  OsmiR156 target region. In addition, we found that mutants with deletions occurred  more frequently than previous reports had indicated and that off-targeting had taken  place in highly similar target sequences. These results proved that multiple  regulators of important traits can be modified in a single cultivar by CRISPR/Cas9,  and thus facilitate the dissection of complex gene regulatory networks in the same  genomic background and the stacking of important traits in cultivated varieties.\\nFAU - Li, Meiru\\nAU  - Li M\\nAD  - Key Laboratory of Plant Resources Conservation and Sustainable Utilization, South  China Botanical Garden, Chinese Academy of SciencesGuangzhou, China; Guangdong  Provincial Key Laboratory of Applied Botany, South China Botanical Garden, Chinese  Academy of SciencesGuangzhou, China.\\nFAU - Li, Xiaoxia\\nAU  - Li X\\nAD  - Guangdong Provincial Key Lab of Biotechnology for Plant Development, South China  Normal University Guangzhou, China.\\nFAU - Zhou, Zejiao\\nAU  - Zhou Z\\nAD  - Guangdong Provincial Key Lab of Biotechnology for Plant Development, South China  Normal University Guangzhou, China.\\nFAU - Wu, Pingzhi\\nAU  - Wu P\\nAD  - Key Laboratory of Plant Resources Conservation and Sustainable Utilization, South  China Botanical Garden, Chinese Academy of SciencesGuangzhou, China; Guangdong  Provincial Key Laboratory of Applied Botany, South China Botanical Garden, Chinese  Academy of SciencesGuangzhou, China.\\nFAU - Fang, Maichun\\nAU  - Fang M\\nAD  - Key Laboratory of Plant Resources Conservation and Sustainable Utilization, South  China Botanical Garden, Chinese Academy of SciencesGuangzhou, China; Guangdong  Provincial Key Laboratory of Applied Botany, South China Botanical Garden, Chinese  Academy of SciencesGuangzhou, China.\\nFAU - Pan, Xiaoping\\nAU  - Pan X\\nAD  - Key Laboratory of Plant Resources Conservation and Sustainable Utilization, South  China Botanical Garden, Chinese Academy of SciencesGuangzhou, China; Guangdong  Provincial Key Laboratory of Applied Botany, South China Botanical Garden, Chinese  Academy of SciencesGuangzhou, China.\\nFAU - Lin, Qiupeng\\nAU  - Lin Q\\nAD  - Guangdong Provincial Key Lab of Biotechnology for Plant Development, South China  Normal University Guangzhou, China.\\nFAU - Luo, Wanbin\\nAU  - Luo W\\nAD  - Guangdong Provincial Key Lab of Biotechnology for Plant Development, South China  Normal University Guangzhou, China.\\nFAU - Wu, Guojiang\\nAU  - Wu G\\nAD  - Key Laboratory of Plant Resources Conservation and Sustainable Utilization, South  China Botanical Garden, Chinese Academy of SciencesGuangzhou, China; Guangdong  Provincial Key Laboratory of Applied Botany, South China Botanical Garden, Chinese  Academy of SciencesGuangzhou, China.\\nFAU - Li, Hongqing\\nAU  - Li H\\nAD  - Guangdong Provincial Key Lab of Biotechnology for Plant Development, South China  Normal University Guangzhou, China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20160330\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC4811884\\nOTO - NOTNLM\\nOT  - CRISPR/Cas9 system\\nOT  - Oryza sativa L.\\nOT  - gene editing\\nOT  - yield-related genes\\nOT  - yield-related traits\\nEDAT- 2016/04/12 06:00\\nMHDA- 2016/04/12 06:01\\nCRDT- 2016/04/12 06:00\\nPHST- 2016/01/20 00:00 [received]\\nPHST- 2016/03/11 00:00 [accepted]\\nPHST- 2016/04/12 06:00 [entrez]\\nPHST- 2016/04/12 06:00 [pubmed]\\nPHST- 2016/04/12 06:01 [medline]\\nAID - 10.3389/fpls.2016.00377 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2016 Mar 30;7:377. doi: 10.3389/fpls.2016.00377. eCollection 2016.\\n\\nPMID- 28730514\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180709\\nLR  - 20181113\\nIS  - 1573-9368 (Electronic)\\nIS  - 0962-8819 (Print)\\nIS  - 0962-8819 (Linking)\\nVI  - 26\\nIP  - 5\\nDP  - 2017 Oct\\nTI  - Fostering responsible research with genome editing technologies: a European  perspective.\\nPG  - 709-713\\nLID - 10.1007/s11248-017-0028-z [doi]\\nAB  - In this consensus paper resulting from a meeting that involved representatives from  more than 20 European partners, we recommend the foundation of an expert group  (European Steering Committee) to assess the potential benefits and draw-backs of  genome editing (off-targets, mosaicisms, etc.), and to design risk matrices and  scenarios for a responsible use of this promising technology. In addition, this  European steering committee will contribute in promoting an open debate on societal  aspects prior to a translation into national and international legislation.\\nFAU - Chneiweiss, HervÃ©\\nAU  - Chneiweiss H\\nAD  - INSERM Ethics Committee, Paris, France. Herve.chneiweiss@inserm.fr.\\nFAU - Hirsch, FranÃ§ois\\nAU  - Hirsch F\\nAUID- ORCID: 0000-0002-1314-6717\\nAD  - INSERM Ethics Committee, Paris, France. francois.hirsch@inserm.fr.\\nFAU - Montoliu, Lluis\\nAU  - Montoliu L\\nAD  - National Centre for Biotechnology (CNB-CSIC), CSIC Ethics Committee and  CIBERER-ISCIII, Madrid, Spain. montoliu@cnb.csic.es.\\nFAU - MÃ¼ller, Albrecht M\\nAU  - MÃ¼ller AM\\nAD  - Institute of Medical Radiology and Cell Research, University of WÃ¼rzburg, WÅ«rzburg,  Germany.\\nFAU - Fenet, Solveig\\nAU  - Fenet S\\nAD  - INSERM Ethics Committee, Paris, France.\\nFAU - Abecassis, Marion\\nAU  - Abecassis M\\nAD  - INSERM Ethics Committee, Paris, France.\\nFAU - Merchant, Jennifer\\nAU  - Merchant J\\nAD  - INSERM Ethics Committee, Paris, France.\\nAD  - University PanthÃ©on-Assas, Paris, France.\\nFAU - Baertschi, Bernard\\nAU  - Baertschi B\\nAD  - INSERM Ethics Committee, Paris, France.\\nAD  - University of Geneva, Geneva, Switzerland.\\nFAU - Botbol-Baum, MylÃ¨ne\\nAU  - Botbol-Baum M\\nAD  - INSERM Ethics Committee, Paris, France.\\nAD  - Catholic University of Louvain, Louvain, Belgium.\\nFAU - Houghton, James A\\nAU  - Houghton JA\\nAD  - National University of Ireland, Galway, Ireland.\\nFAU - Kritikos, Mihalis\\nAU  - Kritikos M\\nAD  - Research Group on Law, Science, Technology and Society (LSTS), Vrije Universiteit  Brussel, Ixelles, Belgium.\\nFAU - Mifsud, Janet\\nAU  - Mifsud J\\nAD  - University of Malta, Msida, Malta.\\nFAU - Bartnik, Ewa\\nAU  - Bartnik E\\nAD  - Faculty of Biology, Institute of Genetics and Biotechnology, University of Warsaw,  Warsaw, Poland.\\nFAU - Rath, Johannes\\nAU  - Rath J\\nAD  - University of Vienna, Vienna, Austria.\\nFAU - Druml, Christiane\\nAU  - Druml C\\nAD  - UNESCO Chair on Bioethics, Medical University of Vienna, Vienna, Austria.\\nFAU - Friedrich, BÃ¤rbel\\nAU  - Friedrich B\\nAD  - German National Academy of Sciences Leopoldina, Halle, Germany.\\nFAU - Carvalho, Ana Sofia\\nAU  - Carvalho AS\\nAD  - Instituto de BioÃ©tica, and Portuguese National Council for Ethics in Life Sciences,  Universidade CatÃ³lica Portuguesa, Porto, Portugal.\\nFAU - Lanzerath, Dirk\\nAU  - Lanzerath D\\nAD  - European Research Ethics Committee Network (EUREC), Ixelles, Belgium.\\nFAU - Saint-Raymond, AgnÃ¨s\\nAU  - Saint-Raymond A\\nAD  - Head of Human Medicines Special Areas, European Medicines Agency, London, UK.\\nLA  - eng\\nGR  - Wellcome Trust/United Kingdom\\nPT  - Letter\\nDEP - 20170720\\nTA  - Transgenic Res\\nJT  - Transgenic research\\nJID - 9209120\\nSB  - IM\\nMH  - Biotechnology/methods/*trends\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Europe\\nMH  - Gene Editing/*methods\\nMH  - Humans\\nPMC - PMC5601998\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas\\nOT  - *Gene editing\\nOT  - *Responsible research and innovation\\nOT  - *Science and society\\nCOIS- The authors declare that they have no conflict of interest.\\nEDAT- 2017/07/22 06:00\\nMHDA- 2018/07/10 06:00\\nCRDT- 2017/07/22 06:00\\nPHST- 2017/05/31 00:00 [received]\\nPHST- 2017/06/07 00:00 [accepted]\\nPHST- 2017/07/22 06:00 [pubmed]\\nPHST- 2018/07/10 06:00 [medline]\\nPHST- 2017/07/22 06:00 [entrez]\\nAID - 10.1007/s11248-017-0028-z [pii]\\nAID - 28 [pii]\\nAID - 10.1007/s11248-017-0028-z [doi]\\nPST - ppublish\\nSO  - Transgenic Res. 2017 Oct;26(5):709-713. doi: 10.1007/s11248-017-0028-z. Epub 2017  Jul 20.\\n\\nPMID- 32476557\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210716\\nLR  - 20210716\\nIS  - 2472-5560 (Electronic)\\nIS  - 2472-5552 (Linking)\\nVI  - 25\\nIP  - 6\\nDP  - 2020 Jul\\nTI  - A Novel Screening Approach for the Dissection of Cellular Regulatory Networks of  NF-ÎºB Using Arrayed CRISPR gRNA Libraries.\\nPG  - 618-633\\nLID - 10.1177/2472555220926160 [doi]\\nAB  - CRISPR/Cas9 is increasingly being used as a tool to prosecute functional genomic  screens. However, it is not yet possible to apply the approach at scale across a  full breadth of cell types and endpoints. In order to address this, we developed a  novel and robust workflow for array-based lentiviral CRISPR/Cas9 screening. We  utilized a Î²-lactamase reporter gene assay to investigate mediators of  TNF-Î±-mediated NF-ÎºB signaling. The system was adapted for CRISPR/Cas9 through the  development of a cell line stably expressing Cas9 and application of a lentiviral  gRNA library comprising mixtures of four gRNAs per gene. We screened a 743-gene  kinome library whereupon hits were independently ranked by percent inhibition, Z\\'  score, strictly standardized mean difference, and T statistic. A consolidated and  optimized ranking was generated using Borda-based methods. Screening data quality  was above acceptable limits (Z\\' â‰¥ 0.5). In order to determine the contribution of  individual gRNAs and to better understand false positives and negatives, a subset of  gRNAs, against 152 genes, were profiled in singlicate format. We highlight the use  of known reference genes and high-throughput, next-generation amplicon and RNA  sequencing to assess screen data quality. Screening with singlicate gRNAs was more  successful than screening with mixtures at identifying genes with known regulatory  roles in TNF-Î±-mediated NF-ÎºB signaling and was found to be superior to previous  RNAi-based methods. These results add to the available data on TNF-Î±-mediated NF-ÎºB  signaling and establish a high-throughput functional genomic screening approach,  utilizing a vector-based arrayed gRNA library, applicable across a wide variety of  endpoints and cell types at a genome-wide scale.\\nFAU - O\\'Shea, Patrick\\nAU  - O\\'Shea P\\nAD  - Discovery Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nFAU - Wildenhain, Jan\\nAU  - Wildenhain J\\nAD  - Syngenta R&D, Bracknell, Berkshire, UK.\\nFAU - Leveridge, Mathew\\nAU  - Leveridge M\\nAD  - Discovery Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nFAU - Revankar, Chetana\\nAU  - Revankar C\\nAD  - Thermo Fisher Scientific, Carlsbad, CA, USA.\\nFAU - Yang, Jian-Ping\\nAU  - Yang JP\\nAD  - Thermo Fisher Scientific, Carlsbad, CA, USA.\\nFAU - Bradley, Jenna\\nAU  - Bradley J\\nAD  - Discovery Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nFAU - Firth, Mike\\nAU  - Firth M\\nAD  - Quantitative Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nFAU - Pilling, James\\nAU  - Pilling J\\nAD  - Discovery Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nFAU - Piper, David\\nAU  - Piper D\\nAD  - Thermo Fisher Scientific, Carlsbad, CA, USA.\\nFAU - Chesnut, Jonathan\\nAU  - Chesnut J\\nAD  - Thermo Fisher Scientific, Carlsbad, CA, USA.\\nFAU - Isherwood, Beverley\\nAU  - Isherwood B\\nAD  - Discovery Biology, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca,  Cambridge, UK.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200601\\nPL  - United States\\nTA  - SLAS Discov\\nJT  - SLAS discovery : advancing life sciences R & D\\nJID - 101697563\\nRN  - 0 (NF-kappa B)\\nRN  - 0 (RNA, Guide)\\nRN  - 0 (Tumor Necrosis Factor-alpha)\\nRN  - EC 2.7.- (Phosphotransferases)\\nRN  - EC 3.5.2.6 (beta-Lactamases)\\nSB  - IM\\nMH  - CRISPR-Cas Systems/*genetics\\nMH  - Clustered Regularly Interspaced Short Palindromic Repeats/*genetics\\nMH  - Gene Library\\nMH  - Genes, Reporter/genetics\\nMH  - Genome, Human/genetics\\nMH  - High-Throughput Screening Assays/methods\\nMH  - Humans\\nMH  - NF-kappa B/*genetics\\nMH  - Phosphotransferases/classification/genetics\\nMH  - RNA, Guide/genetics\\nMH  - Signal Transduction/genetics\\nMH  - Tumor Necrosis Factor-alpha/*genetics\\nMH  - beta-Lactamases/genetics\\nOTO - NOTNLM\\nOT  - CRISPR/Cas systems\\nOT  - NF-ÎºB\\nOT  - cell signaling\\nOT  - functional genomics\\nOT  - gene library\\nEDAT- 2020/06/02 06:00\\nMHDA- 2021/07/17 06:00\\nCRDT- 2020/06/02 06:00\\nPHST- 2020/06/02 06:00 [pubmed]\\nPHST- 2021/07/17 06:00 [medline]\\nPHST- 2020/06/02 06:00 [entrez]\\nAID - 10.1177/2472555220926160 [doi]\\nPST - ppublish\\nSO  - SLAS Discov. 2020 Jul;25(6):618-633. doi: 10.1177/2472555220926160. Epub 2020 Jun 1.\\n\\nPMID- 30928533\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200515\\nLR  - 20200515\\nIS  - 1673-8527 (Print)\\nIS  - 1673-8527 (Linking)\\nVI  - 46\\nIP  - 3\\nDP  - 2019 Mar 20\\nTI  - Control of de novo root regeneration efficiency by developmental status of  Arabidopsis leaf explants.\\nPG  - 133-140\\nLID - S1673-8527(19)30034-7 [pii]\\nLID - 10.1016/j.jgg.2019.03.001 [doi]\\nAB  - De novo root regeneration (DNRR) has wide applications in agriculture such as those  related to cutting technology. Detached Arabidopsis thaliana leaf explants can  regenerate adventitious roots without added hormones. The regenerative ability is  highly dependent on the developmental status of the leaf. An immature leaf has a  higher regenerative ability, while a mature leaf is difficult to regenerate. Using  RNA-Seq analysis, we showed that the expression levels of many genes, including  those in the auxin network, changed during leaf maturation. Particularly, the  expression levels of many YUCCA (YUC) genes in the auxin biosynthesis pathway are  responsive to leaf maturation. Overexpression of YUC1 in the yuc-1D dominant mutant  rescued the rooting defects caused by leaf maturation. In addition, YUC4 expression  levels were also affected by circadian rhythms. The regenerative ability was reduced  in both immature and mature mutant leaf explants from the new wuschel-related  homeobox 11-3 (wox11-3) and wox12-3 mutant alleles created by the CRISPR/Cas9  method. Overall, the transcriptome and genetic data, together with the auxin  concentration analysis, indicate that the ability to upregulate auxin levels upon  detachment may be reduced during leaf maturation. Thus, multiple developmental and  environmental signals may converge to control auxin accumulation, which affects the  efficiency of the WOX11/12-mediated DNRR from leaf explants.\\nCI  - Copyright Â© 2019 Institute of Genetics and Developmental Biology, Chinese Academy of  Sciences, and Genetics Society of China. Published by Elsevier Ltd. All rights  reserved.\\nFAU - Pan, Jing\\nAU  - Pan J\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China; University of Chinese Academy  of Sciences, Beijing, 100049, China.\\nFAU - Zhao, Fei\\nAU  - Zhao F\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China; University of Chinese Academy  of Sciences, Beijing, 100049, China.\\nFAU - Zhang, Guifang\\nAU  - Zhang G\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China; University of Chinese Academy  of Sciences, Beijing, 100049, China.\\nFAU - Pan, Yu\\nAU  - Pan Y\\nAD  - School of Life Sciences, Nantong University, Nantong, 226019, China.\\nFAU - Sun, Lijun\\nAU  - Sun L\\nAD  - School of Life Sciences, Nantong University, Nantong, 226019, China.\\nFAU - Bao, Ning\\nAU  - Bao N\\nAD  - School of Public Health, Nantong University, Nantong, 226019, China.\\nFAU - Qin, Peng\\nAU  - Qin P\\nAD  - Department of Instrument Science and Engineering, Shanghai Jiao Tong University,  Shanghai, 200240, China.\\nFAU - Chen, Lyuqin\\nAU  - Chen L\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China; University of Chinese Academy  of Sciences, Beijing, 100049, China.\\nFAU - Yu, Jie\\nAU  - Yu J\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China. Electronic address:  yujie2016@sibs.ac.cn.\\nFAU - Zhang, Yijing\\nAU  - Zhang Y\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China. Electronic address:  zhangyijing@sibs.ac.cn.\\nFAU - Xu, Lin\\nAU  - Xu L\\nAD  - National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in  Molecular Plant Sciences, Shanghai Institute of Plant Physiology and Ecology,  Chinese Academy of Sciences, Shanghai, 200032, China. Electronic address:  xulin01@sibs.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190305\\nPL  - China\\nTA  - J Genet Genomics\\nJT  - Journal of genetics and genomics = Yi chuan xue bao\\nJID - 101304616\\nRN  - 0 (Arabidopsis Proteins)\\nRN  - 0 (Homeodomain Proteins)\\nRN  - 0 (Indoleacetic Acids)\\nRN  - 0 (WOX11 protein, Arabidopsis)\\nRN  - EC 1.13.- (Oxygenases)\\nRN  - EC 1.13.14.- (YUC protein, Arabidopsis)\\nSB  - IM\\nMH  - Agriculture/*methods\\nMH  - Alleles\\nMH  - Arabidopsis/genetics/*growth & development/metabolism/physiology\\nMH  - Arabidopsis Proteins/genetics\\nMH  - Base Sequence\\nMH  - Gene Expression Regulation, Plant\\nMH  - Homeodomain Proteins/genetics\\nMH  - Indoleacetic Acids/metabolism\\nMH  - Mutation\\nMH  - Oxygenases/genetics\\nMH  - Plant Leaves/*cytology\\nMH  - Plant Roots/*growth & development\\nMH  - *Regeneration\\nOTO - NOTNLM\\nOT  - *Arabidopsis\\nOT  - *Circadian rhythms\\nOT  - *De novo root regeneration\\nOT  - *Developmental status\\nOT  - *WOX11\\nOT  - *YUCCA\\nEDAT- 2019/04/01 06:00\\nMHDA- 2020/05/16 06:00\\nCRDT- 2019/04/01 06:00\\nPHST- 2018/08/08 00:00 [received]\\nPHST- 2019/02/13 00:00 [revised]\\nPHST- 2019/03/04 00:00 [accepted]\\nPHST- 2019/04/01 06:00 [pubmed]\\nPHST- 2020/05/16 06:00 [medline]\\nPHST- 2019/04/01 06:00 [entrez]\\nAID - S1673-8527(19)30034-7 [pii]\\nAID - 10.1016/j.jgg.2019.03.001 [doi]\\nPST - ppublish\\nSO  - J Genet Genomics. 2019 Mar 20;46(3):133-140. doi: 10.1016/j.jgg.2019.03.001. Epub  2019 Mar 5.\\n\\nPMID- 29423270\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 2056-5968 (Print)\\nIS  - 2056-5968 (Electronic)\\nIS  - 2056-5968 (Linking)\\nVI  - 4\\nDP  - 2018\\nTI  - ATF6 safeguards organelle homeostasis and cellular aging in human mesenchymal stem  cells.\\nPG  - 2\\nLID - 10.1038/s41421-017-0003-0 [doi]\\nLID - 2\\nAB  - Loss of organelle homeostasis is a hallmark of aging. However, it remains elusive  how this occurs at gene expression level. Here, we report that human mesenchymal  stem cell (hMSC) aging is associated with dysfunction of double-membrane organelles  and downregulation of transcription factor ATF6. CRISPR/Cas9-mediated inactivation  of ATF6 in hMSCs, not in human embryonic stem cells and human adipocytes, results in  premature cellular aging, characteristic of loss of endomembrane homeostasis.  Transcriptomic analyses uncover cell type-specific constitutive and stress-induced  ATF6-regulated genes implicated in various layers of organelles\\' homeostasis  regulation. FOS was characterized as a constitutive ATF6 responsive gene,  downregulation of which contributes to hMSC aging. Our study unravels the first  ATF6-regulated gene expression network related to homeostatic regulation of membrane  organelles, and provides novel mechanistic insights into aging-associated attrition  of human stem cells.\\nFAU - Wang, Si\\nAU  - Wang S\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 2State Key Laboratory of Stem Cell and Reproductive Biology, Institute of Zoology,  Chinese Academy of Sciences, 100101 Beijing, China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Hu, Boqiang\\nAU  - Hu B\\nAD  - 5Beijing Advanced Innovation Center for Genomics (ICG), College of Life Sciences,  Peking University, 100871 Beijing, China. ISNI: 0000 0001 2256 9319. GRID:  grid.11135.37\\nAD  - 6Peking-Tsinghua Center for Life Sciences, Peking University, 100871 Beijing, China.  ISNI: 0000 0001 2256 9319. GRID: grid.11135.37\\nFAU - Ding, Zhichao\\nAU  - Ding Z\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Dang, Yujiao\\nAU  - Dang Y\\nAD  - 5Beijing Advanced Innovation Center for Genomics (ICG), College of Life Sciences,  Peking University, 100871 Beijing, China. ISNI: 0000 0001 2256 9319. GRID:  grid.11135.37\\nAD  - 6Peking-Tsinghua Center for Life Sciences, Peking University, 100871 Beijing, China.  ISNI: 0000 0001 2256 9319. GRID: grid.11135.37\\nFAU - Wu, Jun\\nAU  - Wu J\\nAD  - 7Gene Expression Laboratory, Salk Institute for Biological Studies, 10010 North  Torrey Pines Road, La Jolla, CA 92037 USA. ISNI: 0000 0001 0662 7144. GRID:  grid.250671.7\\nFAU - Li, Di\\nAU  - Li D\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Liu, Xiaoling\\nAU  - Liu X\\nAD  - 8School of Pharmaceutical Sciences, Tsinghua-Peking Joint Center for Life Sciences,  IDG/McGovern Institute for Brain Research, Tsinghua University, 100084 Beijing,  China. ISNI: 0000 0001 0662 3178. GRID: grid.12527.33\\nFAU - Xiao, Bailong\\nAU  - Xiao B\\nAD  - 8School of Pharmaceutical Sciences, Tsinghua-Peking Joint Center for Life Sciences,  IDG/McGovern Institute for Brain Research, Tsinghua University, 100084 Beijing,  China. ISNI: 0000 0001 0662 3178. GRID: grid.12527.33\\nFAU - Zhang, Weiqi\\nAU  - Zhang W\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Ren, Ruotong\\nAU  - Ren R\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Lei, Jinghui\\nAU  - Lei J\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Hu, Huifang\\nAU  - Hu H\\nAD  - 2State Key Laboratory of Stem Cell and Reproductive Biology, Institute of Zoology,  Chinese Academy of Sciences, 100101 Beijing, China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nFAU - Chen, Chang\\nAU  - Chen C\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Chan, Piu\\nAU  - Chan P\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Li, Dong\\nAU  - Li D\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nFAU - Qu, Jing\\nAU  - Qu J\\nAD  - 2State Key Laboratory of Stem Cell and Reproductive Biology, Institute of Zoology,  Chinese Academy of Sciences, 100101 Beijing, China. ISNI: 0000000119573309. GRID:  grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nFAU - Tang, Fuchou\\nAU  - Tang F\\nAUID- ORCID: 0000-0002-8625-7717\\nAD  - 5Beijing Advanced Innovation Center for Genomics (ICG), College of Life Sciences,  Peking University, 100871 Beijing, China. ISNI: 0000 0001 2256 9319. GRID:  grid.11135.37\\nAD  - 6Peking-Tsinghua Center for Life Sciences, Peking University, 100871 Beijing, China.  ISNI: 0000 0001 2256 9319. GRID: grid.11135.37\\nAD  - 9Ministry of Education Key Laboratory of Cell Proliferation and Differentiation,  100871 Beijing, China. ISNI: 0000 0004 0369 313X. GRID: grid.419897.a\\nAD  - 10Biomedical Institute for Pioneering Investigation via Convergence, Peking  University, 100871 Beijing, China. ISNI: 0000 0001 2256 9319. GRID: grid.11135.37\\nFAU - Liu, Guang-Hui\\nAU  - Liu GH\\nAUID- ORCID: 0000-0001-9289-8177\\nAD  - 1National Laboratory of Biomacromolecules, CAS Center for Excellence in  Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, 100101  Beijing, China. ISNI: 0000000119573309. GRID: grid.9227.e\\nAD  - 3University of Chinese Academy of Sciences, 100049 Beijing, China. ISNI: 0000 0004  1797 8419. GRID: grid.410726.6\\nAD  - 4National Clinical Research Center for Geriatric Disorders, Xuanwu Hospital of  Capital Medical University, 100053 Beijing, China. ISNI: 0000 0004 0632 3337. GRID:  grid.413259.8\\nAD  - 11Key Laboratory of Regenerative Medicine of Ministry of Education, Institute of  Aging and Regenerative Medicine, Jinan University, 510632 Guangzhou, China. ISNI:  0000 0004 1790 3548. GRID: grid.258164.c\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20180105\\nTA  - Cell Discov\\nJT  - Cell discovery\\nJID - 101661034\\nPMC - PMC5798892\\nCOIS- The authors declare that they have no competing interests.\\nEDAT- 2018/02/10 06:00\\nMHDA- 2018/02/10 06:01\\nCRDT- 2018/02/10 06:00\\nPHST- 2017/08/31 00:00 [received]\\nPHST- 2017/10/19 00:00 [revised]\\nPHST- 2017/11/28 00:00 [accepted]\\nPHST- 2018/02/10 06:00 [entrez]\\nPHST- 2018/02/10 06:00 [pubmed]\\nPHST- 2018/02/10 06:01 [medline]\\nAID - 3 [pii]\\nAID - 10.1038/s41421-017-0003-0 [doi]\\nPST - epublish\\nSO  - Cell Discov. 2018 Jan 5;4:2. doi: 10.1038/s41421-017-0003-0. eCollection 2018.\\n\\nPMID- 29033363\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20171031\\nLR  - 20171209\\nIS  - 1878-1551 (Electronic)\\nIS  - 1534-5807 (Linking)\\nVI  - 43\\nIP  - 2\\nDP  - 2017 Oct 23\\nTI  - Convergent Transcriptional Programs Regulate cAMP Levels in C.Â\\xa0elegans GABAergic  Motor Neurons.\\nPG  - 212-226.e7\\nLID - S1534-5807(17)30765-7 [pii]\\nLID - 10.1016/j.devcel.2017.09.013 [doi]\\nAB  - Both transcriptional regulation and signaling pathways play crucial roles in  neuronal differentiation and plasticity. Caenorhabditis elegans possesses 19  GABAergic motor neurons (MNs) called D MNs, which are divided into two subgroups: DD  and VD. DD, but not VD, MNs reverse their cellular polarity in a developmental  process called respecification. UNC-30 and UNC-55 are two critical transcription  factors in D MNs. By using chromatin immunoprecipitation with CRISPR/Cas9 knockin of  GFP fusion, we uncovered the global targets of UNC-30 and UNC-55. UNC-30 and UNC-55  are largely converged to regulate over 1,300 noncoding and coding genes, and genes  in multiple biological processes, including cAMP metabolism, are co-regulated.  Increase in cAMP levels may serve as a timing signal for respecification, whereas  UNC-55 regulates genes such as pde-4 to keep the cAMP levels low in VD. Other genes  modulating DD respecification such as lin-14, irx-1, and oig-1 are also found to  affect cAMP levels.\\nCI  - Copyright Â© 2017 Elsevier Inc. All rights reserved.\\nFAU - Yu, Bin\\nAU  - Yu B\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Wang, Xiaolin\\nAU  - Wang X\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Wei, Shuai\\nAU  - Wei S\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Fu, Tao\\nAU  - Fu T\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Dzakah, Emmanuel Enoch\\nAU  - Dzakah EE\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Waqas, Ahmed\\nAU  - Waqas A\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China.\\nFAU - Walthall, Walter W\\nAU  - Walthall WW\\nAD  - Department of Biology, Georgia State University, Atlanta, GA 30303, USA.\\nFAU - Shan, Ge\\nAU  - Shan G\\nAD  - CAS Key Laboratory of Innate Immunity and Chronic Disease, CAS Center for Excellence  in Molecular Cell Science, School of Life Sciences, University of Science and  Technology of China, Hefei, Anhui Province 230027, China. Electronic address:  shange@ustc.edu.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20171012\\nPL  - United States\\nTA  - Dev Cell\\nJT  - Developmental cell\\nJID - 101120028\\nRN  - 0 (Caenorhabditis elegans Proteins)\\nRN  - 0 (Homeodomain Proteins)\\nRN  - 0 (Nuclear Proteins)\\nRN  - 0 (Receptors, Cell Surface)\\nRN  - 0 (Receptors, Cytoplasmic and Nuclear)\\nRN  - 0 (UNC-55 protein, C elegans)\\nRN  - 0 (unc-30 protein, C elegans)\\nRN  - E0399OZS9N (Cyclic AMP)\\nSB  - IM\\nMH  - Animals\\nMH  - Animals, Genetically Modified/genetics/growth & development/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Caenorhabditis elegans/genetics/growth & development/*metabolism\\nMH  - Caenorhabditis elegans Proteins/antagonists & inhibitors/genetics/*metabolism\\nMH  - Cyclic AMP/*metabolism\\nMH  - GABAergic Neurons/cytology/*metabolism\\nMH  - Gene Expression Regulation, Developmental\\nMH  - Homeodomain Proteins/antagonists & inhibitors/genetics/*metabolism\\nMH  - Nuclear Proteins/antagonists & inhibitors/genetics/*metabolism\\nMH  - Receptors, Cell Surface/antagonists & inhibitors/genetics/*metabolism\\nMH  - Receptors, Cytoplasmic and Nuclear/antagonists & inhibitors/genetics/*metabolism\\nMH  - *Transcription, Genetic\\nOTO - NOTNLM\\nOT  - *cAMP\\nOT  - *developmental plasticity\\nOT  - *differentiation\\nOT  - *gene regulatory network\\nOT  - *motor neurons\\nOT  - *transcription factor\\nEDAT- 2017/10/17 06:00\\nMHDA- 2017/11/01 06:00\\nCRDT- 2017/10/17 06:00\\nPHST- 2017/01/20 00:00 [received]\\nPHST- 2017/06/26 00:00 [revised]\\nPHST- 2017/09/15 00:00 [accepted]\\nPHST- 2017/10/17 06:00 [pubmed]\\nPHST- 2017/11/01 06:00 [medline]\\nPHST- 2017/10/17 06:00 [entrez]\\nAID - S1534-5807(17)30765-7 [pii]\\nAID - 10.1016/j.devcel.2017.09.013 [doi]\\nPST - ppublish\\nSO  - Dev Cell. 2017 Oct 23;43(2):212-226.e7. doi: 10.1016/j.devcel.2017.09.013. Epub 2017  Oct 12.\\n\\nPMID- 31265461\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200124\\nLR  - 20200309\\nIS  - 1545-7885 (Electronic)\\nIS  - 1544-9173 (Print)\\nIS  - 1544-9173 (Linking)\\nVI  - 17\\nIP  - 7\\nDP  - 2019 Jul\\nTI  - Targeted genetic screening in mice through haploid embryonic stem cells identifies  critical genes in bone development.\\nPG  - e3000350\\nLID - 10.1371/journal.pbio.3000350 [doi]\\nLID - e3000350\\nAB  - Mutagenic screening is powerful for identifying key genes involved in developmental  processes. However, such screens are successful only in lower organisms. Here, we  develop a targeted genetic screening approach in mice through combining androgenetic  haploid embryonic stem cells (AG-haESCs) and clustered regularly interspaced  palindromic repeats/CRISPR-associated protein 9 (CRISPR-Cas9) technology. We  produced a mutant semi-cloned (SC) mice pool by oocyte injection of AG-haESCs  carrying constitutively expressed Cas9 and an single guide RNA (sgRNA) library  targeting 72 preselected genes in one step and screened for bone-development-related  genes through skeletal analysis at birth. This yielded 4 genes: Zic1 and Clec11a,  which are required for bone development, and Rln1 and Irx5, which had not been  previously considered. Whereas Rln1-/- mice exhibited small skeletal size only at  birth, Irx5-/- mice showed skeletal abnormalities both in postnatal and adult phases  due to decreased bone mass and increased bone marrow adipogenesis. Mechanistically,  iroquois homeobox 5 (IRX5) promotes osteoblastogenesis and inhibits adipogenesis by  suppressing peroxisome proliferator activated receptor Î³ (PPARÎ³) activation. Thus,  AG-haESC-mediated functional mutagenic screening opens new avenues for genetic  interrogation of developmental processes in mice.\\nFAU - Bai, Meizhu\\nAU  - Bai M\\nAD  - State Key Laboratory of Cell Biology, Shanghai Key Laboratory of Molecular  Andrology, CAS Center for Excellence in Molecular Cell Science, Shanghai Institute  of Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Han, Yujiao\\nAU  - Han Y\\nAD  - State Key Laboratory of Cell Biology, CAS Center for Excellence in Molecular Cell  Science, Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of  Sciences, University of Chinese Academy of Sciences, Shanghai, China.\\nFAU - Wu, Yuxuan\\nAU  - Wu Y\\nAD  - Shanghai Key Laboratory of Regulatory Biology, Institute of Biomedical Sciences and  School of Life Sciences, East China Normal University, Shanghai, China.\\nFAU - Liao, Jiaoyang\\nAU  - Liao J\\nAD  - State Key Laboratory of Cell Biology, Shanghai Key Laboratory of Molecular  Andrology, CAS Center for Excellence in Molecular Cell Science, Shanghai Institute  of Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Li, Lin\\nAU  - Li L\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network, Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences, Shanghai, China.\\nFAU - Wang, Lijun\\nAU  - Wang L\\nAD  - State Key Laboratory of Cell Biology, CAS Center for Excellence in Molecular Cell  Science, Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of  Sciences, University of Chinese Academy of Sciences, Shanghai, China.\\nFAU - Li, Qing\\nAU  - Li Q\\nAD  - State Key Laboratory of Cell Biology, Shanghai Key Laboratory of Molecular  Andrology, CAS Center for Excellence in Molecular Cell Science, Shanghai Institute  of Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nFAU - Xing, Wenhui\\nAU  - Xing W\\nAD  - State Key Laboratory of Cell Biology, CAS Center for Excellence in Molecular Cell  Science, Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of  Sciences, University of Chinese Academy of Sciences, Shanghai, China.\\nFAU - Chen, Luonan\\nAU  - Chen L\\nAD  - Key Laboratory of Systems Biology, CAS Center for Excellence in Molecular Cell  Science, Innovation Center for Cell Signaling Network, Institute of Biochemistry and  Cell Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of  Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nFAU - Zou, Weiguo\\nAU  - Zou W\\nAD  - State Key Laboratory of Cell Biology, CAS Center for Excellence in Molecular Cell  Science, Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of  Sciences, University of Chinese Academy of Sciences, Shanghai, China.\\nFAU - Li, Jinsong\\nAU  - Li J\\nAUID- ORCID: 0000-0003-3456-662X\\nAD  - State Key Laboratory of Cell Biology, Shanghai Key Laboratory of Molecular  Andrology, CAS Center for Excellence in Molecular Cell Science, Shanghai Institute  of Biochemistry and Cell Biology, Chinese Academy of Sciences, University of Chinese  Academy of Sciences, Shanghai, China.\\nAD  - School of Life Science and Technology, Shanghai Tech University, Shanghai, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190702\\nTA  - PLoS Biol\\nJT  - PLoS biology\\nJID - 101183755\\nRN  - 0 (Clec11A protein, mouse)\\nRN  - 0 (Hematopoietic Cell Growth Factors)\\nRN  - 0 (Homeodomain Proteins)\\nRN  - 0 (Irx5 protein, mouse)\\nRN  - 0 (Lectins, C-Type)\\nRN  - 0 (Rln1 protein, mouse)\\nRN  - 0 (Transcription Factors)\\nRN  - 0 (Zic1 protein, mouse)\\nRN  - 9002-69-1 (Relaxin)\\nSB  - IM\\nMH  - Animals\\nMH  - Bone Development/*genetics\\nMH  - CRISPR-Cas Systems\\nMH  - Cells, Cultured\\nMH  - *Gene Expression Regulation, Developmental\\nMH  - Gene Targeting/*methods\\nMH  - Genetic Testing/*methods\\nMH  - Haploidy\\nMH  - Hematopoietic Cell Growth Factors/genetics/metabolism\\nMH  - Homeodomain Proteins/genetics/metabolism\\nMH  - Lectins, C-Type/genetics/metabolism\\nMH  - Mice\\nMH  - Mice, Knockout\\nMH  - Mouse Embryonic Stem Cells/*metabolism\\nMH  - Relaxin/genetics/metabolism\\nMH  - Transcription Factors/genetics/metabolism\\nPMC - PMC6629148\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2019/07/03 06:00\\nMHDA- 2020/01/25 06:00\\nCRDT- 2019/07/03 06:00\\nPHST- 2019/02/04 00:00 [received]\\nPHST- 2019/06/18 00:00 [accepted]\\nPHST- 2019/07/15 00:00 [revised]\\nPHST- 2019/07/03 06:00 [pubmed]\\nPHST- 2020/01/25 06:00 [medline]\\nPHST- 2019/07/03 06:00 [entrez]\\nAID - PBIOLOGY-D-19-00315 [pii]\\nAID - 10.1371/journal.pbio.3000350 [doi]\\nPST - epublish\\nSO  - PLoS Biol. 2019 Jul 2;17(7):e3000350. doi: 10.1371/journal.pbio.3000350. eCollection  2019 Jul.\\n\\nPMID- 29880605\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190911\\nLR  - 20211204\\nIS  - 1592-8721 (Electronic)\\nIS  - 0390-6078 (Print)\\nIS  - 0390-6078 (Linking)\\nVI  - 103\\nIP  - 9\\nDP  - 2018 Sep\\nTI  - Pharmacological inhibition of dihydroorotate dehydrogenase induces apoptosis and  differentiation in acute myeloid leukemia cells.\\nPG  - 1472-1483\\nLID - 10.3324/haematol.2018.188185 [doi]\\nAB  - Acute myeloid leukemia is a disorder characterized by abnormal differentiation of  myeloid cells and a clonal proliferation derived from primitive hematopoietic stem  cells. Interventions that overcome myeloid differentiation have been shown to be a  promising therapeutic strategy for acute myeloid leukemia. In this study, we  demonstrate that CRISPR/Cas9-mediated knockout of dihydroorotate dehydrogenase leads  to apoptosis and normal differentiation of acute myeloid leukemia cells, indicating  that dihydroorotate dehydrogenase is a potential differentiation regulator and a  therapeutic target in acute myeloid leukemia. By screening a library of natural  products, we identified a novel dihydroorotate dehydrogenase inhibitor,  isobavachalcone, derived from the traditional Chinese medicine Psoralea corylifolia  Using enzymatic analysis, thermal shift assay, pull down, nuclear magnetic  resonance, and isothermal titration calorimetry experiments, we demonstrate that  isobavachalcone inhibits human dihydroorotate dehydrogenase directly, and triggers  apoptosis and differentiation of acute myeloid leukemia cells. Oral administration  of isobavachalcone suppresses subcutaneous HL60 xenograft tumor growth without  obvious toxicity. Importantly, our results suggest that a combination of  isobavachalcone and adriamycin prolonged survival in an intravenous HL60 leukemia  model. In summary, this study demonstrates that isobavachalcone triggers apoptosis  and differentiation of acute myeloid leukemia cells via pharmacological inhibition  of human dihydroorotate dehydrogenase, offering a potential therapeutic strategy for  acute myeloid leukemia.\\nCI  - CopyrightÂ© 2018 Ferrata Storti Foundation.\\nFAU - Wu, Dang\\nAU  - Wu D\\nAD  - Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China  University of Science and Technology, China.\\nFAU - Wang, Wanyan\\nAU  - Wang W\\nAD  - Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China  University of Science and Technology, China.\\nFAU - Chen, Wuyan\\nAU  - Chen W\\nAD  - CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica,  Chinese Academy of Sciences (CAS), China.\\nFAU - Lian, Fulin\\nAU  - Lian F\\nAD  - CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica,  Chinese Academy of Sciences (CAS), China.\\nFAU - Lang, Li\\nAU  - Lang L\\nAD  - Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China  University of Science and Technology, China.\\nFAU - Huang, Ying\\nAU  - Huang Y\\nAD  - Guangdong Institute for Drug Control, Guangzhou, China.\\nFAU - Xu, Yechun\\nAU  - Xu Y\\nAD  - CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica,  Chinese Academy of Sciences (CAS), China.\\nFAU - Zhang, Naixia\\nAU  - Zhang N\\nAD  - CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica,  Chinese Academy of Sciences (CAS), China.\\nFAU - Chen, Yinbin\\nAU  - Chen Y\\nAD  - Shanghai Key Laboratory of Regulatory Biology, Institute of Biomedical Sciences and  School of Life Sciences, East China Normal University, China.\\nFAU - Liu, Mingyao\\nAU  - Liu M\\nAD  - Shanghai Key Laboratory of Regulatory Biology, Institute of Biomedical Sciences and  School of Life Sciences, East China Normal University, China.\\nFAU - Nussinov, Ruth\\nAU  - Nussinov R\\nAD  - Cancer and Inflammation Program, Leidos Biomedical Research, Inc., Frederick  National Laboratory for Cancer Research, National Cancer Institute at Frederick, MD,  USA.\\nAD  - Department of Human Molecular Genetics and Biochemistry, Sackler School of Medicine,  Tel Aviv University, Israel.\\nFAU - Cheng, Feixiong\\nAU  - Cheng F\\nAD  - Center for Complex Networks Research and Department of Physics, Northeastern  University, Boston, MA, USA huangjin@ecust.edu.cn wqlu@bio.ecnu.edu.cn  chengf@ccf.org.\\nAD  - Center for Cancer Systems Biology and Department of Cancer Biology, Dana-Farber  Cancer Institute, Harvard Medical School, Boston, MA, USA.\\nAD  - Genomic Medicine Institute, Lerner Research Institute, Cleveland Clinic, OH, USA.\\nAD  - Department of Molecular Medicine, Cleveland Clinic Lerner College of Medicine, Case  Western Reserve University, OH, USA.\\nFAU - Lu, Weiqiang\\nAU  - Lu W\\nAD  - Shanghai Key Laboratory of Regulatory Biology, Institute of Biomedical Sciences and  School of Life Sciences, East China Normal University, China huangjin@ecust.edu.cn  wqlu@bio.ecnu.edu.cn chengf@ccf.org.\\nFAU - Huang, Jin\\nAU  - Huang J\\nAD  - Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China  University of Science and Technology, China huangjin@ecust.edu.cn  wqlu@bio.ecnu.edu.cn chengf@ccf.org.\\nLA  - eng\\nGR  - HHSN261200800001C/RC/CCR NIH HHS/United States\\nGR  - HHSN261200800001E/CA/NCI NIH HHS/United States\\nGR  - K99 HL138272/HL/NHLBI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20180607\\nTA  - Haematologica\\nJT  - Haematologica\\nJID - 0417435\\nRN  - 0 (Antineoplastic Agents)\\nRN  - 0 (Biomarkers, Tumor)\\nRN  - 0 (Chalcones)\\nRN  - 0 (Dihydroorotate Dehydrogenase)\\nRN  - 0 (Enzyme Inhibitors)\\nRN  - 20784-50-3 (isobavachalcone)\\nRN  - EC 1.3.- (Oxidoreductases Acting on CH-CH Group Donors)\\nSB  - IM\\nCIN - Haematologica. 2018 Sep;103(9):1415-1417. PMID: 30171015\\nMH  - Animals\\nMH  - Antineoplastic Agents/*pharmacology\\nMH  - Apoptosis/*drug effects/genetics\\nMH  - Biomarkers, Tumor\\nMH  - Cell Differentiation/*drug effects/genetics\\nMH  - Cell Line, Tumor\\nMH  - Cell Proliferation/drug effects\\nMH  - Chalcones/chemistry/pharmacology\\nMH  - Dihydroorotate Dehydrogenase\\nMH  - Disease Models, Animal\\nMH  - Drug Synergism\\nMH  - Enzyme Activation/drug effects\\nMH  - Enzyme Inhibitors/*pharmacology\\nMH  - Gene Expression\\nMH  - Gene Knockdown Techniques\\nMH  - Humans\\nMH  - Leukemia, Myeloid, Acute/genetics/metabolism/mortality\\nMH  - Mice\\nMH  - Models, Molecular\\nMH  - Molecular Structure\\nMH  - Neoplastic Stem Cells/metabolism\\nMH  - Oxidoreductases Acting on CH-CH Group Donors/*antagonists &  inhibitors/genetics/metabolism\\nMH  - Prognosis\\nMH  - RNA Interference\\nMH  - Structure-Activity Relationship\\nMH  - Xenograft Model Antitumor Assays\\nPMC - PMC6119157\\nEDAT- 2018/06/09 06:00\\nMHDA- 2019/09/12 06:00\\nCRDT- 2018/06/09 06:00\\nPHST- 2018/01/10 00:00 [received]\\nPHST- 2018/05/30 00:00 [accepted]\\nPHST- 2018/06/09 06:00 [pubmed]\\nPHST- 2019/09/12 06:00 [medline]\\nPHST- 2018/06/09 06:00 [entrez]\\nAID - haematol.2018.188185 [pii]\\nAID - 1031472 [pii]\\nAID - 10.3324/haematol.2018.188185 [doi]\\nPST - ppublish\\nSO  - Haematologica. 2018 Sep;103(9):1472-1483. doi: 10.3324/haematol.2018.188185. Epub  2018 Jun 7.\\n\\nPMID- 35131266\\nOWN - NLM\\nSTAT- Publisher\\nLR  - 20220208\\nIS  - 1083-351X (Electronic)\\nIS  - 0021-9258 (Linking)\\nDP  - 2022 Feb 4\\nTI  - The carboxyl-terminal region of SDCCAG8 comprises a functional module essential for  cilia formation as well as organ development and homeostasis.\\nPG  - 101686\\nLID - S0021-9258(22)00126-0 [pii]\\nLID - 10.1016/j.jbc.2022.101686 [doi]\\nAB  - In humans, ciliary dysfunction causes ciliopathies, which present as multiple organ  defects, including developmental and sensory abnormalities. Sdccag8 is a  centrosomal/basal body protein essential for proper cilia formation. Gene mutations  in SDCCAG8 have been found in patients with ciliopathies manifesting a broad  spectrum of symptoms, including hypogonadism. Among these mutations, several that  are predicted to truncate the SDCCAG8 carboxyl (C)-terminus are also associated with  such symptoms; however, the underlying mechanisms are poorly understood. In the  present study, we identified the Sdccag8 C-terminal region (Sdccag8-C) as a module  that interacts with the ciliopathy proteins, Ick/Cilk1 and Mak, which were shown to  be essential for the regulation of ciliary protein trafficking and cilia length in  mammals in our previous studies. We found that Sdccag8-C is essential for Sdccag8  localization to centrosomes and cilia formation in cultured cells. We then generated  a mouse mutant in which Sdccag8-C was truncated (Sdccag8(Î”C/Î”C) mice) using a  CRISPR-mediated stop codon knock-in strategy. In Sdccag8(Î”C/Î”C) mice, we observed  abnormalities in cilia formation and ciliopathy-like organ phenotypes, including  cleft palate, polydactyly, retinal degeneration, and cystic kidney, which partially  overlapped with those previously observed in Ick- and Mak-deficient mice.  Furthermore, Sdccag8(Î”C/Î”C) mice exhibited a defect in spermatogenesis, which was a  previously uncharacterized phenotype of Sdccag8 dysfunction. Together, these results  shed light on the molecular and pathological mechanisms underlying ciliopathies  observed in patients with SDCCAG8 mutations, and may advance our understanding of  protein-protein interaction networks involved in cilia development.\\nCI  - Copyright Â© 2022 The Authors. Published by Elsevier Inc. All rights reserved.\\nFAU - Tsutsumi, Ryotaro\\nAU  - Tsutsumi R\\nAD  - Laboratory for Molecular and Developmental Biology, Institute for Protein Research,  Osaka University, 3-2 Yamadaoka, Suita, Osaka, 565-0871, Japan.\\nFAU - Chaya, Taro\\nAU  - Chaya T\\nAD  - Laboratory for Molecular and Developmental Biology, Institute for Protein Research,  Osaka University, 3-2 Yamadaoka, Suita, Osaka, 565-0871, Japan.\\nFAU - Tsujii, Toshinori\\nAU  - Tsujii T\\nAD  - Laboratory for Molecular and Developmental Biology, Institute for Protein Research,  Osaka University, 3-2 Yamadaoka, Suita, Osaka, 565-0871, Japan.\\nFAU - Furukawa, Takahisa\\nAU  - Furukawa T\\nAD  - Laboratory for Molecular and Developmental Biology, Institute for Protein Research,  Osaka University, 3-2 Yamadaoka, Suita, Osaka, 565-0871, Japan. Electronic address:  takahisa.furukawa@protein.osaka-u.ac.jp.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20220204\\nPL  - United States\\nTA  - J Biol Chem\\nJT  - The Journal of biological chemistry\\nJID - 2985121R\\nSB  - IM\\nOTO - NOTNLM\\nOT  - CRISPR/Cas\\nOT  - cilia\\nOT  - intestinal cell kinase/ciliogenesis-associated kinase 1\\nOT  - kidney\\nOT  - male germ cell-associated kinase\\nOT  - mouse\\nOT  - retina\\nOT  - serologically defined colon cancer antigen 8\\nOT  - sonic hedgehog\\nOT  - spermatogenesis\\nCOIS- Conflict of interest The authors declare that they have no conflicts of interest  with the contents of this article.\\nEDAT- 2022/02/09 06:00\\nMHDA- 2022/02/09 06:00\\nCRDT- 2022/02/08 05:40\\nPHST- 2021/12/10 00:00 [received]\\nPHST- 2022/01/28 00:00 [revised]\\nPHST- 2022/02/01 00:00 [accepted]\\nPHST- 2022/02/08 05:40 [entrez]\\nPHST- 2022/02/09 06:00 [pubmed]\\nPHST- 2022/02/09 06:00 [medline]\\nAID - S0021-9258(22)00126-0 [pii]\\nAID - 10.1016/j.jbc.2022.101686 [doi]\\nPST - aheadofprint\\nSO  - J Biol Chem. 2022 Feb 4:101686. doi: 10.1016/j.jbc.2022.101686.\\n\\nPMID- 30633974\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190618\\nLR  - 20200930\\nIS  - 1096-7184 (Electronic)\\nIS  - 1096-7176 (Linking)\\nVI  - 52\\nDP  - 2019 Mar\\nTI  - Phage serine integrase-mediated genome engineering for efficient expression of  chemical biosynthetic pathway in gas-fermenting Clostridium ljungdahlii.\\nPG  - 293-302\\nLID - S1096-7176(18)30350-1 [pii]\\nLID - 10.1016/j.ymben.2019.01.005 [doi]\\nAB  - The real value of gas-fermenting clostridia, capable of using CO and CO(2), resides  in their potential of being developed into cell factories to produce various bulk  chemicals and fuels. This process requires rapid chromosomal integration of  heterologous chemical biosynthetic pathways, which is impeded by the absence of  genetic tools competent for efficient genome engineering in these anaerobes. Here,  we developed a phage serine integrase-mediated site-specific genome engineering  technique in Clostridium ljungdahlii, one of the major acetogenic gas-fermenting  microbes. Two heterologous phage attachment/integration (Att/Int) systems (from  Clostridium difficile and Streptomyces) were introduced into C. ljungdahlii and  proven to be highly active, achieving efficient chromosomal integration of a whole  donor vector via single-crossover recombination. Based on this, we further realized  markerless chromosomal integration of target DNA fragments through a \"dual integrase  cassette exchange\" (DICE) strategy with the assistance of the CRISPR-Cas9 editing  system. As a proof of concept, a butyric acid production pathway from Clostridium  acetobutylicum was integrated into the C. ljungdahlii genome without the  introduction of extra markers, enabling stable expression of the pathway genes. The  resulting engineered strain produced 1.01â€¯g/L of butyric acid within 3 days by  fermenting synthesis gas (CO(2)/CO). More importantly, the engineered strain showed  good genetic stability and maintained butyric acid production ability after  continuous subculturing. The system developed in this study overcomes the  deficiencies of currently available genetic tools in the chromosomal integration of  large DNA fragments (rapid, markerless and stable) in C. ljungdahlii, and may be  extended to other Clostridium species.\\nCI  - Copyright Â© 2019 International Metabolic Engineering Society. Published by Elsevier  Inc. All rights reserved.\\nFAU - Huang, He\\nAU  - Huang H\\nAD  - Key Laboratory of Synthetic Biology, CAS Center for Excellence in Molecular Plant  Sciences, Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of  Sciences, Shanghai 200032, China.\\nFAU - Chai, Changsheng\\nAU  - Chai C\\nAD  - Key Laboratory of Synthetic Biology, CAS Center for Excellence in Molecular Plant  Sciences, Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of  Sciences, Shanghai 200032, China.\\nFAU - Yang, Sheng\\nAU  - Yang S\\nAD  - Key Laboratory of Synthetic Biology, CAS Center for Excellence in Molecular Plant  Sciences, Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of  Sciences, Shanghai 200032, China; Jiangsu National Synergetic Innovation Center for  Advanced Materials, SICAM, 200 North Zhongshan Road, Nanjing 210009, China.\\nFAU - Jiang, Weihong\\nAU  - Jiang W\\nAD  - Key Laboratory of Synthetic Biology, CAS Center for Excellence in Molecular Plant  Sciences, Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of  Sciences, Shanghai 200032, China; Jiangsu National Synergetic Innovation Center for  Advanced Materials, SICAM, 200 North Zhongshan Road, Nanjing 210009, China.  Electronic address: whjiang@sibs.ac.cn.\\nFAU - Gu, Yang\\nAU  - Gu Y\\nAD  - Key Laboratory of Synthetic Biology, CAS Center for Excellence in Molecular Plant  Sciences, Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of  Sciences, Shanghai 200032, China. Electronic address: ygu02@sibs.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190108\\nPL  - Belgium\\nTA  - Metab Eng\\nJT  - Metabolic engineering\\nJID - 9815657\\nRN  - 0 (DNA, Bacterial)\\nRN  - 107-92-6 (Butyric Acid)\\nRN  - 452VLY9402 (Serine)\\nRN  - EC 2.7.7.- (Integrases)\\nSB  - IM\\nMH  - Attachment Sites, Microbiological/genetics\\nMH  - Bacteriophages/*enzymology/*genetics\\nMH  - Butyric Acid/metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Clostridium/*genetics/*metabolism\\nMH  - DNA, Bacterial/genetics\\nMH  - Fermentation\\nMH  - Genome, Bacterial/genetics\\nMH  - Integrases/*genetics/*metabolism\\nMH  - Metabolic Engineering/*methods\\nMH  - Metabolic Networks and Pathways/genetics\\nMH  - Plasmids/genetics\\nMH  - Serine/metabolism\\nOTO - NOTNLM\\nOT  - *Att/Int system\\nOT  - *Bacteriophage\\nOT  - *Butyric acid production\\nOT  - *Chromosomal integration\\nOT  - *Gas-fermenting clostridia\\nEDAT- 2019/01/12 06:00\\nMHDA- 2019/06/19 06:00\\nCRDT- 2019/01/12 06:00\\nPHST- 2018/09/12 00:00 [received]\\nPHST- 2018/12/19 00:00 [revised]\\nPHST- 2019/01/07 00:00 [accepted]\\nPHST- 2019/01/12 06:00 [pubmed]\\nPHST- 2019/06/19 06:00 [medline]\\nPHST- 2019/01/12 06:00 [entrez]\\nAID - S1096-7176(18)30350-1 [pii]\\nAID - 10.1016/j.ymben.2019.01.005 [doi]\\nPST - ppublish\\nSO  - Metab Eng. 2019 Mar;52:293-302. doi: 10.1016/j.ymben.2019.01.005. Epub 2019 Jan 8.\\n\\nPMID- 28179426\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170727\\nLR  - 20210205\\nIS  - 1083-351X (Electronic)\\nIS  - 0021-9258 (Print)\\nIS  - 0021-9258 (Linking)\\nVI  - 292\\nIP  - 10\\nDP  - 2017 Mar 10\\nTI  - Extracellular Signal-regulated Kinases (ERKs) Phosphorylate Lin28a Protein to  Modulate P19 Cell Proliferation and Differentiation.\\nPG  - 3970-3976\\nLID - 10.1074/jbc.C117.775122 [doi]\\nAB  - Lin28a, originally discovered in the nematode Caenorhabditis elegans and highly  conserved across species, is a well characterized regulator of let-7 microRNA  (miRNA) and is implicated in cell proliferation and pluripotency control. However,  little is known about how Lin28a function is modulated at the post-translational  level and thereby responds to major signaling pathways. Here we show that Lin28a is  directly phosphorylated by ERK1/2 kinases at Ser-200. By editing lin28a gene with  the CRISPR/Cas9-based method, we generated P19 mouse embryonic carcinoma stem cells  expressing Lin28a-S200A (phospho-deficient) and Lin28a-S200D (phospho-mimetic)  mutants, respectively, to study the functional impact of Ser-200 phosphorylation.  Lin28a-S200D-expressing cells, but not Lin28a-S200A-expressing or control P19  embryonic carcinoma cells, displayed impaired inhibition of let-7 miRNA and resulted  in decreased cyclin D1, whereas Lin28a-S200A knock-in cells expressed less let-7  miRNA, proliferated faster, and exhibited differentiation defect upon retinoic acid  induction. Therefore our results support that ERK kinase-mediated Lin28a  phosphorylation may be an important mechanism for pluripotent cells to facilitate  the escape from the self-renewal cycle and start the differentiation process.\\nCI  - Â© 2017 by The American Society for Biochemistry and Molecular Biology, Inc.\\nFAU - Liu, Xiangyuan\\nAU  - Liu X\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China.\\nFAU - Chen, Min\\nAU  - Chen M\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China.\\nFAU - Li, Long\\nAU  - Li L\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China.\\nFAU - Gong, Liyan\\nAU  - Gong L\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China.\\nFAU - Zhou, Hu\\nAU  - Zhou H\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China.\\nFAU - Gao, Daming\\nAU  - Gao D\\nAD  - From the CAS Key Laboratory of Systems Biology, CAS Center for Excellence in  Molecular Cell Science, Innovation Center for Cell Signaling Network, Shanghai  Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, University  of Chinese Academy of Sciences, 320 Yueyang Road, Shanghai 200031, China  dgao@sibcb.ac.cn.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170208\\nTA  - J Biol Chem\\nJT  - The Journal of biological chemistry\\nJID - 2985121R\\nRN  - 0 (LIN28B protein, human)\\nRN  - 0 (Lin-28 protein, mouse)\\nRN  - 0 (RNA-Binding Proteins)\\nRN  - EC 2.7.11.24 (Extracellular Signal-Regulated MAP Kinases)\\nRN  - EC 2.7.11.24 (Mitogen-Activated Protein Kinases)\\nSB  - IM\\nMH  - Animals\\nMH  - *Cell Differentiation\\nMH  - *Cell Proliferation\\nMH  - Embryonal Carcinoma Stem Cells/metabolism/*pathology\\nMH  - Extracellular Signal-Regulated MAP Kinases/*metabolism\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Mice\\nMH  - Mitogen-Activated Protein Kinases/*metabolism\\nMH  - Phosphorylation\\nMH  - RNA-Binding Proteins/genetics/*metabolism\\nMH  - Signal Transduction\\nPMC - PMC5354513\\nOTO - NOTNLM\\nOT  - *ERK kinases\\nOT  - *Lin28a\\nOT  - *differentiation\\nOT  - *extracellular signal-regulated kinase (ERK)\\nOT  - *mitogen-activated protein kinase (MAPK)\\nOT  - *phosphorylation\\nOT  - *proliferation\\nCOIS- The authors declare that they have no conflicts of interest with the contents of  this article\\nEDAT- 2017/02/10 06:00\\nMHDA- 2017/07/28 06:00\\nCRDT- 2017/02/10 06:00\\nPHST- 2017/01/03 00:00 [received]\\nPHST- 2017/02/06 00:00 [revised]\\nPHST- 2017/02/10 06:00 [pubmed]\\nPHST- 2017/07/28 06:00 [medline]\\nPHST- 2017/02/10 06:00 [entrez]\\nAID - S0021-9258(20)41373-0 [pii]\\nAID - C117.775122 [pii]\\nAID - 10.1074/jbc.C117.775122 [doi]\\nPST - ppublish\\nSO  - J Biol Chem. 2017 Mar 10;292(10):3970-3976. doi: 10.1074/jbc.C117.775122. Epub 2017  Feb 8.\\n\\nPMID- 29162967\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200929\\nIS  - 0973-2063 (Print)\\nIS  - 0973-2063 (Electronic)\\nIS  - 0973-2063 (Linking)\\nVI  - 13\\nIP  - 10\\nDP  - 2017\\nTI  - Lubricin: A Principal Modulator of the Psychoneuroendocrine - Osteoimmune  Interactome - Implications for Novel Treatments of Osteoarthritic Pathologies.\\nPG  - 343-346\\nLID - 10.6026/97320630013343 [doi]\\nAB  - Lubricin is a synovial glycoprotein that contributes to joint lubrication. We  propose the hypothesis that lubricin is a key modulator of the  psychoneuroendocrine-osteoimmune interactome, with important clinical relevance for  osteoarthritic pathologies. We consider a variety of neuroendocrine-immune factors,  including inflammatory cytokines and chemokines that may contribute to the  modulation of lubricin in rheumatic complications. Based on our preliminary  immunocytochemistry and fractal analysis data, and in the context of translational  research of modern healthcare, we propose that molecular lubricin gene expression  modification by means of the novel CRISPR/Cas system be considered for  osteoarthritic therapies.\\nFAU - Khakshooy, Allen\\nAU  - Khakshooy A\\nAD  - Laboratory of Human Psychoneuroendocrine-Osteoimmunology; School of Dentistry, UCLA  Center for the Health Sciences, Los Angeles, CA 90095-1668.\\nAD  - Rappaport Faculty of Medicine, Technion-Israel Institute of Technology, Haifa,  Israel 3109601.\\nFAU - Balenton, Nicole\\nAU  - Balenton N\\nAD  - Laboratory of Human Psychoneuroendocrine-Osteoimmunology; School of Dentistry, UCLA  Center for the Health Sciences, Los Angeles, CA 90095-1668.\\nAD  - Department of the Health Sciences, CSUN, Northridge, CA 91330.\\nFAU - Chiappelli, Francesco\\nAU  - Chiappelli F\\nAD  - Laboratory of Human Psychoneuroendocrine-Osteoimmunology; School of Dentistry, UCLA  Center for the Health Sciences, Los Angeles, CA 90095-1668.\\nAD  - Evidence-Based Decision Practice-Based Research Network, DGSO, Los Angeles, CA  91403.\\nAD  - Department of the Health Sciences, CSUN, Northridge, CA 91330.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20171031\\nTA  - Bioinformation\\nJT  - Bioinformation\\nJID - 101258255\\nPMC - PMC5680716\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Lubricin\\nOT  - interactome\\nOT  - osteoarthritis\\nOT  - osteoimmunology\\nOT  - psychoneuroendocrine-osteoimmunology\\nEDAT- 2017/11/23 06:00\\nMHDA- 2017/11/23 06:01\\nCRDT- 2017/11/23 06:00\\nPHST- 2017/10/05 00:00 [received]\\nPHST- 2017/10/05 00:00 [accepted]\\nPHST- 2017/11/23 06:00 [entrez]\\nPHST- 2017/11/23 06:00 [pubmed]\\nPHST- 2017/11/23 06:01 [medline]\\nAID - 97320630013343 [pii]\\nAID - 10.6026/97320630013343 [doi]\\nPST - epublish\\nSO  - Bioinformation. 2017 Oct 31;13(10):343-346. doi: 10.6026/97320630013343. eCollection  2017.\\n\\nPMID- 26649148\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nDCOM- 20151209\\nLR  - 20200930\\nIS  - 1944-3277 (Print)\\nIS  - 1944-3277 (Electronic)\\nIS  - 1944-3277 (Linking)\\nVI  - 10\\nDP  - 2015\\nTI  - Draft genome of Gemmata massiliana sp. nov, a water-borne Planctomycetes species  exhibiting two variants.\\nPG  - 120\\nLID - 10.1186/s40793-015-0103-0 [doi]\\nLID - 120\\nAB  - Gemmata massiliana is a new Planctomycetes bacterium isolated from a hospital water  network in France, using a new culture medium. It is an aerobic microorganism with  optimal growth at pHÂ\\xa08, at 30Â\\xa0Â°C and salinityâ€‰â‰¤â€‰1.25Â\\xa0% NaCl. G. massiliana is  resistant to Î²-lactam antibiotics, due to lack of peptidoglycan in its cell wall.G.  massiliana shares a 97Â\\xa0% 16S rRNA gene sequence similarity with the nearest species,  Gemmata obscuriglobus; and 99Â\\xa0% similarity with unnamed soil isolates. Its  9,249,437-bp genome consists in one chromosome and no detectable plasmid and has a  64.07Â\\xa0%Â\\xa0Gâ€‰+â€‰C content, 32.94Â\\xa0% of genes encoding for hypothetical proteins. The  genome contains an incomplete 19.6-kb phage sequence, 26 CRISPRs, 3 CAS and 15  clusters of secondary metabolites. G. massiliana genome increases knowledge of a  poorly known world of bacteria.\\nFAU - Aghnatios, Rita\\nAU  - Aghnatios R\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Cayrou, Caroline\\nAU  - Cayrou C\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Garibal, Marc\\nAU  - Garibal M\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Robert, Catherine\\nAU  - Robert C\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Azza, Said\\nAU  - Azza S\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Raoult, Didier\\nAU  - Raoult D\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nFAU - Drancourt, Michel\\nAU  - Drancourt M\\nAD  - Aix-Marseille UniversitÃ©, URMITE, UM63, CNRS7278, IRD198, Inserm 1095, FacultÃ© de  mÃ©decine, 27 Boulevard jean Moulin, 13385 Marseille, cedex 05, France.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20151207\\nTA  - Stand Genomic Sci\\nJT  - Standards in genomic sciences\\nJID - 101530505\\nPMC - PMC4672568\\nOTO - NOTNLM\\nOT  - Culture\\nOT  - Gemmata\\nOT  - Gemmata massiliana\\nOT  - Gemmata obscuriglobus\\nOT  - Genome\\nOT  - Hospital water network\\nOT  - Planctomycetes\\nEDAT- 2015/12/10 06:00\\nMHDA- 2015/12/10 06:01\\nCRDT- 2015/12/10 06:00\\nPHST- 2014/08/07 00:00 [received]\\nPHST- 2015/11/17 00:00 [accepted]\\nPHST- 2015/12/10 06:00 [entrez]\\nPHST- 2015/12/10 06:00 [pubmed]\\nPHST- 2015/12/10 06:01 [medline]\\nAID - 103 [pii]\\nAID - 10.1186/s40793-015-0103-0 [doi]\\nPST - epublish\\nSO  - Stand Genomic Sci. 2015 Dec 7;10:120. doi: 10.1186/s40793-015-0103-0. eCollection  2015.\\n\\nPMID- 28972157\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180116\\nLR  - 20210205\\nIS  - 1083-351X (Electronic)\\nIS  - 0021-9258 (Print)\\nIS  - 0021-9258 (Linking)\\nVI  - 292\\nIP  - 48\\nDP  - 2017 Dec 1\\nTI  - Transcriptome analysis reveals determinant stages controlling human embryonic stem  cell commitment to neuronal cells.\\nPG  - 19590-19604\\nLID - 10.1074/jbc.M117.796383 [doi]\\nAB  - Proper neural commitment is essential for ensuring the appropriate development of  the human brain and for preventing neurodevelopmental diseases such as autism  spectrum disorders, schizophrenia, and intellectual disorders. However, the  molecular mechanisms underlying the neural commitment in humans remain elusive.  Here, we report the establishment of a neural differentiation system based on human  embryonic stem cells (hESCs) and on comprehensive RNA sequencing analysis of  transcriptome dynamics during early hESC differentiation. Using weighted gene  co-expression network analysis, we reveal that the hESC neurodevelopmental  trajectory has five stages: pluripotency (day 0); differentiation initiation (days  2, 4, and 6); neural commitment (days 8-10); neural progenitor cell proliferation  (days 12, 14, and 16); and neuronal differentiation (days 18, 20, and 22). These  stages were characterized by unique module genes, which may recapitulate the early  human cortical development. Moreover, a comparison of our RNA-sequencing data with  several other transcriptome profiling datasets from mice and humans indicated that  Module 3 associated with the day 8-10 stage is a critical window of fate switch from  the pluripotency to the neural lineage. Interestingly, at this stage, no key  extrinsic signals were activated. In contrast, using CRISPR/Cas9-mediated gene  knockouts, we also found that intrinsic hub transcription factors, including the  schizophrenia-associated SIX3 gene and septo-optic dysplasia-related HESX1 gene, are  required to program hESC neural determination. Our results improve the understanding  of the mechanism of neural commitment in the human brain and may help elucidate the  etiology of human mental disorders and advance therapies for managing these  conditions.\\nCI  - Â© 2017 by The American Society for Biochemistry and Molecular Biology, Inc.\\nFAU - Li, Yuanyuan\\nAU  - Li Y\\nAD  - From the State Key Laboratory of Cell Biology, CAS Center for Excellence in  Molecular Cell Science, Shanghai Institute of Biochemistry and Cell Biology and.\\nFAU - Wang, Ran\\nAU  - Wang R\\nAD  - From the State Key Laboratory of Cell Biology, CAS Center for Excellence in  Molecular Cell Science, Shanghai Institute of Biochemistry and Cell Biology and.\\nFAU - Qiao, Nan\\nAU  - Qiao N\\nAD  - Chinese Academy of Sciences Key Laboratory of Computational Biology, Chinese Academy  of Sciences-Max Planck Partner Institute for Computational Biology, Shanghai  Institutes for Biological Sciences, Chinese Academy of Sciences, 320 Yue Yang Road,  Shanghai 200031.\\nFAU - Peng, Guangdun\\nAU  - Peng G\\nAD  - From the State Key Laboratory of Cell Biology, CAS Center for Excellence in  Molecular Cell Science, Shanghai Institute of Biochemistry and Cell Biology and.\\nFAU - Zhang, Ke\\nAU  - Zhang K\\nAD  - From the State Key Laboratory of Cell Biology, CAS Center for Excellence in  Molecular Cell Science, Shanghai Institute of Biochemistry and Cell Biology and.\\nFAU - Tang, Ke\\nAU  - Tang K\\nAD  - the Institute of Life Science, Nanchang University, Nanchang, Jiangxi 330031, and.\\nFAU - Han, Jing-Dong J\\nAU  - Han JJ\\nAD  - Chinese Academy of Sciences Key Laboratory of Computational Biology, Chinese Academy  of Sciences-Max Planck Partner Institute for Computational Biology, Shanghai  Institutes for Biological Sciences, Chinese Academy of Sciences, 320 Yue Yang Road,  Shanghai 200031.\\nFAU - Jing, Naihe\\nAU  - Jing N\\nAD  - From the State Key Laboratory of Cell Biology, CAS Center for Excellence in  Molecular Cell Science, Shanghai Institute of Biochemistry and Cell Biology and  njing@sibcb.ac.cn.\\nAD  - the School of Life Science and Technology, ShanghaiTech University, 100 Haike Road,  Shanghai 201210, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170926\\nTA  - J Biol Chem\\nJT  - The Journal of biological chemistry\\nJID - 2985121R\\nRN  - 0 (Eye Proteins)\\nRN  - 0 (HESX1 protein, human)\\nRN  - 0 (Homeodomain Proteins)\\nRN  - 0 (Nerve Tissue Proteins)\\nRN  - 0 (Sine oculis homeobox homolog 3 protein)\\nRN  - 0 (Transcription Factors)\\nSB  - IM\\nMH  - Cell Differentiation/*genetics\\nMH  - Embryonic Stem Cells/chemistry/*metabolism\\nMH  - Eye Proteins/physiology\\nMH  - Homeodomain Proteins/physiology\\nMH  - Humans\\nMH  - Nerve Tissue Proteins/physiology\\nMH  - Neurons/*cytology\\nMH  - Transcription Factors/genetics\\nMH  - *Transcriptome\\nPMC - PMC5712601\\nOTO - NOTNLM\\nOT  - *embryonic stem cell\\nOT  - *human\\nOT  - *neurodevelopment\\nOT  - *neurodifferentiation\\nOT  - *transcriptomics\\nCOIS- The authors declare that they have no conflicts of interest with the contents of  this article\\nEDAT- 2017/10/04 06:00\\nMHDA- 2018/01/18 06:00\\nCRDT- 2017/10/04 06:00\\nPHST- 2017/05/12 00:00 [received]\\nPHST- 2017/09/17 00:00 [revised]\\nPHST- 2017/10/04 06:00 [pubmed]\\nPHST- 2018/01/18 06:00 [medline]\\nPHST- 2017/10/04 06:00 [entrez]\\nAID - S0021-9258(20)32856-8 [pii]\\nAID - M117.796383 [pii]\\nAID - 10.1074/jbc.M117.796383 [doi]\\nPST - ppublish\\nSO  - J Biol Chem. 2017 Dec 1;292(48):19590-19604. doi: 10.1074/jbc.M117.796383. Epub 2017  Sep 26.\\n\\nPMID- 33317552\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210901\\nLR  - 20210901\\nIS  - 1475-2859 (Electronic)\\nIS  - 1475-2859 (Linking)\\nVI  - 19\\nIP  - 1\\nDP  - 2020 Dec 14\\nTI  - Manipulating the position of DNA expression cassettes using location tags fused to  dCas9 (Cas9-Lag) to improve metabolic pathway efficiency.\\nPG  - 229\\nLID - 10.1186/s12934-020-01496-w [doi]\\nLID - 229\\nAB  - BACKGROUND: Deactivated Cas9 (dCas9) led to significant improvement of  CRISPR/Cas9-based techniques because it can be fused with a variety of functional  groups to form diverse molecular devices, which can manipulate or modify target DNA  cassettes. One important metabolic engineering strategy is to localize the enzymes  in proximity of their substrates for improved catalytic efficiency. In this work, we  developed a novel molecular device to manipulate the cellular location of specific  DNA cassettes either on plasmids or on the chromosome, by fusing location tags to  dCas9 (Cas9-Lag), and applied the technique for synthetic biology applications.  Carotenoids like Î²-carotene serve as common intermediates for the synthesis of  derivative compounds, which are hydrophobic and usually accumulate in the membrane  compartment. RESULTS: Carotenoids like Î²-carotene serve as common intermediates for  the synthesis of derivative compounds, which are hydrophobic and usually accumulate  in the membrane components. To improve the functional expression of membrane-bound  enzymes and localize them in proximity to the substrates, Cas9-Lag was used to pull  plasmids or chromosomal DNA expressing carotenoid enzymes onto the cell membrane.  For this purpose, dCas9 was fused to the E. coli membrane docking tag GlpF, and gRNA  was designed to direct this fusion protein to the DNA expression cassettes. With  Cas9-Lag, the zeaxanthin and astaxanthin titer increased by 29.0% and 26.7%  respectively. Due to experimental limitations, the electron microscopy images of  cells expressing Cas9-Lag vaguely indicated that GlpF-Cas9 might have pulled the  target DNA cassettes in close proximity to membrane. Similarly, protein mass  spectrometry analysis of membrane proteins suggested an increased expression of  carotenoid-converting enzymes in the membrane components. CONCLUSION: This work  therefore provides a novel molecular device, Cas9-Lag, which was proved to increase  zeaxanthin and astaxanthin production and might be used to manipulate DNA cassette  location.\\nFAU - Xie, Qianwen\\nAU  - Xie Q\\nAD  - College of Biotechnology, Tianjin University of Science and Technology, Tianjin,  300457, P. R. China.\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China.\\nFAU - Li, Siwei\\nAU  - Li S\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China.\\nFAU - Zhao, Dongdong\\nAU  - Zhao D\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China.\\nFAU - Ye, Lijun\\nAU  - Ye L\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China.\\nFAU - Li, Qingyan\\nAU  - Li Q\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China.\\nFAU - Zhang, Xueli\\nAU  - Zhang X\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China. zhang_xl@tib.cas.cn.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China. zhang_xl@tib.cas.cn.\\nFAU - Zhu, Li\\nAU  - Zhu L\\nAD  - State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of  Biotechnology, Beijing, 100071, China. jewly54@126.com.\\nFAU - Bi, Changhao\\nAU  - Bi C\\nAUID- ORCID: 0000-0002-1940-8511\\nAD  - Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences, Tianjin,  300308, P. R. China. bi_ch@tib.cas.cn.\\nAD  - Key Laboratory of Systems Microbial Biotechnology, Chinese Academy of Sciences,  Tianjin, 300308, China. bi_ch@tib.cas.cn.\\nLA  - eng\\nGR  - 2018YFA0903700/The National Key Research and Development Program of China/\\nGR  - TSBICIP-KJGG-001/Tianjin Synthetic Biotechnology Innovation Capacity Improvement  Project/\\nGR  - 31770105/Innovative Research Group Project of the National Natural Science  Foundation of China/\\nPT  - Journal Article\\nDEP - 20201214\\nTA  - Microb Cell Fact\\nJT  - Microbial cell factories\\nJID - 101139812\\nRN  - 0 (Aquaporins)\\nRN  - 0 (DNA, Bacterial)\\nRN  - 0 (Escherichia coli Proteins)\\nRN  - 0 (Recombinant Fusion Proteins)\\nRN  - 0 (Xanthophylls)\\nRN  - 0 (Zeaxanthins)\\nRN  - 122463-91-6 (GlpF protein, E coli)\\nRN  - 36-88-4 (Carotenoids)\\nRN  - 8XPW32PR7I (astaxanthine)\\nRN  - EC 1.- (Mixed Function Oxygenases)\\nRN  - EC 1.13.- (Oxygenases)\\nRN  - EC 1.13.- (beta-carotene ketolase)\\nRN  - EC 1.14.13.- (beta-carotene hydroxylase)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nSB  - IM\\nMH  - Aquaporins/genetics/metabolism\\nMH  - CRISPR-Associated Protein 9/genetics/*metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Carotenoids/*metabolism\\nMH  - Cell Membrane/enzymology\\nMH  - Chromosomes, Bacterial/genetics/ultrastructure\\nMH  - DNA, Bacterial/genetics\\nMH  - Escherichia coli/*genetics/metabolism/ultrastructure\\nMH  - Escherichia coli Proteins/genetics/metabolism\\nMH  - *Metabolic Engineering\\nMH  - *Metabolic Networks and Pathways\\nMH  - Mixed Function Oxygenases/genetics/metabolism\\nMH  - Oxygenases/genetics/metabolism\\nMH  - Plasmids/genetics\\nMH  - Recombinant Fusion Proteins/metabolism\\nMH  - Xanthophylls/metabolism\\nMH  - Zeaxanthins/*biosynthesis\\nPMC - PMC7737257\\nOTO - NOTNLM\\nOT  - Astaxanthin\\nOT  - Carotenoids\\nOT  - Complex localization\\nOT  - Escherichia coli\\nOT  - dCas9\\nCOIS- The authors declare that they have no competing interests.\\nEDAT- 2020/12/16 06:00\\nMHDA- 2021/09/02 06:00\\nCRDT- 2020/12/15 05:40\\nPHST- 2020/09/26 00:00 [received]\\nPHST- 2020/12/05 00:00 [accepted]\\nPHST- 2020/12/15 05:40 [entrez]\\nPHST- 2020/12/16 06:00 [pubmed]\\nPHST- 2021/09/02 06:00 [medline]\\nAID - 10.1186/s12934-020-01496-w [pii]\\nAID - 1496 [pii]\\nAID - 10.1186/s12934-020-01496-w [doi]\\nPST - epublish\\nSO  - Microb Cell Fact. 2020 Dec 14;19(1):229. doi: 10.1186/s12934-020-01496-w.\\n\\nPMID- 27636589\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170206\\nLR  - 20210109\\nIS  - 1349-7006 (Electronic)\\nIS  - 1347-9032 (Print)\\nIS  - 1347-9032 (Linking)\\nVI  - 107\\nIP  - 12\\nDP  - 2016 Dec\\nTI  - Establishment of MAGEC2-knockout cells and functional investigation of MAGEC2 in  tumor cells.\\nPG  - 1888-1897\\nLID - 10.1111/cas.13082 [doi]\\nAB  - Cancer/testis antigen MAGEC2, a member of the type I melanoma-associated antigen  family, is expressed in a wide variety of cancer types but not in normal somatic  cells. MAGEC2 has long been recognized as a tumor-specific target, however, its  functions remain largely unknown. In this study, we established MAGEC2-knockout A375  melanoma cell lines using the CRISPR/Cas9 system. Seven clonal cell lines were  generated by using four single guide RNAs targeting the coding region of the MAGEC2  gene, which produced indels that abolished MAGEC2 protein expression. To identify  the differentially expressed protein profiles associated with MAGEC2 loss, isobaric  tag for relative quantitation-based comparative proteomics experiments were carried  out on the MAGEC2-knockcout and control A375 cells. Mining of the proteomics data  identified a total 224 (61.6% upregulated and 38.4% downregulated) proteins to be  significantly altered in expression level in MAGEC2-knockcout cells. Ingenuity  Pathway Analysis indicated that the significantly altered proteins were involved in  critical neoplasia-related biological functions such as cell death, proliferation,  and movement. Gene ontology analysis identified \"apoptosis signaling\" as the  top-most upregulated pathway associated with MAGEC2 loss. We showed that knockout or  knockdown of the MAGEC2 gene sensitized melanoma cells to tumor necrosis  factor-Î±-induced apoptosis. Interestingly, actin-based motility by Rho and RhoA  signaling, known to promote cell migration, were also identified as the top  downregulated pathways in MAGEC2-knockout A375 cells. In short, our study provides a  suitable cell model for exploring the biological functions of MAGEC2 in malignant  cells, and sheds light on the molecular pathway by which MAGEC2 promotes tumor  development.\\nCI  - Â© 2016 The Authors. Cancer Science published by John Wiley & Sons Australia, Ltd on  behalf of Japanese Cancer Association.\\nFAU - Wang, Jingjing\\nAU  - Wang J\\nAD  - Key Laboratory of Medical Immunology, Ministry of Health, Department of Immunology,  School of Basic Medical Sciences, Peking University Health Science Center, Beijing,  China.\\nFAU - Song, Xiao\\nAU  - Song X\\nAD  - Key Laboratory of Medical Immunology, Ministry of Health, Department of Immunology,  School of Basic Medical Sciences, Peking University Health Science Center, Beijing,  China.\\nFAU - Guo, Chengli\\nAU  - Guo C\\nAD  - Key Laboratory of Medical Immunology, Ministry of Health, Department of Immunology,  School of Basic Medical Sciences, Peking University Health Science Center, Beijing,  China.\\nFAU - Wang, Ying\\nAU  - Wang Y\\nAD  - Key Laboratory of Medical Immunology, Ministry of Health, Department of Immunology,  School of Basic Medical Sciences, Peking University Health Science Center, Beijing,  China.\\nFAU - Yin, Yanhui\\nAU  - Yin Y\\nAD  - Key Laboratory of Medical Immunology, Ministry of Health, Department of Immunology,  School of Basic Medical Sciences, Peking University Health Science Center, Beijing,  China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20161125\\nTA  - Cancer Sci\\nJT  - Cancer science\\nJID - 101168776\\nRN  - 0 (Antigens, Neoplasm)\\nRN  - 0 (MAGEC2 protein, human)\\nRN  - 0 (Neoplasm Proteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Tumor Necrosis Factor-alpha)\\nSB  - IM\\nMH  - Animals\\nMH  - Antigens, Neoplasm/*genetics/*metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Cell Line, Tumor\\nMH  - Disease Models, Animal\\nMH  - Gene Deletion\\nMH  - Gene Expression Regulation\\nMH  - Gene Knockout Techniques\\nMH  - Gene Regulatory Networks\\nMH  - Gene Targeting\\nMH  - Heterografts\\nMH  - Humans\\nMH  - Mice\\nMH  - Mice, Knockout\\nMH  - Neoplasm Proteins/*genetics/*metabolism\\nMH  - Protein Interaction Maps\\nMH  - Proteome\\nMH  - Proteomics/methods\\nMH  - Signal Transduction\\nMH  - Tumor Necrosis Factor-alpha/metabolism\\nPMC - PMC5198962\\nOTO - NOTNLM\\nOT  - Bioinformatics\\nOT  - CRISPR-Cas systems\\nOT  - MAGEC2\\nOT  - gene knockout techniques\\nOT  - proteomics\\nEDAT- 2016/09/17 06:00\\nMHDA- 2017/02/07 06:00\\nCRDT- 2016/09/17 06:00\\nPHST- 2016/05/23 00:00 [received]\\nPHST- 2016/09/10 00:00 [revised]\\nPHST- 2016/09/13 00:00 [accepted]\\nPHST- 2016/09/17 06:00 [pubmed]\\nPHST- 2017/02/07 06:00 [medline]\\nPHST- 2016/09/17 06:00 [entrez]\\nAID - CAS13082 [pii]\\nAID - 10.1111/cas.13082 [doi]\\nPST - ppublish\\nSO  - Cancer Sci. 2016 Dec;107(12):1888-1897. doi: 10.1111/cas.13082. Epub 2016 Nov 25.\\n\\nPMID- 30081823\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190219\\nLR  - 20190320\\nIS  - 1471-2229 (Electronic)\\nIS  - 1471-2229 (Linking)\\nVI  - 18\\nIP  - 1\\nDP  - 2018 Aug 6\\nTI  - Identification of a G2-like transcription factor, OsPHL3, functions as a negative  regulator of flowering in rice by co-expression and reverse genetic analysis.\\nPG  - 157\\nLID - 10.1186/s12870-018-1382-6 [doi]\\nLID - 157\\nAB  - BACKGROUND: Flowering time is a key trait for regional adaption and seed production  in rice (Oryza sativa L.). Forward and reverse genetic studies have characterized a  number of flowering-time genes. However, co-expression analysis has not been used to  identify the flowering-time genes. RESULTS: We predicted a G2-like family  transcription factor, OsPHL3, by co-expression networks analysis with photoperiodic  flowering pathway genes. OsPHL3 contains a MYB-CC domain, and was localized in the  nucleus with transcriptional activation potential. OsPHL3 was mainly expressed in  the leaves and exhibited a circadian rhythmic expression pattern. Rice lines  overexpressing OsPHL3 showed a delayed flowering time in the genetic background of  TP309 under both long-day (Beijing) and short-day (Hainan) conditions. By contrast,  the knockout rice lines of OsPHL3 by CRISPR/Cas9 technology promoted flowering time  regardless of genetic backgrounds (i.e. Nipponbare and TP309) or day length. Further  analysis indicated that OsPHL3 delayed flowering time by down-regulating the  expression of Hd3a and RFT1 through promoting Hd1 under long-day conditions (LDs),  or suppressing Ehd1/Hd1 under short-day conditions (SDs). CONCLUSIONS: Our results  suggested that co-expression analysis is a useful strategy for identifying novel  flowering-time genes in rice.\\nFAU - Zeng, Liping\\nAU  - Zeng L\\nAD  - CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of  Genomics, Chinese Academy of Sciences, NO.1 Beichen West Road, Chaoyang District,  Beijing, 100101, China.\\nAD  - University of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan  District, Beijing, 100049, China.\\nFAU - Liu, Xue\\nAU  - Liu X\\nAD  - CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of  Genomics, Chinese Academy of Sciences, NO.1 Beichen West Road, Chaoyang District,  Beijing, 100101, China.\\nFAU - Zhou, Zhuangzhi\\nAU  - Zhou Z\\nAD  - State Key Laboratory of Plant Genomics and National Center For Plant Gene Research,  Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, NO.1  Beichen West Road, Chaoyang District, Beijing, 100101, China.\\nFAU - Li, Dayong\\nAU  - Li D\\nAD  - State Key Laboratory of Plant Genomics and National Center For Plant Gene Research,  Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, NO.1  Beichen West Road, Chaoyang District, Beijing, 100101, China.\\nFAU - Zhao, Xianfeng\\nAU  - Zhao X\\nAD  - State Key Laboratory of Plant Genomics and National Center For Plant Gene Research,  Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, NO.1  Beichen West Road, Chaoyang District, Beijing, 100101, China.\\nFAU - Zhu, Lihuang\\nAU  - Zhu L\\nAD  - State Key Laboratory of Plant Genomics and National Center For Plant Gene Research,  Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, NO.1  Beichen West Road, Chaoyang District, Beijing, 100101, China.\\nFAU - Luo, Yingfeng\\nAU  - Luo Y\\nAD  - CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of  Genomics, Chinese Academy of Sciences, NO.1 Beichen West Road, Chaoyang District,  Beijing, 100101, China. luoyf@big.ac.cn.\\nAD  - University of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan  District, Beijing, 100049, China. luoyf@big.ac.cn.\\nFAU - Hu, Songnian\\nAU  - Hu S\\nAD  - CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of  Genomics, Chinese Academy of Sciences, NO.1 Beichen West Road, Chaoyang District,  Beijing, 100101, China. husn@big.ac.cn.\\nAD  - University of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan  District, Beijing, 100049, China. husn@big.ac.cn.\\nLA  - eng\\nGR  - 31000561/TheÂ\\xa0National Natural Science Foundation of China/\\nGR  - 31771473/TheÂ\\xa0National Natural Science Foundation of China/\\nGR  - 2017140/Youth Innovation Promotion Association of the Chinese Academy of Sciences/\\nGR  - XDA08020102/Strategic Priority Research Program of the Chinese Academy of Sciences/\\nPT  - Journal Article\\nDEP - 20180806\\nTA  - BMC Plant Biol\\nJT  - BMC plant biology\\nJID - 100967807\\nRN  - 0 (Plant Proteins)\\nRN  - 0 (Transcription Factors)\\nSB  - IM\\nMH  - Flowers/*genetics/growth & development\\nMH  - Gene Knockdown Techniques\\nMH  - Oryza/*genetics/growth & development/metabolism\\nMH  - Photoperiod\\nMH  - Phylogeny\\nMH  - Plant Proteins/*genetics/physiology\\nMH  - Real-Time Polymerase Chain Reaction\\nMH  - Reverse Genetics\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Transcription Factors/*genetics/physiology\\nMH  - Transcriptome\\nPMC - PMC6091178\\nOTO - NOTNLM\\nOT  - Co-expression\\nOT  - Flowering time\\nOT  - G2-like\\nOT  - MYB-CC transcription factor\\nOT  - OsPHL3\\nOT  - Rice\\nCOIS- The experimental research on plants, including collection of plant material, was  complied with the institutional, national, or international guidelines. And field  studies were conducted in accordance with local legislation. No applicable. The  authors declare that they have no competing interests. Springer Nature remains  neutral with regard to jurisdictional claims in published maps and institutional  affiliations.\\nEDAT- 2018/08/08 06:00\\nMHDA- 2019/03/21 06:00\\nCRDT- 2018/08/08 06:00\\nPHST- 2018/03/30 00:00 [received]\\nPHST- 2018/08/01 00:00 [accepted]\\nPHST- 2018/08/08 06:00 [entrez]\\nPHST- 2018/08/08 06:00 [pubmed]\\nPHST- 2019/03/21 06:00 [medline]\\nAID - 10.1186/s12870-018-1382-6 [pii]\\nAID - 1382 [pii]\\nAID - 10.1186/s12870-018-1382-6 [doi]\\nPST - epublish\\nSO  - BMC Plant Biol. 2018 Aug 6;18(1):157. doi: 10.1186/s12870-018-1382-6.\\n']\n"
     ]
    }
   ],
   "source": [
    "print(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "e6399321",
   "metadata": {},
   "outputs": [],
   "source": [
    "country = re.findall(r'[,]+[\\s]+[\\w]+[.=$]|[,]+[\\s]+[\\w]+[\\s]+[\\w]+[.=$]', my_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "141df177",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1905"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(country)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "8b8d03d8",
   "metadata": {},
   "outputs": [],
   "source": [
    "unique_country = list(set(country))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "26c053e5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[', France.',\n",
       " ', and putrescine.',\n",
       " ',  Korea.',\n",
       " ', 60.',\n",
       " ', and  Itgb3bp.',\n",
       " ', and J.',\n",
       " ', United  States.',\n",
       " ', i.',\n",
       " ', Hungary.',\n",
       " ', China.',\n",
       " ', Co.',\n",
       " ', Raleigh 27695.',\n",
       " ', Tex.',\n",
       " ',  P.',\n",
       " ', USA.',\n",
       " ',  Russia.',\n",
       " ', p.',\n",
       " ',  India.',\n",
       " ',  Canada.',\n",
       " ', ASCR v.',\n",
       " ', New York.',\n",
       " ', Belgium.',\n",
       " ', Luxembourg.',\n",
       " ', Poland.',\n",
       " ', CA 91330.',\n",
       " ', and PGR.',\n",
       " ',  USA.',\n",
       " ', e.',\n",
       " ',  Japan.',\n",
       " ',  South Africa.',\n",
       " ', and aging.',\n",
       " ', Tours France.',\n",
       " ', Germany.',\n",
       " ', and peroxidases.',\n",
       " ', Weinheim.',\n",
       " ', MD 20892.',\n",
       " ', USA wesley.',\n",
       " ', Massachusetts.',\n",
       " ', and Deletion.',\n",
       " ', including mammals.',\n",
       " ', Maryland.',\n",
       " ',  respectively.',\n",
       " ', being D.',\n",
       " ', Australia.',\n",
       " ', TX 78712.',\n",
       " ',  Ireland.',\n",
       " ', PR China.',\n",
       " ', Canada.',\n",
       " ', and biotechnology.',\n",
       " ',  Iran.',\n",
       " ',  Spain.',\n",
       " ', Cat.',\n",
       " ', etc.',\n",
       " ', United States.',\n",
       " ', Gp2.',\n",
       " ', Pakistan.',\n",
       " ', USA  sophie.',\n",
       " ',  Australia.',\n",
       " ', Univ.',\n",
       " ', and rice.',\n",
       " ', Sweden.',\n",
       " ', CA  91403.',\n",
       " ',  Shanghai 200031.',\n",
       " ', degradation.',\n",
       " ', respectively.',\n",
       " ', Italy.',\n",
       " ', D.',\n",
       " ', NO.',\n",
       " ', Denmark.',\n",
       " ', Guangzhou China.',\n",
       " ', Austria.',\n",
       " ',  St.',\n",
       " ', robust logic.',\n",
       " ', and M.',\n",
       " ',  Portugal.',\n",
       " ', India.',\n",
       " ', NY  10016.',\n",
       " ', South Africa.',\n",
       " ', U.',\n",
       " ', The Netherlands.',\n",
       " ', New Zealand.',\n",
       " ', 76.',\n",
       " ', S.',\n",
       " ', Czech Republic.',\n",
       " ', Israel.',\n",
       " ', and medicine.',\n",
       " ', Ireland.',\n",
       " ',  Massachusetts.',\n",
       " ', Sweden gunnar.',\n",
       " ', Colorado.',\n",
       " ', Malta.',\n",
       " ', Dr.',\n",
       " ', Japan.',\n",
       " ', UK.',\n",
       " ', Russia.',\n",
       " ', including hypogonadism.',\n",
       " ', Switzerland.',\n",
       " ', 32.',\n",
       " ', B.',\n",
       " ',  and movement.',\n",
       " ', Egypt.',\n",
       " ', Portugal.',\n",
       " ',  UK.',\n",
       " ', Korea.',\n",
       " ', and 27.',\n",
       " ',  Germany.',\n",
       " ', Fig.',\n",
       " ', United Kingdom.',\n",
       " ', P.',\n",
       " ', a 1.',\n",
       " ', v.',\n",
       " ', Germany  klaus.',\n",
       " ', Slovakia.',\n",
       " ', Czech  Republic.',\n",
       " ',  China.',\n",
       " ', Av.',\n",
       " ',  Bangladesh.',\n",
       " ',  Prof.',\n",
       " ', 5.',\n",
       " ', Inc.',\n",
       " ', Govt.',\n",
       " ', 67.',\n",
       " ', PR  China.',\n",
       " ', ToursFrance.',\n",
       " ',  Israel 3109601.',\n",
       " ', Alabama.',\n",
       " ', adaptive immunity.',\n",
       " ', USA  wesley.',\n",
       " ', and Itgb3bp.',\n",
       " ', Brazil.',\n",
       " ', Spianata S.',\n",
       " ',  Pennsylvania.',\n",
       " ',  Taiwan.',\n",
       " ', New  Zealand.',\n",
       " ',  Alabama.',\n",
       " ', No.',\n",
       " ', The  Netherlands.',\n",
       " ', N.',\n",
       " ', Indonesia.',\n",
       " ', Genetalks Inc.',\n",
       " ', Iran.',\n",
       " ', product titres.',\n",
       " ', Thailand.',\n",
       " ', and.',\n",
       " ', Spain.',\n",
       " ',  Austria.',\n",
       " ', TX 77030.',\n",
       " ', genetic etiology.',\n",
       " ', Washington.',\n",
       " ', 2.',\n",
       " ',  France.',\n",
       " ', 57.']"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "unique_country"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "bababe68",
   "metadata": {},
   "outputs": [],
   "source": [
    "paises=unique_country"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "90834d6c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[', France.',\n",
       " ', and putrescine.',\n",
       " ',  Korea.',\n",
       " ', 60.',\n",
       " ', and  Itgb3bp.',\n",
       " ', and J.',\n",
       " ', United  States.',\n",
       " ', i.',\n",
       " ', Hungary.',\n",
       " ', China.',\n",
       " ', Co.',\n",
       " ', Raleigh 27695.',\n",
       " ', Tex.',\n",
       " ',  P.',\n",
       " ', USA.',\n",
       " ',  Russia.',\n",
       " ', p.',\n",
       " ',  India.',\n",
       " ',  Canada.',\n",
       " ', ASCR v.',\n",
       " ', New York.',\n",
       " ', Belgium.',\n",
       " ', Luxembourg.',\n",
       " ', Poland.',\n",
       " ', CA 91330.',\n",
       " ', and PGR.',\n",
       " ',  USA.',\n",
       " ', e.',\n",
       " ',  Japan.',\n",
       " ',  South Africa.',\n",
       " ', and aging.',\n",
       " ', Tours France.',\n",
       " ', Germany.',\n",
       " ', and peroxidases.',\n",
       " ', Weinheim.',\n",
       " ', MD 20892.',\n",
       " ', USA wesley.',\n",
       " ', Massachusetts.',\n",
       " ', and Deletion.',\n",
       " ', including mammals.',\n",
       " ', Maryland.',\n",
       " ',  respectively.',\n",
       " ', being D.',\n",
       " ', Australia.',\n",
       " ', TX 78712.',\n",
       " ',  Ireland.',\n",
       " ', PR China.',\n",
       " ', Canada.',\n",
       " ', and biotechnology.',\n",
       " ',  Iran.',\n",
       " ',  Spain.',\n",
       " ', Cat.',\n",
       " ', etc.',\n",
       " ', United States.',\n",
       " ', Gp2.',\n",
       " ', Pakistan.',\n",
       " ', USA  sophie.',\n",
       " ',  Australia.',\n",
       " ', Univ.',\n",
       " ', and rice.',\n",
       " ', Sweden.',\n",
       " ', CA  91403.',\n",
       " ',  Shanghai 200031.',\n",
       " ', degradation.',\n",
       " ', respectively.',\n",
       " ', Italy.',\n",
       " ', D.',\n",
       " ', NO.',\n",
       " ', Denmark.',\n",
       " ', Guangzhou China.',\n",
       " ', Austria.',\n",
       " ',  St.',\n",
       " ', robust logic.',\n",
       " ', and M.',\n",
       " ',  Portugal.',\n",
       " ', India.',\n",
       " ', NY  10016.',\n",
       " ', South Africa.',\n",
       " ', U.',\n",
       " ', The Netherlands.',\n",
       " ', New Zealand.',\n",
       " ', 76.',\n",
       " ', S.',\n",
       " ', Czech Republic.',\n",
       " ', Israel.',\n",
       " ', and medicine.',\n",
       " ', Ireland.',\n",
       " ',  Massachusetts.',\n",
       " ', Sweden gunnar.',\n",
       " ', Colorado.',\n",
       " ', Malta.',\n",
       " ', Dr.',\n",
       " ', Japan.',\n",
       " ', UK.',\n",
       " ', Russia.',\n",
       " ', including hypogonadism.',\n",
       " ', Switzerland.',\n",
       " ', 32.',\n",
       " ', B.',\n",
       " ',  and movement.',\n",
       " ', Egypt.',\n",
       " ', Portugal.',\n",
       " ',  UK.',\n",
       " ', Korea.',\n",
       " ', and 27.',\n",
       " ',  Germany.',\n",
       " ', Fig.',\n",
       " ', United Kingdom.',\n",
       " ', P.',\n",
       " ', a 1.',\n",
       " ', v.',\n",
       " ', Germany  klaus.',\n",
       " ', Slovakia.',\n",
       " ', Czech  Republic.',\n",
       " ',  China.',\n",
       " ', Av.',\n",
       " ',  Bangladesh.',\n",
       " ',  Prof.',\n",
       " ', 5.',\n",
       " ', Inc.',\n",
       " ', Govt.',\n",
       " ', 67.',\n",
       " ', PR  China.',\n",
       " ', ToursFrance.',\n",
       " ',  Israel 3109601.',\n",
       " ', Alabama.',\n",
       " ', adaptive immunity.',\n",
       " ', USA  wesley.',\n",
       " ', and Itgb3bp.',\n",
       " ', Brazil.',\n",
       " ', Spianata S.',\n",
       " ',  Pennsylvania.',\n",
       " ',  Taiwan.',\n",
       " ', New  Zealand.',\n",
       " ',  Alabama.',\n",
       " ', No.',\n",
       " ', The  Netherlands.',\n",
       " ', N.',\n",
       " ', Indonesia.',\n",
       " ', Genetalks Inc.',\n",
       " ', Iran.',\n",
       " ', product titres.',\n",
       " ', Thailand.',\n",
       " ', and.',\n",
       " ', Spain.',\n",
       " ',  Austria.',\n",
       " ', TX 77030.',\n",
       " ', genetic etiology.',\n",
       " ', Washington.',\n",
       " ', 2.',\n",
       " ',  France.',\n",
       " ', 57.']"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "paises"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "8e797ed1",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "3d9f400e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>, France.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>, and putrescine.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>,  Korea.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>, 60.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>, and  Itgb3bp.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>, genetic etiology.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>, Washington.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>, 2.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>,  France.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>, 57.</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>152 rows × 1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       0\n",
       "0              , France.\n",
       "1      , and putrescine.\n",
       "2              ,  Korea.\n",
       "3                  , 60.\n",
       "4        , and  Itgb3bp.\n",
       "..                   ...\n",
       "147  , genetic etiology.\n",
       "148        , Washington.\n",
       "149                 , 2.\n",
       "150           ,  France.\n",
       "151                , 57.\n",
       "\n",
       "[152 rows x 1 columns]"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "excel=pd.DataFrame(paises)\n",
    "excel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "03834f34",
   "metadata": {},
   "outputs": [],
   "source": [
    "excel.to_excel('data_excel.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "a194a289",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "c7a30507",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "filename = 'data_excel.csv'\n",
    "data = pandas.read_csv(filename, header=0)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "1efb9a7b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Unnamed: 0                0\n",
      "0            1        Australia\n",
      "1            2          Austria\n",
      "2            3       Bangladesh\n",
      "3            4          Belgium\n",
      "4            5           Canada\n",
      "5            6            China\n",
      "6            7   Czech Republic\n",
      "7            8          Denmark\n",
      "8            9            Egypt\n",
      "9           10           France\n",
      "10          11          Germany\n",
      "11          12          Hungary\n",
      "12          13            India\n",
      "13          14        Indonesia\n",
      "14          15             Iran\n",
      "15          16          Ireland\n",
      "16          17           Israel\n",
      "17          18            Japan\n",
      "18          19            Korea\n",
      "19          20            Malta\n",
      "20          21      New Zealand\n",
      "21          22         Pakistan\n",
      "22          23           Poland\n",
      "23          24         Portugal\n",
      "24          25           Russia\n",
      "25          26         Slovakia\n",
      "26          27     South Africa\n",
      "27          28            Spain\n",
      "28          29            Spain\n",
      "29          30           Sweden\n",
      "30          31      Switzerland\n",
      "31          32           Taiwan\n",
      "32          33  The Netherlands\n",
      "33          34               UK\n",
      "34          35              USA\n"
     ]
    }
   ],
   "source": [
    "print(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6bc4101b",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
